{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Identifiers.org resource metadata exploration"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Step 1\n",
    "\n",
    "Get dataset."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 684 namespaces\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "\n",
    "get_all_dataset_url = \"https://registry.api.identifiers.org/restApi/namespaces?size=1000\"\n",
    "dataset_request = requests.get(get_all_dataset_url)\n",
    "dataset = dataset_request.json()\n",
    "dataset = dataset[\"_embedded\"][\"namespaces\"]\n",
    "\n",
    "print(f\"Found {len(dataset)} namespaces\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Step 2\n",
    "Iterate namespaces in dataset and get metadata from them."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 273,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "* [chebi] has a special LUI.\n",
      "* [go] has a special LUI.\n",
      "* [sbo] has a special LUI.\n",
      "* [mgi] has a special LUI.\n",
      "* [eco] has a special LUI.\n",
      "* [mod] has a special LUI.\n",
      "* [mge] has a special LUI.\n",
      "* [fma] has a special LUI.\n",
      "* [so] has a special LUI.\n",
      "* [psimi] has a special LUI.\n",
      "* [cl] has a special LUI.\n",
      "* [bto] has a special LUI.\n",
      "* [pato] has a special LUI.\n",
      "* [uo] has a special LUI.\n",
      "* [pr] has a special LUI.\n",
      "* [doid] has a special LUI.\n",
      "* [cco] has a special LUI.\n",
      "* [pw] has a special LUI.\n",
      "* [po] has a special LUI.\n",
      "* [vario] has a special LUI.\n",
      "* [ma] has a special LUI.\n",
      "* [uberon] has a special LUI.\n",
      "* [gramene.growthstage] has a special LUI.\n",
      "* [eo] has a special LUI.\n",
      "* [swisslipid] has a special LUI.\n",
      "* [hp] has a special LUI.\n",
      "* [mp] has a special LUI.\n",
      "* [ms] has a special LUI.\n",
      "* [envo] has a special LUI.\n",
      "* [ark] has a special LUI.\n",
      "* [mir] has a special LUI.\n",
      "* [mzspec] has a special LUI.\n",
      "* [nmr] has a special LUI.\n",
      "* [swh] has a special LUI.\n",
      "* [foodon] has a special LUI.\n",
      "* [did] has a special LUI.\n",
      "* [ocid] has a special LUI.\n",
      "* [rrid] has a special LUI.\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/CHEBI:36927: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/uniprot:P0DP23: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ec-code:1.1.1.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ensembl:ENSG00000139618: {'metadata': [{'identifier': 'ENSG00000139618', '@context': 'http://schema.org', '@type': 'DataRecord', 'mainEntity': {'description': 'BRCA2 DNA repair associated [Source:HGNC Symbol;Acc:HGNC:1101]', 'identifier': 'ENSG00000139618', '@type': 'Gene', 'isPartOfBioChemEntity': {'@type': 'BioChemEntity', 'name': 'Chromosome 13'}, 'isPartOf': {'name': 'Homo sapiens', 'codeCategory': {'url': 'http://purl.bioontology.org/ontology/NCBITAXON/9606', 'codeValue': '9606', 'sameAs': 'http://purl.uniprot.org/taxonomy/9606', '@type': 'CategoryCode', 'inCodeSet': {'@type': 'CategoryCodeSet', 'name': 'NCBI taxon'}}, '@type': 'BioChemEntity', 'alternateName': 'Human'}, 'name': 'BRCA2'}, 'isPartOf': 'Ensembl Human Gene Set'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/miriam.collection:MIR:00000008: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/taxonomy:9606: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biomodels.db:BIOMD0000000048: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/intact:EBI-2307691: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/interpro:IPR000100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.pathway:hsa00620: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/icd:C34: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pirsf:PIRSF000100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pubmed:16333295: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mim:603903: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.compound:C12345: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.reaction:R00100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/doi:10.1038/nbt1156: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/reactome:R-HSA-201451: {'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'https://reactome.org/', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://reactome.org/content/query?q={term}', 'query-input': 'required name=term'}}, {'@context': 'http://schema.org', '@type': 'Organization', 'url': 'https://reactome.org', 'logo': 'https://reactome.org/templates/favourite/images/logo/logo.png', 'email': 'help@reactome.org'}, {'name': 'Signaling by BMP', 'description': 'Bone morphogenetic proteins (BMPs) have many biological activities in various tissues, including bone, cartilage, blood vessels, heart, kidney, neurons, liver and lung. They are members of the Transforming growth factor-Beta (TGFB) family. They bind to type II and type I serine-threonine kinase receptors, which transduce signals through SMAD and non-SMAD signalling pathways. BMP signalling is linked to a wide variety of clinical disorders, including vascular diseases, skeletal diseases and cancer. BMPs typically activate BMP type I receptors and signal via SMAD1, 5 and 8. They can be classified into several subgroups, including the BMP2/4 group, the BMP5-8 osteogenic protein-1 (OP1) group, the growth and differentiation factor (GDF) 5-7 group and the BMP9/10 group. Most of the proteins of the BMP2/4, OP1 and BMP9/10 groups induce formation of bone and cartilage tissues in vivo, while the GDF5-7 group induce cartilage and tendon-like, but not bone-like, tissues (Miyazono et al. 2010). Members of the TGFB family bind to two types of serine-threonine kinase receptors, type I and type II (Massagu√© 2012). BMPs can bind type I receptors in the absence of type II receptors, but both types are required for signal transduction. The presence of both types dramatically increases binding affinity (Rozenweig et al. 1995). The type II receptor kinase transphosphorylates the type I receptor, which transmits specific intracellular signals. Type I and type II receptors share similar structural properties, comprised of a relatively short extracellular domain, a single membrane-spanning domain and an intracellular domain containing a serine-threonine kinase domain. Seven receptors, collectively referred to as the Activin receptor-like kinases (ALK), have been identified as type I receptors for the TGFB family in mammals. ALKs are classified into three groups based on their structure and function, the BMPRI group (Bone morphogenetic protein receptor type-1A, ALK3, BMPR1A and Bone morphogenetic protein receptor type-1B, ALK6, BMPR1B), the ALK1 group (Serine/threonine-protein kinase receptor R3, ALK1, ACVRL1 and Activin receptor type-1, ALK2, ACVR1) and the TBetaR1 group (Activin receptor type-1B, ALK4, ACVR1B and  TGF-beta receptor type-1, ALK5, TGFBR1 and Activin receptor type-1C, ALK7, ACVR1C) (Kawabata et al. 1998). ALK1 group and BMPRI group activate SMAD1/5/8 and transduce similar intracellular signals. The TBetaR1 group activate SMAD2/3. BMPR1A and ACVR1 are widely expressed. BMPR1B shows a more restricted expression profile. ACVRL1 is limited to endothelial cells and a few other cell types. The binding specificities of BMPs to type I receptors is affected by the type II receptors that are present (Yu et al. 2005). Typically, BMP2 and BMP4 bind to BMPR1A and BMPR1B (ten Dijke et al. 1994). BMP6 and BMP7 bind strongly to ACVR1 and weakly to BMPR1B. Growth/differentiation factor 5 (BMP14, GDF5) preferentially binds to BMPR1B, but not to other type I receptors (Nishitoh et al. 1995). BMP9 and BMP10 bind to ACVRL1 and ACVRL (Scharpfenecker et al. 2007). BMP type I receptors are shared by other members of the TGFB family. Three receptors, Bone morphogenetic protein receptor type-2 (BMPR2), Activin receptor type-2A (ACVR2A) and Activin receptor type-2B (ACVR2B) are the type II receptors for mammalian BMPs. They are widely expressed in various tissues. BMPR2 is specific for BMPs, whereas ACVR2A and ACVR2B are shared with activins and myostatin. BMP binding and signalling can be affected by coreceptors. Glycosylphosphatidylinositol (GPI)-anchored proteins of the repulsive guidance molecule (RGM) family, including RGMA, RGMB (DRAGON) and Hemojuvelin (HFE2, RGMC) are coreceptors for BMP2 and BMP4, enhancing signaling (Samad et al. 2005, Babitt et al. 2005, 2006). They interact with BMP type I and/or type II receptors and bind BMP2 and BMP4, but not BMP7 or TGFB1. BMP2/4 signalling normally involves BMPR2, not ACVR2A or ACVR2B. Cells transfected with RGMA use both BMPR2 and ACVR2A for BMP-2/4 signalling, suggesting that RGMA facilitates the use of ACVR2A by BMP2/4 (Xia et al. 2007). Endoglin (ENG) is a transmembrane protein expressed in proliferating endothelial cells. It binds various ligands including TGFB1/3, Activin-A and BMP2/7 (Barbara et al. 1999). It inhibits TGFB-induced responses and enhances BMP7-induced responses (Scherner et al. 2007). Mutations in ENG result in hereditary haemorrhagic telangiectasia (HHT1), also known as OslerWeberRendu disease, while mutations in ACVRL1 lead to HHT2, suggesting that they act in a common signalling pathway (McAllister et al. 1994, Johnson et al. 1996). BMP2 is a dimeric protein, having two receptor-binding motifs. One is a high-affinity binding site for BMPR1A, the other is a low-affinity binding site for BMPR2 (Kirsch et al. 2000). In the absence of ligand stimulation, small fractions of type II and type I receptors are present as preexisting homodimers and heterodimers on the cell surface. Ligand-binding increases oligomerization. The intracellular domains of type I receptors have a characteristic GS domain (glycine and serine-rich domain) located N-terminal to the serine-threonine kinase domains. Type II receptor kinases are constitutively active in the absence of ligand. Upon ligand binding, the type II receptor kinase phosphorylates the GS domain of the type I receptor, a critical event in signal transduction by the serine/threonine kinase receptors (Miyazono et al. 2010). Activation of the TGFBR1 receptor has been studied in detail. The inactive conformation is maintained by interaction between the GS domain, the N-terminal lobe and the activation loop of the kinase (Huse et al. 1999). When the GS domain is phosphorylated by the type II receptor kinase, the TGFBR1 kinase is converted to an active conformation. Mutations of Thr-204 in TGFBR1 and the corresponding Gln in BMP type I receptors lead to their constitutive activation. The L45 loop, in the kinase domain of  type I receptors, specifically interacts with receptor-regulated Smads (R-Smads). Neurotrophic tyrosine kinase receptor type 3 (NT-3 growth factor receptor, TrkC, NTRK3) directly binds BMPR2, interfereing with its interaction with BMPR1A, which inhibits downstream signalling (Jin et al. 2007). Tyrosine-protein kinase transmembrane receptor ROR2 and BMPR1B form a heteromeric complex in a ligand independent fashion that modulatesGDF5-BMPR1B signalling by inhibition of Smad1/5 signalling (Sammar et al. 2004). Type I receptor kinases activated by the type II receptor kinases, phosphorylate R-Smads. R-Smads then form a complex with common-partner Smad (co-Smad) and translocate to the nucleus. The oligomeric Smad complexes regulate the transcription of target genes through interaction with various transcription factors and transcriptional coactivators or corepressors. Inhibitory Smads (I-Smads) negatively regulate the action of R-Smads and/or co-Smads. Eight different Smads have been identified in mammals. Smad1, Smad5 and Smad8 are R-Smads in BMP signalling pathways (BMP-specific R-Smads). Smad2 and Smad3 are R-Smads in TGFB/activin<br>signalling pathways. BMP receptors can phosphorylate Smad2 in certain types of cells (Murakami et al. 2009). Smad1, Smad5 and Smad8 are structurally highly similar to each other. The functional differences between them are largely unknown. Smad4 is the only co-Smad in mammals, shared by both BMP and TGFB/activin signalling pathways. Smad6 and Smad7 are I-Smads.', 'url': 'https://reactome.org/PathwayBrowser/#/R-HSA-201451', 'sameAs': None, 'version': '70', 'keywords': ['Pathway'], 'creator': [{'givenName': 'Lukasz', 'familyName': 'Huminiecki', 'affiliation': [{'name': 'Ludwig Institute for Cancer Research', '@type': 'Organization'}], 'url': 'http://europepmc.org/authors/null', '@type': 'Person'}, {'givenName': 'Aris', 'familyName': 'Moustakas', 'affiliation': [{'name': 'Ludwig Institute for Cancer Research', '@type': 'Organization'}], 'url': 'http://europepmc.org/authors/null', '@type': 'Person'}], 'includedInDataCatalog': {'url': 'https://reactome.org', 'name': 'Reactome', '@type': 'DataCatalog'}, 'distribution': [{'contentUrl': 'https://reactome.org/ContentService/exporter/sbml/201451.xml', 'fileFormat': 'SBML', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/ReactomeRESTfulAPI/RESTfulWS/sbgnExporter/201451', 'fileFormat': 'SBGN', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/ReactomeRESTfulAPI/RESTfulWS/biopaxExporter/Level2/201451', 'fileFormat': 'BIOPAX2', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/ReactomeRESTfulAPI/RESTfulWS/biopaxExporter/Level3/201451', 'fileFormat': 'BIOPAX3', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/cgi-bin/pdfexporter?DB=gk_current&ID=201451', 'fileFormat': 'PDF', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/cgi-bin/rtfexporter?DB=gk_current&ID=201451', 'fileFormat': 'DOCX', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/cgi-bin/protegeexporter?DB=gk_current&ID=201451', 'fileFormat': 'OWL', '@type': 'DataDownload'}], 'citation': ['http://www.ncbi.nlm.nih.gov/pubmed/19762341', 'http://www.ncbi.nlm.nih.gov/pubmed/28348038'], 'license': 'https://creativecommons.org/licenses/by/4.0/', '@context': 'http://schema.org/', '@type': 'DataSet'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pdb:2gc4: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/GO:0006915: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.drug:D00123: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sgd:S000003909: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/SBO:0000262: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.glycan:G00123: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/wb:WBGene00000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pfam:PF01234: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/insdc:X58356: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/fb:FBgn0011293: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pubchem.compound:100101: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/prosite:PS00001: {'metadata': [{'@context': ['https://schema.org/'], '@type': 'Organization', 'url': 'https://prosite.expasy.org', 'name': 'PROSITE', 'logo': 'https://prosite.expasy.org//images/prosite/prosite.gif', 'memberof': {'@type': 'NGO', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://sib.swiss/'}, 'parentOrganization': {'@type': 'Organization', 'name': 'ExPASy', 'url': 'https://www.expasy.org/'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/arxiv:0807.4956v1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/wormpep:CE28239: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pubchem.substance:100101: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sabiork.reaction:75: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/arrayexpress:E-MEXP-1712: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/MGI:2442292: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/refseq:NP_012345: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tcdb:5.A.1.1.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/uniparc:UPI000000000A: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mint:MINT-10000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/signaling-gateway:A001094: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dip:DIP-743N: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/resid:AA0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tair.protein:AASequence:1009107926: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tair.locus:2200950: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tair.gene:Gene:2200934: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rgd:2018: {'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'Acly', 'description': \"ENCODES a protein that exhibits ATP citrate synthase activity; INVOLVED IN acetyl-CoA metabolic process; citrate metabolic process; fatty acid biosynthetic process; PARTICIPATES IN fatty acid biosynthetic pathway; citric acid cycle pathway; ASSOCIATED WITH congestive heart failure (ortholog); hepatocellular carcinoma (ortholog); obesity (ortholog); FOUND IN cytosol (ortholog); nucleoplasm (ortholog); plasma membrane (ortholog); INTERACTS WITH 1-naphthyl isothiocyanate; 17alpha-ethynylestradiol; 2,2',4,4'-Tetrabromodiphenyl ether\", 'url': 'https://rgd.mcw.edu/rgdweb/report/gene/main.html?id=2018', 'keywords': 'Rat Gene RGD Genome', 'includedInDataCatalog': 'https://rgd.mcw.edu', 'creator': {'@type': 'Organization', 'name': 'Rat Genome Database'}, 'version': '1', 'license': 'Creative Commons CC BY 4.0'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/CO:0000006: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/peptideatlas:PAp00000009: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lipidmaps:LMPR0102010012: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/geo:GDS1234: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hmdb:HMDB00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sgd.pathways:PWY3O-214: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biogrid:31623: {'metadata': [{'@context': 'https://schema.org/', '@type': 'Dataset', 'name': 'BIOGRID CURATED DATA FOR STE11 (Saccharomyces cerevisiae (S288c))', 'description': 'Protein-Protein, Genetic, and Chemical Interactions for STE11 (Saccharomyces cerevisiae (S288c)) curated by BioGRID (https://thebiogrid.org); DEFINITION: Signal transducing MEK kinase; involved in pheromone response and pseudohyphal/invasive growth pathways where it phosphorylates Ste7p, and the high osmolarity response pathway, via phosphorylation of Pbs2p; regulated by Ste20p and Ste50p; protein abundance increases in response to DNA replication stress', 'url': 'https://thebiogrid.org/31623/summary/saccharomyces-cerevisiae/ste11.html', 'keywords': ['BIOLOGY', 'GENE', 'PROTEIN', 'BIOGRID', 'SACCHAROMYCES CEREVISIAE (S288C)', 'STE11', 'PROTEIN-PROTEIN INTERACTION', 'GENETIC INTERACTION', 'CHEMICAL INTERACTION', 'POST-TRANSLATIONAL MODIFICATION', 'GENE ONTOLOGY', 'CHEMICAL-GENETIC INTERACTION', 'PROTEIN SEQUENCE'], 'creator': {'@type': 'Organization', 'url': 'https://thebiogrid.org', 'name': 'BioGRID Project', 'contactPoint': {'@type': 'ContactPoint', 'contactType': 'Customer Service', 'email': 'biogridadmin@gmail.com', 'url': 'https://wiki.thebiogrid.org/doku.php/aboutus'}}, 'citation': 'https://identifiers.org/biogrid:31623', 'distribution': [{'@type': 'DataDownload', 'encodingFormat': 'ZIP', 'contentURL': 'https://thebiogrid.org/downloadgenerator.php?geneID=31623'}], 'license': {'@type': 'CreativeWork', 'name': 'MIT License', 'url': 'https://en.wikipedia.org/wiki/MIT_License'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/MOD:00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/merops:S01.001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/panther.family:PTHR12345: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/prints:PR00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ligandexpo:ABC: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mge:2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pride:1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/3dmet:B00162: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/isbn:9781584885658: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.genes:syn:ssr3451: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/brenda:1.1.1.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/FMA:67112: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/matrixdb.association:P00747__P07355: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ncbigene:100010: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pubchem.bioassay:1018: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bao:0002989: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mmmp:biomaps:37: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pathwaycommons:485991: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hovergen:HBG004341: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/wikipathways:WP100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/macie:M0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mirbase:MI0000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hgnc:2674: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/zfin:ZDB-GENE-041118-11: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rhea:12345: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/SO:0000704: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/chembl.compound:CHEMBL308052: {'metadata': [{'identifier': 'CHEMBL308052', 'inChIKey': 'BEWLPDVTVIFJAJ-ZPGRZCPFSA-N', '@context': 'http://schema.org', 'biochemicalInteraction': [{'identifier': 'CHEMBL3879801', 'url': 'www.ebi.ac.uk/chembl/target_report_card/CHEMBL3879801/', '@type': 'BioChemEntity'}, {'identifier': 'CHEMBL3974', 'url': 'www.ebi.ac.uk/chembl/target_report_card/CHEMBL3974/', '@type': 'BioChemEntity'}], 'inChI': 'InChI=1S/C40H42Cl2F2N8O3/c41-31-7-5-8-32(42)30(31)24-52-23-26(22-51-16-3-4-17-51)29-11-10-27(20-37(29)52)48-40(55)50-36(19-25-9-12-33(43)34(44)18-25)39(54)49-35(13-14-45)38(53)47-21-28-6-1-2-15-46-28/h1-2,5-12,15,18,20,23,35-36H,3-4,13-14,16-17,19,21-22,24,45H2,(H,47,53)(H,49,54)(H2,48,50,55)/t35-,36-/m0/s1', 'canonical_smiles': ['NCC[C@H](NC(=O)[C@H](Cc1ccc(F)c(F)c1)NC(=O)Nc2ccc3c(CN4CCCC4)cn(Cc5c(Cl)cccc5Cl)c3c2)C(=O)NCc6ccccn6'], '@type': 'MolecularEntity', 'image': 'https://www.ebi.ac.uk/chembl/api/data/image/CHEMBL308052.svg?engine=indigo', 'molecularWeight': 791.73, 'url': 'https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL308052/', 'molecularFormula': 'C40H42Cl2F2N8O3', 'monoisotopicMolecularWeight': 790.2725}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/chembl.target:CHEMBL3467: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sabiork.kineticrecord:5046: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/atc:A10BA02: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/pharmgkb.pathways:PA146123006: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pharmgkb.drug:PA448710: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pharmgkb.disease:PA447218: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ttd.target:TTDS00056: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ttd.drug:DAP000773: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lgic:5HT3Arano: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/neuromorpho:Rosa2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/chemidplus:57-27-2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ctd.chemical:D001151: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biosystems:001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/neurondb:265: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ctd.gene:101: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ctd.disease:D053716: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bionumbers:104674: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/drugbank:DB00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/t3db:T3D0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/smpdb:SMP00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/MI:0308: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genedb:LinJ.20.0070: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biomodels.teddy:TEDDY_0000066: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/phosphosite.protein:12300: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/CL:0000232: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/BTO:0000146: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biomodels.kisao:KISAO_0000057: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pdb-ccd:AB0: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/PATO:0001998: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/glycomedb:G77500AY: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lipidbank:BBA0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/smart:SM00015: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cdd:cd00400: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/prodom:PD10000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.orthology:K00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mmdb:50885: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/imex:IM-19210-3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ro:RO_0002533: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/irefweb:617102: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mpid:172: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/phosphosite.residue:2842: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/neurolex:Birnlex_721: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/asap:ABE-0009634: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sabiork.ec:2.7.1.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/opb:OPB_00573: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/obi:OBI_0000070: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/jws:curien: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/modeldb:45539: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/subtiwiki:BSU29180: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/doqcs.pathway:131: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/doqcs.model:57: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/UO:0000080: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pid.pathway:pi3kcipathway: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/PR\\:000000024: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/clinicaltrials:NCT00222573: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/chemspider:56586: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/omia:1000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cgd:CAL0003079: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/csa:1a05: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ncit:C80519: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/antweb:casent0106247: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biocatalogue.service:614: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pmc:PMC3084216: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/amoebadb:EDI_244000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/atcc:11303: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/microsporidia:ECU03_0820i: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/plasmodb:PF11_0344: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cryptodb:cgd7_230: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/giardiadb:GL50803_102438: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/trichdb:TVAG_386080: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/beetlebase:TC010103: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bdgp.insertion:KG09531: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tritrypdb:Tb927.8.620: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/toxoplasma:TGME49_053730: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bold.taxonomy:27267: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dbprobe:1000000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/unists:456789: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dbest:BP100000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dbsnp:rs121909098: {'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'rs121909098', 'description': 'SNV-Single Nucleotide Variation', 'version': 'Build 153', 'license': 'Open Access', 'distribution': {'@type': 'DataDownload', 'encodingFormat': 'JSON', 'contentUrl': 'https://api.ncbi.nlm.nih.gov/variation/v0/beta/refsnp/121909098'}, 'about': {'@type': 'Thing', 'description': 'Alleles: C>T'}, 'sourceOrganization': 'dbSNP', 'datePublished': 'July 9, 2019'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ecogene:EG10173: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/greengenes:100000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bdgp.est:EY223054.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/grin.taxonomy:19333: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gabi:2679240: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hinv.locus:HIX0004394: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hinv.transcript:HIT000195363: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/homd.seq:SEQF1003: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hinv.protein:HIP000030660: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/homd.taxon:811: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/isfinder:ISA1083-2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ird.segment:CY077097: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/jcm:17254: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/img.taxon:648028003: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mycobank:349124: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/maizegdb.locus:25011: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/img.gene:638309541: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pseudomonas:PSEEN0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/nbrc:00001234: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gramene.protein:78073: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gramene.gene:GR:0080039: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gramene.taxonomy:GR_tax:013681: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gramene.qtl:CQG5: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/xenbase:922462: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sgn:0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/edam:data_1664: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bioportal:1046: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/miriam.resource:MIR:00100005: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/arachnoserver:AS000060: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pmdb:PM0012345: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biocyc:ECOLI:CYT-D-UBIOX-CPLX: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/paleodb:83088: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cazy:GT10: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/goa:P12345: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/compulyeast:O08709: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/eggnog:veNOG12876: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/echobase:EB0170: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ensembl.bacteria:MU9_3181: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ensembl.protist:PF3D7_1328700: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/disprot:DP00003: {'metadata': [{'@context': 'http://schema.org/', '@type': 'DataCatalog', '@id': 'https://disprot.org', 'name': 'DisProt', 'description': 'Database of manually annotated intrisically disordered proteins', 'url': 'https://disprot.org', 'publication': {'@type': 'PublicationEvent', 'name': '', 'url': 'https://doi.org/10.1093/nar/gkz975', 'sameAs': ['https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz975/5622715']}, 'keywords': ['IDP', 'IDPs', 'DisProt', 'intrinsic protein disorder', 'protein annotation', 'protein disorder'], 'sourceOrganization': [{'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}], 'provider': [{'@type': 'Person', 'name': 'Silvio Tosatto', 'email': 'silvio.tosatto@unipd.it'}], 'datePublished': '2019-09', 'dateModified': '2019-09', 'encodingFormat': 'text/html', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000199', 'dataset': {'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'DisProt', 'description': 'Database of manually annotated intrisically disordered proteins', 'url': 'https://disprot.org', 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000199', 'keywords': ['IDP', 'IDPs', 'DisProt', 'intrinsic protein disorder', 'protein annotation', 'protein disorder'], 'includedInDataCatalog': 'http://disprot.org', 'creator': {'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}, 'version': '2019-09', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ensembl.metazoa:FBtr0084214: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hcvdb:M58335: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ensembl.plant:AT1G73965: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ensembl.fungi:CADAFLAT00006211: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genatlas:HBB: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/cath.domain:1cukA01: {'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'http://www.cathdb.info//version/v4_2_0/domain/1cukA01', 'mainEntityOfPage': {'@type': ['DataRecord'], 'identifier': 'cath.domain:1cukA01', 'isPartOf': {'@type': 'Dataset', '@id': 'CATH release v4_2_0'}, 'mainEntity': {'@type': ['StructuredValue', 'BioChemEntity'], 'additionalType': 'http://semanticscience.org/resource/SIO_001379', 'identifier': 'cathdomain:1cukA01', 'name': 'CATH Domain - 1cukA01 (PDB: 1cuk, chain A, domain domain.domain_number', 'isContainedIn': [{'@type': ['StructuredValue', 'BioChemEntity'], 'additionalType': 'http://semanticscience.org/resource/SIO_001379', 'additionalProperty': {'@type': 'BioChemEntity', '@id': 'http://identifiers.org/cath.superfamily:1.10.8.10', 'identifier': '1.10.8.10', 'name': '\"winged helix\" repressor DNA binding domain'}}, {'@type': 'DataRecord', 'identifier': '1cuk', 'isPartOf': {'@type': 'Dataset', '@id': 'http://pdb.org'}, 'name': '1cuk', 'additionalType': 'http://semanticscience.org/resource/SIO_011119', 'additionalProperty': [{'@type': 'StructuredValue', 'additionalType': 'http://semanticscience.org/resource/SIO_010000', 'name': 'organism', 'value': 'http://identifiers.org/taxonomy:9606'}, {'@type': 'StructuredValue', 'name': 'method', 'value': 'xray crystallography'}, {'@type': 'StructuredValue', 'name': 'resolution', 'value': '2.2√É\\x85'}]}], 'additionalProperty': [{'@type': 'http://semanticscience.org/resource/SIO_001379', 'name': 'domain annotation', 'value': {'@type': ['StructuredValue', 'BioChemEntity'], 'additionalType': 'http://semanticscience.org/resource/SIO_001379', 'identifier': 'cath.domain:1cukA01', 'name': 'CATH Domain - 1cukA01 (PDB: 1cuk, chain A, domain domain.domain_number', 'location': [{'@type': 'PropertyValue', 'minValue': 2, 'maxValue': 123}, {'@type': 'PropertyValue', 'minValue': 234, 'maxValue': 431}]}}]}}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cath.superfamily:1.10.10.200: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genefarm:4892: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hogenom:HBG284870: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gpcrdb:RL3R1_HUMAN: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hssp:102l: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genetree:ENSGT00550000074763: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/myco.tuber:Rv1908c: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/myco.marinum:MMAR_2462: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/myco.smeg:MSMEG_3769: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/orthodb:Q9P0K8: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/myco.lepra:ML0224: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/orphanet:85163: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/peroxibase:5282: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pmap.substratedb:1915: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/phylomedb:Phy000CLXM_RAT: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pmap.cutdb:25782: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/protclustdb:O80725: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pmp:Q0VCA6: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/protonet.cluster:4349895: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/DOID\\:11337: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/protonet.proteincard:16941567: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/swiss-model:P23298: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/CCO\\:0000003: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rebase:101: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vectorbase:ISCW007415: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mirbase.mature:MIMAT0000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cas:50-00-0: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/nextprot:NX_O00165: {'metadata': [{'@context': 'http://schema.org', '@graph': [{'@type': 'DataCatalog', '@id': 'http://identifiers.org/nextprot/', 'name': '{{nm.title}}', 'description': '{{nm.description}}', 'url': 'https://www.nextprot.org/', 'keywords': 'nextprot,UniProt,Human,Proteins,Proteome,Proteomics,protein database,protein knowledgebase,protein resource,human protein,human proteome,cancer variants portal,ion channels variants portal,function,medical,disease,expression,interactions,sequence,isoform,mutation,variant,phenotypes,proteomics,peptide,structure,3D,unicity checker,peptide uniqueness checker,annotation,biocuration,chromosomes,protein validation,protein-coding genes,post-translational modifications,ptm,data integration,systems biology,genetic variations,RDF data model,SPARQL endpoint,api,rest', 'thumbnailUrl': 'https://www.nextprot.org/img/np-x.png', 'sourceOrganization': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'The neXtProt knowledgebase on human proteins: 2017 update.', 'author': [{'@type': 'Person', 'name': 'Gaudet P'}, {'@type': 'Person', 'name': 'Michel PA'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Britan A'}, {'@type': 'Person', 'name': 'Cusin I'}, {'@type': 'Person', 'name': 'Domagalski M'}, {'@type': 'Person', 'name': 'Duek PD'}, {'@type': 'Person', 'name': 'Gateau A'}, {'@type': 'Person', 'name': 'Gleizes A'}, {'@type': 'Person', 'name': 'Hinard V'}, {'@type': 'Person', 'name': 'Rech de Laval V'}, {'@type': 'Person', 'name': 'Lin JJ'}, {'@type': 'Person', 'name': 'Nikitin F'}, {'@type': 'Person', 'name': 'Schaeffer M'}, {'@type': 'Person', 'name': 'Teixeira D'}, {'@type': 'Person', 'name': 'Lane L'}, {'@type': 'Person', 'name': 'Bairoch A'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Nucleic acids research'}, {'@type': 'PublicationVolume', 'volumeNumber': '45'}, {'@type': 'PublicationIssue', 'datePublished': '2017', 'issueNumber': 'D1'}], 'sameAs': ['https://doi.org/10.1093/nar/gkw1062', 'http://europepmc.org/abstract/MED/27899619', 'https://www.ncbi.nlm.nih.gov/pubmed/27899619', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210547']}, 'dataset': [{'@type': 'Dataset', 'name': 'neXtProt entries', 'description': 'The collection of neXtProt entries for human proteins', 'url': 'https://www.nextprot.org', 'keywords': 'nextprot,Human,Proteins,Proteome,Proteomics,protein database,protein knowledgebase,protein resource,human protein,human proteome,function,medical,disease,expression,interactions,sequence,isoform,mutation,variant,phenotypes,proteomics,peptide,structure,3D,annotation,biocuration,chromosomes,protein validation,protein-coding genes,post-translational modifications,ptm,data integration,systems biology,genetic variations,UniProt', 'distribution': [{'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/xml/nextprot_all.xml.gz', 'fileFormat': 'XML'}, {'@type': 'DataDownload', 'fileFormat': 'RDF'}, {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/peff/nextprot_all.peff.gz', 'fileFormat': 'PEFF'}, {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/md5/nextprot_sequence_md5.txt', 'fileFormat': 'TXT'}, {'@type': 'DataDownload', 'contentUrl': 'https://api.nextprot.org/export/entries/all.fasta', 'fileFormat': 'FASTA'}], 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.nextprot.org/proteins/search?query={query}', 'query-input': 'required name=query'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}, {'@type': 'Dataset', 'name': 'Caloha', 'description': 'An ontology of human anatomy and human cell types.', 'url': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/caloha.obo', 'keywords': 'human body, anatomy, ontology, tissues, cell types, organs, developmental stage', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/caloha.obo', 'fileFormat': 'OBO'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}, {'@type': 'Dataset', 'name': 'ICEPO', 'description': 'An ontology that allows one to annotate the electrophysiological parameters of the voltage-gated class of ion channels.', 'url': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/icepo.obo', 'keywords': 'ion channels, electrophysiology, ion channel gating, conductance, ion selectivity, activation, inactivation, recovery, deactivation, reopening, closed-state inactivation', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/icepo.obo', 'fileFormat': 'OBO'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'ICEPO: the ion channel electrophysiology ontology.', 'author': [{'@type': 'Person', 'name': 'Hinard V'}, {'@type': 'Person', 'name': 'Britan A'}, {'@type': 'Person', 'name': 'Rougier JS'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Abriel H'}, {'@type': 'Person', 'name': 'Gaudet P'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Database (Oxford)'}, {'@type': 'PublicationVolume', 'volumeNumber': '2016'}, {'@type': 'PublicationIssue', 'datePublished': '2016'}], 'sameAs': ['https://doi.org/10.1093/database/baw017', 'http://europepmc.org/abstract/MED/27055825', 'https://www.ncbi.nlm.nih.gov/pubmed/27055825', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4823818']}}, {'@type': 'Dataset', 'name': 'Chromosome reports', 'description': 'A set of 26 files, each one listing all the neXtProt entries encoded on a given chromosome (chromosomes 1-22, X, Y, mitochondrial genome (MT), or unknown genomic location), along with their genomic coordinates and a brief summary of the available annotations for each of them.', 'url': 'ftp://ftp.nextprot.org/pub/current_release/chr_reports/', 'keywords': 'human genes, chromosomes, mitochondrial genome, genomic coordinates, chromosomal location, coding strand, protein existence, PE1, PE2, PE3, PE4, PE5', 'distribution': {'@type': 'DataDownload', 'fileFormat': 'TXT'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'neXtProt: Organizing Protein Knowledge in the Context of Human Proteome Projects', 'author': [{'@type': 'Person', 'name': 'Gaudet P'}, {'@type': 'Person', 'name': 'Argoud-Puy G'}, {'@type': 'Person', 'name': 'Cusin I'}, {'@type': 'Person', 'name': 'Duek P'}, {'@type': 'Person', 'name': 'Evalet O'}, {'@type': 'Person', 'name': 'Gateau A'}, {'@type': 'Person', 'name': 'Gleizes A'}, {'@type': 'Person', 'name': 'Pereira M'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Zwahlen C'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Lane L'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Journal of proteome research'}, {'@type': 'PublicationVolume', 'volumeNumber': '12'}, {'@type': 'PublicationIssue', 'datePublished': '2013', 'issueNumber': '1'}], 'sameAs': ['https://doi.org/10.1021/pr300830v', 'https://www.ncbi.nlm.nih.gov/pubmed/23205526']}}, {'@type': 'Dataset', 'name': 'Cancer portal', 'description': 'Portal which provides information about the functional impact of genetic variants and artificially generated mutants in cancer genes', 'url': 'https://www.nextprot.org/portals/breast-cancer', 'keywords': 'BRCA1,Biological database,Cancer,Functional defect assessment,Genetic variants,Molecular phenotypes', 'distribution': {'@type': 'DataDownload', 'fileFormat': 'CSV'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'A new bioinformatics tool to help assess the significance of BRCA1 variants.', 'author': [{'@type': 'Person', 'name': 'Cusin I'}, {'@type': 'Person', 'name': 'Teixeira D'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Rech de Laval V'}, {'@type': 'Person', 'name': 'Gleizes A'}, {'@type': 'Person', 'name': 'Viassolo V'}, {'@type': 'Person', 'name': 'Chappuis PO'}, {'@type': 'Person', 'name': 'Hutter P'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Gaudet P'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Human genomics'}, {'@type': 'PublicationVolume', 'volumeNumber': '12'}, {'@type': 'PublicationIssue', 'datePublished': '2018', 'issueNumber': '1'}], 'sameAs': ['https://doi.org/10.1186/s40246-018-0168-0', 'http://europepmc.org/abstract/MED/29996917', 'https://www.ncbi.nlm.nih.gov/pubmed/29996917', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6042458/']}}, {'@type': 'Dataset', 'name': 'Ion channel portal', 'description': 'Portal which provides information about the functional impact of genetic variants and artificially generated mutants in voltage-gated sodium channel genes', 'url': 'https://www.nextprot.org/portals/navmut', 'keywords': 'clinical interpretation of variants, databases, genetics variants,pathogenicity,phenotype,voltage-gated sodium channel', 'distribution': {'@type': 'DataDownload', 'fileFormat': 'CSV'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'Annotation of functional impact of voltage-gated sodium channel mutations.', 'author': [{'@type': 'Person', 'name': 'Hinard V'}, {'@type': 'Person', 'name': 'Britan A'}, {'@type': 'Person', 'name': 'Schaeffer M'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Thomet U'}, {'@type': 'Person', 'name': 'Rougier JS'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Abriel H'}, {'@type': 'Person', 'name': 'Gaudet P'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Human mutation'}, {'@type': 'PublicationVolume', 'volumeNumber': '38'}, {'@type': 'PublicationIssue', 'datePublished': '2017', 'issueNumber': '5'}], 'sameAs': ['https://doi.org/10.1002/humu.23191', 'http://europepmc.org/abstract/MED/28168870', 'https://www.ncbi.nlm.nih.gov/pubmed/28168870', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5413847/']}}, {'@type': 'Dataset', 'name': 'Protein kinase function portal', 'description': 'Portal which provides information about the function of protein kinases', 'url': 'https://www.nextprot.org/portals/kinase-function', 'keywords': 'databases, protein kinase,phosphorylation, protein-protein interactions, biological process, protein function', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'https://www.nextprot.org/viewers/portals/kinase-function/app/index.html#', 'fileFormat': 'CSV'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}]}, {'@type': 'SoftwareApplication', 'name': 'Peptide uniqueness checker', 'url': 'https://www.nextprot.org/tools/peptide-uniqueness-checker', 'author': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'The neXtProt peptide uniqueness checker allows scientists to define which peptides can be used to validate the existence of human proteins, i.e. map uniquely versus multiply to human protein sequences taking into account isobaric substitutions, alternative splicing and single amino acid variants.', 'applicationCategory': 'http://schema.org/WebApplication', 'offers': {'@type': 'Offer', 'price': '0.00', 'priceCurrency': 'CHF'}, 'license': {'@type': 'CreativeWork', 'name': 'GNU General Public License v2.0', 'url': 'https://www.gnu.org/licenses/gpl-2.0.fr.html'}, 'operatingSystem': 'Unix,Windows,OS X,Android', 'citation': {'@type': 'ScholarlyArticle', 'headline': 'The neXtProt peptide uniqueness checker: a tool for the proteomics community.', 'author': [{'@type': 'Person', 'name': 'Schaeffer M'}, {'@type': 'Person', 'name': 'Gateau A'}, {'@type': 'Person', 'name': 'Teixeira D'}, {'@type': 'Person', 'name': 'Michel PA'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Lane L'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Bioinformatics'}, {'@type': 'PublicationVolume', 'volumeNumber': '33'}, {'@type': 'PublicationIssue', 'datePublished': '2017', 'issueNumber': '21'}], 'sameAs': ['https://doi.org/10.1093/bioinformatics/btx318', 'http://europepmc.org/abstract/MED/28520855', 'https://www.ncbi.nlm.nih.gov/pubmed/28520855', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5860159/']}}, {'@type': 'SoftwareApplication', 'name': 'neXtProt SNORQL', 'url': 'https://snorql.nextprot.org/', 'author': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'A web application for using the SPARQL Protocol and RDF Query Language (SPARQL) to access all the neXtProt protein entry data', 'applicationCategory': 'http://schema.org/WebApplication', 'offers': {'@type': 'Offer', 'price': '0.00', 'priceCurrency': 'CHF'}, 'license': {'@type': 'CreativeWork', 'name': 'GNU General Public License v2.0', 'url': 'https://www.gnu.org/licenses/gpl-2.0.fr.html'}, 'operatingSystem': 'Unix,Windows,OS X,Android'}, {'@type': 'Service', 'name': 'neXtProt advanced search (SPARQL)', 'url': 'https://www.nextprot.org/proteins/search?mode=advanced', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'A web service for using the SPARQL Protocol and RDF Query Language (SPARQL) to retrieve a list of neXtProt entries based on all the neXtProt protein entry data.'}, {'@type': 'Service', 'name': 'neXtProt list manager', 'url': 'https://www.nextprot.org/proteins/search?mode=advanced', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'A web service for viewing and managing protein lists (a collection of protein entries created by the user).'}, {'@type': 'Service', 'name': 'neXtProt API', 'description': 'The neXtProt API lets you find entities in neXtProt.', 'url': 'https://api.nextprot.org', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}}, {'@type': 'Service', 'name': 'neXtProt SPARQL endpoint', 'description': 'The neXtProt SPARQL endpoint is a point of presence on an HTTP network that‚Äôs capable of receiving and processing SPARQL Protocol requests.', 'url': 'https://sparql.nextprot.org/', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}}]}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.metagenome:T30002: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/narcis:oai:cwi.nl:4725: {'metadata': [{'@context': 'http://schema.org/', '@type': 'CreativeWork', '@id': 'https://www.persistent-identifier.nl/urn:nbn:nl:ui:18-4725', 'additionalType': 'other', 'name': 'Mathematical modelling in blood coagulation : simulation and parameter estimation', 'headline': 'Mathematical modelling in blood coagulation : simulation and parameter estimation', 'datePublished': '1997-01-01', 'description': \"textabstractThis paper describes the mathematical modelling of a part of the blood coagulation mechanism. The model includes the activation of factor X by a purified enzyme from Russel's Viper Venom (RVV), factor V and prothrombin, and also comprises the inactivation of the products formed. In this study we assume that in principle the mechanism of the process is known. However, the exact structure of the mechanism is unknown, and the process still can be described by different mathematical models. These models are put to test by measuring their capacity to explain the course of thrombin generation as observed in plasma after recalcification in presence of RVV. The mechanism studied is mathematically modelled as a system of differential-algebraic equations (DAEs). Each candidate model contains some freedom, which is expressed in the model equations by the presence of unknown parameters. For example, reaction constants or initial concentrations are unknown. The goal of parameter estimation is to determine these unknown parameters in such a way that the theoretical (i.e., computed) results fit the experimental data within measurement accuracy and to judge which modifications of the chemical reaction scheme allow the best fit. We present results on model discrimination and estimation of reaction constants, which are hard to obtain in another way.\", 'inLanguage': 'en', 'publisher': 'CWI', 'mainEntityOfPage': 'https://www.narcis.nl/publication/RecordID/oai%3Acwi.nl%3A4725', 'url': 'https://ir.cwi.nl/pub/4725', 'identifier': {'@type': 'PropertyValue', 'name': 'nbn', 'value': 'urn:NBN:nl:ui:18-4725'}, 'author': [{'@context': 'http://schema.org/', '@type': 'Person', 'name': 'W.J.H. Stortelder (Walter)'}, {'@context': 'http://schema.org/', '@type': 'Person', '@id': 'http://isni.org/isni/0000000108922067', 'name': 'P.W. Hemker (Piet)', 'identifier': {'@type': 'PropertyValue', 'name': 'isni', 'value': '0000000108922067'}}, {'@context': 'http://schema.org/', '@type': 'Person', 'name': 'H.C. Hemker'}]}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.genome:eco: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/jcsd:J55.713G: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pharmgkb.gene:PA131: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/scretf:RSC3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/insdc.sra:SRX000007: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/PW:0000208: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/napp:351: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mirnest:MNEST029358: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/noncodev3:377550: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/virsirna:virsi1909: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/elm:CLV_MEL_PAP_1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mimodb:1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bykdb:A0A009E7X8: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sitex:1000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/topfind:Q9UKQ2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/combine.specifications:sbgn.er.level-1.version-1.2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cellimage:24801: {'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'Tatyana M. Svitkina, Gary G. Borisy (2011) CIL:24801, Xenopus laevis, fibroblast. CIL. Dataset', 'description': 'Structural differentiation of actin network in lamellipodium. Electron micrograph of Xenopus fibroblasts after regular extraction in the presence of polyethelene glycol (PEG) and phalloidin. While the actin network in the front zone of these cels remains dense, the actin network at the lamellipodial rear disassembled in the course of unprotected extraction (CIL 24800).  Image corresponds to Figure 6c from J Cell Biol. 1999 May 31;145(5):1009-26.', 'identifier': 'doi:10.7295/W9CIL24801', 'license': 'http://creativecommons.org/licenses/by-nc-sa/3.0', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'https://cildata.crbs.ucsd.edu/media/images/24801/24801.zip'}, 'sourceOrganization': 'CIL', 'datePublished': '2011-06-02'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mipmod:HOSAPI0399: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/conoserver:2639: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/huge:KIAA0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cabri:dsmz_mutz-id:ACC 291: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dictybase.gene:DDB_G0267522: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/string:P53350: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bindingdb:e999: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/phenolexplorer:75: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/stitch:BQJCRHHNABKAKU: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/atcvet:QJ51RV02: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/snomedct:284196006: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mesh.2012:17186: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/knapsack:C00000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/massbank:PB000166: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cdpd:63250: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gmd:68513255-fc44-4041-bc4b-4fd2fae7541d: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/homologene:1000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/abs:A0014: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/umbbd.compound:c0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dpv:100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/apd:01001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/chemdb:3966782: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aceview.worm:aap-1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/imgt.ligm:M94112: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/iuphar.receptor:101: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/worfdb:T01B6.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/soybase:BARC-013845-01256: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/nextdb:6b1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hamap:MF_01400: {'metadata': [{'@context': ['https://schema.org/'], '@type': 'Organization', 'url': 'https://hamap.expasy.org', 'name': 'HAMAP', 'logo': 'https://hamap.expasy.org//images/hamap/HAMAP_Red_Logo.png', 'memberof': {'@type': 'NGO', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://sib.swiss/'}, 'parentOrganization': {'@type': 'Organization', 'name': 'ExPASy', 'url': 'https://www.expasy.org/'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cgsc:74: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/arrayexpress.platform:A-GEOD-50: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dragondb.dna:3hB06: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rouge:mKIAA4200: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/merops.family:S1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dragondb.locus:DEF: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dragondb.allele:cho: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dragondb.protein:AMDEFA: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/issn:0745-4570: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/niaest:J0705A10: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/nasc:N1899: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pazar:TF0001053: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mo:ArrayGroup: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rnamods:101: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/PO:0009089: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tgd:TTHERM_00648910: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/treebase:TB2:S1000: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cst.ab:3305: {'metadata': [{'@context': 'http://schema.org/', '@type': 'Product', 'name': 'GCN5L2 (C26A10) Rabbit mAb', 'Category': ' Primary Antibodies', 'additionalProperty': [{'@type': 'PropertyValue', 'name': 'APPLICATIONS', 'value': 'W,IP,IF'}, {'@type': 'PropertyValue', 'name': 'REACTIVITY', 'value': 'H,M,R,Mk'}, {'@type': 'PropertyValue', 'name': 'SENSITIVITY', 'value': 'Endogenous'}, {'@type': 'PropertyValue', 'name': 'MW(kDa)', 'value': '94'}, {'@type': 'PropertyValue', 'name': 'SOURCE', 'value': 'Rabbit'}, {'@type': 'PropertyValue', 'name': 'Product Citations', 'value': '29'}], 'image': '', 'description': '', 'brand': 'Cell Signaling Technology'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tigrfam:TIGR00010: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/atfdb.family:CUT: {'metadata': [{'@context': 'https://schema.org', '@graph': [{'@type': 'WebSite', '@id': 'https://www.bioguo.org/#website', 'url': 'https://www.bioguo.org/', 'name': 'Les produits bio', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.bioguo.org/?s={search_term_string}', 'query-input': 'required name=search_term_string'}}]}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sdbs:4544: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dbg2introns:Cu.me.I1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/iuphar.family:78: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vbase2:humIGHV025: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genecards:ABL1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/spike.map:spike00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/metlin:1455: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/umbbd.enzyme:e0333: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mmrrc:70: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/umbbd.reaction:r0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/umbbd.pathway:ala: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dictybase.est:DDB0016567: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/umbbd.rule:bt0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mirex:165a: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/imgt.hla:A*01:01:01:01: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mobidb:P10636: {'metadata': [{'@context': 'http://schema.org', '@type': 'DataCatalog', '@id': 'http://mobidb.bio.unipd.it', 'name': 'MobiDB', 'description': 'Database of protein disorder and mobility annotations', 'url': 'http://mobidb.bio.unipd.it', 'publication': {'@type': 'PublicationEvent', 'name': 'MobiDB 3.0: more annotations for intrinsic disorder, conformational diversity and interactions in proteins', 'url': 'https://academic.oup.com/nar/article/46/D1/D471/4612964', 'sameAs': ['https://doi.org/10.1093/nar/gkx1071', 'https://europepmc.org/abstract/MED/29136219', 'https://www.ncbi.nlm.nih.gov/pubmed/29136219', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5753340']}, 'keywords': ['protein annotation', 'protein disorder'], 'sourceOrganization': [{'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}], 'provider': [{'@type': 'Person', 'name': 'Silvio Tosatto', 'email': 'silvio.tosatto@unipd.it'}], 'datePublished': '2018-01', 'dateModified': '2018-01', 'encodingFormat': 'text/html', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000637', 'dataset': [{'@type': 'Dataset', 'name': 'MobiDB', 'description': 'MobiDB dataset describes disorder and mobility annotation for proteins', 'url': 'http://mobidb.bio.unipd.it', 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000637', 'keywords': ['protein annotation', 'protein disorder'], 'includedInDataCatalog': 'http://mobidb.bio.unipd.it', 'creator': {'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}, 'version': '2018-01', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'distribution': [{'@type': 'DataDownload', 'name': 'Indirect disorder', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/pdb/indirect.json'}, {'@type': 'DataDownload', 'name': 'Indirect LIPs', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/pdb/lips.json'}, {'@type': 'DataDownload', 'name': 'Manually curated disorder', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/mobidb/curated.json'}, {'@type': 'DataDownload', 'name': 'Manually cuarted LIPs', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/mobidb/lips.json'}, {'@type': 'DataDownload', 'name': 'SwissProt disorder', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/swissprot.json'}, {'@type': 'DataDownload', 'name': 'Indirect dynamic structure', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/bmrb.json'}]}]}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bdsc:33607: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/opm:1h68: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/allergome:1948: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pombase:SPCC13B11.01: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hpa:ENSG00000026508: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/jaxmice:005012: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ubio.namebank:2555646: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/yetfasco:YOR172W_571.0: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tarbase:hsa-let-7a-2-3p: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/charprot:CH_001923: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oma.protein:HUMAN16963: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genpept:CAA71118.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oma.grp:LCSCCPN: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/unigene:4900: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ncbiprotein:CAA71118.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bitterdb.rec:1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bitterdb.cpd:46: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bioproject:PRJDB3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biosample:SAMEA2397676: {'metadata': [{'@context': ['http://schema.org', {'OBI': 'http://purl.obolibrary.org/obo/OBI_', 'biosample': 'http://identifiers.org/biosample/'}], '@type': 'DataRecord', '@id': 'biosample:SAMEA2397676', 'identifier': 'biosample:SAMEA2397676', 'dateCreated': '2014-03-03T16:32:10Z', 'dateModified': '2019-07-24T16:17:02.748Z', 'mainEntity': {'@type': ['Sample', 'OBI:0000747'], 'identifier': ['biosample:SAMEA2397676'], 'name': 'ERS368772', 'subjectOf': ['https://www.ebi.ac.uk/ena/data/view/SAMEA2397676'], 'additionalProperty': [{'@type': 'PropertyValue', 'name': 'ENA checklist', 'value': 'ERC000011'}, {'@type': 'PropertyValue', 'name': 'INSDC center alias', 'value': 'FEM'}, {'@type': 'PropertyValue', 'name': 'INSDC center name', 'value': 'IASMA Research Centre Fondazione Edmund Mach FEM'}, {'@type': 'PropertyValue', 'name': 'INSDC first public', 'value': '2014-03-03T16:32:10Z'}, {'@type': 'PropertyValue', 'name': 'INSDC last update', 'value': '2013-11-08T13:09:45Z'}, {'@type': 'PropertyValue', 'name': 'INSDC status', 'value': 'public'}, {'@type': 'PropertyValue', 'name': 'SRA accession', 'value': 'ERS368772'}, {'@type': 'PropertyValue', 'name': 'alias', 'value': 'CheeseA_24h9'}, {'@type': 'PropertyValue', 'name': 'organism', 'value': 'food metagenome', 'valueReference': [{'@id': 'http://purl.obolibrary.org/obo/NCBITaxon_870726', '@type': 'DefinedTerm'}]}, {'@type': 'PropertyValue', 'name': 'synonym', 'value': 'CheeseA_24h9'}, {'@type': 'PropertyValue', 'name': 'title', 'value': 'CheeseA Period 3 24h ripening'}], 'sameAs': 'http://identifiers.org/biosample/SAMEA2397676'}, 'isPartOf': {'@type': 'Dataset', '@id': 'https://www.ebi.ac.uk/biosamples/samples'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/piroplasma:TA14985: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/unite:UDB000691: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ncim:C0026339: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/proglyc:AC119: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/polbase:19-T4: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/nuclearbd:prgr_human: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/supfam:SSF57615: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pina:Q13485: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bacmap.biog:1050: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ricegap:LOC_Os02g13300: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tissuelist:TS-0285: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hgnc.symbol:DAPK1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/panther.pathway:P00024: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/fairsharing:bsg-000052: {'metadata': [{'@context': 'http://schema.org', '@id': 'https://doi.org/10.25504/FAIRsharing.9qv71f', '@type': 'Dataset', 'alternateName': 'SBML', 'citation': {'@type': 'CreativeWork', 'identifier': 'doi:10.25504/FAIRsharing.9qv71f'}, 'datePublished': '2015-05-14 11:14:01', 'description': 'The Systems Biology Markup Language (SBML) is a machine-readable exchange format for computational models of biological processes. Its strength is in representating phenomena at the scale of biochemical reactions, but it is not limited to that. By supporting SBML as an input and output format, different software tools can operate on the same representation of a model, removing chances for errors in translation and assuring a common starting point for analyses and simulations.', 'identifier': '10.25504/FAIRsharing.9qv71f', 'license': 'https://creativecommons.org/licenses/by-sa/4.0/. Please link to https://fairsharing.org and https://fairsharing.org/static/img/home/svg/FAIRsharing-logo.svg for attribution.', 'name': 'Systems Biology Markup Language', 'url': 'https://doi.org/10.25504/FAIRsharing.9qv71f'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/fungidb:CNBG_0001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/neurovault.image:58788: {'metadata': [{'@context': 'https://schema.org', '@type': 'Organization', 'name': 'NeuroVault', 'url': 'https://neurovault.org', 'logo': 'https://neurovault.org/static/images/neurovault-logo.png', 'sameAs': ['https://en.wikipedia.org/wiki/NeuroVault', 'https://twitter.com/vaultneuro']}, {'@context': 'https://schema.org', '@type': 'WebSite', 'url': 'https://neurovault.org/', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://neurovault.org/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}, {'@context': 'http://schema.org/', '@type': 'Dataset', '@id': 'https://neurovault.org/images/58788', 'url': 'https://neurovault.org/images/58788', 'identifier': ['https://identifiers.org/neurovault.image:58788', 'https://neurovault.org/collections/OELFQSHF/images/58788'], 'name': 'Linked networks for learning and expressing location-specific threat: Threat learning versus object learning during approach periods', 'description': '<p>Activity differences between approaching flowers versus objects during threat and spatial memory aspects of the task. Greater activity was seen when approaching flowers compared with objects in a range of areas, including the insula, medial parietal cortex, PCC, vmPFC, bilateral anterior hippocampus, and amygdala.When approaching objects, a different network was observed with greater activity in the left middle frontal gyrus, middle temporal gyrus, inferior parietal lobule and inferior frontal gyrus, right precentral gyrus, inferior frontal gyrus, and an area extending across lingual and parahippocampal gyri.</p>\\n\\n[![glassbrain](https://neurovault.org/media/images/3303/glass_brain_58788_1.jpg)](https://neurovault.org/images/58788)\\n\\n### Collection description\\n\\n<p>Statistical images from the study in: <br />Suarez-Jimenez B, Bisby JA, Horner AJ, King JA, Pine DS, Burgess N (2018) Linked networks for learning and expressing location-specific threat. PNAS. In press.</p>\\n\\n###Subject species\\n\\nhomo sapiens\\n\\n###Modality\\n\\n fMRI-BOLD\\n\\n### Analysis level\\n\\ngroup\\n\\n### Cognitive paradigm (task)\\n\\nnavigation task\\n\\n### Map type\\n\\nT\\n\\n', 'dateCreated': '2018-01-08T18:52:50', 'dateModified': '2018-06-30T11:08:23', 'keywords': ['neuroscience', 'neuroimaging'], 'includedInDataCatalog': {'@type': 'DataCatalog', 'name': 'NeuroVault', 'url': 'https://neurovault.org'}, 'license': {'@type': 'CreativeWork', 'name': 'CC0 1.0', 'url': 'https://creativecommons.org/publicdomain/zero/1.0/'}, 'distribution': [{'@type': 'DataDownload', 'encodingFormat': 'NIfTI', 'contentUrl': 'https://neurovault.org//media/images/3303/spmT_0007.nii.gz'}], 'citation': 'https://doi.org/10.1073/pnas.1714691115'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/drsc:DRSC05221: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/darc:1250: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oridb.sacch:1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oridb.schizo:1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/scop:47419: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ena.embl:BN000065: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pscdb:051: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/panther.node:PTN000000026: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dommino:2GC4: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ccds:CCDS13573.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gxa.gene:AT4G01080: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gxa.expt:E-MTAB-2037: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hprd:00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lrg:LRG_1: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/metabolights:MTBLS1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/orcid:0000-0002-5355-2576: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/nbn:urn:nbn:fi:tkk-004781: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/phosphopoint.kinase:AURKA: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/uniprot.isoform:Q5BJF6-3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/phosphopoint.protein:AURKA: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/inchikey:RYYVLZVUVIJVGH-UHFFFAOYSA-N: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/inchi:InChI=1S/C2H6O/c1-2-3/h3H,2H2,1H3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/wikipedia.en:SM_UB-81: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cldb:cl3603: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.environ:ev:E00032: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aphidbase.transcript:ACYPI000159: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/efo:0004859: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hgmd:CALM1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/affy.probeset:243002_at: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/treefam:TF101014: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/caps:434: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cubedb:AKR: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ideal:IID00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gold.genome:Gi07796: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pocketome:1433C_TOBAC_1_252: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/stap:1a24: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gold.meta:Gm00047: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bugbase.expt:288: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bugbase.protocol:67: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/VariO:0294: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/tol:98034: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vipr:BeAn 70563: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/fbol:2224: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rfam:RF00230: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/epd:TA_H3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aftol.taxonomy:959: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aspgd.locus:ASPL0000349247: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aspgd.protein:ASPL0000349247: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/autdb:ADA: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bacmap.map:AP011135: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ndc:0002-1975-61: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bgee.gene:FBgn0000015: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bgee.stage:HsapDO:0000004: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bgee.family:ENSFM00500000270089: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bgee.organ:EHDAA:2185: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/panther.pthcmp:P00266: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biocarta.pathway:h_aktPathway: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gmd.gcms:53d583d8-40c6-40e1-9296-23f821cd77a5: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gmd.profile:10b38aaf-b977-4950-85b8-f4775f66658d: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gmd.ref:8cf84adb-b4db-4807-ac98-0004247c35df: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gmd.analyte:4f0fa9b6-514f-4ff4-98cc-0009bc08eb80: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/intact.molecule:EBI-366083: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dailymed:973a9333-fec7-46dd-8eb5-25738f06ee54: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/depod:PTPN1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/phytozome.locus:Glyma0021s00410: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cst:Akt_PKB: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/subtilist:BG11523: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sider.effect:C0017565: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sider.drug:2244: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/broad:S7000002168151102: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/coriell:GM17027: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/wikigenes:3771877: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/corum:100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/UBERON\\:0008203: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ecoliwiki:aaeA: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/MA:0002502: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/jstor:3075966: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genprop:GenProp0699: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/unimod:1200: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vbrc:35742: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rgd.qtl:1354581: {'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'Bp247', 'description': 'QTL Report', 'url': 'https://rgd.mcw.edu/rgdweb/report/qtl/main.html?id=1354581', 'keywords': 'Rat Gene RGD Genome', 'includedInDataCatalog': 'https://rgd.mcw.edu', 'creator': {'@type': 'Organization', 'name': 'Rat Genome Database'}, 'version': '1', 'license': 'Creative Commons CC BY 4.0'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/viralzone:992: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rgd.strain:5688061: {'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'SS.BN-(<i>D13Rat20-D13Got22</i>)/Mcwi-<i>Nckap5<sup>em4Mcwi+/+</sup></i>', 'description': 'Rat Strain', 'url': 'https://rgd.mcw.edu/rgdweb/report/strain/main.html?id=5688061', 'keywords': 'Rat Gene RGD Genome', 'includedInDataCatalog': 'https://rgd.mcw.edu', 'creator': {'@type': 'Organization', 'name': 'Rat Genome Database'}, 'version': '1', 'license': 'Creative Commons CC BY 4.0'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/door:1398574: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/degradome:Ax1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/datf:AT1G01030.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dbd:0045310: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/iuphar.ligand:1755: {'metadata': [{'@context': 'http://schema.org', '@id': 'https://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=1755', '@type': 'DataRecord', 'dct:conformsTo': 'https://bioschemas.org/types/MolecularEntity/', 'includedInDataset': {'@type': 'Dataset', '@id': 'https://www.guidetopharmacology.org/index.jsp#dataset'}, 'mainEntity': {'@id': 'https://identifiers.org/iuphar.ligand:1755', 'name': 'ADP&beta;S', 'alternateName': [\"adenosine 5'-O-(2-thiodiphosphate)\", 'ADPbetaS'], 'description': '', 'identifier': '1755', 'inChI': 'InChI=1S/C10H15N5O9P2S/c11-8-5-9(13-2-12-8)15(3-14-5)10-7(17)6(16)4(23-10)1-22-25(18,19)24-26(20,21)27/h2-4,6-7,10,16-17H,1H2,(H,18,19)(H2,11,12,13)(H2,20,21,27)/t4-,6-,7-,10-/m1/s1', 'inChIKey': 'HCIKUKNAJRJFOW-KQYNXXCUSA-N', 'iupacName': '[(2R,3S,4R,5R)-5-(6-aminopurin-9-yl)-3,4-dihydroxyoxolan-2-yl]methyl dihydroxyphosphinothioyl hydrogen phosphate', 'molecularWeight': 443.01, 'smiles': 'OC1C(COP(=O)(OP(=S)(O)O)O)OC(C1O)n1cnc2c1ncnc2N', 'url': 'https://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=1755'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/molbase:128796-39-4: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/yrcpdr:2673500: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/funcbase.fly:10194: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/funcbase.human:119514: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/yid:SNR17A: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/funcbase.mouse:1351341: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cbioportal:laml_tcga_pub: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/funcbase.yeast:2701: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ydpm:YAL001C: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/wb.rnai:WBRNAi00086878: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pass2:46977: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/iceberg.element:100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/iceberg.family:1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vfdb.gene:VFG2154: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vfdb.genus:Chlamydia: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mesh.2013:17165: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.module:M00002: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg.disease:H00076: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ligandbox:D00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/glycoepitope:EP0311: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/medlineplus:002804: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/jcggdb:JCGG-STR008690: {'metadata': []}\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Metadata for http://127.0.0.1:8082/noncodev4.rna:NONHSAT000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vfb:00000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/noncodev4.gene:NONHSAG00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oryzabase.gene:117: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oryzabase.mutant:21393: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/otl:AADB12: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oryzabase.strain:1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oryzabase.stage:34: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/genewiki:1017: {'metadata': [{'@context': 'https://schema.org', '@type': 'Article', 'name': 'Cyclin-dependent kinase 2', 'url': 'https://en.wikipedia.org/wiki/Cyclin-dependent_kinase_2', 'sameAs': 'http://www.wikidata.org/entity/Q14911732', 'mainEntity': 'http://www.wikidata.org/entity/Q14911732', 'author': {'@type': 'Organization', 'name': 'Contributors to Wikimedia projects'}, 'publisher': {'@type': 'Organization', 'name': 'Wikimedia Foundation, Inc.', 'logo': {'@type': 'ImageObject', 'url': 'https://www.wikimedia.org/static/images/wmf-hor-googpub.png'}}, 'datePublished': '2007-03-05T20:28:34Z', 'dateModified': '2019-11-11T10:45:57Z', 'image': 'https://upload.wikimedia.org/wikipedia/commons/6/6d/Protein_CDK2_PDB_1aq1.png', 'headline': 'protein-coding gene in the species Homo sapiens'}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/paxdb.protein:977869: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pdb.ligand:TRQ: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/merops.inhibitor:I31.952: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/paxdb.organism:9606: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/glida.gpcr:ACM1_HUMAN: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/adw:Lycalopex_vetulus: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/grsdb:10142: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/glida.ligand:L000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/fsnp:rs17852708: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/px:PXD000500: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hdr:63: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biomodels.vocabulary:rateRule: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/norine:NOR00681: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ordb:8497: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/p3db.site:65: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/odor:74: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cattleqtldb:4685: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/p3db.protein:70: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/topdb:AP00378: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/GRO\\:0007133: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/chickenqtldb:14362: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mgnify.samp:SRS086444: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sheepqtldb:19803: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ega.study:EGAS00000000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pigqtldb:14: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ega.dataset:EGAD00000000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/(P)?EO\\:0007404: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/idot:identifierPattern: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/antibodyregistry:493771: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mamo:MAMO_0000026: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pride.project:PXD000440: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/yeastintron:SNR17A: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hgnc.family:PADI: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ardb:CAE46076: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/proteomicsdb.protein:53504: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hpm.protein:1968: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/proteomicsdb.peptide:53504: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/drugbankv4.target:BE0000048: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/foodb.compound:FDB002100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hpm.peptide:9606117: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/zinc:ZINC1084: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/unii:3G6A5W338E: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/orphanet.ordo:Orphanet_C023: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/psipar:PAR:0116: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mgnify.proj:ERP004492: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/clinvar.record:RCV000033555.3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/euclinicaltrials:2008-005144-16: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/google.patent:US4145692: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/uspto:4145692: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/exac.variant:22-46615880-T-C: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cpc:A01M1/026: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gwascentral.study:HGVST1828: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gwascentral.phenotype:HGVPM623: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gwascentral.marker:HGVM15354: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bigg.metabolite:12dgr161: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lincs.cell:LCL-2085: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lincs.protein:200282: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lincs.smallmolecule:LSM-6306: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/exac.transcript:ENST00000407236: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/SLM:000048885: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/exac.gene:ENSG00000169174: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/wikidata:Q2207226: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/unipathway.compound:UPC04349: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bigg.model:iECABU_c1320: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/seed:Biotin_biosynthesis: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/seed.compound:cpd15380: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bigg.compartment:c: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bigg.reaction:13GS: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/umls:C0021390: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/emdb:EMD-1001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mesh:C000100: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mirtarbase:MIRT000002: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/meddra:10015919: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dashr:hsa-mir-200a: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/splash:splash10-0zq2000000-77302b0326a418630a84: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/metanetx.reaction:MNXR101574: {'metadata': [{'@context': 'https://schema.org', '@type': 'DataCatalog', 'description': 'Reconciliation of metabolites and biochemical reactions for automated model construction and genome annotation for large-scale metabolic networks', 'keywords': 'metabolism, network, model, metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MetaNetX/MNXref', 'provider': [{'@id': 'https://www.sib.swiss/', '@type': 'Organization', 'name': 'SIB Swiss Intitute of Bioinformatics'}, {'@id': 'https://www.vital-it.ch/', '@type': 'Organization', 'name': 'Vital-IT', 'logo': 'https://vital-it.ch/images/vital-it_logo.png'}], 'url': 'https://www.metanetx.org/', 'citation': [{'@id': 'https://dx.doi.org/10.1093/nar/gkv1117', '@type': 'CreativeWork', 'name': 'MetaNetX/MNXref - reconciliation of metabolites and biochemical reactions to bring together genome-scale metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bioinformatics/btt036', '@type': 'CreativeWork', 'name': 'MetaNetX.org: a website and repository for accessing, analysing and manipulating metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bib/bbs058', '@type': 'CreativeWork', 'name': 'Reconciliation of metabolites and biochemical reactions for metabolic networks'}], 'dataset': [{'@context': 'https://schema.org', '@type': 'Dataset', 'description': 'The MNXref reconciliation of metabolites and biochemical reactions namespace', 'identifier': [], 'keywords': 'metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MNXref namespace', 'url': 'ftp://ftp.vital-it.ch/databases/metanetx/MNXref/latest/', 'license': 'https://creativecommons.org/licenses/by/4.0/'}], 'dateCreated': '2011', 'license': {'@id': 'https://creativecommons.org/licenses/by/4.0/', '@type': 'CreativeWork', 'name': 'Creative Commons Attribution 4.0 International License'}, 'fileFormat': ['text/plain', 'text/html', 'application/sbml+xml', 'chemical/x-inchi', 'chemical/x-daylight-smiles', 'image/png']}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/metanetx.chemical:MNXM1723: {'metadata': [{'@context': 'https://schema.org', '@type': 'DataCatalog', 'description': 'Reconciliation of metabolites and biochemical reactions for automated model construction and genome annotation for large-scale metabolic networks', 'keywords': 'metabolism, network, model, metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MetaNetX/MNXref', 'provider': [{'@id': 'https://www.sib.swiss/', '@type': 'Organization', 'name': 'SIB Swiss Intitute of Bioinformatics'}, {'@id': 'https://www.vital-it.ch/', '@type': 'Organization', 'name': 'Vital-IT', 'logo': 'https://vital-it.ch/images/vital-it_logo.png'}], 'url': 'https://www.metanetx.org/', 'citation': [{'@id': 'https://dx.doi.org/10.1093/nar/gkv1117', '@type': 'CreativeWork', 'name': 'MetaNetX/MNXref - reconciliation of metabolites and biochemical reactions to bring together genome-scale metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bioinformatics/btt036', '@type': 'CreativeWork', 'name': 'MetaNetX.org: a website and repository for accessing, analysing and manipulating metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bib/bbs058', '@type': 'CreativeWork', 'name': 'Reconciliation of metabolites and biochemical reactions for metabolic networks'}], 'dataset': [{'@context': 'https://schema.org', '@type': 'Dataset', 'description': 'The MNXref reconciliation of metabolites and biochemical reactions namespace', 'identifier': [], 'keywords': 'metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MNXref namespace', 'url': 'ftp://ftp.vital-it.ch/databases/metanetx/MNXref/latest/', 'license': 'https://creativecommons.org/licenses/by/4.0/'}], 'dateCreated': '2011', 'license': {'@id': 'https://creativecommons.org/licenses/by/4.0/', '@type': 'CreativeWork', 'name': 'Creative Commons Attribution 4.0 International License'}, 'fileFormat': ['text/plain', 'text/html', 'application/sbml+xml', 'chemical/x-inchi', 'chemical/x-daylight-smiles', 'image/png']}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dashr.expression:hsa-mir-200a: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/metanetx.compartment:MNXC15: {'metadata': [{'@context': 'https://schema.org', '@type': 'DataCatalog', 'description': 'Reconciliation of metabolites and biochemical reactions for automated model construction and genome annotation for large-scale metabolic networks', 'keywords': 'metabolism, network, model, metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MetaNetX/MNXref', 'provider': [{'@id': 'https://www.sib.swiss/', '@type': 'Organization', 'name': 'SIB Swiss Intitute of Bioinformatics'}, {'@id': 'https://www.vital-it.ch/', '@type': 'Organization', 'name': 'Vital-IT', 'logo': 'https://vital-it.ch/images/vital-it_logo.png'}], 'url': 'https://www.metanetx.org/', 'citation': [{'@id': 'https://dx.doi.org/10.1093/nar/gkv1117', '@type': 'CreativeWork', 'name': 'MetaNetX/MNXref - reconciliation of metabolites and biochemical reactions to bring together genome-scale metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bioinformatics/btt036', '@type': 'CreativeWork', 'name': 'MetaNetX.org: a website and repository for accessing, analysing and manipulating metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bib/bbs058', '@type': 'CreativeWork', 'name': 'Reconciliation of metabolites and biochemical reactions for metabolic networks'}], 'dataset': [{'@context': 'https://schema.org', '@type': 'Dataset', 'description': 'The MNXref reconciliation of metabolites and biochemical reactions namespace', 'identifier': [], 'keywords': 'metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MNXref namespace', 'url': 'ftp://ftp.vital-it.ch/databases/metanetx/MNXref/latest/', 'license': 'https://creativecommons.org/licenses/by/4.0/'}], 'dateCreated': '2011', 'license': {'@id': 'https://creativecommons.org/licenses/by/4.0/', '@type': 'CreativeWork', 'name': 'Creative Commons Attribution 4.0 International License'}, 'fileFormat': ['text/plain', 'text/html', 'application/sbml+xml', 'chemical/x-inchi', 'chemical/x-daylight-smiles', 'image/png']}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/unipathway.reaction:UCR00226: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sasbdb:SASDAX8: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/HP:0000118: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/hgnc.genefamily:141: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mdm:4776: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/apid.interactions:P01116: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/probonto:PROB_c0000005: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/storedb:STOREDB:STUDY1040: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kegg:hsa00190: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/MP:0005452: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/planttfdb:Ath_AT1G01030.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/multicellds.collection:MCDS_C_0000000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/multicellds.cell_line:MCDS_L_0000000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/multicellds.snapshot:MCDS_S_0000000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ecyano.entity:23: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lincs.data:LDS-1110: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ecyano.model:26: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ecyano.rule:56: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/agricola:50018: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/(ark\\:/12345/fk1234: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/MS:1000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ENVO:09200010: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/grid:grid.225360.0: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/medgen:760050: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/clinvar:12345: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/clinvar.submission:SCV000151292: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/MIR:00100037: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cellosaurus:CVCL_0030: {'metadata': [{'@context': ['https://schema.org/'], '@type': 'Organization', 'url': 'https://web.expasy.org/cellosaurus', 'name': 'Cellosaurus', 'memberof': {'@type': 'NGO', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://sib.swiss/'}, 'parentOrganization': {'@type': 'Organization', 'name': 'ExPASy', 'url': 'https://www.expasy.org/'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/pgx:icdom:8500_3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/iao:0000030: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dbgap:phs000768.v2.p1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cosmic:BRAF: {'metadata': [[{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'COSMIC', 'description': \"COSMIC, the Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensive resource for exploring the impact of somatic mutations in human cancer\", 'version': '90', 'identifier': '10.1093/nar/gkw1121', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'https://cancer.sanger.ac.uk/cosmic/download'}, 'sourceOrganization': 'Wellcome Sanger Institute', 'datePublished': '2019-09-05'}, {'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'https://cancer.sanger.ac.uk/cosmic', 'name': 'COSMIC: Catalogue of Somatic Mutations in Cancer', 'description': \"COSMIC, the Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensive resource for exploring the impact of somatic mutations in human cancer\", 'potentialAction': {'@type': 'SearchAction', 'target': 'https://cancer.sanger.ac.uk/cosmic/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}, {'@context': 'http://schema.org', '@type': 'Organization', 'name': 'COSMIC: Catalogue of Somatic Mutations in Cancer', 'description': \"COSMIC, the Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensive resource for exploring the impact of somatic mutations in human cancer\", 'url': 'https://cancer.sanger.ac.uk/cosmic', 'logo': 'https://cancer.sanger.ac.uk/images/banners/cosmic_banner_328x68.png', 'sameAs': ['https://www.facebook.com/cosmic.sanger', 'https://twitter.com/cosmic_sanger', 'https://uk.linkedin.com/company/cosmic-sanger']}]]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/omit:0005506: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/go.ref:GO_REF:0000041: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/insdc.cds:AAA35559: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gdc:ae8c77fe-e6c8-44d5-8265-4a38c637bbef: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/metacyc.reaction:RXN-14904: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/glytoucan:G00054MO: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/metacyc.compound:CPD-10330: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/d1id:00030692-0FE1-4A1B-955E-A2E55D659267: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mmp.ref:MMP3312982.2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cath:1.10.8.10: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mmp.cat:MMP743597.11: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mmp.db:MMP02954345.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aop:98: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biotools:uniprotkb: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ngl:GLDS-141: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/arraymap:icdom:8500_3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/begdb:4214: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/erv:THE1B: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/napdi:28: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/crisprdb:551115: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mzspec:PXD002255::ES_XP_Ubi_97H_HCD_349:scan:9617:LAEIYVNSSFYK/2: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sugarbind:lectins/172: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sisu:17:41223048: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/licebase:EMLSAT00000003403: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gnpis:AY109603: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rnacentral:URS0000759CF4: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/microscope:5601141: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/swissregulon:AHR: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/validatordb:ByStructure/2h6o: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biominder:aef4c195-9cf9-46db-a12a-7cfd1ff3eec3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/neurovault.collection:3304: {'metadata': [{'@context': 'https://schema.org', '@type': 'Organization', 'name': 'NeuroVault', 'url': 'https://neurovault.org', 'logo': 'https://neurovault.org/static/images/neurovault-logo.png', 'sameAs': ['https://en.wikipedia.org/wiki/NeuroVault', 'https://twitter.com/vaultneuro']}, {'@context': 'https://schema.org', '@type': 'WebSite', 'url': 'https://neurovault.org/', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://neurovault.org/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vmhmetabolite:h2o: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/NMR:1000003: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/vmhreaction:HEX1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ascl:1801.012: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/facebase:FB00000917: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/eu89h:jrc-eurl-ecvam-chemagora: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gudmap:Q-2958: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/minid:b97957: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mex:36: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mw.study:ST000900: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gpmdb:GPM32310002988: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mw.project:PR000001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/massive:MSV000082131: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/fplx:RAS: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aop.events:3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aop.stressor:9: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/aop.relationships:5: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/swh:1:rev:309cf2674ee7a0749978cf8265ab91a60aea0f7d: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dev.ga4ghdos:23fa7b4b-9d68-429b-aece-658b11124bb3#2017-03-24T18:43:16.886826-05:00: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/comptox:DTXSID2021028: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/complexportal:CPX-2158: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ligandbook:785: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/rbk:Q-2958: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/kaggle:nasa/kepler-exoplanet-search-results: {'metadata': [{'@context': 'http://schema.org/', '@type': 'Dataset', 'name': 'Kepler Exoplanet Search Results', 'description': '### Context\\n\\nThe Kepler Space Observatory is a NASA-build satellite that was launched in 2009. The telescope is dedicated to searching for exoplanets in star systems besides our own, with the ultimate goal of possibly finding other habitable planets besides our own. The original mission ended in 2013 due to mechanical failures, but the telescope has nevertheless been functional since 2014 on a \"K2\" extended mission.\\n\\nKepler had verified 1284 new exoplanets as of May 2016. As of October 2017 there are over 3000 confirmed exoplanets total (using all detection methods, including ground-based ones). The telescope is still active and continues to collect new data on its extended mission.\\n\\n### Content\\n\\nThis dataset is a cumulative record of all observed Kepler \"objects of interest\" ‚Äî basically, all of the approximately 10,000 exoplanet candidates Kepler has taken observations on.\\n\\nThis dataset has an extensive data dictionary, which can be accessed [here](https://exoplanetarchive.ipac.caltech.edu/docs/API_kepcandidate_columns.html). Highlightable columns of note are:\\n\\n* `kepoi_name`: A KOI is a target identified by the Kepler Project that displays at least one transit-like sequence within Kepler time-series photometry that appears to be of astrophysical origin and initially consistent with a planetary transit hypothesis\\n* `kepler_name`: [These names] are intended to clearly indicate a class of objects that have been confirmed or validated as planets‚Äîa step up from the planet candidate designation.\\n* `koi_disposition`: The disposition in the literature towards this exoplanet candidate. One of CANDIDATE, FALSE POSITIVE, NOT DISPOSITIONED or CONFIRMED.\\n* `koi_pdisposition`: The disposition Kepler data analysis has towards this exoplanet candidate. One of FALSE POSITIVE, NOT DISPOSITIONED, and CANDIDATE.\\n* `koi_score `: A value between 0 and 1 that indicates the confidence in the KOI disposition. For CANDIDATEs, a higher value indicates more confidence in its disposition, while for FALSE POSITIVEs, a higher value indicates less confidence in that disposition.\\n\\n### Acknowledgements\\n\\nThis dataset was published as-is by NASA. You can access the original table [here](https://exoplanetarchive.ipac.caltech.edu/cgi-bin/TblView/nph-tblView?app=ExoTbls&amp;config=koi). More data from the Kepler mission is available from the same source [here](https://exoplanetarchive.ipac.caltech.edu/docs/data.html).\\n\\n### Inspiration\\n\\n* How often are exoplanets confirmed in the existing literature disconfirmed by measurements from Kepler? How about the other way round?\\n* What general characteristics about exoplanets (that we can find) can you derive from this dataset?\\n* What exoplanets get assigned names in the literature? What is the distribution of confidence scores?\\n\\nSee also: the [Kepler Labeled Time Series](https://www.kaggle.com/keplersmachines/kepler-labelled-time-series-data) and [Open Exoplanets Catalogue](https://www.kaggle.com/mrisdal/open-exoplanet-catalogue) datasets.', 'url': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results', 'sameAs': '', 'version': 2, 'keywords': ['society and social sciences &gt; society &gt; business', 'natural and physical sciences', 'technology and applied sciences', 'natural and physical sciences &gt; physical sciences &gt; astronomy', 'natural and physical sciences &gt; physical sciences &gt; space'], 'license': {'@type': 'CreativeWork', 'name': 'CC0: Public Domain', 'url': 'https://creativecommons.org/publicdomain/zero/1.0/'}, 'identifier': '2894', 'includedInDataCatalog': {'@type': 'DataCatalog', 'name': 'Kaggle', 'url': 'https://www.kaggle.com'}, 'creator': {'@type': 'Organization', 'name': 'NASA', 'url': 'https://www.kaggle.com/nasa', 'image': 'https://storage.googleapis.com/kaggle-organizations/147/thumbnail.jpg%3Fr=540'}, 'distribution': [{'@type': 'DataDownload', 'requiresSubscription': True, 'encodingFormat': 'zip', 'fileFormat': 'zip', 'contentUrl': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results/download', 'contentSize': '3695322 bytes'}, {'@type': 'DataDownload', 'requiresSubscription': True, 'encodingFormat': 'csv', 'fileFormat': 'csv', 'contentUrl': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results/download/wYdIhbkWbDC9F8oUvzJ4%2Fversions%2Fde2ssCyY0qPAdJtd3agp%2Ffiles%2Fcumulative.csv?datasetVersionNumber=1', 'contentSize': '3695322 bytes'}], 'commentCount': 0, 'dateModified': '2017-10-10T18:26:59.497', 'discussionUrl': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results/discussion', 'alternateName': '10000 exoplanet candidates examined by the Kepler Space Observatory', 'isAccessibleForFree': True, 'thumbnailUrl': 'https://storage.googleapis.com/kaggle-datasets-images/2894/4876/afd561892005209a55d97b68deb977fe/dataset-card.jpg', 'interactionStatistic': [{'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/CommentAction', 'userInteractionCount': 0}, {'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/DownloadAction', 'userInteractionCount': 2533}, {'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/ViewAction', 'userInteractionCount': 22693}, {'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/LikeAction', 'userInteractionCount': 119}]}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/idoo:DataCollection: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oci:1-18: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/FOODON:03307879: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ricenetdb.gene:LOC_Os01g49190.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ricenetdb.protein:LOC_Os01g49190: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gtex:BRIP1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ricenetdb.compound:OSC1416: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ricenetdb.reaction:OSR0818: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ricenetdb.mirna:osa-miR446: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/encode:ENCSR163RYW: {'metadata': [{'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': {'classification': 'cell line', 'aliases': [], 'dbxrefs': ['Cellosaurus:CVCL_0045'], 'status': 'released', 'organ_slims': ['epithelium', 'kidney'], 'developmental_slims': ['mesoderm'], '@id': '/biosample-types/cell_line_EFO_0001182/', 'name': 'cell_line_EFO_0001182', 'synonyms': ['human embryonic kidney cell'], 'term_name': 'HEK293', 'references': [], 'system_slims': ['excretory system'], 'uuid': '014c9be0-20d4-4df5-b816-b0dca4cfee9d', 'cell_slims': ['epithelial cell'], '@type': ['BiosampleType', 'Item'], 'schema_version': '1', 'term_id': 'EFO:0001182'}, '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'audit': {'WARNING': [{'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF634VOX|/files/ENCFF634VOX/} was generated from a library with PBC1 value of 0.82.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF634VOX|/files/ENCFF634VOX/} was generated from a library with PBC2 value of 5.54.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF431LTO|/files/ENCFF431LTO/} was generated from a library with PBC1 value of 0.84.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF431LTO|/files/ENCFF431LTO/} was generated from a library with PBC2 value of 6.09.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF014SGL|/files/ENCFF014SGL/} was generated from a library with PBC1 value of 0.84.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF014SGL|/files/ENCFF014SGL/} was generated from a library with PBC2 value of 6.09.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF332OVN|/files/ENCFF332OVN/} was generated from a library with PBC1 value of 0.82.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF332OVN|/files/ENCFF332OVN/} was generated from a library with PBC2 value of 5.54.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'missing genetic modification characterization', 'detail': 'Genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} performed for the purpose of tagging is missing validating characterization that is required by ENCODE4 standards.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'missing biosample characterization', 'detail': 'Biosample {ENCBS111GOI|/biosamples/ENCBS111GOI/} which has been modified by genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} is missing characterization validating the modification, Biosample {ENCBS082THU|/biosamples/ENCBS082THU/} which has been modified by genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} is missing characterization validating the modification.', 'path': '/experiments/ENCSR163RYW/'}], 'INTERNAL_ACTION': [{'level': 30, 'level_name': 'INTERNAL_ACTION', 'name': 'audit_genetic_modification_target', 'category': 'inconsistent modification target', 'detail': 'Genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} is targeting {eGFP-ZNF189-human|/targets/eGFP-ZNF189-human/} which has already been modified. Should consider using unmodified target version for genetic modification.', 'path': '/genetic-modifications/ENCGM615AHN/'}, {'level': 30, 'level_name': 'INTERNAL_ACTION', 'name': 'audit_experiment', 'category': 'inconsistent experiment target', 'detail': 'Experiment {ENCSR163RYW|/experiments/ENCSR163RYW/} has a tagged target {eGFP-ZNF189-human|/targets/eGFP-ZNF189-human/}. Should consider using untagged target version for experiment.', 'path': '/experiments/ENCSR163RYW/'}]}, 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': [{'related_datasets': ['/experiments/ENCSR429LAP/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR688NWK/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR157LMX/'], 'alternate_accessions': [], 'accession': 'ENCSR300VIY', 'assembly': ['GRCh38', 'hg19'], 'documents': [], 'hub': '/matched-sets/ENCSR300VIY/@@hub/hub.txt', 'description': 'This set represents the constituent input control libraries sequenced on HiSeq 4000 platform from which a pooled and subsampled control is derived from for the processing of the ChIP-seq experiments done on eGFP-tagged zinc fingers in the HEK293 cell line', 'date_released': '2016-10-07', 'assay_synonyms': ['ChIP-seq assay', 'chromatin immunoprecipitation sequencing assay'], 'organism': ['/organisms/human/'], 'files': ['/files/ENCFF810LLR/', '/files/ENCFF182WHU/'], 'control_type': ['input library'], 'assay_term_id': ['OBI:0000716'], 'aliases': ['michael-snyder:HEK_ZNF_HiSeq4000_control_pool'], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'assay_term_name': ['ChIP-seq'], 'date_created': '2016-08-02T18:59:49.859024+00:00', 'revoked_datasets': [], 'month_released': 'October, 2016', 'lab': '/labs/j-michael-cherry/', '@id': '/matched-sets/ENCSR300VIY/', 'treatment_term_name': [], 'biosample_ontology': ['/biosample-types/cell_line_EFO_0001182/'], 'dbxrefs': [], 'revoked_files': [], 'references': [], 'target': [], 'uuid': '413eaaf8-c433-43dc-aba4-d2d34f148165', 'submitted_by': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'status': 'released', 'internal_tags': [], 'original_files': ['/files/ENCFF810LLR/', '/files/ENCFF182WHU/'], '@type': ['MatchedSet', 'Series', 'Dataset', 'Item'], 'contributing_files': ['/files/ENCFF145UQB/', '/files/ENCFF909YLP/', '/files/ENCFF058AVH/', '/files/ENCFF712WCF/', '/files/ENCFF945YPE/', '/files/ENCFF470DTH/', '/files/ENCFF083WCY/', '/files/ENCFF683AIZ/', '/files/ENCFF370HPX/', '/files/ENCFF876NRR/', '/files/ENCFF848QHI/', '/files/ENCFF361NSP/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR157LMX/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZEB2_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE92227'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': '92beb008-a2c0-4703-92d1-2133939b3504', 'original_files': ['/files/ENCFF557HNG/', '/files/ENCFF453HUS/', '/files/ENCFF370HPX/', '/files/ENCFF045OVA/', '/files/ENCFF087ZSB/', '/files/ENCFF083WCY/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZEB2 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR157LMX/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF557HNG/', '/files/ENCFF453HUS/', '/files/ENCFF370HPX/', '/files/ENCFF045OVA/', '/files/ENCFF087ZSB/', '/files/ENCFF083WCY/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.612597+00:00', 'accession': 'ENCSR157LMX', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/d4ff5fe7-e217-4d27-b541-c6ef5de53e3e/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR274OGE/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-CTCF_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE92228'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': '4b8770d1-5c0d-457a-b559-f928928e0c16', 'original_files': ['/files/ENCFF196VCR/', '/files/ENCFF173EDE/', '/files/ENCFF348IEF/', '/files/ENCFF058AVH/', '/files/ENCFF736ULL/', '/files/ENCFF848QHI/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-CTCF tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR274OGE/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF196VCR/', '/files/ENCFF173EDE/', '/files/ENCFF348IEF/', '/files/ENCFF058AVH/', '/files/ENCFF736ULL/', '/files/ENCFF848QHI/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:08.409664+00:00', 'accession': 'ENCSR274OGE', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/e4e51ac6-910f-4085-99f2-03dc9bbd585a/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR342AGU/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZSCAN5A_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE91681'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': 'de6dcce4-6bca-4f17-a8a4-75a3188f4c4a', 'original_files': ['/files/ENCFF172JDP/', '/files/ENCFF154SYW/', '/files/ENCFF719PXX/', '/files/ENCFF712WCF/', '/files/ENCFF944AKG/', '/files/ENCFF683AIZ/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZSCAN5A tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR342AGU/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF172JDP/', '/files/ENCFF154SYW/', '/files/ENCFF719PXX/', '/files/ENCFF712WCF/', '/files/ENCFF944AKG/', '/files/ENCFF683AIZ/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.394975+00:00', 'accession': 'ENCSR342AGU', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/7864e158-166b-426c-ae95-9cdaa43e57cb/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR354FNX/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZNF18_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE91693'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': 'c013defe-b3f0-4687-94ed-03759899023f', 'original_files': ['/files/ENCFF467VIQ/', '/files/ENCFF085HHB/', '/files/ENCFF771QFT/', '/files/ENCFF470DTH/', '/files/ENCFF348BKF/', '/files/ENCFF876NRR/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZNF18 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR354FNX/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF467VIQ/', '/files/ENCFF085HHB/', '/files/ENCFF771QFT/', '/files/ENCFF470DTH/', '/files/ENCFF348BKF/', '/files/ENCFF876NRR/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.155529+00:00', 'accession': 'ENCSR354FNX', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/b12b2202-3e9b-4f1c-a4af-1422616a63b0/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR429LAP/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-SP7_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE91755'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': 'd472964f-0a6f-4695-8cd4-2e3c4c42fe76', 'original_files': ['/files/ENCFF736BPA/', '/files/ENCFF595HTZ/', '/files/ENCFF084JUL/', '/files/ENCFF945YPE/', '/files/ENCFF854QOQ/', '/files/ENCFF909YLP/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-SP7 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR429LAP/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF736BPA/', '/files/ENCFF595HTZ/', '/files/ENCFF084JUL/', '/files/ENCFF945YPE/', '/files/ENCFF854QOQ/', '/files/ENCFF909YLP/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.808450+00:00', 'accession': 'ENCSR429LAP', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/ff90e222-fa5c-47d3-a8cb-d21f05937bd8/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR688NWK/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZFP62_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE92232'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': '61b405a8-880f-45de-8067-2d5292f657ce', 'original_files': ['/files/ENCFF952FAG/', '/files/ENCFF956RLS/', '/files/ENCFF424ATY/', '/files/ENCFF145UQB/', '/files/ENCFF650VUY/', '/files/ENCFF361NSP/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZFP62 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR688NWK/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF952FAG/', '/files/ENCFF956RLS/', '/files/ENCFF424ATY/', '/files/ENCFF145UQB/', '/files/ENCFF650VUY/', '/files/ENCFF361NSP/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'Control ChIP-seq', 'date_created': '2016-07-29T23:26:08.057247+00:00', 'accession': 'ENCSR688NWK', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': 'WARNING: The tagged TF could not be verified by AP-MS and may not reflect the annotated zinc finger target in the metadata.', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/629454e0-aadb-4844-9abc-3cf56b23e228/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], 'visualize': {'GRCh38': ['UCSC', 'Ensembl'], 'hg19': ['UCSC']}, '@id': '/experiments/ENCSR163RYW/', '@context': '/terms/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': [{'technical_replicates': ['1_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1'], 'libraries': ['9c03b3bd-4cce-438e-ab53-02d96e191ac8'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', 'library': '/libraries/ENCLB195KAR/', 'uuid': 'f26d90d9-53e2-4dff-8b98-c11d3fdb2831', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'biological_replicate_number': 1, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB976SGG/', 'date_created': '2016-12-14T21:13:52.766303+00:00'}, 'read_length_units': 'nt', 'read_count': 58871808, '@id': '/files/ENCFF271YPD/', 'cloud_metadata': {'file_size': 3571917417, 'url': 'https://encode-public.s3.amazonaws.com/2017/01/21/c7b7b658-9281-47ac-8a72-155f105a4c0f/ENCFF271YPD.fastq.gz', 'md5sum_base64': 'aMYg8ojVdYjpHang9fYMtg=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/01/21/c7b7b658-9281-47ac-8a72-155f105a4c0f/ENCFF271YPD.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1-read1'], 'schema_version': '15', 'md5sum': '68c620f288d57588e91da9e0f5f60cb6', 'content_md5sum': 'c6195fcae58cf9b2d55d13a35ebaffa3', 'href': '/files/ENCFF271YPD/@@download/ENCFF271YPD.fastq.gz', 'read_length': 101, 'title': 'ENCFF271YPD', 'library': '/libraries/ENCLB195KAR/', 'superseded_by': [], 'file_size': 3571917417, 'uuid': 'c7b7b658-9281-47ac-8a72-155f105a4c0f', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_160406_HEK-ZNF075_H7KY2_CAGATC_R1.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'CAGATC', 'flowcell': 'AH7KY2BBXX', 'lane': '3'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF736BPA/', '/files/ENCFF196VCR/', '/files/ENCFF154SYW/', '/files/ENCFF952FAG/', '/files/ENCFF557HNG/', '/files/ENCFF085HHB/'], 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'paired_with': '/files/ENCFF456PAU/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-01-21T00:47:20.855848+00:00', 'biological_replicates': [1], 'accession': 'ENCFF271YPD', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '1', 'dbxrefs': ['SRA:SRR8660124'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['H7KY2BBXX:3:1:AAGATC:', 'H7KY2BBXX:3:1:CAAATC:', 'H7KY2BBXX:3:1:CACATC:', 'H7KY2BBXX:3:1:CAGAAC:', 'H7KY2BBXX:3:1:CAGACC:', 'H7KY2BBXX:3:1:CAGAGC:', 'H7KY2BBXX:3:1:CAGATA:', 'H7KY2BBXX:3:1:CAGATC:', 'H7KY2BBXX:3:1:CAGATG:', 'H7KY2BBXX:3:1:CAGATN:', 'H7KY2BBXX:3:1:CAGATT:', 'H7KY2BBXX:3:1:CAGCTC:', 'H7KY2BBXX:3:1:CAGGTC:', 'H7KY2BBXX:3:1:CAGNTC:', 'H7KY2BBXX:3:1:CAGTTC:', 'H7KY2BBXX:3:1:CATATC:', 'H7KY2BBXX:3:1:CCGATC:', 'H7KY2BBXX:3:1:CGGATC:', 'H7KY2BBXX:3:1:CNGATC:', 'H7KY2BBXX:3:1:CTGATC:', 'H7KY2BBXX:3:1:GAGATC:', 'H7KY2BBXX:3:1:NAGATC:', 'H7KY2BBXX:3:1:TAGATC:']}, {'technical_replicates': ['1_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1'], 'libraries': ['9c03b3bd-4cce-438e-ab53-02d96e191ac8'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', 'library': '/libraries/ENCLB195KAR/', 'uuid': 'f26d90d9-53e2-4dff-8b98-c11d3fdb2831', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'biological_replicate_number': 1, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB976SGG/', 'date_created': '2016-12-14T21:13:52.766303+00:00'}, 'read_length_units': 'nt', 'read_count': 58871808, '@id': '/files/ENCFF456PAU/', 'cloud_metadata': {'file_size': 3718247071, 'url': 'https://encode-public.s3.amazonaws.com/2017/01/21/5987964c-5e9a-4d53-8106-df4bd0779e8b/ENCFF456PAU.fastq.gz', 'md5sum_base64': 'cbIh6Ge1gi24Mte6eo/VAw=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/01/21/5987964c-5e9a-4d53-8106-df4bd0779e8b/ENCFF456PAU.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1-read2'], 'schema_version': '15', 'md5sum': '71b221e867b5822db832d7ba7a8fd503', 'content_md5sum': 'd50217375de779a8e9bf10571e7b4f68', 'href': '/files/ENCFF456PAU/@@download/ENCFF456PAU.fastq.gz', 'read_length': 101, 'title': 'ENCFF456PAU', 'library': '/libraries/ENCLB195KAR/', 'superseded_by': [], 'file_size': 3718247071, 'uuid': '5987964c-5e9a-4d53-8106-df4bd0779e8b', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_160406_HEK-ZNF075_H7KY2_CAGATC_R2.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'CAGATC', 'flowcell': 'AH7KY2BBXX', 'lane': '3'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF595HTZ/', '/files/ENCFF173EDE/', '/files/ENCFF172JDP/', '/files/ENCFF956RLS/', '/files/ENCFF467VIQ/', '/files/ENCFF453HUS/'], 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'paired_with': '/files/ENCFF271YPD/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-01-21T00:49:54.579738+00:00', 'biological_replicates': [1], 'accession': 'ENCFF456PAU', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '2', 'dbxrefs': ['SRA:SRR8660124'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['H7KY2BBXX:3:2:AAGATC:', 'H7KY2BBXX:3:2:CAAATC:', 'H7KY2BBXX:3:2:CACATC:', 'H7KY2BBXX:3:2:CAGAAC:', 'H7KY2BBXX:3:2:CAGACC:', 'H7KY2BBXX:3:2:CAGAGC:', 'H7KY2BBXX:3:2:CAGATA:', 'H7KY2BBXX:3:2:CAGATC:', 'H7KY2BBXX:3:2:CAGATG:', 'H7KY2BBXX:3:2:CAGATN:', 'H7KY2BBXX:3:2:CAGATT:', 'H7KY2BBXX:3:2:CAGCTC:', 'H7KY2BBXX:3:2:CAGGTC:', 'H7KY2BBXX:3:2:CAGNTC:', 'H7KY2BBXX:3:2:CAGTTC:', 'H7KY2BBXX:3:2:CATATC:', 'H7KY2BBXX:3:2:CCGATC:', 'H7KY2BBXX:3:2:CGGATC:', 'H7KY2BBXX:3:2:CNGATC:', 'H7KY2BBXX:3:2:CTGATC:', 'H7KY2BBXX:3:2:GAGATC:', 'H7KY2BBXX:3:2:NAGATC:', 'H7KY2BBXX:3:2:TAGATC:']}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/raw_bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '0277afa9c395f02ad0d76bdf01a08190', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KYVgQ03696fqK09j81B2fx\", \"executable\": \"applet-F2pzVVj036944Z350k0FZzYQ\", \"user\": \"user-estherc\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [178276, 0], \"read1\": [58871808, 0], \"singletons\": [505233, 0], \"read2\": [58871808, 0], \"properly_paired\": [111694512, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [112123515, 0], \"mate_mapped_different_chr\": [228953, 0], \"in_total\": [117743616, 0], \"mapped\": [112628748, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY6qQ0369PfB8QBkj0X54b\", \"dx-id\": \"file-F3KfJk80g3F01z8BP7q7Q45Y\"}', '@id': '/files/ENCFF107GHX/', 'cloud_metadata': {'file_size': 6739158178, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/5fe9ef1b-0532-4c8e-b2ed-8f7a2a990ad6/ENCFF107GHX.bam', 'md5sum_base64': 'AnevqcOV8CrQ12vfAaCBkA=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': '5fe9ef1b-0532-4c8e-b2ed-8f7a2a990ad6', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/9820a1d7-427d-4619-85ec-b5dabef7c6e1/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-03-31T04:36:57.153477+00:00', 'biological_replicates': [1], 'accession': 'ENCFF107GHX', 's3_uri': 's3://encode-public/2017/03/31/5fe9ef1b-0532-4c8e-b2ed-8f7a2a990ad6/ENCFF107GHX.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/c729d612-7772-4811-87e2-9039138c23ca/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF107GHX/@@download/ENCFF107GHX.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': '0289badbeed91f8edbc18b598f738751', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 6739158178, 'title': 'ENCFF107GHX', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/raw_bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '3e4899be9f571ee28fb417ca223243c7', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KYXJj0369J51928vXyPxVP\", \"executable\": \"applet-F2pzVVj036944Z350k0FZzYQ\", \"user\": \"user-estherc\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [168174, 0], \"read1\": [58871808, 0], \"singletons\": [468804, 0], \"read2\": [58871808, 0], \"properly_paired\": [112699042, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [113152761, 0], \"mate_mapped_different_chr\": [218393, 0], \"in_total\": [117743616, 0], \"mapped\": [113621565, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY7480369GpQZkPJ206Gqj\", \"dx-id\": \"file-F3KfVg00bbkYZfg207F8PyQG\"}', '@id': '/files/ENCFF027GIB/', 'cloud_metadata': {'file_size': 6745996768, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/a1af7993-4071-419c-aa6f-370bc6eac418/ENCFF027GIB.bam', 'md5sum_base64': 'PkiZvp9XHuKPtBfKIjJDxw=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': 'a1af7993-4071-419c-aa6f-370bc6eac418', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/94162c93-74e4-4eee-a55d-d97622896d06/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-03-31T04:36:33.680804+00:00', 'biological_replicates': [1], 'accession': 'ENCFF027GIB', 's3_uri': 's3://encode-public/2017/03/31/a1af7993-4071-419c-aa6f-370bc6eac418/ENCFF027GIB.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/470e798f-2521-4eca-b451-617ef5d299a1/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF027GIB/@@download/ENCFF027GIB.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'cc8dd109186a91bb50e429c2bd6daac0', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 6745996768, 'title': 'ENCFF027GIB', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': 'c5da8f36fcc4444ec381d5149040e90d', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KY7480369GpQZkPJ206Gqy\", \"executable\": \"applet-F2pzVgj03696Z6g7613j6YyB\", \"user\": \"user-estherc\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1876868\", \"phantomPeakCoef\": \"1.115102\", \"estFragLen\": \"235\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.209290053606703\", \"relPhantomPeakCoef\": \"2.057722\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1981854\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [168174, 0], \"read1\": [58871808, 0], \"singletons\": [468804, 0], \"read2\": [58871808, 0], \"properly_paired\": [112699042, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [113152761, 0], \"mate_mapped_different_chr\": [218393, 0], \"in_total\": [117743616, 0], \"mapped\": [113621565, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"275777225\", \"read_pair_optical_duplicates\": \"4540294\", \"read_pairs_examined\": \"50551963\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.161691\", \"read_pair_duplicates\": \"8173816\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"6.091213\", \"PBC1\": \"0.838029\", \"NRF\": \"0.839997\", \"OneReadPair\": \"35081621\", \"DistinctReadPairs\": \"41862075\", \"TotalReadPairs\": \"49835969\", \"TwoReadPairs\": \"5759382\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [2, 0], \"read1\": [42378147, 0], \"singletons\": [0, 0], \"read2\": [42378147, 0], \"properly_paired\": [84756294, 0], \"paired_in_sequencing\": [84756294, 0], \"with_self_mate_mapped\": [84756294, 0], \"mate_mapped_different_chr\": [2, 0], \"in_total\": [84756294, 0], \"mapped\": [84756294, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY7480369GpQZkPJ206Gqj\", \"dx-id\": \"file-F3KpQv00GBkBqxV70748Y99q\"}', '@id': '/files/ENCFF431LTO/', 'cloud_metadata': {'file_size': 5174099343, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/c4c31fa4-88de-4edf-a09e-d82759523b87/ENCFF431LTO.bam', 'md5sum_base64': 'xdqPNvzERE7DgdUUkEDpDQ=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': 'c4c31fa4-88de-4edf-a09e-d82759523b87', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/54dc81a2-f922-4592-936f-8ca65d3943ef/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-03-31T04:40:42.369433+00:00', 'biological_replicates': [1], 'accession': 'ENCFF431LTO', 's3_uri': 's3://encode-public/2017/03/31/c4c31fa4-88de-4edf-a09e-d82759523b87/ENCFF431LTO.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/afcc99be-6f7c-437c-9ba4-12929bcd5e9b/', '/samtools-flagstats-quality-metrics/51966ee3-048e-40d8-8b79-a2ca31000940/', '/chipseq-filter-quality-metrics/9f34f438-ed83-4c59-b54d-223b1d2eefa2/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF431LTO/@@download/ENCFF431LTO.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': '4965dea8a57851ba2467a038a066cb8a', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5174099343, 'title': 'ENCFF431LTO', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '0f84bd10c4a3cc82ba46fbd641f46ade', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KY6qj0369PfB8QBkj0X54p\", \"executable\": \"applet-F2pzVgj03696Z6g7613j6YyB\", \"user\": \"user-estherc\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1885157\", \"phantomPeakCoef\": \"1.116719\", \"estFragLen\": \"230\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.210519113863597\", \"relPhantomPeakCoef\": \"1.988074\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1995834\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [178276, 0], \"read1\": [58871808, 0], \"singletons\": [505233, 0], \"read2\": [58871808, 0], \"properly_paired\": [111694512, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [112123515, 0], \"mate_mapped_different_chr\": [228953, 0], \"in_total\": [117743616, 0], \"mapped\": [112628748, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"275948160\", \"read_pair_optical_duplicates\": \"4537609\", \"read_pairs_examined\": \"50549702\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.161605\", \"read_pair_duplicates\": \"8169067\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"6.091527\", \"PBC1\": \"0.838035\", \"NRF\": \"0.840003\", \"OneReadPair\": \"35088137\", \"DistinctReadPairs\": \"41869560\", \"TotalReadPairs\": \"49844533\", \"TwoReadPairs\": \"5760155\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [0, 0], \"read1\": [42380635, 0], \"singletons\": [0, 0], \"read2\": [42380635, 0], \"properly_paired\": [84761270, 0], \"paired_in_sequencing\": [84761270, 0], \"with_self_mate_mapped\": [84761270, 0], \"mate_mapped_different_chr\": [0, 0], \"in_total\": [84761270, 0], \"mapped\": [84761270, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY6qQ0369PfB8QBkj0X54b\", \"dx-id\": \"file-F3Kp5300Z9pFfpYXP8ZzJXQ5\"}', '@id': '/files/ENCFF014SGL/', 'cloud_metadata': {'file_size': 5171581198, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/8b25c29d-7435-451a-aebf-1941f74ed6bd/ENCFF014SGL.bam', 'md5sum_base64': 'D4S9EMSjzIK6RvvWQfRq3g=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': '8b25c29d-7435-451a-aebf-1941f74ed6bd', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/529dca74-5983-480d-8ec6-e96760196192/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-03-31T04:41:43.592576+00:00', 'biological_replicates': [1], 'accession': 'ENCFF014SGL', 's3_uri': 's3://encode-public/2017/03/31/8b25c29d-7435-451a-aebf-1941f74ed6bd/ENCFF014SGL.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/b31e92a2-7987-41e5-87ac-0a44982bf947/', '/chipseq-filter-quality-metrics/8286d9c7-87f7-4814-ace7-a75b7de141d9/', '/samtools-flagstats-quality-metrics/2b3b0f42-6f39-431c-a3c5-1ea9fb0dc744/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF014SGL/@@download/ENCFF014SGL.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'd055e93723b8a2f73a3fa75ac5ebea62', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5171581198, 'title': 'ENCFF014SGL', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r1.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '0bc986b2db90c3602cb9826dd0d888d1', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF419YNF/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1710402244, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/9d989b64-4550-45e7-a9f8-a6aabf59dae6/ENCFF419YNF.bigWig', 'md5sum_base64': 'C8mGstuQw2AsuYJt0NiI0Q=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF419YNF/@@download/ENCFF419YNF.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-04-03T17:48:38.208625+00:00', 'biological_replicates': [1], 'accession': 'ENCFF419YNF', 's3_uri': 's3://encode-public/2017/04/03/9d989b64-4550-45e7-a9f8-a6aabf59dae6/ENCFF419YNF.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '9d989b64-4550-45e7-a9f8-a6aabf59dae6', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803690VvxZJGGVzkqv\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VPKj0GX6vKV704y76FzVZ\"}', 'file_size': 1710402244, 'title': 'ENCFF419YNF', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R1.fixcoord.regionPeak.bb', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '6a14c3b4a7cf86fe42ea2997e5cc714b', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF358KJF/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13075865, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/6c47b101-9787-47a5-8dea-88f98754a7f0/ENCFF358KJF.bigBed', 'md5sum_base64': 'ahTDtKfPhv5C6imX5cxxSw=='}, 's3_uri': 's3://encode-public/2017/04/03/6c47b101-9787-47a5-8dea-88f98754a7f0/ENCFF358KJF.bigBed', 'file_size': 13075865, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF358KJF/@@download/ENCFF358KJF.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-04-03T17:49:13.207786+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF358KJF', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': '6c47b101-9787-47a5-8dea-88f98754a7f0', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj036970ZKz5Q51x6Yf\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Q8Q00j5qZ5PXb2Bz99PKk\"}', 'output_category': 'annotation', 'title': 'ENCFF358KJF', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/6751ed59-7d57-4db2-9b60-2fbf8e9d47e2/', 'derived_from': ['/files/ENCFF269RYL/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_stable.narrowPeak.gz', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '4d496fe76ec2a66a91e0ef5975dda88f', 'fastq_signature': [], '@id': '/files/ENCFF201MWD/', 'cloud_metadata': {'file_size': 652851, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/10d9c4a5-ae97-42c3-8aab-a9327b0d90c6/ENCFF201MWD.bed.gz', 'md5sum_base64': 'TUlv527CpmqR4O9Zdd2ojw=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/04/03/10d9c4a5-ae97-42c3-8aab-a9327b0d90c6/ENCFF201MWD.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-unreplicated-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-unreplicated-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-unreplicated-idr-step-v-1/', 'step_label': 'tf-unreplicated-idr-step', 'versions': ['/analysis-step-versions/tf-unreplicated-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '19dec341-2dd2-45aa-a9ad-48dd035a8cf0', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['unreplicated IDR'], 'title': 'TF unreplicated ChIP IDR step', 'aliases': ['encode:tf-unreplicated-idr-step-v-1'], 'name': 'tf-unreplicated-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL493SGC', '@type': ['Pipeline', 'Item'], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': [], '@id': '/pipelines/ENCPL493SGC/', 'assay_term_names': ['ChIP-seq'], 'analysis_steps': ['/analysis-steps/tf-unreplicated-spp-peak-calling-step-v-1/', '/analysis-steps/tf-unreplicated-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-unreplicated-idr-step-v-1/', '/analysis-steps/tf-unreplicated-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-unreplicated-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'de0d77d8-e53b-431a-9897-41f2c5b2a4a8', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq (unreplicated)', 'aliases': [], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-21T00:14:32.126175+00:00'}], 'schema_version': '8', 'output_file_types': ['pseudoreplicated IDR thresholded peaks'], 'date_created': '2017-03-21T00:14:31.005535+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-unreplicated-idr-step-v-1-0/', 'name': 'tf-unreplicated-idr-step-v-1-0', 'uuid': 'abd4401d-96aa-4c37-894d-8e0fee9273ec', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2017-03-21T00:14:31.359299+00:00'}, 'step_run': '/analysis-step-runs/e141362b-fd75-4a69-ab2b-a8db001b515d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-04-03T17:51:15.634518+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF201MWD', 'quality_metrics': ['/idr-quality-metrics/9ffb79a5-99d7-4876-85a5-b6ba93e3fc44/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF201MWD/@@download/ENCFF201MWD.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'pseudoreplicated IDR thresholded peaks', 'status': 'archived', 'technical_replicates': ['1_1'], 'uuid': '10d9c4a5-ae97-42c3-8aab-a9327b0d90c6', 'content_md5sum': 'cbb81e3c9ca28f22e94b2c2ebaad618f', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3PxV200369KFKj3PP5gy7JQ\", \"executable\": \"applet-F2pzbpj0369FZq3f5Q148vbk\", \"user\": \"user-estherc\"}, \"qc\": {\"N1\": 36721, \"Ns\": 36718}, \"dx-parentAnalysis\": \"analysis-F3PxV200369KFKj3PP5gy7JB\", \"dx-id\": \"file-F3Q43Z80yv3gV6bYP2gybb9z\"}', 'file_size': 652851, 'title': 'ENCFF201MWD', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF419RSJ/', '/files/ENCFF646KAN/'], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r1.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'c7cda89308162c4a701264877626b571', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF626DNK/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1828023313, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/58ecf6c1-1255-49fc-98b0-43196c019218/ENCFF626DNK.bigWig', 'md5sum_base64': 'x82okwgWLEpwEmSHdia1cQ=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF626DNK/@@download/ENCFF626DNK.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-04-03T17:48:40.057941+00:00', 'biological_replicates': [1], 'accession': 'ENCFF626DNK', 's3_uri': 's3://encode-public/2017/04/03/58ecf6c1-1255-49fc-98b0-43196c019218/ENCFF626DNK.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '58ecf6c1-1255-49fc-98b0-43196c019218', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGg80369P5x8QJ512zKBG\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Pp380X2G6G7y25V92Fg23\"}', 'file_size': 1828023313, 'title': 'ENCFF626DNK', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r1.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'f12afc2d1a31fe2c4cf6e2ac0239e602', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF414FRL/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1641931334, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/b709c7e6-76ae-4ad4-8291-c8e11a696272/ENCFF414FRL.bigWig', 'md5sum_base64': '8Sr8LRox/ixM9uKsAjnmAg=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF414FRL/@@download/ENCFF414FRL.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-04-03T17:47:30.214722+00:00', 'biological_replicates': [1], 'accession': 'ENCFF414FRL', 's3_uri': 's3://encode-public/2017/04/03/b709c7e6-76ae-4ad4-8291-c8e11a696272/ENCFF414FRL.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b709c7e6-76ae-4ad4-8291-c8e11a696272', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGg80369P5x8QJ512zKBG\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88PpQ80X2G70ZKz5Q51zQ36\"}', 'file_size': 1641931334, 'title': 'ENCFF414FRL', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r1.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '4cdb045fd01caad0c03c227e55e42583', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF752LHF/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1839178351, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/b3b95384-d06b-4dff-a6fd-51c77bbd4dcd/ENCFF752LHF.bigWig', 'md5sum_base64': 'TNsEX9AcqtDAPCJ+VeQlgw=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF752LHF/@@download/ENCFF752LHF.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-04-03T17:49:52.648029+00:00', 'biological_replicates': [1], 'accession': 'ENCFF752LHF', 's3_uri': 's3://encode-public/2017/04/03/b3b95384-d06b-4dff-a6fd-51c77bbd4dcd/ENCFF752LHF.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b3b95384-d06b-4dff-a6fd-51c77bbd4dcd', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803690VvxZJGGVzkqv\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VP1j0GX6vKV704y76FzK0\"}', 'file_size': 1839178351, 'title': 'ENCFF752LHF', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R1.fixcoord.regionPeak.gz', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '649f753c7878fc7227504b1b4f4513f6', 'fastq_signature': [], '@id': '/files/ENCFF646KAN/', 'cloud_metadata': {'file_size': 7211043, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/dc6a7ce7-2d79-42eb-8c97-8f386859b10e/ENCFF646KAN.bed.gz', 'md5sum_base64': 'ZJ91PHh4/HInUEsbT0UT9g=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/04/03/dc6a7ce7-2d79-42eb-8c97-8f386859b10e/ENCFF646KAN.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/e58c3f8a-46e6-45b5-ac74-1a517c4e5de3/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-04-03T17:50:22.884131+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF646KAN', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF646KAN/@@download/ENCFF646KAN.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': 'dc6a7ce7-2d79-42eb-8c97-8f386859b10e', 'content_md5sum': 'e5846a46a60c1728f5004957932c7000', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03694GxQQ2B6F79Kq\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88Vqb00J77f3q5z2B0Yvq8B\"}', 'file_size': 7211043, 'title': 'ENCFF646KAN', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R1.fixcoord.regionPeak.bb', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'e4cd67bee3f7119fad6136ee633e4020', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF790GNI/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13038247, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/6e6012cc-f93b-41d4-8126-b1e4474f8d8a/ENCFF790GNI.bigBed', 'md5sum_base64': '5M1nvuP3EZ+tYTbuYz5AIA=='}, 's3_uri': 's3://encode-public/2017/04/03/6e6012cc-f93b-41d4-8126-b1e4474f8d8a/ENCFF790GNI.bigBed', 'file_size': 13038247, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF790GNI/@@download/ENCFF790GNI.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-04-03T17:50:28.270769+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF790GNI', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': '6e6012cc-f93b-41d4-8126-b1e4474f8d8a', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03694GxQQ2B6F79Kq\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VqZQ0J77Z5PXb2Bz99z9k\"}', 'output_category': 'annotation', 'title': 'ENCFF790GNI', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/55e555ad-73af-47b9-a82f-cb17fd71b1be/', 'derived_from': ['/files/ENCFF646KAN/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_stable.narrowPeak.bb', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '71684444b1a644fb70706e7fea030364', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF593RQD/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1237676, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/eb7722cb-5a41-43b7-b5d9-5661cc2126eb/ENCFF593RQD.bigBed', 'md5sum_base64': 'cWhERLGmRPtwcG5/6gMDZA=='}, 's3_uri': 's3://encode-public/2017/04/03/eb7722cb-5a41-43b7-b5d9-5661cc2126eb/ENCFF593RQD.bigBed', 'file_size': 1237676, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-unreplicated-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['pseudoreplicated IDR thresholded peaks'], '@id': '/analysis-steps/tf-unreplicated-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-unreplicated-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '123a9667-abb2-442a-93ed-3715c67534ea', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF unreplicated ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-unreplicated-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-unreplicated-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL493SGC', '@type': ['Pipeline', 'Item'], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': [], '@id': '/pipelines/ENCPL493SGC/', 'assay_term_names': ['ChIP-seq'], 'analysis_steps': ['/analysis-steps/tf-unreplicated-spp-peak-calling-step-v-1/', '/analysis-steps/tf-unreplicated-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-unreplicated-idr-step-v-1/', '/analysis-steps/tf-unreplicated-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-unreplicated-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'de0d77d8-e53b-431a-9897-41f2c5b2a4a8', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq (unreplicated)', 'aliases': [], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-21T00:14:32.126175+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2017-03-21T00:14:31.136373+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0', 'uuid': 'fa044b11-91ff-4320-a212-5f98f9f7e165', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2017-03-21T00:14:31.444896+00:00'}, 'href': '/files/ENCFF593RQD/@@download/ENCFF593RQD.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'pseudoreplicated IDR thresholded peaks', 'date_created': '2017-04-03T17:51:20.041223+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF593RQD', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/27b03780-0c87-4651-9966-a2e76573b465/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'archived', 'technical_replicates': ['1_1'], 'uuid': 'eb7722cb-5a41-43b7-b5d9-5661cc2126eb', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3PxV200369KFKj3PP5gy7JQ\", \"executable\": \"applet-F2pzbpj0369FZq3f5Q148vbk\", \"user\": \"user-estherc\"}, \"qc\": {\"N1\": 36721, \"Ns\": 36718}, \"dx-parentAnalysis\": \"analysis-F3PxV200369KFKj3PP5gy7JB\", \"dx-id\": \"file-F3Q43Z00yv3VyXVfP2YqyVFq\"}', 'output_category': 'annotation', 'title': 'ENCFF593RQD', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/4aca03d4-12a1-410a-a0e0-8fe8835cf14f/', 'derived_from': ['/files/ENCFF201MWD/']}, {'technical_replicates': ['2_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R2'], 'libraries': ['99423177-fbc8-4bf6-8241-f6b25855eb2b'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/', 'library': '/libraries/ENCLB762ZJL/', 'uuid': 'f5e07b4b-2a7e-4c95-b919-fa54792315a0', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'biological_replicate_number': 2, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB615WUN/', 'date_created': '2017-10-24T20:14:44.935001+00:00'}, 'read_length_units': 'nt', 'read_count': 51749759, '@id': '/files/ENCFF723NCL/', 'cloud_metadata': {'file_size': 3337831258, 'url': 'https://encode-public.s3.amazonaws.com/2017/10/26/771a0f39-fdf5-40b0-80be-2141b3456b8a/ENCFF723NCL.fastq.gz', 'md5sum_base64': '3nR2SKALdbqbzDstkqZCmw=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/10/26/771a0f39-fdf5-40b0-80be-2141b3456b8a/ENCFF723NCL.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R2.fastq.gz'], 'schema_version': '15', 'md5sum': 'de747648a00b75ba9bcc3b2d92a6429b', 'content_md5sum': '3eb4997fc118df543228f910b7dbeda0', 'href': '/files/ENCFF723NCL/@@download/ENCFF723NCL.fastq.gz', 'read_length': 101, 'title': 'ENCFF723NCL', 'library': '/libraries/ENCLB762ZJL/', 'superseded_by': [], 'file_size': 3337831258, 'uuid': '771a0f39-fdf5-40b0-80be-2141b3456b8a', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R2.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'TTAGGC', 'flowcell': 'AHJYFHBBXX', 'lane': '2'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF595HTZ/', '/files/ENCFF173EDE/', '/files/ENCFF172JDP/', '/files/ENCFF956RLS/', '/files/ENCFF467VIQ/', '/files/ENCFF453HUS/'], 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'paired_with': '/files/ENCFF031VYK/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-10-26T00:31:17.967950+00:00', 'biological_replicates': [2], 'accession': 'ENCFF723NCL', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '2', 'dbxrefs': ['SRA:SRR8660123'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['HJYFHBBXX:2:2:ATAGGC:', 'HJYFHBBXX:2:2:CTAGGC:', 'HJYFHBBXX:2:2:GTAGGC:', 'HJYFHBBXX:2:2:NTAGGC:', 'HJYFHBBXX:2:2:TAAGGC:', 'HJYFHBBXX:2:2:TCAGGC:', 'HJYFHBBXX:2:2:TGAGGC:', 'HJYFHBBXX:2:2:TNAGGC:', 'HJYFHBBXX:2:2:TTAAGC:', 'HJYFHBBXX:2:2:TTACGC:', 'HJYFHBBXX:2:2:TTAGAC:', 'HJYFHBBXX:2:2:TTAGCC:', 'HJYFHBBXX:2:2:TTAGGA:', 'HJYFHBBXX:2:2:TTAGGC:', 'HJYFHBBXX:2:2:TTAGGG:', 'HJYFHBBXX:2:2:TTAGGN:', 'HJYFHBBXX:2:2:TTAGGT:', 'HJYFHBBXX:2:2:TTAGNC:', 'HJYFHBBXX:2:2:TTAGTC:', 'HJYFHBBXX:2:2:TTANGC:', 'HJYFHBBXX:2:2:TTATGC:', 'HJYFHBBXX:2:2:TTCGGC:', 'HJYFHBBXX:2:2:TTGGGC:', 'HJYFHBBXX:2:2:TTNGGC:', 'HJYFHBBXX:2:2:TTTGGC:']}, {'technical_replicates': ['2_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R2'], 'libraries': ['99423177-fbc8-4bf6-8241-f6b25855eb2b'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/', 'library': '/libraries/ENCLB762ZJL/', 'uuid': 'f5e07b4b-2a7e-4c95-b919-fa54792315a0', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'biological_replicate_number': 2, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB615WUN/', 'date_created': '2017-10-24T20:14:44.935001+00:00'}, 'read_length_units': 'nt', 'read_count': 51749759, '@id': '/files/ENCFF031VYK/', 'cloud_metadata': {'file_size': 3018903011, 'url': 'https://encode-public.s3.amazonaws.com/2017/10/26/40974f26-fa0e-4d2b-870d-19381d64b943/ENCFF031VYK.fastq.gz', 'md5sum_base64': '7YXJHHjHENCFn2yFHCo9aA=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/10/26/40974f26-fa0e-4d2b-870d-19381d64b943/ENCFF031VYK.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R1.fastq.gz'], 'schema_version': '15', 'md5sum': 'ed85c91c78c710d0859f6c851c2a3d68', 'content_md5sum': '5c8f4c67329db19e1bdf598853567155', 'href': '/files/ENCFF031VYK/@@download/ENCFF031VYK.fastq.gz', 'read_length': 101, 'title': 'ENCFF031VYK', 'library': '/libraries/ENCLB762ZJL/', 'superseded_by': [], 'file_size': 3018903011, 'uuid': '40974f26-fa0e-4d2b-870d-19381d64b943', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R1.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'TTAGGC', 'flowcell': 'AHJYFHBBXX', 'lane': '2'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF736BPA/', '/files/ENCFF196VCR/', '/files/ENCFF154SYW/', '/files/ENCFF952FAG/', '/files/ENCFF557HNG/', '/files/ENCFF085HHB/'], 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'paired_with': '/files/ENCFF723NCL/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-10-26T00:28:53.772514+00:00', 'biological_replicates': [2], 'accession': 'ENCFF031VYK', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '1', 'dbxrefs': ['SRA:SRR8660123'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['HJYFHBBXX:2:1:ATAGGC:', 'HJYFHBBXX:2:1:CTAGGC:', 'HJYFHBBXX:2:1:GTAGGC:', 'HJYFHBBXX:2:1:NTAGGC:', 'HJYFHBBXX:2:1:TAAGGC:', 'HJYFHBBXX:2:1:TCAGGC:', 'HJYFHBBXX:2:1:TGAGGC:', 'HJYFHBBXX:2:1:TNAGGC:', 'HJYFHBBXX:2:1:TTAAGC:', 'HJYFHBBXX:2:1:TTACGC:', 'HJYFHBBXX:2:1:TTAGAC:', 'HJYFHBBXX:2:1:TTAGCC:', 'HJYFHBBXX:2:1:TTAGGA:', 'HJYFHBBXX:2:1:TTAGGC:', 'HJYFHBBXX:2:1:TTAGGG:', 'HJYFHBBXX:2:1:TTAGGN:', 'HJYFHBBXX:2:1:TTAGGT:', 'HJYFHBBXX:2:1:TTAGNC:', 'HJYFHBBXX:2:1:TTAGTC:', 'HJYFHBBXX:2:1:TTANGC:', 'HJYFHBBXX:2:1:TTATGC:', 'HJYFHBBXX:2:1:TTCGGC:', 'HJYFHBBXX:2:1:TTGGGC:', 'HJYFHBBXX:2:1:TTNGGC:', 'HJYFHBBXX:2:1:TTTGGC:']}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/raw_bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '84b740737ef96d35a74693a91bfa11a6', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80P6YQ0369664VY542Vqp2f\", \"executable\": \"applet-F7KPgZQ03692zxFQ2fX49F6Q\", \"user\": \"user-mkajco\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [90142, 0], \"read1\": [51749759, 0], \"singletons\": [359518, 0], \"read2\": [51749759, 0], \"properly_paired\": [99970857, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [100213924, 0], \"mate_mapped_different_chr\": [130362, 0], \"in_total\": [103499518, 0], \"mapped\": [100573442, 0]}}, \"dx-parentAnalysis\": \"analysis-F80KyP8036979gz7P3FV2Kkz\", \"dx-id\": \"file-F80VX800Fj4YK4Bk40PJJ0XG\"}', '@id': '/files/ENCFF504NVE/', 'cloud_metadata': {'file_size': 5904774103, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/86e58ecf-3fc0-45c3-96e4-b30c195db76a/ENCFF504NVE.bam', 'md5sum_base64': 'hLdAc375bTWnRpOpG/oRpg=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': '86e58ecf-3fc0-45c3-96e4-b30c195db76a', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/974e2166-18fa-4360-902d-58b05a4fa4ac/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-11-07T07:36:57.673154+00:00', 'biological_replicates': [2], 'accession': 'ENCFF504NVE', 's3_uri': 's3://encode-public/2017/11/07/86e58ecf-3fc0-45c3-96e4-b30c195db76a/ENCFF504NVE.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/f7c7c0d8-f36a-4120-8686-307365495d3e/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF504NVE/@@download/ENCFF504NVE.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'c7c0b170dfcc6dff0dc01d92dce324e1', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5904774103, 'title': 'ENCFF504NVE', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': 'e8542c35dc806823185369a1e8a8d710', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80KyPQ036979gz7P3FV2Kp4\", \"executable\": \"applet-F7KPgv0036980pKF8jBZy1PZ\", \"user\": \"user-mkajco\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1795315\", \"phantomPeakCoef\": \"1.097095\", \"estFragLen\": \"170\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.1969631869148\", \"relPhantomPeakCoef\": \"2.17445\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1875481\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [90142, 0], \"read1\": [51749759, 0], \"singletons\": [359518, 0], \"read2\": [51749759, 0], \"properly_paired\": [99970857, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [100213924, 0], \"mate_mapped_different_chr\": [130362, 0], \"in_total\": [103499518, 0], \"mapped\": [100573442, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"421896312\", \"read_pair_optical_duplicates\": \"6284927\", \"read_pairs_examined\": \"45638133\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.176706\", \"read_pair_duplicates\": \"8064541\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"5.538004\", \"PBC1\": \"0.821502\", \"NRF\": \"0.823309\", \"OneReadPair\": \"30797474\", \"DistinctReadPairs\": \"37489242\", \"TotalReadPairs\": \"45534857\", \"TwoReadPairs\": \"5561114\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [0, 0], \"read1\": [37573592, 0], \"singletons\": [0, 0], \"read2\": [37573592, 0], \"properly_paired\": [75147184, 0], \"paired_in_sequencing\": [75147184, 0], \"with_self_mate_mapped\": [75147184, 0], \"mate_mapped_different_chr\": [0, 0], \"in_total\": [75147184, 0], \"mapped\": [75147184, 0]}}, \"dx-parentAnalysis\": \"analysis-F80KyP8036979gz7P3FV2Kkz\", \"dx-id\": \"file-F80Zvk802VQ26Vky3zg295Jf\"}', '@id': '/files/ENCFF332OVN/', 'cloud_metadata': {'file_size': 4700745123, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/c3c56564-5abd-4abb-9fab-bd0a2eb0f21c/ENCFF332OVN.bam', 'md5sum_base64': '6FQsNdyAaCMYU2mh6KjXEA=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': 'c3c56564-5abd-4abb-9fab-bd0a2eb0f21c', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/f6d5bf84-a7bf-440d-8142-c0b74350170a/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-11-07T07:42:44.433335+00:00', 'biological_replicates': [2], 'accession': 'ENCFF332OVN', 's3_uri': 's3://encode-public/2017/11/07/c3c56564-5abd-4abb-9fab-bd0a2eb0f21c/ENCFF332OVN.bam', 'quality_metrics': ['/chipseq-filter-quality-metrics/671cbb6f-ad0d-4ff4-a94d-da9e8f993c9e/', '/samtools-flagstats-quality-metrics/63bfab3b-0570-451a-a0b8-eda2d5dfdf6e/', '/samtools-flagstats-quality-metrics/c607ac97-8d0b-4783-8912-c736691e112e/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF332OVN/@@download/ENCFF332OVN.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': '7e177fb913171039ac73a2c45823c474', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 4700745123, 'title': 'ENCFF332OVN', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/raw_bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '3eb9e8cb9a88d073a35cf20f06f00328', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80P6GQ036968By67QJFj879\", \"executable\": \"applet-F7KPgZQ03692zxFQ2fX49F6Q\", \"user\": \"user-mkajco\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [60308, 0], \"read1\": [51749759, 0], \"singletons\": [283366, 0], \"read2\": [51749759, 0], \"properly_paired\": [101272312, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [101523157, 0], \"mate_mapped_different_chr\": [97879, 0], \"in_total\": [103499518, 0], \"mapped\": [101806523, 0]}}, \"dx-parentAnalysis\": \"analysis-F80Kyj003696ZqpyP2qyP81z\", \"dx-id\": \"file-F80X0xQ01JZ4vVqb3zq2fKvK\"}', '@id': '/files/ENCFF876FOW/', 'cloud_metadata': {'file_size': 5914167199, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/a78ee47c-951b-42ee-bee3-1bbc19f40cff/ENCFF876FOW.bam', 'md5sum_base64': 'Prnoy5qI0HOjXPIPBvADKA=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': 'a78ee47c-951b-42ee-bee3-1bbc19f40cff', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/f0690507-36ba-4fa5-81b8-6aaed61d6eb4/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-11-07T08:05:39.165004+00:00', 'biological_replicates': [2], 'accession': 'ENCFF876FOW', 's3_uri': 's3://encode-public/2017/11/07/a78ee47c-951b-42ee-bee3-1bbc19f40cff/ENCFF876FOW.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/70a51172-3fb1-4251-8753-6e29db35a352/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF876FOW/@@download/ENCFF876FOW.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'bff277b4cf47b1aea84ab5766c0dafed', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5914167199, 'title': 'ENCFF876FOW', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '579f54b7c66d8af6b93f60e16f34b2fe', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80Kyj003696ZqpyP2qyP824\", \"executable\": \"applet-F7KPgv0036980pKF8jBZy1PZ\", \"user\": \"user-mkajco\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1844554\", \"phantomPeakCoef\": \"1.094677\", \"estFragLen\": \"170\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.201919048686144\", \"relPhantomPeakCoef\": \"2.315045\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1919989\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [60308, 0], \"read1\": [51749759, 0], \"singletons\": [283366, 0], \"read2\": [51749759, 0], \"properly_paired\": [101272312, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [101523157, 0], \"mate_mapped_different_chr\": [97879, 0], \"in_total\": [103499518, 0], \"mapped\": [101806523, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"422967143\", \"read_pair_optical_duplicates\": \"6272561\", \"read_pairs_examined\": \"45584414\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.176467\", \"read_pair_duplicates\": \"8044135\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"5.543256\", \"PBC1\": \"0.821703\", \"NRF\": \"0.823552\", \"OneReadPair\": \"30776768\", \"DistinctReadPairs\": \"37454873\", \"TotalReadPairs\": \"45479692\", \"TwoReadPairs\": \"5552110\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [2, 0], \"read1\": [37540279, 0], \"singletons\": [0, 0], \"read2\": [37540279, 0], \"properly_paired\": [75080558, 0], \"paired_in_sequencing\": [75080558, 0], \"with_self_mate_mapped\": [75080558, 0], \"mate_mapped_different_chr\": [2, 0], \"in_total\": [75080558, 0], \"mapped\": [75080558, 0]}}, \"dx-parentAnalysis\": \"analysis-F80Kyj003696ZqpyP2qyP81z\", \"dx-id\": \"file-F80bFQQ0Qbf03bzZ87JGYvPG\"}', '@id': '/files/ENCFF634VOX/', 'cloud_metadata': {'file_size': 4700789757, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/fe390a12-7c3f-4943-b837-9e6ba95cd527/ENCFF634VOX.bam', 'md5sum_base64': 'V59Ut8Ztiva5P2DhbzSy/g=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': 'fe390a12-7c3f-4943-b837-9e6ba95cd527', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/74aa85fb-27e6-475e-b58c-9895e3a90452/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-11-07T08:10:58.279948+00:00', 'biological_replicates': [2], 'accession': 'ENCFF634VOX', 's3_uri': 's3://encode-public/2017/11/07/fe390a12-7c3f-4943-b837-9e6ba95cd527/ENCFF634VOX.bam', 'quality_metrics': ['/chipseq-filter-quality-metrics/0b0b2aa4-4c4c-4b5c-bc46-c1eee0e61201/', '/samtools-flagstats-quality-metrics/f100be09-7597-4791-bd79-24c207a32e19/', '/samtools-flagstats-quality-metrics/ece7bfb9-a7b5-4287-b2c3-6a006b1543d2/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF634VOX/@@download/ENCFF634VOX.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'dfa94f34af8d8f2e4169c563950711e4', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 4700789757, 'title': 'ENCFF634VOX', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R1.regionPeak.gz', 'biological_replicates': [1], 'alternate_accessions': ['ENCFF309UAI'], 'flowcell_details': [], 'md5sum': 'ebc9d60199db8c009eb5b490bac21b09', 'fastq_signature': [], '@id': '/files/ENCFF269RYL/', 'cloud_metadata': {'file_size': 7251371, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/df0fd769-c921-4567-8d68-866f5c600652/ENCFF269RYL.bed.gz', 'md5sum_base64': '68nWAZnbjACetbSQusIbCQ=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/df0fd769-c921-4567-8d68-866f5c600652/ENCFF269RYL.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/7560b4b8-eb97-42e2-ad65-8994e668678d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:09.855438+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF269RYL', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF269RYL/@@download/ENCFF269RYL.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': 'df0fd769-c921-4567-8d68-866f5c600652', 'content_md5sum': '94c26eca2d90e55488f7c2ca4138ae36', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj036970ZKz5Q51x6Yf\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Q8Q80j5qZGxQQ2B6F9Z5Q\"}', 'file_size': 7251371, 'title': 'ENCFF269RYL', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF014SGL/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/PL.regionPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '1f7bc4ee6eff6069d77868393a896a02', 'fastq_signature': [], '@id': '/files/ENCFF413EHT/', 'cloud_metadata': {'file_size': 7325551, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/cba4d008-4552-4e05-a4f3-cef34b2b12b1/ENCFF413EHT.bed.gz', 'md5sum_base64': 'H3vE7m7/YGnXeGg5OolqAg=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/cba4d008-4552-4e05-a4f3-cef34b2b12b1/ENCFF413EHT.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/7560b4b8-eb97-42e2-ad65-8994e668678d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:21.622838+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF413EHT', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF413EHT/@@download/ENCFF413EHT.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'cba4d008-4552-4e05-a4f3-cef34b2b12b1', 'content_md5sum': '9814dfc96a89c5ed76dd2d60c59f4a35', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03696G7y25V929JV3\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Vqx80yxgGKXJB5QfK1P3X\"}', 'file_size': 7325551, 'title': 'ENCFF413EHT', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/', '/files/ENCFF014SGL/'], 'superseded_by': []}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/pool.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '1dc8492bc2730d7b1e3bcdefe9598e57', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF127GCV/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 2415172363, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/30e979c3-3b0f-47fb-b75a-3dd99a0bf8cb/ENCFF127GCV.bigWig', 'md5sum_base64': 'HchJK8JzDXseO83v6VmOVw=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF127GCV/@@download/ENCFF127GCV.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:06:36.916409+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF127GCV', 's3_uri': 's3://encode-public/2017/11/21/30e979c3-3b0f-47fb-b75a-3dd99a0bf8cb/ENCFF127GCV.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '30e979c3-3b0f-47fb-b75a-3dd99a0bf8cb', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFBQ036963q5z2B0Yp1b3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88Y9XQ03g36G7y25V92J4QZ\"}', 'file_size': 2415172363, 'title': 'ENCFF127GCV', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r2.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '5de4c2d208d3251354d0acf20d2357f8', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF887IHR/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1665047480, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/ea60aa14-3ac6-4bde-9123-dddde2bf5453/ENCFF887IHR.bigWig', 'md5sum_base64': 'XeTC0gjTJRNU0KzyDSNX+A=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF887IHR/@@download/ENCFF887IHR.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T16:59:36.856844+00:00', 'biological_replicates': [2], 'accession': 'ENCFF887IHR', 's3_uri': 's3://encode-public/2017/11/21/ea60aa14-3ac6-4bde-9123-dddde2bf5453/ENCFF887IHR.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'ea60aa14-3ac6-4bde-9123-dddde2bf5453', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803694GxQQ2B6F79P3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VBGj06zvPvjy4J5z2BB5V\"}', 'file_size': 1665047480, 'title': 'ENCFF887IHR', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.gz', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'df298d239e82b16b3f65eabaf9a3d95e', 'fastq_signature': [], '@id': '/files/ENCFF159RSM/', 'cloud_metadata': {'file_size': 7005883, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/4f4525aa-58a7-40b4-ba35-d3c2286905d6/ENCFF159RSM.bed.gz', 'md5sum_base64': '3ymNI56CsWs/Zeq6+aPZXg=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/4f4525aa-58a7-40b4-ba35-d3c2286905d6/ENCFF159RSM.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/e58c3f8a-46e6-45b5-ac74-1a517c4e5de3/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:07:21.620168+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF159RSM', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF159RSM/@@download/ENCFF159RSM.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': '4f4525aa-58a7-40b4-ba35-d3c2286905d6', 'content_md5sum': 'ee7182c380afddee01261a9cedd73c09', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q036945PXb2Bz971jb\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88QZB80XV7kX5f029x2fZGg\"}', 'file_size': 7005883, 'title': 'ENCFF159RSM', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF634VOX/'], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r2.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '63df25c8fce24248889b7e0f23f37ded', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF304OQI/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1689568852, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/b8fe284f-d7f3-4d0f-9dcc-17bcc1883d22/ENCFF304OQI.bigWig', 'md5sum_base64': 'Y98lyPziQkiIm34PI/N97Q=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF304OQI/@@download/ENCFF304OQI.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:02:53.187619+00:00', 'biological_replicates': [2], 'accession': 'ENCFF304OQI', 's3_uri': 's3://encode-public/2017/11/21/b8fe284f-d7f3-4d0f-9dcc-17bcc1883d22/ENCFF304OQI.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b8fe284f-d7f3-4d0f-9dcc-17bcc1883d22', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369B1Y728jz1X0Kx\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88QxkQ0q912jFY45V2XJYJX\"}', 'file_size': 1689568852, 'title': 'ENCFF304OQI', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r2.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '50fce2cb8f00c97e3f761a5143219223', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF075WGX/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1689246633, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/573ee561-d974-4c24-9bd2-8e86c6609cb4/ENCFF075WGX.bigWig', 'md5sum_base64': 'UPziy48AyX4/dhpRQyGSIw=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF075WGX/@@download/ENCFF075WGX.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:03:47.037981+00:00', 'biological_replicates': [2], 'accession': 'ENCFF075WGX', 's3_uri': 's3://encode-public/2017/11/21/573ee561-d974-4c24-9bd2-8e86c6609cb4/ENCFF075WGX.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '573ee561-d974-4c24-9bd2-8e86c6609cb4', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803694GxQQ2B6F79P3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VB0Q06zv1795j5QvggjVP\"}', 'file_size': 1689246633, 'title': 'ENCFF075WGX', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '2f6121f1ed8a7ad4d2b643597ec02f37', 'fastq_signature': [], '@id': '/files/ENCFF699WXF/', 'cloud_metadata': {'file_size': 722459, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/7fededf2-7502-45f3-a83d-d9118d01a1ac/ENCFF699WXF.bed.gz', 'md5sum_base64': 'L2Eh8e2KetTStkNZfsAvNw=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/7fededf2-7502-45f3-a83d-d9118d01a1ac/ENCFF699WXF.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/b6d04c42-8125-469a-9009-db723d318b37/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:56.150585+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF699WXF', 'quality_metrics': ['/idr-quality-metrics/4043bba6-2c64-4af9-b053-e05556b7a7f9/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF699WXF/@@download/ENCFF699WXF.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'optimal IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '7fededf2-7502-45f3-a83d-d9118d01a1ac', 'content_md5sum': 'b34a920f1b1a142f66da176262d9bfaf', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp980X2k3QP2688Jb8Zv3\"}', 'file_size': 722459, 'title': 'ENCFF699WXF', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF413EHT/', '/files/ENCFF001TDO/', '/files/ENCFF269RYL/', '/files/ENCFF947TJN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.bb', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '436d80e0c8e005b3347ccdb391d1bd1a', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF756PBC/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13409044, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/3ecf312f-37ab-4b73-b050-10039f5fa97c/ENCFF756PBC.bigBed', 'md5sum_base64': 'Q22A4MjgBbM0fM2zkdG9Gg=='}, 's3_uri': 's3://encode-public/2017/11/21/3ecf312f-37ab-4b73-b050-10039f5fa97c/ENCFF756PBC.bigBed', 'file_size': 13409044, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF756PBC/@@download/ENCFF756PBC.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:07:41.326075+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF756PBC', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': '3ecf312f-37ab-4b73-b050-10039f5fa97c', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q036945PXb2Bz971jb\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88QZB00XV7ZGxQQ2B6F9jgp\"}', 'output_category': 'annotation', 'title': 'ENCFF756PBC', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/55e555ad-73af-47b9-a82f-cb17fd71b1be/', 'derived_from': ['/files/ENCFF159RSM/']}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/pool.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '9947868b85751a1b81b382483574c06c', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF643ALW/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 2414374710, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/0bcfab0f-f19b-494b-a0ee-98d522f26319/ENCFF643ALW.bigWig', 'md5sum_base64': 'mUeGi4V1GhuBs4JINXTAbA=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF643ALW/@@download/ENCFF643ALW.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:05:27.591360+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF643ALW', 's3_uri': 's3://encode-public/2017/11/21/0bcfab0f-f19b-494b-a0ee-98d522f26319/ENCFF643ALW.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '0bcfab0f-f19b-494b-a0ee-98d522f26319', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369GQQ8z9240J76X\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88VVp802Y1vbPJ24xQjFY04\"}', 'file_size': 2414374710, 'title': 'ENCFF643ALW', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/PL.fixcoord.regionPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '0f05b53f05c5020b9c75c9f05a8aa8f7', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF753OYI/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13173926, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/8b083846-fa69-4098-bf7f-c8331aca5e42/ENCFF753OYI.bigBed', 'md5sum_base64': 'DwW1PwXFAgucdcnwWoqo9w=='}, 's3_uri': 's3://encode-public/2017/11/21/8b083846-fa69-4098-bf7f-c8331aca5e42/ENCFF753OYI.bigBed', 'file_size': 13173926, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF753OYI/@@download/ENCFF753OYI.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:06:39.198345+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF753OYI', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '8b083846-fa69-4098-bf7f-c8331aca5e42', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03696G7y25V929JV3\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Vqx00yxg3Yyzv5VXVg38Q\"}', 'output_category': 'annotation', 'title': 'ENCFF753OYI', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/6751ed59-7d57-4db2-9b60-2fbf8e9d47e2/', 'derived_from': ['/files/ENCFF413EHT/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '9607aaf7af5e582e2ddc37e01a449102', 'fastq_signature': [], '@id': '/files/ENCFF579SRN/', 'cloud_metadata': {'file_size': 551998, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/6155b816-9597-403d-b5a6-a2ac5dec221a/ENCFF579SRN.bed.gz', 'md5sum_base64': 'lgeq969eWC4t3DfgGkSRAg=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/6155b816-9597-403d-b5a6-a2ac5dec221a/ENCFF579SRN.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/b6d04c42-8125-469a-9009-db723d318b37/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:46.600169+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF579SRN', 'quality_metrics': ['/idr-quality-metrics/4043bba6-2c64-4af9-b053-e05556b7a7f9/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF579SRN/@@download/ENCFF579SRN.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'conservative IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '6155b816-9597-403d-b5a6-a2ac5dec221a', 'content_md5sum': '4048b4e0f305a4b313b79da5368667f6', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp980X2kGbPJ24xQjGV24\"}', 'file_size': 551998, 'title': 'ENCFF579SRN', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF413EHT/', '/files/ENCFF001TDO/', '/files/ENCFF269RYL/', '/files/ENCFF947TJN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'e8cbc0507646020010e84075ba737f0f', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF044QVM/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1333769, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/0a92e9ca-0160-4671-b00d-b264ca588dd3/ENCFF044QVM.bigBed', 'md5sum_base64': '6MvAUHZGAgAQ6EB1unN/Dw=='}, 's3_uri': 's3://encode-public/2017/11/21/0a92e9ca-0160-4671-b00d-b264ca588dd3/ENCFF044QVM.bigBed', 'file_size': 1333769, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF044QVM/@@download/ENCFF044QVM.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'optimal IDR thresholded peaks', 'date_created': '2017-11-21T17:07:00.591299+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF044QVM', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/12315382-15e5-411f-adbe-c022a8dfc50a/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '0a92e9ca-0160-4671-b00d-b264ca588dd3', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp900X2k76KQy4x97252F\"}', 'output_category': 'annotation', 'title': 'ENCFF044QVM', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/a120edc6-130f-4c22-9f25-0d16ef2ce643/', 'derived_from': ['/files/ENCFF699WXF/']}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r2.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '676887a53da5f5658ccf891ce3e3d16f', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF382WGQ/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1498850245, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/7978357e-e69d-4ed2-814d-b2636aad7ac2/ENCFF382WGQ.bigWig', 'md5sum_base64': 'Z2iHpT2l9WWMz4kc4+PRbw=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF382WGQ/@@download/ENCFF382WGQ.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T16:58:55.511167+00:00', 'biological_replicates': [2], 'accession': 'ENCFF382WGQ', 's3_uri': 's3://encode-public/2017/11/21/7978357e-e69d-4ed2-814d-b2636aad7ac2/ENCFF382WGQ.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '7978357e-e69d-4ed2-814d-b2636aad7ac2', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369B1Y728jz1X0Kx\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Qy580q91GKXJB5QfK17Yf\"}', 'file_size': 1498850245, 'title': 'ENCFF382WGQ', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/pool.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'e04a7288efe6d9576e528aa40f8c3e93', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF681MQQ/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1942386368, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/3328d0fe-88a6-4aba-8cd8-d8615286f412/ENCFF681MQQ.bigWig', 'md5sum_base64': '4EpyiO/m2VduUoqkD4w+kw=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF681MQQ/@@download/ENCFF681MQQ.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T17:00:59.177241+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF681MQQ', 's3_uri': 's3://encode-public/2017/11/21/3328d0fe-88a6-4aba-8cd8-d8615286f412/ENCFF681MQQ.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '3328d0fe-88a6-4aba-8cd8-d8615286f412', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369GQQ8z9240J76X\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88VXF802Y1vKV704y76G0gj\"}', 'file_size': 1942386368, 'title': 'ENCFF681MQQ', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/pool.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '4094c8067847429468c491b688fb7ca2', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF939ITT/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1944402799, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/b05e9401-a6c5-44e8-ab27-433b35943380/ENCFF939ITT.bigWig', 'md5sum_base64': 'QJTIBnhHQpRoxJG2iPt8og=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF939ITT/@@download/ENCFF939ITT.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T17:01:48.372219+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF939ITT', 's3_uri': 's3://encode-public/2017/11/21/b05e9401-a6c5-44e8-ab27-433b35943380/ENCFF939ITT.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b05e9401-a6c5-44e8-ab27-433b35943380', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFBQ036963q5z2B0Yp1b3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88YB4Q03g3GKXJB5QfK2548\"}', 'file_size': 1944402799, 'title': 'ENCFF939ITT', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.gz', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '9c7ba6a7b72a87f37d11a6f3e597d365', 'fastq_signature': [], '@id': '/files/ENCFF947TJN/', 'cloud_metadata': {'file_size': 7005891, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/d287107c-15ff-49d5-85d8-553ef23079d8/ENCFF947TJN.bed.gz', 'md5sum_base64': 'nHump7cqh/N9Eabz5ZfTZQ=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/d287107c-15ff-49d5-85d8-553ef23079d8/ENCFF947TJN.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/7560b4b8-eb97-42e2-ad65-8994e668678d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:15.775373+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF947TJN', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF947TJN/@@download/ENCFF947TJN.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': 'd287107c-15ff-49d5-85d8-553ef23079d8', 'content_md5sum': 'c0b8c1c6a939e7ad41befdf7e30d75aa', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03691795j5QvgZvgg\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88PG6806XG2jFY45V2XGx1x\"}', 'file_size': 7005891, 'title': 'ENCFF947TJN', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.bb', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '557f752a63ee193d9233972afd4c38d7', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF372YXK/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13402106, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/58bd8de8-752d-4a6b-bdb1-8e58cefb52fc/ENCFF372YXK.bigBed', 'md5sum_base64': 'VX91KmPuGT2SM5cq/Uw41w=='}, 's3_uri': 's3://encode-public/2017/11/21/58bd8de8-752d-4a6b-bdb1-8e58cefb52fc/ENCFF372YXK.bigBed', 'file_size': 13402106, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF372YXK/@@download/ENCFF372YXK.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:06:32.701602+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF372YXK', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': '58bd8de8-752d-4a6b-bdb1-8e58cefb52fc', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03691795j5QvgZvgg\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88PG5j06XG1795j5Qvgfqvv\"}', 'output_category': 'annotation', 'title': 'ENCFF372YXK', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/6751ed59-7d57-4db2-9b60-2fbf8e9d47e2/', 'derived_from': ['/files/ENCFF947TJN/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'a9ee9c5110e45c9c3d0517f4f896b84e', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF563JWG/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1076586, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/4eb3b64b-bd5b-4f75-84bd-9bd40c89905a/ENCFF563JWG.bigBed', 'md5sum_base64': 'qe6cURDkXJw9BRf0+Ja4Tg=='}, 's3_uri': 's3://encode-public/2017/11/21/4eb3b64b-bd5b-4f75-84bd-9bd40c89905a/ENCFF563JWG.bigBed', 'file_size': 1076586, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF563JWG/@@download/ENCFF563JWG.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'conservative IDR thresholded peaks', 'date_created': '2017-11-21T17:06:51.568790+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF563JWG', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/12315382-15e5-411f-adbe-c022a8dfc50a/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '4eb3b64b-bd5b-4f75-84bd-9bd40c89905a', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp900X2kK66ZB878ZG1yP\"}', 'output_category': 'annotation', 'title': 'ENCFF563JWG', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/a120edc6-130f-4c22-9f25-0d16ef2ce643/', 'derived_from': ['/files/ENCFF579SRN/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/PL.fixcoord.regionPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '049ff8462215837405e58ee458b00707', 'fastq_signature': [], '@id': '/files/ENCFF613WKG/', 'cloud_metadata': {'file_size': 7340148, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/fc67164e-7ae8-4779-af6d-e539cafd4ec8/ENCFF613WKG.bed.gz', 'md5sum_base64': 'BJ/4RiIVg3QF5Y7kWLAHBw=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/fc67164e-7ae8-4779-af6d-e539cafd4ec8/ENCFF613WKG.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/e58c3f8a-46e6-45b5-ac74-1a517c4e5de3/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:07:27.536132+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF613WKG', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF613WKG/@@download/ENCFF613WKG.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'fc67164e-7ae8-4779-af6d-e539cafd4ec8', 'content_md5sum': 'f37aec6c20adf630d986b864ee373a81', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03690VvxZJGGVzkqX\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88YYXQ0f66vQQ8z9240V3y3\"}', 'file_size': 7340148, 'title': 'ENCFF613WKG', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/', '/files/ENCFF634VOX/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '7b6508f7051a9f9bccce5ae306cdcdf5', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF440BSA/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1349888, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/2edfb654-c4ef-4d2d-aa52-e06d7cb0deaf/ENCFF440BSA.bigBed', 'md5sum_base64': 'e2UI9wUan5vMzlrjBs3N9Q=='}, 's3_uri': 's3://encode-public/2017/11/21/2edfb654-c4ef-4d2d-aa52-e06d7cb0deaf/ENCFF440BSA.bigBed', 'file_size': 1349888, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF440BSA/@@download/ENCFF440BSA.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'optimal IDR thresholded peaks', 'date_created': '2017-11-21T17:08:08.725157+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF440BSA', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/dbffd278-f5c4-4188-8df3-1258e74c7960/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '2edfb654-c4ef-4d2d-aa52-e06d7cb0deaf', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX3j0z5kGKV704y76KJ91\"}', 'output_category': 'annotation', 'title': 'ENCFF440BSA', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/fa6a52fa-ca11-4abe-94a5-a96c6b99358a/', 'derived_from': ['/files/ENCFF824YMX/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/PL.fixcoord.regionPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'edb4b2008ed1c2b7b74dcf517893af2c', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF164ASR/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13196767, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/ee4cf664-0fe4-4bbc-bb99-735377fc9623/ENCFF164ASR.bigBed', 'md5sum_base64': '7bSyAI7Rwre3Tc9ReJOvLA=='}, 's3_uri': 's3://encode-public/2017/11/21/ee4cf664-0fe4-4bbc-bb99-735377fc9623/ENCFF164ASR.bigBed', 'file_size': 13196767, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF164ASR/@@download/ENCFF164ASR.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:07:48.580205+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF164ASR', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'ee4cf664-0fe4-4bbc-bb99-735377fc9623', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03690VvxZJGGVzkqX\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88YYX80f66zb63QJ6VQjB6X\"}', 'output_category': 'annotation', 'title': 'ENCFF164ASR', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/55e555ad-73af-47b9-a82f-cb17fd71b1be/', 'derived_from': ['/files/ENCFF613WKG/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '3c3c804b17f026e58e7beb8e00602478', 'fastq_signature': [], '@id': '/files/ENCFF919CRG/', 'cloud_metadata': {'file_size': 559881, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/6dc1ad20-dc6f-42bf-99bf-84bcff0b1f87/ENCFF919CRG.bed.gz', 'md5sum_base64': 'PDyASxfwJuWOe+uOAGAkeA=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/6dc1ad20-dc6f-42bf-99bf-84bcff0b1f87/ENCFF919CRG.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/9a0ff3c5-f598-43c7-96bf-1ff59ef16e28/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:07:54.585967+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF919CRG', 'quality_metrics': ['/idr-quality-metrics/09ffd1cf-5a41-4578-9c38-7d5fa957504d/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF919CRG/@@download/ENCFF919CRG.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'conservative IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '6dc1ad20-dc6f-42bf-99bf-84bcff0b1f87', 'content_md5sum': 'ff2f224b511d503f3f3643c0bb3a8ae7', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX400z5kGbPJ24xQjJyBv\"}', 'file_size': 559881, 'title': 'ENCFF919CRG', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF613WKG/', '/files/ENCFF419RSJ/', '/files/ENCFF159RSM/', '/files/ENCFF646KAN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '03bfee7ee344daa8663e80822480b3ad', 'fastq_signature': [], '@id': '/files/ENCFF824YMX/', 'cloud_metadata': {'file_size': 734290, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/b6a6b3b8-60d8-4254-aea6-1896ed07e5d1/ENCFF824YMX.bed.gz', 'md5sum_base64': 'A7/ufuNE2qhmPoCCJICzrQ=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/b6a6b3b8-60d8-4254-aea6-1896ed07e5d1/ENCFF824YMX.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/9a0ff3c5-f598-43c7-96bf-1ff59ef16e28/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:08:03.802008+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF824YMX', 'quality_metrics': ['/idr-quality-metrics/09ffd1cf-5a41-4578-9c38-7d5fa957504d/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF824YMX/@@download/ENCFF824YMX.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'optimal IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'b6a6b3b8-60d8-4254-aea6-1896ed07e5d1', 'content_md5sum': '9658977633344f45cea253faa14cd970', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX480z5kBbXQj4x1fGZjf\"}', 'file_size': 734290, 'title': 'ENCFF824YMX', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF613WKG/', '/files/ENCFF419RSJ/', '/files/ENCFF159RSM/', '/files/ENCFF646KAN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '267587643028520e24d59d9fe5e54f0c', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF731GZP/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1087913, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/e255f220-43d6-4b0d-b7e3-5f2b48fb922b/ENCFF731GZP.bigBed', 'md5sum_base64': 'JnWHZDAoUg4k1Z2f5eVPDA=='}, 's3_uri': 's3://encode-public/2017/11/21/e255f220-43d6-4b0d-b7e3-5f2b48fb922b/ENCFF731GZP.bigBed', 'file_size': 1087913, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF731GZP/@@download/ENCFF731GZP.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'conservative IDR thresholded peaks', 'date_created': '2017-11-21T17:07:59.133456+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF731GZP', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/dbffd278-f5c4-4188-8df3-1258e74c7960/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'e255f220-43d6-4b0d-b7e3-5f2b48fb922b', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX3j0z5kBbXQj4x1fGZjQ\"}', 'output_category': 'annotation', 'title': 'ENCFF731GZP', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/fa6a52fa-ca11-4abe-94a5-a96c6b99358a/', 'derived_from': ['/files/ENCFF919CRG/']}], 'date_submitted': '2017-01-20', 'award': {'end_date': '2016-07-31', 'rfa': 'ENCODE3', '@type': ['Award', 'Item'], 'status': 'disabled', 'pi': {'uuid': '27e105ca-c741-4459-bf17-90e003508639', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Michael Snyder', '@type': ['User', 'Item'], '@id': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}}, 'start_date': '2012-09-21', '@id': '/awards/U54HG006996/', 'uuid': '4d462953-2da5-4fcf-a695-7206f2d5cf45', 'title': 'PRODUCTION CENTER FOR MAPPING REGULATORY REGIONS OF THE HUMAN GENOME', 'url': 'http://projectreporter.nih.gov/project_info_details.cfm?aid=8402423', 'name': 'U54HG006996', 'description': 'We propose to use our high-throughput production facility for genome-wide identification of the chromosomal locations of regulatory elements in human and mouse cells. Sequence-specific and general transcription factors will be mapped across the genome using chromatin immunoprecipitation followed by DNA sequencing (ChlP-Seq). Long noncoding RNAs (IncRNAs) will be mapped using ChlRP-seq, a method by which specific chromatin-bound RNAs and their associated regions are isolated using oligonucleotides followed by sequencing of the precipitated DNA. The results obtained from our genome wide mapping experiments will greatly advance the efforts of the ENCODE consortium to comprehensively identify all functional elements in the human genome. Our experimental data will be subjected to stringent quality metrics already established by the consortium and the results made publicly available in a timely manner. Targeted biological validation experiments will be performed, along with integrative analyses. All experiments and analyses will be performed by a highly experienced and well-integrated team of experts that has already established a successful working relationship. We expect our center to contribute greatly to the annotation of the human and mouse genomes.', 'schema_version': '7', 'project': 'ENCODE', 'viewing_group': 'ENCODE3'}, 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': {'uuid': '742bd56b-e511-411c-9410-4040c5776597', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Jessika Adrian', '@type': ['User', 'Item'], '@id': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'lab': '/labs/michael-snyder/'}, 'replicates': [{'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1'], 'libraries': [{'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634423'], 'documents': [], '@id': '/libraries/ENCLB195KAR/', 'crosslinking_method': 'formaldehyde', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_L1'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2016-12-14T21:08:40.861500+00:00', 'accession': 'ENCLB195KAR', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': ['/documents/90e0e15b-ca37-4653-8c14-9748db04ae3f/'], '@id': '/biosamples/ENCBS111GOI/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_B1'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'accession': 'ENCBS111GOI', '@type': ['Biosample', 'Item'], 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986910'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', 'parent_of': [], 'sex': 'female', 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': 'e5b2389a-d247-4320-8f55-a17dce5fe03a', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '742bd56b-e511-411c-9410-4040c5776597', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Jessika Adrian', '@type': ['User', 'Item'], '@id': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], 'date_created': '2016-12-14T21:04:33.959774+00:00', 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'description': 'HEK293 cell line stably expressing N-terminal tagged eGFP-ZNF189 under the control of a CMV promoter.'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '9c03b3bd-4cce-438e-ab53-02d96e191ac8', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'antibodies': ['/antibodies/ENCAB976SGG/'], 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/'], '@type': ['Library', 'Item']}], 'status': 'released', 'experiment': '/experiments/ENCSR163RYW/', 'technical_replicate_number': 1, '@id': '/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', 'library': {'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634423'], 'documents': [], '@id': '/libraries/ENCLB195KAR/', 'crosslinking_method': 'formaldehyde', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_L1'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2016-12-14T21:08:40.861500+00:00', 'accession': 'ENCLB195KAR', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': ['/documents/90e0e15b-ca37-4653-8c14-9748db04ae3f/'], '@id': '/biosamples/ENCBS111GOI/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_B1'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'accession': 'ENCBS111GOI', '@type': ['Biosample', 'Item'], 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986910'], 'biosample_ontology': {'classification': 'cell line', 'aliases': [], 'dbxrefs': ['Cellosaurus:CVCL_0045'], 'status': 'released', 'organ_slims': ['epithelium', 'kidney'], 'developmental_slims': ['mesoderm'], '@id': '/biosample-types/cell_line_EFO_0001182/', 'name': 'cell_line_EFO_0001182', 'synonyms': ['human embryonic kidney cell'], 'term_name': 'HEK293', 'references': [], 'system_slims': ['excretory system'], 'uuid': '014c9be0-20d4-4df5-b816-b0dca4cfee9d', 'cell_slims': ['epithelial cell'], '@type': ['BiosampleType', 'Item'], 'schema_version': '1', 'term_id': 'EFO:0001182'}, 'parent_of': [], 'sex': 'female', 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': 'e5b2389a-d247-4320-8f55-a17dce5fe03a', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '742bd56b-e511-411c-9410-4040c5776597', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Jessika Adrian', '@type': ['User', 'Item'], '@id': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], 'date_created': '2016-12-14T21:04:33.959774+00:00', 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'description': 'HEK293 cell line stably expressing N-terminal tagged eGFP-ZNF189 under the control of a CMV promoter.'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '9c03b3bd-4cce-438e-ab53-02d96e191ac8', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'antibodies': ['/antibodies/ENCAB976SGG/'], 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/'], '@type': ['Library', 'Item']}, 'uuid': 'f26d90d9-53e2-4dff-8b98-c11d3fdb2831', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'biological_replicate_number': 1, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': {'lot_reviews': [{'organisms': ['/organisms/human/'], 'targets': ['/targets/eGFP-avictoria/'], 'biosample_term_id': 'EFO:0001182', 'status': 'characterized to standards', 'biosample_term_name': 'HEK293', 'detail': 'Fully characterized.'}], 'antigen_description': '', 'alternate_accessions': [], 'host_organism': '/organisms/rabbit/', 'lot_id': 'GR158277-1', '@id': '/antibodies/ENCAB976SGG/', 'used_by_biosample_characterizations': [], 'aliases': [], 'award': '/awards/U54HG006996/', 'schema_version': '9', 'characterizations': ['/antibody-characterizations/85afbfd6-a683-4db8-b4c6-43231478c6c9/', '/antibody-characterizations/807d39c0-f743-4579-b637-e3a353638565/'], 'source': '/sources/abcam/', 'accession': 'ENCAB976SGG', '@type': ['AntibodyLot', 'Item'], 'purifications': [], 'lab': '/labs/michael-snyder/', 'status': 'released', 'clonality': 'polyclonal', 'lot_id_alias': [], 'dbxrefs': ['AR:AB_303395'], 'uuid': 'af972301-057b-4313-b6a2-39e69a0fed2c', 'submitted_by': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'product_id': 'ab290', 'title': 'ENCAB976SGG', 'url': 'http://www.abcam.com/gfp-antibody-chip-grade-ab290.html', 'date_created': '2016-03-29T16:57:02.862650+00:00', 'targets': ['/targets/eGFP-avictoria/', '/targets/YFP-avictoria/']}, 'date_created': '2016-12-14T21:13:52.766303+00:00'}, {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R2'], 'libraries': [{'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634422'], 'documents': [], '@id': '/libraries/ENCLB762ZJL/', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:L-HEK_eGFP-ZNF189_R2'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2017-10-24T20:05:53.293466+00:00', 'accession': 'ENCLB762ZJL', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': [], '@id': '/biosamples/ENCBS082THU/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:B-HEK_eGFP_ZNF189_R2'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'date_created': '2017-10-24T19:23:45.015546+00:00', 'accession': 'ENCBS082THU', 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986901'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'parent_of': [], 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': '96db7fd4-54b3-4bd0-ab6a-1bd972aeb43e', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], '@type': ['Biosample', 'Item'], 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'sex': 'female'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '99423177-fbc8-4bf6-8241-f6b25855eb2b', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'antibodies': ['/antibodies/ENCAB615WUN/'], 'replicates': ['/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], '@type': ['Library', 'Item']}], 'status': 'released', 'experiment': '/experiments/ENCSR163RYW/', 'technical_replicate_number': 1, '@id': '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/', 'library': {'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634422'], 'documents': [], '@id': '/libraries/ENCLB762ZJL/', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:L-HEK_eGFP-ZNF189_R2'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2017-10-24T20:05:53.293466+00:00', 'accession': 'ENCLB762ZJL', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': [], '@id': '/biosamples/ENCBS082THU/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:B-HEK_eGFP_ZNF189_R2'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'date_created': '2017-10-24T19:23:45.015546+00:00', 'accession': 'ENCBS082THU', 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986901'], 'biosample_ontology': {'classification': 'cell line', 'aliases': [], 'dbxrefs': ['Cellosaurus:CVCL_0045'], 'status': 'released', 'organ_slims': ['epithelium', 'kidney'], 'developmental_slims': ['mesoderm'], '@id': '/biosample-types/cell_line_EFO_0001182/', 'name': 'cell_line_EFO_0001182', 'synonyms': ['human embryonic kidney cell'], 'term_name': 'HEK293', 'references': [], 'system_slims': ['excretory system'], 'uuid': '014c9be0-20d4-4df5-b816-b0dca4cfee9d', 'cell_slims': ['epithelial cell'], '@type': ['BiosampleType', 'Item'], 'schema_version': '1', 'term_id': 'EFO:0001182'}, 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'parent_of': [], 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': '96db7fd4-54b3-4bd0-ab6a-1bd972aeb43e', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], '@type': ['Biosample', 'Item'], 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'sex': 'female'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '99423177-fbc8-4bf6-8241-f6b25855eb2b', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'antibodies': ['/antibodies/ENCAB615WUN/'], 'replicates': ['/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], '@type': ['Library', 'Item']}, 'uuid': 'f5e07b4b-2a7e-4c95-b919-fa54792315a0', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'biological_replicate_number': 2, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': {'status': 'released', 'lot_reviews': [{'organisms': ['/organisms/human/'], 'targets': ['/targets/eGFP-avictoria/'], 'biosample_term_id': 'EFO:0001182', 'status': 'characterized to standards', 'biosample_term_name': 'HEK293', 'detail': 'Fully characterized.'}], 'antigen_description': 'Recombinant full length protein corresponding to GFP', 'alternate_accessions': [], 'host_organism': '/organisms/rabbit/', 'lot_id': 'GR194813-1', '@id': '/antibodies/ENCAB615WUN/', 'used_by_biosample_characterizations': [], 'isotype': 'IgG', 'aliases': ['michael-snyder:AS-1667'], 'award': '/awards/U54HG006996/', 'schema_version': '9', 'characterizations': ['/antibody-characterizations/77d85630-ebee-4af7-a14e-015b9e177f3b/', '/antibody-characterizations/4594a491-584d-45ce-a1da-eb81a4d6c1e8/'], 'source': '/sources/abcam/', 'accession': 'ENCAB615WUN', '@type': ['AntibodyLot', 'Item'], 'purifications': ['antiserum'], 'lab': '/labs/michael-snyder/', 'clonality': 'polyclonal', 'lot_id_alias': [], 'dbxrefs': ['AR:AB_303395'], 'uuid': 'f55443ec-8dce-4373-98ef-28c3302b5d44', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'product_id': 'ab-290 (Lot GR194813-1)', 'title': 'ENCAB615WUN', 'url': 'http://www.abcam.com/gfp-antibody-chip-grade-ab290.html', 'date_created': '2017-02-02T00:33:30.696380+00:00', 'targets': ['/targets/eGFP-avictoria/']}, 'date_created': '2017-10-24T20:14:44.935001+00:00'}], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/asin:0471491039: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oclc:634515043: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ga4ghdos:dg.4503/01b048d0-e128-4cb0-94e9-b2d2cab7563d: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oid:2.16.840: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/flowrepository:FR-FCM-ZYGW: {'metadata': []}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Metadata for http://127.0.0.1:8082/hpscreg:BCRTi001-A: {'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'http://hpscreg.eu/', 'potentialAction': {'@type': 'SearchAction', 'target': 'http://hpscreg.eu/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/addgene:50943: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gdsc:1242: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/bacdive:131392: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ido:0000504: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/isni:000000012281955X: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/lei:HWUPKR0MPOU8FGXBT394: {'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'name': 'Global Legal Entity Identifier Foundation', 'alternateName': 'GLEIF', 'url': 'http://gleif.org', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.gleif.org/en/lei/search#query={search_term_string}', 'query-input': 'required name=search_term_string', 'description': 'Search LEI Records'}}, {'@context': 'http://schema.org', '@type': 'WebSite', 'name': 'Global Legal Entity Identifier Foundation', 'alternateName': 'GLEIF', 'url': 'http://gleif.org', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.gleif.org/en/lei/search#query={search_term_string}', 'query-input': 'required name=search_term_string', 'description': 'Search LEI Records'}}]}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/viaf:75121530: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/sabiork.compound:75: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/spdx:CC-BY-1.0: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/peptideatlas.dataset:PASS01237: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/did:sov:WRfXPg8dantKVubE3HX8pw: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/oryzabase.reference:42840: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/biostudies:S-EPMC6266652: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/seed.reaction:rxn00001: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/occ:br/1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cadsr:3771992: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/erm:ERM00000044: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cid:190000021540: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/classyfire:C0004828: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/gcst:GCST000035: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/mmp.fun:MMP3888430: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/ecyano.experiment:18: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dlxb:6VDC956: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/dlxc:M77F7JM: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/cameo:2019-08-03_00000089_1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/[a-zA-Z]+.+$:AB_262044: None\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/nemo:smp-m3w9hbe: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/insdc.gca:GCA_000155495.1: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/spp:10.1621/vwN2g2HaX3: {'metadata': []}\n",
      "\n",
      "Metadata for http://127.0.0.1:8082/worms:146421: None\n"
     ]
    }
   ],
   "source": [
    "import aiohttp\n",
    "import asyncio\n",
    "\n",
    "metadata_service_url = 'http://127.0.0.1:8082' # https://metadata.api.identifiers.org/\n",
    "\n",
    "\n",
    "# Async request\n",
    "async def fetch(url, session):\n",
    "    async with session.get(url) as response:\n",
    "        if response.status == 200:\n",
    "            return await response.json()\n",
    "        else:\n",
    "            return {}\n",
    "\n",
    "# Async semaphore fetch\n",
    "metadata_results = {}\n",
    "\n",
    "empty_metadata_resource_urls = []\n",
    "\n",
    "async def bound_fetch(sem, url, session, namespace):\n",
    "    async with sem:\n",
    "        metadata = await fetch(url, session)\n",
    "        # print(f\"got metadata for {url}\")\n",
    "        \n",
    "        payload = metadata.get(\"payload\")\n",
    "        print(f'\\nMetadata for {url}: {payload}')\n",
    "        \n",
    "        if payload is not None:\n",
    "            new_metadata_result = payload.get(\"metadata\")\n",
    "            namespace[\"metadata\"] = new_metadata_result\n",
    "            if len(payload[\"metadata\"]) == 0:\n",
    "                empty_metadata_resource_urls.append(url.split('/', 3)[3])\n",
    "        else:\n",
    "            namespace[\"metadata\"] = []\n",
    "\n",
    "\n",
    "async with aiohttp.ClientSession() as session:\n",
    "    metadata_requests = []\n",
    "    sem = asyncio.Semaphore(5)\n",
    "    \n",
    "    for namespace in dataset:\n",
    "        # print(f\"getting metadata for {namespace['prefix']}\")\n",
    "\n",
    "        if namespace['namespaceEmbeddedInLui']:\n",
    "            print(f\"* [{namespace['prefix']}] has a special LUI.\")\n",
    "            prefix = namespace['pattern'][1:].split(':')[0]\n",
    "        else:\n",
    "            prefix = namespace['prefix']\n",
    "\n",
    "        get_metadata_url = f\"{metadata_service_url}/{prefix}:{namespace['sampleId']}\"\n",
    "\n",
    "        task = asyncio.ensure_future(bound_fetch(sem, get_metadata_url, session, namespace))\n",
    "        metadata_requests.append(task)\n",
    "\n",
    "    await asyncio.gather(*metadata_requests)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'prefix': 'chebi', 'mirId': 'MIR:00000002', 'name': 'ChEBI', 'pattern': '^CHEBI:\\\\d+$', 'description': \"Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on 'small' chemical compounds.\", 'created': '2019-06-11T14:15:26.925+0000', 'modified': '2019-06-11T14:15:26.925+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '36927', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1/contactPerson'}}, 'metadata': []}, {'prefix': 'ensembl', 'mirId': 'MIR:00000003', 'name': 'Ensembl', 'pattern': '^((ENS[FPTG]\\\\\\\\d{11}(\\\\\\\\.\\\\\\\\d+)?)|(FB\\\\\\\\w{2}\\\\\\\\d{7})|(Y[A-Z]{2}\\\\\\\\d{3}[a-zA-Z](\\\\\\\\-[A-Z])?)|([A-Z_a-z0-9]+(\\\\\\\\.)?(t)?(\\\\\\\\d+)?([a-z])?))$', 'description': 'Ensembl is a joint project between EMBL - EBI and the Sanger Institute  to develop a software system which produces and maintains automatic annotation on selected eukaryotic genomes. This collections also references outgroup organisms.', 'created': '2019-06-11T14:15:27.869+0000', 'modified': '2019-06-11T14:15:27.869+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ENSG00000139618', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/7'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/7'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/7/contactPerson'}}, 'metadata': [{'identifier': 'ENSG00000139618', '@context': 'http://schema.org', '@type': 'DataRecord', 'mainEntity': {'description': 'BRCA2 DNA repair associated [Source:HGNC Symbol;Acc:HGNC:1101]', 'identifier': 'ENSG00000139618', '@type': 'Gene', 'isPartOfBioChemEntity': {'@type': 'BioChemEntity', 'name': 'Chromosome 13'}, 'isPartOf': {'name': 'Homo sapiens', 'codeCategory': {'url': 'http://purl.bioontology.org/ontology/NCBITAXON/9606', 'codeValue': '9606', 'sameAs': 'http://purl.uniprot.org/taxonomy/9606', '@type': 'CategoryCode', 'inCodeSet': {'@type': 'CategoryCodeSet', 'name': 'NCBI taxon'}}, '@type': 'BioChemEntity', 'alternateName': 'Human'}, 'name': 'BRCA2'}, 'isPartOf': 'Ensembl Human Gene Set'}]}, {'prefix': 'ec-code', 'mirId': 'MIR:00000004', 'name': 'Enzyme Nomenclature', 'pattern': '^\\\\d+\\\\.-\\\\.-\\\\.-|\\\\d+\\\\.\\\\d+\\\\.-\\\\.-|\\\\d+\\\\.\\\\d+\\\\.\\\\d+\\\\.-|\\\\d+\\\\.\\\\d+\\\\.\\\\d+\\\\.(n)?\\\\d+$', 'description': 'The Enzyme Classification contains the recommendations of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology on the nomenclature and classification of enzyme-catalysed reactions.', 'created': '2019-06-11T14:15:28.483+0000', 'modified': '2019-06-11T14:15:28.483+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1.1.1.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/13'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/13'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/13/contactPerson'}}, 'metadata': []}, {'prefix': 'uniprot', 'mirId': 'MIR:00000005', 'name': 'UniProt Knowledgebase', 'pattern': '^([A-N,R-Z][0-9]([A-Z][A-Z, 0-9][A-Z, 0-9][0-9]){1,2})|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])(\\\\.\\\\d+)?$', 'description': 'The UniProt Knowledgebase (UniProtKB) is a comprehensive resource for protein sequence and functional information with extensive cross-references to more than 120 external databases. Besides amino acid sequence and a description, it also provides taxonomic data and citation information.', 'created': '2019-06-11T14:15:29.457+0000', 'modified': '2019-06-11T14:15:29.457+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P0DP23', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/23'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/23'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/23/contactPerson'}}, 'metadata': []}, {'prefix': 'taxonomy', 'mirId': 'MIR:00000006', 'name': 'Taxonomy', 'pattern': '^\\\\d+$', 'description': 'The taxonomy contains the relationships between all living forms for which nucleic acid or protein sequence have been determined.', 'created': '2019-06-11T14:15:29.882+0000', 'modified': '2019-06-11T14:15:29.882+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '9606', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/28'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/28'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/28/contactPerson'}}, 'metadata': []}, {'prefix': 'biomodels.db', 'mirId': 'MIR:00000007', 'name': 'BioModels Database', 'pattern': '^((BIOMD|MODEL)\\\\d{10})|(BMID\\\\d{12})$', 'description': 'BioModels Database is a data resource that allows biologists to store, search and retrieve published mathematical models of biological interests.', 'created': '2019-06-11T14:15:30.877+0000', 'modified': '2019-06-11T14:15:30.877+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BIOMD0000000048', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/36'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/36'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/36/contactPerson'}}, 'metadata': []}, {'prefix': 'miriam.collection', 'mirId': 'MIR:00000008', 'name': 'MIRIAM Registry collection', 'pattern': '^MIR:000\\\\d{5}$', 'description': 'MIRIAM Registry is an online resource created to catalogue collections (Gene Ontology, Taxonomy or PubMed are some examples) and the corresponding resources (physical locations) providing access to those data collections. The Registry provides unique and perennial URIs for each entity of those data collections.', 'created': '2019-06-11T14:15:31.442+0000', 'modified': '2019-06-11T14:15:31.442+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MIR:00000008', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/41'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/41'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/41/contactPerson'}}, 'metadata': []}, {'prefix': 'icd', 'mirId': 'MIR:00000009', 'name': 'ICD', 'pattern': '^[A-Z]\\\\d+(\\\\.[-\\\\d+])?$', 'description': 'The International Classification of Diseases is the international standard diagnostic classification for all general epidemiological and many health management purposes.', 'created': '2019-06-11T14:15:31.658+0000', 'modified': '2019-06-11T14:15:31.658+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C34', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/43'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/43'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/43/contactPerson'}}, 'metadata': []}, {'prefix': 'intact', 'mirId': 'MIR:00000010', 'name': 'IntAct', 'pattern': '^EBI\\\\-[0-9]+$', 'description': 'IntAct provides a freely available, open source database system and analysis tools for protein interaction data.', 'created': '2019-06-11T14:15:31.920+0000', 'modified': '2019-06-11T14:15:31.920+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EBI-2307691', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/46'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/46'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/46/contactPerson'}}, 'metadata': []}, {'prefix': 'interpro', 'mirId': 'MIR:00000011', 'name': 'InterPro', 'pattern': '^IPR\\\\d{6}$', 'description': 'InterPro is a database of protein families, domains and functional sites in which identifiable features found in known proteins can be applied to unknown protein sequences.', 'created': '2019-06-11T14:15:32.153+0000', 'modified': '2019-06-11T14:15:32.153+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'IPR000100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/48'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/48'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/48/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.pathway', 'mirId': 'MIR:00000012', 'name': 'KEGG Pathway', 'pattern': '^\\\\w{2,4}\\\\d{5}$', 'description': 'KEGG PATHWAY is a collection of manually drawn pathway maps representing our knowledge on the molecular interaction and reaction networks.', 'created': '2019-06-11T14:15:32.480+0000', 'modified': '2019-06-11T14:15:32.480+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'hsa00620', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/51'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/51'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/51/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.compound', 'mirId': 'MIR:00000013', 'name': 'KEGG Compound', 'pattern': '^C\\\\d+$', 'description': 'KEGG compound contains our knowledge on the universe of chemical substances that are relevant to life.', 'created': '2019-06-11T14:15:32.730+0000', 'modified': '2019-06-11T14:15:32.730+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C12345', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/53'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/53'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/53/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.reaction', 'mirId': 'MIR:00000014', 'name': 'KEGG Reaction', 'pattern': '^R\\\\d+$', 'description': 'KEGG reaction contains our knowledge on the universe of reactions that are relevant to life.', 'created': '2019-06-11T14:15:32.930+0000', 'modified': '2019-06-11T14:15:32.930+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'R00100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/55'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/55'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/55/contactPerson'}}, 'metadata': []}, {'prefix': 'pubmed', 'mirId': 'MIR:00000015', 'name': 'PubMed', 'pattern': '^\\\\d+$', 'description': 'PubMed is a service of the U.S. National Library of Medicine that includes citations from MEDLINE and other life science journals for biomedical articles back to the 1950s.', 'created': '2019-06-11T14:15:33.142+0000', 'modified': '2019-06-11T14:15:33.142+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '16333295', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/57'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/57'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/57/contactPerson'}}, 'metadata': []}, {'prefix': 'mim', 'mirId': 'MIR:00000016', 'name': 'OMIM', 'pattern': '^[*#+%^]?\\\\d{6}$', 'description': 'Online Mendelian Inheritance in Man is a catalog of human genes and genetic disorders.', 'created': '2019-06-11T14:15:33.878+0000', 'modified': '2019-06-11T14:15:33.878+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '603903', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/65'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/65'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/65/contactPerson'}}, 'metadata': []}, {'prefix': 'pirsf', 'mirId': 'MIR:00000017', 'name': 'PIRSF', 'pattern': '^PIRSF\\\\d{6}$', 'description': 'The PIR SuperFamily concept is being used as a guiding principle to provide comprehensive and non-overlapping clustering of UniProtKB sequences into a hierarchical order to reflect their evolutionary relationships.', 'created': '2019-06-11T14:15:34.294+0000', 'modified': '2019-06-11T14:15:34.294+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PIRSF000100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/69'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/69'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/69/contactPerson'}}, 'metadata': []}, {'prefix': 'reactome', 'mirId': 'MIR:00000018', 'name': 'Reactome', 'pattern': '(^R-[A-Z]{3}-\\\\d+(-\\\\d+)?(\\\\.\\\\d+)?$)|(^REACT_\\\\d+(\\\\.\\\\d+)?$)', 'description': 'The Reactome project is a collaboration to develop a curated resource of core pathways and reactions in human biology.', 'created': '2019-06-11T14:15:34.555+0000', 'modified': '2019-06-11T14:15:34.555+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'R-HSA-201451', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/72'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/72'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/72/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'https://reactome.org/', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://reactome.org/content/query?q={term}', 'query-input': 'required name=term'}}, {'@context': 'http://schema.org', '@type': 'Organization', 'url': 'https://reactome.org', 'logo': 'https://reactome.org/templates/favourite/images/logo/logo.png', 'email': 'help@reactome.org'}, {'name': 'Signaling by BMP', 'description': 'Bone morphogenetic proteins (BMPs) have many biological activities in various tissues, including bone, cartilage, blood vessels, heart, kidney, neurons, liver and lung. They are members of the Transforming growth factor-Beta (TGFB) family. They bind to type II and type I serine-threonine kinase receptors, which transduce signals through SMAD and non-SMAD signalling pathways. BMP signalling is linked to a wide variety of clinical disorders, including vascular diseases, skeletal diseases and cancer. BMPs typically activate BMP type I receptors and signal via SMAD1, 5 and 8. They can be classified into several subgroups, including the BMP2/4 group, the BMP5-8 osteogenic protein-1 (OP1) group, the growth and differentiation factor (GDF) 5-7 group and the BMP9/10 group. Most of the proteins of the BMP2/4, OP1 and BMP9/10 groups induce formation of bone and cartilage tissues in vivo, while the GDF5-7 group induce cartilage and tendon-like, but not bone-like, tissues (Miyazono et al. 2010). Members of the TGFB family bind to two types of serine-threonine kinase receptors, type I and type II (Massagu√© 2012). BMPs can bind type I receptors in the absence of type II receptors, but both types are required for signal transduction. The presence of both types dramatically increases binding affinity (Rozenweig et al. 1995). The type II receptor kinase transphosphorylates the type I receptor, which transmits specific intracellular signals. Type I and type II receptors share similar structural properties, comprised of a relatively short extracellular domain, a single membrane-spanning domain and an intracellular domain containing a serine-threonine kinase domain. Seven receptors, collectively referred to as the Activin receptor-like kinases (ALK), have been identified as type I receptors for the TGFB family in mammals. ALKs are classified into three groups based on their structure and function, the BMPRI group (Bone morphogenetic protein receptor type-1A, ALK3, BMPR1A and Bone morphogenetic protein receptor type-1B, ALK6, BMPR1B), the ALK1 group (Serine/threonine-protein kinase receptor R3, ALK1, ACVRL1 and Activin receptor type-1, ALK2, ACVR1) and the TBetaR1 group (Activin receptor type-1B, ALK4, ACVR1B and  TGF-beta receptor type-1, ALK5, TGFBR1 and Activin receptor type-1C, ALK7, ACVR1C) (Kawabata et al. 1998). ALK1 group and BMPRI group activate SMAD1/5/8 and transduce similar intracellular signals. The TBetaR1 group activate SMAD2/3. BMPR1A and ACVR1 are widely expressed. BMPR1B shows a more restricted expression profile. ACVRL1 is limited to endothelial cells and a few other cell types. The binding specificities of BMPs to type I receptors is affected by the type II receptors that are present (Yu et al. 2005). Typically, BMP2 and BMP4 bind to BMPR1A and BMPR1B (ten Dijke et al. 1994). BMP6 and BMP7 bind strongly to ACVR1 and weakly to BMPR1B. Growth/differentiation factor 5 (BMP14, GDF5) preferentially binds to BMPR1B, but not to other type I receptors (Nishitoh et al. 1995). BMP9 and BMP10 bind to ACVRL1 and ACVRL (Scharpfenecker et al. 2007). BMP type I receptors are shared by other members of the TGFB family. Three receptors, Bone morphogenetic protein receptor type-2 (BMPR2), Activin receptor type-2A (ACVR2A) and Activin receptor type-2B (ACVR2B) are the type II receptors for mammalian BMPs. They are widely expressed in various tissues. BMPR2 is specific for BMPs, whereas ACVR2A and ACVR2B are shared with activins and myostatin. BMP binding and signalling can be affected by coreceptors. Glycosylphosphatidylinositol (GPI)-anchored proteins of the repulsive guidance molecule (RGM) family, including RGMA, RGMB (DRAGON) and Hemojuvelin (HFE2, RGMC) are coreceptors for BMP2 and BMP4, enhancing signaling (Samad et al. 2005, Babitt et al. 2005, 2006). They interact with BMP type I and/or type II receptors and bind BMP2 and BMP4, but not BMP7 or TGFB1. BMP2/4 signalling normally involves BMPR2, not ACVR2A or ACVR2B. Cells transfected with RGMA use both BMPR2 and ACVR2A for BMP-2/4 signalling, suggesting that RGMA facilitates the use of ACVR2A by BMP2/4 (Xia et al. 2007). Endoglin (ENG) is a transmembrane protein expressed in proliferating endothelial cells. It binds various ligands including TGFB1/3, Activin-A and BMP2/7 (Barbara et al. 1999). It inhibits TGFB-induced responses and enhances BMP7-induced responses (Scherner et al. 2007). Mutations in ENG result in hereditary haemorrhagic telangiectasia (HHT1), also known as OslerWeberRendu disease, while mutations in ACVRL1 lead to HHT2, suggesting that they act in a common signalling pathway (McAllister et al. 1994, Johnson et al. 1996). BMP2 is a dimeric protein, having two receptor-binding motifs. One is a high-affinity binding site for BMPR1A, the other is a low-affinity binding site for BMPR2 (Kirsch et al. 2000). In the absence of ligand stimulation, small fractions of type II and type I receptors are present as preexisting homodimers and heterodimers on the cell surface. Ligand-binding increases oligomerization. The intracellular domains of type I receptors have a characteristic GS domain (glycine and serine-rich domain) located N-terminal to the serine-threonine kinase domains. Type II receptor kinases are constitutively active in the absence of ligand. Upon ligand binding, the type II receptor kinase phosphorylates the GS domain of the type I receptor, a critical event in signal transduction by the serine/threonine kinase receptors (Miyazono et al. 2010). Activation of the TGFBR1 receptor has been studied in detail. The inactive conformation is maintained by interaction between the GS domain, the N-terminal lobe and the activation loop of the kinase (Huse et al. 1999). When the GS domain is phosphorylated by the type II receptor kinase, the TGFBR1 kinase is converted to an active conformation. Mutations of Thr-204 in TGFBR1 and the corresponding Gln in BMP type I receptors lead to their constitutive activation. The L45 loop, in the kinase domain of  type I receptors, specifically interacts with receptor-regulated Smads (R-Smads). Neurotrophic tyrosine kinase receptor type 3 (NT-3 growth factor receptor, TrkC, NTRK3) directly binds BMPR2, interfereing with its interaction with BMPR1A, which inhibits downstream signalling (Jin et al. 2007). Tyrosine-protein kinase transmembrane receptor ROR2 and BMPR1B form a heteromeric complex in a ligand independent fashion that modulatesGDF5-BMPR1B signalling by inhibition of Smad1/5 signalling (Sammar et al. 2004). Type I receptor kinases activated by the type II receptor kinases, phosphorylate R-Smads. R-Smads then form a complex with common-partner Smad (co-Smad) and translocate to the nucleus. The oligomeric Smad complexes regulate the transcription of target genes through interaction with various transcription factors and transcriptional coactivators or corepressors. Inhibitory Smads (I-Smads) negatively regulate the action of R-Smads and/or co-Smads. Eight different Smads have been identified in mammals. Smad1, Smad5 and Smad8 are R-Smads in BMP signalling pathways (BMP-specific R-Smads). Smad2 and Smad3 are R-Smads in TGFB/activin<br>signalling pathways. BMP receptors can phosphorylate Smad2 in certain types of cells (Murakami et al. 2009). Smad1, Smad5 and Smad8 are structurally highly similar to each other. The functional differences between them are largely unknown. Smad4 is the only co-Smad in mammals, shared by both BMP and TGFB/activin signalling pathways. Smad6 and Smad7 are I-Smads.', 'url': 'https://reactome.org/PathwayBrowser/#/R-HSA-201451', 'sameAs': None, 'version': '70', 'keywords': ['Pathway'], 'creator': [{'givenName': 'Lukasz', 'familyName': 'Huminiecki', 'affiliation': [{'name': 'Ludwig Institute for Cancer Research', '@type': 'Organization'}], 'url': 'http://europepmc.org/authors/null', '@type': 'Person'}, {'givenName': 'Aris', 'familyName': 'Moustakas', 'affiliation': [{'name': 'Ludwig Institute for Cancer Research', '@type': 'Organization'}], 'url': 'http://europepmc.org/authors/null', '@type': 'Person'}], 'includedInDataCatalog': {'url': 'https://reactome.org', 'name': 'Reactome', '@type': 'DataCatalog'}, 'distribution': [{'contentUrl': 'https://reactome.org/ContentService/exporter/sbml/201451.xml', 'fileFormat': 'SBML', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/ReactomeRESTfulAPI/RESTfulWS/sbgnExporter/201451', 'fileFormat': 'SBGN', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/ReactomeRESTfulAPI/RESTfulWS/biopaxExporter/Level2/201451', 'fileFormat': 'BIOPAX2', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/ReactomeRESTfulAPI/RESTfulWS/biopaxExporter/Level3/201451', 'fileFormat': 'BIOPAX3', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/cgi-bin/pdfexporter?DB=gk_current&ID=201451', 'fileFormat': 'PDF', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/cgi-bin/rtfexporter?DB=gk_current&ID=201451', 'fileFormat': 'DOCX', '@type': 'DataDownload'}, {'contentUrl': 'https://reactome.org/cgi-bin/protegeexporter?DB=gk_current&ID=201451', 'fileFormat': 'OWL', '@type': 'DataDownload'}], 'citation': ['http://www.ncbi.nlm.nih.gov/pubmed/19762341', 'http://www.ncbi.nlm.nih.gov/pubmed/28348038'], 'license': 'https://creativecommons.org/licenses/by/4.0/', '@context': 'http://schema.org/', '@type': 'DataSet'}]}, {'prefix': 'doi', 'mirId': 'MIR:00000019', 'name': 'DOI', 'pattern': '^(doi\\\\:)?\\\\d{2}\\\\.\\\\d{4}.*$', 'description': 'The Digital Object Identifier System is for identifying content objects in the digital environment.', 'created': '2019-06-11T14:15:34.841+0000', 'modified': '2019-06-11T14:15:34.841+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '10.1038/nbt1156', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/75'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/75'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/75/contactPerson'}}, 'metadata': []}, {'prefix': 'pdb', 'mirId': 'MIR:00000020', 'name': 'Protein Data Bank', 'pattern': '^[0-9][A-Za-z0-9]{3}$', 'description': 'The Protein Data Bank is the single worldwide archive of structural data of biological macromolecules.', 'created': '2019-06-11T14:15:35.100+0000', 'modified': '2019-06-11T14:15:35.100+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2gc4', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/78'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/78'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/78/contactPerson'}}, 'metadata': []}, {'prefix': 'go', 'mirId': 'MIR:00000022', 'name': 'Gene Ontology', 'pattern': '^GO:\\\\d{7}$', 'description': 'The Gene Ontology project provides a controlled vocabulary to describe gene and gene product attributes in any organism.', 'created': '2019-06-11T14:15:35.970+0000', 'modified': '2019-06-11T14:15:35.970+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0006915', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/87'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/87'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/87/contactPerson'}}, 'metadata': []}, {'prefix': 'sgd', 'mirId': 'MIR:00000023', 'name': 'SGD', 'pattern': '^((S\\\\d+$)|(Y[A-Z]{2}\\\\d{3}[a-zA-Z](\\\\-[A-Z])?))$', 'description': 'The Saccharomyces Genome Database (SGD) project collects information and maintains a database of the molecular biology of the yeast Saccharomyces cerevisiae.', 'created': '2019-06-11T14:15:37.056+0000', 'modified': '2019-06-11T14:15:37.056+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'S000003909', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/97'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/97'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/97/contactPerson'}}, 'metadata': []}, {'prefix': 'sbo', 'mirId': 'MIR:00000024', 'name': 'Systems Biology Ontology', 'pattern': '^SBO:\\\\d{7}$', 'description': 'The goal of the Systems Biology Ontology is to develop controlled vocabularies and ontologies tailored specifically for the kinds of problems being faced in Systems Biology, especially in the context of computational modeling. SBO is a project of the BioModels.net effort.', 'created': '2019-06-11T14:15:37.756+0000', 'modified': '2019-06-11T14:15:37.756+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000262', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/105'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/105'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/105/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.drug', 'mirId': 'MIR:00000025', 'name': 'KEGG Drug', 'pattern': '^D\\\\d+$', 'description': 'KEGG DRUG contains chemical structures of drugs and additional information such as therapeutic categories and target molecules.', 'created': '2019-06-11T14:15:38.268+0000', 'modified': '2019-06-11T14:15:38.268+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'D00123', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/109'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/109'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/109/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.glycan', 'mirId': 'MIR:00000026', 'name': 'KEGG Glycan', 'pattern': '^G\\\\d+$', 'description': 'KEGG GLYCAN, a part of the KEGG LIGAND database, is a collection of experimentally determined glycan structures. It contains all unique structures taken from CarbBank, structures entered from recent publications, and structures present in KEGG pathways.', 'created': '2019-06-11T14:15:38.579+0000', 'modified': '2019-06-11T14:15:38.579+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'G00123', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/111'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/111'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/111/contactPerson'}}, 'metadata': []}, {'prefix': 'wb', 'mirId': 'MIR:00000027', 'name': 'WormBase', 'pattern': '^WB[A-Z][a-z]+\\\\d+$', 'description': 'WormBase is an online bioinformatics database of the biology and genome of the model organism Caenorhabditis elegans and other nematodes. It is used by the C. elegans research community both as an information resource and as a mode to publish and distribute their results. This collection references WormBase-accessioned entities.', 'created': '2019-06-11T14:15:38.790+0000', 'modified': '2019-06-11T14:15:38.790+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'WBGene00000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/113'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/113'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/113/contactPerson'}}, 'metadata': []}, {'prefix': 'pfam', 'mirId': 'MIR:00000028', 'name': 'Pfam', 'pattern': '^PF\\\\d{5}$', 'description': 'The Pfam database contains information about protein domains and families. For each entry a protein sequence alignment and a Hidden Markov Model is stored.', 'created': '2019-06-11T14:15:39.270+0000', 'modified': '2019-06-11T14:15:39.270+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PF01234', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/117'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/117'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/117/contactPerson'}}, 'metadata': []}, {'prefix': 'insdc', 'mirId': 'MIR:00000029', 'name': 'Nucleotide Sequence Database', 'pattern': '^([A-Z]\\\\d{5}|[A-Z]{2}\\\\d{6}|[A-Z]{4}\\\\d{8}|[A-J][A-Z]{2}\\\\d{5})(\\\\.\\\\d+)?$', 'description': 'The International Nucleotide Sequence Database Collaboration (INSDC) consists of a joint effort to collect and disseminate databases containing DNA and RNA sequences.', 'created': '2019-06-11T14:15:39.496+0000', 'modified': '2019-06-11T14:15:39.496+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'X58356', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/119'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/119'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/119/contactPerson'}}, 'metadata': []}, {'prefix': 'fb', 'mirId': 'MIR:00000030', 'name': 'FlyBase', 'pattern': '^FB\\\\w{2}\\\\d{7}$', 'description': 'FlyBase is the database of the Drosophila Genome Projects and of associated literature.', 'created': '2019-06-11T14:15:40.188+0000', 'modified': '2019-06-11T14:15:40.188+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'FBgn0011293', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/125'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/125'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/125/contactPerson'}}, 'metadata': []}, {'prefix': 'wormpep', 'mirId': 'MIR:00000031', 'name': 'Wormpep', 'pattern': '^CE\\\\d{5}$', 'description': 'Wormpep contains the predicted proteins from the Caenorhabditis elegans genome sequencing project.', 'created': '2019-06-11T14:15:40.709+0000', 'modified': '2019-06-11T14:15:40.709+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CE28239', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/130'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/130'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/130/contactPerson'}}, 'metadata': []}, {'prefix': 'prosite', 'mirId': 'MIR:00000032', 'name': 'PROSITE', 'pattern': '^PS\\\\d{5}$', 'description': 'PROSITE consists of documentation entries describing protein domains, families and functional sites as well as associated patterns and profiles to identify them.', 'created': '2019-06-11T14:15:40.983+0000', 'modified': '2019-06-11T14:15:40.983+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PS00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/133'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/133'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/133/contactPerson'}}, 'metadata': [{'@context': ['https://schema.org/'], '@type': 'Organization', 'url': 'https://prosite.expasy.org', 'name': 'PROSITE', 'logo': 'https://prosite.expasy.org//images/prosite/prosite.gif', 'memberof': {'@type': 'NGO', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://sib.swiss/'}, 'parentOrganization': {'@type': 'Organization', 'name': 'ExPASy', 'url': 'https://www.expasy.org/'}}]}, {'prefix': 'pubchem.substance', 'mirId': 'MIR:00000033', 'name': 'PubChem-substance', 'pattern': '^\\\\d+$', 'description': \"PubChem provides information on the biological activities of small molecules. It is a component of NIH's Molecular Libraries Roadmap Initiative. PubChem Substance archives chemical substance records.\", 'created': '2019-06-11T14:15:41.269+0000', 'modified': '2019-06-11T14:15:41.269+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '100101', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/136'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/136'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/136/contactPerson'}}, 'metadata': []}, {'prefix': 'pubchem.compound', 'mirId': 'MIR:00000034', 'name': 'PubChem-compound', 'pattern': '^\\\\d+$', 'description': \"PubChem provides information on the biological activities of small molecules. It is a component of NIH's Molecular Libraries Roadmap Initiative. PubChem Compound archives chemical structures and records.\", 'created': '2019-06-11T14:15:41.493+0000', 'modified': '2019-06-11T14:15:41.493+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '100101', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/138'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/138'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/138/contactPerson'}}, 'metadata': []}, {'prefix': 'arxiv', 'mirId': 'MIR:00000035', 'name': 'arXiv', 'pattern': '^(\\\\w+(\\\\-\\\\w+)?(\\\\.\\\\w+)?)?\\\\d{4,7}(\\\\.\\\\d+(v\\\\d+)?)?$', 'description': 'arXiv is an e-print service in the fields of physics, mathematics, non-linear science, computer science, and quantitative biology.', 'created': '2019-06-11T14:15:41.707+0000', 'modified': '2019-06-11T14:15:41.707+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0807.4956v1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/140'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/140'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/140/contactPerson'}}, 'metadata': []}, {'prefix': 'arrayexpress', 'mirId': 'MIR:00000036', 'name': 'ArrayExpress', 'pattern': '^[AEP]-\\\\w{4}-\\\\d+$', 'description': 'ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with Microarray Gene Expression Data (MGED) recommendations.', 'created': '2019-06-11T14:15:41.970+0000', 'modified': '2019-06-11T14:15:41.970+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'E-MEXP-1712', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/143'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/143'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/143/contactPerson'}}, 'metadata': []}, {'prefix': 'mgi', 'mirId': 'MIR:00000037', 'name': 'Mouse Genome Database', 'pattern': '^MGI:\\\\d+$', 'description': 'The Mouse Genome Database (MGD) project includes data on gene characterization, nomenclature, mapping, gene homologies among mammals, sequence links, phenotypes, allelic variants and mutants, and strain data.', 'created': '2019-06-11T14:15:42.331+0000', 'modified': '2019-06-11T14:15:42.331+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2442292', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/146'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/146'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/146/contactPerson'}}, 'metadata': []}, {'prefix': 'sabiork.reaction', 'mirId': 'MIR:00000038', 'name': 'SABIO-RK Reaction', 'pattern': '^\\\\d+$', 'description': 'SABIO-RK is a relational database system that contains information about biochemical reactions, their kinetic equations with their parameters, and the experimental conditions under which these parameters were measured. The reaction data set provides information regarding the organism in which a reaction is observed, pathways in which it participates, and links to further information.', 'created': '2019-06-11T14:15:42.838+0000', 'modified': '2019-06-11T14:15:42.838+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '75', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/150'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/150'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/150/contactPerson'}}, 'metadata': []}, {'prefix': 'refseq', 'mirId': 'MIR:00000039', 'name': 'RefSeq', 'pattern': '^((AC|AP|NC|NG|NM|NP|NR|NT|NW|XM|XP|XR|YP|ZP)_\\\\d+|(NZ\\\\_[A-Z]{4}\\\\d+))(\\\\.\\\\d+)?$', 'description': 'The Reference Sequence (RefSeq) collection aims to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, transcript (RNA), and protein products.', 'created': '2019-06-11T14:15:43.085+0000', 'modified': '2019-06-11T14:15:43.085+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'NP_012345', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/153'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/153'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/153/contactPerson'}}, 'metadata': []}, {'prefix': 'tcdb', 'mirId': 'MIR:00000040', 'name': 'Transport Classification Database', 'pattern': '^\\\\d+(\\\\.[A-Z])?(\\\\.\\\\d+)?(\\\\.\\\\d+)?(\\\\.\\\\d+)?$', 'description': 'The database details a comprehensive  IUBMB approved classification system for membrane transport proteins known as the Transporter Classification (TC) system. The TC system is analogous to the Enzyme Commission (EC) system for classification of enzymes, but incorporates phylogenetic information additionally.', 'created': '2019-06-11T14:15:43.342+0000', 'modified': '2019-06-11T14:15:43.342+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '5.A.1.1.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/156'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/156'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/156/contactPerson'}}, 'metadata': []}, {'prefix': 'uniparc', 'mirId': 'MIR:00000041', 'name': 'UniParc', 'pattern': '^UPI[A-F0-9]{10}$', 'description': 'The UniProt Archive (UniParc) is a  database containing non-redundant protein sequence information from many sources. Each unique sequence is given a stable and unique identifier (UPI) making it possible to identify the same protein from different source databases.', 'created': '2019-06-11T14:15:43.590+0000', 'modified': '2019-06-11T14:15:43.590+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'UPI000000000A', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/159'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/159'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/159/contactPerson'}}, 'metadata': []}, {'prefix': 'mint', 'mirId': 'MIR:00000042', 'name': 'MINT', 'pattern': '^MINT\\\\-\\\\d{1,7}$', 'description': 'The Molecular INTeraction database (MINT) stores, in a structured format, information about molecular interactions by extracting experimental details from work published in peer-reviewed journals.', 'created': '2019-06-11T14:15:43.925+0000', 'modified': '2019-06-11T14:15:43.925+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MINT-10000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/162'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/162'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/162/contactPerson'}}, 'metadata': []}, {'prefix': 'dip', 'mirId': 'MIR:00000044', 'name': 'Database of Interacting Proteins', 'pattern': '^DIP(\\\\:)?\\\\-\\\\d{1,}[ENXS]$', 'description': 'The database of interacting protein (DIP) database stores experimentally determined interactions between proteins. It combines information from a variety of sources to create a single, consistent set of protein-protein interactions', 'created': '2019-06-11T14:15:44.329+0000', 'modified': '2019-06-11T14:15:44.329+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DIP-743N', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/166'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/166'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/166/contactPerson'}}, 'metadata': []}, {'prefix': 'signaling-gateway', 'mirId': 'MIR:00000045', 'name': 'Signaling Gateway', 'pattern': 'A\\\\d{6}$', 'description': 'The Signaling Gateway provides  information on  mammalian proteins involved in cellular signaling.', 'created': '2019-06-11T14:15:44.579+0000', 'modified': '2019-06-11T14:15:44.579+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'A001094', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/169'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/169'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/169/contactPerson'}}, 'metadata': []}, {'prefix': 'resid', 'mirId': 'MIR:00000046', 'name': 'RESID', 'pattern': '^AA\\\\d{4}$', 'description': 'The RESID Database of Protein Modifications is a comprehensive collection of annotations and structures for protein modifications including amino-terminal, carboxyl-terminal and peptide chain cross-link post-translational modifications.', 'created': '2019-06-11T14:15:44.821+0000', 'modified': '2019-06-11T14:15:44.821+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AA0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/172'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/172'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/172/contactPerson'}}, 'metadata': []}, {'prefix': 'rgd', 'mirId': 'MIR:00000047', 'name': 'Rat Genome Database', 'pattern': '^\\\\d{4,}$', 'description': 'Rat Genome Database seeks to collect, consolidate, and integrate rat genomic and genetic data with curated functional and physiological data and make these data widely available to the scientific community. This collection references genes.', 'created': '2019-06-11T14:15:45.062+0000', 'modified': '2019-06-11T14:15:45.062+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2018', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/175'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/175'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/175/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'Acly', 'description': \"ENCODES a protein that exhibits ATP citrate synthase activity; INVOLVED IN acetyl-CoA metabolic process; citrate metabolic process; fatty acid biosynthetic process; PARTICIPATES IN fatty acid biosynthetic pathway; citric acid cycle pathway; ASSOCIATED WITH congestive heart failure (ortholog); hepatocellular carcinoma (ortholog); obesity (ortholog); FOUND IN cytosol (ortholog); nucleoplasm (ortholog); plasma membrane (ortholog); INTERACTS WITH 1-naphthyl isothiocyanate; 17alpha-ethynylestradiol; 2,2',4,4'-Tetrabromodiphenyl ether\", 'url': 'https://rgd.mcw.edu/rgdweb/report/gene/main.html?id=2018', 'keywords': 'Rat Gene RGD Genome', 'includedInDataCatalog': 'https://rgd.mcw.edu', 'creator': {'@type': 'Organization', 'name': 'Rat Genome Database'}, 'version': '1', 'license': 'Creative Commons CC BY 4.0'}]}, {'prefix': 'tair.protein', 'mirId': 'MIR:00000048', 'name': 'TAIR Protein', 'pattern': '^AASequence:\\\\d{10}$', 'description': 'The Arabidopsis Information Resource (TAIR) maintains a database of genetic and molecular biology data for the model higher plant Arabidopsis thaliana. This provides protein information for a given gene model and provides links to other sources such as UniProtKB and GenPept', 'created': '2019-06-11T14:15:45.813+0000', 'modified': '2019-06-11T14:15:45.813+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AASequence:1009107926', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/179'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/179'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/179/contactPerson'}}, 'metadata': []}, {'prefix': 'tair.gene', 'mirId': 'MIR:00000049', 'name': 'TAIR Gene', 'pattern': '^Gene:\\\\d{7}$', 'description': 'The Arabidopsis Information Resource (TAIR) maintains a database of genetic and molecular biology data for the model higher plant Arabidopsis thaliana. This is the reference gene model for a given locus.', 'created': '2019-06-11T14:15:46.061+0000', 'modified': '2019-06-11T14:15:46.061+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Gene:2200934', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/182'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/182'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/182/contactPerson'}}, 'metadata': []}, {'prefix': 'tair.locus', 'mirId': 'MIR:00000050', 'name': 'TAIR Locus', 'pattern': '^\\\\d{7}$', 'description': 'The Arabidopsis Information Resource (TAIR) maintains a database of genetic and molecular biology data for the model higher plant Arabidopsis thaliana. The name of a Locus is unique and used by TAIR, TIGR, and MIPS.', 'created': '2019-06-11T14:15:46.274+0000', 'modified': '2019-06-11T14:15:46.274+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2200950', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/184'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/184'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/184/contactPerson'}}, 'metadata': []}, {'prefix': 'hmdb', 'mirId': 'MIR:00000051', 'name': 'HMDB', 'pattern': '^HMDB\\\\d+$', 'description': 'The Human Metabolome Database (HMDB) is a database containing detailed information about small molecule metabolites found in the human body.It contains or links 1) chemical 2) clinical and 3) molecular biology/biochemistry data.', 'created': '2019-06-11T14:15:46.469+0000', 'modified': '2019-06-11T14:15:46.469+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HMDB00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/186'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/186'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/186/contactPerson'}}, 'metadata': []}, {'prefix': 'lipidmaps', 'mirId': 'MIR:00000052', 'name': 'LIPID MAPS', 'pattern': '^LM(FA|GL|GP|SP|ST|PR|SL|PK)[0-9]{4}([0-9a-zA-Z]{4,6})?$', 'description': \"The LIPID MAPS Lipid Classification System is comprised of eight lipid categories, each with its own subclassification hierarchy. All lipids in the LIPID MAPS Structure Database (LMSD) have been classified using this system and have been assigned LIPID MAPS ID's which reflects their position in the classification hierarchy.\", 'created': '2019-06-11T14:15:46.708+0000', 'modified': '2019-06-11T14:15:46.708+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LMPR0102010012', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/189'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/189'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/189/contactPerson'}}, 'metadata': []}, {'prefix': 'peptideatlas', 'mirId': 'MIR:00000053', 'name': 'PeptideAtlas', 'pattern': '^PAp[0-9]{8}$', 'description': 'The PeptideAtlas Project provides a publicly accessible database of peptides identified in tandem mass spectrometry proteomics studies and software tools.', 'created': '2019-06-11T14:15:46.969+0000', 'modified': '2019-06-11T14:15:46.969+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PAp00000009', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/192'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/192'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/192/contactPerson'}}, 'metadata': []}, {'prefix': 'geo', 'mirId': 'MIR:00000054', 'name': 'GEO', 'pattern': '^G(PL|SM|SE|DS)\\\\d+$', 'description': 'The Gene Expression Omnibus (GEO) is a gene expression repository providing a curated, online resource for gene expression data browsing, query and retrieval.', 'created': '2019-06-11T14:15:47.245+0000', 'modified': '2019-06-11T14:15:47.245+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GDS1234', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/195'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/195'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/195/contactPerson'}}, 'metadata': []}, {'prefix': 'eco', 'mirId': 'MIR:00000055', 'name': 'Evidence Code Ontology', 'pattern': 'ECO:\\\\d{7}$', 'description': \"Evidence codes can be used to specify the type of supporting evidence for a piece of knowledge. This allows inference of a 'level of support' between an entity and an annotation made to an entity.\", 'created': '2019-06-11T14:15:47.447+0000', 'modified': '2019-06-11T14:15:47.447+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000006', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/197'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/197'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/197/contactPerson'}}, 'metadata': []}, {'prefix': 'mod', 'mirId': 'MIR:00000056', 'name': 'Protein Modification Ontology', 'pattern': '^MOD:\\\\d{5}', 'description': 'The Proteomics Standards Initiative modification ontology (PSI-MOD)  aims to define a concensus nomenclature and ontology reconciling, in a hierarchical representation, the complementary descriptions of residue modifications.', 'created': '2019-06-11T14:15:47.775+0000', 'modified': '2019-06-11T14:15:47.775+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '00001', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/200'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/200'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/200/contactPerson'}}, 'metadata': []}, {'prefix': 'sgd.pathways', 'mirId': 'MIR:00000057', 'name': 'Saccharomyces genome database pathways', 'pattern': '^PWY\\\\w{2}\\\\-\\\\d{3}$', 'description': 'Curated biochemical pathways for Saccharomyces cerevisiae at Saccharomyces genome database (SGD).', 'created': '2019-06-11T14:15:48.132+0000', 'modified': '2019-06-11T14:15:48.132+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PWY3O-214', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/203'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/203'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/203/contactPerson'}}, 'metadata': []}, {'prefix': 'biogrid', 'mirId': 'MIR:00000058', 'name': 'BioGRID', 'pattern': '^\\\\d+$', 'description': 'BioGRID is a database of physical and genetic interactions in Saccharomyces cerevisiae, Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, and Schizosaccharomyces pombe.', 'created': '2019-06-11T14:15:48.429+0000', 'modified': '2019-06-11T14:15:48.429+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '31623', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/206'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/206'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/206/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org/', '@type': 'Dataset', 'name': 'BIOGRID CURATED DATA FOR STE11 (Saccharomyces cerevisiae (S288c))', 'description': 'Protein-Protein, Genetic, and Chemical Interactions for STE11 (Saccharomyces cerevisiae (S288c)) curated by BioGRID (https://thebiogrid.org); DEFINITION: Signal transducing MEK kinase; involved in pheromone response and pseudohyphal/invasive growth pathways where it phosphorylates Ste7p, and the high osmolarity response pathway, via phosphorylation of Pbs2p; regulated by Ste20p and Ste50p; protein abundance increases in response to DNA replication stress', 'url': 'https://thebiogrid.org/31623/summary/saccharomyces-cerevisiae/ste11.html', 'keywords': ['BIOLOGY', 'GENE', 'PROTEIN', 'BIOGRID', 'SACCHAROMYCES CEREVISIAE (S288C)', 'STE11', 'PROTEIN-PROTEIN INTERACTION', 'GENETIC INTERACTION', 'CHEMICAL INTERACTION', 'POST-TRANSLATIONAL MODIFICATION', 'GENE ONTOLOGY', 'CHEMICAL-GENETIC INTERACTION', 'PROTEIN SEQUENCE'], 'creator': {'@type': 'Organization', 'url': 'https://thebiogrid.org', 'name': 'BioGRID Project', 'contactPoint': {'@type': 'ContactPoint', 'contactType': 'Customer Service', 'email': 'biogridadmin@gmail.com', 'url': 'https://wiki.thebiogrid.org/doku.php/aboutus'}}, 'citation': 'https://identifiers.org/biogrid:31623', 'distribution': [{'@type': 'DataDownload', 'encodingFormat': 'ZIP', 'contentURL': 'https://thebiogrid.org/downloadgenerator.php?geneID=31623'}], 'license': {'@type': 'CreativeWork', 'name': 'MIT License', 'url': 'https://en.wikipedia.org/wiki/MIT_License'}}]}, {'prefix': 'merops', 'mirId': 'MIR:00000059', 'name': 'MEROPS', 'pattern': '^[SCTAGMNU]\\\\d{2}\\\\.([AB]\\\\d{2}|\\\\d{3})$', 'description': 'The MEROPS database is an information resource for peptidases (also termed proteases, proteinases and proteolytic enzymes) and the proteins that inhibit them.', 'created': '2019-06-11T14:15:48.677+0000', 'modified': '2019-06-11T14:15:48.677+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'S01.001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/209'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/209'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/209/contactPerson'}}, 'metadata': []}, {'prefix': 'panther.family', 'mirId': 'MIR:00000060', 'name': 'PANTHER Family', 'pattern': '^PTHR\\\\d{5}(\\\\:SF\\\\d{1,3})?$', 'description': 'The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System is a resource that classifies genes by their functions, using published scientific experimental evidence and evolutionary relationships to predict function even in the absence of direct experimental evidence. This collection references groups of genes that have been organised as families.', 'created': '2019-06-11T14:15:48.923+0000', 'modified': '2019-06-11T14:15:48.923+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PTHR12345', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/212'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/212'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/212/contactPerson'}}, 'metadata': []}, {'prefix': 'prints', 'mirId': 'MIR:00000061', 'name': 'PRINTS', 'pattern': '^PR\\\\d{5}$', 'description': 'PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of a SWISS-PROT/TrEMBL composite. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs, full diagnostic potency deriving from the mutual context provided by motif neighbours.', 'created': '2019-06-11T14:15:49.123+0000', 'modified': '2019-06-11T14:15:49.123+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PR00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/214'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/214'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/214/contactPerson'}}, 'metadata': []}, {'prefix': 'ligandexpo', 'mirId': 'MIR:00000062', 'name': 'Ligand Expo', 'pattern': '^(\\\\w){3}$', 'description': 'Ligand Expo is a data resource for finding information about small molecules bound to proteins and nucleic acids.', 'created': '2019-06-11T14:15:49.385+0000', 'modified': '2019-06-11T14:15:49.385+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ABC', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/217'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/217'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/217/contactPerson'}}, 'metadata': []}, {'prefix': 'mge', 'mirId': 'MIR:00000063', 'name': 'Aclame', 'pattern': '^mge:\\\\d+$', 'description': 'ACLAME is a database dedicated to the collection and classification of mobile genetic elements (MGEs) from various sources, comprising all known phage genomes, plasmids and transposons.', 'created': '2019-06-11T14:15:49.786+0000', 'modified': '2019-06-11T14:15:49.786+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/220'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/220'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/220/contactPerson'}}, 'metadata': []}, {'prefix': 'isbn', 'mirId': 'MIR:00000064', 'name': 'ISBN', 'pattern': '^(ISBN)?(-13|-10)?[:]?[ ]?(\\\\d{2,3}[ -]?)?\\\\d{1,5}[ -]?\\\\d{1,7}[ -]?\\\\d{1,6}[ -]?(\\\\d|X)$', 'description': 'The International Standard Book Number (ISBN) is for identifying printed books.', 'created': '2019-06-11T14:15:50.037+0000', 'modified': '2019-06-11T14:15:50.037+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '9781584885658', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/223'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/223'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/223/contactPerson'}}, 'metadata': []}, {'prefix': 'pride', 'mirId': 'MIR:00000065', 'name': 'PRIDE', 'pattern': '^\\\\d+$', 'description': 'The PRIDE PRoteomics IDEntifications database is a centralized, standards compliant, public data repository that provides protein and peptide identifications together with supporting evidence. This collection references experiments and assays.', 'created': '2019-06-11T14:15:50.454+0000', 'modified': '2019-06-11T14:15:50.454+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/228'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/228'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/228/contactPerson'}}, 'metadata': []}, {'prefix': '3dmet', 'mirId': 'MIR:00000066', 'name': '3DMET', 'pattern': '^B\\\\d{5}$', 'description': '3DMET is a database collecting three-dimensional structures of natural metabolites.', 'created': '2019-06-11T14:15:50.652+0000', 'modified': '2019-06-11T14:15:50.652+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'B00162', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/230'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/230'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/230/contactPerson'}}, 'metadata': []}, {'prefix': 'fma', 'mirId': 'MIR:00000067', 'name': 'FMA', 'pattern': '^FMA:\\\\d+$', 'description': 'The Foundational Model of Anatomy Ontology (FMA) is a biomedical informatics ontology. It is concerned with the representation of classes or types and relationships necessary for the symbolic representation of the phenotypic structure of the human body.  Specifically, the FMA is a domain ontology that represents a coherent body of explicit declarative knowledge about human anatomy.', 'created': '2019-06-11T14:15:50.895+0000', 'modified': '2019-06-11T14:15:50.895+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '67112', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/233'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/233'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/233/contactPerson'}}, 'metadata': []}, {'prefix': 'matrixdb.association', 'mirId': 'MIR:00000068', 'name': 'MatrixDB', 'pattern': '^([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])_.*|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9]_.*)|(GAG_.*)|(MULT_.*)|(PFRAG_.*)|(LIP_.*)|(CAT_.*)$', 'description': 'MatrixDB stores experimentally determined interactions involving at least one extracellular biomolecule. It includes mostly protein-protein and protein-glycosaminoglycan interactions, as well as interactions with lipids and cations.', 'created': '2019-06-11T14:15:51.252+0000', 'modified': '2019-06-11T14:15:51.252+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P00747__P07355', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/236'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/236'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/236/contactPerson'}}, 'metadata': []}, {'prefix': 'ncbigene', 'mirId': 'MIR:00000069', 'name': 'NCBI Gene', 'pattern': '^\\\\d+$', 'description': \"Entrez Gene is the NCBI's database for gene-specific information, focusing on completely sequenced genomes, those with an active research community to contribute gene-specific information, or those that are scheduled for intense sequence analysis.\", 'created': '2019-06-11T14:15:51.483+0000', 'modified': '2019-06-11T14:15:51.483+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '100010', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/239'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/239'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/239/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.genes', 'mirId': 'MIR:00000070', 'name': 'KEGG Genes', 'pattern': '^\\\\w+:[\\\\w\\\\d\\\\.-]*$', 'description': 'KEGG GENES is a collection of gene catalogs for all complete genomes and some partial genomes, generated from publicly available resources.', 'created': '2019-06-11T14:15:51.781+0000', 'modified': '2019-06-11T14:15:51.781+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'syn:ssr3451', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/242'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/242'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/242/contactPerson'}}, 'metadata': []}, {'prefix': 'brenda', 'mirId': 'MIR:00000071', 'name': 'BRENDA', 'pattern': '^((\\\\d+\\\\.-\\\\.-\\\\.-)|(\\\\d+\\\\.\\\\d+\\\\.-\\\\.-)|(\\\\d+\\\\.\\\\d+\\\\.\\\\d+\\\\.-)|(\\\\d+\\\\.\\\\d+\\\\.\\\\d+\\\\.\\\\d+))$', 'description': 'BRENDA is a collection of enzyme functional data available to the scientific community. Data on enzyme function are extracted directly from the primary literature The database covers information on classification and nomenclature, reaction and specificity, functional parameters, occurrence, enzyme structure and stability, mutants and enzyme engineering, preparation and isolation, the application of enzymes, and ligand-related data.', 'created': '2019-06-11T14:15:51.985+0000', 'modified': '2019-06-11T14:15:51.985+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1.1.1.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/244'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/244'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/244/contactPerson'}}, 'metadata': []}, {'prefix': 'pubchem.bioassay', 'mirId': 'MIR:00000072', 'name': 'PubChem-bioassay', 'pattern': '^\\\\d+$', 'description': \"PubChem provides information on the biological activities of small molecules. It is a component of NIH's Molecular Libraries Roadmap Initiative. PubChem bioassay archives active compounds and bioassay results.\", 'created': '2019-06-11T14:15:52.227+0000', 'modified': '2019-06-11T14:15:52.227+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1018', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/247'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/247'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/247/contactPerson'}}, 'metadata': []}, {'prefix': 'bao', 'mirId': 'MIR:00000597', 'name': 'BioAssay Ontology', 'pattern': '^\\\\d{7}$', 'description': 'The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.', 'created': '2019-06-11T14:17:58.671+0000', 'modified': '2019-06-11T14:17:58.671+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0002989', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1659'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1659'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1659/contactPerson'}}, 'metadata': []}, {'prefix': 'pathwaycommons', 'mirId': 'MIR:00000073', 'name': 'Pathway Commons', 'pattern': '^\\\\d+$', 'description': 'Pathway Commons is a convenient point of access to biological pathway information collected from public pathway databases, which you can browse or search. It is a collection of publicly available pathways from multiple organisms that provides researchers with convenient access to a comprehensive collection of pathways from multiple sources represented in a common language.', 'created': '2019-06-11T14:15:52.436+0000', 'modified': '2019-06-11T14:15:52.436+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '485991', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/249'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/249'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/249/contactPerson'}}, 'metadata': []}, {'prefix': 'hovergen', 'mirId': 'MIR:00000074', 'name': 'HOVERGEN', 'pattern': '^HBG\\\\d+$', 'description': 'HOVERGEN is a database of homologous vertebrate genes that allows one to select sets of homologous genes among vertebrate species, and to visualize multiple alignments and phylogenetic trees.', 'created': '2019-06-11T14:15:52.667+0000', 'modified': '2019-06-11T14:15:52.667+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HBG004341', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/252'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/252'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/252/contactPerson'}}, 'metadata': []}, {'prefix': 'mmmp:biomaps', 'mirId': 'MIR:00000075', 'name': 'Melanoma Molecular Map Project Biomaps', 'pattern': '^\\\\d+$', 'description': 'A collection of molecular interaction maps and pathways involved in cancer development and progression with a focus on melanoma.', 'created': '2019-06-11T14:15:52.916+0000', 'modified': '2019-06-11T14:15:52.916+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '37', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/255'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/255'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/255/contactPerson'}}, 'metadata': []}, {'prefix': 'wikipathways', 'mirId': 'MIR:00000076', 'name': 'WikiPathways', 'pattern': 'WP\\\\d{1,5}(\\\\_r\\\\d+)?$', 'description': 'WikiPathways is a resource providing an open and public collection of  pathway maps created and curated by the community in a Wiki like style.\\r\\nAll content is under the Creative Commons Attribution 3.0 Unported license.', 'created': '2019-06-11T14:15:53.125+0000', 'modified': '2019-06-11T14:15:53.125+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'WP100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/258'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/258'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/258/contactPerson'}}, 'metadata': []}, {'prefix': 'macie', 'mirId': 'MIR:00000077', 'name': 'MACiE', 'pattern': '^M\\\\d{4}$', 'description': 'MACiE (Mechanism, Annotation and Classification in Enzymes) is a database of enzyme reaction mechanisms. Each entry in MACiE consists of an overall reaction describing the chemical compounds involved, as well as the species name in which the reaction occurs. The individual reaction stages for each overall reaction are listed with mechanisms, alternative mechanisms, and amino acids involved.', 'created': '2019-06-11T14:15:53.388+0000', 'modified': '2019-06-11T14:15:53.388+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'M0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/261'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/261'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/261/contactPerson'}}, 'metadata': []}, {'prefix': 'mirbase', 'mirId': 'MIR:00000078', 'name': 'miRBase Sequence', 'pattern': 'MI\\\\d{7}', 'description': 'The miRBase Sequence Database is a searchable database of published miRNA sequences and annotation. The data were previously provided by the miRNA Registry. Each entry in the miRBase Sequence database represents a predicted hairpin portion of a miRNA transcript (termed mir in the database), with information on the location and sequence of the mature miRNA sequence (termed miR).', 'created': '2019-06-11T14:15:53.607+0000', 'modified': '2019-06-11T14:15:53.607+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MI0000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/263'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/263'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/263/contactPerson'}}, 'metadata': []}, {'prefix': 'zfin', 'mirId': 'MIR:00000079', 'name': 'ZFIN Bioentity', 'pattern': '^ZDB\\\\-\\\\w+\\\\-\\\\d+\\\\-\\\\d+$', 'description': 'ZFIN serves as the zebrafish model organism database. This collection references all zebrafish biological entities in ZFIN.', 'created': '2019-06-11T14:15:53.859+0000', 'modified': '2019-06-11T14:15:53.859+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ZDB-GENE-041118-11', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/266'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/266'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/266/contactPerson'}}, 'metadata': []}, {'prefix': 'hgnc', 'mirId': 'MIR:00000080', 'name': 'HGNC', 'pattern': '^((HGNC|hgnc):)?\\\\d{1,5}$', 'description': 'The HGNC (HUGO Gene Nomenclature Committee) provides an approved gene name and symbol (short-form abbreviation) for each known human gene.  All approved symbols are stored in the HGNC database, and each symbol is unique. HGNC identifiers refer to records in the HGNC symbol database.', 'created': '2019-06-11T14:15:54.375+0000', 'modified': '2019-06-11T14:15:54.375+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2674', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/271'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/271'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/271/contactPerson'}}, 'metadata': []}, {'prefix': 'so', 'mirId': 'MIR:00000081', 'name': 'Sequence Ontology', 'pattern': '^SO:\\\\d{7}$', 'description': 'The Sequence Ontology (SO) is a structured controlled vocabulary for the parts of a genomic annotation. It provides a common set of terms and definitions to facilitate the exchange, analysis and management of genomic data.', 'created': '2019-06-11T14:15:54.604+0000', 'modified': '2019-06-11T14:15:54.604+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000704', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/273'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/273'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/273/contactPerson'}}, 'metadata': []}, {'prefix': 'rhea', 'mirId': 'MIR:00000082', 'name': 'Rhea', 'pattern': '^\\\\d{5}$', 'description': 'Rhea is a manually annotated reaction database, where all reaction participants (reactants and products) are linked to the ChEBI database (Chemical Entities of Biological Interest), providing detailed information about structure, formulae and charge. It is populated with the reactions found in the EC list, IntEnz and ENZYME databases), as well as other biochemical reactions, including those that are often termed \"spontaneous\".', 'created': '2019-06-11T14:15:55.126+0000', 'modified': '2019-06-11T14:15:55.126+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '12345', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/278'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/278'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/278/contactPerson'}}, 'metadata': []}, {'prefix': 'chembl.compound', 'mirId': 'MIR:00000084', 'name': 'ChEMBL compound', 'pattern': '^CHEMBL\\\\d+$', 'description': 'ChEMBL is a database of bioactive compounds, their quantitative properties and bioactivities (binding constants, pharmacology and ADMET, etc). The data is abstracted and curated from the primary scientific literature.', 'created': '2019-06-11T14:15:55.368+0000', 'modified': '2019-06-11T14:15:55.368+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CHEMBL308052', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/281'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/281'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/281/contactPerson'}}, 'metadata': [{'identifier': 'CHEMBL308052', 'inChIKey': 'BEWLPDVTVIFJAJ-ZPGRZCPFSA-N', '@context': 'http://schema.org', 'biochemicalInteraction': [{'identifier': 'CHEMBL3879801', 'url': 'www.ebi.ac.uk/chembl/target_report_card/CHEMBL3879801/', '@type': 'BioChemEntity'}, {'identifier': 'CHEMBL3974', 'url': 'www.ebi.ac.uk/chembl/target_report_card/CHEMBL3974/', '@type': 'BioChemEntity'}], 'inChI': 'InChI=1S/C40H42Cl2F2N8O3/c41-31-7-5-8-32(42)30(31)24-52-23-26(22-51-16-3-4-17-51)29-11-10-27(20-37(29)52)48-40(55)50-36(19-25-9-12-33(43)34(44)18-25)39(54)49-35(13-14-45)38(53)47-21-28-6-1-2-15-46-28/h1-2,5-12,15,18,20,23,35-36H,3-4,13-14,16-17,19,21-22,24,45H2,(H,47,53)(H,49,54)(H2,48,50,55)/t35-,36-/m0/s1', 'canonical_smiles': ['NCC[C@H](NC(=O)[C@H](Cc1ccc(F)c(F)c1)NC(=O)Nc2ccc3c(CN4CCCC4)cn(Cc5c(Cl)cccc5Cl)c3c2)C(=O)NCc6ccccn6'], '@type': 'MolecularEntity', 'image': 'https://www.ebi.ac.uk/chembl/api/data/image/CHEMBL308052.svg?engine=indigo', 'molecularWeight': 791.73, 'url': 'https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL308052/', 'molecularFormula': 'C40H42Cl2F2N8O3', 'monoisotopicMolecularWeight': 790.2725}]}, {'prefix': 'chembl.target', 'mirId': 'MIR:00000085', 'name': 'ChEMBL target', 'pattern': '^CHEMBL\\\\d+$', 'description': 'ChEMBL is a database of bioactive compounds, their quantitative properties and bioactivities (binding constants, pharmacology and ADMET, etc). The data is abstracted and curated from the primary scientific literature.', 'created': '2019-06-11T14:15:55.880+0000', 'modified': '2019-06-11T14:15:55.880+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CHEMBL3467', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/286'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/286'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/286/contactPerson'}}, 'metadata': []}, {'prefix': 'sabiork.kineticrecord', 'mirId': 'MIR:00000086', 'name': 'SABIO-RK Kinetic Record', 'pattern': '^\\\\d+$', 'description': 'SABIO-RK is a relational database system that contains information about biochemical reactions, their kinetic equations with their parameters, and the experimental conditions under which these parameters were measured. The kinetic record data set provides information regarding the kinetic law, measurement conditions, parameter details and other reference information.', 'created': '2019-06-11T14:15:56.334+0000', 'modified': '2019-06-11T14:15:56.334+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '5046', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/290'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/290'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/290/contactPerson'}}, 'metadata': []}, {'prefix': 'lgic', 'mirId': 'MIR:00000087', 'name': 'Ligand-Gated Ion Channel database', 'pattern': '^\\\\w+$', 'description': 'The Ligand-Gated Ion Channel database provides nucleic and proteic sequences of the subunits of ligand-gated ion channels. These transmembrane proteins can exist under different conformations, at least one of which forms a pore through the membrane connecting two neighbouring compartments. The database can be used to generate multiple sequence alignments from selected subunits, and gives the atomic coordinates of subunits, or portion of subunits, where available.', 'created': '2019-06-11T14:15:56.548+0000', 'modified': '2019-06-11T14:15:56.548+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '5HT3Arano', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/292'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/292'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/292/contactPerson'}}, 'metadata': []}, {'prefix': 'atc', 'mirId': 'MIR:00000088', 'name': 'Anatomical Therapeutic Chemical', 'pattern': '^[A-Z](\\\\d+([A-Z]{1,2}(\\\\d+)?)?)?$', 'description': 'The Anatomical Therapeutic Chemical (ATC) classification system, divides active substances into different groups according to the organ or system on which they act and their therapeutic, pharmacological and chemical properties. Drugs are classified in groups at five different levels;  Drugs are divided into fourteen main groups (1st level), with pharmacological/therapeutic subgroups (2nd level).  The 3rd and 4th levels are chemical/pharmacological/therapeutic subgroups and the 5th level is the chemical substance. The Anatomical Therapeutic Chemical (ATC) classification system and the Defined Daily Dose (DDD) is a tool for exchanging and comparing data on drug use at international, national or local levels.', 'created': '2019-06-11T14:15:56.747+0000', 'modified': '2019-06-11T14:15:56.747+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'A10BA02', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/294'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/294'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/294/contactPerson'}}, 'metadata': []}, {'prefix': 'pharmgkb.pathways', 'mirId': 'MIR:00000089', 'name': 'PharmGKB Pathways', 'pattern': '^PA\\\\d+$', 'description': 'The PharmGKB database is a central repository for genetic, genomic, molecular and cellular phenotype data and clinical information about people who have participated in pharmacogenomics research studies. The data includes, but is not limited to, clinical and basic pharmacokinetic and pharmacogenomic research in the cardiovascular, pulmonary, cancer, pathways, metabolic and transporter domains. \\r\\nPharmGKB Pathways are drug centric, gene based, interactive pathways which focus on candidate genes and gene groups and associated genotype and phenotype data of relevance for pharmacogenetic and pharmacogenomic studies.', 'created': '2019-06-11T14:15:57.006+0000', 'modified': '2019-06-11T14:15:57.006+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PA146123006', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/297'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/297'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/297/contactPerson'}}, 'metadata': []}, {'prefix': 'pharmgkb.disease', 'mirId': 'MIR:00000090', 'name': 'PharmGKB Disease', 'pattern': '^PA\\\\d+$', 'description': 'The PharmGKB database is a central repository for genetic, genomic, molecular and cellular phenotype data and clinical information about people who have participated in pharmacogenomics research studies. The data includes, but is not limited to, clinical and basic pharmacokinetic and pharmacogenomic research in the cardiovascular, pulmonary, cancer, pathways, metabolic and transporter domains.', 'created': '2019-06-11T14:15:57.258+0000', 'modified': '2019-06-11T14:15:57.258+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PA447218', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/300'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/300'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/300/contactPerson'}}, 'metadata': []}, {'prefix': 'pharmgkb.drug', 'mirId': 'MIR:00000091', 'name': 'PharmGKB Drug', 'pattern': '^PA\\\\d+$', 'description': 'The PharmGKB database is a central repository for genetic, genomic, molecular and cellular phenotype data and clinical information about people who have participated in pharmacogenomics research studies. The data includes, but is not limited to, clinical and basic pharmacokinetic and pharmacogenomic research in the cardiovascular, pulmonary, cancer, pathways, metabolic and transporter domains.', 'created': '2019-06-11T14:15:57.451+0000', 'modified': '2019-06-11T14:15:57.451+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PA448710', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/302'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/302'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/302/contactPerson'}}, 'metadata': []}, {'prefix': 'ttd.drug', 'mirId': 'MIR:00000092', 'name': 'TTD Drug', 'pattern': '^DAP\\\\d+$', 'description': 'The Therapeutic Target Database (TTD) is designed to provide information about the known therapeutic protein and nucleic acid targets described in the literature, the targeted disease conditions, the pathway information and the corresponding drugs/ligands directed at each of these targets. Cross-links to other databases allow the access to information about the sequence, 3D structure, function, nomenclature, drug/ligand binding properties, drug usage and effects, and related literature for each target.', 'created': '2019-06-11T14:15:57.654+0000', 'modified': '2019-06-11T14:15:57.654+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DAP000773', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/304'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/304'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/304/contactPerson'}}, 'metadata': []}, {'prefix': 'ttd.target', 'mirId': 'MIR:00000093', 'name': 'TTD Target', 'pattern': '^TTDS\\\\d+$', 'description': 'The Therapeutic Target Database (TTD) is designed to provide information about the known therapeutic protein and nucleic acid targets described in the literature, the targeted disease conditions, the pathway information and the corresponding drugs/ligands directed at each of these targets. Cross-links to other databases are also introduced to facilitate the access of information about the sequence, 3D structure, function, nomenclature, drug/ligand binding properties, drug usage and effects, and related literature for each target.', 'created': '2019-06-11T14:15:57.896+0000', 'modified': '2019-06-11T14:15:57.896+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TTDS00056', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/307'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/307'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/307/contactPerson'}}, 'metadata': []}, {'prefix': 'neurondb', 'mirId': 'MIR:00000094', 'name': 'NeuronDB', 'pattern': '^\\\\d+$', 'description': 'NeuronDB provides a dynamically searchable database of three types of neuronal properties: voltage gated conductances, neurotransmitter receptors, and neurotransmitter substances. It contains tools that provide for integration of these properties in a given type of neuron and compartment, and for comparison of properties across different types of neurons and compartments.', 'created': '2019-06-11T14:15:58.106+0000', 'modified': '2019-06-11T14:15:58.106+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '265', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/309'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/309'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/309/contactPerson'}}, 'metadata': []}, {'prefix': 'neuromorpho', 'mirId': 'MIR:00000095', 'name': 'NeuroMorpho', 'pattern': '^\\\\w+$', 'description': 'NeuroMorpho.Org is a centrally curated inventory of  digitally reconstructed neurons.', 'created': '2019-06-11T14:15:58.342+0000', 'modified': '2019-06-11T14:15:58.342+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Rosa2', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/312'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/312'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/312/contactPerson'}}, 'metadata': []}, {'prefix': 'chemidplus', 'mirId': 'MIR:00000096', 'name': 'ChemIDplus', 'pattern': '^\\\\d+\\\\-\\\\d+\\\\-\\\\d+$', 'description': 'ChemIDplus is a web-based search system that provides access to structure and nomenclature authority files used for the identification of chemical substances cited in National Library of Medicine (NLM) databases. It also provides structure searching and direct links to many biomedical resources at NLM and on the Internet for chemicals of interest.', 'created': '2019-06-11T14:15:58.577+0000', 'modified': '2019-06-11T14:15:58.577+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '57-27-2', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/315'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/315'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/315/contactPerson'}}, 'metadata': []}, {'prefix': 'biosystems', 'mirId': 'MIR:00000097', 'name': 'BioSystems', 'pattern': '^\\\\d+$', 'description': 'The NCBI BioSystems database centralizes and cross-links existing biological systems databases, increasing their utility and target audience by integrating their pathways and systems into NCBI resources.', 'created': '2019-06-11T14:15:58.906+0000', 'modified': '2019-06-11T14:15:58.906+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/318'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/318'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/318/contactPerson'}}, 'metadata': []}, {'prefix': 'ctd.chemical', 'mirId': 'MIR:00000098', 'name': 'CTD Chemical', 'pattern': '^[CD]\\\\d+$', 'description': 'The Comparative Toxicogenomics Database (CTD) presents scientifically reviewed and curated information on chemicals, relevant genes and proteins, and their interactions in vertebrates and invertebrates. It integrates sequence, reference, species, microarray, and general toxicology information to provide a unique centralized resource for toxicogenomic research. The database also provides visualization capabilities that enable cross-species comparisons of gene and protein sequences.', 'created': '2019-06-11T14:15:59.108+0000', 'modified': '2019-06-11T14:15:59.108+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'D001151', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/320'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/320'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/320/contactPerson'}}, 'metadata': []}, {'prefix': 'ctd.disease', 'mirId': 'MIR:00000099', 'name': 'CTD Disease', 'pattern': '^D\\\\d+$', 'description': 'The Comparative Toxicogenomics Database (CTD) presents scientifically reviewed and curated information on chemicals, relevant genes and proteins, and their interactions in vertebrates and invertebrates. It integrates sequence, reference, species, microarray, and general toxicology information to provide a unique centralized resource for toxicogenomic research. The database also provides visualization capabilities that enable cross-species comparisons of gene and protein sequences.', 'created': '2019-06-11T14:15:59.348+0000', 'modified': '2019-06-11T14:15:59.348+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'D053716', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/323'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/323'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/323/contactPerson'}}, 'metadata': []}, {'prefix': 'ctd.gene', 'mirId': 'MIR:00000100', 'name': 'CTD Gene', 'pattern': '^\\\\d+$', 'description': 'The Comparative Toxicogenomics Database (CTD) presents scientifically reviewed and curated information on chemicals, relevant genes and proteins, and their interactions in vertebrates and invertebrates. It integrates sequence, reference, species, microarray, and general toxicology information to provide a unique centralized resource for toxicogenomic research. The database also provides visualization capabilities that enable cross-species comparisons of gene and protein sequences.', 'created': '2019-06-11T14:15:59.547+0000', 'modified': '2019-06-11T14:15:59.547+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '101', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/325'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/325'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/325/contactPerson'}}, 'metadata': []}, {'prefix': 'bionumbers', 'mirId': 'MIR:00000101', 'name': 'BioNumbers', 'pattern': '^\\\\d+$', 'description': 'BioNumbers is a database of key numberical information that may be used in molecular biology. Along with the numbers, it contains references to the original literature, useful comments, and related numeric data.', 'created': '2019-06-11T14:15:59.756+0000', 'modified': '2019-06-11T14:15:59.756+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '104674', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/327'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/327'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/327/contactPerson'}}, 'metadata': []}, {'prefix': 'drugbank', 'mirId': 'MIR:00000102', 'name': 'DrugBank', 'pattern': '^DB\\\\d{5}$', 'description': 'The DrugBank database is a bioinformatics and chemoinformatics resource that combines detailed drug (i.e. chemical, pharmacological and pharmaceutical) data with comprehensive drug target (i.e. sequence, structure, and pathway) information. This collection references drug information.', 'created': '2019-06-11T14:16:00.005+0000', 'modified': '2019-06-11T14:16:00.005+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DB00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/330'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/330'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/330/contactPerson'}}, 'metadata': []}, {'prefix': 't3db', 'mirId': 'MIR:00000103', 'name': 'T3DB', 'pattern': '^T3D\\\\d+$', 'description': 'Toxin and Toxin Target Database (T3DB) is a bioinformatics resource that combines detailed toxin data with comprehensive toxin target information.', 'created': '2019-06-11T14:16:00.251+0000', 'modified': '2019-06-11T14:16:00.251+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'T3D0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/333'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/333'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/333/contactPerson'}}, 'metadata': []}, {'prefix': 'smpdb', 'mirId': 'MIR:00000104', 'name': 'Small Molecule Pathway Database', 'pattern': '^SMP\\\\d+$', 'description': 'The Small Molecule Pathway Database (SMPDB) contains small molecule pathways found in humans, which are presented visually. All SMPDB pathways include information on the relevant organs, subcellular compartments, protein cofactors, protein locations, metabolite locations, chemical structures and protein quaternary structures. Accompanying data includes detailed descriptions and references, providing an overview of the pathway, condition or processes depicted in each diagram.', 'created': '2019-06-11T14:16:00.499+0000', 'modified': '2019-06-11T14:16:00.499+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SMP00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/336'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/336'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/336/contactPerson'}}, 'metadata': []}, {'prefix': 'phosphosite.protein', 'mirId': 'MIR:00000105', 'name': 'PhosphoSite Protein', 'pattern': '^\\\\d{5}$', 'description': 'PhosphoSite is a mammalian protein database that provides information about in vivo phosphorylation sites. This datatype refers to protein-level information, providing a list of phosphorylation sites for each protein in the database.', 'created': '2019-06-11T14:16:00.712+0000', 'modified': '2019-06-11T14:16:00.712+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '12300', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/338'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/338'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/338/contactPerson'}}, 'metadata': []}, {'prefix': 'genedb', 'mirId': 'MIR:00000106', 'name': 'GeneDB', 'pattern': '^[\\\\w\\\\d\\\\.-]*$', 'description': 'GeneDB is a genome database for prokaryotic and eukaryotic organisms and provides a portal through which data generated by the \"Pathogen Genomics\" group at the Wellcome Trust Sanger Institute and other collaborating sequencing centres can be accessed.', 'created': '2019-06-11T14:16:00.951+0000', 'modified': '2019-06-11T14:16:00.951+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LinJ.20.0070', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/341'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/341'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/341/contactPerson'}}, 'metadata': []}, {'prefix': 'biomodels.teddy', 'mirId': 'MIR:00000107', 'name': 'TEDDY', 'pattern': '^TEDDY_\\\\d{7}$', 'description': 'The Terminology for Description of Dynamics (TEDDY) is an ontology for dynamical behaviours, observable dynamical phenomena, and control elements of bio-models and biological systems in Systems Biology and Synthetic Biology.', 'created': '2019-06-11T14:16:01.178+0000', 'modified': '2019-06-11T14:16:01.178+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TEDDY_0000066', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/344'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/344'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/344/contactPerson'}}, 'metadata': []}, {'prefix': 'biomodels.kisao', 'mirId': 'MIR:00000108', 'name': 'KiSAO', 'pattern': '^KISAO_\\\\d+$', 'description': 'The Kinetic Simulation Algorithm Ontology (KiSAO) is an ontology that describes simulation algorithms and methods used for biological kinetic models, and the relationships between them. This provides a means to unambiguously refer to simulation algorithms when describing a simulation experiment.', 'created': '2019-06-11T14:16:01.508+0000', 'modified': '2019-06-11T14:16:01.508+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'KISAO_0000057', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/347'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/347'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/347/contactPerson'}}, 'metadata': []}, {'prefix': 'psimi', 'mirId': 'MIR:00000109', 'name': 'Molecular Interactions Ontology', 'pattern': '^MI:\\\\d{4}$', 'description': 'The Molecular Interactions (MI) ontology forms a structured controlled vocabulary for the annotation of experiments concerned with protein-protein interactions. MI is developed by the HUPO Proteomics Standards Initiative.', 'created': '2019-06-11T14:16:01.860+0000', 'modified': '2019-06-11T14:16:01.860+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0308', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/350'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/350'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/350/contactPerson'}}, 'metadata': []}, {'prefix': 'cl', 'mirId': 'MIR:00000110', 'name': 'Cell Type Ontology', 'pattern': '^CL:\\\\d{7}$', 'description': 'The Cell Ontology is designed as a structured controlled vocabulary for cell types. The ontology was constructed for use by the model organism and other bioinformatics databases, incorporating cell types from prokaryotes to mammals, and includes plants and fungi.', 'created': '2019-06-11T14:16:02.066+0000', 'modified': '2019-06-11T14:16:02.066+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000232', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/352'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/352'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/352/contactPerson'}}, 'metadata': []}, {'prefix': 'bto', 'mirId': 'MIR:00000111', 'name': 'Brenda Tissue Ontology', 'pattern': '^BTO:\\\\d{7}$', 'description': 'The Brenda tissue ontology is a structured controlled vocabulary eastablished to identify the source of an enzyme cited in the Brenda enzyme database. It comprises terms of tissues, cell lines, cell types and cell cultures from uni- and multicellular organisms.', 'created': '2019-06-11T14:16:02.399+0000', 'modified': '2019-06-11T14:16:02.399+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000146', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/355'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/355'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/355/contactPerson'}}, 'metadata': []}, {'prefix': 'pato', 'mirId': 'MIR:00000112', 'name': 'PATO', 'pattern': '^PATO:\\\\d{7}$', 'description': 'PATO is an ontology of phenotypic qualities, intended for use in a number of applications, primarily defining composite phenotypes and phenotype  annotation.', 'created': '2019-06-11T14:16:02.730+0000', 'modified': '2019-06-11T14:16:02.730+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0001998', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/358'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/358'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/358/contactPerson'}}, 'metadata': []}, {'prefix': 'pdb-ccd', 'mirId': 'MIR:00000113', 'name': 'Chemical Component Dictionary', 'pattern': '^\\\\w{3}$', 'description': 'The Chemical Component Dictionary is as an external reference file describing all residue and small molecule components found in Protein Data Bank entries. It contains detailed chemical descriptions for standard and modified amino acids/nucleotides, small molecule ligands, and solvent molecules. Each chemical definition includes descriptions of chemical properties such as stereochemical assignments, aromatic bond assignments, idealized coordinates, chemical descriptors (SMILES & InChI), and systematic chemical names.', 'created': '2019-06-11T14:16:03.054+0000', 'modified': '2019-06-11T14:16:03.054+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AB0', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/361'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/361'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/361/contactPerson'}}, 'metadata': []}, {'prefix': 'glycomedb', 'mirId': 'MIR:00000114', 'name': 'GlycomeDB', 'pattern': '^\\\\w+$', 'description': 'GlycomeDB is the result of a systematic data integration effort, and provides an overview of all carbohydrate structures available in public databases, as well as cross-links.', 'created': '2019-06-11T14:16:03.253+0000', 'modified': '2019-06-11T14:16:03.253+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'G77500AY', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/363'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/363'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/363/contactPerson'}}, 'metadata': []}, {'prefix': 'lipidbank', 'mirId': 'MIR:00000115', 'name': 'LipidBank', 'pattern': '^\\\\w+\\\\d+$', 'description': 'LipidBank is an open, publicly free database of natural lipids including fatty acids, glycerolipids, sphingolipids, steroids, and various vitamins.', 'created': '2019-06-11T14:16:03.490+0000', 'modified': '2019-06-11T14:16:03.490+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BBA0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/366'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/366'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/366/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.orthology', 'mirId': 'MIR:00000116', 'name': 'KEGG Orthology', 'pattern': '^K\\\\d+$', 'description': 'KEGG Orthology (KO) consists of manually defined, generalised ortholog groups that correspond to KEGG pathway nodes and BRITE hierarchy nodes in all organisms.', 'created': '2019-06-11T14:16:03.735+0000', 'modified': '2019-06-11T14:16:03.735+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'K00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/369'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/369'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/369/contactPerson'}}, 'metadata': []}, {'prefix': 'prodom', 'mirId': 'MIR:00000117', 'name': 'ProDom', 'pattern': '^PD\\\\d+$', 'description': 'ProDom is a database of protein domain families generated from the global comparison of all available protein sequences.', 'created': '2019-06-11T14:16:03.923+0000', 'modified': '2019-06-11T14:16:03.923+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PD10000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/371'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/371'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/371/contactPerson'}}, 'metadata': []}, {'prefix': 'smart', 'mirId': 'MIR:00000118', 'name': 'SMART', 'pattern': '^SM\\\\d{5}$', 'description': 'The Simple Modular Architecture Research Tool (SMART) is an online tool for the identification and annotation of protein domains, and the analysis of domain architectures.', 'created': '2019-06-11T14:16:04.151+0000', 'modified': '2019-06-11T14:16:04.151+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SM00015', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/374'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/374'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/374/contactPerson'}}, 'metadata': []}, {'prefix': 'cdd', 'mirId': 'MIR:00000119', 'name': 'Conserved Domain Database', 'pattern': '^(cd)?\\\\d{5}$', 'description': 'The Conserved Domain Database (CDD) is a collection of multiple sequence alignments and derived database search models, which represent protein domains conserved in molecular evolution.', 'created': '2019-06-11T14:16:04.400+0000', 'modified': '2019-06-11T14:16:04.400+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'cd00400', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/377'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/377'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/377/contactPerson'}}, 'metadata': []}, {'prefix': 'ro', 'mirId': 'MIR:00000120', 'name': 'Relation Ontology', 'pattern': '^RO_\\\\d{7}$', 'description': 'The OBO Relation Ontology provides consistent and unambiguous formal definitions of the relational expressions used in biomedical ontologies.', 'created': '2019-06-11T14:16:04.654+0000', 'modified': '2019-06-11T14:16:04.654+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RO_0002533', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/380'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/380'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/380/contactPerson'}}, 'metadata': []}, {'prefix': 'mmdb', 'mirId': 'MIR:00000121', 'name': 'Molecular Modeling Database', 'pattern': '^\\\\d{1,5}$', 'description': 'The Molecular Modeling Database (MMDB) is a database of experimentally determined structures obtained from the Protein Data Bank (PDB). Since structures are known for a large fraction of all protein families, structure homologs may facilitate inference of biological function, or the identification of binding or catalytic sites.', 'created': '2019-06-11T14:16:05.062+0000', 'modified': '2019-06-11T14:16:05.062+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '50885', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/384'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/384'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/384/contactPerson'}}, 'metadata': []}, {'prefix': 'imex', 'mirId': 'MIR:00000122', 'name': 'IMEx', 'pattern': '^IM-\\\\d+(-?)(\\\\d+?)$', 'description': 'The International Molecular Exchange (IMEx) is a consortium of molecular interaction databases which collaborate to share manual curation efforts and provide accessibility to multiple information sources.', 'created': '2019-06-11T14:16:05.272+0000', 'modified': '2019-06-11T14:16:05.272+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'IM-19210-3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/386'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/386'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/386/contactPerson'}}, 'metadata': []}, {'prefix': 'irefweb', 'mirId': 'MIR:00000123', 'name': 'iRefWeb', 'pattern': '^\\\\d+$', 'description': 'iRefWeb is an interface to a relational database containing the latest build of the interaction Reference Index (iRefIndex) which integrates protein interaction data from ten different interaction databases: BioGRID, BIND, CORUM, DIP, HPRD, INTACT, MINT, MPPI, MPACT and OPHID. In addition, iRefWeb associates interactions with the PubMed record from which they are derived.', 'created': '2019-06-11T14:16:05.614+0000', 'modified': '2019-06-11T14:16:05.614+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '617102', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/389'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/389'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/389/contactPerson'}}, 'metadata': []}, {'prefix': 'mpid', 'mirId': 'MIR:00000124', 'name': 'Microbial Protein Interaction Database', 'pattern': '^\\\\d+$', 'description': 'The microbial protein interaction database (MPIDB) provides physical microbial interaction data. The interactions are manually curated from the literature or imported from other databases, and are linked to supporting experimental evidence, as well as evidences based on interaction conservation, protein complex membership, and 3D domain contacts.', 'created': '2019-06-11T14:16:05.849+0000', 'modified': '2019-06-11T14:16:05.849+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '172', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/392'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/392'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/392/contactPerson'}}, 'metadata': []}, {'prefix': 'phosphosite.residue', 'mirId': 'MIR:00000125', 'name': 'PhosphoSite Residue', 'pattern': '^\\\\d+$', 'description': 'PhosphoSite is a mammalian protein database that provides information about in vivo phosphorylation sites. This datatype refers to residue-level information, providing a information about a single modification position in a specific protein sequence.', 'created': '2019-06-11T14:16:06.208+0000', 'modified': '2019-06-11T14:16:06.208+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2842', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/396'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/396'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/396/contactPerson'}}, 'metadata': []}, {'prefix': 'neurolex', 'mirId': 'MIR:00000126', 'name': 'NeuroLex', 'pattern': '^([Bb]irnlex_|Sao|nlx_|GO_|CogPO|HDO|nifext_)\\\\d+$', 'description': 'The NeuroLex project is a dynamic lexicon of terms used in neuroscience.  It is supported by the Neuroscience Information Framework project and incorporates information from the NIF standardised ontology (NIFSTD), and its predecessor, the Biomedical Informatics Research Network Lexicon (BIRNLex).', 'created': '2019-06-11T14:16:06.417+0000', 'modified': '2019-06-11T14:16:06.417+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Birnlex_721', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/398'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/398'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/398/contactPerson'}}, 'metadata': []}, {'prefix': 'obi', 'mirId': 'MIR:00000127', 'name': 'Ontology for Biomedical Investigations', 'pattern': '(^OBI:\\\\d{7}$)|(^OBI_\\\\d{7}$)', 'description': 'The Ontology for Biomedical Investigations (OBI) project is developing an integrated ontology for the description of biological and clinical investigations. The ontology will represent the design of an investigation, the protocols and instrumentation used, the material used, the data generated and the type analysis performed on it. Currently OBI is being built under the Basic Formal Ontology (BFO).', 'created': '2019-06-11T14:16:06.780+0000', 'modified': '2019-06-11T14:16:06.780+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'OBI_0000070', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/402'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/402'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/402/contactPerson'}}, 'metadata': []}, {'prefix': 'asap', 'mirId': 'MIR:00000283', 'name': 'ASAP', 'pattern': '^[A-Za-z0-9-]+$', 'description': 'ASAP (a systematic annotation package for community analysis of genomes) stores bacterial genome sequence and functional characterization data. It includes multiple genome sequences at various stages of analysis, corresponding experimental data and access to collections of related genome resources.', 'created': '2019-06-11T14:16:44.568+0000', 'modified': '2019-06-11T14:16:44.568+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ABE-0009634', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/831'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/831'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/831/contactPerson'}}, 'metadata': []}, {'prefix': 'sabiork.ec', 'mirId': 'MIR:00000128', 'name': 'SABIO-RK EC Record', 'pattern': '^((\\\\d+)|(\\\\d+\\\\.\\\\d+)|(\\\\d+\\\\.\\\\d+\\\\.\\\\d+)|(\\\\d+\\\\.\\\\d+\\\\.\\\\d+\\\\.\\\\d+))$', 'description': 'SABIO-RK is a relational database system that contains information about biochemical reactions, their kinetic equations with their parameters, and the experimental conditions under which these parameters were measured. The EC record provides for a given enzyme classification (EC) the associated list of enzyme-catalysed reactions and their corresponding kinetic data.', 'created': '2019-06-11T14:16:07.366+0000', 'modified': '2019-06-11T14:16:07.366+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2.7.1.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/407'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/407'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/407/contactPerson'}}, 'metadata': []}, {'prefix': 'opb', 'mirId': 'MIR:00000129', 'name': 'Ontology of Physics for Biology', 'pattern': '^OPB_\\\\d+$', 'description': 'The OPB is a reference ontology of classical physics as applied to the dynamics of biological systems. It is designed to encompass the multiple structural scales (multiscale atoms to organisms) and multiple physical domains (multidomain fluid dynamics, chemical kinetics, particle diffusion, etc.) that are encountered in the study and analysis of biological organisms.', 'created': '2019-06-11T14:16:07.624+0000', 'modified': '2019-06-11T14:16:07.624+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'OPB_00573', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/410'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/410'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/410/contactPerson'}}, 'metadata': []}, {'prefix': 'jws', 'mirId': 'MIR:00000130', 'name': 'JWS Online', 'pattern': '^\\\\w+$', 'description': 'JWS Online is a repository of curated biochemical pathway models, and additionally provides the ability to run simulations of these models in a web browser.', 'created': '2019-06-11T14:16:07.820+0000', 'modified': '2019-06-11T14:16:07.820+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'curien', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/412'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/412'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/412/contactPerson'}}, 'metadata': []}, {'prefix': 'modeldb', 'mirId': 'MIR:00000131', 'name': 'ModelDB', 'pattern': '^\\\\d+$', 'description': 'ModelDB is a curated, searchable database of published models in the computational neuroscience domain. It accommodates models expressed in textual form, including procedural or declarative languages (e.g. C++, XML dialects) and source code written for any simulation environment.', 'created': '2019-06-11T14:16:08.429+0000', 'modified': '2019-06-11T14:16:08.429+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '45539', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/419'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/419'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/419/contactPerson'}}, 'metadata': []}, {'prefix': 'subtiwiki', 'mirId': 'MIR:00000132', 'name': 'SubtiWiki', 'pattern': '^BSU\\\\d{5}$', 'description': 'SubtiWiki is a scientific wiki for the model bacterium Bacillus subtilis. It provides comprehensive information on all genes and their proteins and RNA products, as well as information related to the current investigation of the gene/protein. \\r\\nNote: Currently, direct access to RNA products is restricted. This is expected to be rectified soon.', 'created': '2019-06-11T14:16:08.635+0000', 'modified': '2019-06-11T14:16:08.635+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BSU29180', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/421'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/421'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/421/contactPerson'}}, 'metadata': []}, {'prefix': 'pid.pathway', 'mirId': 'MIR:00000133', 'name': 'NCI Pathway Interaction Database: Pathway', 'pattern': '^\\\\w+$', 'description': 'The Pathway Interaction Database is a highly-structured, curated collection of information about known human biomolecular interactions and key cellular processes assembled into signaling pathways. This datatype provides access to pathway information.', 'created': '2019-06-11T14:16:08.872+0000', 'modified': '2019-06-11T14:16:08.872+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'pi3kcipathway', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/424'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/424'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/424/contactPerson'}}, 'metadata': []}, {'prefix': 'doqcs.model', 'mirId': 'MIR:00000134', 'name': 'Database of Quantitative Cellular Signaling: Model', 'pattern': '^\\\\d+$', 'description': 'The Database of Quantitative Cellular Signaling is a repository of models of signaling pathways. It includes reaction schemes, concentrations, rate constants, as well as annotations on the models. The database provides a range of search, navigation, and comparison functions. This datatype provides access to specific models.', 'created': '2019-06-11T14:16:09.111+0000', 'modified': '2019-06-11T14:16:09.111+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '57', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/427'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/427'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/427/contactPerson'}}, 'metadata': []}, {'prefix': 'doqcs.pathway', 'mirId': 'MIR:00000135', 'name': 'Database of Quantitative Cellular Signaling: Pathway', 'pattern': '^\\\\d+$', 'description': 'The Database of Quantitative Cellular Signaling is a repository of models of signaling pathways. It includes reaction schemes, concentrations, rate constants, as well as annotations on the models. The database provides a range of search, navigation, and comparison functions. This datatype provides access to pathways.', 'created': '2019-06-11T14:16:09.372+0000', 'modified': '2019-06-11T14:16:09.372+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '131', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/430'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/430'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/430/contactPerson'}}, 'metadata': []}, {'prefix': 'uo', 'mirId': 'MIR:00000136', 'name': 'Unit Ontology', 'pattern': '^UO:\\\\d{7}?', 'description': 'Ontology of standardized units', 'created': '2019-06-11T14:16:09.596+0000', 'modified': '2019-06-11T14:16:09.596+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000080', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/432'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/432'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/432/contactPerson'}}, 'metadata': []}, {'prefix': 'clinicaltrials', 'mirId': 'MIR:00000137', 'name': 'ClinicalTrials.gov', 'pattern': '^NCT\\\\d{8}$', 'description': 'ClinicalTrials.gov provides free access to information on clinical studies for a wide range of diseases and conditions. Studies listed in the database are conducted in 175 countries', 'created': '2019-06-11T14:16:09.937+0000', 'modified': '2019-06-11T14:16:09.937+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'NCT00222573', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/435'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/435'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/435/contactPerson'}}, 'metadata': []}, {'prefix': 'chemspider', 'mirId': 'MIR:00000138', 'name': 'ChemSpider', 'pattern': '^\\\\d+$', 'description': 'ChemSpider is a collection of compound data from across the web, which aggregates chemical structures and their associated information into a single searchable repository entry. These entries are supplemented with additional properties, related information and links back to original data sources.', 'created': '2019-06-11T14:16:10.254+0000', 'modified': '2019-06-11T14:16:10.254+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '56586', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/438'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/438'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/438/contactPerson'}}, 'metadata': []}, {'prefix': 'ncit', 'mirId': 'MIR:00000139', 'name': 'NCIt', 'pattern': '^C\\\\d+$', 'description': 'NCI Thesaurus (NCIt) provides reference terminology covering vocabulary for clinical care, translational and basic research, and public information and administrative activities, providing a stable and unique identification code.', 'created': '2019-06-11T14:16:10.500+0000', 'modified': '2019-06-11T14:16:10.500+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C80519', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/441'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/441'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/441/contactPerson'}}, 'metadata': []}, {'prefix': 'biocatalogue.service', 'mirId': 'MIR:00000140', 'name': 'BioCatalogue', 'pattern': '^\\\\d+$', 'description': 'The BioCatalogue provides a common interface for registering, browsing and annotating Web Services to the Life Science community. Registered services are monitored, allowing the identification of service problems and changes and the filtering-out of unavailable or unreliable resources. BioCatalogue is free to use, for all.', 'created': '2019-06-11T14:16:10.872+0000', 'modified': '2019-06-11T14:16:10.872+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '614', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/445'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/445'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/445/contactPerson'}}, 'metadata': []}, {'prefix': 'pr', 'mirId': 'MIR:00000141', 'name': 'Protein Ontology', 'pattern': '^PR\\\\:P?\\\\d+$', 'description': 'The PRotein Ontology (PRO) has been designed to describe the relationships of proteins and protein evolutionary classes, to delineate the multiple protein forms of a gene locus (ontology for protein forms), and to interconnect existing ontologies.', 'created': '2019-06-11T14:16:11.083+0000', 'modified': '2019-06-11T14:16:11.083+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '000000024', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/447'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/447'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/447/contactPerson'}}, 'metadata': []}, {'prefix': 'omia', 'mirId': 'MIR:00000142', 'name': 'OMIA', 'pattern': '^\\\\d+$', 'description': 'Online Mendelian Inheritance in Animals is a a database of genes, inherited disorders and traits in animal species (other than human and mouse).', 'created': '2019-06-11T14:16:11.523+0000', 'modified': '2019-06-11T14:16:11.523+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/451'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/451'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/451/contactPerson'}}, 'metadata': []}, {'prefix': 'csa', 'mirId': 'MIR:00000144', 'name': 'CSA', 'pattern': '^[0-9][A-Za-z0-9]{3}$', 'description': 'The Catalytic Site Atlas (CSA) is a database documenting enzyme active sites and catalytic residues in enzymes of 3D structure. It uses a defined classification for catalytic residues which includes only those residues thought to be directly involved in some aspect of the reaction catalysed by an enzyme.', 'created': '2019-06-11T14:16:11.769+0000', 'modified': '2019-06-11T14:16:11.769+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1a05', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/454'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/454'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/454/contactPerson'}}, 'metadata': []}, {'prefix': 'cgd', 'mirId': 'MIR:00000145', 'name': 'Candida Genome Database', 'pattern': '^CAL\\\\d{7}$', 'description': 'The Candida Genome Database (CGD) provides access to genomic sequence data and manually curated functional information about genes and proteins of the human pathogen Candida albicans. It collects gene names and aliases, and assigns gene ontology terms to describe the molecular function, biological process, and subcellular localization of gene products.', 'created': '2019-06-11T14:16:11.955+0000', 'modified': '2019-06-11T14:16:11.955+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CAL0003079', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/456'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/456'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/456/contactPerson'}}, 'metadata': []}, {'prefix': 'antweb', 'mirId': 'MIR:00000146', 'name': 'AntWeb', 'pattern': '^casent\\\\d+(\\\\-D\\\\d+)?$', 'description': 'AntWeb is a website documenting the known species of ants, with records for each species linked to their geographical distribution, life history, and includes pictures.', 'created': '2019-06-11T14:16:12.204+0000', 'modified': '2019-06-11T14:16:12.204+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'casent0106247', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/459'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/459'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/459/contactPerson'}}, 'metadata': []}, {'prefix': 'pmc', 'mirId': 'MIR:00000147', 'name': 'PMC International', 'pattern': 'PMC\\\\d+', 'description': \"PMC International (PMCI) is a free full-text archive of biomedical and life sciences journal literature. PMCI is a collaborative effort between the U.S. National Institutes of Health and the National Library of Medicine, the publishers whose journal content makes up the PMC archive, and organizations in other countries that share NIH's and NLM's interest in archiving life sciences literature.\", 'created': '2019-06-11T14:16:12.446+0000', 'modified': '2019-06-11T14:16:12.446+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PMC3084216', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/462'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/462'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/462/contactPerson'}}, 'metadata': []}, {'prefix': 'atcc', 'mirId': 'MIR:00000284', 'name': 'ATCC', 'pattern': '^\\\\d+$', 'description': 'The American Type Culture Collection (ATCC) is a private, nonprofit biological resource center whose mission focuses on the acquisition, authentication, production, preservation, development and distribution of standard reference microorganisms, cell lines and other materials for research in the life sciences.', 'created': '2019-06-11T14:16:44.811+0000', 'modified': '2019-06-11T14:16:44.811+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '11303', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/834'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/834'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/834/contactPerson'}}, 'metadata': []}, {'prefix': 'amoebadb', 'mirId': 'MIR:00000148', 'name': 'AmoebaDB', 'pattern': '^EDI_\\\\d+$', 'description': 'AmoebaDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:12.793+0000', 'modified': '2019-06-11T14:16:12.793+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EDI_244000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/466'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/466'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/466/contactPerson'}}, 'metadata': []}, {'prefix': 'cryptodb', 'mirId': 'MIR:00000149', 'name': 'CryptoDB', 'pattern': '^\\\\w+', 'description': 'CryptoDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:13.024+0000', 'modified': '2019-06-11T14:16:13.024+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'cgd7_230', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/469'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/469'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/469/contactPerson'}}, 'metadata': []}, {'prefix': 'plasmodb', 'mirId': 'MIR:00000150', 'name': 'PlasmoDB', 'pattern': '^\\\\w+$', 'description': 'AmoebaDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:13.221+0000', 'modified': '2019-06-11T14:16:13.221+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PF11_0344', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/471'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/471'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/471/contactPerson'}}, 'metadata': []}, {'prefix': 'giardiadb', 'mirId': 'MIR:00000151', 'name': 'GiardiaDB', 'pattern': '^\\\\w+$', 'description': 'GiardiaDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:13.424+0000', 'modified': '2019-06-11T14:16:13.424+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GL50803_102438', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/473'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/473'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/473/contactPerson'}}, 'metadata': []}, {'prefix': 'microsporidia', 'mirId': 'MIR:00000152', 'name': 'MicrosporidiaDB', 'pattern': '^\\\\w+$', 'description': 'MicrosporidiaDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:13.630+0000', 'modified': '2019-06-11T14:16:13.630+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ECU03_0820i', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/475'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/475'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/475/contactPerson'}}, 'metadata': []}, {'prefix': 'toxoplasma', 'mirId': 'MIR:00000153', 'name': 'ToxoDB', 'pattern': '^\\\\w+$', 'description': 'ToxoDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:13.817+0000', 'modified': '2019-06-11T14:16:13.817+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TGME49_053730', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/477'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/477'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/477/contactPerson'}}, 'metadata': []}, {'prefix': 'trichdb', 'mirId': 'MIR:00000154', 'name': 'TrichDB', 'pattern': '^\\\\w+$', 'description': 'TrichDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:14.019+0000', 'modified': '2019-06-11T14:16:14.019+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TVAG_386080', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/479'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/479'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/479/contactPerson'}}, 'metadata': []}, {'prefix': 'tritrypdb', 'mirId': 'MIR:00000155', 'name': 'TriTrypDB', 'pattern': '^\\\\w+(\\\\.)?\\\\w+(\\\\.)?\\\\w+', 'description': 'TriTrypDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:14.213+0000', 'modified': '2019-06-11T14:16:14.213+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Tb927.8.620', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/481'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/481'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/481/contactPerson'}}, 'metadata': []}, {'prefix': 'bdgp.insertion', 'mirId': 'MIR:00000156', 'name': 'BDGP insertion DB', 'pattern': '^\\\\w+$', 'description': 'BDGP gene disruption collection provides a public resource of gene disruptions of Drosophila genes using a single transposable element.', 'created': '2019-06-11T14:16:14.437+0000', 'modified': '2019-06-11T14:16:14.437+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'KG09531', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/484'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/484'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/484/contactPerson'}}, 'metadata': []}, {'prefix': 'beetlebase', 'mirId': 'MIR:00000157', 'name': 'BeetleBase', 'pattern': '^TC\\\\d+$', 'description': 'BeetleBase is a comprehensive sequence database and community resource for Tribolium genetics, genomics and developmental biology. It incorporates information about genes, mutants, genetic markers, expressed sequence tags and publications.', 'created': '2019-06-11T14:16:14.695+0000', 'modified': '2019-06-11T14:16:14.695+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TC010103', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/487'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/487'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/487/contactPerson'}}, 'metadata': []}, {'prefix': 'bold.taxonomy', 'mirId': 'MIR:00000158', 'name': 'BOLD Taxonomy', 'pattern': '^\\\\d+$', 'description': 'The Barcode of Life Data System (BOLD) is an informatics workbench aiding the acquisition, storage, analysis and publication of DNA barcode records. The associated taxonomy browser shows the progress of DNA barcoding and provides sample collection site distribution, and taxon occurence information.', 'created': '2019-06-11T14:16:14.938+0000', 'modified': '2019-06-11T14:16:14.938+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '27267', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/490'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/490'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/490/contactPerson'}}, 'metadata': []}, {'prefix': 'dbest', 'mirId': 'MIR:00000159', 'name': 'dbEST', 'pattern': '^([A-Z]+)?\\\\d+(\\\\.\\\\d+)?$', 'description': 'The dbEST contains sequence data and other information on \"single-pass\" cDNA sequences, or \"Expressed Sequence Tags\", from a number of organisms.', 'created': '2019-06-11T14:16:15.166+0000', 'modified': '2019-06-11T14:16:15.166+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BP100000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/493'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/493'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/493/contactPerson'}}, 'metadata': []}, {'prefix': 'dbprobe', 'mirId': 'MIR:00000160', 'name': 'dbProbe', 'pattern': '^\\\\d+$', 'description': 'The NCBI Probe Database is a public registry of nucleic acid reagents designed for use in a wide variety of biomedical research applications, together with information on reagent distributors, probe effectiveness, and computed sequence similarities.', 'created': '2019-06-11T14:16:15.607+0000', 'modified': '2019-06-11T14:16:15.607+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1000000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/497'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/497'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/497/contactPerson'}}, 'metadata': []}, {'prefix': 'dbsnp', 'mirId': 'MIR:00000161', 'name': 'dbSNP', 'pattern': '^rs\\\\d+$', 'description': 'The dbSNP database is a repository for both single base nucleotide subsitutions and short deletion and insertion polymorphisms.', 'created': '2019-06-11T14:16:15.831+0000', 'modified': '2019-06-11T14:16:15.831+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'rs121909098', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/499'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/499'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/499/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'rs121909098', 'description': 'SNV-Single Nucleotide Variation', 'version': 'Build 153', 'license': 'Open Access', 'distribution': {'@type': 'DataDownload', 'encodingFormat': 'JSON', 'contentUrl': 'https://api.ncbi.nlm.nih.gov/variation/v0/beta/refsnp/121909098'}, 'about': {'@type': 'Thing', 'description': 'Alleles: C>T'}, 'sourceOrganization': 'dbSNP', 'datePublished': 'July 9, 2019'}]}, {'prefix': 'unists', 'mirId': 'MIR:00000162', 'name': 'UniSTS', 'pattern': '^\\\\d+$', 'description': 'UniSTS is a comprehensive database of sequence tagged sites (STSs) derived from STS-based maps and other experiments. STSs are defined by PCR primer pairs and are associated with additional information such as genomic position, genes, and sequences.', 'created': '2019-06-11T14:16:16.225+0000', 'modified': '2019-06-11T14:16:16.225+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '456789', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/503'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/503'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/503/contactPerson'}}, 'metadata': []}, {'prefix': 'ecogene', 'mirId': 'MIR:00000163', 'name': 'EcoGene', 'pattern': '^EG\\\\d+$', 'description': 'The EcoGene database contains updated information about the E. coli K-12 genome and proteome sequences, including extensive gene bibliographies. A major EcoGene focus has been the re-evaluation of translation start sites.', 'created': '2019-06-11T14:16:16.429+0000', 'modified': '2019-06-11T14:16:16.429+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EG10173', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/505'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/505'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/505/contactPerson'}}, 'metadata': []}, {'prefix': 'bdgp.est', 'mirId': 'MIR:00000285', 'name': 'BDGP EST', 'pattern': '^\\\\w+(\\\\.)?(\\\\d+)?$', 'description': 'The BDGP EST database collects the expressed sequence tags (ESTs) derived from a variety of tissues and developmental stages for Drosophila melanogaster. All BDGP ESTs are available at dbEST (NCBI).', 'created': '2019-06-11T14:16:45.049+0000', 'modified': '2019-06-11T14:16:45.049+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EY223054.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/837'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/837'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/837/contactPerson'}}, 'metadata': []}, {'prefix': 'gabi', 'mirId': 'MIR:00000164', 'name': 'GABI', 'pattern': '^\\\\w+$', 'description': \"GabiPD (Genome Analysis of Plant Biological Systems Primary Database) constitutes a repository for a wide array of heterogeneous data from high-throughput experiments in several plant species. These data (i.e. genomics, transcriptomics, proteomics and metabolomics), originating from different model or crop species, can be accessed through a central gene 'Green Card'.\", 'created': '2019-06-11T14:16:16.652+0000', 'modified': '2019-06-11T14:16:16.652+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2679240', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/508'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/508'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/508/contactPerson'}}, 'metadata': []}, {'prefix': 'greengenes', 'mirId': 'MIR:00000165', 'name': 'GreenGenes', 'pattern': '^\\\\d+$', 'description': 'A 16S rRNA gene database which provides chimera screening, standard alignment, and taxonomic classification using multiple published taxonomies.', 'created': '2019-06-11T14:16:16.885+0000', 'modified': '2019-06-11T14:16:16.885+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '100000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/511'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/511'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/511/contactPerson'}}, 'metadata': []}, {'prefix': 'grin.taxonomy', 'mirId': 'MIR:00000166', 'name': 'GRIN Plant Taxonomy', 'pattern': '^\\\\d+$', 'description': 'GRIN (Germplasm Resources Information Network) Taxonomy for Plants provides information on scientific and common names, classification, distribution, references, and economic impact.', 'created': '2019-06-11T14:16:17.150+0000', 'modified': '2019-06-11T14:16:17.150+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '19333', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/514'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/514'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/514/contactPerson'}}, 'metadata': []}, {'prefix': 'hinv.locus', 'mirId': 'MIR:00000167', 'name': 'H-InvDb Locus', 'pattern': '^HIX\\\\d{7}(\\\\.\\\\d+)?$', 'description': \"H-Invitational Database (H-InvDB) is an integrated database of human genes and transcripts. It provides curated annotations of human genes and transcripts including gene structures, alternative splicing isoforms, non-coding functional RNAs, protein functions, functional domains, sub-cellular localizations, metabolic pathways, protein 3D structure, genetic polymorphisms (SNPs, indels and microsatellite repeats), relation with diseases, gene expression profiling, molecular evolutionary features, protein-protein interactions (PPIs) and gene families/groups. This datatype provides access to the 'Locus' view.\", 'created': '2019-06-11T14:16:17.382+0000', 'modified': '2019-06-11T14:16:17.382+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HIX0004394', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/517'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/517'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/517/contactPerson'}}, 'metadata': []}, {'prefix': 'hinv.transcript', 'mirId': 'MIR:00000168', 'name': 'H-InvDb Transcript', 'pattern': '^HIT\\\\d{9}(\\\\.\\\\d+)?$', 'description': \"H-Invitational Database (H-InvDB) is an integrated database of human genes and transcripts. It provides curated annotations of human genes and transcripts including gene structures, alternative splicing isoforms, non-coding functional RNAs, protein functions, functional domains, sub-cellular localizations, metabolic pathways, protein 3D structure, genetic polymorphisms (SNPs, indels and microsatellite repeats), relation with diseases, gene expression profiling, molecular evolutionary features, protein-protein interactions (PPIs) and gene families/groups. This datatype provides access to the 'Transcript' view.\", 'created': '2019-06-11T14:16:17.622+0000', 'modified': '2019-06-11T14:16:17.622+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HIT000195363', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/520'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/520'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/520/contactPerson'}}, 'metadata': []}, {'prefix': 'hinv.protein', 'mirId': 'MIR:00000169', 'name': 'H-InvDb Protein', 'pattern': '^HIP\\\\d{9}(\\\\.\\\\d+)?$', 'description': \"H-Invitational Database (H-InvDB) is an integrated database of human genes and transcripts. It provides curated annotations of human genes and transcripts including gene structures, alternative splicing isoforms, non-coding functional RNAs, protein functions, functional domains, sub-cellular localizations, metabolic pathways, protein 3D structure, genetic polymorphisms (SNPs, indels and microsatellite repeats), relation with diseases, gene expression profiling, molecular evolutionary features, protein-protein interactions (PPIs) and gene families/groups. This datatype provides access to the 'Protein' view.\", 'created': '2019-06-11T14:16:17.817+0000', 'modified': '2019-06-11T14:16:17.817+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HIP000030660', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/522'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/522'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/522/contactPerson'}}, 'metadata': []}, {'prefix': 'homd.seq', 'mirId': 'MIR:00000170', 'name': 'HOMD Sequence Metainformation', 'pattern': '^SEQF\\\\d+$', 'description': 'The Human Oral Microbiome Database (HOMD) provides a site-specific comprehensive database for the more than 600 prokaryote species that are present in the human oral cavity. It contains genomic information based on a curated 16S rRNA gene-based provisional naming scheme, and taxonomic information. This datatype contains genomic sequence information.', 'created': '2019-06-11T14:16:18.019+0000', 'modified': '2019-06-11T14:16:18.019+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SEQF1003', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/524'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/524'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/524/contactPerson'}}, 'metadata': []}, {'prefix': 'homd.taxon', 'mirId': 'MIR:00000171', 'name': 'HOMD Taxonomy', 'pattern': '^\\\\d+$', 'description': 'The Human Oral Microbiome Database (HOMD) provides a site-specific comprehensive database for the more than 600 prokaryote species that are present in the human oral cavity. It contains genomic information based on a curated 16S rRNA gene-based provisional naming scheme, and taxonomic information. This datatype contains taxonomic information.', 'created': '2019-06-11T14:16:18.268+0000', 'modified': '2019-06-11T14:16:18.268+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '811', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/527'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/527'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/527/contactPerson'}}, 'metadata': []}, {'prefix': 'ird.segment', 'mirId': 'MIR:00000172', 'name': 'IRD Segment Sequence', 'pattern': '^\\\\w+(\\\\_)?\\\\d+(\\\\.\\\\d+)?$', 'description': 'Influenza Research Database (IRD) contains information related to influenza virus, including genomic sequence, strain, protein, epitope  and bibliographic information. The Segment Details page contains descriptive information and annotation data about a particular genomic segment and its encoded product(s).', 'created': '2019-06-11T14:16:18.461+0000', 'modified': '2019-06-11T14:16:18.461+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CY077097', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/529'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/529'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/529/contactPerson'}}, 'metadata': []}, {'prefix': 'isfinder', 'mirId': 'MIR:00000173', 'name': 'ISFinder', 'pattern': '^IS\\\\w+(\\\\-\\\\d)?$', 'description': 'ISfinder is a database of bacterial insertion sequences (IS). It assigns IS nomenclature and acts as a repository for ISs. Each IS is annotated with information such as the open reading frame DNA sequence, the sequence of the ends of the element and target sites, its origin and distribution together with a bibliography, where available.', 'created': '2019-06-11T14:16:18.690+0000', 'modified': '2019-06-11T14:16:18.690+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ISA1083-2', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/532'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/532'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/532/contactPerson'}}, 'metadata': []}, {'prefix': 'jcm', 'mirId': 'MIR:00000174', 'name': 'Japan Collection of Microorganisms', 'pattern': '^\\\\d+$', 'description': 'The Japan Collection of Microorganisms (JCM) collects, catalogues, and distributes cultured microbial strains, restricted to those classified in Risk Group 1 or 2.', 'created': '2019-06-11T14:16:18.921+0000', 'modified': '2019-06-11T14:16:18.921+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '17254', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/535'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/535'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/535/contactPerson'}}, 'metadata': []}, {'prefix': 'img.taxon', 'mirId': 'MIR:00000175', 'name': 'Integrated Microbial Genomes Taxon', 'pattern': '^\\\\d+$', 'description': 'The integrated microbial genomes (IMG) system is a data management, analysis and annotation platform for all publicly available genomes. IMG contains both draft and complete JGI (DoE Joint Genome Institute) microbial genomes integrated with all other publicly available genomes from all three domains of life, together with a large number of plasmids and viruses. This datatype refers to taxon information.', 'created': '2019-06-11T14:16:19.173+0000', 'modified': '2019-06-11T14:16:19.173+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '648028003', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/538'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/538'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/538/contactPerson'}}, 'metadata': []}, {'prefix': 'img.gene', 'mirId': 'MIR:00000176', 'name': 'Integrated Microbial Genomes Gene', 'pattern': '^\\\\d+$', 'description': 'The integrated microbial genomes (IMG) system is a data management, analysis and annotation platform for all publicly available genomes. IMG contains both draft and complete JGI (DoE Joint Genome Institute) microbial genomes integrated with all other publicly available genomes from all three domains of life, together with a large number of plasmids and viruses. This datatype refers to gene information.', 'created': '2019-06-11T14:16:19.420+0000', 'modified': '2019-06-11T14:16:19.420+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '638309541', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/541'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/541'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/541/contactPerson'}}, 'metadata': []}, {'prefix': 'maizegdb.locus', 'mirId': 'MIR:00000177', 'name': 'MaizeGDB Locus', 'pattern': '^\\\\d+$', 'description': \"MaizeGDB is the maize research community's central repository for genetics and genomics information.\", 'created': '2019-06-11T14:16:19.610+0000', 'modified': '2019-06-11T14:16:19.610+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '25011', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/543'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/543'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/543/contactPerson'}}, 'metadata': []}, {'prefix': 'mycobank', 'mirId': 'MIR:00000178', 'name': 'MycoBank', 'pattern': '^\\\\d+$', 'description': 'MycoBank is an online database, documenting new mycological names and combinations, eventually combined with descriptions and illustrations.', 'created': '2019-06-11T14:16:19.843+0000', 'modified': '2019-06-11T14:16:19.843+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '349124', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/546'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/546'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/546/contactPerson'}}, 'metadata': []}, {'prefix': 'nbrc', 'mirId': 'MIR:00000179', 'name': 'NITE Biological Research Center Catalogue', 'pattern': '^\\\\d+$', 'description': 'NITE Biological Research Center (NBRC) provides a collection of microbial resources, performing taxonomic characterization of individual microorganisms such as bacteria including actinomycetes and archaea, yeasts, fungi, algaes, bacteriophages and DNA resources for academic research and industrial applications. A catalogue is maintained which states strain nomenclature, synonyms, and culture and sequence information.', 'created': '2019-06-11T14:16:20.103+0000', 'modified': '2019-06-11T14:16:20.103+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '00001234', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/549'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/549'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/549/contactPerson'}}, 'metadata': []}, {'prefix': 'pseudomonas', 'mirId': 'MIR:00000180', 'name': 'Pseudomonas Genome Database', 'pattern': '^P\\\\w+$', 'description': 'The Pseudomonas Genome Database is a resource for peer-reviewed, continually updated annotation for all Pseudomonas species. It includes gene and protein sequence information, as well as regulation and predicted function and annotation.', 'created': '2019-06-11T14:16:20.362+0000', 'modified': '2019-06-11T14:16:20.362+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PSEEN0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/552'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/552'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/552/contactPerson'}}, 'metadata': []}, {'prefix': 'gramene.protein', 'mirId': 'MIR:00000181', 'name': 'Gramene protein', 'pattern': '^\\\\d+$', 'description': 'Gramene is a comparative genome mapping database for grasses and crop plants. It combines a semi-automatically generated database of cereal genomic and expressed sequence tag sequences, genetic maps, map relations, quantitative trait loci (QTL), and publications, with a curated database of mutants (genes and alleles), molecular markers, and proteins. This datatype refers to proteins in Gramene.', 'created': '2019-06-11T14:16:20.589+0000', 'modified': '2019-06-11T14:16:20.589+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '78073', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/555'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/555'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/555/contactPerson'}}, 'metadata': []}, {'prefix': 'gramene.gene', 'mirId': 'MIR:00000182', 'name': 'Gramene genes', 'pattern': '^GR\\\\:\\\\d+$', 'description': 'Gramene is a comparative genome mapping database for grasses and crop plants. It combines a semi-automatically generated database of cereal genomic and expressed sequence tag sequences, genetic maps, map relations, quantitative trait loci (QTL), and publications, with a curated database of mutants (genes and alleles), molecular markers, and proteins. This datatype refers to genes in Gramene.', 'created': '2019-06-11T14:16:20.812+0000', 'modified': '2019-06-11T14:16:20.812+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GR:0080039', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/558'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/558'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/558/contactPerson'}}, 'metadata': []}, {'prefix': 'gramene.taxonomy', 'mirId': 'MIR:00000183', 'name': 'Gramene Taxonomy', 'pattern': '^GR\\\\_tax\\\\:\\\\d+$', 'description': 'Gramene is a comparative genome mapping database for grasses and crop plants. It combines a semi-automatically generated database of cereal genomic and expressed sequence tag sequences, genetic maps, map relations, quantitative trait loci (QTL), and publications, with a curated database of mutants (genes and alleles), molecular markers, and proteins. This datatype refers to taxonomic information in Gramene.', 'created': '2019-06-11T14:16:21.008+0000', 'modified': '2019-06-11T14:16:21.008+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GR_tax:013681', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/560'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/560'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/560/contactPerson'}}, 'metadata': []}, {'prefix': 'gramene.qtl', 'mirId': 'MIR:00000184', 'name': 'Gramene QTL', 'pattern': '^\\\\w+$', 'description': 'Gramene is a comparative genome mapping database for grasses and crop plants. It combines a semi-automatically generated database of cereal genomic and expressed sequence tag sequences, genetic maps, map relations, quantitative trait loci (QTL), and publications, with a curated database of mutants (genes and alleles), molecular markers, and proteins. This datatype refers to quantitative trait loci identified in Gramene.', 'created': '2019-06-11T14:16:21.219+0000', 'modified': '2019-06-11T14:16:21.219+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CQG5', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/562'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/562'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/562/contactPerson'}}, 'metadata': []}, {'prefix': 'sgn', 'mirId': 'MIR:00000185', 'name': 'Sol Genomics Network', 'pattern': '^\\\\d+$', 'description': 'The Sol Genomics Network (SGN) is a database and website dedicated to the genomic information of the nightshade family, which includes species such as tomato, potato, pepper, petunia and eggplant.', 'created': '2019-06-11T14:16:21.425+0000', 'modified': '2019-06-11T14:16:21.425+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/564'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/564'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/564/contactPerson'}}, 'metadata': []}, {'prefix': 'xenbase', 'mirId': 'MIR:00000186', 'name': 'Xenbase', 'pattern': '^(XB-GENE-)?\\\\d+$', 'description': 'Xenbase is the model organism database for Xenopus laevis and X. (Silurana) tropicalis. It contains genomic, development data and community information for Xenopus research. it includes gene expression patterns that incorporates image data from the literature, large scale screens and community submissions.', 'created': '2019-06-11T14:16:21.651+0000', 'modified': '2019-06-11T14:16:21.651+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '922462', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/567'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/567'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/567/contactPerson'}}, 'metadata': []}, {'prefix': 'bioportal', 'mirId': 'MIR:00000187', 'name': 'BioPortal', 'pattern': '^\\\\d+$', 'description': 'BioPortal is an open repository of biomedical ontologies that provides access via Web services and Web browsers to ontologies developed in OWL, RDF, OBO format and Prot√©g√© frames. BioPortal functionality includes the ability to browse, search and visualize ontologies.', 'created': '2019-06-11T14:16:21.882+0000', 'modified': '2019-06-11T14:16:21.882+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1046', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/570'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/570'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/570/contactPerson'}}, 'metadata': []}, {'prefix': 'miriam.resource', 'mirId': 'MIR:00000188', 'name': 'MIRIAM Registry resource', 'pattern': '^MIR:001\\\\d{5}$', 'description': 'MIRIAM Registry is an online resource created to catalogue data types (Gene Ontology, Taxonomy or PubMed are some examples), their URIs and the corresponding resources (or physical locations), whether these are controlled vocabularies or databases.', 'created': '2019-06-11T14:16:22.127+0000', 'modified': '2019-06-11T14:16:22.127+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MIR:00100005', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/573'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/573'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/573/contactPerson'}}, 'metadata': []}, {'prefix': 'edam', 'mirId': 'MIR:00000189', 'name': 'EDAM Ontology', 'pattern': '^(data|topic|operation|format)\\\\_\\\\d{4}$', 'description': 'EDAM is an ontology of general bioinformatics concepts, including topics, data types, formats, identifiers and operations. EDAM provides a controlled vocabulary for the description, in semantic terms, of things such as: web services (e.g. WSDL files), applications, tool collections and packages, work-benches and workflow software, databases and ontologies, XSD data schema and data objects, data syntax and file formats, web portals and pages, resource catalogues and documents (such as scientific publications).', 'created': '2019-06-11T14:16:22.318+0000', 'modified': '2019-06-11T14:16:22.318+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'data_1664', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/575'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/575'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/575/contactPerson'}}, 'metadata': []}, {'prefix': 'pmdb', 'mirId': 'MIR:00000190', 'name': 'Protein Model Database', 'pattern': '^PM\\\\d{7}', 'description': 'The Protein Model DataBase (PMDB), is a database that collects manually built three dimensional protein models, obtained by different structure prediction techniques.', 'created': '2019-06-11T14:16:22.644+0000', 'modified': '2019-06-11T14:16:22.644+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PM0012345', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/578'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/578'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/578/contactPerson'}}, 'metadata': []}, {'prefix': 'arachnoserver', 'mirId': 'MIR:00000193', 'name': 'ArachnoServer', 'pattern': '^AS\\\\d{6}$', 'description': 'ArachnoServer (www.arachnoserver.org) is a manually curated database providing information on the sequence, structure and biological activity of protein toxins from spider venoms. It include a molecular target ontology designed specifically for venom toxins, as well as current and historic taxonomic information.', 'created': '2019-06-11T14:16:22.902+0000', 'modified': '2019-06-11T14:16:22.902+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AS000060', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/581'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/581'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/581/contactPerson'}}, 'metadata': []}, {'prefix': 'biocyc', 'mirId': 'MIR:00000194', 'name': 'BioCyc', 'pattern': '^[A-Z-0-9]+(\\\\:)?[A-Za-z0-9+_.%-:]+$', 'description': 'BioCyc is a collection of Pathway/Genome Databases (PGDBs) which provides an electronic reference source on the genomes and metabolic pathways of sequenced organisms.', 'created': '2019-06-11T14:16:23.132+0000', 'modified': '2019-06-11T14:16:23.132+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ECOLI:CYT-D-UBIOX-CPLX', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/584'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/584'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/584/contactPerson'}}, 'metadata': []}, {'prefix': 'cazy', 'mirId': 'MIR:00000195', 'name': 'CAZy', 'pattern': '^(GT|GH|PL|CE|CBM)\\\\d+(\\\\_\\\\d+)?$', 'description': 'The Carbohydrate-Active Enzyme (CAZy) database is a resource specialized in enzymes that build and breakdown complex carbohydrates and glycoconjugates. These enzymes are classified into families based on structural features.', 'created': '2019-06-11T14:16:23.358+0000', 'modified': '2019-06-11T14:16:23.358+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GT10', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/587'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/587'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/587/contactPerson'}}, 'metadata': []}, {'prefix': 'goa', 'mirId': 'MIR:00000196', 'name': 'GOA', 'pattern': '^(([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9]))|(URS[0-9A-F]{10}(_[0-9]+){0,1})|(EBI-[0-9]+)$', 'description': 'The GOA (Gene Ontology Annotation) project provides high-quality Gene Ontology (GO) annotations to proteins in the UniProt Knowledgebase (UniProtKB) and International Protein Index (IPI). This involves electronic annotation and the integration of high-quality manual GO annotation from all GO Consortium model organism groups and specialist groups.', 'created': '2019-06-11T14:16:23.594+0000', 'modified': '2019-06-11T14:16:23.594+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P12345', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/590'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/590'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/590/contactPerson'}}, 'metadata': []}, {'prefix': 'paleodb', 'mirId': 'MIR:00000197', 'name': 'PaleoDB', 'pattern': '^\\\\d+$', 'description': 'The Paleobiology Database seeks to provide researchers and the public with information about the entire fossil record. It stores global, collection-based occurrence and taxonomic data for marine and terrestrial animals and plants of any geological age, as well as web-based software for statistical analysis of the data.', 'created': '2019-06-11T14:16:23.782+0000', 'modified': '2019-06-11T14:16:23.782+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '83088', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/592'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/592'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/592/contactPerson'}}, 'metadata': []}, {'prefix': 'compulyeast', 'mirId': 'MIR:00000198', 'name': 'Compulyeast', 'pattern': '^([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])$', 'description': 'Compluyeast-2D-DB is a two-dimensional polyacrylamide gel electrophoresis federated database. This collection references a subset of Uniprot, and contains general information about the protein record.', 'created': '2019-06-11T14:16:24.045+0000', 'modified': '2019-06-11T14:16:24.045+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'O08709', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/595'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/595'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/595/contactPerson'}}, 'metadata': []}, {'prefix': 'disprot', 'mirId': 'MIR:00000199', 'name': 'DisProt', 'pattern': '^DP\\\\d{5}$', 'description': 'The Database of Protein Disorder (DisProt) is a curated database that provides information about proteins that lack fixed 3D structure in their putatively native states, either in their entirety or in part.', 'created': '2019-06-11T14:16:24.278+0000', 'modified': '2019-06-11T14:16:24.278+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DP00003', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/598'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/598'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/598/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org/', '@type': 'DataCatalog', '@id': 'https://disprot.org', 'name': 'DisProt', 'description': 'Database of manually annotated intrisically disordered proteins', 'url': 'https://disprot.org', 'publication': {'@type': 'PublicationEvent', 'name': '', 'url': 'https://doi.org/10.1093/nar/gkz975', 'sameAs': ['https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz975/5622715']}, 'keywords': ['IDP', 'IDPs', 'DisProt', 'intrinsic protein disorder', 'protein annotation', 'protein disorder'], 'sourceOrganization': [{'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}], 'provider': [{'@type': 'Person', 'name': 'Silvio Tosatto', 'email': 'silvio.tosatto@unipd.it'}], 'datePublished': '2019-09', 'dateModified': '2019-09', 'encodingFormat': 'text/html', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000199', 'dataset': {'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'DisProt', 'description': 'Database of manually annotated intrisically disordered proteins', 'url': 'https://disprot.org', 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000199', 'keywords': ['IDP', 'IDPs', 'DisProt', 'intrinsic protein disorder', 'protein annotation', 'protein disorder'], 'includedInDataCatalog': 'http://disprot.org', 'creator': {'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}, 'version': '2019-09', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}}]}, {'prefix': 'echobase', 'mirId': 'MIR:00000200', 'name': 'EchoBASE', 'pattern': '^EB\\\\d+$', 'description': 'EchoBASE is a database designed to contain and manipulate information from post-genomic experiments using the model bacterium Escherichia coli K-12. The database is built on an enhanced annotation of the updated genome sequence of strain MG1655 and the association of experimental data with the E.coli genes and their products.', 'created': '2019-06-11T14:16:24.496+0000', 'modified': '2019-06-11T14:16:24.496+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EB0170', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/601'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/601'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/601/contactPerson'}}, 'metadata': []}, {'prefix': 'eggnog', 'mirId': 'MIR:00000201', 'name': 'eggNOG', 'pattern': '^\\\\w+$', 'description': 'eggNOG (evolutionary genealogy of genes: Non-supervised Orthologous Groups) is a database of orthologous groups of genes. The orthologous groups are annotated with functional description lines (derived by identifying a common denominator for the genes based on their various annotations), with functional categories (i.e derived from the original COG/KOG categories).', 'created': '2019-06-11T14:16:24.719+0000', 'modified': '2019-06-11T14:16:24.719+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'veNOG12876', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/604'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/604'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/604/contactPerson'}}, 'metadata': []}, {'prefix': 'ensembl.bacteria', 'mirId': 'MIR:00000202', 'name': 'Ensembl Bacteria', 'pattern': '^((EB\\\\w+)|([A-Z0-9]+\\\\_[A-Z0-9]+))$', 'description': 'Ensembl Genomes consists of five sub-portals (for bacteria, protists, fungi, plants and invertebrate metazoa) designed to complement the availability of vertebrate genomes in Ensembl. This collection is concerned with bacterial genomes.', 'created': '2019-06-11T14:16:24.944+0000', 'modified': '2019-06-11T14:16:24.944+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MU9_3181', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/607'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/607'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/607/contactPerson'}}, 'metadata': []}, {'prefix': 'ensembl.protist', 'mirId': 'MIR:00000203', 'name': 'Ensembl Protists', 'pattern': '^\\\\w+$', 'description': 'Ensembl Genomes consists of five sub-portals (for bacteria, protists, fungi, plants and invertebrate metazoa) designed to complement the availability of vertebrate genomes in Ensembl. This collection is concerned with protist genomes.', 'created': '2019-06-11T14:16:25.183+0000', 'modified': '2019-06-11T14:16:25.183+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PF3D7_1328700', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/609'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/609'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/609/contactPerson'}}, 'metadata': []}, {'prefix': 'ensembl.metazoa', 'mirId': 'MIR:00000204', 'name': 'Ensembl Metazoa', 'pattern': '^\\\\w+(\\\\.)?\\\\d+$', 'description': 'Ensembl Genomes consists of five sub-portals (for bacteria, protists, fungi, plants and invertebrate metazoa) designed to complement the availability of vertebrate genomes in Ensembl. This collection is concerned with metazoa genomes.', 'created': '2019-06-11T14:16:25.396+0000', 'modified': '2019-06-11T14:16:25.396+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'FBtr0084214', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/611'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/611'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/611/contactPerson'}}, 'metadata': []}, {'prefix': 'ensembl.plant', 'mirId': 'MIR:00000205', 'name': 'Ensembl Plants', 'pattern': '^\\\\w+(\\\\.\\\\d+)?(\\\\.\\\\d+)?$', 'description': 'Ensembl Genomes consists of five sub-portals (for bacteria, protists, fungi, plants and invertebrate metazoa) designed to complement the availability of vertebrate genomes in Ensembl. This collection is concerned with plant genomes.', 'created': '2019-06-11T14:16:25.597+0000', 'modified': '2019-06-11T14:16:25.597+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AT1G73965', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/613'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/613'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/613/contactPerson'}}, 'metadata': []}, {'prefix': 'ensembl.fungi', 'mirId': 'MIR:00000206', 'name': 'Ensembl Fungi', 'pattern': '^[A-Z-a-z0-9]+$', 'description': 'Ensembl Genomes consists of five sub-portals (for bacteria, protists, fungi, plants and invertebrate metazoa) designed to complement the availability of vertebrate genomes in Ensembl. This collection is concerned with fungal genomes.', 'created': '2019-06-11T14:16:25.804+0000', 'modified': '2019-06-11T14:16:25.804+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CADAFLAT00006211', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/615'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/615'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/615/contactPerson'}}, 'metadata': []}, {'prefix': 'hcvdb', 'mirId': 'MIR:00000207', 'name': 'HCVDB', 'pattern': '^M\\\\d{5}$', 'description': 'the European Hepatitis C Virus Database (euHCVdb, http://euhcvdb.ibcp.fr), a collection of computer-annotated sequences based on reference genomes.mainly dedicated to HCV protein sequences, 3D structures and functional analyses.', 'created': '2019-06-11T14:16:26.018+0000', 'modified': '2019-06-11T14:16:26.018+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'M58335', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/617'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/617'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/617/contactPerson'}}, 'metadata': []}, {'prefix': 'genatlas', 'mirId': 'MIR:00000208', 'name': 'Genatlas', 'pattern': '^\\\\w+$', 'description': 'GenAtlas is a database containing information on human genes, markers and phenotypes.', 'created': '2019-06-11T14:16:26.301+0000', 'modified': '2019-06-11T14:16:26.301+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HBB', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/620'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/620'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/620/contactPerson'}}, 'metadata': []}, {'prefix': 'cath.superfamily', 'mirId': 'MIR:00000209', 'name': 'CATH superfamily', 'pattern': '^\\\\d+(\\\\.\\\\d+(\\\\.\\\\d+(\\\\.\\\\d+)?)?)?$', 'description': 'The CATH database is a hierarchical domain classification of protein structures in the Protein Data Bank. Protein structures are classified using a combination of automated and manual procedures. There are four major levels in this hierarchy; Class (secondary structure classification, e.g. mostly alpha), Architecture (classification based on overall shape), Topology (fold family) and  Homologous superfamily (protein domains which are thought to share a common ancestor). This colelction is concerned with superfamily classification.', 'created': '2019-06-11T14:16:26.635+0000', 'modified': '2019-06-11T14:16:26.635+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1.10.10.200', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/623'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/623'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/623/contactPerson'}}, 'metadata': []}, {'prefix': 'cath.domain', 'mirId': 'MIR:00000210', 'name': 'CATH domain', 'pattern': '^\\\\w+$', 'description': 'The CATH database is a hierarchical domain classification of protein structures in the Protein Data Bank. Protein structures are classified using a combination of automated and manual procedures. There are four major levels in this hierarchy; Class (secondary structure classification, e.g. mostly alpha), Architecture (classification based on overall shape), Topology (fold family) and Homologous superfamily (protein domains which are thought to share a common ancestor). This colelction is concerned with CATH domains.', 'created': '2019-06-11T14:16:26.899+0000', 'modified': '2019-06-11T14:16:26.899+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1cukA01', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/626'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/626'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/626/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'http://www.cathdb.info//version/v4_2_0/domain/1cukA01', 'mainEntityOfPage': {'@type': ['DataRecord'], 'identifier': 'cath.domain:1cukA01', 'isPartOf': {'@type': 'Dataset', '@id': 'CATH release v4_2_0'}, 'mainEntity': {'@type': ['StructuredValue', 'BioChemEntity'], 'additionalType': 'http://semanticscience.org/resource/SIO_001379', 'identifier': 'cathdomain:1cukA01', 'name': 'CATH Domain - 1cukA01 (PDB: 1cuk, chain A, domain domain.domain_number', 'isContainedIn': [{'@type': ['StructuredValue', 'BioChemEntity'], 'additionalType': 'http://semanticscience.org/resource/SIO_001379', 'additionalProperty': {'@type': 'BioChemEntity', '@id': 'http://identifiers.org/cath.superfamily:1.10.8.10', 'identifier': '1.10.8.10', 'name': '\"winged helix\" repressor DNA binding domain'}}, {'@type': 'DataRecord', 'identifier': '1cuk', 'isPartOf': {'@type': 'Dataset', '@id': 'http://pdb.org'}, 'name': '1cuk', 'additionalType': 'http://semanticscience.org/resource/SIO_011119', 'additionalProperty': [{'@type': 'StructuredValue', 'additionalType': 'http://semanticscience.org/resource/SIO_010000', 'name': 'organism', 'value': 'http://identifiers.org/taxonomy:9606'}, {'@type': 'StructuredValue', 'name': 'method', 'value': 'xray crystallography'}, {'@type': 'StructuredValue', 'name': 'resolution', 'value': '2.2√É\\x85'}]}], 'additionalProperty': [{'@type': 'http://semanticscience.org/resource/SIO_001379', 'name': 'domain annotation', 'value': {'@type': ['StructuredValue', 'BioChemEntity'], 'additionalType': 'http://semanticscience.org/resource/SIO_001379', 'identifier': 'cath.domain:1cukA01', 'name': 'CATH Domain - 1cukA01 (PDB: 1cuk, chain A, domain domain.domain_number', 'location': [{'@type': 'PropertyValue', 'minValue': 2, 'maxValue': 123}, {'@type': 'PropertyValue', 'minValue': 234, 'maxValue': 431}]}}]}}}]}, {'prefix': 'genefarm', 'mirId': 'MIR:00000211', 'name': 'GeneFarm', 'pattern': '^\\\\d+$', 'description': 'GeneFarm is a database whose purpose is to store traceable annotations for Arabidopsis nuclear genes and gene products.', 'created': '2019-06-11T14:16:27.145+0000', 'modified': '2019-06-11T14:16:27.145+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4892', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/628'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/628'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/628/contactPerson'}}, 'metadata': []}, {'prefix': 'gpcrdb', 'mirId': 'MIR:00000212', 'name': 'GPCRDB', 'pattern': '^\\\\w+$', 'description': 'The G protein-coupled receptor database (GPCRDB) collects, large amounts of heterogeneous data on GPCRs. It contains experimental data on sequences, ligand-binding constants, mutations and oligomers, and derived data such as multiple sequence alignments and homology models.', 'created': '2019-06-11T14:16:27.419+0000', 'modified': '2019-06-11T14:16:27.419+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RL3R1_HUMAN', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/631'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/631'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/631/contactPerson'}}, 'metadata': []}, {'prefix': 'hogenom', 'mirId': 'MIR:00000213', 'name': 'HOGENOM', 'pattern': '^\\\\w+$', 'description': 'HOGENOM is a database of homologous genes from fully sequenced organisms (bacteria, archeae and eukarya). This collection references phylogenetic trees which can be retrieved using either UniProt accession numbers, or HOGENOM tree family identifier.', 'created': '2019-06-11T14:16:27.678+0000', 'modified': '2019-06-11T14:16:27.678+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HBG284870', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/634'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/634'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/634/contactPerson'}}, 'metadata': []}, {'prefix': 'genetree', 'mirId': 'MIR:00000214', 'name': 'GeneTree', 'pattern': '^ENSGT\\\\d+$', 'description': 'Genetree displays the maximum likelihood phylogenetic (protein) trees representing the evolutionary history of the genes. These are constructed using the canonical protein for every gene in Ensembl.', 'created': '2019-06-11T14:16:27.950+0000', 'modified': '2019-06-11T14:16:27.950+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ENSGT00550000074763', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/637'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/637'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/637/contactPerson'}}, 'metadata': []}, {'prefix': 'hssp', 'mirId': 'MIR:00000215', 'name': 'HSSP', 'pattern': '^\\\\w{4}$', 'description': 'HSSP (homology-derived structures of proteins) is a derived database merging structural (2-D and 3-D) and sequence information (1-D). For each protein of known 3D structure from the Protein Data Bank, the database has a file with all sequence homologues, properly aligned to the PDB protein.', 'created': '2019-06-11T14:16:28.184+0000', 'modified': '2019-06-11T14:16:28.184+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '102l', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/639'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/639'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/639/contactPerson'}}, 'metadata': []}, {'prefix': 'myco.tuber', 'mirId': 'MIR:00000216', 'name': 'MycoBrowser tuberculosis', 'pattern': '^Rv\\\\d{4}(A|B|c)?$', 'description': 'Mycobrowser is a resource that provides both in silico generated and manually reviewed information within databases dedicated to the complete genomes of Mycobacterium tuberculosis, Mycobacterium leprae, Mycobacterium marinum and Mycobacterium smegmatis. This collection references Mycobacteria tuberculosis information.', 'created': '2019-06-11T14:16:28.639+0000', 'modified': '2019-06-11T14:16:28.639+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Rv1908c', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/644'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/644'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/644/contactPerson'}}, 'metadata': []}, {'prefix': 'myco.lepra', 'mirId': 'MIR:00000217', 'name': 'MycoBrowser leprae', 'pattern': '^ML\\\\w+$', 'description': 'Mycobrowser is a resource that provides both in silico generated and manually reviewed information within databases dedicated to the complete genomes of Mycobacterium tuberculosis, Mycobacterium leprae, Mycobacterium marinum and Mycobacterium smegmatis. This collection references Mycobacteria leprae information.', 'created': '2019-06-11T14:16:28.887+0000', 'modified': '2019-06-11T14:16:28.887+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ML0224', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/647'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/647'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/647/contactPerson'}}, 'metadata': []}, {'prefix': 'myco.marinum', 'mirId': 'MIR:00000218', 'name': 'MycoBrowser marinum', 'pattern': '^MMAR\\\\_\\\\d+$', 'description': 'Mycobrowser is a resource that provides both in silico generated and manually reviewed information within databases dedicated to the complete genomes of Mycobacterium tuberculosis, Mycobacterium leprae, Mycobacterium marinum and Mycobacterium smegmatis. This collection references Mycobacteria marinum information.', 'created': '2019-06-11T14:16:29.087+0000', 'modified': '2019-06-11T14:16:29.087+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MMAR_2462', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/649'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/649'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/649/contactPerson'}}, 'metadata': []}, {'prefix': 'myco.smeg', 'mirId': 'MIR:00000219', 'name': 'MycoBrowser smegmatis', 'pattern': '^MSMEG\\\\w+$', 'description': 'Mycobrowser is a resource that provides both in silico generated and manually reviewed information within databases dedicated to the complete genomes of Mycobacterium tuberculosis, Mycobacterium leprae, Mycobacterium marinum and Mycobacterium smegmatis. This collection references Mycobacteria smegmatis information.', 'created': '2019-06-11T14:16:29.310+0000', 'modified': '2019-06-11T14:16:29.310+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MSMEG_3769', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/651'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/651'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/651/contactPerson'}}, 'metadata': []}, {'prefix': 'orphanet', 'mirId': 'MIR:00000220', 'name': 'Orphanet', 'pattern': '^\\\\d+$', 'description': 'Orphanet is a reference portal for information on rare diseases and orphan drugs. It‚Äôs aim is to help improve the diagnosis, care and treatment of patients with rare diseases.', 'created': '2019-06-11T14:16:29.544+0000', 'modified': '2019-06-11T14:16:29.544+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '85163', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/653'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/653'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/653/contactPerson'}}, 'metadata': []}, {'prefix': 'orthodb', 'mirId': 'MIR:00000221', 'name': 'OrthoDB', 'pattern': '^\\\\w+$', 'description': 'OrthoDB presents a catalog of eukaryotic orthologous protein-coding genes across vertebrates, arthropods, and fungi. Orthology refers to the last common ancestor of the species under consideration, and thus OrthoDB explicitly delineates orthologs at each radiation along the species phylogeny. The database of orthologs presents available protein descriptors, together with Gene Ontology and InterPro attributes, which serve to provide general descriptive annotations of the orthologous groups', 'created': '2019-06-11T14:16:29.860+0000', 'modified': '2019-06-11T14:16:29.860+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q9P0K8', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/657'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/657'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/657/contactPerson'}}, 'metadata': []}, {'prefix': 'peroxibase', 'mirId': 'MIR:00000222', 'name': 'Peroxibase', 'pattern': '^\\\\d+$', 'description': 'Peroxibase provides access to peroxidase sequences from all kingdoms of life, and provides a series of bioinformatics tools and facilities suitable for analysing these sequences.', 'created': '2019-06-11T14:16:30.097+0000', 'modified': '2019-06-11T14:16:30.097+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '5282', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/660'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/660'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/660/contactPerson'}}, 'metadata': []}, {'prefix': 'phylomedb', 'mirId': 'MIR:00000223', 'name': 'PhylomeDB', 'pattern': '^\\\\w+$', 'description': 'PhylomeDB is a database of complete phylomes derived for different genomes within a specific taxonomic range. It provides alignments, phylogentic trees and tree-based orthology predictions for all encoded proteins.', 'created': '2019-06-11T14:16:30.326+0000', 'modified': '2019-06-11T14:16:30.326+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Phy000CLXM_RAT', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/663'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/663'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/663/contactPerson'}}, 'metadata': []}, {'prefix': 'pmap.substratedb', 'mirId': 'MIR:00000224', 'name': 'SubstrateDB', 'pattern': '^\\\\d+$', 'description': 'The Proteolysis MAP is a resource for proteolytic networks and pathways. PMAP is comprised of five databases, linked together in one environment. SubstrateDB contains molecular information on documented protease substrates.', 'created': '2019-06-11T14:16:30.547+0000', 'modified': '2019-06-11T14:16:30.547+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1915', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/666'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/666'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/666/contactPerson'}}, 'metadata': []}, {'prefix': 'pmap.cutdb', 'mirId': 'MIR:00000225', 'name': 'CutDB', 'pattern': '^\\\\d+$', 'description': 'The Proteolysis MAP is a resource for proteolytic networks and pathways. PMAP is comprised of five databases, linked together in one environment. CutDB is a database of individual proteolytic events (cleavage sites).', 'created': '2019-06-11T14:16:30.793+0000', 'modified': '2019-06-11T14:16:30.793+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '25782', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/669'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/669'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/669/contactPerson'}}, 'metadata': []}, {'prefix': 'protclustdb', 'mirId': 'MIR:00000226', 'name': 'ProtClustDB', 'pattern': '^\\\\w+$', 'description': 'ProtClustDB is a collection of related protein sequences (clusters) consisting of Reference Sequence proteins encoded by complete genomes. This database contains both curated and non-curated clusters.', 'created': '2019-06-11T14:16:31.086+0000', 'modified': '2019-06-11T14:16:31.086+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'O80725', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/671'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/671'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/671/contactPerson'}}, 'metadata': []}, {'prefix': 'pmp', 'mirId': 'MIR:00000227', 'name': 'PMP', 'pattern': '^([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])$', 'description': 'The number of known protein sequences exceeds those of experimentally solved protein structures. Homology (or comparative) modeling methods make use of experimental protein structures to build models for evolutionary related proteins. The Protein Model Portal (PMP) provides a single portal to access these models, which are accessed through their UniProt identifiers.', 'created': '2019-06-11T14:16:31.282+0000', 'modified': '2019-06-11T14:16:31.282+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q0VCA6', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/673'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/673'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/673/contactPerson'}}, 'metadata': []}, {'prefix': 'protonet.proteincard', 'mirId': 'MIR:00000228', 'name': 'ProtoNet ProteinCard', 'pattern': '^\\\\d+$', 'description': 'ProtoNet provides automatic hierarchical classification of protein sequences in the UniProt database, partitioning the protein space into clusters of similar proteins. This collection references protein information.', 'created': '2019-06-11T14:16:31.529+0000', 'modified': '2019-06-11T14:16:31.529+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '16941567', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/676'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/676'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/676/contactPerson'}}, 'metadata': []}, {'prefix': 'protonet.cluster', 'mirId': 'MIR:00000229', 'name': 'ProtoNet Cluster', 'pattern': '^\\\\d+$', 'description': 'ProtoNet provides automatic hierarchical classification of protein sequences in the UniProt database, partitioning the protein space into clusters of similar proteins. This collection references cluster information.', 'created': '2019-06-11T14:16:31.786+0000', 'modified': '2019-06-11T14:16:31.786+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4349895', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/679'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/679'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/679/contactPerson'}}, 'metadata': []}, {'prefix': 'rebase', 'mirId': 'MIR:00000230', 'name': 'REBASE', 'pattern': '^\\\\d+$', 'description': 'REBASE is a comprehensive database of information about restriction enzymes, DNA methyltransferases and related proteins involved in the biological process of restriction-modification (R-M). It contains fully referenced information about recognition and cleavage sites, isoschizomers, neoschizomers, commercial availability, methylation sensitivity, crystal and sequence data.', 'created': '2019-06-11T14:16:32.004+0000', 'modified': '2019-06-11T14:16:32.004+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '101', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/681'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/681'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/681/contactPerson'}}, 'metadata': []}, {'prefix': 'swiss-model', 'mirId': 'MIR:00000231', 'name': 'SWISS-MODEL Repository', 'pattern': '^\\\\w+$', 'description': 'The SWISS-MODEL Repository is a database of 3D protein structure models generated by the SWISS-MODEL homology-modelling pipeline for UniProtKB protein sequences.', 'created': '2019-06-11T14:16:32.267+0000', 'modified': '2019-06-11T14:16:32.267+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P23298', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/684'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/684'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/684/contactPerson'}}, 'metadata': []}, {'prefix': 'vectorbase', 'mirId': 'MIR:00000232', 'name': 'VectorBase', 'pattern': '^\\\\D{4}\\\\d{6}(\\\\-\\\\D{2})?$', 'description': 'VectorBase is an NIAID-funded Bioinformatic Resource Center focused on invertebrate vectors of human pathogens. VectorBase annotates and curates vector genomes providing a web accessible integrated resource for the research community. Currently, VectorBase contains genome information for three mosquito species: Aedes aegypti, Anopheles gambiae and Culex quinquefasciatus, a body louse Pediculus humanus and a tick species Ixodes scapularis.', 'created': '2019-06-11T14:16:32.515+0000', 'modified': '2019-06-11T14:16:32.515+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ISCW007415', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/687'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/687'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/687/contactPerson'}}, 'metadata': []}, {'prefix': 'doid', 'mirId': 'MIR:00000233', 'name': 'Human Disease Ontology', 'pattern': '^DOID\\\\:\\\\d+$', 'description': 'The Disease Ontology has been developed as a standardized ontology for human disease with the purpose of providing the biomedical community with consistent, reusable and sustainable descriptions of human disease terms, phenotype characteristics and related medical vocabulary disease concepts.', 'created': '2019-06-11T14:16:32.719+0000', 'modified': '2019-06-11T14:16:32.719+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '11337', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/689'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/689'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/689/contactPerson'}}, 'metadata': []}, {'prefix': 'cco', 'mirId': 'MIR:00000234', 'name': 'Cell Cycle Ontology', 'pattern': '^CCO\\\\:\\\\w+$', 'description': 'The Cell Cycle Ontology is an application ontology that captures and integrates detailed knowledge on the cell cycle process.', 'created': '2019-06-11T14:16:33.213+0000', 'modified': '2019-06-11T14:16:33.213+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000003', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/694'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/694'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/694/contactPerson'}}, 'metadata': []}, {'prefix': 'mirbase.mature', 'mirId': 'MIR:00000235', 'name': 'miRBase mature sequence', 'pattern': 'MIMAT\\\\d{7}', 'description': 'The miRBase Sequence Database is a searchable database of published miRNA sequences and annotation. This collection refers specifically to the mature miRNA sequence.', 'created': '2019-06-11T14:16:33.415+0000', 'modified': '2019-06-11T14:16:33.415+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MIMAT0000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/696'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/696'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/696/contactPerson'}}, 'metadata': []}, {'prefix': 'nextprot', 'mirId': 'MIR:00000236', 'name': 'nextProt', 'pattern': '^NX_\\\\w+', 'description': 'neXtProt is a resource on human proteins, and includes information such as proteins‚Äô function, subcellular location, expression, interactions and role in diseases.', 'created': '2019-06-11T14:16:33.634+0000', 'modified': '2019-06-11T14:16:33.634+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'NX_O00165', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/698'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/698'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/698/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@graph': [{'@type': 'DataCatalog', '@id': 'http://identifiers.org/nextprot/', 'name': '{{nm.title}}', 'description': '{{nm.description}}', 'url': 'https://www.nextprot.org/', 'keywords': 'nextprot,UniProt,Human,Proteins,Proteome,Proteomics,protein database,protein knowledgebase,protein resource,human protein,human proteome,cancer variants portal,ion channels variants portal,function,medical,disease,expression,interactions,sequence,isoform,mutation,variant,phenotypes,proteomics,peptide,structure,3D,unicity checker,peptide uniqueness checker,annotation,biocuration,chromosomes,protein validation,protein-coding genes,post-translational modifications,ptm,data integration,systems biology,genetic variations,RDF data model,SPARQL endpoint,api,rest', 'thumbnailUrl': 'https://www.nextprot.org/img/np-x.png', 'sourceOrganization': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'The neXtProt knowledgebase on human proteins: 2017 update.', 'author': [{'@type': 'Person', 'name': 'Gaudet P'}, {'@type': 'Person', 'name': 'Michel PA'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Britan A'}, {'@type': 'Person', 'name': 'Cusin I'}, {'@type': 'Person', 'name': 'Domagalski M'}, {'@type': 'Person', 'name': 'Duek PD'}, {'@type': 'Person', 'name': 'Gateau A'}, {'@type': 'Person', 'name': 'Gleizes A'}, {'@type': 'Person', 'name': 'Hinard V'}, {'@type': 'Person', 'name': 'Rech de Laval V'}, {'@type': 'Person', 'name': 'Lin JJ'}, {'@type': 'Person', 'name': 'Nikitin F'}, {'@type': 'Person', 'name': 'Schaeffer M'}, {'@type': 'Person', 'name': 'Teixeira D'}, {'@type': 'Person', 'name': 'Lane L'}, {'@type': 'Person', 'name': 'Bairoch A'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Nucleic acids research'}, {'@type': 'PublicationVolume', 'volumeNumber': '45'}, {'@type': 'PublicationIssue', 'datePublished': '2017', 'issueNumber': 'D1'}], 'sameAs': ['https://doi.org/10.1093/nar/gkw1062', 'http://europepmc.org/abstract/MED/27899619', 'https://www.ncbi.nlm.nih.gov/pubmed/27899619', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210547']}, 'dataset': [{'@type': 'Dataset', 'name': 'neXtProt entries', 'description': 'The collection of neXtProt entries for human proteins', 'url': 'https://www.nextprot.org', 'keywords': 'nextprot,Human,Proteins,Proteome,Proteomics,protein database,protein knowledgebase,protein resource,human protein,human proteome,function,medical,disease,expression,interactions,sequence,isoform,mutation,variant,phenotypes,proteomics,peptide,structure,3D,annotation,biocuration,chromosomes,protein validation,protein-coding genes,post-translational modifications,ptm,data integration,systems biology,genetic variations,UniProt', 'distribution': [{'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/xml/nextprot_all.xml.gz', 'fileFormat': 'XML'}, {'@type': 'DataDownload', 'fileFormat': 'RDF'}, {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/peff/nextprot_all.peff.gz', 'fileFormat': 'PEFF'}, {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/md5/nextprot_sequence_md5.txt', 'fileFormat': 'TXT'}, {'@type': 'DataDownload', 'contentUrl': 'https://api.nextprot.org/export/entries/all.fasta', 'fileFormat': 'FASTA'}], 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.nextprot.org/proteins/search?query={query}', 'query-input': 'required name=query'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}, {'@type': 'Dataset', 'name': 'Caloha', 'description': 'An ontology of human anatomy and human cell types.', 'url': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/caloha.obo', 'keywords': 'human body, anatomy, ontology, tissues, cell types, organs, developmental stage', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/caloha.obo', 'fileFormat': 'OBO'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}, {'@type': 'Dataset', 'name': 'ICEPO', 'description': 'An ontology that allows one to annotate the electrophysiological parameters of the voltage-gated class of ion channels.', 'url': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/icepo.obo', 'keywords': 'ion channels, electrophysiology, ion channel gating, conductance, ion selectivity, activation, inactivation, recovery, deactivation, reopening, closed-state inactivation', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'ftp://ftp.nextprot.org/pub/current_release/controlled_vocabularies/icepo.obo', 'fileFormat': 'OBO'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'ICEPO: the ion channel electrophysiology ontology.', 'author': [{'@type': 'Person', 'name': 'Hinard V'}, {'@type': 'Person', 'name': 'Britan A'}, {'@type': 'Person', 'name': 'Rougier JS'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Abriel H'}, {'@type': 'Person', 'name': 'Gaudet P'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Database (Oxford)'}, {'@type': 'PublicationVolume', 'volumeNumber': '2016'}, {'@type': 'PublicationIssue', 'datePublished': '2016'}], 'sameAs': ['https://doi.org/10.1093/database/baw017', 'http://europepmc.org/abstract/MED/27055825', 'https://www.ncbi.nlm.nih.gov/pubmed/27055825', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4823818']}}, {'@type': 'Dataset', 'name': 'Chromosome reports', 'description': 'A set of 26 files, each one listing all the neXtProt entries encoded on a given chromosome (chromosomes 1-22, X, Y, mitochondrial genome (MT), or unknown genomic location), along with their genomic coordinates and a brief summary of the available annotations for each of them.', 'url': 'ftp://ftp.nextprot.org/pub/current_release/chr_reports/', 'keywords': 'human genes, chromosomes, mitochondrial genome, genomic coordinates, chromosomal location, coding strand, protein existence, PE1, PE2, PE3, PE4, PE5', 'distribution': {'@type': 'DataDownload', 'fileFormat': 'TXT'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'neXtProt: Organizing Protein Knowledge in the Context of Human Proteome Projects', 'author': [{'@type': 'Person', 'name': 'Gaudet P'}, {'@type': 'Person', 'name': 'Argoud-Puy G'}, {'@type': 'Person', 'name': 'Cusin I'}, {'@type': 'Person', 'name': 'Duek P'}, {'@type': 'Person', 'name': 'Evalet O'}, {'@type': 'Person', 'name': 'Gateau A'}, {'@type': 'Person', 'name': 'Gleizes A'}, {'@type': 'Person', 'name': 'Pereira M'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Zwahlen C'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Lane L'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Journal of proteome research'}, {'@type': 'PublicationVolume', 'volumeNumber': '12'}, {'@type': 'PublicationIssue', 'datePublished': '2013', 'issueNumber': '1'}], 'sameAs': ['https://doi.org/10.1021/pr300830v', 'https://www.ncbi.nlm.nih.gov/pubmed/23205526']}}, {'@type': 'Dataset', 'name': 'Cancer portal', 'description': 'Portal which provides information about the functional impact of genetic variants and artificially generated mutants in cancer genes', 'url': 'https://www.nextprot.org/portals/breast-cancer', 'keywords': 'BRCA1,Biological database,Cancer,Functional defect assessment,Genetic variants,Molecular phenotypes', 'distribution': {'@type': 'DataDownload', 'fileFormat': 'CSV'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'A new bioinformatics tool to help assess the significance of BRCA1 variants.', 'author': [{'@type': 'Person', 'name': 'Cusin I'}, {'@type': 'Person', 'name': 'Teixeira D'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Rech de Laval V'}, {'@type': 'Person', 'name': 'Gleizes A'}, {'@type': 'Person', 'name': 'Viassolo V'}, {'@type': 'Person', 'name': 'Chappuis PO'}, {'@type': 'Person', 'name': 'Hutter P'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Gaudet P'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Human genomics'}, {'@type': 'PublicationVolume', 'volumeNumber': '12'}, {'@type': 'PublicationIssue', 'datePublished': '2018', 'issueNumber': '1'}], 'sameAs': ['https://doi.org/10.1186/s40246-018-0168-0', 'http://europepmc.org/abstract/MED/29996917', 'https://www.ncbi.nlm.nih.gov/pubmed/29996917', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6042458/']}}, {'@type': 'Dataset', 'name': 'Ion channel portal', 'description': 'Portal which provides information about the functional impact of genetic variants and artificially generated mutants in voltage-gated sodium channel genes', 'url': 'https://www.nextprot.org/portals/navmut', 'keywords': 'clinical interpretation of variants, databases, genetics variants,pathogenicity,phenotype,voltage-gated sodium channel', 'distribution': {'@type': 'DataDownload', 'fileFormat': 'CSV'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'citation': {'@type': 'ScholarlyArticle', 'headline': 'Annotation of functional impact of voltage-gated sodium channel mutations.', 'author': [{'@type': 'Person', 'name': 'Hinard V'}, {'@type': 'Person', 'name': 'Britan A'}, {'@type': 'Person', 'name': 'Schaeffer M'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Thomet U'}, {'@type': 'Person', 'name': 'Rougier JS'}, {'@type': 'Person', 'name': 'Bairoch A'}, {'@type': 'Person', 'name': 'Abriel H'}, {'@type': 'Person', 'name': 'Gaudet P'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Human mutation'}, {'@type': 'PublicationVolume', 'volumeNumber': '38'}, {'@type': 'PublicationIssue', 'datePublished': '2017', 'issueNumber': '5'}], 'sameAs': ['https://doi.org/10.1002/humu.23191', 'http://europepmc.org/abstract/MED/28168870', 'https://www.ncbi.nlm.nih.gov/pubmed/28168870', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5413847/']}}, {'@type': 'Dataset', 'name': 'Protein kinase function portal', 'description': 'Portal which provides information about the function of protein kinases', 'url': 'https://www.nextprot.org/portals/kinase-function', 'keywords': 'databases, protein kinase,phosphorylation, protein-protein interactions, biological process, protein function', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'https://www.nextprot.org/viewers/portals/kinase-function/app/index.html#', 'fileFormat': 'CSV'}, 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC BY 4.0 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}}]}, {'@type': 'SoftwareApplication', 'name': 'Peptide uniqueness checker', 'url': 'https://www.nextprot.org/tools/peptide-uniqueness-checker', 'author': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'The neXtProt peptide uniqueness checker allows scientists to define which peptides can be used to validate the existence of human proteins, i.e. map uniquely versus multiply to human protein sequences taking into account isobaric substitutions, alternative splicing and single amino acid variants.', 'applicationCategory': 'http://schema.org/WebApplication', 'offers': {'@type': 'Offer', 'price': '0.00', 'priceCurrency': 'CHF'}, 'license': {'@type': 'CreativeWork', 'name': 'GNU General Public License v2.0', 'url': 'https://www.gnu.org/licenses/gpl-2.0.fr.html'}, 'operatingSystem': 'Unix,Windows,OS X,Android', 'citation': {'@type': 'ScholarlyArticle', 'headline': 'The neXtProt peptide uniqueness checker: a tool for the proteomics community.', 'author': [{'@type': 'Person', 'name': 'Schaeffer M'}, {'@type': 'Person', 'name': 'Gateau A'}, {'@type': 'Person', 'name': 'Teixeira D'}, {'@type': 'Person', 'name': 'Michel PA'}, {'@type': 'Person', 'name': 'Zahn-Zabal M'}, {'@type': 'Person', 'name': 'Lane L'}], 'isPartOf': [{'@type': 'Periodical', 'name': 'Bioinformatics'}, {'@type': 'PublicationVolume', 'volumeNumber': '33'}, {'@type': 'PublicationIssue', 'datePublished': '2017', 'issueNumber': '21'}], 'sameAs': ['https://doi.org/10.1093/bioinformatics/btx318', 'http://europepmc.org/abstract/MED/28520855', 'https://www.ncbi.nlm.nih.gov/pubmed/28520855', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5860159/']}}, {'@type': 'SoftwareApplication', 'name': 'neXtProt SNORQL', 'url': 'https://snorql.nextprot.org/', 'author': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'A web application for using the SPARQL Protocol and RDF Query Language (SPARQL) to access all the neXtProt protein entry data', 'applicationCategory': 'http://schema.org/WebApplication', 'offers': {'@type': 'Offer', 'price': '0.00', 'priceCurrency': 'CHF'}, 'license': {'@type': 'CreativeWork', 'name': 'GNU General Public License v2.0', 'url': 'https://www.gnu.org/licenses/gpl-2.0.fr.html'}, 'operatingSystem': 'Unix,Windows,OS X,Android'}, {'@type': 'Service', 'name': 'neXtProt advanced search (SPARQL)', 'url': 'https://www.nextprot.org/proteins/search?mode=advanced', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'A web service for using the SPARQL Protocol and RDF Query Language (SPARQL) to retrieve a list of neXtProt entries based on all the neXtProt protein entry data.'}, {'@type': 'Service', 'name': 'neXtProt list manager', 'url': 'https://www.nextprot.org/proteins/search?mode=advanced', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}, 'description': 'A web service for viewing and managing protein lists (a collection of protein entries created by the user).'}, {'@type': 'Service', 'name': 'neXtProt API', 'description': 'The neXtProt API lets you find entities in neXtProt.', 'url': 'https://api.nextprot.org', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}}, {'@type': 'Service', 'name': 'neXtProt SPARQL endpoint', 'description': 'The neXtProt SPARQL endpoint is a point of presence on an HTTP network that‚Äôs capable of receiving and processing SPARQL Protocol requests.', 'url': 'https://sparql.nextprot.org/', 'provider': {'@type': 'Organization', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://www.sib.swiss/', 'logo': 'https://www.nextprot.org/img/sib.png'}}]}]}, {'prefix': 'cas', 'mirId': 'MIR:00000237', 'name': 'CAS', 'pattern': '^\\\\d{1,7}\\\\-\\\\d{2}\\\\-\\\\d$', 'description': 'CAS (Chemical Abstracts Service) is a division of the American Chemical Society and is the producer of comprehensive databases of chemical information.', 'created': '2019-06-11T14:16:33.869+0000', 'modified': '2019-06-11T14:16:33.869+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '50-00-0', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/701'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/701'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/701/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.genome', 'mirId': 'MIR:00000238', 'name': 'KEGG Genome', 'pattern': '^(T0\\\\d+|\\\\w{3,5})$', 'description': 'KEGG Genome is a collection of organisms whose genomes have been completely sequenced.', 'created': '2019-06-11T14:16:34.113+0000', 'modified': '2019-06-11T14:16:34.113+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'eco', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/704'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/704'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/704/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.metagenome', 'mirId': 'MIR:00000239', 'name': 'KEGG Metagenome', 'pattern': '^T3\\\\d+$', 'description': 'The KEGG Metagenome Database collection information on environmental samples (ecosystems) of genome sequences for multiple species.', 'created': '2019-06-11T14:16:34.345+0000', 'modified': '2019-06-11T14:16:34.345+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'T30002', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/707'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/707'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/707/contactPerson'}}, 'metadata': []}, {'prefix': 'narcis', 'mirId': 'MIR:00000240', 'name': 'NARCIS', 'pattern': '^oai\\\\:cwi\\\\.nl\\\\:\\\\d+$', 'description': 'NARCIS provides access to scientific information, including (open access) publications from the repositories of all the Dutch universities, KNAW, NWO and a number of research institutes, which is not referenced in other citation databases.', 'created': '2019-06-11T14:16:34.542+0000', 'modified': '2019-06-11T14:16:34.542+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'oai:cwi.nl:4725', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/709'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/709'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/709/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org/', '@type': 'CreativeWork', '@id': 'https://www.persistent-identifier.nl/urn:nbn:nl:ui:18-4725', 'additionalType': 'other', 'name': 'Mathematical modelling in blood coagulation : simulation and parameter estimation', 'headline': 'Mathematical modelling in blood coagulation : simulation and parameter estimation', 'datePublished': '1997-01-01', 'description': \"textabstractThis paper describes the mathematical modelling of a part of the blood coagulation mechanism. The model includes the activation of factor X by a purified enzyme from Russel's Viper Venom (RVV), factor V and prothrombin, and also comprises the inactivation of the products formed. In this study we assume that in principle the mechanism of the process is known. However, the exact structure of the mechanism is unknown, and the process still can be described by different mathematical models. These models are put to test by measuring their capacity to explain the course of thrombin generation as observed in plasma after recalcification in presence of RVV. The mechanism studied is mathematically modelled as a system of differential-algebraic equations (DAEs). Each candidate model contains some freedom, which is expressed in the model equations by the presence of unknown parameters. For example, reaction constants or initial concentrations are unknown. The goal of parameter estimation is to determine these unknown parameters in such a way that the theoretical (i.e., computed) results fit the experimental data within measurement accuracy and to judge which modifications of the chemical reaction scheme allow the best fit. We present results on model discrimination and estimation of reaction constants, which are hard to obtain in another way.\", 'inLanguage': 'en', 'publisher': 'CWI', 'mainEntityOfPage': 'https://www.narcis.nl/publication/RecordID/oai%3Acwi.nl%3A4725', 'url': 'https://ir.cwi.nl/pub/4725', 'identifier': {'@type': 'PropertyValue', 'name': 'nbn', 'value': 'urn:NBN:nl:ui:18-4725'}, 'author': [{'@context': 'http://schema.org/', '@type': 'Person', 'name': 'W.J.H. Stortelder (Walter)'}, {'@context': 'http://schema.org/', '@type': 'Person', '@id': 'http://isni.org/isni/0000000108922067', 'name': 'P.W. Hemker (Piet)', 'identifier': {'@type': 'PropertyValue', 'name': 'isni', 'value': '0000000108922067'}}, {'@context': 'http://schema.org/', '@type': 'Person', 'name': 'H.C. Hemker'}]}]}, {'prefix': 'jcsd', 'mirId': 'MIR:00000241', 'name': 'Japan Chemical Substance Dictionary', 'pattern': '^J\\\\d{1,3}(\\\\.\\\\d{3})?(\\\\.\\\\d{1,3})?[A-Za-z]$', 'description': 'The Japan Chemical Substance Dictionary is an organic compound dictionary database prepared by the Japan Science and Technology Agency (JST).', 'created': '2019-06-11T14:16:34.772+0000', 'modified': '2019-06-11T14:16:34.772+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'J55.713G', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/712'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/712'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/712/contactPerson'}}, 'metadata': []}, {'prefix': 'pw', 'mirId': 'MIR:00000242', 'name': 'Pathway Ontology', 'pattern': '^PW:\\\\d{7}$', 'description': 'The Pathway Ontology captures information on biological networks, the relationships between netweorks and the alterations or malfunctioning of such networks within a hierarchical structure. The five main branches of the ontology are: classic metabolic pathways, regulatory, signaling, drug, and disease pathwaysfor complex human conditions.', 'created': '2019-06-11T14:16:35.052+0000', 'modified': '2019-06-11T14:16:35.052+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000208', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/715'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/715'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/715/contactPerson'}}, 'metadata': []}, {'prefix': 'insdc.sra', 'mirId': 'MIR:00000243', 'name': 'Sequence Read Archive', 'pattern': '^[SED]R[APRSXZ]\\\\d+$', 'description': 'The Sequence Read Archive (SRA) stores raw sequencing data from the next generation of sequencing platforms Data submitted to SRA. It is organized using a metadata model consisting of six objects: study, sample, experiment, run, analysis and submission. The SRA study contains high-level information including goals of the study and literature references, and may be linked to the INSDC BioProject database.', 'created': '2019-06-11T14:16:35.560+0000', 'modified': '2019-06-11T14:16:35.560+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SRX000007', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/721'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/721'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/721/contactPerson'}}, 'metadata': []}, {'prefix': 'scretf', 'mirId': 'MIR:00000244', 'name': 'ScerTF', 'pattern': '^\\\\w+$', 'description': 'ScerTF is a database of position weight matrices (PWMs) for transcription factors in Saccharomyces species. It identifies a single matrix for each TF that best predicts in vivo data, providing metrics related to the performance of that matrix in accurately representing the DNA binding specificity of the annotated transcription factor.', 'created': '2019-06-11T14:16:35.987+0000', 'modified': '2019-06-11T14:16:35.987+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RSC3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/725'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/725'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/725/contactPerson'}}, 'metadata': []}, {'prefix': 'pharmgkb.gene', 'mirId': 'MIR:00000245', 'name': 'PharmGKB Gene', 'pattern': '^PA\\\\w+$', 'description': 'The PharmGKB database is a central repository for genetic, genomic, molecular and cellular phenotype data and clinical information about people who have participated in pharmacogenomics research studies. The data includes, but is not limited to, clinical and basic pharmacokinetic and pharmacogenomic research in the cardiovascular, pulmonary, cancer, pathways, metabolic and transporter domains.', 'created': '2019-06-11T14:16:36.222+0000', 'modified': '2019-06-11T14:16:36.222+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PA131', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/728'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/728'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/728/contactPerson'}}, 'metadata': []}, {'prefix': 'mirnest', 'mirId': 'MIR:00000246', 'name': 'miRNEST', 'pattern': '^MNEST\\\\d+$', 'description': 'miRNEST is a database of animal, plant and virus microRNAs, containing  miRNA predictions conducted on Expressed Sequence Tags of animal and  plant species.', 'created': '2019-06-11T14:16:36.416+0000', 'modified': '2019-06-11T14:16:36.416+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MNEST029358', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/730'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/730'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/730/contactPerson'}}, 'metadata': []}, {'prefix': 'napp', 'mirId': 'MIR:00000247', 'name': 'NAPP', 'pattern': '^\\\\d+$', 'description': 'NAPP (Nucleic Acids Phylogenetic Profiling is a clustering method based on conserved noncoding RNA (ncRNA) elements in a bacterial genomes. Short intergenic regions from a reference genome are compared with other genomes to identify RNA rich clusters.', 'created': '2019-06-11T14:16:36.669+0000', 'modified': '2019-06-11T14:16:36.669+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '351', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/733'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/733'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/733/contactPerson'}}, 'metadata': []}, {'prefix': 'noncodev3', 'mirId': 'MIR:00000248', 'name': 'NONCODE v3', 'pattern': '^\\\\d+$', 'description': 'NONCODE is a database of expression and functional lncRNA (long noncoding RNA) data obtained from microarray studies. LncRNAs have been shown to play key roles in various biological processes such as imprinting control, circuitry controlling pluripotency and differentiation, immune responses and chromosome dynamics. The collection references NONCODE version 3. This was replaced in 2013 by version 4.', 'created': '2019-06-11T14:16:36.900+0000', 'modified': '2019-06-11T14:16:36.900+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '377550', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/736'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/736'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/736/contactPerson'}}, 'metadata': []}, {'prefix': 'virsirna', 'mirId': 'MIR:00000249', 'name': 'VIRsiRNA', 'pattern': '^virsi\\\\d+$', 'description': 'The VIRsiRNA database contains details of siRNA/shRNA which target viral genome regions. It provides efficacy information where available, as well as the siRNA sequence, viral target and subtype, as well as the target genomic region.', 'created': '2019-06-11T14:16:37.163+0000', 'modified': '2019-06-11T14:16:37.163+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'virsi1909', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/739'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/739'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/739/contactPerson'}}, 'metadata': []}, {'prefix': 'elm', 'mirId': 'MIR:00000250', 'name': 'ELM', 'pattern': '^[A-Za-z_0-9]+$', 'description': 'Linear motifs are short, evolutionarily plastic components of regulatory proteins. Mainly focused on the eukaryotic sequences,the Eukaryotic Linear Motif resource (ELM) is a database of curated motif classes and instances.', 'created': '2019-06-11T14:16:37.386+0000', 'modified': '2019-06-11T14:16:37.386+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CLV_MEL_PAP_1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/742'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/742'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/742/contactPerson'}}, 'metadata': []}, {'prefix': 'mimodb', 'mirId': 'MIR:00000251', 'name': 'MimoDB', 'pattern': '^\\\\d+$', 'description': 'MimoDB is a database collecting peptides that have been selected from random peptide libraries based on their ability to bind small compounds, nucleic acids, proteins, cells, tissues and organs. It also stores other information such as the corresponding target, template, library, and structures. As of March 2016, this database was renamed Biopanning Data Bank.', 'created': '2019-06-11T14:16:37.622+0000', 'modified': '2019-06-11T14:16:37.622+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/745'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/745'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/745/contactPerson'}}, 'metadata': []}, {'prefix': 'sitex', 'mirId': 'MIR:00000252', 'name': 'SitEx', 'pattern': '^\\\\d+$', 'description': 'SitEx is a database containing information on eukaryotic protein functional sites. It stores the amino acid sequence positions in the functional site, in relation to the exon structure of encoding gene This can be used to detect the exons involved in shuffling in protein evolution, or to design protein-engineering experiments.', 'created': '2019-06-11T14:16:37.867+0000', 'modified': '2019-06-11T14:16:37.867+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/748'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/748'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/748/contactPerson'}}, 'metadata': []}, {'prefix': 'bykdb', 'mirId': 'MIR:00000253', 'name': 'BYKdb', 'pattern': '^[A-Z0-9]+$', 'description': 'The bacterial tyrosine kinase database (BYKdb) that collects sequences of putative and authentic bacterial tyrosine kinases,  providing structural and functional information.', 'created': '2019-06-11T14:16:38.105+0000', 'modified': '2019-06-11T14:16:38.105+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'A0A009E7X8', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/751'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/751'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/751/contactPerson'}}, 'metadata': []}, {'prefix': 'conoserver', 'mirId': 'MIR:00000254', 'name': 'Conoserver', 'pattern': '^\\\\d+$', 'description': 'ConoServer is a database specialized in the sequence and structures of conopeptides, which are peptides expressed by carnivorous marine cone snails.', 'created': '2019-06-11T14:16:38.340+0000', 'modified': '2019-06-11T14:16:38.340+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2639', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/754'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/754'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/754/contactPerson'}}, 'metadata': []}, {'prefix': 'topfind', 'mirId': 'MIR:00000255', 'name': 'TopFind', 'pattern': '^([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])$', 'description': 'TopFIND is a database of protein termini, terminus modifications and their proteolytic processing in the species: Homo sapiens, Mus musculus, Arabidopsis thaliana, Saccharomyces cerevisiae and Escherichia coli.', 'created': '2019-06-11T14:16:38.521+0000', 'modified': '2019-06-11T14:16:38.521+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q9UKQ2', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/756'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/756'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/756/contactPerson'}}, 'metadata': []}, {'prefix': 'mipmod', 'mirId': 'MIR:00000256', 'name': 'MIPModDB', 'pattern': '^\\\\w+$', 'description': 'MIPModDb is a database of comparative protein structure models of MIP (Major Intrinsic Protein) family of proteins, identified from complete genome sequence. It provides key information of MIPs based on their sequence and structures.', 'created': '2019-06-11T14:16:38.756+0000', 'modified': '2019-06-11T14:16:38.756+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HOSAPI0399', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/759'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/759'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/759/contactPerson'}}, 'metadata': []}, {'prefix': 'cellimage', 'mirId': 'MIR:00000257', 'name': 'Cell Image Library', 'pattern': '^\\\\d+$', 'description': 'The Cell: An Image Library‚Ñ¢ is a freely accessible, public repository of reviewed and annotated images, videos, and animations of cells from a variety of organisms, showcasing cell architecture, intracellular functionalities, and both normal and abnormal processes.', 'created': '2019-06-11T14:16:38.984+0000', 'modified': '2019-06-11T14:16:38.984+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '24801', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/762'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/762'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/762/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'Tatyana M. Svitkina, Gary G. Borisy (2011) CIL:24801, Xenopus laevis, fibroblast. CIL. Dataset', 'description': 'Structural differentiation of actin network in lamellipodium. Electron micrograph of Xenopus fibroblasts after regular extraction in the presence of polyethelene glycol (PEG) and phalloidin. While the actin network in the front zone of these cels remains dense, the actin network at the lamellipodial rear disassembled in the course of unprotected extraction (CIL 24800).  Image corresponds to Figure 6c from J Cell Biol. 1999 May 31;145(5):1009-26.', 'identifier': 'doi:10.7295/W9CIL24801', 'license': 'http://creativecommons.org/licenses/by-nc-sa/3.0', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'https://cildata.crbs.ucsd.edu/media/images/24801/24801.zip'}, 'sourceOrganization': 'CIL', 'datePublished': '2011-06-02'}]}, {'prefix': 'combine.specifications', 'mirId': 'MIR:00000258', 'name': 'COMBINE specifications', 'pattern': '^\\\\w+(\\\\-|\\\\.|\\\\w)*$', 'description': \"The 'COmputational Modeling in BIology' NEtwork (COMBINE) is an initiative to coordinate the development of the various community standards and formats for computational models, initially in Systems Biology and related fields. This collection pertains to specifications of the standard formats developed by the Computational Modeling in Biology Network.\", 'created': '2019-06-11T14:16:39.210+0000', 'modified': '2019-06-11T14:16:39.210+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'sbgn.er.level-1.version-1.2', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/765'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/765'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/765/contactPerson'}}, 'metadata': []}, {'prefix': 'cabri', 'mirId': 'MIR:00000261', 'name': 'CABRI', 'pattern': '^([A-Za-z]+)?(\\\\_)?([A-Za-z-]+)\\\\:([A-Za-z0-9 ]+)$', 'description': 'CABRI (Common Access to Biotechnological Resources and Information) is an online service where users can search a number of European Biological Resource Centre catalogues. It lists the availability of a particular organism or genetic resource and defines the set of technical specifications and procedures which should be used to handle it.', 'created': '2019-06-11T14:16:39.396+0000', 'modified': '2019-06-11T14:16:39.396+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'dsmz_mutz-id:ACC 291', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/767'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/767'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/767/contactPerson'}}, 'metadata': []}, {'prefix': 'huge', 'mirId': 'MIR:00000263', 'name': 'HUGE', 'pattern': '^KIAA\\\\d{4}$', 'description': 'The Human Unidentified Gene-Encoded (HUGE) protein database contains  results from sequence analysis of human novel large (>4 kb) cDNAs identified in the Kazusa cDNA sequencing project.', 'created': '2019-06-11T14:16:39.763+0000', 'modified': '2019-06-11T14:16:39.763+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'KIAA0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/772'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/772'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/772/contactPerson'}}, 'metadata': []}, {'prefix': 'bindingdb', 'mirId': 'MIR:00000264', 'name': 'BindingDB', 'pattern': '^\\\\w\\\\d+$', 'description': 'BindingDB is the first public database of protein-small molecule affinity data.', 'created': '2019-06-11T14:16:39.982+0000', 'modified': '2019-06-11T14:16:39.982+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'e999', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/775'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/775'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/775/contactPerson'}}, 'metadata': []}, {'prefix': 'dictybase.gene', 'mirId': 'MIR:00000286', 'name': 'Dictybase Gene', 'pattern': '^DDB_G\\\\d+$', 'description': 'The dictyBase database provides data on the model organism Dictyostelium discoideum and related species. It contains the complete genome sequence, ESTs, gene models and functional annotations. This collection references gene information.', 'created': '2019-06-11T14:16:45.247+0000', 'modified': '2019-06-11T14:16:45.247+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DDB_G0267522', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/839'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/839'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/839/contactPerson'}}, 'metadata': []}, {'prefix': 'string', 'mirId': 'MIR:00000265', 'name': 'STRING', 'pattern': '^([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])|([0-9][A-Za-z0-9]{3})$', 'description': 'STRING (Search Tool for Retrieval of Interacting Genes/Proteins) is a database of known and predicted protein interactions.\\r\\nThe interactions include direct (physical) and indirect (functional) associations; they are derived from four sources:Genomic Context, High-throughput Experiments,(Conserved) Coexpression, Previous Knowledge. STRING quantitatively integrates interaction data from these sources for a large number of organisms, and transfers information between these organisms where applicable.', 'created': '2019-06-11T14:16:40.213+0000', 'modified': '2019-06-11T14:16:40.213+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P53350', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/778'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/778'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/778/contactPerson'}}, 'metadata': []}, {'prefix': 'stitch', 'mirId': 'MIR:00000266', 'name': 'STITCH', 'pattern': '^\\\\w{14}$', 'description': 'STITCH is a resource to explore known and predicted interactions of chemicals and proteins. Chemicals are linked to other chemicals and proteins by evidence derived from experiments, databases and the literature.', 'created': '2019-06-11T14:16:40.524+0000', 'modified': '2019-06-11T14:16:40.524+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BQJCRHHNABKAKU', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/781'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/781'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/781/contactPerson'}}, 'metadata': []}, {'prefix': 'atcvet', 'mirId': 'MIR:00000267', 'name': 'Anatomical Therapeutic Chemical Vetinary', 'pattern': '^Q[A-Z0-9]+$', 'description': 'The ATCvet system for the classification of veterinary medicines is based on the same overall principles as the ATC system for substances used in human medicine. In ATCvet systems, preparations are divided into groups, according to their therapeutic use. First, they are divided into 15 anatomical groups (1st level), classified as QA-QV in the ATCvet system, on the basis of their main therapeutic use.', 'created': '2019-06-11T14:16:40.712+0000', 'modified': '2019-06-11T14:16:40.712+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'QJ51RV02', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/783'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/783'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/783/contactPerson'}}, 'metadata': []}, {'prefix': 'phenolexplorer', 'mirId': 'MIR:00000268', 'name': 'Phenol-Explorer', 'pattern': '^\\\\d+$', 'description': 'Phenol-Explorer is an electronic database on polyphenol content in foods. Polyphenols form a wide group of natural antioxidants present in a large number of foods and beverages. They contribute to food characteristics such as taste, colour or shelf-life. They also participate in the prevention of several major chronic diseases such as cardiovascular diseases, diabetes, cancers, neurodegenerative diseases or osteoporosis.', 'created': '2019-06-11T14:16:40.928+0000', 'modified': '2019-06-11T14:16:40.928+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '75', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/785'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/785'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/785/contactPerson'}}, 'metadata': []}, {'prefix': 'snomedct', 'mirId': 'MIR:00000269', 'name': 'SNOMED CT', 'pattern': '^(\\\\w+)?\\\\d+$', 'description': 'SNOMED CT (Systematized Nomenclature of Medicine -- Clinical Terms), is a systematically organized computer processable collection of medical terminology covering most areas of clinical information such as diseases, findings, procedures, microorganisms, pharmaceuticals, etc.', 'created': '2019-06-11T14:16:41.184+0000', 'modified': '2019-06-11T14:16:41.184+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '284196006', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/788'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/788'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/788/contactPerson'}}, 'metadata': []}, {'prefix': 'mesh.2012', 'mirId': 'MIR:00000270', 'name': 'MeSH 2012', 'pattern': '^[A-Za-z0-9]+$', 'description': \"MeSH (Medical Subject Headings) is the National Library of Medicine's controlled vocabulary thesaurus. It consists of sets of terms naming descriptors in a hierarchical structure that permits searching at various levels of specificity. This thesaurus is used by NLM for indexing articles from biomedical journals, cataloging of books, documents, etc. This collection references MeSH terms published in 2012.\", 'created': '2019-06-11T14:16:41.417+0000', 'modified': '2019-06-11T14:16:41.417+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '17186', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/791'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/791'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/791/contactPerson'}}, 'metadata': []}, {'prefix': 'knapsack', 'mirId': 'MIR:00000271', 'name': 'KnapSack', 'pattern': '^C\\\\d{8}', 'description': 'Knapsack provides information on metabolites and  the taxonomic class with which they are associated.', 'created': '2019-06-11T14:16:41.635+0000', 'modified': '2019-06-11T14:16:41.635+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C00000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/794'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/794'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/794/contactPerson'}}, 'metadata': []}, {'prefix': 'cdpd', 'mirId': 'MIR:00000272', 'name': 'Canadian Drug Product Database', 'pattern': '^\\\\d+$', 'description': \"The Canadian Drug Product Database (DPD) contains product specific information on drugs approved for use in Canada, and includes human pharmaceutical and biological drugs, veterinary drugs and disinfectant products. This information includes 'brand name', 'route of administration' and a Canadian 'Drug Identification Number' (DIN).\", 'created': '2019-06-11T14:16:41.852+0000', 'modified': '2019-06-11T14:16:41.852+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '63250', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/797'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/797'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/797/contactPerson'}}, 'metadata': []}, {'prefix': 'massbank', 'mirId': 'MIR:00000273', 'name': 'MassBank', 'pattern': '^[A-Z]{2}[A-Z0-9][0-9]{5}$', 'description': 'MassBank is a federated database of reference spectra from different instruments, including high-resolution mass spectra of small metabolites (<3000 Da).', 'created': '2019-06-11T14:16:42.071+0000', 'modified': '2019-06-11T14:16:42.071+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PB000166', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/800'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/800'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/800/contactPerson'}}, 'metadata': []}, {'prefix': 'gmd', 'mirId': 'MIR:00000274', 'name': 'Golm Metabolome Database', 'pattern': '^([0-9a-fA-F]){8}(-([0-9a-fA-F]){4}){3}-([0-9a-fA-F]){12}$', 'description': 'Golm Metabolome Database (GMD) provides public access to custom mass spectral libraries, metabolite profiling experiments as well as additional information and tools. This collection references metabolite information, relating the biologically active substance to metabolic pathways or signalling phenomena.', 'created': '2019-06-11T14:16:42.514+0000', 'modified': '2019-06-11T14:16:42.514+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '68513255-fc44-4041-bc4b-4fd2fae7541d', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/805'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/805'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/805/contactPerson'}}, 'metadata': []}, {'prefix': 'homologene', 'mirId': 'MIR:00000275', 'name': 'HomoloGene', 'pattern': '^\\\\d+$', 'description': 'HomoloGene is a system for automated detection of homologs among the annotated genes of several completely sequenced eukaryotic genomes.', 'created': '2019-06-11T14:16:42.733+0000', 'modified': '2019-06-11T14:16:42.733+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/808'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/808'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/808/contactPerson'}}, 'metadata': []}, {'prefix': 'umbbd.compound', 'mirId': 'MIR:00000276', 'name': 'UM-BBD Compound', 'pattern': '^c\\\\d+$', 'description': 'The University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD) contains information on microbial biocatalytic reactions and biodegradation pathways for primarily xenobiotic, chemical compounds. The goal of the UM-BBD is to provide information on microbial enzyme-catalyzed reactions that are important for biotechnology. This collection refers to compound information.', 'created': '2019-06-11T14:16:42.994+0000', 'modified': '2019-06-11T14:16:42.994+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'c0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/811'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/811'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/811/contactPerson'}}, 'metadata': []}, {'prefix': 'abs', 'mirId': 'MIR:00000277', 'name': 'ABS', 'pattern': '^A\\\\d+$', 'description': 'The database of Annotated regulatory Binding Sites (from orthologous promoters), ABS, is a public database of known binding sites identified in promoters of orthologous vertebrate genes that have been manually curated from bibliography.', 'created': '2019-06-11T14:16:43.224+0000', 'modified': '2019-06-11T14:16:43.224+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'A0014', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/814'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/814'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/814/contactPerson'}}, 'metadata': []}, {'prefix': 'apd', 'mirId': 'MIR:00000278', 'name': 'APD', 'pattern': '^\\\\d{5}$', 'description': 'The antimicrobial peptide database (APD) provides information on anticancer, antiviral, antifungal and antibacterial peptides.', 'created': '2019-06-11T14:16:43.451+0000', 'modified': '2019-06-11T14:16:43.451+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '01001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/817'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/817'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/817/contactPerson'}}, 'metadata': []}, {'prefix': 'chemdb', 'mirId': 'MIR:00000279', 'name': 'ChemDB', 'pattern': '^\\\\d+$', 'description': 'ChemDB is a chemical database containing commercially available small molecules, important for use as synthetic building blocks, probes in systems biology and as leads for the discovery of drugs and other useful compounds.', 'created': '2019-06-11T14:16:43.696+0000', 'modified': '2019-06-11T14:16:43.696+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3966782', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/820'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/820'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/820/contactPerson'}}, 'metadata': []}, {'prefix': 'dpv', 'mirId': 'MIR:00000280', 'name': 'DPV', 'pattern': '^\\\\d+$', 'description': 'Description of Plant Viruses (DPV) provides information about viruses, viroids and satellites of plants, fungi and protozoa. It provides taxonomic information, including brief descriptions of each family and genus, and classified lists of virus sequences. The database also holds detailed information for all sequences of viruses, viroids and satellites of plants, fungi and protozoa that are complete or that contain at least one complete gene.', 'created': '2019-06-11T14:16:43.916+0000', 'modified': '2019-06-11T14:16:43.916+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/823'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/823'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/823/contactPerson'}}, 'metadata': []}, {'prefix': 'iuphar.receptor', 'mirId': 'MIR:00000281', 'name': 'IUPHAR receptor', 'pattern': '^\\\\d+$', 'description': 'The IUPHAR Compendium details the molecular, biophysical and pharmacological properties of identified mammalian sodium, calcium and potassium channels, as well as the related cyclic nucleotide-modulated ion channels and transient receptor potential channels. It includes information on nomenclature systems, and on inter and intra-species molecular structure variation. This collection references individual receptors or subunits.', 'created': '2019-06-11T14:16:44.139+0000', 'modified': '2019-06-11T14:16:44.139+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '101', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/826'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/826'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/826/contactPerson'}}, 'metadata': []}, {'prefix': 'aceview.worm', 'mirId': 'MIR:00000282', 'name': 'Aceview Worm', 'pattern': '^[a-z0-9-]+$', 'description': 'AceView provides a curated sequence representation of all public mRNA sequences (mRNAs from GenBank or RefSeq, and single pass cDNA sequences from dbEST and Trace). These are aligned on the genome and clustered into a minimal number of alternative transcript variants and grouped into genes. In addition, alternative features such as promoters, and expression in tissues is recorded. This collection references C. elegans genes and expression.', 'created': '2019-06-11T14:16:44.374+0000', 'modified': '2019-06-11T14:16:44.374+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'aap-1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/829'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/829'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/829/contactPerson'}}, 'metadata': []}, {'prefix': 'imgt.ligm', 'mirId': 'MIR:00000287', 'name': 'IMGT LIGM', 'pattern': '^M\\\\d+$', 'description': 'IMGT, the international ImMunoGeneTics project, is a collection of high-quality integrated databases specialising in Immunoglobulins, T cell receptors and the Major Histocompatibility Complex (MHC) of all vertebrate species. IMGT/LIGM is a comprehensive database of fully annotated sequences of Immunoglobulins and T cell receptors from human and other vertebrates.', 'created': '2019-06-11T14:16:45.469+0000', 'modified': '2019-06-11T14:16:45.469+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'M94112', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/842'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/842'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/842/contactPerson'}}, 'metadata': []}, {'prefix': 'worfdb', 'mirId': 'MIR:00000288', 'name': 'Worfdb', 'pattern': '^\\\\w+(\\\\.\\\\d+)?', 'description': 'WOrfDB (Worm ORFeome DataBase) contains data from the cloning of complete set of predicted protein-encoding Open Reading Frames (ORFs) of Caenorhabditis elegans. This collection describes experimentally defined transcript structures of unverified genes through RACE (Rapid Amplification of cDNA Ends).', 'created': '2019-06-11T14:16:45.846+0000', 'modified': '2019-06-11T14:16:45.846+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'T01B6.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/847'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/847'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/847/contactPerson'}}, 'metadata': []}, {'prefix': 'nextdb', 'mirId': 'MIR:00000289', 'name': 'NEXTDB', 'pattern': '^[A-Za-z0-9]+$', 'description': \"NextDb is a database that provides information on the expression pattern map of the 100Mb genome of the nematode Caenorhabditis elegans. This was done through EST analysis and systematic whole mount in situ hybridization. Information available includes 5' and 3' ESTs, and in-situ hybridization images of 11,237 cDNA clones.\", 'created': '2019-06-11T14:16:46.065+0000', 'modified': '2019-06-11T14:16:46.065+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '6b1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/850'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/850'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/850/contactPerson'}}, 'metadata': []}, {'prefix': 'soybase', 'mirId': 'MIR:00000291', 'name': 'SoyBase', 'pattern': '^\\\\w+(\\\\-)?\\\\w+(\\\\-)?\\\\w+$', 'description': 'SoyBase is a repository for curated genetics, genomics and related data resources for soybean.', 'created': '2019-06-11T14:16:46.310+0000', 'modified': '2019-06-11T14:16:46.310+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BARC-013845-01256', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/853'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/853'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/853/contactPerson'}}, 'metadata': []}, {'prefix': 'hamap', 'mirId': 'MIR:00000292', 'name': 'HAMAP', 'pattern': '^MF_\\\\d+$', 'description': 'HAMAP is a system that identifies and semi-automatically annotates proteins that are part of well-conserved and orthologous microbial families or subfamilies. These are used to build rules which are used to propagate annotations to member bacterial, archaeal and plastid-encoded protein entries.', 'created': '2019-06-11T14:16:46.532+0000', 'modified': '2019-06-11T14:16:46.532+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MF_01400', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/856'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/856'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/856/contactPerson'}}, 'metadata': [{'@context': ['https://schema.org/'], '@type': 'Organization', 'url': 'https://hamap.expasy.org', 'name': 'HAMAP', 'logo': 'https://hamap.expasy.org//images/hamap/HAMAP_Red_Logo.png', 'memberof': {'@type': 'NGO', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://sib.swiss/'}, 'parentOrganization': {'@type': 'Organization', 'name': 'ExPASy', 'url': 'https://www.expasy.org/'}}]}, {'prefix': 'rouge', 'mirId': 'MIR:00000293', 'name': 'Rouge', 'pattern': '^m\\\\w+$', 'description': 'The Rouge protein database contains results from sequence analysis of novel large (>4 kb) cDNAs identified in the Kazusa cDNA sequencing project.', 'created': '2019-06-11T14:16:46.721+0000', 'modified': '2019-06-11T14:16:46.721+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'mKIAA4200', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/858'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/858'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/858/contactPerson'}}, 'metadata': []}, {'prefix': 'arrayexpress.platform', 'mirId': 'MIR:00000294', 'name': 'ArrayExpress Platform', 'pattern': '^[AEP]-\\\\w{4}-\\\\d+$', 'description': 'ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with Microarray Gene Expression Data (MGED) recommendations.This collection references the specific platforms used in the generation of experimental results.', 'created': '2019-06-11T14:16:46.944+0000', 'modified': '2019-06-11T14:16:46.944+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'A-GEOD-50', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/861'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/861'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/861/contactPerson'}}, 'metadata': []}, {'prefix': 'cgsc', 'mirId': 'MIR:00000295', 'name': 'CGSC Strain', 'pattern': '^\\\\d+$', 'description': 'The CGSC Database of E. coli genetic information includes genotypes and reference information for the strains in the CGSC collection, the names, synonyms, properties, and map position for genes, gene product information, and information on specific mutations and references to primary literature.', 'created': '2019-06-11T14:16:47.138+0000', 'modified': '2019-06-11T14:16:47.138+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '74', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/863'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/863'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/863/contactPerson'}}, 'metadata': []}, {'prefix': 'dragondb.dna', 'mirId': 'MIR:00000297', 'name': 'DragonDB DNA', 'pattern': '^\\\\d\\\\w+$', 'description': 'DragonDB is a genetic and genomic database for Antirrhinum majus (Snapdragon). This collection refers to DNA sequence information.', 'created': '2019-06-11T14:16:47.375+0000', 'modified': '2019-06-11T14:16:47.375+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3hB06', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/866'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/866'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/866/contactPerson'}}, 'metadata': []}, {'prefix': 'dragondb.protein', 'mirId': 'MIR:00000298', 'name': 'DragonDB Protein', 'pattern': '^\\\\w+$', 'description': 'DragonDB is a genetic and genomic database for Antirrhinum majus (Snapdragon). This collection refers to protein sequence information.', 'created': '2019-06-11T14:16:47.607+0000', 'modified': '2019-06-11T14:16:47.607+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AMDEFA', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/869'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/869'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/869/contactPerson'}}, 'metadata': []}, {'prefix': 'dragondb.locus', 'mirId': 'MIR:00000299', 'name': 'DragonDB Locus', 'pattern': '^\\\\w+$', 'description': 'DragonDB is a genetic and genomic database for Antirrhinum majus (Snapdragon). This collection refers to Locus information.', 'created': '2019-06-11T14:16:47.798+0000', 'modified': '2019-06-11T14:16:47.798+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DEF', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/871'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/871'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/871/contactPerson'}}, 'metadata': []}, {'prefix': 'dragondb.allele', 'mirId': 'MIR:00000300', 'name': 'DragonDB Allele', 'pattern': '^\\\\w+$', 'description': 'DragonDB is a genetic and genomic database for Antirrhinum majus (Snapdragon). This collection refers to allele information.', 'created': '2019-06-11T14:16:47.981+0000', 'modified': '2019-06-11T14:16:47.981+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'cho', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/873'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/873'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/873/contactPerson'}}, 'metadata': []}, {'prefix': 'issn', 'mirId': 'MIR:00000301', 'name': 'ISSN', 'pattern': '^\\\\d{4}-\\\\d{3}[\\\\dX]$', 'description': 'The International Standard Serial Number (ISSN) is a unique eight-digit number used to identify a print or electronic periodical publication, rather than individual articles or books.', 'created': '2019-06-11T14:16:48.189+0000', 'modified': '2019-06-11T14:16:48.189+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0745-4570', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/875'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/875'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/875/contactPerson'}}, 'metadata': []}, {'prefix': 'merops.family', 'mirId': 'MIR:00000302', 'name': 'MEROPS Family', 'pattern': '^[SCTAGMNU]\\\\d+$', 'description': 'The MEROPS database is an information resource for peptidases (also termed proteases, proteinases and proteolytic enzymes) and the proteins that inhibit them. These are hierarchically classified and assigned to a Family on the basis of statistically significant similarities in amino acid sequence. Families thought to be homologous are grouped together in a Clan. This collection references peptidase families.', 'created': '2019-06-11T14:16:48.579+0000', 'modified': '2019-06-11T14:16:48.579+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'S1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/880'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/880'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/880/contactPerson'}}, 'metadata': []}, {'prefix': 'mo', 'mirId': 'MIR:00000303', 'name': 'MGED Ontology', 'pattern': '^\\\\w+$', 'description': 'The MGED Ontology (MO) provides terms for annotating all aspects of a microarray experiment from the design of the experiment and array layout, through to the preparation of the biological sample and the protocols used to hybridize the RNA and analyze the data.', 'created': '2019-06-11T14:16:48.772+0000', 'modified': '2019-06-11T14:16:48.772+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ArrayGroup', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/882'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/882'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/882/contactPerson'}}, 'metadata': []}, {'prefix': 'nasc', 'mirId': 'MIR:00000304', 'name': 'NASC code', 'pattern': '^(\\\\w+)?\\\\d+$', 'description': 'The Nottingham Arabidopsis Stock Centre (NASC) provides seed and information resources to the International Arabidopsis Genome Programme and the wider research community.', 'created': '2019-06-11T14:16:49.131+0000', 'modified': '2019-06-11T14:16:49.131+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'N1899', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/886'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/886'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/886/contactPerson'}}, 'metadata': []}, {'prefix': 'niaest', 'mirId': 'MIR:00000305', 'name': 'NIAEST', 'pattern': '^\\\\w\\\\d{4}\\\\w\\\\d{2}(\\\\-[35])?$', 'description': 'A catalog of mouse genes expressed in early embryos, embryonic and adult stem cells, including 250000 ESTs, was assembled by the NIA (National Institute on Aging) assembled.This collection represents the name and sequence from individual cDNA clones.', 'created': '2019-06-11T14:16:49.376+0000', 'modified': '2019-06-11T14:16:49.376+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'J0705A10', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/889'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/889'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/889/contactPerson'}}, 'metadata': []}, {'prefix': 'pazar', 'mirId': 'MIR:00000306', 'name': 'Pazar Transcription Factor', 'pattern': '^TF\\\\w+$', 'description': 'The PAZAR database unites independently created and maintained data collections of transcription factor and regulatory sequence annotation. It provides information on the sequence and target of individual transcription factors.', 'created': '2019-06-11T14:16:49.605+0000', 'modified': '2019-06-11T14:16:49.605+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TF0001053', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/892'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/892'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/892/contactPerson'}}, 'metadata': []}, {'prefix': 'po', 'mirId': 'MIR:00000307', 'name': 'Plant Ontology', 'pattern': '^PO:\\\\d+$', 'description': 'The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data.', 'created': '2019-06-11T14:16:49.830+0000', 'modified': '2019-06-11T14:16:49.830+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0009089', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/895'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/895'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/895/contactPerson'}}, 'metadata': []}, {'prefix': 'rnamods', 'mirId': 'MIR:00000308', 'name': 'RNA Modification Database', 'pattern': '^\\\\d{3}$', 'description': 'The RNA modification database provides a comprehensive listing of post-transcriptionally modified nucleosides from RNA. The database consists of all RNA-derived ribonucleosides of known structure, including those from established sequence positions, as well as those detected or characterized from hydrolysates of RNA.', 'created': '2019-06-11T14:16:50.326+0000', 'modified': '2019-06-11T14:16:50.326+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '101', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/900'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/900'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/900/contactPerson'}}, 'metadata': []}, {'prefix': 'treebase', 'mirId': 'MIR:00000312', 'name': 'TreeBASE', 'pattern': '^TB[1,2]?:[A-Z][a-z]?\\\\d+$', 'description': 'TreeBASE is a relational database designed to manage and explore information on phylogenetic relationships. It includes phylogenetic trees and data matrices, together with information about the relevant publication, taxa, morphological and sequence-based characters, and published analyses. Data in TreeBASE are exposed to the public if they are used in a publication that is in press or published in a peer-reviewed scientific journal, etc.', 'created': '2019-06-11T14:16:50.556+0000', 'modified': '2019-06-11T14:16:50.556+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TB2:S1000', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/903'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/903'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/903/contactPerson'}}, 'metadata': []}, {'prefix': 'cst.ab', 'mirId': 'MIR:00000430', 'name': 'Cell Signaling Technology Antibody', 'pattern': '^\\\\d+$', 'description': 'Cell Signaling Technology is a commercial organisation which provides a pathway portal to showcase their phospho-antibody products. This collection references antibody products.', 'created': '2019-06-11T14:17:18.559+0000', 'modified': '2019-06-11T14:17:18.559+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3305', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1235'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1235'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1235/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org/', '@type': 'Product', 'name': 'GCN5L2 (C26A10) Rabbit mAb', 'Category': ' Primary Antibodies', 'additionalProperty': [{'@type': 'PropertyValue', 'name': 'APPLICATIONS', 'value': 'W,IP,IF'}, {'@type': 'PropertyValue', 'name': 'REACTIVITY', 'value': 'H,M,R,Mk'}, {'@type': 'PropertyValue', 'name': 'SENSITIVITY', 'value': 'Endogenous'}, {'@type': 'PropertyValue', 'name': 'MW(kDa)', 'value': '94'}, {'@type': 'PropertyValue', 'name': 'SOURCE', 'value': 'Rabbit'}, {'@type': 'PropertyValue', 'name': 'Product Citations', 'value': '29'}], 'image': '', 'description': '', 'brand': 'Cell Signaling Technology'}]}, {'prefix': 'tgd', 'mirId': 'MIR:00000313', 'name': 'Tetrahymena Genome Database', 'pattern': '^TTHERM\\\\_\\\\d+$', 'description': 'The Tetrahymena Genome Database (TGD) Wiki is a database of information about the Tetrahymena thermophila genome sequence. It provides information curated from the literature about each published gene, including a standardized gene name, a link to the genomic locus, gene product annotations utilizing the Gene Ontology, and links to published literature.', 'created': '2019-06-11T14:16:50.779+0000', 'modified': '2019-06-11T14:16:50.779+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TTHERM_00648910', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/906'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/906'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/906/contactPerson'}}, 'metadata': []}, {'prefix': 'tigrfam', 'mirId': 'MIR:00000315', 'name': 'TIGRFAMS', 'pattern': '^TIGR\\\\d+$', 'description': 'TIGRFAMs is a resource consisting of curated multiple sequence alignments, Hidden Markov Models (HMMs) for protein sequence classification, and associated information designed to support automated annotation of (mostly prokaryotic) proteins.', 'created': '2019-06-11T14:16:50.975+0000', 'modified': '2019-06-11T14:16:50.975+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TIGR00010', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/908'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/908'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/908/contactPerson'}}, 'metadata': []}, {'prefix': 'atfdb.family', 'mirId': 'MIR:00000316', 'name': 'Animal TFDB Family', 'pattern': '^\\\\w+$', 'description': 'The Animal Transcription Factor DataBase (AnimalTFDB) classifies TFs in sequenced animal genomes, as well as collecting the transcription co-factors and chromatin remodeling factors of those genomes. This collections refers to transcription factor families, and the species in which they are found.', 'created': '2019-06-11T14:16:51.241+0000', 'modified': '2019-06-11T14:16:51.241+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CUT', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/911'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/911'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/911/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org', '@graph': [{'@type': 'WebSite', '@id': 'https://www.bioguo.org/#website', 'url': 'https://www.bioguo.org/', 'name': 'Les produits bio', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.bioguo.org/?s={search_term_string}', 'query-input': 'required name=search_term_string'}}]}]}, {'prefix': 'iuphar.family', 'mirId': 'MIR:00000317', 'name': 'IUPHAR family', 'pattern': '^\\\\d+$', 'description': 'The IUPHAR Compendium details the molecular, biophysical and pharmacological properties of identified mammalian sodium, calcium and potassium channels, as well as the related cyclic nucleotide-modulated ion channels and the recently described transient receptor potential channels. It includes information on nomenclature systems, and on inter and intra-species molecular structure variation. This collection references families of receptors or subunits.', 'created': '2019-06-11T14:16:51.481+0000', 'modified': '2019-06-11T14:16:51.481+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '78', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/914'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/914'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/914/contactPerson'}}, 'metadata': []}, {'prefix': 'dbg2introns', 'mirId': 'MIR:00000318', 'name': 'DBG2 Introns', 'pattern': '^\\\\w{1,2}\\\\.(\\\\w{1,2}\\\\.)?[A-Za-z0-9]+$', 'description': 'The Database for Bacterial Group II Introns  provides a catalogue of full-length, non-redundant group II introns present in bacterial DNA sequences in GenBank.', 'created': '2019-06-11T14:16:51.708+0000', 'modified': '2019-06-11T14:16:51.708+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Cu.me.I1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/917'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/917'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/917/contactPerson'}}, 'metadata': []}, {'prefix': 'sdbs', 'mirId': 'MIR:00000319', 'name': 'Spectral Database for Organic Compounds', 'pattern': '\\\\d+$', 'description': 'The Spectral Database for Organic Compounds (SDBS) is an integrated spectral database system for organic compounds. It provides access to 6 different types of spectra for each compound, including Mass spectrum (EI-MS), a Fourier transform infrared spectrum (FT-IR), and NMR spectra.', 'created': '2019-06-11T14:16:51.940+0000', 'modified': '2019-06-11T14:16:51.940+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4544', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/920'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/920'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/920/contactPerson'}}, 'metadata': []}, {'prefix': 'vbase2', 'mirId': 'MIR:00000320', 'name': 'Vbase2', 'pattern': '^\\\\w+$', 'description': 'The database VBASE2 provides germ-line sequences of human and mouse immunoglobulin variable (V) genes.', 'created': '2019-06-11T14:16:52.167+0000', 'modified': '2019-06-11T14:16:52.167+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'humIGHV025', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/923'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/923'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/923/contactPerson'}}, 'metadata': []}, {'prefix': 'spike.map', 'mirId': 'MIR:00000321', 'name': 'SPIKE Map', 'pattern': '^spike\\\\d{5}$', 'description': 'SPIKE (Signaling Pathways Integrated Knowledge Engine) is a repository that can store, organise and allow retrieval of pathway information in a way that will be useful for the research community. The database currently focuses primarily on pathways describing DNA damage response, cell cycle, programmed cell death and hearing related pathways. Pathways are regularly updated, and additional pathways are gradually added. The complete database and the individual maps are freely exportable in several formats. This collection references pathway maps.', 'created': '2019-06-11T14:16:52.396+0000', 'modified': '2019-06-11T14:16:52.396+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'spike00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/926'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/926'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/926/contactPerson'}}, 'metadata': []}, {'prefix': 'metlin', 'mirId': 'MIR:00000322', 'name': 'METLIN', 'pattern': '^\\\\d{4}$', 'description': 'The METLIN (Metabolite and Tandem Mass Spectrometry) Database is a repository of metabolite information as well as tandem mass spectrometry data, providing public access to its comprehensive MS and MS/MS metabolite data. An annotated list of known metabolites and their mass, chemical formula, and structure are available, with each metabolite linked to external resources for further reference and inquiry.', 'created': '2019-06-11T14:16:52.640+0000', 'modified': '2019-06-11T14:16:52.640+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1455', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/929'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/929'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/929/contactPerson'}}, 'metadata': []}, {'prefix': 'genecards', 'mirId': 'MIR:00000323', 'name': 'GeneCards', 'pattern': '^[A-Za-z-0-9_]+(\\\\@)?$', 'description': 'The GeneCards human gene database stores gene related transcriptomic, genetic, proteomic, functional and disease information. It uses standard nomenclature and approved gene symbols. GeneCards presents a complete summary for each human gene.', 'created': '2019-06-11T14:16:52.858+0000', 'modified': '2019-06-11T14:16:52.858+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ABL1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/932'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/932'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/932/contactPerson'}}, 'metadata': []}, {'prefix': 'mmrrc', 'mirId': 'MIR:00000324', 'name': 'MMRRC', 'pattern': '^\\\\d+$', 'description': 'The MMRRC database is a repository of available mouse stocks and embryonic stem cell line collections.', 'created': '2019-06-11T14:16:53.082+0000', 'modified': '2019-06-11T14:16:53.082+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '70', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/935'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/935'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/935/contactPerson'}}, 'metadata': []}, {'prefix': 'umbbd.reaction', 'mirId': 'MIR:00000325', 'name': 'UM-BBD Reaction', 'pattern': '^r\\\\d+$', 'description': 'The University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD) contains information on microbial biocatalytic reactions and biodegradation pathways for primarily xenobiotic, chemical compounds. The goal of the UM-BBD is to provide information on microbial enzyme-catalyzed reactions that are important for biotechnology. This collection refers to reaction information.', 'created': '2019-06-11T14:16:53.335+0000', 'modified': '2019-06-11T14:16:53.335+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'r0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/938'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/938'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/938/contactPerson'}}, 'metadata': []}, {'prefix': 'umbbd.enzyme', 'mirId': 'MIR:00000326', 'name': 'UM-BBD Enzyme', 'pattern': '^e\\\\d+$', 'description': 'The University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD) contains information on microbial biocatalytic reactions and biodegradation pathways for primarily xenobiotic, chemical compounds. The goal of the UM-BBD is to provide information on microbial enzyme-catalyzed reactions that are important for biotechnology. This collection refers to enzyme information.', 'created': '2019-06-11T14:16:53.546+0000', 'modified': '2019-06-11T14:16:53.546+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'e0333', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/940'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/940'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/940/contactPerson'}}, 'metadata': []}, {'prefix': 'umbbd.pathway', 'mirId': 'MIR:00000327', 'name': 'UM-BBD Pathway', 'pattern': '^\\\\w+$', 'description': 'The University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD) contains information on microbial biocatalytic reactions and biodegradation pathways for primarily xenobiotic, chemical compounds. The goal of the UM-BBD is to provide information on microbial enzyme-catalyzed reactions that are important for biotechnology. This collection refers to pathway information.', 'created': '2019-06-11T14:16:53.741+0000', 'modified': '2019-06-11T14:16:53.741+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ala', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/942'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/942'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/942/contactPerson'}}, 'metadata': []}, {'prefix': 'umbbd.rule', 'mirId': 'MIR:00000328', 'name': 'UM-BBD Biotransformation Rule', 'pattern': '^bt\\\\d+$', 'description': 'The University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD) contains information on microbial biocatalytic reactions and biodegradation pathways for primarily xenobiotic, chemical compounds. The UM-BBD Pathway Prediction System (PPS) predicts microbial catabolic reactions using substructure searching, a rule-base, and atom-to-atom mapping. The PPS recognizes organic functional groups found in a compound and predicts transformations based on biotransformation rules. These rules are based on reactions found in the UM-BBD database. This collection references those rules.', 'created': '2019-06-11T14:16:53.927+0000', 'modified': '2019-06-11T14:16:53.927+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'bt0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/944'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/944'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/944/contactPerson'}}, 'metadata': []}, {'prefix': 'mirex', 'mirId': 'MIR:00000329', 'name': 'mirEX', 'pattern': '^\\\\d+(\\\\w+)?$', 'description': 'mirEX is a comprehensive platform for comparative analysis of primary microRNA expression data, storing RT‚ÄìqPCR-based gene expression profile over seven development stages of Arabidopsis. It also provides RNA structural models, publicly available deep sequencing results and experimental procedure details. This collection provides profile information for a single microRNA over all development stages.', 'created': '2019-06-11T14:16:54.147+0000', 'modified': '2019-06-11T14:16:54.147+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '165a', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/946'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/946'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/946/contactPerson'}}, 'metadata': []}, {'prefix': 'dictybase.est', 'mirId': 'MIR:00000330', 'name': 'Dictybase EST', 'pattern': '^DDB\\\\d+$', 'description': 'The dictyBase database provides data on the model organism Dictyostelium discoideum and related species. It contains the complete genome sequence, ESTs, gene models and functional annotations. This collection references expressed sequence tag (EST) information.', 'created': '2019-06-11T14:16:54.345+0000', 'modified': '2019-06-11T14:16:54.345+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DDB0016567', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/948'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/948'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/948/contactPerson'}}, 'metadata': []}, {'prefix': 'mobidb', 'mirId': 'MIR:00000637', 'name': 'MobiDB', 'pattern': '^[OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2}$', 'description': 'MobiDB is a database of protein disorder and mobility annotations.', 'created': '2019-06-11T14:18:08.223+0000', 'modified': '2019-06-11T14:18:08.223+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P10636', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1768'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1768'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1768/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'DataCatalog', '@id': 'http://mobidb.bio.unipd.it', 'name': 'MobiDB', 'description': 'Database of protein disorder and mobility annotations', 'url': 'http://mobidb.bio.unipd.it', 'publication': {'@type': 'PublicationEvent', 'name': 'MobiDB 3.0: more annotations for intrinsic disorder, conformational diversity and interactions in proteins', 'url': 'https://academic.oup.com/nar/article/46/D1/D471/4612964', 'sameAs': ['https://doi.org/10.1093/nar/gkx1071', 'https://europepmc.org/abstract/MED/29136219', 'https://www.ncbi.nlm.nih.gov/pubmed/29136219', 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5753340']}, 'keywords': ['protein annotation', 'protein disorder'], 'sourceOrganization': [{'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}], 'provider': [{'@type': 'Person', 'name': 'Silvio Tosatto', 'email': 'silvio.tosatto@unipd.it'}], 'datePublished': '2018-01', 'dateModified': '2018-01', 'encodingFormat': 'text/html', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000637', 'dataset': [{'@type': 'Dataset', 'name': 'MobiDB', 'description': 'MobiDB dataset describes disorder and mobility annotation for proteins', 'url': 'http://mobidb.bio.unipd.it', 'identifier': 'https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000637', 'keywords': ['protein annotation', 'protein disorder'], 'includedInDataCatalog': 'http://mobidb.bio.unipd.it', 'creator': {'@type': 'Organization', 'name': 'University of Padua, Department of Biomedical Sciences, BioComputing UP', 'url': 'http://protein.bio.unipd.it'}, 'version': '2018-01', 'license': {'@type': 'CreativeWork', 'name': 'Creative Commons CC4 Attribution', 'url': 'https://creativecommons.org/licenses/by/4.0/'}, 'distribution': [{'@type': 'DataDownload', 'name': 'Indirect disorder', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/pdb/indirect.json'}, {'@type': 'DataDownload', 'name': 'Indirect LIPs', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/pdb/lips.json'}, {'@type': 'DataDownload', 'name': 'Manually curated disorder', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/mobidb/curated.json'}, {'@type': 'DataDownload', 'name': 'Manually cuarted LIPs', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/mobidb/lips.json'}, {'@type': 'DataDownload', 'name': 'SwissProt disorder', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/swissprot.json'}, {'@type': 'DataDownload', 'name': 'Indirect dynamic structure', 'encodingFormat': 'application/json', 'contentURL': 'http://mobidb.bio.unipd.it/ws/dataset/bmrb.json'}]}]}]}, {'prefix': 'imgt.hla', 'mirId': 'MIR:00000331', 'name': 'IMGT HLA', 'pattern': '^[A-Z0-9*:]+$', 'description': 'IMGT, the international ImMunoGeneTics project, is a collection of high-quality integrated databases specialising in Immunoglobulins, T cell receptors and the Major Histocompatibility Complex (MHC) of all vertebrate species. IMGT/HLA is a database for sequences of the human MHC, referred to as HLA. It includes all the official sequences for the WHO Nomenclature Committee For Factors of the HLA System. This collection references allele information through the WHO nomenclature.', 'created': '2019-06-11T14:16:54.535+0000', 'modified': '2019-06-11T14:16:54.535+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'A*01:01:01:01', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/950'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/950'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/950/contactPerson'}}, 'metadata': []}, {'prefix': 'bdsc', 'mirId': 'MIR:00000332', 'name': 'Bloomington Drosophila Stock Center', 'pattern': '^\\\\d+$', 'description': 'The Bloomington Drosophila Stock Center collects, maintains and distributes Drosophila melanogaster strains for research.', 'created': '2019-06-11T14:16:54.741+0000', 'modified': '2019-06-11T14:16:54.741+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '33607', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/952'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/952'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/952/contactPerson'}}, 'metadata': []}, {'prefix': 'opm', 'mirId': 'MIR:00000333', 'name': 'OPM', 'pattern': '^[0-9][A-Za-z0-9]{3}$', 'description': 'The Orientations of Proteins in Membranes (OPM) database provides spatial positions of membrane-bound peptides and proteins of known three-dimensional structure in the lipid bilayer, together with their structural classification, topology and intracellular localization.', 'created': '2019-06-11T14:16:54.970+0000', 'modified': '2019-06-11T14:16:54.970+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1h68', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/955'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/955'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/955/contactPerson'}}, 'metadata': []}, {'prefix': 'allergome', 'mirId': 'MIR:00000334', 'name': 'Allergome', 'pattern': '^\\\\d+$', 'description': 'Allergome is a repository of data related to all IgE-binding compounds. Its purpose is to collect a list of allergenic sources and molecules by using the widest selection criteria and sources.', 'created': '2019-06-11T14:16:55.214+0000', 'modified': '2019-06-11T14:16:55.214+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1948', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/958'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/958'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/958/contactPerson'}}, 'metadata': []}, {'prefix': 'pombase', 'mirId': 'MIR:00000335', 'name': 'PomBase', 'pattern': '^S\\\\w+(\\\\.)?\\\\w+(\\\\.)?$', 'description': 'PomBase is a model organism database established to provide access to molecular data and biological information for the fission yeast Schizosaccharomyces pombe. It encompasses annotation of genomic sequence and features, comprehensive manual literature curation and genome-wide data sets.', 'created': '2019-06-11T14:16:55.438+0000', 'modified': '2019-06-11T14:16:55.438+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SPCC13B11.01', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/961'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/961'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/961/contactPerson'}}, 'metadata': []}, {'prefix': 'hpa', 'mirId': 'MIR:00000336', 'name': 'HPA', 'pattern': '^ENSG\\\\d{11}$', 'description': 'The Human Protein Atlas (HPA) is a publicly available database with high-resolution images showing the spatial distribution of proteins in different normal and cancer human cell lines. Primary access to this collection is through Ensembl Gene identifiers.', 'created': '2019-06-11T14:16:55.662+0000', 'modified': '2019-06-11T14:16:55.662+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ENSG00000026508', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/964'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/964'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/964/contactPerson'}}, 'metadata': []}, {'prefix': 'jaxmice', 'mirId': 'MIR:00000337', 'name': 'JAX Mice', 'pattern': '^\\\\d+$', 'description': 'JAX Mice is a catalogue of mouse strains supplied by the Jackson Laboratory.', 'created': '2019-06-11T14:16:55.877+0000', 'modified': '2019-06-11T14:16:55.877+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '005012', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/967'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/967'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/967/contactPerson'}}, 'metadata': []}, {'prefix': 'ubio.namebank', 'mirId': 'MIR:00000338', 'name': 'uBio NameBank', 'pattern': '^\\\\d+$', 'description': 'NameBank is a \"biological name server\" focused on storing names and objectively-derived nomenclatural attributes. NameBank is a repository for all recorded names including scientific names, vernacular (or common names), misspelled names, as well as ad-hoc nomenclatural labels that may have limited context.', 'created': '2019-06-11T14:16:56.122+0000', 'modified': '2019-06-11T14:16:56.122+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2555646', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/970'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/970'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/970/contactPerson'}}, 'metadata': []}, {'prefix': 'yetfasco', 'mirId': 'MIR:00000339', 'name': 'YeTFasCo', 'pattern': '^\\\\w+\\\\_\\\\d+(\\\\.\\\\d+)?$', 'description': 'The Yeast Transcription Factor Specificity Compendium (YeTFasCO) is a database of transcription factor specificities for the yeast Saccharomyces cerevisiae in Position Frequency Matrix (PFM) or Position Weight Matrix (PWM) formats.', 'created': '2019-06-11T14:16:56.366+0000', 'modified': '2019-06-11T14:16:56.366+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'YOR172W_571.0', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/973'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/973'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/973/contactPerson'}}, 'metadata': []}, {'prefix': 'tarbase', 'mirId': 'MIR:00000340', 'name': 'TarBase', 'pattern': '^[a-z]{3}\\\\-(mir|let|lin)\\\\-\\\\w+(\\\\-\\\\w+\\\\-\\\\w+)?', 'description': 'TarBase stores microRNA (miRNA) information for miRNA‚Äìgene interactions, as well as miRNA- and gene-related facts to information specific to the interaction and the experimental validation methodologies used.', 'created': '2019-06-11T14:16:56.599+0000', 'modified': '2019-06-11T14:16:56.599+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'hsa-let-7a-2-3p', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/976'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/976'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/976/contactPerson'}}, 'metadata': []}, {'prefix': 'charprot', 'mirId': 'MIR:00000341', 'name': 'CharProt', 'pattern': '^CH_\\\\d+$', 'description': 'CharProt is a database of biochemically characterized proteins designed to support automated annotation pipelines. Entries are annotated with gene name, symbol and various controlled vocabulary terms, including Gene Ontology terms, Enzyme Commission number and TransportDB accession.', 'created': '2019-06-11T14:16:56.827+0000', 'modified': '2019-06-11T14:16:56.827+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CH_001923', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/979'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/979'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/979/contactPerson'}}, 'metadata': []}, {'prefix': 'oma.protein', 'mirId': 'MIR:00000342', 'name': 'OMA Protein', 'pattern': '^[A-Z0-9]{5}\\\\d+$', 'description': 'OMA (Orthologous MAtrix) is a database that identifies orthologs among publicly available, complete genome sequences. It identifies orthologous relationships which can be accessed either group-wise, where all group members are orthologous to all other group members, or on a sequence-centric basis, where for a given protein all its orthologs in all other species are displayed. This collection references individual protein records.', 'created': '2019-06-11T14:16:57.058+0000', 'modified': '2019-06-11T14:16:57.058+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HUMAN16963', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/982'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/982'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/982/contactPerson'}}, 'metadata': []}, {'prefix': 'oma.grp', 'mirId': 'MIR:00000343', 'name': 'OMA Group', 'pattern': '^[A-Z]+$', 'description': 'OMA (Orthologous MAtrix) is a database that identifies orthologs among publicly available, complete genome sequences. It identifies orthologous relationships which can be accessed either group-wise, where all group members are orthologous to all other group members, or on a sequence-centric basis, where for a given protein all its orthologs in all other species are displayed. This collection references groupings of orthologs.', 'created': '2019-06-11T14:16:57.285+0000', 'modified': '2019-06-11T14:16:57.285+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LCSCCPN', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/985'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/985'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/985/contactPerson'}}, 'metadata': []}, {'prefix': 'ncbiprotein', 'mirId': 'MIR:00000344', 'name': 'NCBI Protein', 'pattern': '^(\\\\w+\\\\d+(\\\\.\\\\d+)?)|(NP_\\\\d+)$', 'description': 'The Protein database is a collection of sequences from several sources, including translations from annotated coding regions in GenBank, RefSeq and TPA, as well as records from SwissProt, PIR, PRF, and PDB.', 'created': '2019-06-11T14:16:57.477+0000', 'modified': '2019-06-11T14:16:57.477+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CAA71118.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/987'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/987'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/987/contactPerson'}}, 'metadata': []}, {'prefix': 'genpept', 'mirId': 'MIR:00000345', 'name': 'GenPept', 'pattern': '^\\\\w{3}\\\\d{5}(\\\\.\\\\d+)?$', 'description': 'The GenPept database is a collection of sequences based on translations from annotated coding regions in GenBank.', 'created': '2019-06-11T14:16:57.671+0000', 'modified': '2019-06-11T14:16:57.671+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CAA71118.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/989'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/989'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/989/contactPerson'}}, 'metadata': []}, {'prefix': 'unigene', 'mirId': 'MIR:00000346', 'name': 'UniGene', 'pattern': '^\\\\d+$', 'description': 'A UniGene entry is a set of transcript sequences that appear to come from the same transcription locus (gene or expressed pseudogene), together with information on protein similarities, gene expression, cDNA clone reagents, and genomic location.', 'created': '2019-06-11T14:16:57.856+0000', 'modified': '2019-06-11T14:16:57.856+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4900', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/991'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/991'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/991/contactPerson'}}, 'metadata': []}, {'prefix': 'bitterdb.rec', 'mirId': 'MIR:00000347', 'name': 'BitterDB Receptor', 'pattern': '^\\\\d+$', 'description': 'BitterDB is a database of compounds reported to taste bitter to humans. The compounds can be searched by name, chemical structure, similarity to other bitter compounds, association with a particular human bitter taste receptor, and so on. The database also contains information on mutations in bitter taste receptors that were shown to influence receptor activation by bitter compounds. The aim of BitterDB is to facilitate studying the chemical features associated with bitterness. This collection references receptors.', 'created': '2019-06-11T14:16:58.089+0000', 'modified': '2019-06-11T14:16:58.089+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/994'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/994'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/994/contactPerson'}}, 'metadata': []}, {'prefix': 'bitterdb.cpd', 'mirId': 'MIR:00000348', 'name': 'BitterDB Compound', 'pattern': '^\\\\d+$', 'description': 'BitterDB is a database of compounds reported to taste bitter to humans. The compounds can be searched by name, chemical structure, similarity to other bitter compounds, association with a particular human bitter taste receptor, and so on. The database also contains information on mutations in bitter taste receptors that were shown to influence receptor activation by bitter compounds. The aim of BitterDB is to facilitate studying the chemical features associated with bitterness. This collection references compounds.', 'created': '2019-06-11T14:16:58.347+0000', 'modified': '2019-06-11T14:16:58.347+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '46', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/997'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/997'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/997/contactPerson'}}, 'metadata': []}, {'prefix': 'bioproject', 'mirId': 'MIR:00000349', 'name': 'BioProject', 'pattern': '^PRJ[DEN][A-Z]\\\\d+$', 'description': 'BioProject provides an organizational framework to access metadata about research projects and the data from the projects that are deposited into different databases. It provides information about a project‚Äôs scope, material, objectives, funding source and general relevance categories.', 'created': '2019-06-11T14:16:58.543+0000', 'modified': '2019-06-11T14:16:58.543+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PRJDB3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/999'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/999'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/999/contactPerson'}}, 'metadata': []}, {'prefix': 'biosample', 'mirId': 'MIR:00000350', 'name': 'BioSample', 'pattern': '^SAM[NED](\\\\w)?\\\\d+$', 'description': 'The BioSample Database stores information about biological samples used in molecular experiments, such as sequencing, gene expression or proteomics. It includes reference samples, such as cell lines, which are repeatedly used in experiments. Accession numbers for the reference samples will be exchanged with a similar database at NCBI, and DDBJ (Japan). Record access may be affected due to different release cycles and inter-institutional synchronisation.', 'created': '2019-06-11T14:16:59.017+0000', 'modified': '2019-06-11T14:16:59.017+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SAMEA2397676', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1004'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1004'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1004/contactPerson'}}, 'metadata': [{'@context': ['http://schema.org', {'OBI': 'http://purl.obolibrary.org/obo/OBI_', 'biosample': 'http://identifiers.org/biosample/'}], '@type': 'DataRecord', '@id': 'biosample:SAMEA2397676', 'identifier': 'biosample:SAMEA2397676', 'dateCreated': '2014-03-03T16:32:10Z', 'dateModified': '2019-07-24T16:17:02.748Z', 'mainEntity': {'@type': ['Sample', 'OBI:0000747'], 'identifier': ['biosample:SAMEA2397676'], 'name': 'ERS368772', 'subjectOf': ['https://www.ebi.ac.uk/ena/data/view/SAMEA2397676'], 'additionalProperty': [{'@type': 'PropertyValue', 'name': 'ENA checklist', 'value': 'ERC000011'}, {'@type': 'PropertyValue', 'name': 'INSDC center alias', 'value': 'FEM'}, {'@type': 'PropertyValue', 'name': 'INSDC center name', 'value': 'IASMA Research Centre Fondazione Edmund Mach FEM'}, {'@type': 'PropertyValue', 'name': 'INSDC first public', 'value': '2014-03-03T16:32:10Z'}, {'@type': 'PropertyValue', 'name': 'INSDC last update', 'value': '2013-11-08T13:09:45Z'}, {'@type': 'PropertyValue', 'name': 'INSDC status', 'value': 'public'}, {'@type': 'PropertyValue', 'name': 'SRA accession', 'value': 'ERS368772'}, {'@type': 'PropertyValue', 'name': 'alias', 'value': 'CheeseA_24h9'}, {'@type': 'PropertyValue', 'name': 'organism', 'value': 'food metagenome', 'valueReference': [{'@id': 'http://purl.obolibrary.org/obo/NCBITaxon_870726', '@type': 'DefinedTerm'}]}, {'@type': 'PropertyValue', 'name': 'synonym', 'value': 'CheeseA_24h9'}, {'@type': 'PropertyValue', 'name': 'title', 'value': 'CheeseA Period 3 24h ripening'}], 'sameAs': 'http://identifiers.org/biosample/SAMEA2397676'}, 'isPartOf': {'@type': 'Dataset', '@id': 'https://www.ebi.ac.uk/biosamples/samples'}}]}, {'prefix': 'piroplasma', 'mirId': 'MIR:00000351', 'name': 'PiroplasmaDB', 'pattern': '^TA\\\\d+$', 'description': 'PiroplasmaDB is one of the databases that can be accessed through the EuPathDB (http://EuPathDB.org; formerly ApiDB) portal, covering eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.', 'created': '2019-06-11T14:16:59.494+0000', 'modified': '2019-06-11T14:16:59.494+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TA14985', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1009'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1009'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1009/contactPerson'}}, 'metadata': []}, {'prefix': 'unite', 'mirId': 'MIR:00000352', 'name': 'Unite', 'pattern': '^UDB\\\\d{6}$', 'description': 'UNITE is a fungal rDNA internal transcribed spacer (ITS) sequence database. It focuses on high-quality ITS sequences generated from fruiting bodies collected and identified by experts and deposited in public herbaria. Entries may be supplemented with metadata on describing locality, habitat, soil, climate, and interacting taxa.', 'created': '2019-06-11T14:16:59.723+0000', 'modified': '2019-06-11T14:16:59.723+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'UDB000691', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1012'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1012'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1012/contactPerson'}}, 'metadata': []}, {'prefix': 'ncim', 'mirId': 'MIR:00000353', 'name': 'NCIm', 'pattern': '^C\\\\d+$', 'description': 'NCI Metathesaurus (NCIm) is a wide-ranging biomedical terminology database that covers most terminologies used by NCI for clinical care, translational and basic research, and public information and administrative activities. It integrates terms and definitions from different terminologies, including NCI Thesaurus, however the representation is not identical.', 'created': '2019-06-11T14:16:59.945+0000', 'modified': '2019-06-11T14:16:59.945+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C0026339', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1015'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1015'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1015/contactPerson'}}, 'metadata': []}, {'prefix': 'proglyc', 'mirId': 'MIR:00000354', 'name': 'ProGlycProt', 'pattern': '^[A-Z]C\\\\d{1,3}$', 'description': 'ProGlycProt (Prokaryotic Glycoprotein) is a repository of bacterial and archaeal glycoproteins with at least one experimentally validated glycosite (glycosylated residue). Each entry in the database is fully cross-referenced and enriched with available published information about source organism, coding gene, protein, glycosites, glycosylation type, attached glycan, associated oligosaccharyl/glycosyl transferases (OSTs/GTs), supporting references, and applicable additional information.', 'created': '2019-06-11T14:17:00.162+0000', 'modified': '2019-06-11T14:17:00.162+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AC119', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1017'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1017'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1017/contactPerson'}}, 'metadata': []}, {'prefix': 'polbase', 'mirId': 'MIR:00000355', 'name': 'PolBase', 'pattern': '^[A-Za-z-0-9]+$', 'description': 'Polbase is a database of DNA polymerases providing information on polymerase protein sequence, target DNA sequence, enzyme structure, sequence mutations and details on polymerase activity.', 'created': '2019-06-11T14:17:00.403+0000', 'modified': '2019-06-11T14:17:00.403+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '19-T4', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1020'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1020'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1020/contactPerson'}}, 'metadata': []}, {'prefix': 'nuclearbd', 'mirId': 'MIR:00000356', 'name': 'NucleaRDB', 'pattern': '^\\\\w+\\\\_\\\\w+$', 'description': 'NucleaRDB is an information system that stores heterogenous data on Nuclear Hormone Receptors (NHRs). It contains data on sequences, ligand binding constants and mutations for NHRs.', 'created': '2019-06-11T14:17:00.593+0000', 'modified': '2019-06-11T14:17:00.593+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'prgr_human', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1022'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1022'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1022/contactPerson'}}, 'metadata': []}, {'prefix': 'supfam', 'mirId': 'MIR:00000357', 'name': 'SUPFAM', 'pattern': '^\\\\w+$', 'description': 'SUPERFAMILY provides structural, functional and evolutionary information for proteins from all completely sequenced genomes, and large sequence collections such as UniProt.', 'created': '2019-06-11T14:17:00.815+0000', 'modified': '2019-06-11T14:17:00.815+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SSF57615', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1025'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1025'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1025/contactPerson'}}, 'metadata': []}, {'prefix': 'ricegap', 'mirId': 'MIR:00000358', 'name': 'Rice Genome Annotation Project', 'pattern': '^LOC\\\\_Os\\\\d{1,2}g\\\\d{5}$', 'description': 'The objective of this project is to provide high quality annotation for the rice genome Oryza sativa spp japonica cv Nipponbare. All genes are annotated with functional annotation including expression data, gene ontologies, and tagged lines.', 'created': '2019-06-11T14:17:01.041+0000', 'modified': '2019-06-11T14:17:01.041+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LOC_Os02g13300', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1028'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1028'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1028/contactPerson'}}, 'metadata': []}, {'prefix': 'pina', 'mirId': 'MIR:00000359', 'name': 'PINA', 'pattern': '^([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])$', 'description': 'Protein Interaction Network Analysis (PINA) platform is an integrated platform for protein interaction network construction, filtering, analysis, visualization and management. It integrates protein-protein interaction data from six public curated databases and builds a complete, non-redundant protein interaction dataset for six model organisms.', 'created': '2019-06-11T14:17:01.276+0000', 'modified': '2019-06-11T14:17:01.276+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q13485', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1030'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1030'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1030/contactPerson'}}, 'metadata': []}, {'prefix': 'tissuelist', 'mirId': 'MIR:00000360', 'name': 'Tissue List', 'pattern': '^TS-\\\\d{4}$', 'description': 'The UniProt Tissue List is a controlled vocabulary of terms used to annotate biological tissues. It also contains cross-references to other ontologies where tissue types are specified.', 'created': '2019-06-11T14:17:01.500+0000', 'modified': '2019-06-11T14:17:01.500+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TS-0285', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1033'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1033'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1033/contactPerson'}}, 'metadata': []}, {'prefix': 'bacmap.biog', 'mirId': 'MIR:00000361', 'name': 'BacMap Biography', 'pattern': '^\\\\d+$', 'description': \"BacMap is an electronic, interactive atlas of fully sequenced bacterial genomes. It contains labeled, zoomable and searchable chromosome maps for sequenced prokaryotic (archaebacterial and eubacterial) species. Each map can be zoomed to the level of individual genes and each gene is hyperlinked to a richly annotated gene card. All bacterial genome maps are supplemented with separate prophage genome maps as well as separate tRNA and rRNA maps. Each bacterial chromosome entry in BacMap contains graphs and tables on a variety of gene and protein statistics. Likewise, every bacterial species entry contains a bacterial 'biography' card, with taxonomic details, phenotypic details, textual descriptions and images. This collection references 'biography' information.\", 'created': '2019-06-11T14:17:01.735+0000', 'modified': '2019-06-11T14:17:01.735+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1050', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1036'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1036'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1036/contactPerson'}}, 'metadata': []}, {'prefix': 'hgnc.symbol', 'mirId': 'MIR:00000362', 'name': 'HGNC Symbol', 'pattern': '^[A-Za-z-0-9_]+(\\\\@)?$', 'description': 'The HGNC (HUGO Gene Nomenclature Committee) provides an approved gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database, and each symbol is unique. This collection refers to records using the HGNC symbol.', 'created': '2019-06-11T14:17:01.955+0000', 'modified': '2019-06-11T14:17:01.955+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DAPK1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1039'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1039'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1039/contactPerson'}}, 'metadata': []}, {'prefix': 'panther.pathway', 'mirId': 'MIR:00000363', 'name': 'PANTHER Pathway', 'pattern': '^P\\\\d{5}$', 'description': 'The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System is a resource that classifies genes by their functions, using published scientific experimental evidence and evolutionary relationships to predict function even in the absence of direct experimental evidence. The PANTHER Pathway collection references pathway information, primarily for signaling pathways, each with subfamilies and protein sequences mapped to individual pathway components.', 'created': '2019-06-11T14:17:02.153+0000', 'modified': '2019-06-11T14:17:02.153+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P00024', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1041'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1041'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1041/contactPerson'}}, 'metadata': []}, {'prefix': 'fairsharing', 'mirId': 'MIR:00000364', 'name': 'FAIRsharing', 'pattern': '^bsg-[dscp]?\\\\d{6}$', 'description': \"The web-based FAIRSharing catalogues aim to centralize bioscience data policies, reporting standards and links to other related portals. This collection references bioinformatics data exchange standards, which includes 'Reporting Guidelines', Format Specifications and Terminologies.\", 'created': '2019-06-11T14:17:02.358+0000', 'modified': '2019-06-11T14:17:02.358+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'bsg-000052', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1043'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1043'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1043/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@id': 'https://doi.org/10.25504/FAIRsharing.9qv71f', '@type': 'Dataset', 'alternateName': 'SBML', 'citation': {'@type': 'CreativeWork', 'identifier': 'doi:10.25504/FAIRsharing.9qv71f'}, 'datePublished': '2015-05-14 11:14:01', 'description': 'The Systems Biology Markup Language (SBML) is a machine-readable exchange format for computational models of biological processes. Its strength is in representating phenomena at the scale of biochemical reactions, but it is not limited to that. By supporting SBML as an input and output format, different software tools can operate on the same representation of a model, removing chances for errors in translation and assuring a common starting point for analyses and simulations.', 'identifier': '10.25504/FAIRsharing.9qv71f', 'license': 'https://creativecommons.org/licenses/by-sa/4.0/. Please link to https://fairsharing.org and https://fairsharing.org/static/img/home/svg/FAIRsharing-logo.svg for attribution.', 'name': 'Systems Biology Markup Language', 'url': 'https://doi.org/10.25504/FAIRsharing.9qv71f'}]}, {'prefix': 'fungidb', 'mirId': 'MIR:00000365', 'name': 'FungiDB', 'pattern': '^[A-Za-z_0-9]+$', 'description': \"FungiDB is a genomic resource for fungal genomes. It contains contains genome sequence and annotation from several fungal classes, including the Ascomycota classes, Eurotiomycetes, Sordariomycetes, Saccharomycetes and the Basidiomycota orders, Pucciniomycetes and Tremellomycetes, and the basal 'Zygomycete' lineage Mucormycotina.\", 'created': '2019-06-11T14:17:02.591+0000', 'modified': '2019-06-11T14:17:02.591+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CNBG_0001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1046'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1046'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1046/contactPerson'}}, 'metadata': []}, {'prefix': 'neurovault.image', 'mirId': 'MIR:00000638', 'name': 'NeuroVault Image', 'pattern': '^[1-9][0-9]*$', 'description': 'Neurovault is an online repository for statistical maps, parcellations and atlases of the brain. This collection references individual images.', 'created': '2019-06-11T14:18:08.473+0000', 'modified': '2019-06-11T14:18:08.473+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '58788', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1771'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1771'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1771/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org', '@type': 'Organization', 'name': 'NeuroVault', 'url': 'https://neurovault.org', 'logo': 'https://neurovault.org/static/images/neurovault-logo.png', 'sameAs': ['https://en.wikipedia.org/wiki/NeuroVault', 'https://twitter.com/vaultneuro']}, {'@context': 'https://schema.org', '@type': 'WebSite', 'url': 'https://neurovault.org/', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://neurovault.org/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}, {'@context': 'http://schema.org/', '@type': 'Dataset', '@id': 'https://neurovault.org/images/58788', 'url': 'https://neurovault.org/images/58788', 'identifier': ['https://identifiers.org/neurovault.image:58788', 'https://neurovault.org/collections/OELFQSHF/images/58788'], 'name': 'Linked networks for learning and expressing location-specific threat: Threat learning versus object learning during approach periods', 'description': '<p>Activity differences between approaching flowers versus objects during threat and spatial memory aspects of the task. Greater activity was seen when approaching flowers compared with objects in a range of areas, including the insula, medial parietal cortex, PCC, vmPFC, bilateral anterior hippocampus, and amygdala.When approaching objects, a different network was observed with greater activity in the left middle frontal gyrus, middle temporal gyrus, inferior parietal lobule and inferior frontal gyrus, right precentral gyrus, inferior frontal gyrus, and an area extending across lingual and parahippocampal gyri.</p>\\n\\n[![glassbrain](https://neurovault.org/media/images/3303/glass_brain_58788_1.jpg)](https://neurovault.org/images/58788)\\n\\n### Collection description\\n\\n<p>Statistical images from the study in: <br />Suarez-Jimenez B, Bisby JA, Horner AJ, King JA, Pine DS, Burgess N (2018) Linked networks for learning and expressing location-specific threat. PNAS. In press.</p>\\n\\n###Subject species\\n\\nhomo sapiens\\n\\n###Modality\\n\\n fMRI-BOLD\\n\\n### Analysis level\\n\\ngroup\\n\\n### Cognitive paradigm (task)\\n\\nnavigation task\\n\\n### Map type\\n\\nT\\n\\n', 'dateCreated': '2018-01-08T18:52:50', 'dateModified': '2018-06-30T11:08:23', 'keywords': ['neuroscience', 'neuroimaging'], 'includedInDataCatalog': {'@type': 'DataCatalog', 'name': 'NeuroVault', 'url': 'https://neurovault.org'}, 'license': {'@type': 'CreativeWork', 'name': 'CC0 1.0', 'url': 'https://creativecommons.org/publicdomain/zero/1.0/'}, 'distribution': [{'@type': 'DataDownload', 'encodingFormat': 'NIfTI', 'contentUrl': 'https://neurovault.org//media/images/3303/spmT_0007.nii.gz'}], 'citation': 'https://doi.org/10.1073/pnas.1714691115'}]}, {'prefix': 'darc', 'mirId': 'MIR:00000366', 'name': 'DARC', 'pattern': '^\\\\d+$', 'description': 'DARC (Database of Aligned Ribosomal Complexes) stores available cryo-EM (electron microscopy) data and atomic coordinates of ribosomal particles from the PDB, which are aligned within a common coordinate system. The aligned coordinate system simplifies direct visualization of conformational changes in the ribosome, such as subunit rotation and head-swiveling, as well as direct comparison of bound ligands, such as antibiotics or translation factors.', 'created': '2019-06-11T14:17:02.814+0000', 'modified': '2019-06-11T14:17:02.814+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1250', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1049'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1049'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1049/contactPerson'}}, 'metadata': []}, {'prefix': 'drsc', 'mirId': 'MIR:00000367', 'name': 'DRSC', 'pattern': '^DRSC\\\\d+$', 'description': 'The DRSC (Drosophila RNAi Screening Cente) tracks both production of reagents for RNA interference (RNAi) screening in Drosophila cells and RNAi screen results. It maintains a list of Drosophila gene names, identifiers, symbols and synonyms and provides  information for cell-based or in vivo RNAi reagents, other types of reagents, screen results, etc. corresponding for a given gene.', 'created': '2019-06-11T14:17:03.033+0000', 'modified': '2019-06-11T14:17:03.033+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DRSC05221', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1052'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1052'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1052/contactPerson'}}, 'metadata': []}, {'prefix': 'oridb.schizo', 'mirId': 'MIR:00000368', 'name': 'OriDB Schizosaccharomyces', 'pattern': '^\\\\d+$', 'description': 'OriDB is a database of collated genome-wide mapping studies of confirmed and predicted replication origin sites in Saccharomyces cerevisiae and the fission yeast Schizosaccharomyces pombe. This collection references Schizosaccharomyces pombe.', 'created': '2019-06-11T14:17:03.255+0000', 'modified': '2019-06-11T14:17:03.255+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1055'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1055'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1055/contactPerson'}}, 'metadata': []}, {'prefix': 'oridb.sacch', 'mirId': 'MIR:00000369', 'name': 'OriDB Saccharomyces', 'pattern': '^\\\\d+$', 'description': 'OriDB is a database of collated genome-wide mapping studies of confirmed and predicted replication origin sites in Saccharomyces cerevisiae and the fission yeast Schizosaccharomyces pombe. This collection references Saccharomyces cerevisiae.', 'created': '2019-06-11T14:17:03.475+0000', 'modified': '2019-06-11T14:17:03.475+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1058'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1058'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1058/contactPerson'}}, 'metadata': []}, {'prefix': 'pscdb', 'mirId': 'MIR:00000370', 'name': 'PSCDB', 'pattern': '^\\\\d+$', 'description': 'The PSCDB (Protein Structural Change DataBase) collects information on the relationship between protein structural change upon ligand binding. Each entry page provides detailed information about this structural motion.', 'created': '2019-06-11T14:17:03.650+0000', 'modified': '2019-06-11T14:17:03.650+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '051', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1060'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1060'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1060/contactPerson'}}, 'metadata': []}, {'prefix': 'scop', 'mirId': 'MIR:00000371', 'name': 'SCOP', 'pattern': '^\\\\d+$', 'description': 'The SCOP (Structural Classification of Protein) database is a comprehensive ordering of all proteins of known structure according to their evolutionary, functional and structural relationships. The basic classification unit is the protein domain. Domains are hierarchically classified into species, proteins, families, superfamilies, folds, and classes.', 'created': '2019-06-11T14:17:03.870+0000', 'modified': '2019-06-11T14:17:03.870+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '47419', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1063'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1063'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1063/contactPerson'}}, 'metadata': []}, {'prefix': 'ena.embl', 'mirId': 'MIR:00000372', 'name': 'ENA', 'pattern': '^[A-Z]+[0-9]+(\\\\.\\\\d+)?$', 'description': 'The European Nucleotide Archive (ENA) captures and presents information relating to experimental workflows that are based around nucleotide sequencing. ENA is made up of a number of distinct databases that includes EMBL-Bank, the Sequence Read Archive (SRA) and the Trace Archive each with their own data formats and standards. This collection references Embl-Bank identifiers.', 'created': '2019-06-11T14:17:04.223+0000', 'modified': '2019-06-11T14:17:04.223+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BN000065', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1068'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1068'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1068/contactPerson'}}, 'metadata': []}, {'prefix': 'dommino', 'mirId': 'MIR:00000373', 'name': 'DOMMINO', 'pattern': '^[0-9][A-Za-z0-9]{3}$', 'description': 'DOMMINO is a database of macromolecular interactions that includes the interactions between protein domains, interdomain linkers, N- and C-terminal regions and protein peptides.', 'created': '2019-06-11T14:17:04.514+0000', 'modified': '2019-06-11T14:17:04.514+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2GC4', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1071'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1071'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1071/contactPerson'}}, 'metadata': []}, {'prefix': 'panther.node', 'mirId': 'MIR:00000374', 'name': 'PANTHER Node', 'pattern': '^PTN\\\\d{9}$', 'description': 'The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System is a resource that classifies genes by their functions, using published scientific experimental evidence and evolutionary relationships to predict function even in the absence of direct experimental evidence. PANTHER tree is a key element of the PANTHER System to represent ‚Äòall‚Äô of the evolutionary events in the gene family. PANTHER nodes represent the evolutionary events, either speciation or duplication, within the tree. PANTHER is maintaining stable identifier for these nodes.', 'created': '2019-06-11T14:17:04.735+0000', 'modified': '2019-06-11T14:17:04.735+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PTN000000026', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1074'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1074'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1074/contactPerson'}}, 'metadata': []}, {'prefix': 'ccds', 'mirId': 'MIR:00000375', 'name': 'Consensus CDS', 'pattern': '^CCDS\\\\d+\\\\.\\\\d+$', 'description': 'The Consensus CDS (CCDS) project is a collaborative effort to identify a core set of human and mouse protein coding regions that are consistently annotated and of high quality. The CCDS set is calculated following coordinated whole genome annotation updates carried out by the NCBI, WTSI, and Ensembl. The long term goal is to support convergence towards a standard set of gene annotations.', 'created': '2019-06-11T14:17:04.925+0000', 'modified': '2019-06-11T14:17:04.925+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CCDS13573.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1076'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1076'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1076/contactPerson'}}, 'metadata': []}, {'prefix': 'lrg', 'mirId': 'MIR:00000376', 'name': 'Locus Reference Genomic', 'pattern': '^LRG_\\\\d+$', 'description': 'Locus Reference Genomic (LRG) provides identifiers to stable genomic DNA sequences for regions of the human genome, providing a recognized reference-sequence standard for reporting sequence variants. LRG is maintained by the NCBI and the European Bioinformatics Institute (EBI).', 'created': '2019-06-11T14:17:05.115+0000', 'modified': '2019-06-11T14:17:05.115+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LRG_1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1078'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1078'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1078/contactPerson'}}, 'metadata': []}, {'prefix': 'hprd', 'mirId': 'MIR:00000377', 'name': 'HPRD', 'pattern': '^\\\\d+$', 'description': 'The Human Protein Reference Database (HPRD) represents a centralized platform to visually depict and integrate information pertaining to domain architecture, post-translational modifications, interaction networks and disease association for each protein in the human proteome.', 'created': '2019-06-11T14:17:05.741+0000', 'modified': '2019-06-11T14:17:05.741+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1086'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1086'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1086/contactPerson'}}, 'metadata': []}, {'prefix': 'gxa.gene', 'mirId': 'MIR:00000378', 'name': 'GXA Gene', 'pattern': '^\\\\w+$', 'description': 'The Gene Expression Atlas (GXA) is a semantically enriched database of meta-analysis based summary statistics over a curated subset of ArrayExpress Archive, servicing queries for condition-specific gene expression patterns as well as broader exploratory searches for biologically interesting genes/samples. This collection references genes.', 'created': '2019-06-11T14:17:05.964+0000', 'modified': '2019-06-11T14:17:05.964+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AT4G01080', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1089'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1089'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1089/contactPerson'}}, 'metadata': []}, {'prefix': 'gxa.expt', 'mirId': 'MIR:00000379', 'name': 'GXA Expt', 'pattern': '^[AEP]-\\\\w{4}-\\\\d+$', 'description': 'The Gene Expression Atlas (GXA) is a semantically enriched database of meta-analysis based summary statistics over a curated subset of ArrayExpress Archive, servicing queries for condition-specific gene expression patterns as well as broader exploratory searches for biologically interesting genes/samples. This collection references experiments.', 'created': '2019-06-11T14:17:06.163+0000', 'modified': '2019-06-11T14:17:06.163+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'E-MTAB-2037', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1091'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1091'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1091/contactPerson'}}, 'metadata': []}, {'prefix': 'metabolights', 'mirId': 'MIR:00000380', 'name': 'MetaboLights', 'pattern': '^MTBLS\\\\d+$', 'description': 'MetaboLights is a database for Metabolomics experiments and derived information. The database is cross-species, cross-technique and covers metabolite structures and their reference spectra as well as their biological roles, locations and concentrations, and experimental data from metabolic experiments. This collection references individual metabolomics studies.', 'created': '2019-06-11T14:17:06.457+0000', 'modified': '2019-06-11T14:17:06.457+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MTBLS1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1094'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1094'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1094/contactPerson'}}, 'metadata': []}, {'prefix': 'nbn', 'mirId': 'MIR:00000381', 'name': 'National Bibliography Number', 'pattern': '^urn\\\\:nbn\\\\:[A-Za-z_0-9]+\\\\:([A-Za-z_0-9]+)-[A-Za-z_0-9]+$', 'description': 'The National Bibliography Number (NBN), is a URN-based publication identifier system employed by a variety of national libraries such as those of Germany, the Netherlands and Switzerland.  They are used to identify documents archived in national libraries, in their native format or language, and are typically used for documents which do not have a publisher-assigned identifier.', 'created': '2019-06-11T14:17:06.748+0000', 'modified': '2019-06-11T14:17:06.748+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'urn:nbn:fi:tkk-004781', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1097'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1097'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1097/contactPerson'}}, 'metadata': []}, {'prefix': 'orcid', 'mirId': 'MIR:00000382', 'name': 'ORCID', 'pattern': '^\\\\d{4}-\\\\d{4}-\\\\d{4}-\\\\d{3}(\\\\d|X)$', 'description': 'ORCID (Open Researcher and Contributor ID) is an open, non-profit, community-based effort to create and maintain a registry of unique identifiers for individual researchers. ORCID records hold non-sensitive information such as name, email, organization name, and research activities.', 'created': '2019-06-11T14:17:06.994+0000', 'modified': '2019-06-11T14:17:06.994+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000-0002-5355-2576', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1100'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1100'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1100/contactPerson'}}, 'metadata': []}, {'prefix': 'inchi', 'mirId': 'MIR:00000383', 'name': 'InChI', 'pattern': '^InChI\\\\=1S?\\\\/[A-Za-z0-9\\\\.]+(\\\\+[0-9]+)?(\\\\/[cnpqbtmsih][A-Za-z0-9\\\\-\\\\+\\\\(\\\\)\\\\,\\\\/\\\\?\\\\;\\\\.]+)*$', 'description': 'The IUPAC International Chemical Identifier (InChI) is a non-proprietary identifier for chemical substances that can be used in printed and electronic data sources. It is derived solely from a structural representation of that substance, such that a single compound always yields the same identifier.', 'created': '2019-06-11T14:17:07.236+0000', 'modified': '2019-06-11T14:17:07.236+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'InChI=1S/C2H6O/c1-2-3/h3H,2H2,1H3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1103'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1103'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1103/contactPerson'}}, 'metadata': []}, {'prefix': 'wikipedia.en', 'mirId': 'MIR:00000384', 'name': 'Wikipedia (En)', 'pattern': '^[A-Za-z-0-9_]+$', 'description': 'Wikipedia is a multilingual, web-based, free-content encyclopedia project based on an openly editable model. It is written collaboratively by largely anonymous Internet volunteers who write without pay.', 'created': '2019-06-11T14:17:07.845+0000', 'modified': '2019-06-11T14:17:07.845+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SM_UB-81', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1110'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1110'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1110/contactPerson'}}, 'metadata': []}, {'prefix': 'phosphopoint.kinase', 'mirId': 'MIR:00000385', 'name': 'PhosphoPoint Kinase', 'pattern': '^\\\\w+$', 'description': 'PhosphoPOINT is a database of the human kinase and phospho-protein interactome. It describes the interactions among kinases, their potential substrates and their interacting (phospho)-proteins. It also incorporates gene expression and uses gene ontology (GO) terms to annotate interactions. This collection references kinase information.', 'created': '2019-06-11T14:17:08.222+0000', 'modified': '2019-06-11T14:17:08.222+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AURKA', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1115'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1115'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1115/contactPerson'}}, 'metadata': []}, {'prefix': 'phosphopoint.protein', 'mirId': 'MIR:00000386', 'name': 'PhosphoPoint Phosphoprotein', 'pattern': '^\\\\w+$', 'description': 'PhosphoPOINT is a database of the human kinase and phospho-protein interactome. It describes the interactions among kinases, their potential substrates and their interacting (phospho)-proteins. It also incorporates gene expression and uses gene ontology (GO) terms to annotate interactions. This collection references phosphoprotein information.', 'created': '2019-06-11T14:17:08.458+0000', 'modified': '2019-06-11T14:17:08.458+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AURKA', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1118'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1118'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1118/contactPerson'}}, 'metadata': []}, {'prefix': 'inchikey', 'mirId': 'MIR:00000387', 'name': 'InChIKey', 'pattern': '^[A-Z]{14}\\\\-[A-Z]{10}(\\\\-[A-Z])?', 'description': 'The IUPAC International Chemical Identifier (InChI, see MIR:00000383) is an identifier for chemical substances, and is derived solely from a structural representation of that substance. Since these can be quite unwieldly, particularly for web use, the InChIKey was developed. These are of a fixed length (25 character) and were created as a condensed, more web friendly, digital representation of the InChI.', 'created': '2019-06-11T14:17:08.649+0000', 'modified': '2019-06-11T14:17:08.649+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RYYVLZVUVIJVGH-UHFFFAOYSA-N', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1120'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1120'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1120/contactPerson'}}, 'metadata': []}, {'prefix': 'uniprot.isoform', 'mirId': 'MIR:00000388', 'name': 'UniProt Isoform', 'pattern': '^([A-N,R-Z][0-9][A-Z][A-Z, 0-9][A-Z, 0-9][0-9])|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])(\\\\-\\\\d+)$', 'description': 'The UniProt Knowledgebase (UniProtKB) is a comprehensive resource for protein sequence and functional information with extensive cross-references to more than 120 external databases. This collection is a subset of UniProtKB, and provides a means to reference isoform information.', 'created': '2019-06-11T14:17:08.943+0000', 'modified': '2019-06-11T14:17:08.943+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q5BJF6-3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1123'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1123'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1123/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.environ', 'mirId': 'MIR:00000389', 'name': 'KEGG Environ', 'pattern': '^(ev\\\\:)?E\\\\d+$', 'description': 'KEGG ENVIRON (renamed from EDRUG) is a collection of crude drugs, essential oils, and other health-promoting substances, which are mostly natural products of plants. It will contain environmental substances and other health-damagine substances as well. Each KEGG ENVIRON entry is identified by the E number and is associated with the chemical component, efficacy information, and source species information whenever applicable.', 'created': '2019-06-11T14:17:09.349+0000', 'modified': '2019-06-11T14:17:09.349+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ev:E00032', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1127'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1127'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1127/contactPerson'}}, 'metadata': []}, {'prefix': 'cldb', 'mirId': 'MIR:00000390', 'name': 'CLDB', 'pattern': '^(cl|tum)\\\\d+$', 'description': 'The Cell Line Data Base (CLDB) is a reference information source for human and animal cell lines. It provides the characteristics of the cell lines and their availability through  distributors, allowing cell line requests to be made from collections and laboratories.', 'created': '2019-06-11T14:17:09.531+0000', 'modified': '2019-06-11T14:17:09.531+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'cl3603', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1129'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1129'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1129/contactPerson'}}, 'metadata': []}, {'prefix': 'efo', 'mirId': 'MIR:00000391', 'name': 'Experimental Factor Ontology', 'pattern': '^\\\\d{7}$', 'description': 'The Experimental Factor Ontology (EFO) provides a systematic description of many experimental variables available in EBI databases. It combines parts of several biological ontologies, such as anatomy, disease and chemical compounds. The scope of EFO is to support the annotation, analysis and visualization of data handled by the EBI Functional Genomics Team.', 'created': '2019-06-11T14:17:09.753+0000', 'modified': '2019-06-11T14:17:09.753+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0004859', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1132'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1132'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1132/contactPerson'}}, 'metadata': []}, {'prefix': 'hgmd', 'mirId': 'MIR:00000392', 'name': 'HGMD', 'pattern': '^[A-Z_0-9]+$', 'description': 'The Human Gene Mutation Database (HGMD) collates data on germ-line mutations in nuclear genes associated with human inherited disease. It includes information on single base-pair substitutions in coding, regulatory and splicing-relevant regions; micro-deletions and micro-insertions; indels; triplet repeat expansions as well as gross deletions; insertions; duplications; and complex rearrangements. Each mutation entry is unique, and includes cDNA reference sequences for most genes, splice junction sequences, disease-associated and functional polymorphisms, as well as links to data present in publicly available online locus-specific mutation databases.', 'created': '2019-06-11T14:17:10.208+0000', 'modified': '2019-06-11T14:17:10.208+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CALM1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1136'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1136'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1136/contactPerson'}}, 'metadata': []}, {'prefix': 'aphidbase.transcript', 'mirId': 'MIR:00000393', 'name': 'AphidBase Transcript', 'pattern': '^ACYPI\\\\d{6}(-RA)?$', 'description': 'AphidBase is a centralized bioinformatic resource that was developed to facilitate community annotation of the pea aphid genome by the International Aphid Genomics Consortium (IAGC). The AphidBase Information System was designed to organize and distribute genomic data and annotations for a large international community. This collection references the transcript report, which describes genomic location, sequence and exon information.', 'created': '2019-06-11T14:17:10.425+0000', 'modified': '2019-06-11T14:17:10.425+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ACYPI000159', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1139'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1139'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1139/contactPerson'}}, 'metadata': []}, {'prefix': 'affy.probeset', 'mirId': 'MIR:00000394', 'name': 'Affymetrix Probeset', 'pattern': '\\\\d{4,}((_[asx])?_at)?', 'description': 'An Affymetrix ProbeSet is a collection of up to 11 short (~22 nucleotide) microarray probes designed to measure a single gene or a family of genes as a unit. Multiple probe sets may be available for each gene under consideration.', 'created': '2019-06-11T14:17:10.647+0000', 'modified': '2019-06-11T14:17:10.647+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '243002_at', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1142'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1142'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1142/contactPerson'}}, 'metadata': []}, {'prefix': 'treefam', 'mirId': 'MIR:00000395', 'name': 'TreeFam', 'pattern': '^\\\\w{1,2}\\\\d+$', 'description': 'TreeFam is a database of phylogenetic trees of gene families found in animals. Automatically generated trees are curated, to create a curated resource that presents the accurate evolutionary history of all animal gene families, as well as reliable ortholog and paralog assignments.', 'created': '2019-06-11T14:17:10.973+0000', 'modified': '2019-06-11T14:17:10.973+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TF101014', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1146'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1146'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1146/contactPerson'}}, 'metadata': []}, {'prefix': 'caps', 'mirId': 'MIR:00000396', 'name': 'CAPS-DB', 'pattern': '^\\\\d+$', 'description': 'CAPS-DB is a structural classification of helix-cappings or caps compiled from protein structures. The regions of the polypeptide chain immediately preceding or following an alpha-helix are known as Nt- and Ct cappings, respectively. Caps extracted from protein structures have been structurally classified based on geometry and conformation and organized in a tree-like hierarchical classification where the different levels correspond to different properties of the caps.', 'created': '2019-06-11T14:17:11.199+0000', 'modified': '2019-06-11T14:17:11.199+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '434', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1149'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1149'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1149/contactPerson'}}, 'metadata': []}, {'prefix': 'cubedb', 'mirId': 'MIR:00000397', 'name': 'Cube db', 'pattern': '^[A-Za-z_0-9]+$', 'description': 'Cube-DB is a database of pre-evaluated results for detection of functional divergence in human/vertebrate protein families. It analyzes comparable taxonomical samples for all paralogues under consideration, storing functional specialisation at the level of residues. The data are presented as a table of per-residue scores, and mapped onto related structures where available.', 'created': '2019-06-11T14:17:11.437+0000', 'modified': '2019-06-11T14:17:11.437+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AKR', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1152'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1152'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1152/contactPerson'}}, 'metadata': []}, {'prefix': 'ideal', 'mirId': 'MIR:00000398', 'name': 'IDEAL', 'pattern': '^IID\\\\d+$', 'description': 'IDEAL provides a collection of knowledge on experimentally verified intrinsically disordered proteins. It contains manual annotations by curators on intrinsically disordered regions, interaction regions to other molecules, post-translational modification sites, references and structural domain assignments.', 'created': '2019-06-11T14:17:11.651+0000', 'modified': '2019-06-11T14:17:11.651+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'IID00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1155'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1155'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1155/contactPerson'}}, 'metadata': []}, {'prefix': 'stap', 'mirId': 'MIR:00000399', 'name': 'STAP', 'pattern': '^[0-9][A-Za-z0-9]{3}$', 'description': 'STAP (Statistical Torsional Angles Potentials) was developed since, according to several studies, some nuclear magnetic resonance (NMR) structures are of lower quality, are less reliable and less suitable for structural analysis than high-resolution X-ray crystallographic structures. The refined NMR solution structures (statistical torsion angle potentials; STAP) in the database are refined from the Protein Data Bank (PDB).', 'created': '2019-06-11T14:17:11.871+0000', 'modified': '2019-06-11T14:17:11.871+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1a24', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1158'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1158'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1158/contactPerson'}}, 'metadata': []}, {'prefix': 'pocketome', 'mirId': 'MIR:00000400', 'name': 'Pocketome', 'pattern': '^[A-Za-z_0-9]+', 'description': 'Pocketome is an encyclopedia of conformational ensembles of all druggable binding sites that can be identified experimentally from co-crystal structures in the Protein Data Bank. Each Pocketome entry corresponds to a small molecule binding site in a protein which has been co-crystallized in complex with at least one drug-like small molecule, and is represented in at least two PDB entries.', 'created': '2019-06-11T14:17:12.099+0000', 'modified': '2019-06-11T14:17:12.099+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1433C_TOBAC_1_252', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1161'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1161'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1161/contactPerson'}}, 'metadata': []}, {'prefix': 'gold.genome', 'mirId': 'MIR:00000401', 'name': 'GOLD genome', 'pattern': '^[Gi|Gc]\\\\d+$', 'description': 'The GOLD (Genomes OnLine Database)is a resource for centralized monitoring of genome and metagenome projects worldwide. It stores information on complete and ongoing projects, along with their associated metadata. This collection references the sequencing status of individual genomes.', 'created': '2019-06-11T14:17:12.343+0000', 'modified': '2019-06-11T14:17:12.343+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Gi07796', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1164'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1164'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1164/contactPerson'}}, 'metadata': []}, {'prefix': 'gold.meta', 'mirId': 'MIR:00000402', 'name': 'GOLD metadata', 'pattern': '^Gm\\\\d+$', 'description': 'The GOLD (Genomes OnLine Database)is a resource for centralized monitoring of genome and metagenome projects worldwide. It stores information on complete and ongoing projects, along with their associated metadata. This collection references metadata associated with samples.', 'created': '2019-06-11T14:17:12.564+0000', 'modified': '2019-06-11T14:17:12.564+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Gm00047', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1167'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1167'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1167/contactPerson'}}, 'metadata': []}, {'prefix': 'bugbase.protocol', 'mirId': 'MIR:00000403', 'name': 'BugBase Protocol', 'pattern': '^\\\\d+$', 'description': 'BugBase is a MIAME-compliant microbial gene expression and comparative genomic database. It stores experimental annotation and multiple raw and analysed data formats, as well as protocols for bacterial microarray designs. This collection references design protocols.', 'created': '2019-06-11T14:17:12.749+0000', 'modified': '2019-06-11T14:17:12.749+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '67', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1169'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1169'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1169/contactPerson'}}, 'metadata': []}, {'prefix': 'bugbase.expt', 'mirId': 'MIR:00000404', 'name': 'BugBase Expt', 'pattern': '^\\\\d+$', 'description': 'BugBase is a MIAME-compliant microbial gene expression and comparative genomic database. It stores experimental annotation and multiple raw and analysed data formats, as well as protocols for bacterial microarray designs. This collection references microarray experiments.', 'created': '2019-06-11T14:17:12.968+0000', 'modified': '2019-06-11T14:17:12.968+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '288', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1172'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1172'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1172/contactPerson'}}, 'metadata': []}, {'prefix': 'tol', 'mirId': 'MIR:00000405', 'name': 'Tree of Life', 'pattern': '^\\\\d+$', 'description': 'The Tree of Life Web Project (ToL) is a collaborative effort of biologists and nature enthusiasts from around the world. On more than 10,000 World Wide Web pages, the project provides information about biodiversity, the characteristics of different groups of organisms, and their evolutionary history (phylogeny). \\r\\n\\r\\nEach page contains information about a particular group, with pages linked one to another hierarchically, in the form of the evolutionary tree of life. Starting with the root of all Life on Earth and moving out along diverging branches to individual species, the structure of the ToL project thus illustrates the genetic connections between all living things.', 'created': '2019-06-11T14:17:13.167+0000', 'modified': '2019-06-11T14:17:13.167+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '98034', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1174'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1174'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1174/contactPerson'}}, 'metadata': []}, {'prefix': 'vario', 'mirId': 'MIR:00000406', 'name': 'VariO', 'pattern': '^VariO:\\\\d+$', 'description': 'The Variation Ontology (VariO) is an ontology for the standardized, systematic description of effects, consequences and mechanisms of variations. It describes the effects of variations at the DNA, RNA and/or protein level.', 'created': '2019-06-11T14:17:13.429+0000', 'modified': '2019-06-11T14:17:13.429+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0294', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1177'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1177'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1177/contactPerson'}}, 'metadata': []}, {'prefix': 'vipr', 'mirId': 'MIR:00000407', 'name': 'ViPR Strain', 'pattern': '^[A-Za-z 0-9]+$', 'description': 'The Virus Pathogen Database and Analysis Resource (ViPR) supports bioinformatics workflows for a broad range of human virus pathogens and other related viruses. It provides access to sequence records, gene and protein annotations, immune epitopes, 3D structures,  and host factor data. This collection references viral strain information.', 'created': '2019-06-11T14:17:13.893+0000', 'modified': '2019-06-11T14:17:13.893+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BeAn 70563', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1182'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1182'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1182/contactPerson'}}, 'metadata': []}, {'prefix': 'epd', 'mirId': 'MIR:00000408', 'name': 'EPD', 'pattern': '^[A-Z-_0-9]+$', 'description': 'The Eukaryotic Promoter Database (EPD) is an annotated non-redundant collection of eukaryotic POL II promoters, for which the transcription start site has been determined experimentally. Access to promoter sequences is provided by pointers to positions in nucleotide sequence entries. The annotation part of an entry includes description of the initiation site mapping data, cross-references to other databases, and bibliographic references. EPD is structured in a way that facilitates dynamic extraction of biologically meaningful promoter subsets for comparative sequence analysis.', 'created': '2019-06-11T14:17:14.109+0000', 'modified': '2019-06-11T14:17:14.109+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TA_H3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1185'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1185'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1185/contactPerson'}}, 'metadata': []}, {'prefix': 'rfam', 'mirId': 'MIR:00000409', 'name': 'RFAM', 'pattern': '^RF\\\\d{5}$', 'description': 'The Rfam database is a collection of RNA families, each represented by multiple sequence alignments, consensus secondary structures and covariance models (CMs). The families in Rfam break down into three broad functional classes: non-coding RNA genes, structured cis-regulatory elements and self-splicing RNAs. Typically these functional RNAs often have a conserved secondary structure which may be better preserved than the RNA sequence. The CMs used to describe each family are a slightly more complicated relative of the profile hidden Markov models (HMMs) used by Pfam. CMs can simultaneously model RNA sequence and the structure in an elegant and accurate fashion.', 'created': '2019-06-11T14:17:14.331+0000', 'modified': '2019-06-11T14:17:14.331+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RF00230', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1187'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1187'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1187/contactPerson'}}, 'metadata': []}, {'prefix': 'fbol', 'mirId': 'MIR:00000410', 'name': 'Fungal Barcode', 'pattern': '^\\\\d+$', 'description': 'DNA barcoding is the use of short standardised segments of the genome for identification of species in all the Kingdoms of Life. The goal of the Fungal Barcoding site is to promote the DNA barcoding of fungi and other fungus-like organisms.', 'created': '2019-06-11T14:17:14.533+0000', 'modified': '2019-06-11T14:17:14.533+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2224', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1189'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1189'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1189/contactPerson'}}, 'metadata': []}, {'prefix': 'aftol.taxonomy', 'mirId': 'MIR:00000411', 'name': 'AFTOL', 'pattern': '^\\\\d+$', 'description': 'The Assembling the Fungal Tree of Life (AFTOL) project is dedicated to significantly enhancing our understanding of the evolution of the Kingdom Fungi, which represents one of the major clades of life. There are roughly 80,000 described species of Fungi, but the actual diversity in the group has been estimated to be about 1.5 million species.', 'created': '2019-06-11T14:17:14.756+0000', 'modified': '2019-06-11T14:17:14.756+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '959', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1192'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1192'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1192/contactPerson'}}, 'metadata': []}, {'prefix': 'aspgd.locus', 'mirId': 'MIR:00000412', 'name': 'AspGD Locus', 'pattern': '^[A-Za-z_0-9]+$', 'description': 'The Aspergillus Genome Database (AspGD) is a repository for information relating to fungi of the genus Aspergillus, which includes organisms of clinical, agricultural and industrial importance. AspGD facilitates comparative genomics by providing a full-featured genomics viewer, as well as matched and standardized sets of genomic information for the sequenced aspergilli. This collection references gene information.', 'created': '2019-06-11T14:17:14.975+0000', 'modified': '2019-06-11T14:17:14.975+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ASPL0000349247', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1195'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1195'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1195/contactPerson'}}, 'metadata': []}, {'prefix': 'aspgd.protein', 'mirId': 'MIR:00000413', 'name': 'AspGD Protein', 'pattern': '^[A-Za-z_0-9]+$', 'description': 'The Aspergillus Genome Database (AspGD) is a repository for information relating to fungi of the genus Aspergillus, which includes organisms of clinical, agricultural and industrial importance. AspGD facilitates comparative genomics by providing a full-featured genomics viewer, as well as matched and standardized sets of genomic information for the sequenced aspergilli. This collection references protein information.', 'created': '2019-06-11T14:17:15.227+0000', 'modified': '2019-06-11T14:17:15.227+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ASPL0000349247', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1198'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1198'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1198/contactPerson'}}, 'metadata': []}, {'prefix': 'ndc', 'mirId': 'MIR:00000431', 'name': 'National Drug Code', 'pattern': '^\\\\d+\\\\-\\\\d+\\\\-\\\\d+', 'description': 'The National Drug Code (NDC) is a unique, three-segment number used by the Food and Drug Administration (FDA) to identify drug products for commercial use. This is required by the Drug Listing Act of 1972. The FDA publishes and updates the listed NDC numbers daily.', 'created': '2019-06-11T14:17:18.754+0000', 'modified': '2019-06-11T14:17:18.754+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0002-1975-61', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1237'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1237'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1237/contactPerson'}}, 'metadata': []}, {'prefix': 'autdb', 'mirId': 'MIR:00000415', 'name': 'AutDB', 'pattern': '^[A-Z]+[A-Z-0-9]{2,}$', 'description': 'AutDB is a curated database for autism research. It is built on information extracted from the studies on molecular genetics and biology of Autism Spectrum Disorders (ASD). The four modules of AutDB include information on Human Genes, Animal models, Protein Interactions (PIN) and Copy Number Variants (CNV) respectively. It provides an annotated list of ASD candidate genes in the form of reference dataset for interrogating molecular mechanisms underlying the disorder.', 'created': '2019-06-11T14:17:15.426+0000', 'modified': '2019-06-11T14:17:15.426+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ADA', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1200'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1200'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1200/contactPerson'}}, 'metadata': []}, {'prefix': 'bacmap.map', 'mirId': 'MIR:00000416', 'name': 'BacMap Map', 'pattern': '^\\\\w+(\\\\_)?\\\\d+(\\\\.\\\\d+)?$', 'description': \"BacMap is an electronic, interactive atlas of fully sequenced bacterial genomes. It contains labeled, zoomable and searchable chromosome maps for sequenced prokaryotic (archaebacterial and eubacterial) species. Each map can be zoomed to the level of individual genes and each gene is hyperlinked to a richly annotated gene card. All bacterial genome maps are supplemented with separate prophage genome maps as well as separate tRNA and rRNA maps. Each bacterial chromosome entry in BacMap contains graphs and tables on a variety of gene and protein statistics. Likewise, every bacterial species entry contains a bacterial 'biography' card, with taxonomic details, phenotypic details, textual descriptions and images. This collection references genome map information.\", 'created': '2019-06-11T14:17:15.634+0000', 'modified': '2019-06-11T14:17:15.634+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AP011135', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1203'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1203'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1203/contactPerson'}}, 'metadata': []}, {'prefix': 'bgee.family', 'mirId': 'MIR:00000417', 'name': 'Bgee family', 'pattern': '^(ENSFM|ENSGTV:)\\\\d+$', 'description': 'Bgee is a database of gene expression patterns within particular anatomical structures within a species, and between different animal species. This collection refers to expression across species.', 'created': '2019-06-11T14:17:15.821+0000', 'modified': '2019-06-11T14:17:15.821+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ENSFM00500000270089', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1205'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1205'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1205/contactPerson'}}, 'metadata': []}, {'prefix': 'bgee.gene', 'mirId': 'MIR:00000418', 'name': 'Bgee gene', 'pattern': '^[A-Za-z]+\\\\d+$', 'description': 'Bgee is a database to retrieve and compare gene expression patterns in multiple species, produced from multiple data types (RNA-Seq, Affymetrix, in situ hybridization, and EST data). This collection references genes in Bgee.', 'created': '2019-06-11T14:17:16.067+0000', 'modified': '2019-06-11T14:17:16.067+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'FBgn0000015', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1208'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1208'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1208/contactPerson'}}, 'metadata': []}, {'prefix': 'bgee.stage', 'mirId': 'MIR:00000419', 'name': 'Bgee stage', 'pattern': '^(FBvd|XtroDO|HsapDO|MmusDO)\\\\:\\\\d+$', 'description': 'Bgee is a database of gene expression patterns within particular anatomical structures within a species, and between different animal species. This collection refers to developmental stages.', 'created': '2019-06-11T14:17:16.333+0000', 'modified': '2019-06-11T14:17:16.333+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HsapDO:0000004', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1211'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1211'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1211/contactPerson'}}, 'metadata': []}, {'prefix': 'bgee.organ', 'mirId': 'MIR:00000420', 'name': 'Bgee organ', 'pattern': '^(XAO|ZFA|EHDAA|EMAPA|EV|MA)\\\\:\\\\d+$', 'description': 'Bgee is a database of gene expression patterns within particular anatomical structures within a species, and between different animal species. This collection refers to anatomical structures.', 'created': '2019-06-11T14:17:16.514+0000', 'modified': '2019-06-11T14:17:16.514+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EHDAA:2185', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1213'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1213'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1213/contactPerson'}}, 'metadata': []}, {'prefix': 'biocarta.pathway', 'mirId': 'MIR:00000421', 'name': 'BioCarta Pathway', 'pattern': '^([hm]\\\\_)?\\\\w+Pathway$', 'description': 'BioCarta is a supplier and distributor of characterized reagents and assays for biopharmaceutical and academic research. It catalogs community produced online maps depicting molecular relationships from areas of active research, generating classical pathways as well as suggestions for new pathways. This collections references pathway maps.', 'created': '2019-06-11T14:17:16.688+0000', 'modified': '2019-06-11T14:17:16.688+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'h_aktPathway', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1215'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1215'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1215/contactPerson'}}, 'metadata': []}, {'prefix': 'panther.pthcmp', 'mirId': 'MIR:00000422', 'name': 'PANTHER Pathway Component', 'pattern': '^G|P|U|C|S\\\\d{5}$', 'description': 'The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System is a resource that classifies genes by their functions, using published scientific experimental evidence and evolutionary relationships to predict function even in the absence of direct experimental evidence. The PANTHER Pathway Component collection references specific classes of molecules that play the same mechanistic role within a pathway, across species. Pathway\\r\\ncomponents may be proteins, genes/DNA, RNA, or simple molecules. Where the identified component is a protein, DNA, or transcribed RNA, it is associated with protein sequences in the PANTHER protein family trees through manual curation.', 'created': '2019-06-11T14:17:16.858+0000', 'modified': '2019-06-11T14:17:16.858+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P00266', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1217'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1217'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1217/contactPerson'}}, 'metadata': []}, {'prefix': 'gmd.profile', 'mirId': 'MIR:00000423', 'name': 'Golm Metabolome Database Profile', 'pattern': '^([0-9a-fA-F]){8}(-([0-9a-fA-F]){4}){3}-([0-9a-fA-F]){12}$', 'description': \"Golm Metabolome Database (GMD) provides public access to custom mass spectral libraries, metabolite profiling experiments as well as additional information and tools. GMD's metabolite profiles provide relative metabolite concentrations normalised according to fresh weight (or comparable quantitative data, such as volume, cell count, etc.) and internal standards (e.g. ribotol) of biological reference conditions and tissues.\", 'created': '2019-06-11T14:17:17.038+0000', 'modified': '2019-06-11T14:17:17.038+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '10b38aaf-b977-4950-85b8-f4775f66658d', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1219'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1219'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1219/contactPerson'}}, 'metadata': []}, {'prefix': 'gmd.gcms', 'mirId': 'MIR:00000424', 'name': 'Golm Metabolome Database GC-MS spectra', 'pattern': '^([0-9a-fA-F]){8}(-([0-9a-fA-F]){4}){3}-([0-9a-fA-F]){12}$', 'description': 'Golm Metabolome Database (GMD) provides public access to custom mass spectral libraries, metabolite profiling experiments as well as additional information and tools. Analytes are subjected to a gas chromatograph coupled to a mass spectrometer, which records the mass spectrum and the retention time linked to an analyte. This collection references GC-MS spectra.', 'created': '2019-06-11T14:17:17.255+0000', 'modified': '2019-06-11T14:17:17.255+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '53d583d8-40c6-40e1-9296-23f821cd77a5', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1221'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1221'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1221/contactPerson'}}, 'metadata': []}, {'prefix': 'gmd.ref', 'mirId': 'MIR:00000425', 'name': 'Golm Metabolome Database Reference Substance', 'pattern': '^([0-9a-fA-F]){8}(-([0-9a-fA-F]){4}){3}-([0-9a-fA-F]){12}$', 'description': 'Golm Metabolome Database (GMD) provides public access to custom mass spectral libraries, metabolite profiling experiments as well as additional information and tools. Since metabolites often cannot be obtained in their respective native biological state, for example organic acids may be only acquirable as salts, the concept of reference substance was introduced. This collection references reference substances.', 'created': '2019-06-11T14:17:17.475+0000', 'modified': '2019-06-11T14:17:17.475+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '8cf84adb-b4db-4807-ac98-0004247c35df', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1223'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1223'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1223/contactPerson'}}, 'metadata': []}, {'prefix': 'gmd.analyte', 'mirId': 'MIR:00000426', 'name': 'Golm Metabolome Database Analyte', 'pattern': '^([0-9a-fA-F]){8}(-([0-9a-fA-F]){4}){3}-([0-9a-fA-F]){12}$', 'description': 'Golm Metabolome Database (GMD) provides public access to custom mass spectral libraries, metabolite profiling experiments as well as additional information and tools. For GC-MS profiling analyses, polar metabolite extracts are chemically converted, i.e. derivatised into less polar and volatile compounds, so called analytes.  This collection references analytes.', 'created': '2019-06-11T14:17:17.661+0000', 'modified': '2019-06-11T14:17:17.661+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4f0fa9b6-514f-4ff4-98cc-0009bc08eb80', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1225'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1225'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1225/contactPerson'}}, 'metadata': []}, {'prefix': 'intact.molecule', 'mirId': 'MIR:00000427', 'name': 'IntAct Molecule', 'pattern': '^EBI\\\\-[0-9]+$', 'description': 'IntAct provides a freely available, open source database system and analysis tools for protein interaction data. This collection references interactor molecules.', 'created': '2019-06-11T14:17:17.850+0000', 'modified': '2019-06-11T14:17:17.850+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EBI-366083', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1227'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1227'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1227/contactPerson'}}, 'metadata': []}, {'prefix': 'depod', 'mirId': 'MIR:00000428', 'name': 'DEPOD', 'pattern': '^[A-Z0-9]+$', 'description': 'The human DEPhOsphorylation Database (DEPOD) contains information on known human active phosphatases and their experimentally verified protein and nonprotein substrates. Reliability scores are provided for dephosphorylation interactions, according to the type of assay used, as well as the number of laboratories that have confirmed such interaction. Phosphatase and substrate entries are listed along with the dephosphorylation site, bioassay type, and original literature, and contain links to other resources.', 'created': '2019-06-11T14:17:18.034+0000', 'modified': '2019-06-11T14:17:18.034+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PTPN1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1229'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1229'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1229/contactPerson'}}, 'metadata': []}, {'prefix': 'cst', 'mirId': 'MIR:00000429', 'name': 'Cell Signaling Technology Pathways', 'pattern': '^[A-Za-z0-9_-]+$', 'description': 'Cell Signaling Technology is a commercial organisation which provides a pathway portal to showcase their phospho-antibody products. This collection references pathways.', 'created': '2019-06-11T14:17:18.324+0000', 'modified': '2019-06-11T14:17:18.324+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Akt_PKB', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1232'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1232'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1232/contactPerson'}}, 'metadata': []}, {'prefix': 'phytozome.locus', 'mirId': 'MIR:00000432', 'name': 'Phytozome Locus', 'pattern': '^[A-Za-z0-9]+$', 'description': 'Phytozome is a project to facilitate comparative genomic studies amongst green plants. Famlies of orthologous and paralogous genes that represent the modern descendents of ancestral gene sets are constructed at key phylogenetic nodes. These families allow easy access to clade specific orthology/paralogy relationships as well as clade specific genes and gene expansions. This collection references locus information.', 'created': '2019-06-11T14:17:19.049+0000', 'modified': '2019-06-11T14:17:19.049+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Glyma0021s00410', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1241'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1241'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1241/contactPerson'}}, 'metadata': []}, {'prefix': 'subtilist', 'mirId': 'MIR:00000433', 'name': 'SubtiList', 'pattern': '^BG\\\\d+$', 'description': 'SubtiList serves to collate and integrate various aspects of the genomic information from B. subtilis, the paradigm of sporulating Gram-positive bacteria.\\r\\nSubtiList provides a complete dataset of DNA and protein sequences derived from the paradigm strain B. subtilis 168, linked to the relevant annotations and functional assignments.', 'created': '2019-06-11T14:17:19.310+0000', 'modified': '2019-06-11T14:17:19.310+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BG11523', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1244'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1244'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1244/contactPerson'}}, 'metadata': []}, {'prefix': 'dailymed', 'mirId': 'MIR:00000434', 'name': 'DailyMed', 'pattern': '^[A-Za-z0-9-]+', 'description': 'DailyMed provides information about marketed drugs. This information includes FDA labels (package inserts). The Web site provides a standard, comprehensive, up-to-date, look-up and download resource of medication content and labeling as found in medication package inserts. Drug labeling is the most recent submitted to the Food and Drug Administration (FDA) and currently in use; it may include, for example, strengthened warnings undergoing FDA review or minor editorial changes. These labels have been reformatted to make them easier to read.', 'created': '2019-06-11T14:17:19.532+0000', 'modified': '2019-06-11T14:17:19.532+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '973a9333-fec7-46dd-8eb5-25738f06ee54', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1247'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1247'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1247/contactPerson'}}, 'metadata': []}, {'prefix': 'sider.drug', 'mirId': 'MIR:00000435', 'name': 'SIDER Drug', 'pattern': '^\\\\d+$', 'description': 'SIDER (Side Effect Resource) is a public, computer-readable side effect resource that connects drugs to side effect terms. It aggregates dispersed public information on side effects. This collection references drugs in SIDER.', 'created': '2019-06-11T14:17:19.846+0000', 'modified': '2019-06-11T14:17:19.846+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2244', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1250'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1250'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1250/contactPerson'}}, 'metadata': []}, {'prefix': 'sider.effect', 'mirId': 'MIR:00000436', 'name': 'SIDER Side Effect', 'pattern': '^C\\\\d+$', 'description': 'SIDER (Side Effect Resource) is a public, computer-readable side effect resource that connects drugs to side effect terms. It aggregates dispersed public information on side effects. This collection references side effects of drugs as referenced in SIDER.', 'created': '2019-06-11T14:17:20.079+0000', 'modified': '2019-06-11T14:17:20.079+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C0017565', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1253'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1253'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1253/contactPerson'}}, 'metadata': []}, {'prefix': 'wikigenes', 'mirId': 'MIR:00000437', 'name': 'WikiGenes', 'pattern': '^\\\\d+$', 'description': 'WikiGenes is a collaborative knowledge resource for the life sciences, which is based on the general wiki idea but employs specifically developed technology to serve as a rigorous scientific tool. The rationale behind WikiGenes is to provide a platform for the scientific community to collect, communicate and evaluate knowledge about genes, chemicals, diseases and other biomedical concepts in a bottom-up process.', 'created': '2019-06-11T14:17:20.303+0000', 'modified': '2019-06-11T14:17:20.303+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3771877', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1255'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1255'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1255/contactPerson'}}, 'metadata': []}, {'prefix': 'broad', 'mirId': 'MIR:00000438', 'name': 'Broad Fungal Genome Initiative', 'pattern': '^S\\\\d+$', 'description': 'Magnaporthe grisea, the causal agent of rice blast disease, is one of the most devasting threats to food security worldwide and is a model organism for studying fungal phytopathogenicity and host-parasite interactions. The Magnaporthe comparative genomics database provides accesses to multiple fungal genomes from the Magnaporthaceae family to facilitate the comparative analysis. As part of the Broad Fungal Genome Initiative, the Magnaporthe comparative project includes the finished M. oryzae (formerly M. grisea) genome, as well as the draft assemblies of Gaeumannomyces graminis var. tritici and M. poae.', 'created': '2019-06-11T14:17:20.539+0000', 'modified': '2019-06-11T14:17:20.539+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'S7000002168151102', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1258'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1258'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1258/contactPerson'}}, 'metadata': []}, {'prefix': 'coriell', 'mirId': 'MIR:00000439', 'name': 'Coriell Cell Repositories', 'pattern': '^[A-Z]{2}\\\\d+$', 'description': 'The Coriell Cell Repositories provide essential research reagents to the scientific community by establishing, verifying, maintaining, and distributing cell cultures and DNA derived from cell cultures. These collections, supported by funds from the National Institutes of Health (NIH) and several foundations, are extensively utilized by research scientists around the world.', 'created': '2019-06-11T14:17:20.763+0000', 'modified': '2019-06-11T14:17:20.763+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GM17027', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1261'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1261'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1261/contactPerson'}}, 'metadata': []}, {'prefix': 'corum', 'mirId': 'MIR:00000440', 'name': 'CORUM', 'pattern': '^\\\\d+$', 'description': 'The CORUM database provides a resource of manually annotated protein complexes from mammalian organisms. Annotation includes protein complex function, localization, subunit composition, literature references and more. All information is obtained from individual experiments published in scientific articles, data from high-throughput experiments is excluded.', 'created': '2019-06-11T14:17:20.995+0000', 'modified': '2019-06-11T14:17:20.995+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1264'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1264'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1264/contactPerson'}}, 'metadata': []}, {'prefix': 'ecoliwiki', 'mirId': 'MIR:00000442', 'name': 'EcoliWiki', 'pattern': '^[A-Za-z0-9-]+$', 'description': 'EcoliWiki is a wiki-based resource to store information related to non-pathogenic E. coli, its phages, plasmids, and mobile genetic elements. This collection references genes.', 'created': '2019-06-11T14:17:21.287+0000', 'modified': '2019-06-11T14:17:21.287+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'aaeA', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1267'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1267'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1267/contactPerson'}}, 'metadata': []}, {'prefix': 'genprop', 'mirId': 'MIR:00000443', 'name': 'Genome Properties', 'pattern': '^GenProp\\\\d+$', 'description': 'Genome properties is an annotation system whereby functional attributes can be assigned to a genome, based on the presence of a defined set of protein signatures within that genome.', 'created': '2019-06-11T14:17:21.507+0000', 'modified': '2019-06-11T14:17:21.507+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GenProp0699', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1270'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1270'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1270/contactPerson'}}, 'metadata': []}, {'prefix': 'jstor', 'mirId': 'MIR:00000444', 'name': 'JSTOR', 'pattern': '^\\\\d+$', 'description': 'JSTOR (Journal Storage) is a digital library containing digital versions of historical academic journals, as well as books, pamphlets and current issues of journals. Some public domain content is free to access, while other articles require registration.', 'created': '2019-06-11T14:17:21.690+0000', 'modified': '2019-06-11T14:17:21.690+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3075966', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1272'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1272'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1272/contactPerson'}}, 'metadata': []}, {'prefix': 'ma', 'mirId': 'MIR:00000445', 'name': 'Mouse Adult Gross Anatomy', 'pattern': '^MA:\\\\d+$', 'description': 'A structured controlled vocabulary of the adult anatomy of the mouse (Mus)', 'created': '2019-06-11T14:17:21.919+0000', 'modified': '2019-06-11T14:17:21.919+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0002502', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1275'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1275'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1275/contactPerson'}}, 'metadata': []}, {'prefix': 'uberon', 'mirId': 'MIR:00000446', 'name': 'UBERON', 'pattern': '^UBERON\\\\:\\\\d+$', 'description': 'Uberon is an integrated cross-species anatomy ontology representing a variety of entities classified according to traditional anatomical criteria such as structure, function and developmental lineage. The ontology includes comprehensive relationships to taxon-specific anatomical ontologies, allowing integration of functional, phenotype and expression data.', 'created': '2019-06-11T14:17:22.433+0000', 'modified': '2019-06-11T14:17:22.433+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0008203', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1280'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1280'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1280/contactPerson'}}, 'metadata': []}, {'prefix': 'unimod', 'mirId': 'MIR:00000447', 'name': 'Unimod', 'pattern': '^\\\\d+$', 'description': 'Unimod is a public domain database created to provide a community supported, comprehensive database of protein modifications for mass spectrometry applications. That is, accurate and verifiable values, derived from elemental compositions, for the mass differences introduced by all types of natural and artificial modifications. Other important information includes any mass change, (neutral loss), that occurs during MS/MS analysis, and site specificity, (which residues are susceptible to modification and any constraints on the position of the modification within the protein or peptide).', 'created': '2019-06-11T14:17:22.748+0000', 'modified': '2019-06-11T14:17:22.748+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1200', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1283'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1283'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1283/contactPerson'}}, 'metadata': []}, {'prefix': 'vbrc', 'mirId': 'MIR:00000448', 'name': 'VBRC', 'pattern': '^\\\\d+$', 'description': 'The VBRC provides bioinformatics resources to support scientific research directed at viruses belonging to the Arenaviridae, Bunyaviridae, Filoviridae, Flaviviridae, Paramyxoviridae, Poxviridae, and Togaviridae families. The Center consists of a relational database and web application that support the data storage, annotation, analysis, and information exchange goals of this work. Each data release contains the complete genomic sequences for all viral pathogens and related strains that are available for species in the above-named families. In addition to sequence data, the VBRC provides a curation for each virus species, resulting in a searchable, comprehensive mini-review of gene function relating genotype to biological phenotype, with special emphasis on pathogenesis.', 'created': '2019-06-11T14:17:22.962+0000', 'modified': '2019-06-11T14:17:22.962+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '35742', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1286'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1286'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1286/contactPerson'}}, 'metadata': []}, {'prefix': 'viralzone', 'mirId': 'MIR:00000449', 'name': 'ViralZone', 'pattern': '^\\\\d+$', 'description': 'ViralZone is a resource bridging textbook knowledge with genomic and proteomic sequences. It provides fact sheets on all known virus families/genera with easy access to sequence data. A selection of reference strains (RefStrain) provides annotated standards to circumvent the exponential increase of virus sequences. Moreover ViralZone offers a complete set of detailed and accurate virion pictures.', 'created': '2019-06-11T14:17:23.207+0000', 'modified': '2019-06-11T14:17:23.207+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '992', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1289'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1289'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1289/contactPerson'}}, 'metadata': []}, {'prefix': 'go.ref', 'mirId': 'MIR:00000450', 'name': 'Gene Ontology Reference', 'pattern': '^GO_REF:\\\\d{7}$', 'description': 'The GO reference collection is a set of abstracts that can be cited in the GO ontologies (e.g. as dbxrefs for term definitions) and annotation files (in the Reference column). It provides two types of reference; It can be used to provide details of why specific Evidence codes (see http://identifiers.org/eco/) are assigned, or to present abstract-style descriptions of \"GO content\" meetings at which substantial changes in the ontologies are discussed and made.', 'created': '2019-06-11T14:17:23.452+0000', 'modified': '2019-06-11T14:17:23.452+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GO_REF:0000041', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1292'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1292'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1292/contactPerson'}}, 'metadata': []}, {'prefix': 'rgd.qtl', 'mirId': 'MIR:00000451', 'name': 'Rat Genome Database qTL', 'pattern': '^\\\\d+$', 'description': 'Rat Genome Database seeks to collect, consolidate, and integrate rat genomic and genetic data with curated functional and physiological data and make these data widely available to the scientific community. This collection references quantitative trait loci (qTLs), providing phenotype and disease descriptions, mapping, and strain information as well as links to markers and candidate genes.', 'created': '2019-06-11T14:17:23.659+0000', 'modified': '2019-06-11T14:17:23.659+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1354581', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1294'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1294'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1294/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'Bp247', 'description': 'QTL Report', 'url': 'https://rgd.mcw.edu/rgdweb/report/qtl/main.html?id=1354581', 'keywords': 'Rat Gene RGD Genome', 'includedInDataCatalog': 'https://rgd.mcw.edu', 'creator': {'@type': 'Organization', 'name': 'Rat Genome Database'}, 'version': '1', 'license': 'Creative Commons CC BY 4.0'}]}, {'prefix': 'rgd.strain', 'mirId': 'MIR:00000452', 'name': 'Rat Genome Database strain', 'pattern': '^\\\\d+$', 'description': 'Rat Genome Database seeks to collect, consolidate, and integrate rat genomic and genetic data with curated functional and physiological data and make these data widely available to the scientific community. This collection references strain reports, which include a description of strain origin, disease, phenotype, genetics and immunology.', 'created': '2019-06-11T14:17:23.876+0000', 'modified': '2019-06-11T14:17:23.876+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '5688061', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1297'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1297'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1297/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'SS.BN-(<i>D13Rat20-D13Got22</i>)/Mcwi-<i>Nckap5<sup>em4Mcwi+/+</sup></i>', 'description': 'Rat Strain', 'url': 'https://rgd.mcw.edu/rgdweb/report/strain/main.html?id=5688061', 'keywords': 'Rat Gene RGD Genome', 'includedInDataCatalog': 'https://rgd.mcw.edu', 'creator': {'@type': 'Organization', 'name': 'Rat Genome Database'}, 'version': '1', 'license': 'Creative Commons CC BY 4.0'}]}, {'prefix': 'door', 'mirId': 'MIR:00000453', 'name': 'DOOR', 'pattern': '^\\\\d+$', 'description': 'DOOR (Database for prOkaryotic OpeRons) contains computationally predicted operons of all the sequenced prokaryotic genomes. It includes operons for RNA genes.', 'created': '2019-06-11T14:17:24.062+0000', 'modified': '2019-06-11T14:17:24.062+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1398574', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1299'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1299'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1299/contactPerson'}}, 'metadata': []}, {'prefix': 'degradome', 'mirId': 'MIR:00000454', 'name': 'Degradome Database', 'pattern': '^[AMCST][0-9x][0-9]$', 'description': 'The Degradome Database contains information on the complete set of predicted proteases present in a a variety of mammalian species that have been subjected to whole genome sequencing. Each protease sequence is curated and, when necessary, cloned and sequenced.', 'created': '2019-06-11T14:17:24.294+0000', 'modified': '2019-06-11T14:17:24.294+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Ax1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1302'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1302'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1302/contactPerson'}}, 'metadata': []}, {'prefix': 'dbd', 'mirId': 'MIR:00000455', 'name': 'DBD', 'pattern': '^\\\\d+$', 'description': 'The DBD (transcription factor database) provides genome-wide transcription factor predictions for organisms across the tree of life. The prediction method identifies sequence-specific DNA-binding transcription factors through homology using profile hidden Markov models (HMMs) of domains from Pfam and SUPERFAMILY. It does not include basal transcription factors or chromatin-associated proteins.', 'created': '2019-06-11T14:17:24.514+0000', 'modified': '2019-06-11T14:17:24.514+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0045310', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1305'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1305'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1305/contactPerson'}}, 'metadata': []}, {'prefix': 'datf', 'mirId': 'MIR:00000456', 'name': 'DATF', 'pattern': '^AT[1-5]G\\\\d{5}(\\\\.\\\\d+)?$', 'description': 'DATF contains known and predicted Arabidopsis transcription factors (1827 genes in 56 families) with the unique information of 1177 cloned sequences and many other features including 3D structure templates, EST expression information, transcription factor binding sites and nuclear location signals.', 'created': '2019-06-11T14:17:24.701+0000', 'modified': '2019-06-11T14:17:24.701+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AT1G01030.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1307'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1307'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1307/contactPerson'}}, 'metadata': []}, {'prefix': 'iuphar.ligand', 'mirId': 'MIR:00000457', 'name': 'IUPHAR ligand', 'pattern': '^\\\\d+$', 'description': 'The IUPHAR Compendium details the molecular, biophysical and pharmacological properties of identified mammalian sodium, calcium and potassium channels, as well as the related cyclic nucleotide-modulated ion channels and the recently described transient receptor potential channels. It includes information on nomenclature systems, and on inter and intra-species molecular structure variation. This collection references ligands.', 'created': '2019-06-11T14:17:24.936+0000', 'modified': '2019-06-11T14:17:24.936+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1755', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1310'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1310'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1310/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@id': 'https://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=1755', '@type': 'DataRecord', 'dct:conformsTo': 'https://bioschemas.org/types/MolecularEntity/', 'includedInDataset': {'@type': 'Dataset', '@id': 'https://www.guidetopharmacology.org/index.jsp#dataset'}, 'mainEntity': {'@id': 'https://identifiers.org/iuphar.ligand:1755', 'name': 'ADP&beta;S', 'alternateName': [\"adenosine 5'-O-(2-thiodiphosphate)\", 'ADPbetaS'], 'description': '', 'identifier': '1755', 'inChI': 'InChI=1S/C10H15N5O9P2S/c11-8-5-9(13-2-12-8)15(3-14-5)10-7(17)6(16)4(23-10)1-22-25(18,19)24-26(20,21)27/h2-4,6-7,10,16-17H,1H2,(H,18,19)(H2,11,12,13)(H2,20,21,27)/t4-,6-,7-,10-/m1/s1', 'inChIKey': 'HCIKUKNAJRJFOW-KQYNXXCUSA-N', 'iupacName': '[(2R,3S,4R,5R)-5-(6-aminopurin-9-yl)-3,4-dihydroxyoxolan-2-yl]methyl dihydroxyphosphinothioyl hydrogen phosphate', 'molecularWeight': 443.01, 'smiles': 'OC1C(COP(=O)(OP(=S)(O)O)O)OC(C1O)n1cnc2c1ncnc2N', 'url': 'https://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=1755'}}]}, {'prefix': 'molbase', 'mirId': 'MIR:00000458', 'name': 'Molbase', 'pattern': '^(\\\\d{1,7}\\\\-\\\\d{2}\\\\-\\\\d)|([A-Za-z0-9\\\\+\\\\-\\\\_]+)$', 'description': 'Molbase provides compound data information for researchers as well as listing suppliers and price information. It can be searched by keyword or CAS indetifier.', 'created': '2019-06-11T14:17:25.144+0000', 'modified': '2019-06-11T14:17:25.144+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '128796-39-4', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1312'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1312'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1312/contactPerson'}}, 'metadata': []}, {'prefix': 'yrcpdr', 'mirId': 'MIR:00000459', 'name': 'YRC PDR', 'pattern': '^\\\\d+$', 'description': \"The Yeast Resource Center Public Data Repository (YRC PDR) serves as a single point of access for the experimental data produced from many collaborations typically studying Saccharomyces cerevisiae (baker's yeast). The experimental data include large amounts of mass spectrometry results from protein co-purification experiments, yeast two-hybrid interaction experiments, fluorescence microscopy images and protein structure predictions.\", 'created': '2019-06-11T14:17:25.397+0000', 'modified': '2019-06-11T14:17:25.397+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2673500', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1315'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1315'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1315/contactPerson'}}, 'metadata': []}, {'prefix': 'yid', 'mirId': 'MIR:00000460', 'name': 'Yeast Intron Database v3', 'pattern': '^[A-Z0-9]+$', 'description': 'The YEast Intron Database (version 3) contains information on the spliceosomal introns of the yeast Saccharomyces cerevisiae. It includes expression data that relates to the efficiency of splicing relative to other processes in strains of yeast lacking nonessential splicing factors.  The data are displayed on each intron page. An updated version of the database is available through [MIR:00000521].', 'created': '2019-06-11T14:17:25.610+0000', 'modified': '2019-06-11T14:17:25.610+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SNR17A', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1318'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1318'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1318/contactPerson'}}, 'metadata': []}, {'prefix': 'funcbase.fly', 'mirId': 'MIR:00000461', 'name': 'FuncBase Fly', 'pattern': '^\\\\d+$', 'description': 'Computational gene function prediction can serve to focus experimental resources on high-priority experimental tasks. FuncBase is a web resource for viewing quantitative machine learning-based gene function annotations. Quantitative annotations of genes, including fungal and mammalian genes, with Gene Ontology terms are accompanied by a community feedback system. Evidence underlying function annotations is shown. FuncBase provides links to external resources, and may be accessed directly or via links from species-specific databases. This collection references Drosophila data.', 'created': '2019-06-11T14:17:25.854+0000', 'modified': '2019-06-11T14:17:25.854+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '10194', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1321'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1321'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1321/contactPerson'}}, 'metadata': []}, {'prefix': 'funcbase.human', 'mirId': 'MIR:00000462', 'name': 'FuncBase Human', 'pattern': '^\\\\d+$', 'description': 'Computational gene function prediction can serve to focus experimental resources on high-priority experimental tasks. FuncBase is a web resource for viewing quantitative machine learning-based gene function annotations. Quantitative annotations of genes, including fungal and mammalian genes, with Gene Ontology terms are accompanied by a community feedback system. Evidence underlying function annotations is shown. FuncBase provides links to external resources, and may be accessed directly or via links from species-specific databases. This collection references human data.', 'created': '2019-06-11T14:17:26.074+0000', 'modified': '2019-06-11T14:17:26.074+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '119514', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1324'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1324'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1324/contactPerson'}}, 'metadata': []}, {'prefix': 'funcbase.mouse', 'mirId': 'MIR:00000463', 'name': 'FuncBase Mouse', 'pattern': '^\\\\d+$', 'description': 'Computational gene function prediction can serve to focus experimental resources on high-priority experimental tasks. FuncBase is a web resource for viewing quantitative machine learning-based gene function annotations. Quantitative annotations of genes, including fungal and mammalian genes, with Gene Ontology terms are accompanied by a community feedback system. Evidence underlying function annotations is shown. FuncBase provides links to external resources, and may be accessed directly or via links from species-specific databases. This collection references mouse.', 'created': '2019-06-11T14:17:26.285+0000', 'modified': '2019-06-11T14:17:26.285+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1351341', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1326'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1326'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1326/contactPerson'}}, 'metadata': []}, {'prefix': 'cbioportal', 'mirId': 'MIR:00000679', 'name': 'The cBioPortal for Cancer Genomics', 'pattern': '^[a-z0-9\\\\_]+$', 'description': 'The cBioPortal for Cancer Genomics provides visualization, analysis and download of large-scale cancer genomics data sets.', 'created': '2019-06-11T14:18:18.874+0000', 'modified': '2019-06-11T14:18:18.874+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'laml_tcga_pub', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1881'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1881'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1881/contactPerson'}}, 'metadata': []}, {'prefix': 'funcbase.yeast', 'mirId': 'MIR:00000464', 'name': 'FuncBase Yeast', 'pattern': '^\\\\d+$', 'description': 'Computational gene function prediction can serve to focus experimental resources on high-priority experimental tasks. FuncBase is a web resource for viewing quantitative machine learning-based gene function annotations. Quantitative annotations of genes, including fungal and mammalian genes, with Gene Ontology terms are accompanied by a community feedback system. Evidence underlying function annotations is shown. FuncBase provides links to external resources, and may be accessed directly or via links from species-specific databases. This collection references yeast.', 'created': '2019-06-11T14:17:26.470+0000', 'modified': '2019-06-11T14:17:26.470+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2701', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1328'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1328'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1328/contactPerson'}}, 'metadata': []}, {'prefix': 'ydpm', 'mirId': 'MIR:00000465', 'name': 'YDPM', 'pattern': '^Y[A-Z]{2}\\\\d+[CW]$', 'description': 'The YDPM database serves to support the Yeast Deletion and the Mitochondrial Proteomics Project. The project aims to increase the understanding of mitochondrial function and biogenesis in the context of the cell. In the Deletion Project, strains from the deletion collection were monitored under 9 different media conditions selected for the study of mitochondrial function. The YDPM database contains both the raw data and growth rates calculated for each strain in each media condition.', 'created': '2019-06-11T14:17:26.654+0000', 'modified': '2019-06-11T14:17:26.654+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'YAL001C', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1330'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1330'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1330/contactPerson'}}, 'metadata': []}, {'prefix': 'wb.rnai', 'mirId': 'MIR:00000466', 'name': 'WormBase RNAi', 'pattern': '^WBRNAi\\\\d{8}$', 'description': 'WormBase is an online bioinformatics database of the biology and genome of the model organism Caenorhabditis elegans and related nematodes. It is used by the C. elegans research community both as an information resource and as a mode to publish and distribute their results. This collection references RNAi experiments, detailing target and phenotypes.', 'created': '2019-06-11T14:17:26.886+0000', 'modified': '2019-06-11T14:17:26.886+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'WBRNAi00086878', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1333'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1333'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1333/contactPerson'}}, 'metadata': []}, {'prefix': 'pass2', 'mirId': 'MIR:00000468', 'name': 'PASS2', 'pattern': '^\\\\d+$', 'description': 'The PASS2 database provides alignments of proteins related at the superfamily level and are characterized by low sequence identity.', 'created': '2019-06-11T14:17:27.086+0000', 'modified': '2019-06-11T14:17:27.086+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '46977', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1335'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1335'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1335/contactPerson'}}, 'metadata': []}, {'prefix': 'iceberg.element', 'mirId': 'MIR:00000469', 'name': 'ICEberg element', 'pattern': '^\\\\d+$', 'description': 'ICEberg (Integrative and conjugative elements) is a database of integrative and conjugative elements (ICEs) found in bacteria. ICEs are conjugative self-transmissible elements that can integrate into and excise from a host chromosome, and can carry likely virulence determinants, antibiotic-resistant factors and/or genes coding for other beneficial traits. It contains details of ICEs found in representatives bacterial species, and which are organised as families. This collection references ICE elements.', 'created': '2019-06-11T14:17:27.338+0000', 'modified': '2019-06-11T14:17:27.338+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1338'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1338'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1338/contactPerson'}}, 'metadata': []}, {'prefix': 'iceberg.family', 'mirId': 'MIR:00000470', 'name': 'ICEberg family', 'pattern': '^\\\\d+$', 'description': 'ICEberg (Integrative and conjugative elements) is a database of integrative and conjugative elements (ICEs) found in bacteria. ICEs are conjugative self-transmissible elements that can integrate into and excise from a host chromosome, and can carry likely virulence determinants, antibiotic-resistant factors and/or genes coding for other beneficial traits. It contains details of ICEs found in representatives bacterial species, and which are organised as families. This collection references ICE families.', 'created': '2019-06-11T14:17:27.601+0000', 'modified': '2019-06-11T14:17:27.601+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1341'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1341'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1341/contactPerson'}}, 'metadata': []}, {'prefix': 'vfdb.genus', 'mirId': 'MIR:00000471', 'name': 'VFDB Genus', 'pattern': '^\\\\w+$', 'description': 'VFDB is a repository of virulence factors (VFs) of pathogenic bacteria.This collection references VF information by Genus.', 'created': '2019-06-11T14:17:27.811+0000', 'modified': '2019-06-11T14:17:27.811+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Chlamydia', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1344'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1344'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1344/contactPerson'}}, 'metadata': []}, {'prefix': 'vfdb.gene', 'mirId': 'MIR:00000472', 'name': 'VFDB Gene', 'pattern': '^\\\\w+$', 'description': 'VFDB is a repository of virulence factors (VFs) of pathogenic bacteria.This collection references VF genes.', 'created': '2019-06-11T14:17:28.027+0000', 'modified': '2019-06-11T14:17:28.027+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'VFG2154', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1347'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1347'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1347/contactPerson'}}, 'metadata': []}, {'prefix': 'mesh.2013', 'mirId': 'MIR:00000473', 'name': 'MeSH 2013', 'pattern': '^[A-Za-z0-9]+$', 'description': \"MeSH (Medical Subject Headings) is the National Library of Medicine's controlled vocabulary thesaurus. It consists of sets of terms naming descriptors in a hierarchical structure that permits searching at various levels of specificity. This thesaurus is used by NLM for indexing articles from biomedical journals, cataloging of books, documents, etc. This collection references MeSH terms published in 2013.\", 'created': '2019-06-11T14:17:28.224+0000', 'modified': '2019-06-11T14:17:28.224+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '17165', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1349'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1349'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1349/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.module', 'mirId': 'MIR:00000474', 'name': 'KEGG Module', 'pattern': '^([a-z]{3,5}_)?M\\\\d{5}$', 'description': \"KEGG Modules are manually defined functional units used in the annotation and biological interpretation of sequenced genomes. Each module corresponds to a set of 'KEGG Orthology' (MIR:00000116) entries. KEGG Modules can represent pathway, structural, functional or signature modules.\", 'created': '2019-06-11T14:17:28.414+0000', 'modified': '2019-06-11T14:17:28.414+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'M00002', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1351'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1351'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1351/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg.disease', 'mirId': 'MIR:00000475', 'name': 'KEGG Disease', 'pattern': '^H\\\\d+$', 'description': 'The KEGG DISEASE database is a collection of disease entries capturing knowledge on genetic and environmental perturbations. Each disease entry contains a list of known genetic factors (disease genes), environmental factors, diagnostic markers, and therapeutic drugs. Diseases are viewed as perturbed states of the molecular system, and drugs as perturbants to the molecular system.', 'created': '2019-06-11T14:17:28.601+0000', 'modified': '2019-06-11T14:17:28.601+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'H00076', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1353'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1353'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1353/contactPerson'}}, 'metadata': []}, {'prefix': 'medlineplus', 'mirId': 'MIR:00000476', 'name': 'MedlinePlus', 'pattern': '^\\\\d+$', 'description': \"MedlinePlus is the National Institutes of Health's Web site for patients and their families and friends. Produced by the National Library of Medicine, it provides information about diseases, conditions, and wellness issues using non-technical terms and language.\", 'created': '2019-06-11T14:17:28.795+0000', 'modified': '2019-06-11T14:17:28.795+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '002804', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1355'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1355'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1355/contactPerson'}}, 'metadata': []}, {'prefix': 'ligandbox', 'mirId': 'MIR:00000477', 'name': 'LigandBox', 'pattern': '^D\\\\d{5}', 'description': 'LigandBox is a database of 3D compound structures.  Compound information is collected from the catalogues of various commercial suppliers, with approved drugs and biochemical compounds taken from KEGG and PDB databases. Each chemical compound in the database has several 3D conformers with hydrogen atoms and atomic charges, which are ready to be docked into receptors using docking programs. Various physical properties, such as aqueous solubility (LogS) and carcinogenicity have also been calculated to characterize the ADME-Tox properties of the compounds.', 'created': '2019-06-11T14:17:28.979+0000', 'modified': '2019-06-11T14:17:28.979+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'D00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1357'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1357'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1357/contactPerson'}}, 'metadata': []}, {'prefix': 'glycoepitope', 'mirId': 'MIR:00000478', 'name': 'GlycoEpitope', 'pattern': '^EP\\\\d{4}$', 'description': 'GlycoEpitope is a database containing useful information about carbohydrate antigens (glyco-epitopes) and the antibodies (polyclonal or monoclonal) that can be used to analyze their expression. This collection references Glycoepitopes.', 'created': '2019-06-11T14:17:29.214+0000', 'modified': '2019-06-11T14:17:29.214+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EP0311', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1360'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1360'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1360/contactPerson'}}, 'metadata': []}, {'prefix': 'jcggdb', 'mirId': 'MIR:00000479', 'name': 'JCGGDB', 'pattern': '^JCGG-STR\\\\d{6}$', 'description': 'JCGGDB (Japan Consortium for Glycobiology and Glycotechnology DataBase) is a database that aims to integrate all glycan-related data held in various repositories in Japan. This includes databases for large-quantity synthesis of glycogenes and glycans, analysis and detection of glycan structure and glycoprotein, glycan-related differentiation markers, glycan functions, glycan-related diseases and transgenic and knockout animals, etc.', 'created': '2019-06-11T14:17:29.437+0000', 'modified': '2019-06-11T14:17:29.437+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'JCGG-STR008690', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1363'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1363'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1363/contactPerson'}}, 'metadata': []}, {'prefix': 'noncodev4.gene', 'mirId': 'MIR:00000480', 'name': 'NONCODE v4 Gene', 'pattern': '^NONHSAG\\\\d{5}$', 'description': 'NONCODE is a database of expression and functional lncRNA (long noncoding RNA) data obtained from microarray studies. LncRNAs have been shown to play key roles in various biological processes such as imprinting control, circuitry controlling pluripotency and differentiation, immune responses and chromosome dynamics. The collection references NONCODE version 4 and relates to gene regions.', 'created': '2019-06-11T14:17:29.649+0000', 'modified': '2019-06-11T14:17:29.649+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'NONHSAG00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1366'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1366'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1366/contactPerson'}}, 'metadata': []}, {'prefix': 'vfb', 'mirId': 'MIR:00000680', 'name': 'Virtual Fly Brain', 'pattern': '^[0-9a-zA-Z]{8}$', 'description': 'An interactive tool for neurobiologists to explore the detailed neuroanatomy, neuron connectivity and gene expression of the Drosophila melanogaster.', 'created': '2019-06-11T14:18:19.150+0000', 'modified': '2019-06-11T14:18:19.150+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '00000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1884'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1884'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1884/contactPerson'}}, 'metadata': []}, {'prefix': 'noncodev4.rna', 'mirId': 'MIR:00000481', 'name': 'NONCODE v4 Transcript', 'pattern': '^NONHSAT\\\\d{6}$', 'description': 'NONCODE is a database of expression and functional lncRNA (long noncoding RNA) data obtained from microarray studies. LncRNAs have been shown to play key roles in various biological processes such as imprinting control, circuitry controlling pluripotency and differentiation, immune responses and chromosome dynamics. The collection references NONCODE version 4 and relates to individual transcripts.', 'created': '2019-06-11T14:17:29.833+0000', 'modified': '2019-06-11T14:17:29.833+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'NONHSAT000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1368'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1368'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1368/contactPerson'}}, 'metadata': []}, {'prefix': 'oryzabase.gene', 'mirId': 'MIR:00000482', 'name': 'Oryzabase Gene', 'pattern': '^\\\\d+$', 'description': 'Oryzabase provides a view of rice (Oryza sativa) as a model monocot plant by integrating biological data with molecular genomic information. It contains information about rice development and anatomy, rice mutants, and genetic resources, especially for wild varieties of rice. Developmental and anatomical descriptions include in situ gene expression data serving as stage and tissue markers. This collection references gene information.', 'created': '2019-06-11T14:17:30.027+0000', 'modified': '2019-06-11T14:17:30.027+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '117', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1370'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1370'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1370/contactPerson'}}, 'metadata': []}, {'prefix': 'oryzabase.mutant', 'mirId': 'MIR:00000483', 'name': 'Oryzabase Mutant', 'pattern': '^\\\\d+$', 'description': 'Oryzabase provides a view of rice (Oryza sativa) as a model monocot plant by integrating biological data with molecular genomic information. It contains information about rice development and anatomy, rice mutants, and genetic resources, especially for wild varieties of rice. Developmental and anatomical descriptions include in situ gene expression data serving as stage and tissue markers. This collection references mutant strain information.', 'created': '2019-06-11T14:17:30.248+0000', 'modified': '2019-06-11T14:17:30.248+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '21393', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1373'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1373'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1373/contactPerson'}}, 'metadata': []}, {'prefix': 'oryzabase.strain', 'mirId': 'MIR:00000484', 'name': 'Oryzabase Strain', 'pattern': '^\\\\d+$', 'description': 'Oryzabase provides a view of rice (Oryza sativa) as a model monocot plant by integrating biological data with molecular genomic information. It contains information about rice development and anatomy, rice mutants, and genetic resources, especially for wild varieties of rice. Developmental and anatomical descriptions include in situ gene expression data serving as stage and tissue markers. This collection references wild strain information.', 'created': '2019-06-11T14:17:30.425+0000', 'modified': '2019-06-11T14:17:30.425+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1375'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1375'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1375/contactPerson'}}, 'metadata': []}, {'prefix': 'oryzabase.stage', 'mirId': 'MIR:00000485', 'name': 'Oryzabase Stage', 'pattern': '^\\\\d+$', 'description': 'Oryzabase provides a view of rice (Oryza sativa) as a model monocot plant by integrating biological data with molecular genomic information. It contains information about rice development and anatomy, rice mutants, and genetic resources, especially for wild varieties of rice. Developmental and anatomical descriptions include in situ gene expression data serving as stage and tissue markers. This collection references development stage information.', 'created': '2019-06-11T14:17:30.616+0000', 'modified': '2019-06-11T14:17:30.616+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '34', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1377'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1377'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1377/contactPerson'}}, 'metadata': []}, {'prefix': 'otl', 'mirId': 'MIR:00000486', 'name': 'Oryza Tag Line', 'pattern': '^A[A-Z]+\\\\d+$', 'description': 'Oryza Tag Line is a database that was developed to collect information generated from the characterization of rice (Oryza sativa L cv. Nipponbare) insertion lines resulting in potential gene disruptions. It collates morpho-physiological alterations observed during field evaluation, with each insertion line documented through a generic passport data including production records, seed stocks and FST information.', 'created': '2019-06-11T14:17:30.833+0000', 'modified': '2019-06-11T14:17:30.833+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AADB12', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1379'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1379'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1379/contactPerson'}}, 'metadata': []}, {'prefix': 'genewiki', 'mirId': 'MIR:00000487', 'name': 'Gene Wiki', 'pattern': '^\\\\d+$', 'description': \"The Gene Wiki is project which seeks to provide detailed information on human genes. Initial 'stub' articles are created in an automated manner, with further information added by the community. Gene Wiki can be accessed in wikipedia using Gene identifiers from NCBI.\", 'created': '2019-06-11T14:17:31.086+0000', 'modified': '2019-06-11T14:17:31.086+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1017', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1382'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1382'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1382/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org', '@type': 'Article', 'name': 'Cyclin-dependent kinase 2', 'url': 'https://en.wikipedia.org/wiki/Cyclin-dependent_kinase_2', 'sameAs': 'http://www.wikidata.org/entity/Q14911732', 'mainEntity': 'http://www.wikidata.org/entity/Q14911732', 'author': {'@type': 'Organization', 'name': 'Contributors to Wikimedia projects'}, 'publisher': {'@type': 'Organization', 'name': 'Wikimedia Foundation, Inc.', 'logo': {'@type': 'ImageObject', 'url': 'https://www.wikimedia.org/static/images/wmf-hor-googpub.png'}}, 'datePublished': '2007-03-05T20:28:34Z', 'dateModified': '2019-11-11T10:45:57Z', 'image': 'https://upload.wikimedia.org/wikipedia/commons/6/6d/Protein_CDK2_PDB_1aq1.png', 'headline': 'protein-coding gene in the species Homo sapiens'}]}, {'prefix': 'paxdb.organism', 'mirId': 'MIR:00000488', 'name': 'PaxDb Organism', 'pattern': '^\\\\d+$', 'description': 'PaxDb is a resource dedicated to integrating information on absolute protein abundance levels across different organisms. Publicly available experimental data are mapped onto a common namespace and, in the case of tandem mass spectrometry data, re-processed using a standardized spectral counting pipeline. Data sets are scored and ranked to assess consistency against externally provided protein-network information. PaxDb provides whole-organism data as well as tissue-resolved data, for numerous proteins. This collection references protein abundance information by species.', 'created': '2019-06-11T14:17:31.315+0000', 'modified': '2019-06-11T14:17:31.315+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '9606', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1384'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1384'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1384/contactPerson'}}, 'metadata': []}, {'prefix': 'paxdb.protein', 'mirId': 'MIR:00000489', 'name': 'PaxDb Protein', 'pattern': '^\\\\d+$', 'description': 'PaxDb is a resource dedicated to integrating information on absolute protein abundance levels across different organisms. Publicly available experimental data are mapped onto a common namespace and, in the case of tandem mass spectrometry data, re-processed using a standardized spectral counting pipeline. Data sets are scored and ranked to assess consistency against externally provided protein-network information. PaxDb provides whole-organism data as well as tissue-resolved data, for numerous proteins. This collection references individual protein abundance levels.', 'created': '2019-06-11T14:17:31.566+0000', 'modified': '2019-06-11T14:17:31.566+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '977869', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1387'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1387'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1387/contactPerson'}}, 'metadata': []}, {'prefix': 'pdb.ligand', 'mirId': 'MIR:00000490', 'name': 'Protein Data Bank Ligand', 'pattern': '^[A-Za-z0-9]+$', 'description': 'The Protein Data Bank is the single worldwide archive of structural data of biological macromolecules. This collection references ligands.', 'created': '2019-06-11T14:17:31.784+0000', 'modified': '2019-06-11T14:17:31.784+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'TRQ', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1389'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1389'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1389/contactPerson'}}, 'metadata': []}, {'prefix': 'merops.inhibitor', 'mirId': 'MIR:00000491', 'name': 'MEROPS Inhibitor', 'pattern': '^I\\\\d{2}\\\\.\\\\d{3}$', 'description': 'The MEROPS database is an information resource for peptidases (also termed proteases, proteinases and proteolytic enzymes) and the proteins that inhibit them. This collections references inhibitors.', 'created': '2019-06-11T14:17:32.129+0000', 'modified': '2019-06-11T14:17:32.129+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'I31.952', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1392'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1392'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1392/contactPerson'}}, 'metadata': []}, {'prefix': 'adw', 'mirId': 'MIR:00000492', 'name': 'Animal Diversity Web', 'pattern': '^[A-Z_a-z]+$', 'description': 'Animal Diversity Web (ADW) is an online database of animal natural history, distribution, classification, and conservation biology.', 'created': '2019-06-11T14:17:32.366+0000', 'modified': '2019-06-11T14:17:32.366+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Lycalopex_vetulus', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1395'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1395'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1395/contactPerson'}}, 'metadata': []}, {'prefix': 'glida.gpcr', 'mirId': 'MIR:00000493', 'name': 'GLIDA GPCR', 'pattern': '^[A-Z-_0-9]+$', 'description': 'The GPCR-LIgand DAtabase (GLIDA) is a GPCR-related chemical genomic database that is primarily focused on the correlation of information between GPCRs and their ligands. It provides correlation data between GPCRs and their ligands, along with chemical information on the ligands. This collection references G-protein coupled receptors.', 'created': '2019-06-11T14:17:32.633+0000', 'modified': '2019-06-11T14:17:32.633+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ACM1_HUMAN', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1398'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1398'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1398/contactPerson'}}, 'metadata': []}, {'prefix': 'glida.ligand', 'mirId': 'MIR:00000494', 'name': 'GLIDA Ligand', 'pattern': '^L\\\\d+$', 'description': 'The GPCR-LIgand DAtabase (GLIDA) is a GPCR-related chemical genomic database that is primarily focused on the correlation of information between GPCRs and their ligands. It provides correlation data between GPCRs and their ligands, along with chemical information on the ligands. This collection references ligands.', 'created': '2019-06-11T14:17:32.872+0000', 'modified': '2019-06-11T14:17:32.872+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'L000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1401'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1401'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1401/contactPerson'}}, 'metadata': []}, {'prefix': 'grsdb', 'mirId': 'MIR:00000495', 'name': 'GRSDB', 'pattern': '^\\\\d+$', 'description': 'GRSDB is a database of G-quadruplexes and contains information on composition and distribution of putative Quadruplex-forming G-Rich Sequences (QGRS) mapped in the eukaryotic pre-mRNA sequences, including those that are alternatively processed (alternatively spliced or alternatively polyadenylated). The data stored in the GRSDB is based on computational analysis of NCBI Entrez Gene entries and their corresponding annotated genomic nucleotide sequences of RefSeq/GenBank.', 'created': '2019-06-11T14:17:33.087+0000', 'modified': '2019-06-11T14:17:33.087+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '10142', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1403'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1403'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1403/contactPerson'}}, 'metadata': []}, {'prefix': 'px', 'mirId': 'MIR:00000513', 'name': 'ProteomeXchange', 'pattern': '^(R)?PXD\\\\d{6}$', 'description': 'The ProteomeXchange provides a single point of submission of Mass Spectrometry (MS) proteomics data for the main existing proteomics repositories, and encourages the data exchange between them for optimal data dissemination.', 'created': '2019-06-11T14:17:37.549+0000', 'modified': '2019-06-11T14:17:37.549+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PXD000500', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1447'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1447'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1447/contactPerson'}}, 'metadata': []}, {'prefix': 'biomodels.vocabulary', 'mirId': 'MIR:00000514', 'name': 'SBML RDF Vocabulary', 'pattern': '^[A-Za-z]+$', 'description': 'Vocabulary used in the RDF representation of SBML models.', 'created': '2019-06-11T14:17:37.784+0000', 'modified': '2019-06-11T14:17:37.784+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'rateRule', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1450'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1450'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1450/contactPerson'}}, 'metadata': []}, {'prefix': 'fsnp', 'mirId': 'MIR:00000496', 'name': 'F-SNP', 'pattern': '^rs\\\\d+$', 'description': 'The Functional Single Nucleotide Polymorphism (F-SNP) database integrates information obtained from databases about the functional effects of SNPs. These effects are predicted and indicated at the splicing, transcriptional, translational and post-translational level. In particular, users can retrieve SNPs that disrupt genomic regions known to be functional, including splice sites and transcriptional regulatory regions. Users can also identify non-synonymous SNPs that may have deleterious effects on protein structure or function, interfere with protein translation or impede post-translational modification.', 'created': '2019-06-11T14:17:33.340+0000', 'modified': '2019-06-11T14:17:33.340+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'rs17852708', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1406'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1406'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1406/contactPerson'}}, 'metadata': []}, {'prefix': 'hdr', 'mirId': 'MIR:00000497', 'name': 'Homeodomain Research', 'pattern': '^\\\\d+$', 'description': 'The Homeodomain Resource is a curated collection of sequence, structure, interaction, genomic and functional information on the homeodomain family. It contains sets of curated homeodomain sequences from fully sequenced genomes, including experimentally derived homeodomain structures, homeodomain protein-protein interactions, homeodomain DNA-binding sites and homeodomain proteins implicated in human genetic disorders.', 'created': '2019-06-11T14:17:33.593+0000', 'modified': '2019-06-11T14:17:33.593+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '63', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1409'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1409'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1409/contactPerson'}}, 'metadata': []}, {'prefix': 'norine', 'mirId': 'MIR:00000498', 'name': 'NORINE', 'pattern': '^NOR\\\\d+$', 'description': 'Norine is a database dedicated to nonribosomal peptides (NRPs). In bacteria and fungi, in addition to the traditional ribosomal proteic biosynthesis, an alternative ribosome-independent pathway called NRP synthesis allows peptide production. The molecules synthesized by NRPS contain a high proportion of nonproteogenic amino acids whose primary structure is not always linear, often being more complex and containing cycles and branchings.', 'created': '2019-06-11T14:17:33.862+0000', 'modified': '2019-06-11T14:17:33.862+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'NOR00681', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1412'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1412'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1412/contactPerson'}}, 'metadata': []}, {'prefix': 'ordb', 'mirId': 'MIR:00000499', 'name': 'Olfactory Receptor Database', 'pattern': '^\\\\d+$', 'description': 'The Olfactory Receptor Database (ORDB) is a repository of genomics and proteomics information of olfactory receptors (ORs). It includes a broad range of chemosensory genes and proteins, that includes in addition to ORs the taste papilla receptors (TPRs), vomeronasal organ receptors (VNRs), insect olfactory receptors (IORs), Caenorhabditis elegans chemosensory receptors (CeCRs), fungal pheromone receptors (FPRs).', 'created': '2019-06-11T14:17:34.104+0000', 'modified': '2019-06-11T14:17:34.104+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '8497', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1415'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1415'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1415/contactPerson'}}, 'metadata': []}, {'prefix': 'odor', 'mirId': 'MIR:00000500', 'name': 'Odor Molecules DataBase', 'pattern': '^\\\\d+$', 'description': 'OdorDB stores information related to odorous compounds, specifically identifying those that have been shown to interact with olfactory receptors', 'created': '2019-06-11T14:17:34.340+0000', 'modified': '2019-06-11T14:17:34.340+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '74', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1418'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1418'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1418/contactPerson'}}, 'metadata': []}, {'prefix': 'p3db.protein', 'mirId': 'MIR:00000501', 'name': 'P3DB Protein', 'pattern': '^\\\\d+$', 'description': 'Plant Protein Phosphorylation DataBase (P3DB) is a database that provides information on experimentally determined phosphorylation sites in the proteins of various plant species. This collection references plant proteins that contain phosphorylation sites.', 'created': '2019-06-11T14:17:34.528+0000', 'modified': '2019-06-11T14:17:34.528+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '70', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1420'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1420'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1420/contactPerson'}}, 'metadata': []}, {'prefix': 'p3db.site', 'mirId': 'MIR:00000502', 'name': 'P3DB Site', 'pattern': '^\\\\d+$', 'description': 'Plant Protein Phosphorylation DataBase (P3DB) is a database that provides information on experimentally determined phosphorylation sites in the proteins of various plant species. This collection references phosphorylation sites in proteins.', 'created': '2019-06-11T14:17:34.763+0000', 'modified': '2019-06-11T14:17:34.763+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '65', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1423'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1423'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1423/contactPerson'}}, 'metadata': []}, {'prefix': 'topdb', 'mirId': 'MIR:00000503', 'name': 'TOPDB', 'pattern': '^[A-Z0-9]+$', 'description': 'The Topology Data Bank of Transmembrane Proteins (TOPDB) is a collection of transmembrane protein datasets containing experimentally derived topology information. It contains information gathered from the literature and from public databases availableon transmembrane proteins. Each record in TOPDB also contains information on the given protein sequence, name, organism and cross references to various other databases.', 'created': '2019-06-11T14:17:35.217+0000', 'modified': '2019-06-11T14:17:35.217+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AP00378', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1425'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1425'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1425/contactPerson'}}, 'metadata': []}, {'prefix': 'cattleqtldb', 'mirId': 'MIR:00000504', 'name': 'Animal Genome Cattle QTL', 'pattern': '^\\\\d+$', 'description': 'The Animal Quantitative Trait Loci (QTL) database (Animal QTLdb) is designed to house publicly all available QTL and single-nucleotide polymorphism/gene association data on livestock animal species. This collection references cattle QTLs.', 'created': '2019-06-11T14:17:35.485+0000', 'modified': '2019-06-11T14:17:35.485+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4685', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1428'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1428'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1428/contactPerson'}}, 'metadata': []}, {'prefix': 'chickenqtldb', 'mirId': 'MIR:00000505', 'name': 'Animal Genome Chicken QTL', 'pattern': '^\\\\d+$', 'description': 'The Animal Quantitative Trait Loci (QTL) database (Animal QTLdb) is designed to house publicly all available QTL and single-nucleotide polymorphism/gene association data on livestock animal species. This collection references chicken QTLs.', 'created': '2019-06-11T14:17:35.752+0000', 'modified': '2019-06-11T14:17:35.752+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '14362', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1431'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1431'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1431/contactPerson'}}, 'metadata': []}, {'prefix': 'pigqtldb', 'mirId': 'MIR:00000506', 'name': 'Animal Genome Pig QTL', 'pattern': '^\\\\d+$', 'description': 'The Animal Quantitative Trait Loci (QTL) database (Animal QTLdb) is designed to house publicly all available QTL and single-nucleotide polymorphism/gene association data on livestock animal species. This collection references pig QTLs.', 'created': '2019-06-11T14:17:35.969+0000', 'modified': '2019-06-11T14:17:35.969+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '14', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1433'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1433'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1433/contactPerson'}}, 'metadata': []}, {'prefix': 'sheepqtldb', 'mirId': 'MIR:00000507', 'name': 'Animal Genome Sheep QTL', 'pattern': '^\\\\d+$', 'description': 'The Animal Quantitative Trait Loci (QTL) database (Animal QTLdb) is designed to house publicly all available QTL and single-nucleotide polymorphism/gene association data on livestock animal species. This collection references sheep QTLs.', 'created': '2019-06-11T14:17:36.183+0000', 'modified': '2019-06-11T14:17:36.183+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '19803', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1435'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1435'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1435/contactPerson'}}, 'metadata': []}, {'prefix': 'gramene.growthstage', 'mirId': 'MIR:00000508', 'name': 'Gramene Growth Stage Ontology', 'pattern': '^GRO\\\\:\\\\d+$', 'description': 'Gramene is a comparative genome mapping database for grasses and crop plants. It combines a semi-automatically generated database of cereal genomic and expressed sequence tag sequences, genetic maps, map relations, quantitative trait loci (QTL), and publications, with a curated database of mutants (genes and alleles), molecular markers, and proteins. This collection refers to growth stage ontology information in Gramene.', 'created': '2019-06-11T14:17:36.384+0000', 'modified': '2019-06-11T14:17:36.384+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0007133', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1437'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1437'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1437/contactPerson'}}, 'metadata': []}, {'prefix': 'mgnify.samp', 'mirId': 'MIR:00000510', 'name': 'MGnify Sample', 'pattern': '^[A-Z]+[0-9]+$', 'description': 'The EBI Metagenomics service is an automated pipeline for the analysis and archiving of metagenomic data that aims to provide insights into the phylogenetic diversity as well as the functional and metabolic potential of a sample. Metagenomics is the study of all genomes present in any given environment without the need for prior individual identification or amplification. This collection references samples.', 'created': '2019-06-11T14:17:36.594+0000', 'modified': '2019-06-11T14:17:36.594+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SRS086444', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1439'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1439'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1439/contactPerson'}}, 'metadata': []}, {'prefix': 'ega.study', 'mirId': 'MIR:00000511', 'name': 'European Genome-phenome Archive Study', 'pattern': '^EGAS\\\\d{11}$', 'description': \"The EGA is a service for permanent archiving and sharing of all types of personally identifiable genetic and phenotypic data resulting from biomedical research projects. The EGA contains exclusive data collected from individuals whose consent agreements authorize data release only for specific research use or to bona fide researchers. Strict protocols govern how information is managed, stored and distributed by the EGA project. This collection references 'Studies' which are experimental investigations of a particular phenomenon, often drawn from different datasets.\", 'created': '2019-06-11T14:17:36.823+0000', 'modified': '2019-06-11T14:17:36.823+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EGAS00000000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1441'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1441'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1441/contactPerson'}}, 'metadata': []}, {'prefix': 'ega.dataset', 'mirId': 'MIR:00000512', 'name': 'European Genome-phenome Archive Dataset', 'pattern': '^EGAD\\\\d{11}$', 'description': \"The EGA is a service for permanent archiving and sharing of all types of personally identifiable genetic and phenotypic data resulting from biomedical research projects. The EGA contains exclusive data collected from individuals whose consent agreements authorize data release only for specific research use or to bona fide researchers. Strict protocols govern how information is managed, stored and distributed by the EGA project. This collection references 'Datasets'.\", 'created': '2019-06-11T14:17:37.233+0000', 'modified': '2019-06-11T14:17:37.233+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EGAD00000000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1444'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1444'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1444/contactPerson'}}, 'metadata': []}, {'prefix': 'pride.project', 'mirId': 'MIR:00000515', 'name': 'PRIDE Project', 'pattern': '^P(X|R)D\\\\d{6}$', 'description': 'The PRIDE PRoteomics IDEntifications database is a centralized, standards compliant, public data repository that provides protein and peptide identifications together with supporting evidence. This collection references projects.', 'created': '2019-06-11T14:17:38.030+0000', 'modified': '2019-06-11T14:17:38.030+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PXD000440', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1453'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1453'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1453/contactPerson'}}, 'metadata': []}, {'prefix': 'antibodyregistry', 'mirId': 'MIR:00000516', 'name': 'Antibody Registry', 'pattern': '^\\\\d{6}$', 'description': 'The Antibody Registry provides identifiers for antibodies used in publications. It lists commercial antibodies from numerous vendors, each assigned with a unique identifier. Unlisted antibodies can be submitted by providing the catalog number and vendor information.', 'created': '2019-06-11T14:17:38.379+0000', 'modified': '2019-06-11T14:17:38.379+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '493771', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1456'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1456'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1456/contactPerson'}}, 'metadata': []}, {'prefix': 'mamo', 'mirId': 'MIR:00000517', 'name': 'Mathematical Modelling Ontology', 'pattern': '^MAMO_\\\\d{7}$', 'description': 'The Mathematical Modelling Ontology (MAMO) is a classification of the types of mathematical models used mostly in the life sciences, their variables, relationships and other relevant features.', 'created': '2019-06-11T14:17:38.625+0000', 'modified': '2019-06-11T14:17:38.625+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MAMO_0000026', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1459'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1459'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1459/contactPerson'}}, 'metadata': []}, {'prefix': 'eo', 'mirId': 'MIR:00000518', 'name': 'Plant Environment Ontology', 'pattern': '^(P)?EO\\\\:\\\\d{7}$', 'description': 'The Plant Environment Ontology is a set of standardized controlled vocabularies to describe various types of treatments given to an individual plant / a population or a cultured tissue and/or cell type sample to evaluate the response on its exposure. It also includes the study types, where the terms can be used to identify the growth study facility. Each growth facility such as field study, growth chamber, green house etc is a environment on its own it may also involve instances of biotic and abiotic environments as supplemental treatments used in these studies.', 'created': '2019-06-11T14:17:38.985+0000', 'modified': '2019-06-11T14:17:38.985+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0007404', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1462'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1462'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1462/contactPerson'}}, 'metadata': []}, {'prefix': 'idot', 'mirId': 'MIR:00000519', 'name': 'Identifiers.org Terms', 'pattern': '^[A-Za-z]+$', 'description': 'Identifiers.org Terms (idot) is an RDF vocabulary providing useful terms for describing datasets.', 'created': '2019-06-11T14:17:39.487+0000', 'modified': '2019-06-11T14:17:39.487+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'identifierPattern', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1466'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1466'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1466/contactPerson'}}, 'metadata': []}, {'prefix': 'hgnc.family', 'mirId': 'MIR:00000520', 'name': 'HGNC Family', 'pattern': '^[A-Z0-9-]+(#[A-Z0-9-]+)?$', 'description': 'The HGNC (HUGO Gene Nomenclature Committee) provides an approved gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database, and each symbol is unique. In addition, HGNC also provides symbols for both structural and functional gene families. This collection refers to records using the HGNC family symbol.', 'created': '2019-06-11T14:17:39.700+0000', 'modified': '2019-06-11T14:17:39.700+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PADI', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1468'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1468'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1468/contactPerson'}}, 'metadata': []}, {'prefix': 'yeastintron', 'mirId': 'MIR:00000521', 'name': 'Yeast Intron Database v4.3', 'pattern': '^[A-Z0-9]+$', 'description': 'The YEast Intron Database (version 4.3) contains information on the spliceosomal introns of the yeast Saccharomyces cerevisiae. It includes expression data that relates to the efficiency of splicing relative to other processes in strains of yeast lacking nonessential splicing factors. The data are displayed on each intron page. This is an updated version of the previous dataset, which can be accessed through [MIR:00000460].', 'created': '2019-06-11T14:17:39.906+0000', 'modified': '2019-06-11T14:17:39.906+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SNR17A', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1470'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1470'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1470/contactPerson'}}, 'metadata': []}, {'prefix': 'ardb', 'mirId': 'MIR:00000522', 'name': 'Antibiotic Resistance Genes Database', 'pattern': '^[A-Z_]{3}[0-9]{4,}$', 'description': 'The Antibiotic Resistance Genes Database (ARDB) is a manually curated database which characterises genes involved in antibiotic resistance. Each gene and resistance type is annotated with information, including resistance profile, mechanism of action, ontology, COG and CDD annotations, as well as external links to sequence and protein databases. This collection references resistance genes.', 'created': '2019-06-11T14:17:40.104+0000', 'modified': '2019-06-11T14:17:40.104+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CAE46076', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1472'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1472'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1472/contactPerson'}}, 'metadata': []}, {'prefix': 'proteomicsdb.protein', 'mirId': 'MIR:00000524', 'name': 'ProteomicsDB Protein', 'pattern': '^\\\\d+$', 'description': 'ProteomicsDB is an effort dedicated to expedite the identification of the human proteome and its use across the scientific community. This human proteome data is assembled primarily using information  from liquid chromatography tandem-mass-spectrometry (LC-MS/MS) experiments involving human tissues, cell lines and body fluids. Information is accessible for individual proteins, or on the basis of protein coverage on the encoding chromosome, and for peptide components of a protein. This collection provides access to individual proteins.', 'created': '2019-06-11T14:17:40.344+0000', 'modified': '2019-06-11T14:17:40.344+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '53504', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1475'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1475'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1475/contactPerson'}}, 'metadata': []}, {'prefix': 'proteomicsdb.peptide', 'mirId': 'MIR:00000525', 'name': 'ProteomicsDB Peptide', 'pattern': '^\\\\d+$', 'description': 'ProteomicsDB is an effort dedicated to expedite the identification of the human proteome and its use across the scientific community. This human proteome data is assembled primarily using information from liquid chromatography tandem-mass-spectrometry (LC-MS/MS) experiments involving human tissues, cell lines and body fluids. Information is accessible for individual proteins, or on the basis of protein coverage on the encoding chromosome, and for peptide components of a protein. This collection provides access to the peptides identified for a given protein.', 'created': '2019-06-11T14:17:40.583+0000', 'modified': '2019-06-11T14:17:40.583+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '53504', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1478'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1478'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1478/contactPerson'}}, 'metadata': []}, {'prefix': 'hpm.protein', 'mirId': 'MIR:00000526', 'name': 'Human Proteome Map Protein', 'pattern': '^\\\\d+$', 'description': 'The Human Proteome Map (HPM) portal integrates the peptide sequencing result from the draft map of the human proteome project. The project was based on LC-MS/MS by utilizing of high resolution and high accuracy Fourier transform mass spectrometry. The HPM contains direct evidence of translation of a number of protein products derived from human genes, based on peptide identifications of multiple organs/tissues and cell types from individuals with clinically defined healthy tissues. The HPM portal provides data on individual proteins, as well as on individual peptide spectra. This collection references proteins.', 'created': '2019-06-11T14:17:40.825+0000', 'modified': '2019-06-11T14:17:40.825+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1968', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1481'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1481'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1481/contactPerson'}}, 'metadata': []}, {'prefix': 'hpm.peptide', 'mirId': 'MIR:00000527', 'name': 'Human Proteome Map Peptide', 'pattern': '^\\\\d+$', 'description': 'The Human Proteome Map (HPM) portal integrates the peptide sequencing result from the draft map of the human proteome project. The project was based on LC-MS/MS by utilizing of high resolution and high accuracy Fourier transform mass spectrometry. The HPM contains direct evidence of translation of a number of protein products derived from human genes, based on peptide identifications of multiple organs/tissues and cell types from individuals with clinically defined healthy tissues. The HPM portal provides data on individual proteins, as well as on individual peptide spectra. This collection references individual peptides through spectra.', 'created': '2019-06-11T14:17:41.032+0000', 'modified': '2019-06-11T14:17:41.032+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '9606117', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1483'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1483'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1483/contactPerson'}}, 'metadata': []}, {'prefix': 'drugbankv4.target', 'mirId': 'MIR:00000528', 'name': 'DrugBank Target v4', 'pattern': '^BE\\\\d{7}$', 'description': 'The DrugBank database is a bioinformatics and chemoinformatics resource that combines detailed drug (i.e. chemical, pharmacological and pharmaceutical) data with comprehensive drug target (i.e. sequence, structure, and pathway) information. This collection references target information from version 4 of the database.', 'created': '2019-06-11T14:17:41.273+0000', 'modified': '2019-06-11T14:17:41.273+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BE0000048', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1485'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1485'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1485/contactPerson'}}, 'metadata': []}, {'prefix': 'zinc', 'mirId': 'MIR:00000529', 'name': 'ZINC', 'pattern': '^(ZINC)?\\\\d+$', 'description': 'ZINC is a free public resource for ligand discovery. The database contains over twenty million commercially available molecules in biologically relevant representations that may be downloaded in popular ready-to-dock formats and subsets. The Web site enables searches by structure, biological activity, physical property, vendor, catalog number, name, and CAS number.', 'created': '2019-06-11T14:17:41.559+0000', 'modified': '2019-06-11T14:17:41.559+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ZINC1084', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1488'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1488'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1488/contactPerson'}}, 'metadata': []}, {'prefix': 'foodb.compound', 'mirId': 'MIR:00000530', 'name': 'FooDB Compound', 'pattern': '^FDB\\\\d+$', 'description': 'FooDB is resource on food and its constituent compounds. It includes data on the compound‚Äôs nomenclature, its description, information on its structure, chemical class, its physico-chemical data, its food source(s), its color, its aroma, its taste, its physiological effect, presumptive health effects (from published studies), and concentrations in various foods. This collection references compounds.', 'created': '2019-06-11T14:17:41.796+0000', 'modified': '2019-06-11T14:17:41.796+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'FDB002100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1491'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1491'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1491/contactPerson'}}, 'metadata': []}, {'prefix': 'unii', 'mirId': 'MIR:00000531', 'name': 'UNII', 'pattern': '^[A-Z0-9]+$', 'description': 'The purpose of the joint FDA/USP Substance Registration System (SRS) is to support health information technology initiatives by generating unique ingredient identifiers (UNIIs) for substances in drugs, biologics, foods, and devices. The UNII is a non- proprietary, free, unique, unambiguous, non semantic, alphanumeric identifier based on a substance‚Äôs molecular structure and/or descriptive information.', 'created': '2019-06-11T14:17:41.988+0000', 'modified': '2019-06-11T14:17:41.988+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3G6A5W338E', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1493'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1493'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1493/contactPerson'}}, 'metadata': []}, {'prefix': 'orphanet.ordo', 'mirId': 'MIR:00000532', 'name': 'Orphanet Rare Disease Ontology', 'pattern': '^Orphanet(_|:)C?\\\\d+$', 'description': 'The Orphanet Rare Disease ontology (ORDO) is a structured vocabulary for rare diseases, capturing relationships between diseases, genes and other relevant features which will form a useful resource for the computational analysis of rare diseases.\\r\\nIt integrates a nosology (classification of rare diseases), relationships (gene-disease relations, epiemological data) and connections with other terminologies (MeSH, UMLS, MedDRA), databases (OMIM, UniProtKB, HGNC, ensembl, Reactome, IUPHAR, Geantlas) and classifications (ICD10).', 'created': '2019-06-11T14:17:42.194+0000', 'modified': '2019-06-11T14:17:42.194+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Orphanet_C023', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1495'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1495'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1495/contactPerson'}}, 'metadata': []}, {'prefix': 'psipar', 'mirId': 'MIR:00000533', 'name': 'Protein Affinity Reagents', 'pattern': '^PAR:\\\\d+$', 'description': 'Protein Affinity Reagents (PSI-PAR) provides a structured controlled vocabulary for the annotation of experiments concerned with interactions, and interactor production methods. PAR is developed by the HUPO Proteomics Standards Initiative and contains the majority of the terms from the PSI-MI controlled vocabular, as well as additional terms.', 'created': '2019-06-11T14:17:42.461+0000', 'modified': '2019-06-11T14:17:42.461+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PAR:0116', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1497'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1497'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1497/contactPerson'}}, 'metadata': []}, {'prefix': 'clinvar.record', 'mirId': 'MIR:00000534', 'name': 'ClinVar Record', 'pattern': '^RCV\\\\d+(\\\\.\\\\d+)?$', 'description': 'ClinVar archives reports of relationships among medically important variants and phenotypes. It records human variation, interpretations of the relationship specific variations to human health, and supporting evidence for each interpretation. Each ClinVar record (RCV identifier) represents an aggregated view of interpretations of  the same variation and condition from one or more submitters. Submissions for individual variation/phenotype combinations (SCV identifier) are also collected and made available separately. This collection references the Record Report, based on RCV accession.', 'created': '2019-06-11T14:17:42.685+0000', 'modified': '2019-06-11T14:17:42.685+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RCV000033555.3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1499'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1499'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1499/contactPerson'}}, 'metadata': []}, {'prefix': 'mgnify.proj', 'mirId': 'MIR:00000535', 'name': 'MGnify Project', 'pattern': '^[A-Z]+[0-9]+$', 'description': 'MGnify is a resource for the analysis and archiving of microbiome data to help determine the taxonomic diversity and functional & metabolic potential of environmental samples. Users can submit their own data for analysis or freely browse all of the analysed public datasets held within the repository. In addition, users can request analysis of any appropriate dataset within the European Nucleotide Archive (ENA). User-submitted or ENA-derived datasets can also be assembled on request, prior to analysis.', 'created': '2019-06-11T14:17:42.938+0000', 'modified': '2019-06-11T14:17:42.938+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ERP004492', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1502'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1502'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1502/contactPerson'}}, 'metadata': []}, {'prefix': 'euclinicaltrials', 'mirId': 'MIR:00000536', 'name': 'EU Clinical Trials', 'pattern': '^\\\\d{4}\\\\-\\\\d{6}\\\\-\\\\d{2}$', 'description': 'The EU Clinical Trials Register contains information on clinical trials conducted in the European Union (EU), or the European Economic Area (EEA) which started after 1 May 2004.\\r\\nIt also includes trials conducted outside these areas if they form part of a paediatric investigation plan (PIP), or are sponsored by a marketing authorisation holder, and involve the use of a medicine in the paediatric population.', 'created': '2019-06-11T14:17:43.129+0000', 'modified': '2019-06-11T14:17:43.129+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2008-005144-16', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1504'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1504'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1504/contactPerson'}}, 'metadata': []}, {'prefix': 'google.patent', 'mirId': 'MIR:00000537', 'name': 'Google Patents', 'pattern': '^[A-Z]{2}\\\\d+([A-Z])?$', 'description': 'Google Patents covers the entire collection of granted patents and published patent applications from the USPTO, EPO, and WIPO. US patent documents date back to 1790, EPO and WIPO to 1978. Google Patents can be searched using patent number, inventor, classification, and filing date.', 'created': '2019-06-11T14:17:43.371+0000', 'modified': '2019-06-11T14:17:43.371+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'US4145692', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1507'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1507'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1507/contactPerson'}}, 'metadata': []}, {'prefix': 'uspto', 'mirId': 'MIR:00000538', 'name': 'USPTO', 'pattern': '^(D|PP|R|T|H|X|AI)?\\\\d+$', 'description': \"The United States Patent and Trademark Office (USPTO) is the federal agency for granting U.S. patents and registering trademarks. As a mechanism that protects new ideas and investments in innovation and creativity, the USPTO is at the cutting edge of the nation's technological progress and achievement.\", 'created': '2019-06-11T14:17:43.583+0000', 'modified': '2019-06-11T14:17:43.583+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4145692', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1510'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1510'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1510/contactPerson'}}, 'metadata': []}, {'prefix': 'cpc', 'mirId': 'MIR:00000539', 'name': 'Cooperative Patent Classification', 'pattern': '^([A-H,Y]|[A-H, Y]\\\\d{2}|[A-H, Y]\\\\d{2}[A-Z]|[A-H, Y]\\\\d{2}[A-Z]\\\\d{1,3}|[A-H, Y]\\\\d{2}[A-Z]\\\\d{1,3}(\\\\/)?\\\\d{2,})$', 'description': 'The Cooperative Patent Classification (CPC) is a patent classification system, developed jointly by the European Patent Office (EPO) and the United States Patent and Trademark Office (USPTO). It is based on the previous European classification system (ECLA), which itself was a version of the International Patent Classification (IPC) system. The CPC patent classification system has been used by EPO and USPTO since 1st January, 2013.', 'created': '2019-06-11T14:17:43.853+0000', 'modified': '2019-06-11T14:17:43.853+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'A01M1/026', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1513'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1513'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1513/contactPerson'}}, 'metadata': []}, {'prefix': 'gwascentral.study', 'mirId': 'MIR:00000540', 'name': 'GWAS Central Study', 'pattern': '^HGVST\\\\d+$', 'description': 'GWAS Central (previously the Human Genome Variation database of Genotype-to-Phenotype information) is a database of summary level findings from genetic association studies, both large and small. It gathers datasets from public domain projects, and accepts direct data submission. It is based upon Marker information encompassing SNP and variant information from public databases, to which allele and genotype frequency data, and genetic association findings are additionally added. A Study (most generic level) contains one or more Experiments, one or more Sample Panels of test subjects, and one or more Phenotypes. This collection references a GWAS Central Study.', 'created': '2019-06-11T14:17:44.062+0000', 'modified': '2019-06-11T14:17:44.062+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HGVST1828', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1516'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1516'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1516/contactPerson'}}, 'metadata': []}, {'prefix': 'exac.variant', 'mirId': 'MIR:00000541', 'name': 'ExAC Variant', 'pattern': '^\\\\d{1,2}\\\\-\\\\d+\\\\-[GATC]\\\\-[GATC]$', 'description': 'The Exome Aggregation Consortium (ExAC) is a coalition of investigators seeking to aggregate and harmonize exome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. The data pertains to unrelated individuals sequenced as part of various disease-specific and population genetic studies and serves as a reference set of allele frequencies for severe disease studies. This collection references variant information.', 'created': '2019-06-11T14:17:44.275+0000', 'modified': '2019-06-11T14:17:44.275+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '22-46615880-T-C', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1518'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1518'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1518/contactPerson'}}, 'metadata': []}, {'prefix': 'gwascentral.marker', 'mirId': 'MIR:00000542', 'name': 'GWAS Central Marker', 'pattern': '^HGVM\\\\d+$', 'description': 'GWAS Central (previously the Human Genome Variation database of Genotype-to-Phenotype information) is a database of summary level findings from genetic association studies, both large and small. It gathers datasets from public domain projects, and accepts direct data submission. It is based upon Marker information encompassing SNP and variant information from public databases, to which allele and genotype frequency data, and genetic association findings are additionally added. A Study (most generic level) contains one or more Experiments, one or more Sample Panels of test subjects, and one or more Phenotypes. This collection references a GWAS Central Marker.', 'created': '2019-06-11T14:17:44.511+0000', 'modified': '2019-06-11T14:17:44.511+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HGVM15354', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1521'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1521'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1521/contactPerson'}}, 'metadata': []}, {'prefix': 'bigg.metabolite', 'mirId': 'MIR:00000556', 'name': 'BiGG Metabolite', 'pattern': '^[a-z_A-Z0-9]+$', 'description': 'BiGG is a knowledgebase of Biochemically, Genetically and Genomically structured genome-scale metabolic network reconstructions. It more published genome-scale metabolic networks into a single database with a set of stardized identifiers called BiGG IDs. Genes in the BiGG models are mapped to NCBI genome annotations, and metabolites are linked to many external databases (KEGG, PubChem, and many more). This collection references individual metabolotes.', 'created': '2019-06-11T14:17:47.850+0000', 'modified': '2019-06-11T14:17:47.850+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '12dgr161', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1555'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1555'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1555/contactPerson'}}, 'metadata': []}, {'prefix': 'gwascentral.phenotype', 'mirId': 'MIR:00000543', 'name': 'GWAS Central Phenotype', 'pattern': '^HGVP\\\\d+$', 'description': 'GWAS Central (previously the Human Genome Variation database of Genotype-to-Phenotype information) is a database of summary level findings from genetic association studies, both large and small. It gathers datasets from public domain projects, and accepts direct data submission. It is based upon Marker information encompassing SNP and variant information from public databases, to which allele and genotype frequency data, and genetic association findings are additionally added. A Study (most generic level) contains one or more Experiments, one or more Sample Panels of test subjects, and one or more Phenotypes. This collection references a GWAS Central Phenotype.', 'created': '2019-06-11T14:17:44.698+0000', 'modified': '2019-06-11T14:17:44.698+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HGVPM623', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1523'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1523'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1523/contactPerson'}}, 'metadata': []}, {'prefix': 'lincs.cell', 'mirId': 'MIR:00000544', 'name': 'LINCS Cell', 'pattern': '(^LCL-\\\\d+$)|(^LDC-\\\\d+$)|(^ES-\\\\d+$)|(^LSC-\\\\d+$)|(^LPC-\\\\d+$)', 'description': 'The Library of Network-Based Cellular Signatures (LINCS) Program aims to  create a network-based understanding of biology by cataloging changes in gene expression and other cellular processes that occur when cells are exposed to a variety of perturbing agents. The LINCS cell model system can have the following cell categories: cell lines, primary cells, induced pluripotent stem cells, differentiated cells, and embryonic stem cells. The metadata contains information provided by each LINCS Data and Signature Generation Center (DSGC) and the association with a tissue or organ from which the cells were derived, in many cases are also associated to a disease.', 'created': '2019-06-11T14:17:44.903+0000', 'modified': '2019-06-11T14:17:44.903+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LCL-2085', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1525'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1525'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1525/contactPerson'}}, 'metadata': []}, {'prefix': 'lincs.protein', 'mirId': 'MIR:00000545', 'name': 'LINCS Protein', 'pattern': '^\\\\d+$', 'description': 'The HMS LINCS Database currently contains information on experimental reagents (small molecule perturbagens, cells, and proteins). It aims to collect and disseminate information relating to the fundamental principles of cellular response in humans to perturbation. This collection references proteins.', 'created': '2019-06-11T14:17:45.145+0000', 'modified': '2019-06-11T14:17:45.145+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '200282', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1528'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1528'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1528/contactPerson'}}, 'metadata': []}, {'prefix': 'lincs.smallmolecule', 'mirId': 'MIR:00000546', 'name': 'LINCS Small Molecule', 'pattern': '^LSM-\\\\d+$', 'description': 'The Library of Network-Based Cellular Signatures (LINCS) Program aims to create a network-based understanding of biology by cataloging changes in gene expression and other cellular processes that occur when cells are exposed to a variety of perturbing agents. The LINCS small molecule collection is used as perturbagens in LINCS experiments. The small molecule metadata includes substance-specific batch information provided by each LINCS Data and Signature Generation Center (DSGC).', 'created': '2019-06-11T14:17:45.396+0000', 'modified': '2019-06-11T14:17:45.396+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LSM-6306', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1530'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1530'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1530/contactPerson'}}, 'metadata': []}, {'prefix': 'exac.transcript', 'mirId': 'MIR:00000547', 'name': 'ExAC Transcript', 'pattern': '^ENST\\\\d{11}$', 'description': 'The Exome Aggregation Consortium (ExAC) is a coalition of investigators seeking to aggregate and harmonize exome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. The data pertains to unrelated individuals sequenced as part of various disease-specific and population genetic studies and serves as a reference set of allele frequencies for severe disease studies. This collection references transcript information.', 'created': '2019-06-11T14:17:45.691+0000', 'modified': '2019-06-11T14:17:45.691+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ENST00000407236', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1533'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1533'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1533/contactPerson'}}, 'metadata': []}, {'prefix': 'exac.gene', 'mirId': 'MIR:00000548', 'name': 'ExAC Gene', 'pattern': '^ENSG\\\\d{11}$', 'description': 'The Exome Aggregation Consortium (ExAC) is a coalition of investigators seeking to aggregate and harmonize exome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. The data pertains to unrelated individuals sequenced as part of various disease-specific and population genetic studies and serves as a reference set of allele frequencies for severe disease studies. This collection references gene information.', 'created': '2019-06-11T14:17:45.890+0000', 'modified': '2019-06-11T14:17:45.890+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ENSG00000169174', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1535'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1535'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1535/contactPerson'}}, 'metadata': []}, {'prefix': 'wikidata', 'mirId': 'MIR:00000549', 'name': 'Wikidata', 'pattern': '^Q\\\\d+$', 'description': 'Wikidata is a collaboratively edited knowledge base operated by the Wikimedia Foundation. It is intended to provide a common source of certain types of data which can be used by Wikimedia projects such as Wikipedia. Wikidata functions as a document-oriented database, centred on individual items. Items represent topics, for which basic information is stored that identifies each topic.', 'created': '2019-06-11T14:17:46.087+0000', 'modified': '2019-06-11T14:17:46.087+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q2207226', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1537'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1537'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1537/contactPerson'}}, 'metadata': []}, {'prefix': 'swisslipid', 'mirId': 'MIR:00000550', 'name': 'SwissLipids', 'pattern': '^SLM:\\\\d+$', 'description': 'SwissLipids is a curated resource that provides information about known lipids, including lipid structure, metabolism, interactions, and subcellular and tissue localization. Information is curated from peer-reviewed literature and referenced using established ontologies, and provided with full provenance and evidence codes for curated assertions.', 'created': '2019-06-11T14:17:46.314+0000', 'modified': '2019-06-11T14:17:46.314+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '000048885', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1539'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1539'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1539/contactPerson'}}, 'metadata': []}, {'prefix': 'unipathway.compound', 'mirId': 'MIR:00000551', 'name': 'UniPathway Compound', 'pattern': '^UPC\\\\d{5}$', 'description': 'UniPathway is a manually curated resource of enzyme-catalyzed and spontaneous chemical reactions. It provides a hierarchical representation of metabolic pathways and a controlled vocabulary for pathway annotation in UniProtKB. UniPathway data are cross-linked to existing metabolic resources such as ChEBI/Rhea, KEGG and MetaCyc. This collection references compounds.', 'created': '2019-06-11T14:17:46.612+0000', 'modified': '2019-06-11T14:17:46.612+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'UPC04349', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1542'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1542'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1542/contactPerson'}}, 'metadata': []}, {'prefix': 'seed', 'mirId': 'MIR:00000552', 'name': 'SEED Subsystem', 'pattern': '^\\\\w+$', 'description': 'This cooperative effort, which includes Fellowship for Interpretation of Genomes (FIG), Argonne National Laboratory, and the University of Chicago, focuses on the development of the comparative genomics environment called the SEED. It is a framework to support comparative analysis and annotation of genomes, and the development of curated genomic data (annotation). Curation is performed at the level of subsystems by an expert annotator, across many genomes, and not on a gene by gene basis. This collection references subsystems.', 'created': '2019-06-11T14:17:46.875+0000', 'modified': '2019-06-11T14:17:46.875+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Biotin_biosynthesis', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1545'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1545'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1545/contactPerson'}}, 'metadata': []}, {'prefix': 'seed.compound', 'mirId': 'MIR:00000553', 'name': 'SEED Compound', 'pattern': '^cpd\\\\d+$', 'description': 'This cooperative effort, which includes Fellowship for Interpretation of Genomes (FIG), Argonne National Laboratory, and the University of Chicago, focuses on the development of the comparative genomics environment called the SEED. It is a framework to support comparative analysis and annotation of genomes, and the development of curated genomic data (annotation). Curation is performed at the level of subsystems by an expert annotator, across many genomes, and not on a gene by gene basis. This collection references subsystems.', 'created': '2019-06-11T14:17:47.133+0000', 'modified': '2019-06-11T14:17:47.133+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'cpd15380', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1548'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1548'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1548/contactPerson'}}, 'metadata': []}, {'prefix': 'bigg.model', 'mirId': 'MIR:00000554', 'name': 'BiGG Model', 'pattern': '^[a-z_A-Z0-9]+$', 'description': 'BiGG is a knowledgebase of Biochemically, Genetically and Genomically structured genome-scale metabolic network reconstructions. It more published genome-scale metabolic networks into a single database with a set of stardized identifiers called BiGG IDs. Genes in the BiGG models are mapped to NCBI genome annotations, and metabolites are linked to many external databases (KEGG, PubChem, and many more). This collection references individual models.', 'created': '2019-06-11T14:17:47.362+0000', 'modified': '2019-06-11T14:17:47.362+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'iECABU_c1320', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1550'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1550'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1550/contactPerson'}}, 'metadata': []}, {'prefix': 'bigg.compartment', 'mirId': 'MIR:00000555', 'name': 'BiGG Compartment', 'pattern': '^[a-z_A-Z]+$', 'description': 'BiGG is a knowledgebase of Biochemically, Genetically and Genomically structured genome-scale metabolic network reconstructions. It more published genome-scale metabolic networks into a single database with a set of stardized identifiers called BiGG IDs. Genes in the BiGG models are mapped to NCBI genome annotations, and metabolites are linked to many external databases (KEGG, PubChem, and many more). This collection references model compartments.', 'created': '2019-06-11T14:17:47.621+0000', 'modified': '2019-06-11T14:17:47.621+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'c', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1553'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1553'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1553/contactPerson'}}, 'metadata': []}, {'prefix': 'bigg.reaction', 'mirId': 'MIR:00000557', 'name': 'BiGG Reaction', 'pattern': '^[a-z_A-Z0-9]+$', 'description': 'BiGG is a knowledgebase of Biochemically, Genetically and Genomically structured genome-scale metabolic network reconstructions. It more published genome-scale metabolic networks into a single database with a set of stardized identifiers called BiGG IDs. Genes in the BiGG models are mapped to NCBI genome annotations, and metabolites are linked to many external databases (KEGG, PubChem, and many more). This collection references reactions.', 'created': '2019-06-11T14:17:48.059+0000', 'modified': '2019-06-11T14:17:48.059+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '13GS', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1557'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1557'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1557/contactPerson'}}, 'metadata': []}, {'prefix': 'umls', 'mirId': 'MIR:00000559', 'name': 'UMLS', 'pattern': '^C\\\\d+$', 'description': 'The Unified Medical Language System is a repository of biomedical vocabularies. Vocabularies integrated in the UMLS Metathesaurus include the NCBI taxonomy, Gene Ontology, the Medical Subject Headings (MeSH), OMIM and the Digital Anatomist Symbolic Knowledge Base. UMLS concepts are not only inter-related, but may also be linked to external resources such as GenBank.', 'created': '2019-06-11T14:17:48.502+0000', 'modified': '2019-06-11T14:17:48.502+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C0021390', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1561'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1561'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1561/contactPerson'}}, 'metadata': []}, {'prefix': 'mesh', 'mirId': 'MIR:00000560', 'name': 'MeSH', 'pattern': '^(C|D)\\\\d{6}$', 'description': \"MeSH (Medical Subject Headings) is the National Library of Medicine's controlled vocabulary thesaurus. It consists of sets of terms naming descriptors in a hierarchical structure that permits searching at various levels of specificity. This thesaurus is used by NLM for indexing articles from biomedical journals, cataloguing of books, documents, etc.\", 'created': '2019-06-11T14:17:48.752+0000', 'modified': '2019-06-11T14:17:48.752+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C000100', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1563'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1563'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1563/contactPerson'}}, 'metadata': []}, {'prefix': 'emdb', 'mirId': 'MIR:00000561', 'name': 'Electron Microscopy Data Bank', 'pattern': '^EMD-\\\\d{4}$', 'description': 'The Electron Microscopy Data Bank (EMDB) is a public repository for electron microscopy density maps of macromolecular complexes and subcellular structures. It covers a variety of techniques, including single-particle analysis, electron tomography, and electron (2D) crystallography. The EMDB map distribution format follows the CCP4 definition, which is widely recognized by software packages used by the structural biology community.', 'created': '2019-06-11T14:17:48.947+0000', 'modified': '2019-06-11T14:17:48.947+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EMD-1001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1565'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1565'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1565/contactPerson'}}, 'metadata': []}, {'prefix': 'mirtarbase', 'mirId': 'MIR:00000562', 'name': 'miRTarBase', 'pattern': '^MIRT\\\\d{6}$', 'description': 'miRTarBase is a database of miRNA-target interactions (MTIs), collected manually from relevant literature, following Natural Language Processing of the text to identify research articles related to functional studies of miRNAs. Generally, the collected MTIs are validated experimentally by reporter assay, western blot, microarray and next-generation sequencing experiments.', 'created': '2019-06-11T14:17:49.171+0000', 'modified': '2019-06-11T14:17:49.171+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MIRT000002', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1567'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1567'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1567/contactPerson'}}, 'metadata': []}, {'prefix': 'meddra', 'mirId': 'MIR:00000563', 'name': 'MedDRA', 'pattern': '^\\\\d+$', 'description': 'The Medical Dictionary for Regulatory Activities (MedDRA) was developed by the International Council for Harmonisation of Technical Requirements for Registration of Pharmaceuticals for Human Use (ICH)to provide a standardised medical terminology to facilitate sharing of regulatory information internationally for medical products used by humans. It is used within regulatory processes, safety monitoring, as well as for marketing activities. Products covered by the scope of MedDRA include pharmaceuticals, biologics, vaccines and drug-device combination products. The MedDRA dictionary is organized by System Organ Class (SOC), divided into High-Level Group Terms (HLGT), High-Level Terms (HLT), Preferred Terms (PT) and finally into Lowest Level Terms (LLT).', 'created': '2019-06-11T14:17:49.428+0000', 'modified': '2019-06-11T14:17:49.428+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '10015919', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1570'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1570'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1570/contactPerson'}}, 'metadata': []}, {'prefix': 'dashr', 'mirId': 'MIR:00000564', 'name': 'DASHR', 'pattern': '^(hsa-(let|mir)-\\\\w+(-\\\\w+)?)|(piR-\\\\d+)|(chr\\\\w+.tRNA\\\\d+-\\\\w+)|(chr\\\\w+.tRNA\\\\d+-\\\\w+-tRF\\\\d)|((SNORD|SNORA|ACA|HBII|HBI|U)(-)?\\\\w+)|(HY\\\\d\\\\+(-L\\\\d+)?)|((LSU|SSU|5S)(-rRNA_Hsa)?(-L\\\\d+)?)$', 'description': 'DASHR reports the annotation, expression and evidence for specific RNA processing (cleavage specificity scores/entropy) of human sncRNA genes, precursor and mature sncRNA products across different human tissues and cell types. DASHR integrates information from multiple existing annotation resources for small non-coding RNAs, including microRNAs (miRNAs), Piwi-interacting (piRNAs), small nuclear (snRNAs), nucleolar (snoRNAs), cytoplasmic (scRNAs), transfer (tRNAs), tRNA fragments (tRFs), and ribosomal RNAs (rRNAs). These datasets were obtained from non-diseased human tissues and cell types and were generated for studying or profiling small non-coding RNAs. This collection references RNA records.', 'created': '2019-06-11T14:17:49.634+0000', 'modified': '2019-06-11T14:17:49.634+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'hsa-mir-200a', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1572'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1572'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1572/contactPerson'}}, 'metadata': []}, {'prefix': 'dashr.expression', 'mirId': 'MIR:00000565', 'name': 'DASHR expression', 'pattern': '^(hsa-(let|mir)-\\\\w+(-\\\\w+)?)|(piR-\\\\d+)|(chr\\\\w+.tRNA\\\\d+-\\\\w+)|(chr\\\\w+.tRNA\\\\d+-\\\\w+-tRF\\\\d)|((SNORD|SNORA|ACA|HBII|HBI|U)(-)?\\\\w+)|(HY\\\\d\\\\+(-L\\\\d+)?)|((LSU|SSU|5S)(-rRNA_Hsa)?(-L\\\\d+)?)$', 'description': 'DASHR reports the annotation, expression and evidence for specific RNA processing (cleavage specificity scores/entropy) of human sncRNA genes, precursor and mature sncRNA products across different human tissues and cell types. DASHR integrates information from multiple existing annotation resources for small non-coding RNAs, including microRNAs (miRNAs), Piwi-interacting (piRNAs), small nuclear (snRNAs), nucleolar (snoRNAs), cytoplasmic (scRNAs), transfer (tRNAs), tRNA fragments (tRFs), and ribosomal RNAs (rRNAs). These datasets were obtained from non-diseased human tissues and cell types and were generated for studying or profiling small non-coding RNAs. This collection references RNA expression.', 'created': '2019-06-11T14:17:49.866+0000', 'modified': '2019-06-11T14:17:49.866+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'hsa-mir-200a', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1575'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1575'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1575/contactPerson'}}, 'metadata': []}, {'prefix': 'splash', 'mirId': 'MIR:00000566', 'name': 'SPLASH', 'pattern': '^splash\\\\d[A-Z-a-z0-9]-[A-Za-z0-9]+-[A-Za-z0-9]+$', 'description': 'The spectra hash code (SPLASH) is a unique and non-proprietary identifier for spectra, and is independent of how the spectra were acquired or processed. It can be easily calculated for a wide range of spectra, including Mass spectroscopy,  infrared spectroscopy, ultraviolet and nuclear magnetic resonance.', 'created': '2019-06-11T14:17:50.083+0000', 'modified': '2019-06-11T14:17:50.083+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'splash10-0zq2000000-77302b0326a418630a84', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1577'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1577'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1577/contactPerson'}}, 'metadata': []}, {'prefix': 'metanetx.chemical', 'mirId': 'MIR:00000567', 'name': 'MetaNetX chemical', 'pattern': '^(MNXM\\\\d+|BIOMASS)$', 'description': 'MetaNetX/MNXref integrates various information from genome-scale metabolic network reconstructions such as information on reactions, metabolites and compartments. This information undergoes a reconciliation process to minimise for discrepancies between different data sources, and makes the data accessible under a common namespace. This collection references chemical or metabolic components.', 'created': '2019-06-11T14:17:50.332+0000', 'modified': '2019-06-11T14:17:50.332+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MNXM1723', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1580'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1580'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1580/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org', '@type': 'DataCatalog', 'description': 'Reconciliation of metabolites and biochemical reactions for automated model construction and genome annotation for large-scale metabolic networks', 'keywords': 'metabolism, network, model, metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MetaNetX/MNXref', 'provider': [{'@id': 'https://www.sib.swiss/', '@type': 'Organization', 'name': 'SIB Swiss Intitute of Bioinformatics'}, {'@id': 'https://www.vital-it.ch/', '@type': 'Organization', 'name': 'Vital-IT', 'logo': 'https://vital-it.ch/images/vital-it_logo.png'}], 'url': 'https://www.metanetx.org/', 'citation': [{'@id': 'https://dx.doi.org/10.1093/nar/gkv1117', '@type': 'CreativeWork', 'name': 'MetaNetX/MNXref - reconciliation of metabolites and biochemical reactions to bring together genome-scale metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bioinformatics/btt036', '@type': 'CreativeWork', 'name': 'MetaNetX.org: a website and repository for accessing, analysing and manipulating metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bib/bbs058', '@type': 'CreativeWork', 'name': 'Reconciliation of metabolites and biochemical reactions for metabolic networks'}], 'dataset': [{'@context': 'https://schema.org', '@type': 'Dataset', 'description': 'The MNXref reconciliation of metabolites and biochemical reactions namespace', 'identifier': [], 'keywords': 'metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MNXref namespace', 'url': 'ftp://ftp.vital-it.ch/databases/metanetx/MNXref/latest/', 'license': 'https://creativecommons.org/licenses/by/4.0/'}], 'dateCreated': '2011', 'license': {'@id': 'https://creativecommons.org/licenses/by/4.0/', '@type': 'CreativeWork', 'name': 'Creative Commons Attribution 4.0 International License'}, 'fileFormat': ['text/plain', 'text/html', 'application/sbml+xml', 'chemical/x-inchi', 'chemical/x-daylight-smiles', 'image/png']}]}, {'prefix': 'metanetx.reaction', 'mirId': 'MIR:00000568', 'name': 'MetaNetX reaction', 'pattern': '^MNXR\\\\d+$', 'description': 'MetaNetX/MNXref integrates various information from genome-scale metabolic network reconstructions such as information on reactions, metabolites and compartments. This information undergoes a reconciliation process to minimise for discrepancies between different data sources, and makes the data accessible under a common namespace. This collection references reactions.', 'created': '2019-06-11T14:17:50.605+0000', 'modified': '2019-06-11T14:17:50.605+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MNXR101574', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1583'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1583'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1583/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org', '@type': 'DataCatalog', 'description': 'Reconciliation of metabolites and biochemical reactions for automated model construction and genome annotation for large-scale metabolic networks', 'keywords': 'metabolism, network, model, metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MetaNetX/MNXref', 'provider': [{'@id': 'https://www.sib.swiss/', '@type': 'Organization', 'name': 'SIB Swiss Intitute of Bioinformatics'}, {'@id': 'https://www.vital-it.ch/', '@type': 'Organization', 'name': 'Vital-IT', 'logo': 'https://vital-it.ch/images/vital-it_logo.png'}], 'url': 'https://www.metanetx.org/', 'citation': [{'@id': 'https://dx.doi.org/10.1093/nar/gkv1117', '@type': 'CreativeWork', 'name': 'MetaNetX/MNXref - reconciliation of metabolites and biochemical reactions to bring together genome-scale metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bioinformatics/btt036', '@type': 'CreativeWork', 'name': 'MetaNetX.org: a website and repository for accessing, analysing and manipulating metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bib/bbs058', '@type': 'CreativeWork', 'name': 'Reconciliation of metabolites and biochemical reactions for metabolic networks'}], 'dataset': [{'@context': 'https://schema.org', '@type': 'Dataset', 'description': 'The MNXref reconciliation of metabolites and biochemical reactions namespace', 'identifier': [], 'keywords': 'metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MNXref namespace', 'url': 'ftp://ftp.vital-it.ch/databases/metanetx/MNXref/latest/', 'license': 'https://creativecommons.org/licenses/by/4.0/'}], 'dateCreated': '2011', 'license': {'@id': 'https://creativecommons.org/licenses/by/4.0/', '@type': 'CreativeWork', 'name': 'Creative Commons Attribution 4.0 International License'}, 'fileFormat': ['text/plain', 'text/html', 'application/sbml+xml', 'chemical/x-inchi', 'chemical/x-daylight-smiles', 'image/png']}]}, {'prefix': 'metanetx.compartment', 'mirId': 'MIR:00000569', 'name': 'MetaNetX compartment', 'pattern': '^(MNX[CD]\\\\d+|BOUNDARY)$', 'description': 'MetaNetX/MNXref integrates various information from genome-scale metabolic network reconstructions such as information on reactions, metabolites and compartments. This information undergoes a reconciliation process to minimise for discrepancies between different data sources, and makes the data accessible under a common namespace. This collection references cellular compartments.', 'created': '2019-06-11T14:17:50.805+0000', 'modified': '2019-06-11T14:17:50.805+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MNXC15', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1585'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1585'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1585/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org', '@type': 'DataCatalog', 'description': 'Reconciliation of metabolites and biochemical reactions for automated model construction and genome annotation for large-scale metabolic networks', 'keywords': 'metabolism, network, model, metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MetaNetX/MNXref', 'provider': [{'@id': 'https://www.sib.swiss/', '@type': 'Organization', 'name': 'SIB Swiss Intitute of Bioinformatics'}, {'@id': 'https://www.vital-it.ch/', '@type': 'Organization', 'name': 'Vital-IT', 'logo': 'https://vital-it.ch/images/vital-it_logo.png'}], 'url': 'https://www.metanetx.org/', 'citation': [{'@id': 'https://dx.doi.org/10.1093/nar/gkv1117', '@type': 'CreativeWork', 'name': 'MetaNetX/MNXref - reconciliation of metabolites and biochemical reactions to bring together genome-scale metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bioinformatics/btt036', '@type': 'CreativeWork', 'name': 'MetaNetX.org: a website and repository for accessing, analysing and manipulating metabolic networks'}, {'@id': 'https://dx.doi.org/10.1093/bib/bbs058', '@type': 'CreativeWork', 'name': 'Reconciliation of metabolites and biochemical reactions for metabolic networks'}], 'dataset': [{'@context': 'https://schema.org', '@type': 'Dataset', 'description': 'The MNXref reconciliation of metabolites and biochemical reactions namespace', 'identifier': [], 'keywords': 'metabolite, compound, chemical, biochemical, reaction, namespace, reconciliation', 'name': 'MNXref namespace', 'url': 'ftp://ftp.vital-it.ch/databases/metanetx/MNXref/latest/', 'license': 'https://creativecommons.org/licenses/by/4.0/'}], 'dateCreated': '2011', 'license': {'@id': 'https://creativecommons.org/licenses/by/4.0/', '@type': 'CreativeWork', 'name': 'Creative Commons Attribution 4.0 International License'}, 'fileFormat': ['text/plain', 'text/html', 'application/sbml+xml', 'chemical/x-inchi', 'chemical/x-daylight-smiles', 'image/png']}]}, {'prefix': 'unipathway.reaction', 'mirId': 'MIR:00000570', 'name': 'UniPathway Reaction', 'pattern': '^UCR\\\\d{5}$', 'description': 'UniPathway is a manually curated resource of enzyme-catalyzed and spontaneous chemical reactions. It provides a hierarchical representation of metabolic pathways and a controlled vocabulary for pathway annotation in UniProtKB. UniPathway data are cross-linked to existing metabolic resources such as ChEBI/Rhea, KEGG and MetaCyc. This collection references individual reactions.', 'created': '2019-06-11T14:17:51.017+0000', 'modified': '2019-06-11T14:17:51.017+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'UCR00226', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1587'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1587'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1587/contactPerson'}}, 'metadata': []}, {'prefix': 'hp', 'mirId': 'MIR:00000571', 'name': 'Human Phenotype Ontology', 'pattern': '^HP:\\\\d{7}$', 'description': 'The Human Phenotype Ontology (HPO) aims to provide a standardized vocabulary of phenotypic abnormalities encountered in human disease. Each term in the HPO describes a phenotypic abnormality, such as atrial septal defect. The HPO is currently being developed using the medical literature, Orphanet, DECIPHER, and OMIM.', 'created': '2019-06-11T14:17:51.237+0000', 'modified': '2019-06-11T14:17:51.237+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000118', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1589'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1589'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1589/contactPerson'}}, 'metadata': []}, {'prefix': 'sasbdb', 'mirId': 'MIR:00000572', 'name': 'SASBDB', 'pattern': '^[Ss][Aa][Ss][A-Za-z0-9]{3}[0-9]$', 'description': 'Small Angle Scattering Biological Data Bank (SASBDB) is a curated repository for small angle X-ray scattering (SAXS) and neutron scattering (SANS) data and derived models. Small angle scattering (SAS) of X-ray and neutrons provides structural information on biological macromolecules in solution at a resolution of 1-2 nm. SASBDB provides freely accessible and downloadable experimental data, which are deposited together with the relevant experimental conditions, sample details, derived models and their fits to the data.', 'created': '2019-06-11T14:17:51.650+0000', 'modified': '2019-06-11T14:17:51.650+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SASDAX8', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1593'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1593'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1593/contactPerson'}}, 'metadata': []}, {'prefix': 'hgnc.genefamily', 'mirId': 'MIR:00000573', 'name': 'HGNC gene family', 'pattern': '^\\\\d+$', 'description': 'The HGNC (HUGO Gene Nomenclature Committee) provides an approved gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database, and each symbol is unique. In addition, HGNC also provides a unique numerical ID to identify gene families, providing a display of curated hierarchical relationships between families.', 'created': '2019-06-11T14:17:51.916+0000', 'modified': '2019-06-11T14:17:51.916+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '141', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1596'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1596'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1596/contactPerson'}}, 'metadata': []}, {'prefix': 'mdm', 'mirId': 'MIR:00000574', 'name': 'MDM', 'pattern': '^\\\\d+$', 'description': 'The MDM (Medical Data Models) Portal is a meta-data registry for creating, analysing, sharing and reusing medical forms. Electronic forms are central in numerous processes involving data, including the collection of data through electronic health records (EHRs), Electronic Data Capture (EDC), and as case report forms (CRFs) for clinical trials. The MDM Portal provides medical forms in numerous export formats, facilitating the sharing and reuse of medical data models and exchange between information systems.', 'created': '2019-06-11T14:17:52.138+0000', 'modified': '2019-06-11T14:17:52.138+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4776', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1598'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1598'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1598/contactPerson'}}, 'metadata': []}, {'prefix': 'probonto', 'mirId': 'MIR:00000575', 'name': 'ProbOnto', 'pattern': '^PROB_c\\\\d+$', 'description': 'ProbOnto, is an ontology-based knowledge base of probability distributions, featuring uni- and multivariate distributions with their defining functions, characteristics, relationships and reparameterisation formulae.  It can be used for annotation of models, facilitating the encoding of distribution-based models, related functions and quantities.', 'created': '2019-06-11T14:17:52.483+0000', 'modified': '2019-06-11T14:17:52.483+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PROB_c0000005', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1601'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1601'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1601/contactPerson'}}, 'metadata': []}, {'prefix': 'apid.interactions', 'mirId': 'MIR:00000576', 'name': 'APID Interactomes', 'pattern': '^([A-N,R-Z][0-9]([A-Z][A-Z, 0-9][A-Z, 0-9][0-9]){1,2})|([O,P,Q][0-9][A-Z, 0-9][A-Z, 0-9][A-Z, 0-9][0-9])(\\\\.\\\\d+)?$', 'description': 'APID (Agile Protein Interactomes DataServer) provides information on the protein interactomes of numerous  organisms, based on the integration of known experimentally validated protein-protein physical interactions (PPIs). Interactome data includes a report on  quality levels and coverage over the proteomes for each organism included. APID integrates PPIs from primary databases of molecular interactions (BIND, BioGRID, DIP, HPRD, IntAct, MINT) and also from experimentally resolved 3D structures (PDB) where more than two distinct proteins have been identified. This collection references protein interactors, through a UniProt identifier.', 'created': '2019-06-11T14:17:52.731+0000', 'modified': '2019-06-11T14:17:52.731+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'P01116', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1603'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1603'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1603/contactPerson'}}, 'metadata': []}, {'prefix': 'storedb', 'mirId': 'MIR:00000577', 'name': 'STOREDB', 'pattern': '^STOREDB:(STUDY|FILE|DATASET)\\\\d+$', 'description': 'STOREDB database is a repository for data used by the international radiobiology community, archiving and sharing primary data outputs from research on low dose radiation. It also provides a directory of bioresources and databases for radiobiology projects containing information and materials that investigators are willing to share. STORE supports the creation of a low dose radiation research commons.', 'created': '2019-06-11T14:17:53.021+0000', 'modified': '2019-06-11T14:17:53.021+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'STOREDB:STUDY1040', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1606'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1606'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1606/contactPerson'}}, 'metadata': []}, {'prefix': 'kegg', 'mirId': 'MIR:00000578', 'name': 'Kyoto Encyclopedia of Genes and Genomes', 'pattern': '^([CHDEGTMKR]\\\\d+)|(\\\\w+:[\\\\w\\\\d\\\\.-]*)|([a-z]{3,5})|(\\\\w{2,4}\\\\d{5})$', 'description': 'Kyoto Encyclopedia of Genes and Genomes (KEGG) is a database resource for understanding high-level functions and utilities of the biological system, such as the cell, the organism and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by genome sequencing and other high-throughput experimental technologies.', 'created': '2019-06-11T14:17:53.298+0000', 'modified': '2019-06-11T14:17:53.298+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'hsa00190', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1609'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1609'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1609/contactPerson'}}, 'metadata': []}, {'prefix': 'planttfdb', 'mirId': 'MIR:00000579', 'name': 'Plant Transcription Factor Database', 'pattern': '^[A-Z][a-z]{2}_([A-Za-z]{3}[0-9]{6})|([A-Za-z0-9\\\\._\\\\-#]*)$', 'description': 'The Plant TF database (PlantTFDB) systematically identifies transcription factors for plant species. It includes annotation for identified TFs, including information on expression, regulation, interaction, conserved elements, phenotype information. It also provides curated descriptions and cross-references to other life science databases, as well as identifying evolutionary relationship among identified factors.', 'created': '2019-06-11T14:17:53.641+0000', 'modified': '2019-06-11T14:17:53.641+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Ath_AT1G01030.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1611'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1611'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1611/contactPerson'}}, 'metadata': []}, {'prefix': 'mp', 'mirId': 'MIR:00000580', 'name': 'Mammalian Phenotype Ontology', 'pattern': '^MP:\\\\d{7}$', 'description': 'The Mammalian Phenotype Ontology (MP) classifies and organises phenotypic information related to the mouse and other mammalian species. This ontology has been applied to mouse phenotype descriptions in various databases allowing comparisons of data from diverse mammalian sources. It can facilitate in the identification of appropriate experimental disease models, and aid in the discovery of candidate disease genes and molecular signaling pathways.', 'created': '2019-06-11T14:17:53.926+0000', 'modified': '2019-06-11T14:17:53.926+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0005452', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1614'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1614'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1614/contactPerson'}}, 'metadata': []}, {'prefix': 'worms', 'mirId': 'MIR:00000581', 'name': 'World Register of Marine Species', 'pattern': '^\\\\d+$', 'description': 'The World Register of Marine Species (WoRMS) provides an authoritative and comprehensive list of names of marine organisms. It includes synonyms for valid taxonomic names allowing a more complete interpretation of taxonomic literature. The content of WoRMS is administered by taxonomic experts.', 'created': '2019-06-11T14:17:54.476+0000', 'modified': '2019-06-11T14:17:54.476+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '146421', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1618'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1618'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1618/contactPerson'}}, 'metadata': []}, {'prefix': 'multicellds.collection', 'mirId': 'MIR:00000582', 'name': 'MultiCellDS collection', 'pattern': '^MCDS_C_[a-zA-Z0-9]{1,10}$', 'description': 'MultiCellDS is data standard for multicellular simulation, experimental, and clinical data. A collection groups one or more individual uniquely identified cell lines, snapshots, or collections. Primary uses are times series (collections of snapshots), patient cohorts (collections of cell lines), and studies (collections of time series collections).', 'created': '2019-06-11T14:17:54.753+0000', 'modified': '2019-06-11T14:17:54.753+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MCDS_C_0000000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1621'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1621'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1621/contactPerson'}}, 'metadata': []}, {'prefix': 'multicellds.snapshot', 'mirId': 'MIR:00000583', 'name': 'MultiCellDS Digital snapshot', 'pattern': '^MCDS_S_[a-zA-Z0-9]{1,10}$', 'description': 'MultiCellDS is data standard for multicellular simulation, experimental, and clinical data. A digital snapshot is a single-time output of the microenvironment (including basement membranes and the vascular network), any cells contained within, and essential metadata. Cells may include phenotypic data.', 'created': '2019-06-11T14:17:55.012+0000', 'modified': '2019-06-11T14:17:55.012+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MCDS_S_0000000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1624'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1624'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1624/contactPerson'}}, 'metadata': []}, {'prefix': 'multicellds.cell_line', 'mirId': 'MIR:00000584', 'name': 'MultiCellDS Digital Cell Line', 'pattern': '^MCDS_L_[a-zA-Z0-9]{1,10}$', 'description': 'MultiCellDS is data standard for multicellular simulation, experimental, and clinical data. A digital cell line is a hierarchical organization of quantitative phenotype data for a single biological cell line, including the microenvironmental context of the measurements and essential metadata.', 'created': '2019-06-11T14:17:55.320+0000', 'modified': '2019-06-11T14:17:55.320+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MCDS_L_0000000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1626'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1626'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1626/contactPerson'}}, 'metadata': []}, {'prefix': 'ecyano.entity', 'mirId': 'MIR:00000585', 'name': 'E-cyanobacterium entity', 'pattern': '^\\\\d+$', 'description': 'E-cyanobacterium.org is a web-based platform for public sharing, annotation, analysis, and visualisation of dynamical models and wet-lab experiments related to cyanobacteria. It allows models to be represented at different levels of abstraction ‚Äî as biochemical reaction networks or ordinary differential equations.It provides concise mappings of mathematical models to a formalised consortium-agreed biochemical description, with the aim of connecting the world of biological knowledge with benefits of mathematical description of dynamic processes. This collection references entities.', 'created': '2019-06-11T14:17:55.541+0000', 'modified': '2019-06-11T14:17:55.541+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '23', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1628'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1628'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1628/contactPerson'}}, 'metadata': []}, {'prefix': 'ecyano.model', 'mirId': 'MIR:00000586', 'name': 'E-cyanobacterium model', 'pattern': '^\\\\d+$', 'description': 'E-cyanobacterium.org is a web-based platform for public sharing, annotation, analysis, and visualisation of dynamical models and wet-lab experiments related to cyanobacteria. It allows models to be represented at different levels of abstraction ‚Äî as biochemical reaction networks or ordinary differential equations.It provides concise mappings of mathematical models to a formalised consortium-agreed biochemical description, with the aim of connecting the world of biological knowledge with benefits of mathematical description of dynamic processes. This collection references models.', 'created': '2019-06-11T14:17:55.864+0000', 'modified': '2019-06-11T14:17:55.864+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '26', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1631'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1631'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1631/contactPerson'}}, 'metadata': []}, {'prefix': 'ecyano.rule', 'mirId': 'MIR:00000587', 'name': 'E-cyanobacterium rule', 'pattern': '^\\\\d+$', 'description': 'E-cyanobacterium.org is a web-based platform for public sharing, annotation, analysis, and visualisation of dynamical models and wet-lab experiments related to cyanobacteria. It allows models to be represented at different levels of abstraction ‚Äî as biochemical reaction networks or ordinary differential equations.It provides concise mappings of mathematical models to a formalised consortium-agreed biochemical description, with the aim of connecting the world of biological knowledge with benefits of mathematical description of dynamic processes. This collection references rules.', 'created': '2019-06-11T14:17:56.089+0000', 'modified': '2019-06-11T14:17:56.089+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '56', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1633'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1633'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1633/contactPerson'}}, 'metadata': []}, {'prefix': 'lincs.data', 'mirId': 'MIR:00000588', 'name': 'LINCS Data', 'pattern': '^[EL]D[SG]-\\\\d+$', 'description': 'The Library of Network-Based Cellular Signatures (LINCS) Program aims to create a network-based understanding of biology by cataloguing changes in gene expression and other cellular processes that occur when cells are exposed to perturbing agents. The data is organized and available as datasets, each including experimental data, metadata and a description of the dataset and assay. The dataset group comprises datasets for the same experiment but with different data level results (data processed to a different level).', 'created': '2019-06-11T14:17:56.343+0000', 'modified': '2019-06-11T14:17:56.343+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LDS-1110', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1635'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1635'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1635/contactPerson'}}, 'metadata': []}, {'prefix': 'agricola', 'mirId': 'MIR:00000589', 'name': 'AGRICOLA', 'pattern': '^\\\\d+$', 'description': 'AGRICOLA (AGRICultural OnLine Access) serves as the catalog and index to the collections of the National Agricultural Library, as well as a primary public source for world-wide access to agricultural information. The database covers materials in all formats and periods, including printed works from as far back as the 15th century.', 'created': '2019-06-11T14:17:56.660+0000', 'modified': '2019-06-11T14:17:56.660+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '50018', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1638'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1638'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1638/contactPerson'}}, 'metadata': []}, {'prefix': 'ms', 'mirId': 'MIR:00000590', 'name': 'Mass Spectrometry Controlled Vocabulary', 'pattern': '^MS:\\\\d{7}$', 'description': 'The PSI-Mass Spectrometry (MS) CV contains all the terms used in the PSI MS-related data standards. The CV contains a logical hierarchical structure to ensure ease of maintenance and the development of software that makes use of complex semantics. The CV contains terms required for a complete description of an MS analysis pipeline used in proteomics, including sample labeling, digestion enzymes, instrumentation parts and parameters, software used for identification and quantification of peptides/proteins and the parameters and scores used to determine their significance.', 'created': '2019-06-11T14:17:56.919+0000', 'modified': '2019-06-11T14:17:56.919+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1000001', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1641'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1641'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1641/contactPerson'}}, 'metadata': []}, {'prefix': 'envo', 'mirId': 'MIR:00000591', 'name': 'Environment Ontology', 'pattern': '^ENVO:\\\\d{8}$', 'description': \"The Environment Ontology is a resource and research target for the semantically controlled description of environmental entities. The ontology's initial aim was the representation of the biomes, environmental features, and environmental materials pertinent to genomic and microbiome-related investigations.\", 'created': '2019-06-11T14:17:57.267+0000', 'modified': '2019-06-11T14:17:57.267+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '09200010', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1644'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1644'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1644/contactPerson'}}, 'metadata': []}, {'prefix': 'ark', 'mirId': 'MIR:00000592', 'name': 'ARK', 'pattern': '^(ark\\\\:)/*[0-9A-Za-z]+(?:/[\\\\w/.=*+@\\\\$-]*)?(?:\\\\?.*)?$', 'description': 'An Archival Resource Key (ARK) is a Uniform Resource Locator (URL) that is a multi-purpose persistent identifier for information objects of any type.', 'created': '2019-06-11T14:17:57.583+0000', 'modified': '2019-06-11T14:17:57.583+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '/12345/fk1234', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1647'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1647'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1647/contactPerson'}}, 'metadata': []}, {'prefix': 'grid', 'mirId': 'MIR:00000593', 'name': 'GRID', 'pattern': '^grid\\\\.[0-9]+\\\\.[a-f0-9]{1,2}$', 'description': \"International coverage of the world's leading research organisations, indexing 92% of funding allocated globally.\", 'created': '2019-06-11T14:17:57.806+0000', 'modified': '2019-06-11T14:17:57.806+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'grid.225360.0', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1650'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1650'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1650/contactPerson'}}, 'metadata': []}, {'prefix': 'medgen', 'mirId': 'MIR:00000594', 'name': 'MedGen', 'pattern': '^[CN]*\\\\d{4,7}$', 'description': 'MedGen is a portal for information about conditions and phenotypes related to Medical Genetics. Terms from multiple sources are aggregated into concepts, each of which is assigned a unique identifier and a preferred name and symbol. The core content of the record may include names, identifiers used by other databases, mode of inheritance, clinical features, and map location of the loci affecting the disorder.', 'created': '2019-06-11T14:17:58.068+0000', 'modified': '2019-06-11T14:17:58.068+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '760050', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1653'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1653'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1653/contactPerson'}}, 'metadata': []}, {'prefix': 'clinvar.submission', 'mirId': 'MIR:00000595', 'name': 'ClinVar Submission', 'pattern': '^SCV\\\\d+(\\\\.\\\\d+)?$', 'description': 'ClinVar archives reports of relationships among medically important variants and phenotypes. It records human variation, interpretations of the relationship specific variations to human health, and supporting evidence for each interpretation. Each ClinVar record (RCV identifier) represents an aggregated view of interpretations of the same variation and condition from one or more submitters. Submissions for individual variation/phenotype combinations (SCV identifier) are also collected and made available separately. This collection references submissions, and is based on SCV accession.', 'created': '2019-06-11T14:17:58.273+0000', 'modified': '2019-06-11T14:17:58.273+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'SCV000151292', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1655'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1655'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1655/contactPerson'}}, 'metadata': []}, {'prefix': 'clinvar', 'mirId': 'MIR:00000596', 'name': 'ClinVar Variant', 'pattern': '^\\\\d+$', 'description': 'ClinVar archives reports of relationships among medically important variants and phenotypes. It records human variation, interpretations of the relationship specific variations to human health, and supporting evidence for each interpretation. Each ClinVar record (RCV identifier) represents an aggregated view of interpretations of the same variation and condition from one or more submitters. Submissions for individual variation/phenotype combinations (SCV identifier) are also collected and made available separately. This collection references the Variant identifier.', 'created': '2019-06-11T14:17:58.490+0000', 'modified': '2019-06-11T14:17:58.490+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '12345', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1657'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1657'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1657/contactPerson'}}, 'metadata': []}, {'prefix': 'cellosaurus', 'mirId': 'MIR:00000598', 'name': 'Cellosaurus', 'pattern': 'CVCL_[A-Z0-9]{4}', 'description': 'The Cellosaurus is a knowledge resource on cell lines. It attempts to describe all cell lines used in biomedical research. Its scope includes: Immortalized cell lines; naturally immortal cell lines (example: stem cell lines); finite life cell lines when those are distributed and used widely; vertebrate cell line with an emphasis on human, mouse and rat cell lines; and invertebrate (insects and ticks) cell lines. Its scope does not include primary cell lines (with the exception of the finite life cell lines described above) and plant cell lines.', 'created': '2019-06-11T14:17:59.000+0000', 'modified': '2019-06-11T14:17:59.000+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CVCL_0030', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1662'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1662'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1662/contactPerson'}}, 'metadata': [{'@context': ['https://schema.org/'], '@type': 'Organization', 'url': 'https://web.expasy.org/cellosaurus', 'name': 'Cellosaurus', 'memberof': {'@type': 'NGO', 'name': 'SIB Swiss Institute of Bioinformatics', 'url': 'https://sib.swiss/'}, 'parentOrganization': {'@type': 'Organization', 'name': 'ExPASy', 'url': 'https://www.expasy.org/'}}]}, {'prefix': 'mir', 'mirId': 'MIR:00000599', 'name': 'Identifiers.org Registry', 'pattern': '^MIR:\\\\d{8}$', 'description': 'The Identifiers.org registry contains registered namespace and provider prefixes with associated access URIs for a large number of high quality data collections. These prefixes are used in web resolution of compact identifiers of the form ‚ÄúPREFIX:ACCESSION‚Äù or \"PROVIDER/PREFIX:ACCESSION‚Äù commonly used to specify bioinformatics and other data resources.', 'created': '2019-06-11T14:17:59.221+0000', 'modified': '2019-06-11T14:17:59.221+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '00100037', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1664'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1664'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1664/contactPerson'}}, 'metadata': []}, {'prefix': 'pgx', 'mirId': 'MIR:00000600', 'name': 'Progenetix', 'pattern': '^[\\\\w\\\\-:,]{3,64}$', 'description': 'The Progenetix database provides an overview of copy number abnormalities in human cancer from currently 32317 array and chromosomal Comparative Genomic Hybridization (CGH) experiments, as well as Whole Genome or Whole Exome Sequencing (WGS, WES) studies. The cancer profile data in Progenetix was curated from 1026 articles and represents 364 different cancer types, according to the International classification of Diseases in Oncology (ICD-O).', 'created': '2019-06-11T14:17:59.445+0000', 'modified': '2019-06-11T14:17:59.445+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'icdom:8500_3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1666'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1666'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1666/contactPerson'}}, 'metadata': []}, {'prefix': 'cosmic', 'mirId': 'MIR:00000601', 'name': 'COSMIC Gene', 'pattern': '^[A-Z0-9]+$', 'description': 'COSMIC is a comprehensive global resource for information on somatic mutations in human cancer, combining curation of the scientific literature with tumor resequencing data from the Cancer Genome Project at the Sanger Institute, U.K. This collection references genes.', 'created': '2019-06-11T14:17:59.699+0000', 'modified': '2019-06-11T14:17:59.699+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BRAF', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1669'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1669'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1669/contactPerson'}}, 'metadata': [[{'@context': 'http://schema.org', '@type': 'Dataset', 'name': 'COSMIC', 'description': \"COSMIC, the Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensive resource for exploring the impact of somatic mutations in human cancer\", 'version': '90', 'identifier': '10.1093/nar/gkw1121', 'distribution': {'@type': 'DataDownload', 'contentUrl': 'https://cancer.sanger.ac.uk/cosmic/download'}, 'sourceOrganization': 'Wellcome Sanger Institute', 'datePublished': '2019-09-05'}, {'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'https://cancer.sanger.ac.uk/cosmic', 'name': 'COSMIC: Catalogue of Somatic Mutations in Cancer', 'description': \"COSMIC, the Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensive resource for exploring the impact of somatic mutations in human cancer\", 'potentialAction': {'@type': 'SearchAction', 'target': 'https://cancer.sanger.ac.uk/cosmic/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}, {'@context': 'http://schema.org', '@type': 'Organization', 'name': 'COSMIC: Catalogue of Somatic Mutations in Cancer', 'description': \"COSMIC, the Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensive resource for exploring the impact of somatic mutations in human cancer\", 'url': 'https://cancer.sanger.ac.uk/cosmic', 'logo': 'https://cancer.sanger.ac.uk/images/banners/cosmic_banner_328x68.png', 'sameAs': ['https://www.facebook.com/cosmic.sanger', 'https://twitter.com/cosmic_sanger', 'https://uk.linkedin.com/company/cosmic-sanger']}]]}, {'prefix': 'dbgap', 'mirId': 'MIR:00000602', 'name': 'dbGaP', 'pattern': '^phs[0-9]{6}(.v\\\\d+.p\\\\d+)?$', 'description': 'The database of Genotypes and Phenotypes (dbGaP) archives and distributes the results of studies that have investigated the interaction of genotype and phenotype.', 'created': '2019-06-11T14:17:59.890+0000', 'modified': '2019-06-11T14:17:59.890+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'phs000768.v2.p1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1671'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1671'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1671/contactPerson'}}, 'metadata': []}, {'prefix': 'iao', 'mirId': 'MIR:00000603', 'name': 'Information Artifact Ontology', 'pattern': '^\\\\d{7}$', 'description': 'An ontology of information entities, originally driven by work by the Ontology of Biomedical Investigation (OBI) digital entity and realizable information entity branch.', 'created': '2019-06-11T14:18:00.081+0000', 'modified': '2019-06-11T14:18:00.081+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000030', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1673'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1673'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1673/contactPerson'}}, 'metadata': []}, {'prefix': 'gdc', 'mirId': 'MIR:00000604', 'name': 'Genomic Data Commons Data Portal', 'pattern': '^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$', 'description': 'The GDC Data Portal is a robust data-driven platform that allows cancer researchers and bioinformaticians to search and download cancer data for analysis.', 'created': '2019-06-11T14:18:00.311+0000', 'modified': '2019-06-11T14:18:00.311+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ae8c77fe-e6c8-44d5-8265-4a38c637bbef', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1676'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1676'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1676/contactPerson'}}, 'metadata': []}, {'prefix': 'omit', 'mirId': 'MIR:00000605', 'name': 'OMIT', 'pattern': '^\\\\d{7}$', 'description': \"The purpose of the OMIT ontology is to establish data exchange standards and common data elements in the microRNA (miR) domain. Biologists (cell biologists in particular) and bioinformaticians can make use of OMIT to leverage emerging semantic technologies in knowledge acquisition and discovery for more effective identification of important roles performed by miRs in humans' various diseases and biological processes (usually through miRs' respective target genes).\", 'created': '2019-06-11T14:18:00.566+0000', 'modified': '2019-06-11T14:18:00.566+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0005506', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1679'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1679'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1679/contactPerson'}}, 'metadata': []}, {'prefix': 'insdc.cds', 'mirId': 'MIR:00000606', 'name': 'INSDC CDS', 'pattern': '^([A-Z]\\\\d{5}|[A-Z]{2}\\\\d{6}|[A-Z]{4}\\\\d{8}|[A-J][A-Z]{2}\\\\d{5})(\\\\.\\\\d+)?$', 'description': 'The coding sequence or protein identifiers as maintained in INSDC.', 'created': '2019-06-11T14:18:00.782+0000', 'modified': '2019-06-11T14:18:00.782+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AAA35559', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1681'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1681'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1681/contactPerson'}}, 'metadata': []}, {'prefix': 'glytoucan', 'mirId': 'MIR:00000608', 'name': 'GlyTouCan', 'pattern': '^G[0-9]{5}[A-Z]{2}$', 'description': 'GlyTouCan is the single worldwide registry of glycan (carbohydrate sugar chain) data.', 'created': '2019-06-11T14:18:01.543+0000', 'modified': '2019-06-11T14:18:01.543+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'G00054MO', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1688'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1688'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1688/contactPerson'}}, 'metadata': []}, {'prefix': 'metacyc.compound', 'mirId': 'MIR:00000609', 'name': 'MetaCyc Compound', 'pattern': '^CPD-\\\\d{5}$', 'description': 'MetaCyc is a curated database of experimentally elucidated metabolic pathways from all domains of life. MetaCyc contains 2526 pathways from 2844 different organisms. MetaCyc contains pathways involved in both primary and secondary metabolism, as well as associated metabolites, reactions, enzymes, and genes. The goal of MetaCyc is to catalog the universe of metabolism by storing a representative sample of each experimentally elucidated pathway.', 'created': '2019-06-11T14:18:01.783+0000', 'modified': '2019-06-11T14:18:01.783+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CPD-10330', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1691'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1691'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1691/contactPerson'}}, 'metadata': []}, {'prefix': 'metacyc.reaction', 'mirId': 'MIR:00000610', 'name': 'MetaCyc Reaction', 'pattern': '^RXN-\\\\d{5}$', 'description': 'MetaCyc is a curated database of experimentally elucidated metabolic pathways from all domains of life. MetaCyc contains 2526 pathways from 2844 different organisms. MetaCyc contains pathways involved in both primary and secondary metabolism, as well as associated metabolites, reactions, enzymes, and genes. The goal of MetaCyc is to catalog the universe of metabolism by storing a representative sample of each experimentally elucidated pathway.', 'created': '2019-06-11T14:18:02.043+0000', 'modified': '2019-06-11T14:18:02.043+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RXN-14904', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1694'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1694'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1694/contactPerson'}}, 'metadata': []}, {'prefix': 'd1id', 'mirId': 'MIR:00000611', 'name': 'DataONE', 'pattern': '\\\\S+', 'description': 'DataONE provides infrastructure facilitating long-term access to scientific research data of relevance to the earth sciences.', 'created': '2019-06-11T14:18:02.239+0000', 'modified': '2019-06-11T14:18:02.239+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '00030692-0FE1-4A1B-955E-A2E55D659267', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1696'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1696'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1696/contactPerson'}}, 'metadata': []}, {'prefix': 'cath', 'mirId': 'MIR:00000612', 'name': 'CATH Protein Structural Domain Superfamily', 'pattern': '^[1-4]\\\\.[0-9]+\\\\.[0-9]+\\\\.[0-9]+$', 'description': 'CATH is a classification of protein structural domains. We group protein domains into superfamilies when there is sufficient evidence they have diverged from a common ancestor. CATH can be used to predict structural and functional information directly from protein sequence.', 'created': '2019-06-11T14:18:02.470+0000', 'modified': '2019-06-11T14:18:02.470+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1.10.8.10', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1699'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1699'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1699/contactPerson'}}, 'metadata': []}, {'prefix': 'mmp.ref', 'mirId': 'MIR:00000613', 'name': 'MarRef', 'pattern': '^MMP\\\\d+.\\\\d+$', 'description': 'MarRef is a manually curated marine microbial reference genome database that contains completely sequenced genomes.', 'created': '2019-06-11T14:18:02.696+0000', 'modified': '2019-06-11T14:18:02.696+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MMP3312982.2', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1702'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1702'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1702/contactPerson'}}, 'metadata': []}, {'prefix': 'mmp.db', 'mirId': 'MIR:00000614', 'name': 'MarDB', 'pattern': '^MMP\\\\d+.\\\\d+$', 'description': 'MarDB includes all sequenced marine microbial genomes regardless of level of completeness.', 'created': '2019-06-11T14:18:02.934+0000', 'modified': '2019-06-11T14:18:02.934+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MMP02954345.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1705'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1705'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1705/contactPerson'}}, 'metadata': []}, {'prefix': 'mmp.cat', 'mirId': 'MIR:00000615', 'name': 'MarCat', 'pattern': '^MMP\\\\d+.\\\\d+$', 'description': 'MarCat is a gene (protein) catalogue of uncultivable and cultivable marine genes and proteins derived from metagenomics samples.', 'created': '2019-06-11T14:18:03.141+0000', 'modified': '2019-06-11T14:18:03.141+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MMP743597.11', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1707'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1707'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1707/contactPerson'}}, 'metadata': []}, {'prefix': 'biotools', 'mirId': 'MIR:00000616', 'name': 'BioTools', 'pattern': '^[A-Za-z0-9\\\\_]*$', 'description': 'Tool and data services registry.', 'created': '2019-06-11T14:18:03.353+0000', 'modified': '2019-06-11T14:18:03.353+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'uniprotkb', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1709'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1709'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1709/contactPerson'}}, 'metadata': []}, {'prefix': 'aop', 'mirId': 'MIR:00000617', 'name': 'AOPWiki', 'pattern': '^\\\\d+$', 'description': 'International repository of Adverse Outcome Pathways.', 'created': '2019-06-11T14:18:03.584+0000', 'modified': '2019-06-11T14:18:03.584+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '98', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1712'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1712'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1712/contactPerson'}}, 'metadata': []}, {'prefix': 'ngl', 'mirId': 'MIR:00000618', 'name': 'NASA GeneLab', 'pattern': '^GLDS-\\\\d+$', 'description': \"NASA's GeneLab gathers spaceflight genomic data, RNA and protein expression, and metabolic profiles, interfaces with existing databases for expanded research, will offer tools to conduct data analysis, and is in the process of creating a place online where scientists, researchers, teachers and students can connect with their peers, share their results, and communicate with NASA.\", 'created': '2019-06-11T14:18:03.826+0000', 'modified': '2019-06-11T14:18:03.826+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GLDS-141', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1715'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1715'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1715/contactPerson'}}, 'metadata': []}, {'prefix': 'erv', 'mirId': 'MIR:00000619', 'name': 'Human Endogenous Retrovirus Database', 'pattern': '^[A-Za-z0-9\\\\-\\\\_]+$', 'description': 'Endogenous retroviruses (ERVs) are common in vertebrate genomes; a typical mammalian genome contains tens to hundreds of thousands of ERV elements. Most ERVs are evolutionarily old and have accumulated multiple mutations, playing important roles in physiology and disease processes. The Human Endogenous Retrovirus Database (hERV) is compiled from the human genome nucleotide sequences obtained from Human Genome Projects, and screens those sequences for hERVs, whilst continuously improving classification and characterization of retroviral families. It provides access to individual reconstructed HERV elements, their sequence, structure and features.', 'created': '2019-06-11T14:18:04.072+0000', 'modified': '2019-06-11T14:18:04.072+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'THE1B', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1718'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1718'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1718/contactPerson'}}, 'metadata': []}, {'prefix': 'napdi', 'mirId': 'MIR:00000620', 'name': 'Natural Product-Drug Interaction Research Data Repository', 'pattern': '^[0-9]+$', 'description': 'The Natural Product-Drug Interaction Research Data Repository, a publicly accessible database where researchers can access scientific results, raw data, and recommended approaches to optimally assess the clinical significance of pharmacokinetic natural product-drug interactions (PK-NPDIs).', 'created': '2019-06-11T14:18:04.316+0000', 'modified': '2019-06-11T14:18:04.316+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '28', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1721'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1721'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1721/contactPerson'}}, 'metadata': []}, {'prefix': 'arraymap', 'mirId': 'MIR:00000621', 'name': 'ArrayMap', 'pattern': '^[\\\\w\\\\-:,]{3,64}$', 'description': 'arrayMap is a collection of pre-processed oncogenomic array data sets and CNA (somatic copy number aberrations) profiles. CNA are a type of mutation commonly found in cancer genomes. arrayMap data is assembled from public repositories and supplemented with additional sources, using custom curation pipelines. This information has been mapped to multiple editions of the reference human genome.', 'created': '2019-06-11T14:18:04.557+0000', 'modified': '2019-06-11T14:18:04.557+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'icdom:8500_3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1724'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1724'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1724/contactPerson'}}, 'metadata': []}, {'prefix': 'begdb', 'mirId': 'MIR:00000622', 'name': 'Benchmark Energy & Geometry Database', 'pattern': '^[0-9]+$', 'description': 'The Benchmark Energy & Geometry Database (BEGDB) collects results of highly accurate quantum mechanics (QM) calculations of molecular structures, energies and properties. These data can serve as benchmarks for testing and parameterization of other computational methods.', 'created': '2019-06-11T14:18:04.748+0000', 'modified': '2019-06-11T14:18:04.748+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '4214', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1726'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1726'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1726/contactPerson'}}, 'metadata': []}, {'prefix': 'sisu', 'mirId': 'MIR:00000624', 'name': 'SISu', 'pattern': '^[0-9]+:[0-9]+$', 'description': 'The Sequencing Initiative Suomi (SISu) project is an international collaboration to harmonize and aggregate whole genome and exome sequence data from Finnish samples, providing data for researchers and clinicians. The SISu project allows for the search of variants to determine their attributes and occurrence in Finnish cohorts, and provides summary data on single nucleotide variants and indels from exomes, sequenced in disease-specific and population genetic studies.', 'created': '2019-06-11T14:18:05.215+0000', 'modified': '2019-06-11T14:18:05.215+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '17:41223048', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1732'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1732'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1732/contactPerson'}}, 'metadata': []}, {'prefix': 'mzspec', 'mirId': 'MIR:00000625', 'name': 'Universal Spectrum Identifier', 'pattern': '^mzspec:.+$', 'description': 'The Universal Spectrum Identifier (USI) is a compound identifier that provides an abstract path to refer to a single spectrum generated by a mass spectrometer, and potentially the ion that is thought to have produced it.', 'created': '2019-06-11T14:18:05.468+0000', 'modified': '2019-06-11T14:18:05.468+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PXD002255::ES_XP_Ubi_97H_HCD_349:scan:9617:LAEIYVNSSFYK/2', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1735'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1735'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1735/contactPerson'}}, 'metadata': []}, {'prefix': 'crisprdb', 'mirId': 'MIR:00000626', 'name': 'CRISPRdb', 'pattern': '^[0-9]+$', 'description': 'Repeated CRISPR (\"clustered regularly interspaced short palindromic repeats\") elements found in archaebacteria and eubacteria are believed to defend against viral infection, potentially targeting invading DNA for degradation. CRISPRdb is a database that stores information on CRISPRs that are automatically extracted from newly released genome sequence data.', 'created': '2019-06-11T14:18:05.694+0000', 'modified': '2019-06-11T14:18:05.694+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '551115', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1738'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1738'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1738/contactPerson'}}, 'metadata': []}, {'prefix': 'gnpis', 'mirId': 'MIR:00000627', 'name': 'GnpIS', 'pattern': '^[A-Za-z0-9]+$', 'description': 'GnpIS is an integrative information system focused on plants and fungal pests. It provides both genetic (e.g. genetic maps, quantitative trait loci, markers, single nucleotide polymorphisms, germplasms and genotypes) and genomic data (e.g. genomic sequences, physical maps, genome annotation and expression data) for species of agronomical interest.', 'created': '2019-06-11T14:18:05.948+0000', 'modified': '2019-06-11T14:18:05.948+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AY109603', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1741'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1741'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1741/contactPerson'}}, 'metadata': []}, {'prefix': 'licebase', 'mirId': 'MIR:00000628', 'name': 'LiceBase', 'pattern': '^[A-Za-z0-9\\\\-\\\\/]+$', 'description': 'Sea lice (Lepeophtheirus salmonis and Caligus species) are the major pathogens of salmon,  significantly impacting upon the global salmon farming industry. Lice control is primarily accomplished through chemotherapeutants, though emerging resistance necessitates the development of new treatment methods (biological agents, prophylactics and new drugs). LiceBase is a database for sea lice genomics, providing genome annotation of the Atlantic salmon louse Lepeophtheirus salmonis, a genome browser, and access to related high-thoughput genomics data. LiceBase also mines and stores data from related genome sequencing and functional genomics projects.', 'created': '2019-06-11T14:18:06.185+0000', 'modified': '2019-06-11T14:18:06.185+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'EMLSAT00000003403', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1744'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1744'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1744/contactPerson'}}, 'metadata': []}, {'prefix': 'sugarbind', 'mirId': 'MIR:00000629', 'name': 'SugarBind', 'pattern': '^[A-Za-z]+\\\\/[0-9]+$', 'description': 'The SugarBind Database captures knowledge of glycan binding of human pathogen lectins and adhesins, where each glycan-protein binding pair is associated with at least one published reference. It provides information on the pathogen agent, the lectin/adhesin involved, and the human glycan ligand. This collection provides information on ligands.', 'created': '2019-06-11T14:18:06.412+0000', 'modified': '2019-06-11T14:18:06.412+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'lectins/172', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1747'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1747'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1747/contactPerson'}}, 'metadata': []}, {'prefix': 'rnacentral', 'mirId': 'MIR:00000630', 'name': 'RNAcentral', 'pattern': '^URS[0-9A-F]{10}(\\\\_\\\\d+)?$', 'description': 'RNAcentral is a public resource that offers integrated access to a comprehensive and up-to-date set of non-coding RNA sequences provided by a collaborating group of Expert Databases.', 'created': '2019-06-11T14:18:06.653+0000', 'modified': '2019-06-11T14:18:06.653+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'URS0000759CF4', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1750'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1750'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1750/contactPerson'}}, 'metadata': []}, {'prefix': 'microscope', 'mirId': 'MIR:00000631', 'name': 'MicroScope', 'pattern': '^\\\\d+$', 'description': 'MicroScope is an integrative resource that supports systematic and efficient revision of microbial genome annotation, data management and comparative analysis.', 'created': '2019-06-11T14:18:06.853+0000', 'modified': '2019-06-11T14:18:06.853+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '5601141', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1752'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1752'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1752/contactPerson'}}, 'metadata': []}, {'prefix': 'swissregulon', 'mirId': 'MIR:00000632', 'name': 'SwissRegulon', 'pattern': '^[A-Za-z0-9]+$', 'description': 'A database of genome-wide annotations of regulatory sites. It contains annotations for 17 prokaryotes and 3 eukaryotes. The database frontend offers an intuitive interface showing genomic information in a graphical form.', 'created': '2019-06-11T14:18:07.113+0000', 'modified': '2019-06-11T14:18:07.113+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AHR', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1755'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1755'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1755/contactPerson'}}, 'metadata': []}, {'prefix': 'validatordb', 'mirId': 'MIR:00000633', 'name': 'ValidatorDB', 'pattern': '^[A-Za-z0-9\\\\/]+$', 'description': 'Database of validation results for ligands and non-standard residues in the Protein Data Bank.', 'created': '2019-06-11T14:18:07.321+0000', 'modified': '2019-06-11T14:18:07.321+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ByStructure/2h6o', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1757'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1757'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1757/contactPerson'}}, 'metadata': []}, {'prefix': 'biominder', 'mirId': 'MIR:00000634', 'name': 'Bio-MINDER Tissue Database', 'pattern': '^[a-z0-9\\\\-]+$', 'description': 'Database of the dielectric properties of biological tissues.', 'created': '2019-06-11T14:18:07.548+0000', 'modified': '2019-06-11T14:18:07.548+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'aef4c195-9cf9-46db-a12a-7cfd1ff3eec3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1760'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1760'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1760/contactPerson'}}, 'metadata': []}, {'prefix': 'neurovault.collection', 'mirId': 'MIR:00000635', 'name': 'NeuroVault Collection', 'pattern': '^[1-9][0-9]*$', 'description': 'Neurovault is an online repository for statistical maps, parcellations and atlases of the brain. This collection references sets (collections) of images.', 'created': '2019-06-11T14:18:07.777+0000', 'modified': '2019-06-11T14:18:07.777+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3304', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1763'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1763'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1763/contactPerson'}}, 'metadata': [{'@context': 'https://schema.org', '@type': 'Organization', 'name': 'NeuroVault', 'url': 'https://neurovault.org', 'logo': 'https://neurovault.org/static/images/neurovault-logo.png', 'sameAs': ['https://en.wikipedia.org/wiki/NeuroVault', 'https://twitter.com/vaultneuro']}, {'@context': 'https://schema.org', '@type': 'WebSite', 'url': 'https://neurovault.org/', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://neurovault.org/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}]}, {'prefix': 'vmhmetabolite', 'mirId': 'MIR:00000636', 'name': 'VMH metabolite', 'pattern': '[a-zA-Z0-9_\\\\(\\\\_\\\\)\\\\[\\\\]]+', 'description': 'The Virtual Metabolic Human (VMH) is a resource that combines human and gut microbiota metabolism with nutrition and disease.', 'created': '2019-06-11T14:18:07.981+0000', 'modified': '2019-06-11T14:18:07.981+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'h2o', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1765'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1765'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1765/contactPerson'}}, 'metadata': []}, {'prefix': 'nmr', 'mirId': 'MIR:00000639', 'name': 'Nuclear Magnetic Resonance Controlled Vocabulary', 'pattern': '^NMR:\\\\d+$', 'description': 'nmrCV is a controlled vocabulary to deliver standardized descriptors for the open mark-up language for NMR raw and spectrum data, sanctioned by the metabolomics standards initiative msi.', 'created': '2019-06-11T14:18:08.710+0000', 'modified': '2019-06-11T14:18:08.710+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1000003', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1773'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1773'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1773/contactPerson'}}, 'metadata': []}, {'prefix': 'vmhreaction', 'mirId': 'MIR:00000640', 'name': 'VMH reaction', 'pattern': '[a-zA-Z0-9_\\\\(\\\\_\\\\)\\\\[\\\\]]+', 'description': 'The Virtual Metabolic Human (VMH) is a resource that combines human and gut microbiota metabolism with nutrition and disease.', 'created': '2019-06-11T14:18:08.921+0000', 'modified': '2019-06-11T14:18:08.921+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HEX1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1775'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1775'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1775/contactPerson'}}, 'metadata': []}, {'prefix': 'ascl', 'mirId': 'MIR:00000641', 'name': 'Astrophysics Source Code Library', 'pattern': '^[0-9\\\\.]+$', 'description': \"The Astrophysics Source Code Library (ASCL) is a free online registry for software that have been used in research that has appeared in, or been submitted to, peer-reviewed publications. The ASCL is indexed by the SAO/NASA Astrophysics Data System (ADS) and Web of Science's Data Citation Index (WoS DCI), and is citable by using the unique ascl ID assigned to each code. The ascl ID can be used to link to the code entry by prefacing the number with ascl.net (i.e., ascl.net/1201.001).\", 'created': '2019-06-11T14:18:09.136+0000', 'modified': '2019-06-11T14:18:09.136+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1801.012', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1777'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1777'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1777/contactPerson'}}, 'metadata': []}, {'prefix': 'facebase', 'mirId': 'MIR:00000642', 'name': 'FaceBase Data Repository', 'pattern': '^FB\\\\d{8}$', 'description': 'FaceBase is a collaborative NIDCR-funded consortium to generate data in support of advancing research into craniofacial development and malformation. It serves as a community resource by generating large datasets of a variety of types and making them available to the wider research community via this website. Practices emphasize a comprehensive and multidisciplinary approach to understanding the developmental processes that create the face. The data offered spotlights high-throughput genetic, molecular, biological, imaging and computational techniques. One of the missions of this consortium is to facilitate cooperation and collaboration between projects.', 'created': '2019-06-11T14:18:09.365+0000', 'modified': '2019-06-11T14:18:09.365+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'FB00000917', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1780'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1780'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1780/contactPerson'}}, 'metadata': []}, {'prefix': 'gudmap', 'mirId': 'MIR:00000643', 'name': 'GUDMAP', 'pattern': '^[-0-9a-zA-Z]+(@[-0-9a-zA-Z]+)?$', 'description': 'The GenitoUrinary Development Molecular Anatomy Project (GUDMAP) is a consortium of laboratories working to provide the scientific and medical community with tools to facilitate research on the GenitoUrinary (GU) tract.', 'created': '2019-06-11T14:18:09.598+0000', 'modified': '2019-06-11T14:18:09.598+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q-2958', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1783'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1783'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1783/contactPerson'}}, 'metadata': []}, {'prefix': 'eu89h', 'mirId': 'MIR:00000644', 'name': 'JRC Data Catalogue', 'pattern': '^[a-z0-9\\\\-_]+$', 'description': \"The JRC Data Catalogue gives access to the multidisciplinary data produced and maintained by the Joint Research Centre, the European Commission's in-house science service providing independent scientific advice and support to policies of the European Union.\", 'created': '2019-06-11T14:18:09.859+0000', 'modified': '2019-06-11T14:18:09.859+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'jrc-eurl-ecvam-chemagora', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1786'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1786'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1786/contactPerson'}}, 'metadata': []}, {'prefix': 'minid', 'mirId': 'MIR:00000645', 'name': 'Minimal Identifiers for Transient Data', 'pattern': '^[A-Za-z0-9]+$', 'description': 'Minid are identifiers used to provide robust reference to intermediate data generated during the course of a research investigation.', 'created': '2019-06-11T14:18:10.091+0000', 'modified': '2019-06-11T14:18:10.091+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'b97957', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1789'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1789'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1789/contactPerson'}}, 'metadata': []}, {'prefix': 'mw.study', 'mirId': 'MIR:00000646', 'name': 'Metabolomics Workbench Study', 'pattern': '^ST[0-9]{6}$', 'description': 'Metabolomics Workbench stores metabolomics data for small and large studies on cells, tissues and organisms for the Metabolomics Consortium Data Repository and Coordinating Center (DRCC).', 'created': '2019-06-11T14:18:10.316+0000', 'modified': '2019-06-11T14:18:10.316+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ST000900', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1792'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1792'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1792/contactPerson'}}, 'metadata': []}, {'prefix': 'mw.project', 'mirId': 'MIR:00000647', 'name': 'Metabolomics Workbench Project', 'pattern': '^PR[0-9]{6}$', 'description': 'Metabolomics Workbench stores metabolomics data for small and large studies on cells, tissues and organisms for the Metabolomics Consortium Data Repository and Coordinating Center (DRCC).', 'created': '2019-06-11T14:18:10.677+0000', 'modified': '2019-06-11T14:18:10.677+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PR000001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1796'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1796'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1796/contactPerson'}}, 'metadata': []}, {'prefix': 'mex', 'mirId': 'MIR:00000648', 'name': 'Metabolome Express', 'pattern': '^\\\\d+$', 'description': 'A public place to process, interpret and share GC/MS metabolomics datasets.', 'created': '2019-06-11T14:18:10.861+0000', 'modified': '2019-06-11T14:18:10.861+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '36', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1798'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1798'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1798/contactPerson'}}, 'metadata': []}, {'prefix': 'gpmdb', 'mirId': 'MIR:00000649', 'name': 'GPMDB', 'pattern': '^GPM\\\\d+$', 'description': 'The Global Proteome Machine Database was constructed to utilize the information obtained by GPM servers to aid in the difficult process of validating peptide MS/MS spectra as well as protein coverage patterns.', 'created': '2019-06-11T14:18:11.249+0000', 'modified': '2019-06-11T14:18:11.249+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GPM32310002988', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1802'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1802'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1802/contactPerson'}}, 'metadata': []}, {'prefix': 'massive', 'mirId': 'MIR:00000650', 'name': 'MassIVE', 'pattern': '^MSV\\\\d+$', 'description': 'MassIVE is a community resource developed by the NIH-funded Center for Computational Mass Spectrometry to promote the global, free exchange of mass spectrometry data.', 'created': '2019-06-11T14:18:11.594+0000', 'modified': '2019-06-11T14:18:11.594+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MSV000082131', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1806'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1806'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1806/contactPerson'}}, 'metadata': []}, {'prefix': 'fplx', 'mirId': 'MIR:00000651', 'name': 'FamPlex', 'pattern': '^[a-zA-Z0-9][A-Za-z0-9_]+$', 'description': 'FamPlex is a collection of resources for grounding biological entities from text and describing their hierarchical relationships.', 'created': '2019-06-11T14:18:11.924+0000', 'modified': '2019-06-11T14:18:11.924+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'RAS', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1809'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1809'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1809/contactPerson'}}, 'metadata': []}, {'prefix': 'aop.events', 'mirId': 'MIR:00000652', 'name': 'AOPWiki (Key Event)', 'pattern': '^\\\\d+$', 'description': 'International repository of Adverse Outcome Pathways.', 'created': '2019-06-11T14:18:12.154+0000', 'modified': '2019-06-11T14:18:12.154+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1811'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1811'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1811/contactPerson'}}, 'metadata': []}, {'prefix': 'aop.relationships', 'mirId': 'MIR:00000653', 'name': 'AOPWiki (Key Event Relationship)', 'pattern': '^\\\\d+$', 'description': 'International repository of Adverse Outcome Pathways.', 'created': '2019-06-11T14:18:12.357+0000', 'modified': '2019-06-11T14:18:12.357+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '5', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1813'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1813'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1813/contactPerson'}}, 'metadata': []}, {'prefix': 'aop.stressor', 'mirId': 'MIR:00000654', 'name': 'AOPWiki (Stressor)', 'pattern': '^\\\\d+$', 'description': 'International repository of Adverse Outcome Pathways.', 'created': '2019-06-11T14:18:12.557+0000', 'modified': '2019-06-11T14:18:12.557+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '9', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1815'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1815'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1815/contactPerson'}}, 'metadata': []}, {'prefix': 'swh', 'mirId': 'MIR:00000655', 'name': 'Software Heritage', 'pattern': '^swh:[1-9]:[a-z][a-z][a-z]:[a-f0-9]+$', 'description': 'Software Heritage is the universal archive of software source code.', 'created': '2019-06-11T14:18:12.760+0000', 'modified': '2019-06-11T14:18:12.760+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1:rev:309cf2674ee7a0749978cf8265ab91a60aea0f7d', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1817'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1817'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1817/contactPerson'}}, 'metadata': []}, {'prefix': 'dev.ga4ghdos', 'mirId': 'MIR:00000656', 'name': 'Development Data Object Service', 'pattern': '^[a-zA-Z0-9\\\\-:#\\\\.]+$', 'description': 'Assists in resolving data across cloud resources.', 'created': '2019-06-11T14:18:12.998+0000', 'modified': '2019-06-11T14:18:12.998+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '23fa7b4b-9d68-429b-aece-658b11124bb3#2017-03-24T18:43:16.886826-05:00', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1820'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1820'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1820/contactPerson'}}, 'metadata': []}, {'prefix': 'complexportal', 'mirId': 'MIR:00000657', 'name': 'Complex Portal', 'pattern': '^CPX-[0-9]+$', 'description': 'A database that describes manually curated macromolecular complexes and provides links to details about these complexes in other databases.', 'created': '2019-06-11T14:18:13.254+0000', 'modified': '2019-06-11T14:18:13.254+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CPX-2158', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1823'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1823'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1823/contactPerson'}}, 'metadata': []}, {'prefix': 'comptox', 'mirId': 'MIR:00000658', 'name': 'CompTox Chemistry Dashboard', 'pattern': '^DTXSID\\\\d+$', 'description': \"The Chemistry Dashboard is a part of a suite of databases and web applications developed by the US Environmental Protection Agency's Chemical Safety for Sustainability Research Program. These databases and apps support EPA's computational toxicology research efforts to develop innovative methods to change how chemicals are currently evaluated for potential health risks.\", 'created': '2019-06-11T14:18:13.452+0000', 'modified': '2019-06-11T14:18:13.452+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DTXSID2021028', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1825'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1825'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1825/contactPerson'}}, 'metadata': []}, {'prefix': 'rbk', 'mirId': 'MIR:00000659', 'name': 'Rebuilding a Kidney', 'pattern': '^[-0-9a-zA-Z]+(@[-0-9a-zA-Z]+)?$', 'description': '(Re)Building a Kidney is an NIDDK-funded consortium of research projects working to optimize approaches for the isolation, expansion, and differentiation of appropriate kidney cell types and their integration into complex structures that replicate human kidney function.', 'created': '2019-06-11T14:18:13.721+0000', 'modified': '2019-06-11T14:18:13.721+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'Q-2958', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1828'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1828'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1828/contactPerson'}}, 'metadata': []}, {'prefix': 'ligandbook', 'mirId': 'MIR:00000681', 'name': 'LigandBook', 'pattern': '^[0-9]+$', 'description': 'Ligandbook is a public repository for force field parameters with a special emphasis on small molecules and known ligands of proteins. It acts as a warehouse for parameter files that are supplied by the community.', 'created': '2019-06-11T14:18:19.416+0000', 'modified': '2019-06-11T14:18:19.416+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '785', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1887'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1887'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1887/contactPerson'}}, 'metadata': []}, {'prefix': 'kaggle', 'mirId': 'MIR:00000682', 'name': 'Kaggle', 'pattern': '^[0-9a-zA-Z\\\\-]+\\\\/[0-9a-zA-Z\\\\-]+$', 'description': 'Kaggle is a platform for sharing data, performing reproducible analyses, interactive data analysis tutorials, and machine learning competitions.', 'created': '2019-06-11T14:18:19.662+0000', 'modified': '2019-06-11T14:18:19.662+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'nasa/kepler-exoplanet-search-results', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1890'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1890'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1890/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org/', '@type': 'Dataset', 'name': 'Kepler Exoplanet Search Results', 'description': '### Context\\n\\nThe Kepler Space Observatory is a NASA-build satellite that was launched in 2009. The telescope is dedicated to searching for exoplanets in star systems besides our own, with the ultimate goal of possibly finding other habitable planets besides our own. The original mission ended in 2013 due to mechanical failures, but the telescope has nevertheless been functional since 2014 on a \"K2\" extended mission.\\n\\nKepler had verified 1284 new exoplanets as of May 2016. As of October 2017 there are over 3000 confirmed exoplanets total (using all detection methods, including ground-based ones). The telescope is still active and continues to collect new data on its extended mission.\\n\\n### Content\\n\\nThis dataset is a cumulative record of all observed Kepler \"objects of interest\" ‚Äî basically, all of the approximately 10,000 exoplanet candidates Kepler has taken observations on.\\n\\nThis dataset has an extensive data dictionary, which can be accessed [here](https://exoplanetarchive.ipac.caltech.edu/docs/API_kepcandidate_columns.html). Highlightable columns of note are:\\n\\n* `kepoi_name`: A KOI is a target identified by the Kepler Project that displays at least one transit-like sequence within Kepler time-series photometry that appears to be of astrophysical origin and initially consistent with a planetary transit hypothesis\\n* `kepler_name`: [These names] are intended to clearly indicate a class of objects that have been confirmed or validated as planets‚Äîa step up from the planet candidate designation.\\n* `koi_disposition`: The disposition in the literature towards this exoplanet candidate. One of CANDIDATE, FALSE POSITIVE, NOT DISPOSITIONED or CONFIRMED.\\n* `koi_pdisposition`: The disposition Kepler data analysis has towards this exoplanet candidate. One of FALSE POSITIVE, NOT DISPOSITIONED, and CANDIDATE.\\n* `koi_score `: A value between 0 and 1 that indicates the confidence in the KOI disposition. For CANDIDATEs, a higher value indicates more confidence in its disposition, while for FALSE POSITIVEs, a higher value indicates less confidence in that disposition.\\n\\n### Acknowledgements\\n\\nThis dataset was published as-is by NASA. You can access the original table [here](https://exoplanetarchive.ipac.caltech.edu/cgi-bin/TblView/nph-tblView?app=ExoTbls&amp;config=koi). More data from the Kepler mission is available from the same source [here](https://exoplanetarchive.ipac.caltech.edu/docs/data.html).\\n\\n### Inspiration\\n\\n* How often are exoplanets confirmed in the existing literature disconfirmed by measurements from Kepler? How about the other way round?\\n* What general characteristics about exoplanets (that we can find) can you derive from this dataset?\\n* What exoplanets get assigned names in the literature? What is the distribution of confidence scores?\\n\\nSee also: the [Kepler Labeled Time Series](https://www.kaggle.com/keplersmachines/kepler-labelled-time-series-data) and [Open Exoplanets Catalogue](https://www.kaggle.com/mrisdal/open-exoplanet-catalogue) datasets.', 'url': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results', 'sameAs': '', 'version': 2, 'keywords': ['society and social sciences &gt; society &gt; business', 'natural and physical sciences', 'technology and applied sciences', 'natural and physical sciences &gt; physical sciences &gt; astronomy', 'natural and physical sciences &gt; physical sciences &gt; space'], 'license': {'@type': 'CreativeWork', 'name': 'CC0: Public Domain', 'url': 'https://creativecommons.org/publicdomain/zero/1.0/'}, 'identifier': '2894', 'includedInDataCatalog': {'@type': 'DataCatalog', 'name': 'Kaggle', 'url': 'https://www.kaggle.com'}, 'creator': {'@type': 'Organization', 'name': 'NASA', 'url': 'https://www.kaggle.com/nasa', 'image': 'https://storage.googleapis.com/kaggle-organizations/147/thumbnail.jpg%3Fr=540'}, 'distribution': [{'@type': 'DataDownload', 'requiresSubscription': True, 'encodingFormat': 'zip', 'fileFormat': 'zip', 'contentUrl': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results/download', 'contentSize': '3695322 bytes'}, {'@type': 'DataDownload', 'requiresSubscription': True, 'encodingFormat': 'csv', 'fileFormat': 'csv', 'contentUrl': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results/download/wYdIhbkWbDC9F8oUvzJ4%2Fversions%2Fde2ssCyY0qPAdJtd3agp%2Ffiles%2Fcumulative.csv?datasetVersionNumber=1', 'contentSize': '3695322 bytes'}], 'commentCount': 0, 'dateModified': '2017-10-10T18:26:59.497', 'discussionUrl': 'https://www.kaggle.com/nasa/kepler-exoplanet-search-results/discussion', 'alternateName': '10000 exoplanet candidates examined by the Kepler Space Observatory', 'isAccessibleForFree': True, 'thumbnailUrl': 'https://storage.googleapis.com/kaggle-datasets-images/2894/4876/afd561892005209a55d97b68deb977fe/dataset-card.jpg', 'interactionStatistic': [{'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/CommentAction', 'userInteractionCount': 0}, {'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/DownloadAction', 'userInteractionCount': 2533}, {'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/ViewAction', 'userInteractionCount': 22693}, {'@type': 'InteractionCounter', 'interactionType': 'http://schema.org/LikeAction', 'userInteractionCount': 119}]}]}, {'prefix': 'idoo', 'mirId': 'MIR:00000683', 'name': 'Identifiers.org Ontology', 'pattern': '^[0-9a-zA-Z]+$', 'description': 'Identifiers.org Ontology', 'created': '2019-06-11T14:18:19.905+0000', 'modified': '2019-06-11T14:18:19.905+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'DataCollection', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1893'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1893'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1893/contactPerson'}}, 'metadata': []}, {'prefix': 'oci', 'mirId': 'MIR:00000660', 'name': 'OCI', 'pattern': '^[0-9]+-[0-9]+$', 'description': 'Each OCI (Open Citation Identifier) has a simple structure: oci:number-number, where ‚Äúoci:‚Äù is the identifier prefix, and is used to identify a citation as a first-class data entitiy - see https://opencitations.wordpress.com/2018/02/19/citations-as-first-class-data-entities-introduction/ for additional information.\\r\\n\\r\\nOCIs for citations stored within the OpenCitations Corpus are constructed by combining the OpenCitations Corpus local identifiers for the citing and cited bibliographic resources, separating them with a dash. For example, oci:2544384-7295288 is a valid OCI for the citation between two papers stored within the OpenCitations Corpus.\\r\\n\\r\\nOCIs can also be created for bibliographic resources described in an external bibliographic database, if they are similarly identified there by identifiers having a unique numerical part. For example, the OCI for the citation that exists between Wikidata resources Q27931310 and Q22252312 is oci:01027931310‚Äì01022252312.\\r\\n\\r\\nOCIs can also be created for bibliographic resources described in external bibliographic database such as Crossref or DataCite where they are identified by alphanumeric Digital Object Identifiers (DOIs), rather than purely numerical strings.', 'created': '2019-06-11T14:18:13.968+0000', 'modified': '2019-06-11T14:18:13.968+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1-18', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1831'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1831'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1831/contactPerson'}}, 'metadata': []}, {'prefix': 'foodon', 'mirId': 'MIR:00000661', 'name': 'FoodOn Food Ontology', 'pattern': '^FOODON:[0-9]{8}$', 'description': 'FoodOn is a comprehensive and easily accessible global farm-to-fork ontology about food that accurately and consistently describes foods commonly known in cultures from around the world. It is a consortium-driven project built to interoperate with the The Open Biological and Biomedical Ontology Foundry library of ontologies.', 'created': '2019-06-11T14:18:14.340+0000', 'modified': '2019-06-11T14:18:14.340+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '03307879', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1834'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1834'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1834/contactPerson'}}, 'metadata': []}, {'prefix': 'encode', 'mirId': 'MIR:00000662', 'name': 'ENCODE: Encyclopedia of DNA Elements', 'pattern': '^ENC[A-Za-z]{2}[0-9]{3}[A-Za-z]{3}$', 'description': 'The ENCODE Consortium is integrating multiple technologies and approaches in a collective effort to discover and define the functional elements encoded in the human genome, including genes, transcripts, and transcriptional regulatory regions, together with their attendant chromatin states and DNA methylation patterns.', 'created': '2019-06-11T14:18:14.565+0000', 'modified': '2019-06-11T14:18:14.565+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ENCSR163RYW', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1836'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1836'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1836/contactPerson'}}, 'metadata': [{'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': {'classification': 'cell line', 'aliases': [], 'dbxrefs': ['Cellosaurus:CVCL_0045'], 'status': 'released', 'organ_slims': ['epithelium', 'kidney'], 'developmental_slims': ['mesoderm'], '@id': '/biosample-types/cell_line_EFO_0001182/', 'name': 'cell_line_EFO_0001182', 'synonyms': ['human embryonic kidney cell'], 'term_name': 'HEK293', 'references': [], 'system_slims': ['excretory system'], 'uuid': '014c9be0-20d4-4df5-b816-b0dca4cfee9d', 'cell_slims': ['epithelial cell'], '@type': ['BiosampleType', 'Item'], 'schema_version': '1', 'term_id': 'EFO:0001182'}, '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'audit': {'WARNING': [{'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF634VOX|/files/ENCFF634VOX/} was generated from a library with PBC1 value of 0.82.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF634VOX|/files/ENCFF634VOX/} was generated from a library with PBC2 value of 5.54.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF431LTO|/files/ENCFF431LTO/} was generated from a library with PBC1 value of 0.84.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF431LTO|/files/ENCFF431LTO/} was generated from a library with PBC2 value of 6.09.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF014SGL|/files/ENCFF014SGL/} was generated from a library with PBC1 value of 0.84.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF014SGL|/files/ENCFF014SGL/} was generated from a library with PBC2 value of 6.09.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC1 (PCR Bottlenecking Coefficient 1, M1/M_distinct) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where some reads map (M_distinct). A PBC1 value in the range 0 - 0.5 is severe bottlenecking, 0.5 - 0.8 is moderate bottlenecking, 0.8 - 0.9 is mild bottlenecking, and > 0.9 is no bottlenecking. PBC1 value > 0.9 is recommended, but > 0.8 is acceptable.  ENCODE processed alignments file {ENCFF332OVN|/files/ENCFF332OVN/} was generated from a library with PBC1 value of 0.82.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'mild to moderate bottlenecking', 'detail': 'PBC2 (PCR Bottlenecking Coefficient 2, M1/M2) is the ratio of the number of genomic locations where exactly one read maps uniquely (M1) to the number of genomic locations where two reads map uniquely (M2). A PBC2 value in the range 0 - 1 is severe bottlenecking, 1 - 3 is moderate bottlenecking, 3 - 10 is mild bottlenecking, > 10 is no bottlenecking. PBC2 value > 10 is recommended, but > 3 is acceptable.  ENCODE processed alignments file {ENCFF332OVN|/files/ENCFF332OVN/} was generated from a library with PBC2 value of 5.54.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'missing genetic modification characterization', 'detail': 'Genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} performed for the purpose of tagging is missing validating characterization that is required by ENCODE4 standards.', 'path': '/experiments/ENCSR163RYW/'}, {'level': 40, 'level_name': 'WARNING', 'name': 'audit_experiment', 'category': 'missing biosample characterization', 'detail': 'Biosample {ENCBS111GOI|/biosamples/ENCBS111GOI/} which has been modified by genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} is missing characterization validating the modification, Biosample {ENCBS082THU|/biosamples/ENCBS082THU/} which has been modified by genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} is missing characterization validating the modification.', 'path': '/experiments/ENCSR163RYW/'}], 'INTERNAL_ACTION': [{'level': 30, 'level_name': 'INTERNAL_ACTION', 'name': 'audit_genetic_modification_target', 'category': 'inconsistent modification target', 'detail': 'Genetic modification {ENCGM615AHN|/genetic-modifications/ENCGM615AHN/} is targeting {eGFP-ZNF189-human|/targets/eGFP-ZNF189-human/} which has already been modified. Should consider using unmodified target version for genetic modification.', 'path': '/genetic-modifications/ENCGM615AHN/'}, {'level': 30, 'level_name': 'INTERNAL_ACTION', 'name': 'audit_experiment', 'category': 'inconsistent experiment target', 'detail': 'Experiment {ENCSR163RYW|/experiments/ENCSR163RYW/} has a tagged target {eGFP-ZNF189-human|/targets/eGFP-ZNF189-human/}. Should consider using untagged target version for experiment.', 'path': '/experiments/ENCSR163RYW/'}]}, 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': [{'related_datasets': ['/experiments/ENCSR429LAP/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR688NWK/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR157LMX/'], 'alternate_accessions': [], 'accession': 'ENCSR300VIY', 'assembly': ['GRCh38', 'hg19'], 'documents': [], 'hub': '/matched-sets/ENCSR300VIY/@@hub/hub.txt', 'description': 'This set represents the constituent input control libraries sequenced on HiSeq 4000 platform from which a pooled and subsampled control is derived from for the processing of the ChIP-seq experiments done on eGFP-tagged zinc fingers in the HEK293 cell line', 'date_released': '2016-10-07', 'assay_synonyms': ['ChIP-seq assay', 'chromatin immunoprecipitation sequencing assay'], 'organism': ['/organisms/human/'], 'files': ['/files/ENCFF810LLR/', '/files/ENCFF182WHU/'], 'control_type': ['input library'], 'assay_term_id': ['OBI:0000716'], 'aliases': ['michael-snyder:HEK_ZNF_HiSeq4000_control_pool'], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'assay_term_name': ['ChIP-seq'], 'date_created': '2016-08-02T18:59:49.859024+00:00', 'revoked_datasets': [], 'month_released': 'October, 2016', 'lab': '/labs/j-michael-cherry/', '@id': '/matched-sets/ENCSR300VIY/', 'treatment_term_name': [], 'biosample_ontology': ['/biosample-types/cell_line_EFO_0001182/'], 'dbxrefs': [], 'revoked_files': [], 'references': [], 'target': [], 'uuid': '413eaaf8-c433-43dc-aba4-d2d34f148165', 'submitted_by': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'status': 'released', 'internal_tags': [], 'original_files': ['/files/ENCFF810LLR/', '/files/ENCFF182WHU/'], '@type': ['MatchedSet', 'Series', 'Dataset', 'Item'], 'contributing_files': ['/files/ENCFF145UQB/', '/files/ENCFF909YLP/', '/files/ENCFF058AVH/', '/files/ENCFF712WCF/', '/files/ENCFF945YPE/', '/files/ENCFF470DTH/', '/files/ENCFF083WCY/', '/files/ENCFF683AIZ/', '/files/ENCFF370HPX/', '/files/ENCFF876NRR/', '/files/ENCFF848QHI/', '/files/ENCFF361NSP/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR157LMX/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZEB2_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE92227'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': '92beb008-a2c0-4703-92d1-2133939b3504', 'original_files': ['/files/ENCFF557HNG/', '/files/ENCFF453HUS/', '/files/ENCFF370HPX/', '/files/ENCFF045OVA/', '/files/ENCFF087ZSB/', '/files/ENCFF083WCY/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZEB2 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR157LMX/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF557HNG/', '/files/ENCFF453HUS/', '/files/ENCFF370HPX/', '/files/ENCFF045OVA/', '/files/ENCFF087ZSB/', '/files/ENCFF083WCY/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.612597+00:00', 'accession': 'ENCSR157LMX', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/d4ff5fe7-e217-4d27-b541-c6ef5de53e3e/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR274OGE/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-CTCF_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE92228'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': '4b8770d1-5c0d-457a-b559-f928928e0c16', 'original_files': ['/files/ENCFF196VCR/', '/files/ENCFF173EDE/', '/files/ENCFF348IEF/', '/files/ENCFF058AVH/', '/files/ENCFF736ULL/', '/files/ENCFF848QHI/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-CTCF tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR274OGE/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF196VCR/', '/files/ENCFF173EDE/', '/files/ENCFF348IEF/', '/files/ENCFF058AVH/', '/files/ENCFF736ULL/', '/files/ENCFF848QHI/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:08.409664+00:00', 'accession': 'ENCSR274OGE', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/e4e51ac6-910f-4085-99f2-03dc9bbd585a/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR342AGU/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZSCAN5A_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE91681'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': 'de6dcce4-6bca-4f17-a8a4-75a3188f4c4a', 'original_files': ['/files/ENCFF172JDP/', '/files/ENCFF154SYW/', '/files/ENCFF719PXX/', '/files/ENCFF712WCF/', '/files/ENCFF944AKG/', '/files/ENCFF683AIZ/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZSCAN5A tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR342AGU/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF172JDP/', '/files/ENCFF154SYW/', '/files/ENCFF719PXX/', '/files/ENCFF712WCF/', '/files/ENCFF944AKG/', '/files/ENCFF683AIZ/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.394975+00:00', 'accession': 'ENCSR342AGU', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/7864e158-166b-426c-ae95-9cdaa43e57cb/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR354FNX/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZNF18_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE91693'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': 'c013defe-b3f0-4687-94ed-03759899023f', 'original_files': ['/files/ENCFF467VIQ/', '/files/ENCFF085HHB/', '/files/ENCFF771QFT/', '/files/ENCFF470DTH/', '/files/ENCFF348BKF/', '/files/ENCFF876NRR/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZNF18 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR354FNX/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF467VIQ/', '/files/ENCFF085HHB/', '/files/ENCFF771QFT/', '/files/ENCFF470DTH/', '/files/ENCFF348BKF/', '/files/ENCFF876NRR/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.155529+00:00', 'accession': 'ENCSR354FNX', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/b12b2202-3e9b-4f1c-a4af-1422616a63b0/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR429LAP/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-SP7_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE91755'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_title': 'Control ChIP-seq', 'uuid': 'd472964f-0a6f-4695-8cd4-2e3c4c42fe76', 'original_files': ['/files/ENCFF736BPA/', '/files/ENCFF595HTZ/', '/files/ENCFF084JUL/', '/files/ENCFF945YPE/', '/files/ENCFF854QOQ/', '/files/ENCFF909YLP/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-SP7 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR429LAP/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF736BPA/', '/files/ENCFF595HTZ/', '/files/ENCFF084JUL/', '/files/ENCFF945YPE/', '/files/ENCFF854QOQ/', '/files/ENCFF909YLP/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_term_name': 'ChIP-seq', 'date_created': '2016-07-29T23:26:07.808450+00:00', 'accession': 'ENCSR429LAP', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/ff90e222-fa5c-47d3-a8cb-d21f05937bd8/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}, {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/'], 'hub': '/experiments/ENCSR688NWK/@@hub/hub.txt', 'replication_type': 'unreplicated', 'revoked_files': [], 'related_series': ['/matched-sets/ENCSR300VIY/'], 'aliases': ['michael-snyder:HEK_eGFP-ZFP62_FRT_ChIP_input_4000'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE92232'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': '61b405a8-880f-45de-8067-2d5292f657ce', 'original_files': ['/files/ENCFF952FAG/', '/files/ENCFF956RLS/', '/files/ENCFF424ATY/', '/files/ENCFF145UQB/', '/files/ENCFF650VUY/', '/files/ENCFF361NSP/'], 'internal_status': 'pipeline completed', 'description': 'Input ChIP-seq on eGFP-ZFP62 tagged human HEK293 cells.', 'possible_controls': [], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR688NWK/', 'date_released': '2016-10-07', 'internal_tags': [], 'files': ['/files/ENCFF952FAG/', '/files/ENCFF956RLS/', '/files/ENCFF424ATY/', '/files/ENCFF145UQB/', '/files/ENCFF650VUY/', '/files/ENCFF361NSP/'], 'control_type': 'input library', 'date_submitted': '2016-08-01', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'Control ChIP-seq', 'date_created': '2016-07-29T23:26:08.057247+00:00', 'accession': 'ENCSR688NWK', 'month_released': 'October, 2016', 'lab': '/labs/michael-snyder/', 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': 'WARNING: The tagged TF could not be verified by AP-MS and may not reflect the annotated zinc finger target in the metadata.', 'references': [], 'type_slims': [], 'related_files': [], 'submitted_by': '/users/7dfa474a-d01d-417f-af5f-657c63cb254c/', 'replicates': ['/replicates/629454e0-aadb-4844-9abc-3cf56b23e228/'], 'contributing_files': ['/files/ENCFF643CGH/', '/files/ENCFF807MUK/']}], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], 'visualize': {'GRCh38': ['UCSC', 'Ensembl'], 'hg19': ['UCSC']}, '@id': '/experiments/ENCSR163RYW/', '@context': '/terms/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': [{'technical_replicates': ['1_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1'], 'libraries': ['9c03b3bd-4cce-438e-ab53-02d96e191ac8'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', 'library': '/libraries/ENCLB195KAR/', 'uuid': 'f26d90d9-53e2-4dff-8b98-c11d3fdb2831', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'biological_replicate_number': 1, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB976SGG/', 'date_created': '2016-12-14T21:13:52.766303+00:00'}, 'read_length_units': 'nt', 'read_count': 58871808, '@id': '/files/ENCFF271YPD/', 'cloud_metadata': {'file_size': 3571917417, 'url': 'https://encode-public.s3.amazonaws.com/2017/01/21/c7b7b658-9281-47ac-8a72-155f105a4c0f/ENCFF271YPD.fastq.gz', 'md5sum_base64': 'aMYg8ojVdYjpHang9fYMtg=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/01/21/c7b7b658-9281-47ac-8a72-155f105a4c0f/ENCFF271YPD.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1-read1'], 'schema_version': '15', 'md5sum': '68c620f288d57588e91da9e0f5f60cb6', 'content_md5sum': 'c6195fcae58cf9b2d55d13a35ebaffa3', 'href': '/files/ENCFF271YPD/@@download/ENCFF271YPD.fastq.gz', 'read_length': 101, 'title': 'ENCFF271YPD', 'library': '/libraries/ENCLB195KAR/', 'superseded_by': [], 'file_size': 3571917417, 'uuid': 'c7b7b658-9281-47ac-8a72-155f105a4c0f', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_160406_HEK-ZNF075_H7KY2_CAGATC_R1.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'CAGATC', 'flowcell': 'AH7KY2BBXX', 'lane': '3'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF736BPA/', '/files/ENCFF196VCR/', '/files/ENCFF154SYW/', '/files/ENCFF952FAG/', '/files/ENCFF557HNG/', '/files/ENCFF085HHB/'], 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'paired_with': '/files/ENCFF456PAU/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-01-21T00:47:20.855848+00:00', 'biological_replicates': [1], 'accession': 'ENCFF271YPD', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '1', 'dbxrefs': ['SRA:SRR8660124'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['H7KY2BBXX:3:1:AAGATC:', 'H7KY2BBXX:3:1:CAAATC:', 'H7KY2BBXX:3:1:CACATC:', 'H7KY2BBXX:3:1:CAGAAC:', 'H7KY2BBXX:3:1:CAGACC:', 'H7KY2BBXX:3:1:CAGAGC:', 'H7KY2BBXX:3:1:CAGATA:', 'H7KY2BBXX:3:1:CAGATC:', 'H7KY2BBXX:3:1:CAGATG:', 'H7KY2BBXX:3:1:CAGATN:', 'H7KY2BBXX:3:1:CAGATT:', 'H7KY2BBXX:3:1:CAGCTC:', 'H7KY2BBXX:3:1:CAGGTC:', 'H7KY2BBXX:3:1:CAGNTC:', 'H7KY2BBXX:3:1:CAGTTC:', 'H7KY2BBXX:3:1:CATATC:', 'H7KY2BBXX:3:1:CCGATC:', 'H7KY2BBXX:3:1:CGGATC:', 'H7KY2BBXX:3:1:CNGATC:', 'H7KY2BBXX:3:1:CTGATC:', 'H7KY2BBXX:3:1:GAGATC:', 'H7KY2BBXX:3:1:NAGATC:', 'H7KY2BBXX:3:1:TAGATC:']}, {'technical_replicates': ['1_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1'], 'libraries': ['9c03b3bd-4cce-438e-ab53-02d96e191ac8'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', 'library': '/libraries/ENCLB195KAR/', 'uuid': 'f26d90d9-53e2-4dff-8b98-c11d3fdb2831', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'biological_replicate_number': 1, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB976SGG/', 'date_created': '2016-12-14T21:13:52.766303+00:00'}, 'read_length_units': 'nt', 'read_count': 58871808, '@id': '/files/ENCFF456PAU/', 'cloud_metadata': {'file_size': 3718247071, 'url': 'https://encode-public.s3.amazonaws.com/2017/01/21/5987964c-5e9a-4d53-8106-df4bd0779e8b/ENCFF456PAU.fastq.gz', 'md5sum_base64': 'cbIh6Ge1gi24Mte6eo/VAw=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/01/21/5987964c-5e9a-4d53-8106-df4bd0779e8b/ENCFF456PAU.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1-read2'], 'schema_version': '15', 'md5sum': '71b221e867b5822db832d7ba7a8fd503', 'content_md5sum': 'd50217375de779a8e9bf10571e7b4f68', 'href': '/files/ENCFF456PAU/@@download/ENCFF456PAU.fastq.gz', 'read_length': 101, 'title': 'ENCFF456PAU', 'library': '/libraries/ENCLB195KAR/', 'superseded_by': [], 'file_size': 3718247071, 'uuid': '5987964c-5e9a-4d53-8106-df4bd0779e8b', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_160406_HEK-ZNF075_H7KY2_CAGATC_R2.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'CAGATC', 'flowcell': 'AH7KY2BBXX', 'lane': '3'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF595HTZ/', '/files/ENCFF173EDE/', '/files/ENCFF172JDP/', '/files/ENCFF956RLS/', '/files/ENCFF467VIQ/', '/files/ENCFF453HUS/'], 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'paired_with': '/files/ENCFF271YPD/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-01-21T00:49:54.579738+00:00', 'biological_replicates': [1], 'accession': 'ENCFF456PAU', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '2', 'dbxrefs': ['SRA:SRR8660124'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['H7KY2BBXX:3:2:AAGATC:', 'H7KY2BBXX:3:2:CAAATC:', 'H7KY2BBXX:3:2:CACATC:', 'H7KY2BBXX:3:2:CAGAAC:', 'H7KY2BBXX:3:2:CAGACC:', 'H7KY2BBXX:3:2:CAGAGC:', 'H7KY2BBXX:3:2:CAGATA:', 'H7KY2BBXX:3:2:CAGATC:', 'H7KY2BBXX:3:2:CAGATG:', 'H7KY2BBXX:3:2:CAGATN:', 'H7KY2BBXX:3:2:CAGATT:', 'H7KY2BBXX:3:2:CAGCTC:', 'H7KY2BBXX:3:2:CAGGTC:', 'H7KY2BBXX:3:2:CAGNTC:', 'H7KY2BBXX:3:2:CAGTTC:', 'H7KY2BBXX:3:2:CATATC:', 'H7KY2BBXX:3:2:CCGATC:', 'H7KY2BBXX:3:2:CGGATC:', 'H7KY2BBXX:3:2:CNGATC:', 'H7KY2BBXX:3:2:CTGATC:', 'H7KY2BBXX:3:2:GAGATC:', 'H7KY2BBXX:3:2:NAGATC:', 'H7KY2BBXX:3:2:TAGATC:']}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/raw_bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '0277afa9c395f02ad0d76bdf01a08190', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KYVgQ03696fqK09j81B2fx\", \"executable\": \"applet-F2pzVVj036944Z350k0FZzYQ\", \"user\": \"user-estherc\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [178276, 0], \"read1\": [58871808, 0], \"singletons\": [505233, 0], \"read2\": [58871808, 0], \"properly_paired\": [111694512, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [112123515, 0], \"mate_mapped_different_chr\": [228953, 0], \"in_total\": [117743616, 0], \"mapped\": [112628748, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY6qQ0369PfB8QBkj0X54b\", \"dx-id\": \"file-F3KfJk80g3F01z8BP7q7Q45Y\"}', '@id': '/files/ENCFF107GHX/', 'cloud_metadata': {'file_size': 6739158178, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/5fe9ef1b-0532-4c8e-b2ed-8f7a2a990ad6/ENCFF107GHX.bam', 'md5sum_base64': 'AnevqcOV8CrQ12vfAaCBkA=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': '5fe9ef1b-0532-4c8e-b2ed-8f7a2a990ad6', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/9820a1d7-427d-4619-85ec-b5dabef7c6e1/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-03-31T04:36:57.153477+00:00', 'biological_replicates': [1], 'accession': 'ENCFF107GHX', 's3_uri': 's3://encode-public/2017/03/31/5fe9ef1b-0532-4c8e-b2ed-8f7a2a990ad6/ENCFF107GHX.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/c729d612-7772-4811-87e2-9039138c23ca/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF107GHX/@@download/ENCFF107GHX.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': '0289badbeed91f8edbc18b598f738751', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 6739158178, 'title': 'ENCFF107GHX', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/raw_bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '3e4899be9f571ee28fb417ca223243c7', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KYXJj0369J51928vXyPxVP\", \"executable\": \"applet-F2pzVVj036944Z350k0FZzYQ\", \"user\": \"user-estherc\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [168174, 0], \"read1\": [58871808, 0], \"singletons\": [468804, 0], \"read2\": [58871808, 0], \"properly_paired\": [112699042, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [113152761, 0], \"mate_mapped_different_chr\": [218393, 0], \"in_total\": [117743616, 0], \"mapped\": [113621565, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY7480369GpQZkPJ206Gqj\", \"dx-id\": \"file-F3KfVg00bbkYZfg207F8PyQG\"}', '@id': '/files/ENCFF027GIB/', 'cloud_metadata': {'file_size': 6745996768, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/a1af7993-4071-419c-aa6f-370bc6eac418/ENCFF027GIB.bam', 'md5sum_base64': 'PkiZvp9XHuKPtBfKIjJDxw=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': 'a1af7993-4071-419c-aa6f-370bc6eac418', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/94162c93-74e4-4eee-a55d-d97622896d06/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-03-31T04:36:33.680804+00:00', 'biological_replicates': [1], 'accession': 'ENCFF027GIB', 's3_uri': 's3://encode-public/2017/03/31/a1af7993-4071-419c-aa6f-370bc6eac418/ENCFF027GIB.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/470e798f-2521-4eca-b451-617ef5d299a1/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF027GIB/@@download/ENCFF027GIB.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'cc8dd109186a91bb50e429c2bd6daac0', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 6745996768, 'title': 'ENCFF027GIB', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': 'c5da8f36fcc4444ec381d5149040e90d', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KY7480369GpQZkPJ206Gqy\", \"executable\": \"applet-F2pzVgj03696Z6g7613j6YyB\", \"user\": \"user-estherc\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1876868\", \"phantomPeakCoef\": \"1.115102\", \"estFragLen\": \"235\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.209290053606703\", \"relPhantomPeakCoef\": \"2.057722\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1981854\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [168174, 0], \"read1\": [58871808, 0], \"singletons\": [468804, 0], \"read2\": [58871808, 0], \"properly_paired\": [112699042, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [113152761, 0], \"mate_mapped_different_chr\": [218393, 0], \"in_total\": [117743616, 0], \"mapped\": [113621565, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"275777225\", \"read_pair_optical_duplicates\": \"4540294\", \"read_pairs_examined\": \"50551963\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.161691\", \"read_pair_duplicates\": \"8173816\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"6.091213\", \"PBC1\": \"0.838029\", \"NRF\": \"0.839997\", \"OneReadPair\": \"35081621\", \"DistinctReadPairs\": \"41862075\", \"TotalReadPairs\": \"49835969\", \"TwoReadPairs\": \"5759382\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [2, 0], \"read1\": [42378147, 0], \"singletons\": [0, 0], \"read2\": [42378147, 0], \"properly_paired\": [84756294, 0], \"paired_in_sequencing\": [84756294, 0], \"with_self_mate_mapped\": [84756294, 0], \"mate_mapped_different_chr\": [2, 0], \"in_total\": [84756294, 0], \"mapped\": [84756294, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY7480369GpQZkPJ206Gqj\", \"dx-id\": \"file-F3KpQv00GBkBqxV70748Y99q\"}', '@id': '/files/ENCFF431LTO/', 'cloud_metadata': {'file_size': 5174099343, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/c4c31fa4-88de-4edf-a09e-d82759523b87/ENCFF431LTO.bam', 'md5sum_base64': 'xdqPNvzERE7DgdUUkEDpDQ=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': 'c4c31fa4-88de-4edf-a09e-d82759523b87', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/54dc81a2-f922-4592-936f-8ca65d3943ef/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-03-31T04:40:42.369433+00:00', 'biological_replicates': [1], 'accession': 'ENCFF431LTO', 's3_uri': 's3://encode-public/2017/03/31/c4c31fa4-88de-4edf-a09e-d82759523b87/ENCFF431LTO.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/afcc99be-6f7c-437c-9ba4-12929bcd5e9b/', '/samtools-flagstats-quality-metrics/51966ee3-048e-40d8-8b79-a2ca31000940/', '/chipseq-filter-quality-metrics/9f34f438-ed83-4c59-b54d-223b1d2eefa2/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF431LTO/@@download/ENCFF431LTO.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': '4965dea8a57851ba2467a038a066cb8a', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5174099343, 'title': 'ENCFF431LTO', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/bams/ENCSR163RYW/rep1/ENCFF271YPDENCFF456PAU.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '0f84bd10c4a3cc82ba46fbd641f46ade', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3KY6qj0369PfB8QBkj0X54p\", \"executable\": \"applet-F2pzVgj03696Z6g7613j6YyB\", \"user\": \"user-estherc\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1885157\", \"phantomPeakCoef\": \"1.116719\", \"estFragLen\": \"230\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.210519113863597\", \"relPhantomPeakCoef\": \"1.988074\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1995834\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [178276, 0], \"read1\": [58871808, 0], \"singletons\": [505233, 0], \"read2\": [58871808, 0], \"properly_paired\": [111694512, 0], \"paired_in_sequencing\": [117743616, 0], \"with_self_mate_mapped\": [112123515, 0], \"mate_mapped_different_chr\": [228953, 0], \"in_total\": [117743616, 0], \"mapped\": [112628748, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"275948160\", \"read_pair_optical_duplicates\": \"4537609\", \"read_pairs_examined\": \"50549702\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.161605\", \"read_pair_duplicates\": \"8169067\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"6.091527\", \"PBC1\": \"0.838035\", \"NRF\": \"0.840003\", \"OneReadPair\": \"35088137\", \"DistinctReadPairs\": \"41869560\", \"TotalReadPairs\": \"49844533\", \"TwoReadPairs\": \"5760155\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [0, 0], \"read1\": [42380635, 0], \"singletons\": [0, 0], \"read2\": [42380635, 0], \"properly_paired\": [84761270, 0], \"paired_in_sequencing\": [84761270, 0], \"with_self_mate_mapped\": [84761270, 0], \"mate_mapped_different_chr\": [0, 0], \"in_total\": [84761270, 0], \"mapped\": [84761270, 0]}}, \"dx-parentAnalysis\": \"analysis-F3KY6qQ0369PfB8QBkj0X54b\", \"dx-id\": \"file-F3Kp5300Z9pFfpYXP8ZzJXQ5\"}', '@id': '/files/ENCFF014SGL/', 'cloud_metadata': {'file_size': 5171581198, 'url': 'https://encode-public.s3.amazonaws.com/2017/03/31/8b25c29d-7435-451a-aebf-1941f74ed6bd/ENCFF014SGL.bam', 'md5sum_base64': 'D4S9EMSjzIK6RvvWQfRq3g=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF271YPD/', '/files/ENCFF456PAU/'], 'uuid': '8b25c29d-7435-451a-aebf-1941f74ed6bd', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-0/', 'name': 'bwa-alignment-step-v-1-0', 'uuid': 'ced7264f-9206-4838-99ea-3badb3fb124c', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'step_run': '/analysis-step-runs/529dca74-5983-480d-8ec6-e96760196192/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-03-31T04:41:43.592576+00:00', 'biological_replicates': [1], 'accession': 'ENCFF014SGL', 's3_uri': 's3://encode-public/2017/03/31/8b25c29d-7435-451a-aebf-1941f74ed6bd/ENCFF014SGL.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/b31e92a2-7987-41e5-87ac-0a44982bf947/', '/chipseq-filter-quality-metrics/8286d9c7-87f7-4814-ace7-a75b7de141d9/', '/samtools-flagstats-quality-metrics/2b3b0f42-6f39-431c-a3c5-1ea9fb0dc744/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF014SGL/@@download/ENCFF014SGL.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'd055e93723b8a2f73a3fa75ac5ebea62', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5171581198, 'title': 'ENCFF014SGL', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r1.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '0bc986b2db90c3602cb9826dd0d888d1', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF419YNF/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1710402244, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/9d989b64-4550-45e7-a9f8-a6aabf59dae6/ENCFF419YNF.bigWig', 'md5sum_base64': 'C8mGstuQw2AsuYJt0NiI0Q=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF419YNF/@@download/ENCFF419YNF.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-04-03T17:48:38.208625+00:00', 'biological_replicates': [1], 'accession': 'ENCFF419YNF', 's3_uri': 's3://encode-public/2017/04/03/9d989b64-4550-45e7-a9f8-a6aabf59dae6/ENCFF419YNF.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '9d989b64-4550-45e7-a9f8-a6aabf59dae6', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803690VvxZJGGVzkqv\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VPKj0GX6vKV704y76FzVZ\"}', 'file_size': 1710402244, 'title': 'ENCFF419YNF', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R1.fixcoord.regionPeak.bb', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '6a14c3b4a7cf86fe42ea2997e5cc714b', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF358KJF/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13075865, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/6c47b101-9787-47a5-8dea-88f98754a7f0/ENCFF358KJF.bigBed', 'md5sum_base64': 'ahTDtKfPhv5C6imX5cxxSw=='}, 's3_uri': 's3://encode-public/2017/04/03/6c47b101-9787-47a5-8dea-88f98754a7f0/ENCFF358KJF.bigBed', 'file_size': 13075865, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF358KJF/@@download/ENCFF358KJF.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-04-03T17:49:13.207786+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF358KJF', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': '6c47b101-9787-47a5-8dea-88f98754a7f0', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj036970ZKz5Q51x6Yf\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Q8Q00j5qZ5PXb2Bz99PKk\"}', 'output_category': 'annotation', 'title': 'ENCFF358KJF', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/6751ed59-7d57-4db2-9b60-2fbf8e9d47e2/', 'derived_from': ['/files/ENCFF269RYL/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_stable.narrowPeak.gz', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '4d496fe76ec2a66a91e0ef5975dda88f', 'fastq_signature': [], '@id': '/files/ENCFF201MWD/', 'cloud_metadata': {'file_size': 652851, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/10d9c4a5-ae97-42c3-8aab-a9327b0d90c6/ENCFF201MWD.bed.gz', 'md5sum_base64': 'TUlv527CpmqR4O9Zdd2ojw=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/04/03/10d9c4a5-ae97-42c3-8aab-a9327b0d90c6/ENCFF201MWD.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-unreplicated-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-unreplicated-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-unreplicated-idr-step-v-1/', 'step_label': 'tf-unreplicated-idr-step', 'versions': ['/analysis-step-versions/tf-unreplicated-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '19dec341-2dd2-45aa-a9ad-48dd035a8cf0', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['unreplicated IDR'], 'title': 'TF unreplicated ChIP IDR step', 'aliases': ['encode:tf-unreplicated-idr-step-v-1'], 'name': 'tf-unreplicated-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL493SGC', '@type': ['Pipeline', 'Item'], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': [], '@id': '/pipelines/ENCPL493SGC/', 'assay_term_names': ['ChIP-seq'], 'analysis_steps': ['/analysis-steps/tf-unreplicated-spp-peak-calling-step-v-1/', '/analysis-steps/tf-unreplicated-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-unreplicated-idr-step-v-1/', '/analysis-steps/tf-unreplicated-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-unreplicated-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'de0d77d8-e53b-431a-9897-41f2c5b2a4a8', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq (unreplicated)', 'aliases': [], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-21T00:14:32.126175+00:00'}], 'schema_version': '8', 'output_file_types': ['pseudoreplicated IDR thresholded peaks'], 'date_created': '2017-03-21T00:14:31.005535+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-unreplicated-idr-step-v-1-0/', 'name': 'tf-unreplicated-idr-step-v-1-0', 'uuid': 'abd4401d-96aa-4c37-894d-8e0fee9273ec', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2017-03-21T00:14:31.359299+00:00'}, 'step_run': '/analysis-step-runs/e141362b-fd75-4a69-ab2b-a8db001b515d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-04-03T17:51:15.634518+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF201MWD', 'quality_metrics': ['/idr-quality-metrics/9ffb79a5-99d7-4876-85a5-b6ba93e3fc44/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF201MWD/@@download/ENCFF201MWD.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'pseudoreplicated IDR thresholded peaks', 'status': 'archived', 'technical_replicates': ['1_1'], 'uuid': '10d9c4a5-ae97-42c3-8aab-a9327b0d90c6', 'content_md5sum': 'cbb81e3c9ca28f22e94b2c2ebaad618f', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3PxV200369KFKj3PP5gy7JQ\", \"executable\": \"applet-F2pzbpj0369FZq3f5Q148vbk\", \"user\": \"user-estherc\"}, \"qc\": {\"N1\": 36721, \"Ns\": 36718}, \"dx-parentAnalysis\": \"analysis-F3PxV200369KFKj3PP5gy7JB\", \"dx-id\": \"file-F3Q43Z80yv3gV6bYP2gybb9z\"}', 'file_size': 652851, 'title': 'ENCFF201MWD', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF419RSJ/', '/files/ENCFF646KAN/'], 'superseded_by': []}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r1.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'c7cda89308162c4a701264877626b571', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF626DNK/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1828023313, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/58ecf6c1-1255-49fc-98b0-43196c019218/ENCFF626DNK.bigWig', 'md5sum_base64': 'x82okwgWLEpwEmSHdia1cQ=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF626DNK/@@download/ENCFF626DNK.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-04-03T17:48:40.057941+00:00', 'biological_replicates': [1], 'accession': 'ENCFF626DNK', 's3_uri': 's3://encode-public/2017/04/03/58ecf6c1-1255-49fc-98b0-43196c019218/ENCFF626DNK.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '58ecf6c1-1255-49fc-98b0-43196c019218', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGg80369P5x8QJ512zKBG\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Pp380X2G6G7y25V92Fg23\"}', 'file_size': 1828023313, 'title': 'ENCFF626DNK', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r1.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'f12afc2d1a31fe2c4cf6e2ac0239e602', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF414FRL/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1641931334, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/b709c7e6-76ae-4ad4-8291-c8e11a696272/ENCFF414FRL.bigWig', 'md5sum_base64': '8Sr8LRox/ixM9uKsAjnmAg=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF414FRL/@@download/ENCFF414FRL.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-04-03T17:47:30.214722+00:00', 'biological_replicates': [1], 'accession': 'ENCFF414FRL', 's3_uri': 's3://encode-public/2017/04/03/b709c7e6-76ae-4ad4-8291-c8e11a696272/ENCFF414FRL.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b709c7e6-76ae-4ad4-8291-c8e11a696272', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGg80369P5x8QJ512zKBG\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88PpQ80X2G70ZKz5Q51zQ36\"}', 'file_size': 1641931334, 'title': 'ENCFF414FRL', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r1.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '4cdb045fd01caad0c03c227e55e42583', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF752LHF/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1839178351, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/b3b95384-d06b-4dff-a6fd-51c77bbd4dcd/ENCFF752LHF.bigWig', 'md5sum_base64': 'TNsEX9AcqtDAPCJ+VeQlgw=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF752LHF/@@download/ENCFF752LHF.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-04-03T17:49:52.648029+00:00', 'biological_replicates': [1], 'accession': 'ENCFF752LHF', 's3_uri': 's3://encode-public/2017/04/03/b3b95384-d06b-4dff-a6fd-51c77bbd4dcd/ENCFF752LHF.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b3b95384-d06b-4dff-a6fd-51c77bbd4dcd', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803690VvxZJGGVzkqv\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VP1j0GX6vKV704y76FzK0\"}', 'file_size': 1839178351, 'title': 'ENCFF752LHF', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R1.fixcoord.regionPeak.gz', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '649f753c7878fc7227504b1b4f4513f6', 'fastq_signature': [], '@id': '/files/ENCFF646KAN/', 'cloud_metadata': {'file_size': 7211043, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/dc6a7ce7-2d79-42eb-8c97-8f386859b10e/ENCFF646KAN.bed.gz', 'md5sum_base64': 'ZJ91PHh4/HInUEsbT0UT9g=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/04/03/dc6a7ce7-2d79-42eb-8c97-8f386859b10e/ENCFF646KAN.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/e58c3f8a-46e6-45b5-ac74-1a517c4e5de3/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-04-03T17:50:22.884131+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF646KAN', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF646KAN/@@download/ENCFF646KAN.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': 'dc6a7ce7-2d79-42eb-8c97-8f386859b10e', 'content_md5sum': 'e5846a46a60c1728f5004957932c7000', 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03694GxQQ2B6F79Kq\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88Vqb00J77f3q5z2B0Yvq8B\"}', 'file_size': 7211043, 'title': 'ENCFF646KAN', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R1.fixcoord.regionPeak.bb', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'e4cd67bee3f7119fad6136ee633e4020', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF790GNI/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13038247, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/6e6012cc-f93b-41d4-8126-b1e4474f8d8a/ENCFF790GNI.bigBed', 'md5sum_base64': '5M1nvuP3EZ+tYTbuYz5AIA=='}, 's3_uri': 's3://encode-public/2017/04/03/6e6012cc-f93b-41d4-8126-b1e4474f8d8a/ENCFF790GNI.bigBed', 'file_size': 13038247, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF790GNI/@@download/ENCFF790GNI.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-04-03T17:50:28.270769+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF790GNI', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': '6e6012cc-f93b-41d4-8126-b1e4474f8d8a', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03694GxQQ2B6F79Kq\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VqZQ0J77Z5PXb2Bz99z9k\"}', 'output_category': 'annotation', 'title': 'ENCFF790GNI', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/55e555ad-73af-47b9-a82f-cb17fd71b1be/', 'derived_from': ['/files/ENCFF646KAN/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_stable.narrowPeak.bb', 'biological_replicates': [1], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '71684444b1a644fb70706e7fea030364', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF593RQD/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1237676, 'url': 'https://encode-public.s3.amazonaws.com/2017/04/03/eb7722cb-5a41-43b7-b5d9-5661cc2126eb/ENCFF593RQD.bigBed', 'md5sum_base64': 'cWhERLGmRPtwcG5/6gMDZA=='}, 's3_uri': 's3://encode-public/2017/04/03/eb7722cb-5a41-43b7-b5d9-5661cc2126eb/ENCFF593RQD.bigBed', 'file_size': 1237676, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-unreplicated-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['pseudoreplicated IDR thresholded peaks'], '@id': '/analysis-steps/tf-unreplicated-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-unreplicated-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '123a9667-abb2-442a-93ed-3715c67534ea', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF unreplicated ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-unreplicated-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-unreplicated-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL493SGC', '@type': ['Pipeline', 'Item'], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': [], '@id': '/pipelines/ENCPL493SGC/', 'assay_term_names': ['ChIP-seq'], 'analysis_steps': ['/analysis-steps/tf-unreplicated-spp-peak-calling-step-v-1/', '/analysis-steps/tf-unreplicated-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-unreplicated-idr-step-v-1/', '/analysis-steps/tf-unreplicated-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-unreplicated-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'de0d77d8-e53b-431a-9897-41f2c5b2a4a8', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq (unreplicated)', 'aliases': [], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-21T00:14:32.126175+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2017-03-21T00:14:31.136373+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-unreplicated-idr-peaks-to-bigbed-step-v-1-0', 'uuid': 'fa044b11-91ff-4320-a212-5f98f9f7e165', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2017-03-21T00:14:31.444896+00:00'}, 'href': '/files/ENCFF593RQD/@@download/ENCFF593RQD.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'pseudoreplicated IDR thresholded peaks', 'date_created': '2017-04-03T17:51:20.041223+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF593RQD', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/27b03780-0c87-4651-9966-a2e76573b465/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'archived', 'technical_replicates': ['1_1'], 'uuid': 'eb7722cb-5a41-43b7-b5d9-5661cc2126eb', 'superseded_by': [], 'submitted_by': {'uuid': '4c23ec32-c7c8-4ac0-affb-04befcc881d4', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Esther Chan', '@type': ['User', 'Item'], '@id': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F3PxV200369KFKj3PP5gy7JQ\", \"executable\": \"applet-F2pzbpj0369FZq3f5Q148vbk\", \"user\": \"user-estherc\"}, \"qc\": {\"N1\": 36721, \"Ns\": 36718}, \"dx-parentAnalysis\": \"analysis-F3PxV200369KFKj3PP5gy7JB\", \"dx-id\": \"file-F3Q43Z00yv3VyXVfP2YqyVFq\"}', 'output_category': 'annotation', 'title': 'ENCFF593RQD', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/4aca03d4-12a1-410a-a0e0-8fe8835cf14f/', 'derived_from': ['/files/ENCFF201MWD/']}, {'technical_replicates': ['2_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R2'], 'libraries': ['99423177-fbc8-4bf6-8241-f6b25855eb2b'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/', 'library': '/libraries/ENCLB762ZJL/', 'uuid': 'f5e07b4b-2a7e-4c95-b919-fa54792315a0', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'biological_replicate_number': 2, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB615WUN/', 'date_created': '2017-10-24T20:14:44.935001+00:00'}, 'read_length_units': 'nt', 'read_count': 51749759, '@id': '/files/ENCFF723NCL/', 'cloud_metadata': {'file_size': 3337831258, 'url': 'https://encode-public.s3.amazonaws.com/2017/10/26/771a0f39-fdf5-40b0-80be-2141b3456b8a/ENCFF723NCL.fastq.gz', 'md5sum_base64': '3nR2SKALdbqbzDstkqZCmw=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/10/26/771a0f39-fdf5-40b0-80be-2141b3456b8a/ENCFF723NCL.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R2.fastq.gz'], 'schema_version': '15', 'md5sum': 'de747648a00b75ba9bcc3b2d92a6429b', 'content_md5sum': '3eb4997fc118df543228f910b7dbeda0', 'href': '/files/ENCFF723NCL/@@download/ENCFF723NCL.fastq.gz', 'read_length': 101, 'title': 'ENCFF723NCL', 'library': '/libraries/ENCLB762ZJL/', 'superseded_by': [], 'file_size': 3337831258, 'uuid': '771a0f39-fdf5-40b0-80be-2141b3456b8a', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R2.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'TTAGGC', 'flowcell': 'AHJYFHBBXX', 'lane': '2'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF595HTZ/', '/files/ENCFF173EDE/', '/files/ENCFF172JDP/', '/files/ENCFF956RLS/', '/files/ENCFF467VIQ/', '/files/ENCFF453HUS/'], 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'paired_with': '/files/ENCFF031VYK/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-10-26T00:31:17.967950+00:00', 'biological_replicates': [2], 'accession': 'ENCFF723NCL', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '2', 'dbxrefs': ['SRA:SRR8660123'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['HJYFHBBXX:2:2:ATAGGC:', 'HJYFHBBXX:2:2:CTAGGC:', 'HJYFHBBXX:2:2:GTAGGC:', 'HJYFHBBXX:2:2:NTAGGC:', 'HJYFHBBXX:2:2:TAAGGC:', 'HJYFHBBXX:2:2:TCAGGC:', 'HJYFHBBXX:2:2:TGAGGC:', 'HJYFHBBXX:2:2:TNAGGC:', 'HJYFHBBXX:2:2:TTAAGC:', 'HJYFHBBXX:2:2:TTACGC:', 'HJYFHBBXX:2:2:TTAGAC:', 'HJYFHBBXX:2:2:TTAGCC:', 'HJYFHBBXX:2:2:TTAGGA:', 'HJYFHBBXX:2:2:TTAGGC:', 'HJYFHBBXX:2:2:TTAGGG:', 'HJYFHBBXX:2:2:TTAGGN:', 'HJYFHBBXX:2:2:TTAGGT:', 'HJYFHBBXX:2:2:TTAGNC:', 'HJYFHBBXX:2:2:TTAGTC:', 'HJYFHBBXX:2:2:TTANGC:', 'HJYFHBBXX:2:2:TTATGC:', 'HJYFHBBXX:2:2:TTCGGC:', 'HJYFHBBXX:2:2:TTGGGC:', 'HJYFHBBXX:2:2:TTNGGC:', 'HJYFHBBXX:2:2:TTTGGC:']}, {'technical_replicates': ['2_1'], 'file_format': 'fastq', 'replicate': {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R2'], 'libraries': ['99423177-fbc8-4bf6-8241-f6b25855eb2b'], 'status': 'released', 'experiment': {'documents': ['/documents/0994a79b-44ae-49ef-a44c-6b2ebf85a230/', '/documents/7047cb04-68a7-48be-b58b-6991f41ffbb6/'], 'hub': '/experiments/ENCSR163RYW/@@hub/hub.txt', 'replication_type': 'isogenic', 'revoked_files': [], 'related_series': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP'], 'assembly': ['GRCh38', 'hg19'], 'category_slims': [], 'assay_slims': ['DNA binding'], 'supersedes': [], 'dbxrefs': ['GEO:GSE127450'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', '@type': ['Experiment', 'Dataset', 'Item'], 'schema_version': '28', 'assay_term_name': 'ChIP-seq', 'uuid': 'faa59f8b-0bca-4f7d-b23c-472244a578f8', 'original_files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'internal_status': 'pipeline completed', 'description': 'eGFP-ZNF189 ChIP-seq on human HEK293 cells.', 'possible_controls': ['/matched-sets/ENCSR300VIY/', '/experiments/ENCSR157LMX/', '/experiments/ENCSR274OGE/', '/experiments/ENCSR342AGU/', '/experiments/ENCSR354FNX/', '/experiments/ENCSR429LAP/', '/experiments/ENCSR688NWK/'], 'superseded_by': [], 'assay_term_id': 'OBI:0000716', 'alternate_accessions': [], '@id': '/experiments/ENCSR163RYW/', 'date_released': '2017-11-28', 'internal_tags': ['ENCYCLOPEDIAv5'], 'files': ['/files/ENCFF271YPD/', '/files/ENCFF456PAU/', '/files/ENCFF107GHX/', '/files/ENCFF027GIB/', '/files/ENCFF431LTO/', '/files/ENCFF014SGL/', '/files/ENCFF419YNF/', '/files/ENCFF358KJF/', '/files/ENCFF201MWD/', '/files/ENCFF626DNK/', '/files/ENCFF414FRL/', '/files/ENCFF752LHF/', '/files/ENCFF646KAN/', '/files/ENCFF790GNI/', '/files/ENCFF593RQD/', '/files/ENCFF723NCL/', '/files/ENCFF031VYK/', '/files/ENCFF504NVE/', '/files/ENCFF332OVN/', '/files/ENCFF876FOW/', '/files/ENCFF634VOX/', '/files/ENCFF269RYL/', '/files/ENCFF413EHT/', '/files/ENCFF127GCV/', '/files/ENCFF887IHR/', '/files/ENCFF159RSM/', '/files/ENCFF304OQI/', '/files/ENCFF075WGX/', '/files/ENCFF699WXF/', '/files/ENCFF756PBC/', '/files/ENCFF643ALW/', '/files/ENCFF753OYI/', '/files/ENCFF579SRN/', '/files/ENCFF044QVM/', '/files/ENCFF382WGQ/', '/files/ENCFF681MQQ/', '/files/ENCFF939ITT/', '/files/ENCFF947TJN/', '/files/ENCFF372YXK/', '/files/ENCFF563JWG/', '/files/ENCFF613WKG/', '/files/ENCFF440BSA/', '/files/ENCFF164ASR/', '/files/ENCFF919CRG/', '/files/ENCFF824YMX/', '/files/ENCFF731GZP/'], 'date_submitted': '2017-01-20', 'award': '/awards/U54HG006996/', 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': '/organisms/human/', 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}, 'technical_replicate_number': 1, '@id': '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/', 'library': '/libraries/ENCLB762ZJL/', 'uuid': 'f5e07b4b-2a7e-4c95-b919-fa54792315a0', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'biological_replicate_number': 2, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': '/antibodies/ENCAB615WUN/', 'date_created': '2017-10-24T20:14:44.935001+00:00'}, 'read_length_units': 'nt', 'read_count': 51749759, '@id': '/files/ENCFF031VYK/', 'cloud_metadata': {'file_size': 3018903011, 'url': 'https://encode-public.s3.amazonaws.com/2017/10/26/40974f26-fa0e-4d2b-870d-19381d64b943/ENCFF031VYK.fastq.gz', 'md5sum_base64': '7YXJHHjHENCFn2yFHCo9aA=='}, 'output_category': 'raw data', 's3_uri': 's3://encode-public/2017/10/26/40974f26-fa0e-4d2b-870d-19381d64b943/ENCFF031VYK.fastq.gz', 'platform': {'@type': ['Platform', 'Item'], 'status': 'released', '@id': '/platforms/OBI%3A0002049/', 'term_name': 'Illumina HiSeq 4000', 'dbxrefs': [], 'uuid': '6bf2f062-a8f9-4758-bf77-7864b6fc3dd7', 'submitted_by': '/users/eb26c7d8-cdb4-4370-8c6b-204b441ef987/', 'title': 'Illumina HiSeq 4000', 'url': 'http://www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/hiseq-3000-4000-specification-sheet-770-2014-057.pdf', 'aliases': ['encode:HiSeq4000'], 'schema_version': '7', 'term_id': 'OBI:0002049', 'date_created': '2015-12-14T22:10:07.941975+00:00'}, 'no_file_available': False, 'aliases': ['michael-snyder:SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R1.fastq.gz'], 'schema_version': '15', 'md5sum': 'ed85c91c78c710d0859f6c851c2a3d68', 'content_md5sum': '5c8f4c67329db19e1bdf598853567155', 'href': '/files/ENCFF031VYK/@@download/ENCFF031VYK.fastq.gz', 'read_length': 101, 'title': 'ENCFF031VYK', 'library': '/libraries/ENCLB762ZJL/', 'superseded_by': [], 'file_size': 3018903011, 'uuid': '40974f26-fa0e-4d2b-870d-19381d64b943', '@type': ['File', 'Item'], 'run_type': 'paired-ended', 'submitted_file_name': '/srv/gsfs0/software/gbsc/encode/current/encode/dcc_submit/DnanexusFastqDownloads/SCGPM_HEK-ZNF120_HJYFH_L2_TTAGGC_R1.fastq.gz', 'alternate_accessions': [], 'flowcell_details': [{'machine': 'GADGET', 'barcode': 'TTAGGC', 'flowcell': 'AHJYFHBBXX', 'lane': '2'}], 'file_type': 'fastq', 'controlled_by': ['/files/ENCFF736BPA/', '/files/ENCFF196VCR/', '/files/ENCFF154SYW/', '/files/ENCFF952FAG/', '/files/ENCFF557HNG/', '/files/ENCFF085HHB/'], 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'paired_with': '/files/ENCFF723NCL/', 'award': '/awards/U54HG006996/', 'output_type': 'reads', 'date_created': '2017-10-26T00:28:53.772514+00:00', 'biological_replicates': [2], 'accession': 'ENCFF031VYK', 'quality_metrics': [], 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'paired_end': '1', 'dbxrefs': ['SRA:SRR8660123'], 'dataset': '/experiments/ENCSR163RYW/', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'fastq_signature': ['HJYFHBBXX:2:1:ATAGGC:', 'HJYFHBBXX:2:1:CTAGGC:', 'HJYFHBBXX:2:1:GTAGGC:', 'HJYFHBBXX:2:1:NTAGGC:', 'HJYFHBBXX:2:1:TAAGGC:', 'HJYFHBBXX:2:1:TCAGGC:', 'HJYFHBBXX:2:1:TGAGGC:', 'HJYFHBBXX:2:1:TNAGGC:', 'HJYFHBBXX:2:1:TTAAGC:', 'HJYFHBBXX:2:1:TTACGC:', 'HJYFHBBXX:2:1:TTAGAC:', 'HJYFHBBXX:2:1:TTAGCC:', 'HJYFHBBXX:2:1:TTAGGA:', 'HJYFHBBXX:2:1:TTAGGC:', 'HJYFHBBXX:2:1:TTAGGG:', 'HJYFHBBXX:2:1:TTAGGN:', 'HJYFHBBXX:2:1:TTAGGT:', 'HJYFHBBXX:2:1:TTAGNC:', 'HJYFHBBXX:2:1:TTAGTC:', 'HJYFHBBXX:2:1:TTANGC:', 'HJYFHBBXX:2:1:TTATGC:', 'HJYFHBBXX:2:1:TTCGGC:', 'HJYFHBBXX:2:1:TTGGGC:', 'HJYFHBBXX:2:1:TTNGGC:', 'HJYFHBBXX:2:1:TTTGGC:']}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/raw_bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '84b740737ef96d35a74693a91bfa11a6', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80P6YQ0369664VY542Vqp2f\", \"executable\": \"applet-F7KPgZQ03692zxFQ2fX49F6Q\", \"user\": \"user-mkajco\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [90142, 0], \"read1\": [51749759, 0], \"singletons\": [359518, 0], \"read2\": [51749759, 0], \"properly_paired\": [99970857, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [100213924, 0], \"mate_mapped_different_chr\": [130362, 0], \"in_total\": [103499518, 0], \"mapped\": [100573442, 0]}}, \"dx-parentAnalysis\": \"analysis-F80KyP8036979gz7P3FV2Kkz\", \"dx-id\": \"file-F80VX800Fj4YK4Bk40PJJ0XG\"}', '@id': '/files/ENCFF504NVE/', 'cloud_metadata': {'file_size': 5904774103, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/86e58ecf-3fc0-45c3-96e4-b30c195db76a/ENCFF504NVE.bam', 'md5sum_base64': 'hLdAc375bTWnRpOpG/oRpg=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': '86e58ecf-3fc0-45c3-96e4-b30c195db76a', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/974e2166-18fa-4360-902d-58b05a4fa4ac/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-11-07T07:36:57.673154+00:00', 'biological_replicates': [2], 'accession': 'ENCFF504NVE', 's3_uri': 's3://encode-public/2017/11/07/86e58ecf-3fc0-45c3-96e4-b30c195db76a/ENCFF504NVE.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/f7c7c0d8-f36a-4120-8686-307365495d3e/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF504NVE/@@download/ENCFF504NVE.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'c7c0b170dfcc6dff0dc01d92dce324e1', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5904774103, 'title': 'ENCFF504NVE', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_hg19/bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': 'e8542c35dc806823185369a1e8a8d710', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80KyPQ036979gz7P3FV2Kp4\", \"executable\": \"applet-F7KPgv0036980pKF8jBZy1PZ\", \"user\": \"user-mkajco\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1795315\", \"phantomPeakCoef\": \"1.097095\", \"estFragLen\": \"170\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.1969631869148\", \"relPhantomPeakCoef\": \"2.17445\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1875481\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [90142, 0], \"read1\": [51749759, 0], \"singletons\": [359518, 0], \"read2\": [51749759, 0], \"properly_paired\": [99970857, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [100213924, 0], \"mate_mapped_different_chr\": [130362, 0], \"in_total\": [103499518, 0], \"mapped\": [100573442, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"421896312\", \"read_pair_optical_duplicates\": \"6284927\", \"read_pairs_examined\": \"45638133\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.176706\", \"read_pair_duplicates\": \"8064541\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"5.538004\", \"PBC1\": \"0.821502\", \"NRF\": \"0.823309\", \"OneReadPair\": \"30797474\", \"DistinctReadPairs\": \"37489242\", \"TotalReadPairs\": \"45534857\", \"TwoReadPairs\": \"5561114\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [0, 0], \"read1\": [37573592, 0], \"singletons\": [0, 0], \"read2\": [37573592, 0], \"properly_paired\": [75147184, 0], \"paired_in_sequencing\": [75147184, 0], \"with_self_mate_mapped\": [75147184, 0], \"mate_mapped_different_chr\": [0, 0], \"in_total\": [75147184, 0], \"mapped\": [75147184, 0]}}, \"dx-parentAnalysis\": \"analysis-F80KyP8036979gz7P3FV2Kkz\", \"dx-id\": \"file-F80Zvk802VQ26Vky3zg295Jf\"}', '@id': '/files/ENCFF332OVN/', 'cloud_metadata': {'file_size': 4700745123, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/c3c56564-5abd-4abb-9fab-bd0a2eb0f21c/ENCFF332OVN.bam', 'md5sum_base64': '6FQsNdyAaCMYU2mh6KjXEA=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF807MUK/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': 'c3c56564-5abd-4abb-9fab-bd0a2eb0f21c', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/f6d5bf84-a7bf-440d-8142-c0b74350170a/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-11-07T07:42:44.433335+00:00', 'biological_replicates': [2], 'accession': 'ENCFF332OVN', 's3_uri': 's3://encode-public/2017/11/07/c3c56564-5abd-4abb-9fab-bd0a2eb0f21c/ENCFF332OVN.bam', 'quality_metrics': ['/chipseq-filter-quality-metrics/671cbb6f-ad0d-4ff4-a94d-da9e8f993c9e/', '/samtools-flagstats-quality-metrics/63bfab3b-0570-451a-a0b8-eda2d5dfdf6e/', '/samtools-flagstats-quality-metrics/c607ac97-8d0b-4783-8912-c736691e112e/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF332OVN/@@download/ENCFF332OVN.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': '7e177fb913171039ac73a2c45823c474', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 4700745123, 'title': 'ENCFF332OVN', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/raw_bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '3eb9e8cb9a88d073a35cf20f06f00328', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80P6GQ036968By67QJFj879\", \"executable\": \"applet-F7KPgZQ03692zxFQ2fX49F6Q\", \"user\": \"user-mkajco\"}, \"qc\": {\"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [60308, 0], \"read1\": [51749759, 0], \"singletons\": [283366, 0], \"read2\": [51749759, 0], \"properly_paired\": [101272312, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [101523157, 0], \"mate_mapped_different_chr\": [97879, 0], \"in_total\": [103499518, 0], \"mapped\": [101806523, 0]}}, \"dx-parentAnalysis\": \"analysis-F80Kyj003696ZqpyP2qyP81z\", \"dx-id\": \"file-F80X0xQ01JZ4vVqb3zq2fKvK\"}', '@id': '/files/ENCFF876FOW/', 'cloud_metadata': {'file_size': 5914167199, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/a78ee47c-951b-42ee-bee3-1bbc19f40cff/ENCFF876FOW.bam', 'md5sum_base64': 'Prnoy5qI0HOjXPIPBvADKA=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': 'a78ee47c-951b-42ee-bee3-1bbc19f40cff', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/f0690507-36ba-4fa5-81b8-6aaed61d6eb4/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'unfiltered alignments', 'date_created': '2017-11-07T08:05:39.165004+00:00', 'biological_replicates': [2], 'accession': 'ENCFF876FOW', 's3_uri': 's3://encode-public/2017/11/07/a78ee47c-951b-42ee-bee3-1bbc19f40cff/ENCFF876FOW.bam', 'quality_metrics': ['/samtools-flagstats-quality-metrics/70a51172-3fb1-4251-8753-6e29db35a352/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF876FOW/@@download/ENCFF876FOW.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'bff277b4cf47b1aea84ab5766c0dafed', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 5914167199, 'title': 'ENCFF876FOW', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/mapping_GRCh38/bams/ENCSR163RYW/rep2/ENCFF031VYKENCFF723NCL.raw.srt.filt.srt.nodup.bam', 'alternate_accessions': [], 'flowcell_details': [], 'read_length_units': 'nt', 'md5sum': '579f54b7c66d8af6b93f60e16f34b2fe', '@type': ['File', 'Item'], 'notes': '{\"dx-createdBy\": {\"job\": \"job-F80Kyj003696ZqpyP2qyP824\", \"executable\": \"applet-F7KPgv0036980pKF8jBZy1PZ\", \"user\": \"user-mkajco\"}, \"qc\": {\"xcor_qc\": {\"QualityTag\": \"2\", \"min_corr\": \"0.1844554\", \"phantomPeakCoef\": \"1.094677\", \"estFragLen\": \"170\", \"PhantomPeak\": \"105\", \"corr_estFragLen\": \"0.201919048686144\", \"relPhantomPeakCoef\": \"2.315045\", \"numReads\": \"15000000\", \"corr_phantomPeak\": \"0.1919989\", \"argmin_corr\": \"1500\"}, \"qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [60308, 0], \"read1\": [51749759, 0], \"singletons\": [283366, 0], \"read2\": [51749759, 0], \"properly_paired\": [101272312, 0], \"paired_in_sequencing\": [103499518, 0], \"with_self_mate_mapped\": [101523157, 0], \"mate_mapped_different_chr\": [97879, 0], \"in_total\": [103499518, 0], \"mapped\": [101806523, 0]}, \"dup_qc\": {\"unmapped_reads\": \"0\", \"estimated_library_size\": \"422967143\", \"read_pair_optical_duplicates\": \"6272561\", \"read_pairs_examined\": \"45584414\", \"unpaired_reads_examined\": \"0\", \"percent_duplication\": \"0.176467\", \"read_pair_duplicates\": \"8044135\", \"unpaired_read_duplicates\": \"0\"}, \"pbc_qc\": {\"PBC2\": \"5.543256\", \"PBC1\": \"0.821703\", \"NRF\": \"0.823552\", \"OneReadPair\": \"30776768\", \"DistinctReadPairs\": \"37454873\", \"TotalReadPairs\": \"45479692\", \"TwoReadPairs\": \"5552110\"}, \"filtered_qc\": {\"duplicates\": [0, 0], \"mate_mapped_different_chr_hiQ\": [2, 0], \"read1\": [37540279, 0], \"singletons\": [0, 0], \"read2\": [37540279, 0], \"properly_paired\": [75080558, 0], \"paired_in_sequencing\": [75080558, 0], \"with_self_mate_mapped\": [75080558, 0], \"mate_mapped_different_chr\": [2, 0], \"in_total\": [75080558, 0], \"mapped\": [75080558, 0]}}, \"dx-parentAnalysis\": \"analysis-F80Kyj003696ZqpyP2qyP81z\", \"dx-id\": \"file-F80bFQQ0Qbf03bzZ87JGYvPG\"}', '@id': '/files/ENCFF634VOX/', 'cloud_metadata': {'file_size': 4700789757, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/07/fe390a12-7c3f-4943-b837-9e6ba95cd527/ENCFF634VOX.bam', 'md5sum_base64': 'V59Ut8Ztiva5P2DhbzSy/g=='}, 'output_category': 'alignment', 'derived_from': ['/files/ENCFF643CGH/', '/files/ENCFF031VYK/', '/files/ENCFF723NCL/'], 'uuid': 'fe390a12-7c3f-4943-b837-9e6ba95cd527', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/bwa-indexing-step-v-1/'], 'current_version': '/analysis-step-versions/bwa-alignment-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['genome index', 'reads'], '@id': '/analysis-steps/bwa-alignment-step-v-1/', 'step_label': 'bwa-alignment-step', 'versions': ['/analysis-step-versions/bwa-alignment-step-v-1-0/', '/analysis-step-versions/bwa-alignment-step-v-1-1/'], 'major_version': 1, 'uuid': 'd72eced7-ad33-4cdc-8767-5a5ccaeb674c', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['fastq concatenation', 'alignment', 'filtering'], 'title': 'BWA alignment and filtration step', 'aliases': ['encode:bwa-alignment-step-v-1'], 'name': 'bwa-alignment-step-v-1', 'pipelines': [{'accession': 'ENCPL220NBH', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'references': [], 'documents': ['/documents/6f6351d4-9310-4a3b-a3c2-70ecac47b28b/'], '@id': '/pipelines/ENCPL220NBH/', 'assay_term_names': ['ChIP-seq'], 'description': 'ENCODE defined analysis pipeline for ChIP-seq read mapping', 'analysis_steps': ['/analysis-steps/bwa-indexing-step-v-1/', '/analysis-steps/bwa-alignment-step-v-1/'], 'uuid': '4168b003-a56e-452c-b550-1c9eaa9a1895', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/U41HG006992/', 'title': 'ChIP-seq read mapping', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/', 'status': 'released', 'schema_version': '10', 'date_created': '2017-03-09T18:29:33.666735+00:00'}], 'schema_version': '8', 'output_file_types': ['alignments', 'unfiltered alignments'], 'date_created': '2015-06-24T04:10:17.022631+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/bwa-alignment-step-v-1-1/', 'name': 'bwa-alignment-step-v-1-1', 'uuid': 'dc6dac54-0a5f-4356-815c-975c4b052249', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:bwa-alignment-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-07T02:06:41.028946+00:00'}, 'step_run': '/analysis-step-runs/74aa85fb-27e6-475e-b58c-9895e3a90452/', 'mapped_read_length': 101, 'no_file_available': False, 'file_type': 'bam', 'file_format': 'bam', 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'alignments', 'date_created': '2017-11-07T08:10:58.279948+00:00', 'biological_replicates': [2], 'accession': 'ENCFF634VOX', 's3_uri': 's3://encode-public/2017/11/07/fe390a12-7c3f-4943-b837-9e6ba95cd527/ENCFF634VOX.bam', 'quality_metrics': ['/chipseq-filter-quality-metrics/0b0b2aa4-4c4c-4b5c-bc46-c1eee0e61201/', '/samtools-flagstats-quality-metrics/f100be09-7597-4791-bd79-24c207a32e19/', '/samtools-flagstats-quality-metrics/ece7bfb9-a7b5-4287-b2c3-6a006b1543d2/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF634VOX/@@download/ENCFF634VOX.bam', 'schema_version': '15', 'dbxrefs': [], 'dataset': '/experiments/ENCSR163RYW/', 'content_md5sum': 'dfa94f34af8d8f2e4169c563950711e4', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'status': 'released', 'file_size': 4700789757, 'title': 'ENCFF634VOX', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'fastq_signature': [], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R1.regionPeak.gz', 'biological_replicates': [1], 'alternate_accessions': ['ENCFF309UAI'], 'flowcell_details': [], 'md5sum': 'ebc9d60199db8c009eb5b490bac21b09', 'fastq_signature': [], '@id': '/files/ENCFF269RYL/', 'cloud_metadata': {'file_size': 7251371, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/df0fd769-c921-4567-8d68-866f5c600652/ENCFF269RYL.bed.gz', 'md5sum_base64': '68nWAZnbjACetbSQusIbCQ=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/df0fd769-c921-4567-8d68-866f5c600652/ENCFF269RYL.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/7560b4b8-eb97-42e2-ad65-8994e668678d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:09.855438+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF269RYL', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF269RYL/@@download/ENCFF269RYL.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1'], 'uuid': 'df0fd769-c921-4567-8d68-866f5c600652', 'content_md5sum': '94c26eca2d90e55488f7c2ca4138ae36', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj036970ZKz5Q51x6Yf\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Q8Q80j5qZGxQQ2B6F9Z5Q\"}', 'file_size': 7251371, 'title': 'ENCFF269RYL', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF014SGL/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/PL.regionPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '1f7bc4ee6eff6069d77868393a896a02', 'fastq_signature': [], '@id': '/files/ENCFF413EHT/', 'cloud_metadata': {'file_size': 7325551, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/cba4d008-4552-4e05-a4f3-cef34b2b12b1/ENCFF413EHT.bed.gz', 'md5sum_base64': 'H3vE7m7/YGnXeGg5OolqAg=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/cba4d008-4552-4e05-a4f3-cef34b2b12b1/ENCFF413EHT.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/7560b4b8-eb97-42e2-ad65-8994e668678d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:21.622838+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF413EHT', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF413EHT/@@download/ENCFF413EHT.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'cba4d008-4552-4e05-a4f3-cef34b2b12b1', 'content_md5sum': '9814dfc96a89c5ed76dd2d60c59f4a35', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03696G7y25V929JV3\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Vqx80yxgGKXJB5QfK1P3X\"}', 'file_size': 7325551, 'title': 'ENCFF413EHT', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/', '/files/ENCFF014SGL/'], 'superseded_by': []}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/pool.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '1dc8492bc2730d7b1e3bcdefe9598e57', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF127GCV/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 2415172363, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/30e979c3-3b0f-47fb-b75a-3dd99a0bf8cb/ENCFF127GCV.bigWig', 'md5sum_base64': 'HchJK8JzDXseO83v6VmOVw=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF127GCV/@@download/ENCFF127GCV.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:06:36.916409+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF127GCV', 's3_uri': 's3://encode-public/2017/11/21/30e979c3-3b0f-47fb-b75a-3dd99a0bf8cb/ENCFF127GCV.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '30e979c3-3b0f-47fb-b75a-3dd99a0bf8cb', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFBQ036963q5z2B0Yp1b3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88Y9XQ03g36G7y25V92J4QZ\"}', 'file_size': 2415172363, 'title': 'ENCFF127GCV', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r2.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '5de4c2d208d3251354d0acf20d2357f8', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF887IHR/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1665047480, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/ea60aa14-3ac6-4bde-9123-dddde2bf5453/ENCFF887IHR.bigWig', 'md5sum_base64': 'XeTC0gjTJRNU0KzyDSNX+A=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF887IHR/@@download/ENCFF887IHR.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T16:59:36.856844+00:00', 'biological_replicates': [2], 'accession': 'ENCFF887IHR', 's3_uri': 's3://encode-public/2017/11/21/ea60aa14-3ac6-4bde-9123-dddde2bf5453/ENCFF887IHR.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'ea60aa14-3ac6-4bde-9123-dddde2bf5453', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803694GxQQ2B6F79P3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VBGj06zvPvjy4J5z2BB5V\"}', 'file_size': 1665047480, 'title': 'ENCFF887IHR', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.gz', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'df298d239e82b16b3f65eabaf9a3d95e', 'fastq_signature': [], '@id': '/files/ENCFF159RSM/', 'cloud_metadata': {'file_size': 7005883, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/4f4525aa-58a7-40b4-ba35-d3c2286905d6/ENCFF159RSM.bed.gz', 'md5sum_base64': '3ymNI56CsWs/Zeq6+aPZXg=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/4f4525aa-58a7-40b4-ba35-d3c2286905d6/ENCFF159RSM.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/e58c3f8a-46e6-45b5-ac74-1a517c4e5de3/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:07:21.620168+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF159RSM', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF159RSM/@@download/ENCFF159RSM.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': '4f4525aa-58a7-40b4-ba35-d3c2286905d6', 'content_md5sum': 'ee7182c380afddee01261a9cedd73c09', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q036945PXb2Bz971jb\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88QZB80XV7kX5f029x2fZGg\"}', 'file_size': 7005883, 'title': 'ENCFF159RSM', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF634VOX/'], 'superseded_by': []}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r2.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '63df25c8fce24248889b7e0f23f37ded', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF304OQI/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1689568852, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/b8fe284f-d7f3-4d0f-9dcc-17bcc1883d22/ENCFF304OQI.bigWig', 'md5sum_base64': 'Y98lyPziQkiIm34PI/N97Q=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF304OQI/@@download/ENCFF304OQI.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:02:53.187619+00:00', 'biological_replicates': [2], 'accession': 'ENCFF304OQI', 's3_uri': 's3://encode-public/2017/11/21/b8fe284f-d7f3-4d0f-9dcc-17bcc1883d22/ENCFF304OQI.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b8fe284f-d7f3-4d0f-9dcc-17bcc1883d22', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369B1Y728jz1X0Kx\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88QxkQ0q912jFY45V2XJYJX\"}', 'file_size': 1689568852, 'title': 'ENCFF304OQI', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/r2.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '50fce2cb8f00c97e3f761a5143219223', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF075WGX/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1689246633, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/573ee561-d974-4c24-9bd2-8e86c6609cb4/ENCFF075WGX.bigWig', 'md5sum_base64': 'UPziy48AyX4/dhpRQyGSIw=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF075WGX/@@download/ENCFF075WGX.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:03:47.037981+00:00', 'biological_replicates': [2], 'accession': 'ENCFF075WGX', 's3_uri': 's3://encode-public/2017/11/21/573ee561-d974-4c24-9bd2-8e86c6609cb4/ENCFF075WGX.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '573ee561-d974-4c24-9bd2-8e86c6609cb4', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFB803694GxQQ2B6F79P3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88VB0Q06zv1795j5QvggjVP\"}', 'file_size': 1689246633, 'title': 'ENCFF075WGX', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '2f6121f1ed8a7ad4d2b643597ec02f37', 'fastq_signature': [], '@id': '/files/ENCFF699WXF/', 'cloud_metadata': {'file_size': 722459, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/7fededf2-7502-45f3-a83d-d9118d01a1ac/ENCFF699WXF.bed.gz', 'md5sum_base64': 'L2Eh8e2KetTStkNZfsAvNw=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/7fededf2-7502-45f3-a83d-d9118d01a1ac/ENCFF699WXF.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/b6d04c42-8125-469a-9009-db723d318b37/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:56.150585+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF699WXF', 'quality_metrics': ['/idr-quality-metrics/4043bba6-2c64-4af9-b053-e05556b7a7f9/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF699WXF/@@download/ENCFF699WXF.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'optimal IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '7fededf2-7502-45f3-a83d-d9118d01a1ac', 'content_md5sum': 'b34a920f1b1a142f66da176262d9bfaf', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp980X2k3QP2688Jb8Zv3\"}', 'file_size': 722459, 'title': 'ENCFF699WXF', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF413EHT/', '/files/ENCFF001TDO/', '/files/ENCFF269RYL/', '/files/ENCFF947TJN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.bb', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '436d80e0c8e005b3347ccdb391d1bd1a', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF756PBC/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13409044, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/3ecf312f-37ab-4b73-b050-10039f5fa97c/ENCFF756PBC.bigBed', 'md5sum_base64': 'Q22A4MjgBbM0fM2zkdG9Gg=='}, 's3_uri': 's3://encode-public/2017/11/21/3ecf312f-37ab-4b73-b050-10039f5fa97c/ENCFF756PBC.bigBed', 'file_size': 13409044, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF756PBC/@@download/ENCFF756PBC.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:07:41.326075+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF756PBC', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': '3ecf312f-37ab-4b73-b050-10039f5fa97c', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q036945PXb2Bz971jb\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88QZB00XV7ZGxQQ2B6F9jgp\"}', 'output_category': 'annotation', 'title': 'ENCFF756PBC', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/55e555ad-73af-47b9-a82f-cb17fd71b1be/', 'derived_from': ['/files/ENCFF159RSM/']}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/pool.fc_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '9947868b85751a1b81b382483574c06c', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF643ALW/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 2414374710, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/0bcfab0f-f19b-494b-a0ee-98d522f26319/ENCFF643ALW.bigWig', 'md5sum_base64': 'mUeGi4V1GhuBs4JINXTAbA=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF643ALW/@@download/ENCFF643ALW.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'fold change over control', 'date_created': '2017-11-21T17:05:27.591360+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF643ALW', 's3_uri': 's3://encode-public/2017/11/21/0bcfab0f-f19b-494b-a0ee-98d522f26319/ENCFF643ALW.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '0bcfab0f-f19b-494b-a0ee-98d522f26319', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369GQQ8z9240J76X\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88VVp802Y1vbPJ24xQjFY04\"}', 'file_size': 2414374710, 'title': 'ENCFF643ALW', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/PL.fixcoord.regionPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '0f05b53f05c5020b9c75c9f05a8aa8f7', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF753OYI/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13173926, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/8b083846-fa69-4098-bf7f-c8331aca5e42/ENCFF753OYI.bigBed', 'md5sum_base64': 'DwW1PwXFAgucdcnwWoqo9w=='}, 's3_uri': 's3://encode-public/2017/11/21/8b083846-fa69-4098-bf7f-c8331aca5e42/ENCFF753OYI.bigBed', 'file_size': 13173926, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF753OYI/@@download/ENCFF753OYI.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:06:39.198345+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF753OYI', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '8b083846-fa69-4098-bf7f-c8331aca5e42', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03696G7y25V929JV3\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Vqx00yxg3Yyzv5VXVg38Q\"}', 'output_category': 'annotation', 'title': 'ENCFF753OYI', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/6751ed59-7d57-4db2-9b60-2fbf8e9d47e2/', 'derived_from': ['/files/ENCFF413EHT/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '9607aaf7af5e582e2ddc37e01a449102', 'fastq_signature': [], '@id': '/files/ENCFF579SRN/', 'cloud_metadata': {'file_size': 551998, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/6155b816-9597-403d-b5a6-a2ac5dec221a/ENCFF579SRN.bed.gz', 'md5sum_base64': 'lgeq969eWC4t3DfgGkSRAg=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/6155b816-9597-403d-b5a6-a2ac5dec221a/ENCFF579SRN.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/b6d04c42-8125-469a-9009-db723d318b37/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:46.600169+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF579SRN', 'quality_metrics': ['/idr-quality-metrics/4043bba6-2c64-4af9-b053-e05556b7a7f9/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF579SRN/@@download/ENCFF579SRN.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'conservative IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '6155b816-9597-403d-b5a6-a2ac5dec221a', 'content_md5sum': '4048b4e0f305a4b313b79da5368667f6', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp980X2kGbPJ24xQjGV24\"}', 'file_size': 551998, 'title': 'ENCFF579SRN', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF413EHT/', '/files/ENCFF001TDO/', '/files/ENCFF269RYL/', '/files/ENCFF947TJN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'e8cbc0507646020010e84075ba737f0f', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF044QVM/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1333769, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/0a92e9ca-0160-4671-b00d-b264ca588dd3/ENCFF044QVM.bigBed', 'md5sum_base64': '6MvAUHZGAgAQ6EB1unN/Dw=='}, 's3_uri': 's3://encode-public/2017/11/21/0a92e9ca-0160-4671-b00d-b264ca588dd3/ENCFF044QVM.bigBed', 'file_size': 1333769, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF044QVM/@@download/ENCFF044QVM.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'optimal IDR thresholded peaks', 'date_created': '2017-11-21T17:07:00.591299+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF044QVM', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/12315382-15e5-411f-adbe-c022a8dfc50a/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '0a92e9ca-0160-4671-b00d-b264ca588dd3', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp900X2k76KQy4x97252F\"}', 'output_category': 'annotation', 'title': 'ENCFF044QVM', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/a120edc6-130f-4c22-9f25-0d16ef2ce643/', 'derived_from': ['/files/ENCFF699WXF/']}, {'technical_replicates': ['2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/r2.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '676887a53da5f5658ccf891ce3e3d16f', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF382WGQ/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1498850245, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/7978357e-e69d-4ed2-814d-b2636aad7ac2/ENCFF382WGQ.bigWig', 'md5sum_base64': 'Z2iHpT2l9WWMz4kc4+PRbw=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF382WGQ/@@download/ENCFF382WGQ.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T16:58:55.511167+00:00', 'biological_replicates': [2], 'accession': 'ENCFF382WGQ', 's3_uri': 's3://encode-public/2017/11/21/7978357e-e69d-4ed2-814d-b2636aad7ac2/ENCFF382WGQ.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '7978357e-e69d-4ed2-814d-b2636aad7ac2', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369B1Y728jz1X0Kx\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Qy580q91GKXJB5QfK17Yf\"}', 'file_size': 1498850245, 'title': 'ENCFF382WGQ', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_macs2/pool.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'e04a7288efe6d9576e528aa40f8c3e93', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF681MQQ/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1942386368, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/3328d0fe-88a6-4aba-8cd8-d8615286f412/ENCFF681MQQ.bigWig', 'md5sum_base64': '4EpyiO/m2VduUoqkD4w+kw=='}, 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/', '/files/ENCFF014SGL/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF681MQQ/@@download/ENCFF681MQQ.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T17:00:59.177241+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF681MQQ', 's3_uri': 's3://encode-public/2017/11/21/3328d0fe-88a6-4aba-8cd8-d8615286f412/ENCFF681MQQ.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': '3328d0fe-88a6-4aba-8cd8-d8615286f412', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGgQ0369GQQ8z9240J76X\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88VXF802Y1vKV704y76G0gj\"}', 'file_size': 1942386368, 'title': 'ENCFF681MQQ', 'aliases': [], 'step_run': '/analysis-step-runs/0b3456fd-5c8b-4b03-acb4-2b5fa3c9009b/'}, {'technical_replicates': ['1_1', '2_1'], 'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_macs2/pool.pvalue_signal.bw', 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '4094c8067847429468c491b688fb7ca2', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF939ITT/', 'file_type': 'bigWig', 'cloud_metadata': {'file_size': 1944402799, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/b05e9401-a6c5-44e8-ab27-433b35943380/ENCFF939ITT.bigWig', 'md5sum_base64': 'QJTIBnhHQpRoxJG2iPt8og=='}, 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/', '/files/ENCFF634VOX/'], 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-macs2-signal-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-macs2-signal-calling-step-v-1/', 'step_label': 'tf-macs2-signal-calling-step', 'versions': ['/analysis-step-versions/tf-macs2-signal-calling-step-v-1-0/', '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7c02066c-da5f-4316-8492-cd70419284b3', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['signal generation'], 'title': 'TF ChIP MACS2 signal calling step', 'name': 'tf-macs2-signal-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['fold change over control', 'signal p-value'], 'date_created': '2015-10-30T22:08:29.178581+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-macs2-signal-calling-step-v-1-1/', 'name': 'tf-macs2-signal-calling-step-v-1-1', 'uuid': '37284851-f58c-4c4e-898d-0b595699cd20', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/'], 'aliases': ['encode:tf-macs2-signal-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:34:35.859422+00:00'}, 'href': '/files/ENCFF939ITT/@@download/ENCFF939ITT.bigWig', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'output_category': 'signal', 'award': '/awards/U41HG006992/', 'output_type': 'signal p-value', 'date_created': '2017-11-21T17:01:48.372219+00:00', 'biological_replicates': [1, 2], 'accession': 'ENCFF939ITT', 's3_uri': 's3://encode-public/2017/11/21/b05e9401-a6c5-44e8-ab27-433b35943380/ENCFF939ITT.bigWig', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'status': 'released', 'file_format': 'bigWig', 'dataset': '/experiments/ENCSR163RYW/', 'schema_version': '15', 'dbxrefs': [], 'uuid': 'b05e9401-a6c5-44e8-ab27-433b35943380', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FFBQ036963q5z2B0Yp1b3\", \"executable\": \"applet-F7KPjf0036980pKF8jBZy1Pz\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88YB4Q03g3GKXJB5QfK2548\"}', 'file_size': 1944402799, 'title': 'ENCFF939ITT', 'aliases': [], 'step_run': '/analysis-step-runs/bfd09857-c819-4197-b4d7-ef39ae15d13a/'}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.gz', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '9c7ba6a7b72a87f37d11a6f3e597d365', 'fastq_signature': [], '@id': '/files/ENCFF947TJN/', 'cloud_metadata': {'file_size': 7005891, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/d287107c-15ff-49d5-85d8-553ef23079d8/ENCFF947TJN.bed.gz', 'md5sum_base64': 'nHump7cqh/N9Eabz5ZfTZQ=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/d287107c-15ff-49d5-85d8-553ef23079d8/ENCFF947TJN.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/7560b4b8-eb97-42e2-ad65-8994e668678d/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:06:15.775373+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF947TJN', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF947TJN/@@download/ENCFF947TJN.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': 'd287107c-15ff-49d5-85d8-553ef23079d8', 'content_md5sum': 'c0b8c1c6a939e7ad41befdf7e30d75aa', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03691795j5QvgZvgg\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88PG6806XG2jFY45V2XGx1x\"}', 'file_size': 7005891, 'title': 'ENCFF947TJN', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF182WHU/', '/files/ENCFF332OVN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/encode_spp/R2.fixcoord.regionPeak.bb', 'biological_replicates': [2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '557f752a63ee193d9233972afd4c38d7', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF372YXK/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13402106, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/58bd8de8-752d-4a6b-bdb1-8e58cefb52fc/ENCFF372YXK.bigBed', 'md5sum_base64': 'VX91KmPuGT2SM5cq/Uw41w=='}, 's3_uri': 's3://encode-public/2017/11/21/58bd8de8-752d-4a6b-bdb1-8e58cefb52fc/ENCFF372YXK.bigBed', 'file_size': 13402106, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF372YXK/@@download/ENCFF372YXK.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:06:32.701602+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF372YXK', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['2_1'], 'uuid': '58bd8de8-752d-4a6b-bdb1-8e58cefb52fc', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FGpj03691795j5QvgZvgg\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88PG5j06XG1795j5Qvgfqvv\"}', 'output_category': 'annotation', 'title': 'ENCFF372YXK', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/6751ed59-7d57-4db2-9b60-2fbf8e9d47e2/', 'derived_from': ['/files/ENCFF947TJN/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_hg19/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'a9ee9c5110e45c9c3d0517f4f896b84e', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF563JWG/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1076586, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/4eb3b64b-bd5b-4f75-84bd-9bd40c89905a/ENCFF563JWG.bigBed', 'md5sum_base64': 'qe6cURDkXJw9BRf0+Ja4Tg=='}, 's3_uri': 's3://encode-public/2017/11/21/4eb3b64b-bd5b-4f75-84bd-9bd40c89905a/ENCFF563JWG.bigBed', 'file_size': 1076586, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF563JWG/@@download/ENCFF563JWG.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'hg19', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'conservative IDR thresholded peaks', 'date_created': '2017-11-21T17:06:51.568790+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF563JWG', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/12315382-15e5-411f-adbe-c022a8dfc50a/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '4eb3b64b-bd5b-4f75-84bd-9bd40c89905a', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F7q003696z8KxG3g7ZF6V\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3437627951412583, \"Np\": 39383, \"N1\": 36365, \"N2\": 22857, \"Nt\": 29308, \"self_consistency_ratio\": 1.590978693616835}, \"dx-parentAnalysis\": \"analysis-F88F7pj03696z8KxG3g7ZF68\", \"dx-id\": \"file-F88Yp900X2kK66ZB878ZG1yP\"}', 'output_category': 'annotation', 'title': 'ENCFF563JWG', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/a120edc6-130f-4c22-9f25-0d16ef2ce643/', 'derived_from': ['/files/ENCFF579SRN/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/PL.fixcoord.regionPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '049ff8462215837405e58ee458b00707', 'fastq_signature': [], '@id': '/files/ENCFF613WKG/', 'cloud_metadata': {'file_size': 7340148, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/fc67164e-7ae8-4779-af6d-e539cafd4ec8/ENCFF613WKG.bed.gz', 'md5sum_base64': 'BJ/4RiIVg3QF5Y7kWLAHBw=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/fc67164e-7ae8-4779-af6d-e539cafd4ec8/ENCFF613WKG.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'aliases': ['encode:tf-spp-peak-calling-step-v-1'], 'current_version': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['alignments'], '@id': '/analysis-steps/tf-spp-peak-calling-step-v-1/', 'step_label': 'tf-spp-peak-calling-step', 'versions': ['/analysis-step-versions/tf-spp-peak-calling-step-v-1-0/', '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/'], 'major_version': 1, 'uuid': '7a74345f-f75f-49e5-9001-eca3d3dfdc64', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['peak calling'], 'title': 'TF ChIP SPP peak calling step', 'name': 'tf-spp-peak-calling-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['peaks'], 'date_created': '2015-10-28T21:42:25.477498+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-spp-peak-calling-step-v-1-1/', 'name': 'tf-spp-peak-calling-step-v-1-1', 'uuid': '63326924-00a7-414f-ba7d-923c0f25da92', 'minor_version': 1, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/'], 'aliases': ['encode:tf-spp-peak-calling-step-v-1.2'], 'schema_version': '4', 'date_created': '2016-10-18T04:36:53.223945+00:00'}, 'step_run': '/analysis-step-runs/e58c3f8a-46e6-45b5-ac74-1a517c4e5de3/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:07:27.536132+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF613WKG', 'quality_metrics': [], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF613WKG/@@download/ENCFF613WKG.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'peaks and background as input for IDR', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'fc67164e-7ae8-4779-af6d-e539cafd4ec8', 'content_md5sum': 'f37aec6c20adf630d986b864ee373a81', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03690VvxZJGGVzkqX\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88YYXQ0f66vQQ8z9240V3y3\"}', 'file_size': 7340148, 'title': 'ENCFF613WKG', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF810LLR/', '/files/ENCFF431LTO/', '/files/ENCFF634VOX/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '7b6508f7051a9f9bccce5ae306cdcdf5', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF440BSA/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1349888, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/2edfb654-c4ef-4d2d-aa52-e06d7cb0deaf/ENCFF440BSA.bigBed', 'md5sum_base64': 'e2UI9wUan5vMzlrjBs3N9Q=='}, 's3_uri': 's3://encode-public/2017/11/21/2edfb654-c4ef-4d2d-aa52-e06d7cb0deaf/ENCFF440BSA.bigBed', 'file_size': 1349888, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF440BSA/@@download/ENCFF440BSA.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'optimal IDR thresholded peaks', 'date_created': '2017-11-21T17:08:08.725157+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF440BSA', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/dbffd278-f5c4-4188-8df3-1258e74c7960/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '2edfb654-c4ef-4d2d-aa52-e06d7cb0deaf', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX3j0z5kGKV704y76KJ91\"}', 'output_category': 'annotation', 'title': 'ENCFF440BSA', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/fa6a52fa-ca11-4abe-94a5-a96c6b99358a/', 'derived_from': ['/files/ENCFF824YMX/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/encode_spp/PL.fixcoord.regionPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': 'edb4b2008ed1c2b7b74dcf517893af2c', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF164ASR/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 13196767, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/ee4cf664-0fe4-4bbc-bb99-735377fc9623/ENCFF164ASR.bigBed', 'md5sum_base64': '7bSyAI7Rwre3Tc9ReJOvLA=='}, 's3_uri': 's3://encode-public/2017/11/21/ee4cf664-0fe4-4bbc-bb99-735377fc9623/ENCFF164ASR.bigBed', 'file_size': 13196767, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': '7b8e4925-6401-4737-8043-7722e81a02ee', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP peaks to bigBed file format conversion step', 'aliases': ['encode:tf-peaks-to-bigbed-step-v-1'], 'name': 'tf-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.111701+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-peaks-to-bigbed-step-v-1-0', 'uuid': '511ba3d7-4b15-49a8-901e-c7516f3312c5', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.234825+00:00'}, 'href': '/files/ENCFF164ASR/@@download/ENCFF164ASR.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'peaks and background as input for IDR', 'date_created': '2017-11-21T17:07:48.580205+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF164ASR', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': [], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'ee4cf664-0fe4-4bbc-bb99-735377fc9623', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88FF9Q03690VvxZJGGVzkqX\", \"executable\": \"applet-F7KPjP80369K49260Y8kbyf1\", \"user\": \"user-mkajco\"}, \"qc\": {}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88YYX80f66zb63QJ6VQjB6X\"}', 'output_category': 'annotation', 'title': 'ENCFF164ASR', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/55e555ad-73af-47b9-a82f-cb17fd71b1be/', 'derived_from': ['/files/ENCFF613WKG/']}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '3c3c804b17f026e58e7beb8e00602478', 'fastq_signature': [], '@id': '/files/ENCFF919CRG/', 'cloud_metadata': {'file_size': 559881, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/6dc1ad20-dc6f-42bf-99bf-84bcff0b1f87/ENCFF919CRG.bed.gz', 'md5sum_base64': 'PDyASxfwJuWOe+uOAGAkeA=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/6dc1ad20-dc6f-42bf-99bf-84bcff0b1f87/ENCFF919CRG.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/9a0ff3c5-f598-43c7-96bf-1ff59ef16e28/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:07:54.585967+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF919CRG', 'quality_metrics': ['/idr-quality-metrics/09ffd1cf-5a41-4578-9c38-7d5fa957504d/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF919CRG/@@download/ENCFF919CRG.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'conservative IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': '6dc1ad20-dc6f-42bf-99bf-84bcff0b1f87', 'content_md5sum': 'ff2f224b511d503f3f3643c0bb3a8ae7', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX400z5kGbPJ24xQjJyBv\"}', 'file_size': 559881, 'title': 'ENCFF919CRG', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF613WKG/', '/files/ENCFF419RSJ/', '/files/ENCFF159RSM/', '/files/ENCFF646KAN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_optimal.narrowPeak.gz', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '03bfee7ee344daa8663e80822480b3ad', 'fastq_signature': [], '@id': '/files/ENCFF824YMX/', 'cloud_metadata': {'file_size': 734290, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/b6a6b3b8-60d8-4254-aea6-1896ed07e5d1/ENCFF824YMX.bed.gz', 'md5sum_base64': 'A7/ufuNE2qhmPoCCJICzrQ=='}, 'output_category': 'annotation', 's3_uri': 's3://encode-public/2017/11/21/b6a6b3b8-60d8-4254-aea6-1896ed07e5d1/ENCFF824YMX.bed.gz', 'file_format_type': 'narrowPeak', 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-spp-peak-calling-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-step-v-1/', 'step_label': 'tf-idr-step', 'versions': ['/analysis-step-versions/tf-idr-step-v-1-0/'], 'major_version': 1, 'uuid': '0c3377fe-a83e-46d5-b548-d7e0669dde1a', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['IDR'], 'title': 'TF ChIP IDR step', 'aliases': ['encode:tf-idr-step-v-1'], 'name': 'tf-idr-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['optimal IDR thresholded peaks', 'conservative IDR thresholded peaks'], 'date_created': '2015-10-28T21:42:25.788823+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-step-v-1-0/', 'name': 'tf-idr-step-v-1-0', 'uuid': 'ad55c9a7-76f4-483d-9feb-ff47c067c523', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:48.974340+00:00'}, 'step_run': '/analysis-step-runs/9a0ff3c5-f598-43c7-96bf-1ff59ef16e28/', 'no_file_available': False, 'file_type': 'bed narrowPeak', 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'schema_version': '15', 'date_created': '2017-11-21T17:08:03.802008+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF824YMX', 'quality_metrics': ['/idr-quality-metrics/09ffd1cf-5a41-4578-9c38-7d5fa957504d/'], 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'href': '/files/ENCFF824YMX/@@download/ENCFF824YMX.bed.gz', 'dbxrefs': [], 'file_format': 'bed', 'output_type': 'optimal IDR thresholded peaks', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'b6a6b3b8-60d8-4254-aea6-1896ed07e5d1', 'content_md5sum': '9658977633344f45cea253faa14cd970', 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX480z5kBbXQj4x1fGZjf\"}', 'file_size': 734290, 'title': 'ENCFF824YMX', 'dataset': '/experiments/ENCSR163RYW/', '@type': ['File', 'Item'], 'derived_from': ['/files/ENCFF613WKG/', '/files/ENCFF419RSJ/', '/files/ENCFF159RSM/', '/files/ENCFF646KAN/'], 'superseded_by': []}, {'submitted_file_name': 'project-BpBZvb003694v3fj3QJGK3Zb:/peaks_TF_GRCh38/ENCSR163RYW/peaks/idr2/IDR_final_conservative.narrowPeak.bb', 'biological_replicates': [1, 2], 'alternate_accessions': [], 'flowcell_details': [], 'md5sum': '267587643028520e24d59d9fe5e54f0c', '@type': ['File', 'Item'], 'fastq_signature': [], '@id': '/files/ENCFF731GZP/', 'file_type': 'bigBed narrowPeak', 'cloud_metadata': {'file_size': 1087913, 'url': 'https://encode-public.s3.amazonaws.com/2017/11/21/e255f220-43d6-4b0d-b7e3-5f2b48fb922b/ENCFF731GZP.bigBed', 'md5sum_base64': 'JnWHZDAoUg4k1Z2f5eVPDA=='}, 's3_uri': 's3://encode-public/2017/11/21/e255f220-43d6-4b0d-b7e3-5f2b48fb922b/ENCFF731GZP.bigBed', 'file_size': 1087913, 'analysis_step_version': {'analysis_step': {'parents': ['/analysis-steps/tf-idr-step-v-1/'], 'current_version': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', '@type': ['AnalysisStep', 'Item'], 'status': 'released', 'documents': [], 'input_file_types': ['peaks'], '@id': '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/', 'step_label': 'tf-idr-peaks-to-bigbed-step', 'versions': ['/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/'], 'major_version': 1, 'uuid': 'c590043a-3bb1-403c-bc4b-67740aa8cb6b', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'analysis_step_types': ['file format conversion'], 'title': 'TF ChIP IDR peaks to bigBed file format conversion step', 'aliases': ['encode:tf-idr-peaks-to-bigbed-step-v-1'], 'name': 'tf-idr-peaks-to-bigbed-step-v-1', 'pipelines': [{'accession': 'ENCPL138KID', 'aliases': [], 'alternate_accessions': [], 'lab': '/labs/encode-processing-pipeline/', 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline', 'references': [], 'documents': ['/documents/7009beb8-340b-4e71-b9db-53bb020c7fe2/'], '@id': '/pipelines/ENCPL138KID/', 'assay_term_names': ['ChIP-seq'], 'description': '', 'analysis_steps': ['/analysis-steps/tf-spp-peak-calling-step-v-1/', '/analysis-steps/tf-macs2-signal-calling-step-v-1/', '/analysis-steps/tf-idr-step-v-1/', '/analysis-steps/tf-peaks-to-bigbed-step-v-1/', '/analysis-steps/tf-idr-peaks-to-bigbed-step-v-1/'], 'uuid': 'd7d8ed09-5caa-47bc-8548-430edcbdbcaa', 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'award': '/awards/ENCODE/', 'title': 'Transcription factor ChIP-seq', '@type': ['Pipeline', 'Item'], 'standards_page': '/chip-seq/transcription_factor/', 'status': 'released', 'schema_version': '10', 'date_created': '2015-10-28T21:43:04.192637+00:00'}], 'schema_version': '8', 'output_file_types': ['bigBed'], 'date_created': '2015-10-28T21:42:26.383107+00:00'}, 'status': 'released', '@type': ['AnalysisStepVersion', 'Item'], '@id': '/analysis-step-versions/tf-idr-peaks-to-bigbed-step-v-1-0/', 'name': 'tf-idr-peaks-to-bigbed-step-v-1-0', 'uuid': '30b7cd3c-55fd-42c2-8175-19bc98d58752', 'minor_version': 0, 'submitted_by': '/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/', 'software_versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/'], 'aliases': [], 'schema_version': '4', 'date_created': '2015-10-28T21:42:49.467762+00:00'}, 'href': '/files/ENCFF731GZP/@@download/ENCFF731GZP.bigBed', 'no_file_available': False, 'replicate_libraries': ['/libraries/ENCLB195KAR/', '/libraries/ENCLB762ZJL/'], 'assembly': 'GRCh38', 'aliases': [], 'award': '/awards/U41HG006992/', 'output_type': 'conservative IDR thresholded peaks', 'date_created': '2017-11-21T17:07:59.133456+00:00', 'file_format_specifications': ['/documents/948203bb-8eb2-42a2-8b12-1c10f356c998/'], 'accession': 'ENCFF731GZP', 'file_format_type': 'narrowPeak', 'lab': {'name': 'encode-processing-pipeline', 'phone1': '', 'phone2': '', 'state': '', 'status': 'current', '@id': '/labs/encode-processing-pipeline/', 'fax': '', 'pi': '/users/8b1f8780-b5d6-4fb7-a5a2-ddcec9054288/', 'institute_label': '', 'country': 'USA', 'institute_name': '', 'address1': '', 'uuid': 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d', 'postal_code': '', 'title': 'ENCODE Processing Pipeline', '@type': ['Lab', 'Item'], 'awards': ['/awards/U41HG007000/', '/awards/U41HG006992/'], 'schema_version': '5', 'city': ''}, 'dbxrefs': [], 'quality_metrics': ['/idr-quality-metrics/dbffd278-f5c4-4188-8df3-1258e74c7960/'], 'file_format': 'bigBed', 'schema_version': '15', 'status': 'released', 'technical_replicates': ['1_1', '2_1'], 'uuid': 'e255f220-43d6-4b0d-b7e3-5f2b48fb922b', 'superseded_by': [], 'submitted_by': {'uuid': '85978cd9-131e-48e2-a389-f752ab05b0a6', 'submits_for': ['/labs/j-michael-cherry/'], 'title': 'Kathrina Onate', '@type': ['User', 'Item'], '@id': '/users/85978cd9-131e-48e2-a389-f752ab05b0a6/', 'lab': '/labs/j-michael-cherry/'}, 'notes': '{\"dx-createdBy\": {\"job\": \"job-F88F6yj036978Y69G3PVZyV9\", \"executable\": \"applet-F7KPjy003694gb8k2gB5vjQq\", \"user\": \"user-mkajco\"}, \"qc\": {\"reproducibility_test\": \"pass\", \"rescue_ratio\": 1.3494106857783363, \"Np\": 39728, \"N1\": 36721, \"N2\": 22264, \"Nt\": 29441, \"self_consistency_ratio\": 1.6493442328422565}, \"dx-parentAnalysis\": \"analysis-F88F6yQ036978Y69G3PVZyV0\", \"dx-id\": \"file-F88bX3j0z5kBbXQj4x1fGZjQ\"}', 'output_category': 'annotation', 'title': 'ENCFF731GZP', 'dataset': '/experiments/ENCSR163RYW/', 'step_run': '/analysis-step-runs/fa6a52fa-ca11-4abe-94a5-a96c6b99358a/', 'derived_from': ['/files/ENCFF919CRG/']}], 'date_submitted': '2017-01-20', 'award': {'end_date': '2016-07-31', 'rfa': 'ENCODE3', '@type': ['Award', 'Item'], 'status': 'disabled', 'pi': {'uuid': '27e105ca-c741-4459-bf17-90e003508639', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Michael Snyder', '@type': ['User', 'Item'], '@id': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}}, 'start_date': '2012-09-21', '@id': '/awards/U54HG006996/', 'uuid': '4d462953-2da5-4fcf-a695-7206f2d5cf45', 'title': 'PRODUCTION CENTER FOR MAPPING REGULATORY REGIONS OF THE HUMAN GENOME', 'url': 'http://projectreporter.nih.gov/project_info_details.cfm?aid=8402423', 'name': 'U54HG006996', 'description': 'We propose to use our high-throughput production facility for genome-wide identification of the chromosomal locations of regulatory elements in human and mouse cells. Sequence-specific and general transcription factors will be mapped across the genome using chromatin immunoprecipitation followed by DNA sequencing (ChlP-Seq). Long noncoding RNAs (IncRNAs) will be mapped using ChlRP-seq, a method by which specific chromatin-bound RNAs and their associated regions are isolated using oligonucleotides followed by sequencing of the precipitated DNA. The results obtained from our genome wide mapping experiments will greatly advance the efforts of the ENCODE consortium to comprehensively identify all functional elements in the human genome. Our experimental data will be subjected to stringent quality metrics already established by the consortium and the results made publicly available in a timely manner. Targeted biological validation experiments will be performed, along with integrative analyses. All experiments and analyses will be performed by a highly experienced and well-integrated team of experts that has already established a successful working relationship. We expect our center to contribute greatly to the annotation of the human and mouse genomes.', 'schema_version': '7', 'project': 'ENCODE', 'viewing_group': 'ENCODE3'}, 'objective_slims': [], 'assay_title': 'TF ChIP-seq', 'date_created': '2016-12-14T21:11:45.870691+00:00', 'accession': 'ENCSR163RYW', 'month_released': 'November, 2017', 'lab': {'name': 'michael-snyder', 'phone1': '650-723-4668', 'phone2': '650-736-8099', 'state': 'CA', 'status': 'current', '@id': '/labs/michael-snyder/', 'fax': '650-725-1534', 'pi': '/users/27e105ca-c741-4459-bf17-90e003508639/', 'institute_label': 'Stanford', 'schema_version': '5', 'country': 'USA', 'institute_name': 'Stanford University', 'address1': 'Department of Genetics', 'uuid': '72a9c49a-4e17-4df1-96d1-fd19aa46b175', 'postal_code': '94305-5120', 'title': 'Michael Snyder, Stanford', '@type': ['Lab', 'Item'], 'awards': ['/awards/RC2HG005602/', '/awards/U54HG006996/', '/awards/U54HG004558/', '/awards/U01HG007919/', '/awards/UM1HG009442/'], 'address2': '300 Pasteur Drive; MC5120', 'city': 'Stanford'}, 'status': 'released', 'biosample_summary': 'HEK293 genetically modified using site-specific recombination originated from HEK293', 'submitter_comment': \"This experiment uses two slightly different protocols for each replicate. Replicate 1 uses 'BravoTruSeq protocol' whereas replicate 2 uses 'TruSeq' protocol.\", 'references': [], 'target': {'aliases': [], 'uuid': '33d4e470-7497-4c29-acc1-eba93f334948', '@type': ['Target', 'Item'], 'label': 'eGFP-ZNF189', 'status': 'released', '@id': '/targets/eGFP-ZNF189-human/', 'dbxref': ['ENSEMBL:ENSG00000136870', 'GeneID:7743', 'HGNC:12980', 'UniProtKB:O75820'], 'modifications': [{'position': 1, 'modification': 'eGFP'}], 'genes': [{'uuid': '3fdb268a-ef98-4c9c-b787-2d58e78da877', 'geneid': '7743', '@type': ['Gene', 'Item'], 'ncbi_entrez_status': 'live', 'dbxrefs': ['MIM:603132', 'HGNC:12980', 'ENSEMBL:ENSG00000136870', 'Vega:OTTHUMG00000020383', 'UniProtKB:O75820', 'UniProtKB:B7ZLK9', 'UniProtKB:A0A087X0K2'], '@id': '/genes/7743/', 'schema_version': '1', 'status': 'released', 'organism': '/organisms/human/', 'symbol': 'ZNF189', 'title': 'ZNF189 (Homo sapiens)', 'name': 'zinc finger protein 189', 'go_annotations': [{'go_name': 'nucleus', 'go_aspect': 'C', 'go_evidence_code': 'IEA', 'go_id': 'GO:0005634'}, {'go_name': 'negative regulation of transcription by RNA polymerase II', 'go_aspect': 'P', 'go_evidence_code': 'TAS', 'go_id': 'GO:0000122'}, {'go_name': 'DNA-binding transcription factor activity', 'go_aspect': 'F', 'go_evidence_code': 'TAS', 'go_id': 'GO:0003700'}, {'go_name': 'transcription, DNA-templated', 'go_aspect': 'P', 'go_evidence_code': 'IEA', 'go_id': 'GO:0006351'}, {'go_name': 'metal ion binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0046872'}, {'go_name': 'DNA-binding transcription factor activity, RNA polymerase II-specific', 'go_aspect': 'F', 'go_evidence_code': 'ISA', 'go_id': 'GO:0000981'}, {'go_name': 'DNA binding', 'go_aspect': 'F', 'go_evidence_code': 'IEA', 'go_id': 'GO:0003677'}], 'targets': ['/targets/ZNF189-human/', '/targets/eGFP-ZNF189-human/']}], 'title': 'eGFP-ZNF189 (Homo sapiens)', 'investigated_as': ['recombinant protein', 'transcription factor'], 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'name': 'eGFP-ZNF189-human', 'schema_version': '11'}, 'type_slims': [], 'related_files': [], 'submitted_by': {'uuid': '742bd56b-e511-411c-9410-4040c5776597', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Jessika Adrian', '@type': ['User', 'Item'], '@id': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'lab': '/labs/michael-snyder/'}, 'replicates': [{'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R1'], 'libraries': [{'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634423'], 'documents': [], '@id': '/libraries/ENCLB195KAR/', 'crosslinking_method': 'formaldehyde', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_L1'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2016-12-14T21:08:40.861500+00:00', 'accession': 'ENCLB195KAR', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': ['/documents/90e0e15b-ca37-4653-8c14-9748db04ae3f/'], '@id': '/biosamples/ENCBS111GOI/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_B1'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'accession': 'ENCBS111GOI', '@type': ['Biosample', 'Item'], 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986910'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', 'parent_of': [], 'sex': 'female', 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': 'e5b2389a-d247-4320-8f55-a17dce5fe03a', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '742bd56b-e511-411c-9410-4040c5776597', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Jessika Adrian', '@type': ['User', 'Item'], '@id': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], 'date_created': '2016-12-14T21:04:33.959774+00:00', 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'description': 'HEK293 cell line stably expressing N-terminal tagged eGFP-ZNF189 under the control of a CMV promoter.'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '9c03b3bd-4cce-438e-ab53-02d96e191ac8', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'antibodies': ['/antibodies/ENCAB976SGG/'], 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/'], '@type': ['Library', 'Item']}], 'status': 'released', 'experiment': '/experiments/ENCSR163RYW/', 'technical_replicate_number': 1, '@id': '/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/', 'library': {'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634423'], 'documents': [], '@id': '/libraries/ENCLB195KAR/', 'crosslinking_method': 'formaldehyde', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_L1'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2016-12-14T21:08:40.861500+00:00', 'accession': 'ENCLB195KAR', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': ['/documents/90e0e15b-ca37-4653-8c14-9748db04ae3f/'], '@id': '/biosamples/ENCBS111GOI/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_B1'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'accession': 'ENCBS111GOI', '@type': ['Biosample', 'Item'], 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986910'], 'biosample_ontology': {'classification': 'cell line', 'aliases': [], 'dbxrefs': ['Cellosaurus:CVCL_0045'], 'status': 'released', 'organ_slims': ['epithelium', 'kidney'], 'developmental_slims': ['mesoderm'], '@id': '/biosample-types/cell_line_EFO_0001182/', 'name': 'cell_line_EFO_0001182', 'synonyms': ['human embryonic kidney cell'], 'term_name': 'HEK293', 'references': [], 'system_slims': ['excretory system'], 'uuid': '014c9be0-20d4-4df5-b816-b0dca4cfee9d', 'cell_slims': ['epithelial cell'], '@type': ['BiosampleType', 'Item'], 'schema_version': '1', 'term_id': 'EFO:0001182'}, 'parent_of': [], 'sex': 'female', 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': 'e5b2389a-d247-4320-8f55-a17dce5fe03a', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '742bd56b-e511-411c-9410-4040c5776597', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/peggy-farnham/', '/labs/sherman-weissman/'], 'title': 'Jessika Adrian', '@type': ['User', 'Item'], '@id': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], 'date_created': '2016-12-14T21:04:33.959774+00:00', 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'description': 'HEK293 cell line stably expressing N-terminal tagged eGFP-ZNF189 under the control of a CMV promoter.'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '9c03b3bd-4cce-438e-ab53-02d96e191ac8', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'antibodies': ['/antibodies/ENCAB976SGG/'], 'replicates': ['/replicates/f26d90d9-53e2-4dff-8b98-c11d3fdb2831/'], '@type': ['Library', 'Item']}, 'uuid': 'f26d90d9-53e2-4dff-8b98-c11d3fdb2831', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'biological_replicate_number': 1, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': {'lot_reviews': [{'organisms': ['/organisms/human/'], 'targets': ['/targets/eGFP-avictoria/'], 'biosample_term_id': 'EFO:0001182', 'status': 'characterized to standards', 'biosample_term_name': 'HEK293', 'detail': 'Fully characterized.'}], 'antigen_description': '', 'alternate_accessions': [], 'host_organism': '/organisms/rabbit/', 'lot_id': 'GR158277-1', '@id': '/antibodies/ENCAB976SGG/', 'used_by_biosample_characterizations': [], 'aliases': [], 'award': '/awards/U54HG006996/', 'schema_version': '9', 'characterizations': ['/antibody-characterizations/85afbfd6-a683-4db8-b4c6-43231478c6c9/', '/antibody-characterizations/807d39c0-f743-4579-b637-e3a353638565/'], 'source': '/sources/abcam/', 'accession': 'ENCAB976SGG', '@type': ['AntibodyLot', 'Item'], 'purifications': [], 'lab': '/labs/michael-snyder/', 'status': 'released', 'clonality': 'polyclonal', 'lot_id_alias': [], 'dbxrefs': ['AR:AB_303395'], 'uuid': 'af972301-057b-4313-b6a2-39e69a0fed2c', 'submitted_by': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'product_id': 'ab290', 'title': 'ENCAB976SGG', 'url': 'http://www.abcam.com/gfp-antibody-chip-grade-ab290.html', 'date_created': '2016-03-29T16:57:02.862650+00:00', 'targets': ['/targets/eGFP-avictoria/', '/targets/YFP-avictoria/']}, 'date_created': '2016-12-14T21:13:52.766303+00:00'}, {'aliases': ['michael-snyder:HEK_eGFP-ZNF189_FRT_ChIP_R2'], 'libraries': [{'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634422'], 'documents': [], '@id': '/libraries/ENCLB762ZJL/', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:L-HEK_eGFP-ZNF189_R2'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2017-10-24T20:05:53.293466+00:00', 'accession': 'ENCLB762ZJL', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': [], '@id': '/biosamples/ENCBS082THU/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:B-HEK_eGFP_ZNF189_R2'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'date_created': '2017-10-24T19:23:45.015546+00:00', 'accession': 'ENCBS082THU', 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986901'], 'biosample_ontology': '/biosample-types/cell_line_EFO_0001182/', 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'parent_of': [], 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': '96db7fd4-54b3-4bd0-ab6a-1bd972aeb43e', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], '@type': ['Biosample', 'Item'], 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'sex': 'female'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '99423177-fbc8-4bf6-8241-f6b25855eb2b', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'antibodies': ['/antibodies/ENCAB615WUN/'], 'replicates': ['/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], '@type': ['Library', 'Item']}], 'status': 'released', 'experiment': '/experiments/ENCSR163RYW/', 'technical_replicate_number': 1, '@id': '/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/', 'library': {'barcode_details': [], 'alternate_accessions': [], 'dbxrefs': ['GEO:GSM3634422'], 'documents': [], '@id': '/libraries/ENCLB762ZJL/', 'treatments': [], 'size_range': '450-650', 'aliases': ['michael-snyder:L-HEK_eGFP-ZNF189_R2'], 'nucleic_acid_term_name': 'DNA', 'schema_version': '9', 'date_created': '2017-10-24T20:05:53.293466+00:00', 'accession': 'ENCLB762ZJL', 'spikeins_used': [], 'strand_specificity': False, 'lab': '/labs/michael-snyder/', 'biosample': {'genetic_modifications': ['/genetic-modifications/ENCGM615AHN/'], 'status': 'released', 'alternate_accessions': [], 'age': 'unknown', 'documents': [], '@id': '/biosamples/ENCBS082THU/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'treatments': [], 'aliases': ['michael-snyder:B-HEK_eGFP_ZNF189_R2'], 'award': '/awards/U54HG006996/', 'schema_version': '24', 'applied_modifications': [{'biosamples_modified': ['/biosamples/ENCBS111GOI/', '/biosamples/ENCBS082THU/'], 'alternate_accessions': [], 'documents': ['/documents/672a2de5-5bfe-41a9-ac2d-e7e43c697c51/'], 'modified_site_by_target_id': '/targets/eGFP-ZNF189-human/', '@id': '/genetic-modifications/ENCGM615AHN/', 'perturbation': False, 'introduced_tags': [{'location': 'N-terminal', 'name': 'eGFP'}], 'treatments': ['/treatments/33bde07b-1162-4dbd-a867-333a74bcab40/'], 'method': 'site-specific recombination', 'aliases': ['michael-snyder:eGFP_ZNF189_FRT_construct'], 'award': '/awards/U54HG006996/', 'schema_version': '8', 'characterizations': [], 'date_created': '2017-09-22T02:13:56.689948+00:00', 'accession': 'ENCGM615AHN', 'lab': '/labs/michael-snyder/', 'donors_modified': [], 'status': 'released', 'category': 'insertion', 'reagents': [{'url': 'http://horfdb.dfci.harvard.edu/hv7/index.php?page=getresults&by=detail&qury=54338', 'identifier': 'human-orfeome:54338', 'source': '/sources/human-orfeome/'}], 'modified_site_nonspecific': 'random', 'purpose': 'tagging', 'uuid': 'e1020b38-2913-4b41-b759-8a1aca5cb305', 'submitted_by': '/users/742bd56b-e511-411c-9410-4040c5776597/', 'notes': 'The pcDNA5/FRT vector is from Thermo Fisher, product id = V601020 (https://www.thermofisher.com/order/catalog/product/V601020).', '@type': ['GeneticModification', 'Item'], 'description': \"This construct contains an eGFP N-terminal tag to ZNF189 under control of a CMV promoter, introduced to expression vector pcDNA5/FRT as part of Thermo Fisher's Flp-In system. It is stably integrated at the introduced FLP recombinase target (FRT) site in the host genome.\"}], 'date_created': '2017-10-24T19:23:45.015546+00:00', 'accession': 'ENCBS082THU', 'lab': '/labs/michael-snyder/', 'dbxrefs': ['GEO:SAMN10986901'], 'biosample_ontology': {'classification': 'cell line', 'aliases': [], 'dbxrefs': ['Cellosaurus:CVCL_0045'], 'status': 'released', 'organ_slims': ['epithelium', 'kidney'], 'developmental_slims': ['mesoderm'], '@id': '/biosample-types/cell_line_EFO_0001182/', 'name': 'cell_line_EFO_0001182', 'synonyms': ['human embryonic kidney cell'], 'term_name': 'HEK293', 'references': [], 'system_slims': ['excretory system'], 'uuid': '014c9be0-20d4-4df5-b816-b0dca4cfee9d', 'cell_slims': ['epithelial cell'], '@type': ['BiosampleType', 'Item'], 'schema_version': '1', 'term_id': 'EFO:0001182'}, 'source': {'aliases': [], 'name': 'timothy-hughes', 'uuid': '5d81682e-ff53-48c6-bdc3-a6c595c3de9e', '@id': '/sources/timothy-hughes/', 'status': 'released', 'title': 'Timothy Hughes', 'url': 'http://hugheslab.med.utoronto.ca/', '@type': ['Source', 'Item'], 'description': 'Timothy Hughes at University of Toronto', 'schema_version': '6'}, 'parent_of': [], 'summary': 'Homo sapiens HEK293 cell line genetically modified (insertion) using site-specific recombination targeting eGFP-ZNF189 originated from HEK293', 'references': [], 'uuid': '96db7fd4-54b3-4bd0-ab6a-1bd972aeb43e', 'life_stage': 'embryonic', 'submitted_by': {'uuid': '51d0b8cb-6bb4-46a7-a167-8db20f23e562', 'submits_for': ['/labs/michael-snyder/', '/labs/kevin-white/', '/labs/sherman-weissman/', '/labs/peggy-farnham/'], 'title': 'Nathaniel Watson', '@type': ['User', 'Item'], '@id': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'lab': '/labs/michael-snyder/'}, 'internal_tags': [], 'characterizations': [], '@type': ['Biosample', 'Item'], 'originated_from': '/biosamples/ENCBS749QJL/', 'donor': {'alternate_accessions': [], 'siblings': [], 'age': 'unknown', 'documents': [], '@id': '/human-donors/ENCDO000ABF/', 'health_status': 'unknown', 'organism': {'name': 'human', 'uuid': '7745b647-ff15-4ff3-9ced-b897d4e2983c', 'status': 'released', 'scientific_name': 'Homo sapiens', '@type': ['Organism', 'Item'], '@id': '/organisms/human/', 'schema_version': '6', 'taxon_id': '9606'}, 'references': ['/publications/b42243ab-26d8-4ff6-b801-d368df994167/'], 'aliases': ['encode:donor of HEK293', 'encode:donor of HEK293T'], 'award': '/awards/U41HG006992/', 'schema_version': '11', 'characterizations': [], 'date_created': '2013-12-06T21:06:24.449752+00:00', 'parents': [], 'accession': 'ENCDO000ABF', 'children': [], 'lab': '/labs/encode-consortium/', 'external_ids': ['GEO:SAMN06122596'], 'status': 'released', 'sex': 'female', 'dbxrefs': [], 'uuid': '28fbb880-be14-44c9-9104-c2a93b59a249', 'life_stage': 'embryonic', 'submitted_by': '/users/3787a0ac-f13a-40fc-a524-69628b04cd59/', 'notes': 'Donor of HEK293.  Embryonic kidney, cells contain Adenovirus 5 DNA (PMID: 11967234); hypotriploid, XXX', '@type': ['HumanDonor', 'Donor', 'Item']}, 'sex': 'female'}, 'status': 'released', 'award': '/awards/U54HG006996/', 'nucleic_acid_term_id': 'SO:0000352', 'uuid': '99423177-fbc8-4bf6-8241-f6b25855eb2b', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'antibodies': ['/antibodies/ENCAB615WUN/'], 'replicates': ['/replicates/f5e07b4b-2a7e-4c95-b919-fa54792315a0/'], '@type': ['Library', 'Item']}, 'uuid': 'f5e07b4b-2a7e-4c95-b919-fa54792315a0', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'biological_replicate_number': 2, '@type': ['Replicate', 'Item'], 'schema_version': '9', 'antibody': {'status': 'released', 'lot_reviews': [{'organisms': ['/organisms/human/'], 'targets': ['/targets/eGFP-avictoria/'], 'biosample_term_id': 'EFO:0001182', 'status': 'characterized to standards', 'biosample_term_name': 'HEK293', 'detail': 'Fully characterized.'}], 'antigen_description': 'Recombinant full length protein corresponding to GFP', 'alternate_accessions': [], 'host_organism': '/organisms/rabbit/', 'lot_id': 'GR194813-1', '@id': '/antibodies/ENCAB615WUN/', 'used_by_biosample_characterizations': [], 'isotype': 'IgG', 'aliases': ['michael-snyder:AS-1667'], 'award': '/awards/U54HG006996/', 'schema_version': '9', 'characterizations': ['/antibody-characterizations/77d85630-ebee-4af7-a14e-015b9e177f3b/', '/antibody-characterizations/4594a491-584d-45ce-a1da-eb81a4d6c1e8/'], 'source': '/sources/abcam/', 'accession': 'ENCAB615WUN', '@type': ['AntibodyLot', 'Item'], 'purifications': ['antiserum'], 'lab': '/labs/michael-snyder/', 'clonality': 'polyclonal', 'lot_id_alias': [], 'dbxrefs': ['AR:AB_303395'], 'uuid': 'f55443ec-8dce-4373-98ef-28c3302b5d44', 'submitted_by': '/users/51d0b8cb-6bb4-46a7-a167-8db20f23e562/', 'product_id': 'ab-290 (Lot GR194813-1)', 'title': 'ENCAB615WUN', 'url': 'http://www.abcam.com/gfp-antibody-chip-grade-ab290.html', 'date_created': '2017-02-02T00:33:30.696380+00:00', 'targets': ['/targets/eGFP-avictoria/']}, 'date_created': '2017-10-24T20:14:44.935001+00:00'}], 'contributing_files': ['/files/ENCFF182WHU/', '/files/ENCFF810LLR/', '/files/ENCFF807MUK/', '/files/ENCFF419RSJ/', '/files/ENCFF001TDO/', '/files/ENCFF643CGH/']}]}, {'prefix': 'gtex', 'mirId': 'MIR:00000663', 'name': 'GTEx', 'pattern': '^\\\\w.+$', 'description': 'The Genotype-Tissue Expression (GTEx) project aims to provide to the scientific community a resource with which to study human gene expression and regulation and its relationship to genetic variation.', 'created': '2019-06-11T14:18:14.830+0000', 'modified': '2019-06-11T14:18:14.830+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BRIP1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1839'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1839'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1839/contactPerson'}}, 'metadata': []}, {'prefix': 'ricenetdb.gene', 'mirId': 'MIR:00000664', 'name': 'RiceNetDB Gene', 'pattern': '^LOC\\\\_Os\\\\d{1,2}g\\\\d{5}\\\\.\\\\d$', 'description': 'RiceNetDB is currently the most comprehensive regulatory database on Oryza Sativa based on genome annotation. It was displayed in three levels: GEM, PPIs and GRNs to facilitate biomolecular regulatory analysis and gene-metabolite mapping.', 'created': '2019-06-11T14:18:15.139+0000', 'modified': '2019-06-11T14:18:15.139+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LOC_Os01g49190.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1842'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1842'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1842/contactPerson'}}, 'metadata': []}, {'prefix': 'ricenetdb.protein', 'mirId': 'MIR:00000665', 'name': 'RiceNetDB Protein', 'pattern': '^LOC\\\\_Os\\\\d{1,2}g\\\\d{5}$', 'description': 'RiceNetDB is currently the most comprehensive regulatory database on Oryza Sativa based on genome annotation. It was displayed in three levels: GEM, PPIs and GRNs to facilitate biomolecular regulatory analysis and gene-metabolite mapping.', 'created': '2019-06-11T14:18:15.429+0000', 'modified': '2019-06-11T14:18:15.429+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'LOC_Os01g49190', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1845'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1845'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1845/contactPerson'}}, 'metadata': []}, {'prefix': 'ricenetdb.compound', 'mirId': 'MIR:00000666', 'name': 'RiceNetDB Compound', 'pattern': '^OSC\\\\d{4}$', 'description': 'RiceNetDB is currently the most comprehensive regulatory database on Oryza Sativa based on genome annotation. It was displayed in three levels: GEM, PPIs and GRNs to facilitate biomolecular regulatory analysis and gene-metabolite mapping.', 'created': '2019-06-11T14:18:15.639+0000', 'modified': '2019-06-11T14:18:15.639+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'OSC1416', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1847'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1847'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1847/contactPerson'}}, 'metadata': []}, {'prefix': 'ricenetdb.reaction', 'mirId': 'MIR:00000667', 'name': 'RiceNetDB Reaction', 'pattern': '^OSR\\\\d{4}$', 'description': 'RiceNetDB is currently the most comprehensive regulatory database on Oryza Sativa based on genome annotation. It was displayed in three levels: GEM, PPIs and GRNs to facilitate biomolecular regulatory analysis and gene-metabolite mapping.', 'created': '2019-06-11T14:18:15.856+0000', 'modified': '2019-06-11T14:18:15.856+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'OSR0818', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1849'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1849'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1849/contactPerson'}}, 'metadata': []}, {'prefix': 'ricenetdb.mirna', 'mirId': 'MIR:00000668', 'name': 'RiceNetDB miRNA', 'pattern': '^osa-miR\\\\d{3,5}[a-z]{0,1}$', 'description': 'RiceNetDB is currently the most comprehensive regulatory database on Oryza Sativa based on genome annotation. It was displayed in three levels: GEM, PPIs and GRNs to facilitate biomolecular regulatory analysis and gene-metabolite mapping.', 'created': '2019-06-11T14:18:16.076+0000', 'modified': '2019-06-11T14:18:16.076+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'osa-miR446', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1851'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1851'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1851/contactPerson'}}, 'metadata': []}, {'prefix': 'oclc', 'mirId': 'MIR:00000669', 'name': 'Online Computer Library Center (OCLC) WorldCat', 'pattern': '^[0-9]+$', 'description': \"The global library cooperative OCLC maintains WorldCat. WorldCat is the world's largest network of library content and services. WorldCat libraries are dedicated to providing access to their resources on the Web, where most people start their search for information.\", 'created': '2019-06-11T14:18:16.344+0000', 'modified': '2019-06-11T14:18:16.344+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '634515043', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1853'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1853'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1853/contactPerson'}}, 'metadata': []}, {'prefix': 'asin', 'mirId': 'MIR:00000670', 'name': 'Amazon Standard Identification Number (ASIN)', 'pattern': '^[0-9]{10}$', 'description': 'Almost every product on our site has its own ASIN, a unique code we use to identify it. For books, the ASIN is the same as the ISBN number, but for all other products a new ASIN is created when the item is uploaded to our catalogue.', 'created': '2019-06-11T14:18:16.611+0000', 'modified': '2019-06-11T14:18:16.611+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0471491039', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1856'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1856'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1856/contactPerson'}}, 'metadata': []}, {'prefix': 'ga4ghdos', 'mirId': 'MIR:00000671', 'name': 'Data Object Service', 'pattern': '^[a-zA-Z0-9\\\\-:#/\\\\.]+$', 'description': 'Assists in resolving data across cloud resources.', 'created': '2019-06-11T14:18:16.879+0000', 'modified': '2019-06-11T14:18:16.879+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'dg.4503/01b048d0-e128-4cb0-94e9-b2d2cab7563d', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1859'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1859'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1859/contactPerson'}}, 'metadata': []}, {'prefix': 'flowrepository', 'mirId': 'MIR:00000672', 'name': 'FlowRepository', 'pattern': '^FR\\\\-FCM\\\\-\\\\w{4}$', 'description': 'FlowRepository is a database of flow cytometry experiments where you can query and download data collected and annotated according to the MIFlowCyt standard. It is primarily used as a data deposition place for experimental findings published in peer-reviewed journals in the flow cytometry field.', 'created': '2019-06-11T14:18:17.129+0000', 'modified': '2019-06-11T14:18:17.129+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'FR-FCM-ZYGW', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1861'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1861'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1861/contactPerson'}}, 'metadata': []}, {'prefix': 'oid', 'mirId': 'MIR:00000673', 'name': 'OID Repository', 'pattern': '^[\\\\d.]+$', 'description': 'OIDs provide a persistent identification of objects based on a hierarchical structure of Registration Authorities (RA), where each parent has an object identifier and allocates object identifiers to child nodes.', 'created': '2019-06-11T14:18:17.389+0000', 'modified': '2019-06-11T14:18:17.389+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2.16.840', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1864'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1864'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1864/contactPerson'}}, 'metadata': []}, {'prefix': 'hpscreg', 'mirId': 'MIR:00000674', 'name': 'Human Pluripotent Stem Cell Registry', 'pattern': '^[A-Z]{2,6}(e|i)[A-Za-z0-9]{3}-[A-Z]{1,2}(-[A-Za-z0-9]{1,2})?$', 'description': 'hPSCreg is a freely accessible global registry for human pluripotent stem cell lines (hPSC-lines).', 'created': '2019-06-11T14:18:17.630+0000', 'modified': '2019-06-11T14:18:17.630+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'BCRTi001-A', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1867'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1867'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1867/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'url': 'http://hpscreg.eu/', 'potentialAction': {'@type': 'SearchAction', 'target': 'http://hpscreg.eu/search?q={search_term_string}', 'query-input': 'required name=search_term_string'}}]}, {'prefix': 'addgene', 'mirId': 'MIR:00000675', 'name': 'Addgene Plasmid Repository', 'pattern': '^[0-9]{5}(-[a-zA-Z0-9-]{0,7})?$|^[0-9]{10}$', 'description': 'Addgene is a non-profit plasmid repository. Addgene facilitates the exchange of genetic material between laboratories by offering plasmids and their associated cloning data to not-for-profit laboratories around the world.', 'created': '2019-06-11T14:18:17.878+0000', 'modified': '2019-06-11T14:18:17.878+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '50943', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1870'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1870'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1870/contactPerson'}}, 'metadata': []}, {'prefix': 'bacdive', 'mirId': 'MIR:00000676', 'name': 'Bacterial Diversity Metadatabase', 'pattern': '^[0-9]+$', 'description': 'BacDive‚Äîthe Bacterial Diversity Metadatabase merges detailed strain-linked information on the different aspects of bacterial and archaeal biodiversity.', 'created': '2019-06-11T14:18:18.125+0000', 'modified': '2019-06-11T14:18:18.125+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '131392', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1873'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1873'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1873/contactPerson'}}, 'metadata': []}, {'prefix': 'ido', 'mirId': 'MIR:00000677', 'name': 'Infectious Disease Ontology', 'pattern': '^[0-9]+$', 'description': 'Infectious Disease Ontology holds entities relevant to both biomedical and clinical aspects of most infectious diseases.', 'created': '2019-06-11T14:18:18.372+0000', 'modified': '2019-06-11T14:18:18.372+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '0000504', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1876'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1876'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1876/contactPerson'}}, 'metadata': []}, {'prefix': 'gdsc', 'mirId': 'MIR:00000678', 'name': 'Genomics of Drug Sensitivity in Cancer', 'pattern': '^[0-9]+$', 'description': 'The Genomics of Drug Sensitivity in Cancer (GDSC) database is designed to facilitate an increased understanding of the molecular features that influence drug response in cancer cells and which will enable the design of improved cancer therapies.', 'created': '2019-06-11T14:18:18.593+0000', 'modified': '2019-06-11T14:18:18.593+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '1242', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1878'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1878'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1878/contactPerson'}}, 'metadata': []}, {'prefix': 'isni', 'mirId': 'MIR:00000684', 'name': 'International Standard Name Identifier', 'pattern': '^[0-9]{15}[0-9X]{1}$', 'description': 'ISNI is the ISO certified global standard number for identifying the millions of contributors to creative works and those active in their distribution, including researchers, inventors, writers, artists, visual creators, performers, producers, publishers, aggregators, and more. It is part of a family of international standard identifiers that includes identifiers of works, recordings, products and right holders in all repertoires, e.g. DOI, ISAN, ISBN, ISRC, ISSN, ISTC, and ISWC.\\r\\n\\r\\nThe mission of the ISNI International Authority (ISNI-IA) is to assign to the public name(s) of a researcher, inventor, writer, artist, performer, publisher, etc. a persistent unique identifying number in order to resolve the problem of name ambiguity in search and discovery; and diffuse each assigned ISNI across all repertoires in the global supply chain so that every published work can be unambiguously attributed to its creator wherever that work is described.', 'created': '2019-06-11T14:18:20.121+0000', 'modified': '2019-06-11T14:18:20.121+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '000000012281955X', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1895'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1895'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1895/contactPerson'}}, 'metadata': []}, {'prefix': 'viaf', 'mirId': 'MIR:00000685', 'name': 'Virtual International Authority File', 'pattern': '^\\\\d+$', 'description': 'The VIAF¬Æ (Virtual International Authority File) combines multiple name authority files into a single OCLC-hosted name authority service. The goal of the service is to lower the cost and increase the utility of library authority files by matching and linking widely-used authority files and making that information available on the Web.', 'created': '2019-06-11T14:18:20.364+0000', 'modified': '2019-06-11T14:18:20.364+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '75121530', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1898'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1898'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1898/contactPerson'}}, 'metadata': []}, {'prefix': 'lei', 'mirId': 'MIR:00000686', 'name': 'Global LEI Index', 'pattern': '^[0-9A-Z]{4}[0-9A-Z]{14}[0-9A-Z]{2}$', 'description': 'Established by the Financial Stability Board in June 2014, the Global Legal Entity Identifier Foundation (GLEIF) is tasked to support the implementation and use of the Legal Entity Identifier (LEI). The foundation is backed and overseen by the LEI Regulatory Oversight Committee, representing public authorities from around the globe that have come together to jointly drive forward transparency within the global financial markets. GLEIF is a supra-national not-for-profit organization headquartered in Basel, Switzerland.', 'created': '2019-06-11T14:18:20.599+0000', 'modified': '2019-06-11T14:18:20.599+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'HWUPKR0MPOU8FGXBT394', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1901'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1901'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1901/contactPerson'}}, 'metadata': [{'@context': 'http://schema.org', '@type': 'WebSite', 'name': 'Global Legal Entity Identifier Foundation', 'alternateName': 'GLEIF', 'url': 'http://gleif.org', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.gleif.org/en/lei/search#query={search_term_string}', 'query-input': 'required name=search_term_string', 'description': 'Search LEI Records'}}, {'@context': 'http://schema.org', '@type': 'WebSite', 'name': 'Global Legal Entity Identifier Foundation', 'alternateName': 'GLEIF', 'url': 'http://gleif.org', 'potentialAction': {'@type': 'SearchAction', 'target': 'https://www.gleif.org/en/lei/search#query={search_term_string}', 'query-input': 'required name=search_term_string', 'description': 'Search LEI Records'}}]}, {'prefix': 'spdx', 'mirId': 'MIR:00000687', 'name': 'SPDX License List', 'pattern': '^[0-9A-Za-z\\\\-.]+$', 'description': 'The SPDX License List is a list of commonly found licenses and exceptions used in free and open source and other collaborative software or documentation. The purpose of the SPDX License List is to enable easy and efficient identification of such licenses and exceptions in an SPDX document, in source files or elsewhere. The SPDX License List includes a standardized short identifier, full name, vetted license text including matching guidelines markup as appropriate, and a canonical permanent URL for each license and exception.', 'created': '2019-06-11T14:18:20.855+0000', 'modified': '2019-06-11T14:18:20.855+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'CC-BY-1.0', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1904'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1904'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1904/contactPerson'}}, 'metadata': []}, {'prefix': 'sabiork.compound', 'mirId': 'MIR:00000688', 'name': 'SABIO-RK Compound', 'pattern': '^\\\\d+$', 'description': 'SABIO-RK is a relational database system that contains information about biochemical reactions, their kinetic equations with their parameters, and the experimental conditions under which these parameters were measured. The compound data set provides information regarding the reactions in which a compound participates as substrate, product or modifier (e.g. inhibitor, cofactor), and links to further information.', 'created': '2019-06-11T14:18:21.119+0000', 'modified': '2019-06-11T14:18:21.119+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '75', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1907'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1907'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1907/contactPerson'}}, 'metadata': []}, {'prefix': 'oryzabase.reference', 'mirId': 'MIR:00000689', 'name': 'Oryzabase', 'pattern': '^\\\\d+$', 'description': \"The Oryzabase is a comprehensive rice science database established in 2000 by rice researcher's committee in Japan.\", 'created': '2019-06-11T14:18:21.361+0000', 'modified': '2019-06-11T14:18:21.361+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '42840', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1909'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1909'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1909/contactPerson'}}, 'metadata': []}, {'prefix': 'peptideatlas.dataset', 'mirId': 'MIR:00000690', 'name': 'PeptideAtlas Dataset', 'pattern': '^PASS\\\\d{5}$', 'description': 'Experiment details about PeptideAtlas entries. Each PASS entry provides direct access to the data files submitted to PeptideAtlas.', 'created': '2019-06-11T14:18:21.571+0000', 'modified': '2019-06-11T14:18:21.571+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'PASS01237', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1911'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1911'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1911/contactPerson'}}, 'metadata': []}, {'prefix': 'did', 'mirId': 'MIR:00000691', 'name': 'Decentralized Identifiers (DIDs)', 'pattern': '^did:[a-z0-9]+:[A-Za-z0-9.\\\\-:]+$', 'description': 'DIDs are an effort by the W3C Credentials Community Group and the wider Internet identity community to define identifiers that can be registered, updated, resolved, and revoked without any dependency on a central authority or intermediary.', 'created': '2019-06-11T14:18:21.755+0000', 'modified': '2019-06-11T14:18:21.755+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'sov:WRfXPg8dantKVubE3HX8pw', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1913'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1913'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1913/contactPerson'}}, 'metadata': []}, {'prefix': 'seed.reaction', 'mirId': 'MIR:00000692', 'name': 'SEED Reactions', 'pattern': '^rxn\\\\d+$', 'description': 'ModelSEED is a platform for creating genome-scale metabolic network reconstructions for microbes and plants. As part of the platform, a biochemistry database is managed that contains reactions unique to ModelSEED as well as reactions aggregated from other databases or from manually-curated genome-scale metabolic network reconstructions.', 'created': '2019-06-11T14:18:21.981+0000', 'modified': '2019-06-11T14:18:21.981+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'rxn00001', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1916'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1916'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1916/contactPerson'}}, 'metadata': []}, {'prefix': 'occ', 'mirId': 'MIR:00000693', 'name': 'OpenCitations Corpus', 'pattern': '^[a-z][a-z]/[0-9]+$', 'description': 'The OpenCitations Corpus is open repository of scholarly citation data made available under a Creative Commons public domain dedication (CC0), which provides accurate bibliographic references harvested from the scholarly literature that others may freely build upon, enhance and reuse for any purpose, without restriction under copyright or database law.', 'created': '2019-06-11T14:18:22.186+0000', 'modified': '2019-06-11T14:18:22.186+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'br/1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1918'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1918'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1918/contactPerson'}}, 'metadata': []}, {'prefix': 'biostudies', 'mirId': 'MIR:00000694', 'name': 'BioStudies database', 'pattern': '^S-[A-Z]{4}[A-Z\\\\d\\\\-]+$', 'description': 'The BioStudies database holds descriptions of biological studies, links to data from these studies in other databases at EMBL-EBI or outside, as well as data that do not fit in the structured archives at EMBL-EBI. The database can accept a wide range of types of studies described via a simple format. It also enables manuscript authors to submit supplementary information and link to it from the publication.', 'created': '2019-06-11T14:18:22.448+0000', 'modified': '2019-06-11T14:18:22.448+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'S-EPMC6266652', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1921'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1921'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1921/contactPerson'}}, 'metadata': []}, {'prefix': 'cadsr', 'mirId': 'MIR:00000695', 'name': 'Cancer Data Standards Registry and Repository', 'pattern': '^[0-9]*$', 'description': 'The US National Cancer Institute (NCI) maintains and administers data elements, forms, models, and components of these items in a metadata registry referred to as the Cancer Data Standards Registry and Repository, or caDSR.', 'created': '2019-06-11T14:18:22.646+0000', 'modified': '2019-06-11T14:18:22.646+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '3771992', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1923'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1923'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1923/contactPerson'}}, 'metadata': []}, {'prefix': 'gcst', 'mirId': 'MIR:00000715', 'name': 'GWAS Catalog', 'pattern': '^GCST\\\\d{6}$', 'description': 'The GWAS Catalog provides a consistent, searchable, visualisable and freely available database of published SNP-trait associations, which can be easily integrated with other resources, and is accessed by scientists, clinicians and other users worldwide.', 'created': '2019-06-17T12:55:17.868+0000', 'modified': '2019-06-17T12:55:17.868+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GCST000035', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1934'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1934'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1934/contactPerson'}}, 'metadata': []}, {'prefix': 'erm', 'mirId': 'MIR:00000763', 'name': 'European Registry of Materials', 'pattern': '^ERM[0-9]{8}$', 'description': 'The European Registry of Materials is a simple registry with the sole purpose to mint material identifiers to be used by research projects throughout the life cycle of their project.', 'created': '2019-06-19T15:38:51.382+0000', 'modified': '2019-06-19T15:38:51.382+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'ERM00000044', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1942'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1942'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1942/contactPerson'}}, 'metadata': []}, {'prefix': 'classyfire', 'mirId': 'MIR:00000770', 'name': 'ClassyFire', 'pattern': '^C[0-9]{7}$', 'description': 'ClassyFire is a web-based application for automated structural classification of chemical entities. This application uses a rule-based approach that relies on a comprehensible, comprehensive, and computable chemical taxonomy. ClassyFire provides a hierarchical chemical classification of chemical entities (mostly small molecules and short peptide sequences), as well as a structure-based textual description, based on a chemical taxonomy named ChemOnt, which covers 4825 chemical classes of organic and inorganic compounds. Moreover, ClassyFire allows for text-based search via its web interface. It can be accessed via the web interface or via the ClassyFire API.', 'created': '2019-09-05T11:46:36.162+0000', 'modified': '2019-09-05T11:46:36.162+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'C0004828', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1977'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1977'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1977/contactPerson'}}, 'metadata': []}, {'prefix': 'ocid', 'mirId': 'MIR:00000738', 'name': 'Ontology Concept Identifiers', 'pattern': 'ocid:[0-9]{12}', 'description': '\\'ocid\\' stands for \"Ontology Concept Identifiers\" and are 12 digit long integers covering IDs in topical ontologies from anatomy up to toxicology.', 'created': '2019-09-10T09:27:59.693+0000', 'modified': '2019-09-10T09:27:59.693+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '190000021540', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1989'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1989'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1989/contactPerson'}}, 'metadata': []}, {'prefix': 'mmp.fun', 'mirId': 'MIR:00000729', 'name': 'MarFun', 'pattern': '^MMP\\\\d+.\\\\d+$', 'description': 'MarFun is manually curated database for marine fungi which is a part of the MAR databases.', 'created': '2019-09-10T10:50:49.547+0000', 'modified': '2019-09-10T10:50:49.547+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'MMP3888430', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1997'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1997'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1997/contactPerson'}}, 'metadata': []}, {'prefix': 'ecyano.experiment', 'mirId': 'MIR:00000760', 'name': 'E-cyanobacterium Experimental Data', 'pattern': '^\\\\d+$', 'description': 'E-cyanobacterium experiments is a repository of wet-lab experiments related to cyanobacteria. The emphasis is placed on annotation via mapping to local database of biological knowledge and mathematical models along with the complete experimental setup supporting the reproducibility of the experiments.', 'created': '2019-09-10T12:51:24.637+0000', 'modified': '2019-09-10T12:51:24.637+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '18', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2003'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2003'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2003/contactPerson'}}, 'metadata': []}, {'prefix': 'dlxc', 'mirId': 'MIR:00000742', 'name': 'Circular double stranded DNA sequences composed', 'pattern': '^[A-Z0-9]{6,7}$', 'description': 'DOULIX lab-tested standard biological parts, in this case, full length constructs.', 'created': '2019-09-24T11:03:08.927+0000', 'modified': '2019-09-24T11:03:08.927+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'M77F7JM', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2022'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2022'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2022/contactPerson'}}, 'metadata': []}, {'prefix': 'dlxb', 'mirId': 'MIR:00000747', 'name': 'Linear double stranded DNA sequences', 'pattern': '^[A-Z0-9]{6,7}$', 'description': 'DOULIX lab-tested standard biological parts, in this case linear double stranded DNA sequences.', 'created': '2019-10-01T11:40:36.666+0000', 'modified': '2019-10-01T11:40:36.666+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '6VDC956', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2026'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2026'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2026/contactPerson'}}, 'metadata': []}, {'prefix': 'cameo', 'mirId': 'MIR:00000623', 'name': 'CAMEO', 'pattern': '^[0-9\\\\-_]+$', 'description': 'The goal of the CAMEO (Continuous Automated Model EvaluatiOn) community project is to continuously evaluate the accuracy and reliability of protein structure prediction servers, offering scores on  tertiary and quaternary structure prediction, model quality estimation, accessible surface area prediction, ligand binding site residue prediction and contact prediction services in a fully automated manner. These predictions are regularly compared against reference structures from PDB.', 'created': '2019-06-11T14:18:04.976+0000', 'modified': '2019-10-01T11:46:06.302+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '2019-08-03_00000089_1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1729'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1729'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1729/contactPerson'}}, 'metadata': []}, {'prefix': 'nemo', 'mirId': 'MIR:00000744', 'name': 'Neuroscience Multi-Omic BRAIN Initiative Data', 'pattern': '[a-z]{3}-[a-km-z0-9]{7}', 'description': 'This namespace is about Neuroscience Multi-Omic data, specially focused on that data generated from the BRAIN Initiative and related brain research projects.', 'created': '2019-09-19T13:46:35.999+0000', 'modified': '2019-10-02T08:18:15.555+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'smp-m3w9hbe', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2013'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2013'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2013/contactPerson'}}, 'metadata': []}, {'prefix': 'insdc.gca', 'mirId': 'MIR:00000607', 'name': 'Genome assembly database', 'pattern': '^GC[AF]_[0-9]{9}\\\\.[0-9]$', 'description': 'The genome assembly database contains detailed information about genome assemblies for eukaryota, bacteria and archaea. The scope of the genome collections database does not extend to viruses, viroids and bacteriophage.', 'created': '2019-06-11T14:18:01.236+0000', 'modified': '2019-11-04T16:22:10.920+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'GCA_000155495.1', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1685'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1685'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1685/contactPerson'}}, 'metadata': []}, {'prefix': 'rrid', 'mirId': 'MIR:00000558', 'name': 'RRID', 'pattern': '^[a-zA-Z]+.+$', 'description': 'The Research Resource Identification Initiative provides RRIDs to 4 main classes of resources: Antibodies, Cell Lines, Model Organisms, and Databases / Software tools.: Antibodies, Model Organisms, and Databases / Software tools.\\r\\nThe initiative works with participating journals to intercept manuscripts in the publication process that use these resources, and allows publication authors to incorporate RRIDs within the methods sections. It also provides resolver services that access curated data from 10 data sources: the antibody registry (a curated catalog of antibodies), the SciCrunch registry (a curated catalog of software tools and databases), and model organism nomenclature authority databases (MGI, FlyBase, WormBase, RGD), as well as various stock centers. These RRIDs are aggregated and can be searched through SciCrunch.', 'created': '2019-06-11T14:17:48.275+0000', 'modified': '2019-11-07T09:50:56.983+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': 'AB_262044', 'namespaceEmbeddedInLui': True, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1559'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1559'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/1559/contactPerson'}}, 'metadata': []}, {'prefix': 'spp', 'mirId': 'MIR:00000726', 'name': 'Signaling Pathways Project', 'pattern': '^10.\\\\w{4}/\\\\w{10}$', 'description': \"The Signaling Pathways Project is an integrated 'omics knowledgebase based upon public, manually curated transcriptomic and cistromic (ChIP-Seq) datasets involving genetic and small molecule manipulations of cellular receptors, enzymes and transcription factors. Our goal is to create a resource where scientists can routinely generate research hypotheses or validate bench data relevant to cellular signaling pathways.\", 'created': '2019-11-07T10:59:42.032+0000', 'modified': '2019-11-07T10:59:42.032+0000', 'deprecated': False, 'deprecationDate': None, 'sampleId': '10.1621/vwN2g2HaX3', 'namespaceEmbeddedInLui': False, '_links': {'self': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2037'}, 'namespace': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2037'}, 'contactPerson': {'href': 'https://registry.api.identifiers.org/restApi/namespaces/2037/contactPerson'}}, 'metadata': []}]\n"
     ]
    }
   ],
   "source": [
    "print(dataset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 204,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Bad metadata: 0\n",
      "No metadata: 651\n",
      "Contains metadata: 33\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "\n",
    "dataset_keys_to_keep = [\"prefix\", \"metadata\"]\n",
    "final_dataset = [{ key: namespace[key] for key in dataset_keys_to_keep } for namespace in dataset]\n",
    "\n",
    "df = pd.DataFrame.from_dict(final_dataset)\n",
    "\n",
    "bad_metadata_count = df[df.metadata == 'ERROR'].shape[0]\n",
    "no_metadata_count = df[df.metadata.map(len) == 0].shape[0]\n",
    "total_metadata = df.shape[0] - bad_metadata_count - no_metadata_count\n",
    "\n",
    "print(f\"Bad metadata: {bad_metadata_count}\")\n",
    "print(f\"No metadata: {no_metadata_count}\")\n",
    "print(f\"Contains metadata: {total_metadata}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>prefix</th>\n",
       "      <th>metadata</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ensembl</td>\n",
       "      <td>[{'identifier': 'ENSG00000139618', '@context':...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>reactome</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>prosite</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>rgd</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>biogrid</td>\n",
       "      <td>[{'@context': 'https://schema.org/', '@type': ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>chembl.compound</td>\n",
       "      <td>[{'identifier': 'CHEMBL308052', 'inChIKey': 'B...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>dbsnp</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>disprot</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>cath.domain</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>nextprot</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@graph': [...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>narcis</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>cellimage</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>hamap</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>cst.ab</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>atfdb.family</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@graph': ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>mobidb</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>biosample</td>\n",
       "      <td>[{'@context': ['http://schema.org', {'OBI': 'h...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>fairsharing</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@id': 'htt...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>neurovault.image</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>rgd.qtl</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>rgd.strain</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>iuphar.ligand</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@id': 'htt...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>genewiki</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>metanetx.chemical</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>metanetx.reaction</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>metanetx.compartment</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>cellosaurus</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>cosmic</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>neurovault.collection</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>kaggle</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>encode</td>\n",
       "      <td>[{'documents': ['/documents/0994a79b-44ae-49ef...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>hpscreg</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>lei</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    prefix                                           metadata\n",
       "1                  ensembl  [{'identifier': 'ENSG00000139618', '@context':...\n",
       "16                reactome  [{'@context': 'http://schema.org', '@type': 'W...\n",
       "29                 prosite  [{'@context': ['https://schema.org/'], '@type'...\n",
       "43                     rgd  [{'@context': 'http://schema.org', '@type': 'D...\n",
       "54                 biogrid  [{'@context': 'https://schema.org/', '@type': ...\n",
       "80         chembl.compound  [{'identifier': 'CHEMBL308052', 'inChIKey': 'B...\n",
       "158                  dbsnp  [{'@context': 'http://schema.org', '@type': 'D...\n",
       "195                disprot  [{'@context': 'http://schema.org/', '@type': '...\n",
       "206            cath.domain  [{'@context': 'http://schema.org', '@type': 'W...\n",
       "232               nextprot  [{'@context': 'http://schema.org', '@graph': [...\n",
       "236                 narcis  [{'@context': 'http://schema.org/', '@type': '...\n",
       "253              cellimage  [{'@context': 'http://schema.org', '@type': 'D...\n",
       "281                  hamap  [{'@context': ['https://schema.org/'], '@type'...\n",
       "298                 cst.ab  [{'@context': 'http://schema.org/', '@type': '...\n",
       "301           atfdb.family  [{'@context': 'https://schema.org', '@graph': ...\n",
       "316                 mobidb  [{'@context': 'http://schema.org', '@type': 'D...\n",
       "336              biosample  [{'@context': ['http://schema.org', {'OBI': 'h...\n",
       "350            fairsharing  [{'@context': 'http://schema.org', '@id': 'htt...\n",
       "352       neurovault.image  [{'@context': 'https://schema.org', '@type': '...\n",
       "435                rgd.qtl  [{'@context': 'http://schema.org', '@type': 'D...\n",
       "436             rgd.strain  [{'@context': 'http://schema.org', '@type': 'D...\n",
       "441          iuphar.ligand  [{'@context': 'http://schema.org', '@id': 'htt...\n",
       "472               genewiki  [{'@context': 'https://schema.org', '@type': '...\n",
       "549      metanetx.chemical  [{'@context': 'https://schema.org', '@type': '...\n",
       "550      metanetx.reaction  [{'@context': 'https://schema.org', '@type': '...\n",
       "551   metanetx.compartment  [{'@context': 'https://schema.org', '@type': '...\n",
       "579            cellosaurus  [{'@context': ['https://schema.org/'], '@type'...\n",
       "582                 cosmic  [{'@context': 'http://schema.org', '@type': 'D...\n",
       "614  neurovault.collection  [{'@context': 'https://schema.org', '@type': '...\n",
       "638                 kaggle  [{'@context': 'http://schema.org/', '@type': '...\n",
       "642                 encode  [{'documents': ['/documents/0994a79b-44ae-49ef...\n",
       "654                hpscreg  [{'@context': 'http://schema.org', '@type': 'W...\n",
       "661                    lei  [{'@context': 'http://schema.org', '@type': 'W..."
      ]
     },
     "execution_count": 205,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "namespaces_with_metadata = df[df.metadata.map(len) != 0]\n",
    "namespaces_with_metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Pie chart style\n",
    "custom_style = Style(\n",
    "    opacity='0.8',\n",
    "    opacity_hover='0.5',\n",
    "    title_font_size=36,\n",
    "    tooltip_font_size=10,\n",
    "    inner_radius=0.75,\n",
    "    plot_background=\"rgba(249, 249, 249, 1)\"\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <!DOCTYPE html>\n",
       "    <html>\n",
       "      <head>\n",
       "      <script type=\"text/javascript\" src=\"http://kozea.github.com/pygal.js/javascripts/svg.jquery.js\"></script>\n",
       "      <script type=\"text/javascript\" src=\"https://kozea.github.io/pygal.js/2.0.x/pygal-tooltips.min.js\"\"></script>\n",
       "      </head>\n",
       "      <body>\n",
       "        <figure>\n",
       "          <?xml version='1.0' encoding='utf-8'?>\n",
       "<svg xmlns:xlink=\"http://www.w3.org/1999/xlink\" class=\"pygal-chart\" id=\"chart-c12009ed-8655-443a-99b8-681999f12a67\" viewBox=\"0 0 800 400\" xmlns=\"http://www.w3.org/2000/svg\"><!--Generated with pygal 2.4.0 (etree) ¬©Kozea 2012-2016 on 2019-11-20--><!--http://pygal.org--><!--http://github.com/Kozea/pygal--><defs><style type=\"text/css\">#chart-c12009ed-8655-443a-99b8-681999f12a67{-webkit-user-select:none;-webkit-font-smoothing:antialiased;font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace}#chart-c12009ed-8655-443a-99b8-681999f12a67 .title{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:36px}#chart-c12009ed-8655-443a-99b8-681999f12a67 .legends .legend text{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:14px}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis text{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis text.major{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-c12009ed-8655-443a-99b8-681999f12a67 .text-overlay text.value{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:16px}#chart-c12009ed-8655-443a-99b8-681999f12a67 .text-overlay text.label{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-c12009ed-8655-443a-99b8-681999f12a67 text.no_data{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:64px}\n",
       "#chart-c12009ed-8655-443a-99b8-681999f12a67{background-color:rgba(249,249,249,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 path,#chart-c12009ed-8655-443a-99b8-681999f12a67 line,#chart-c12009ed-8655-443a-99b8-681999f12a67 rect,#chart-c12009ed-8655-443a-99b8-681999f12a67 circle{-webkit-transition:150ms;-moz-transition:150ms;transition:150ms}#chart-c12009ed-8655-443a-99b8-681999f12a67 .graph &gt; .background{fill:rgba(249,249,249,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .plot &gt; .background{fill:rgba(249,249,249,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .graph{fill:rgba(0,0,0,.87)}#chart-c12009ed-8655-443a-99b8-681999f12a67 text.no_data{fill:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .title{fill:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .legends .legend text{fill:rgba(0,0,0,.87)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .legends .legend:hover text{fill:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis .line{stroke:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis .guide.line{stroke:rgba(0,0,0,.54)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis .major.line{stroke:rgba(0,0,0,.87)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis text.major{fill:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.y .guides:hover .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .line-graph .axis.x .guides:hover .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .stackedline-graph .axis.x .guides:hover .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .xy-graph .axis.x .guides:hover .guide.line{stroke:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis .guides:hover text{fill:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .reactive{fill-opacity:0.8;stroke-opacity:.8}#chart-c12009ed-8655-443a-99b8-681999f12a67 .ci{stroke:rgba(0,0,0,.87)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .reactive.active,#chart-c12009ed-8655-443a-99b8-681999f12a67 .active .reactive{fill-opacity:0.5;stroke-opacity:.9;stroke-width:4}#chart-c12009ed-8655-443a-99b8-681999f12a67 .ci .reactive.active{stroke-width:1.5}#chart-c12009ed-8655-443a-99b8-681999f12a67 .series text{fill:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip rect{fill:rgba(249,249,249,1);stroke:rgba(0,0,0,1);-webkit-transition:opacity 150ms;-moz-transition:opacity 150ms;transition:opacity 150ms}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip .label{fill:rgba(0,0,0,.87)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip .label{fill:rgba(0,0,0,.87)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip .legend{font-size:.8em;fill:rgba(0,0,0,.54)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip .x_label{font-size:.6em;fill:rgba(0,0,0,1)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip .xlink{font-size:.5em;text-decoration:underline}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip .value{font-size:1.5em}#chart-c12009ed-8655-443a-99b8-681999f12a67 .bound{font-size:.5em}#chart-c12009ed-8655-443a-99b8-681999f12a67 .max-value{font-size:.75em;fill:rgba(0,0,0,.54)}#chart-c12009ed-8655-443a-99b8-681999f12a67 .map-element{fill:rgba(249,249,249,1);stroke:rgba(0,0,0,.54) !important}#chart-c12009ed-8655-443a-99b8-681999f12a67 .map-element .reactive{fill-opacity:inherit;stroke-opacity:inherit}#chart-c12009ed-8655-443a-99b8-681999f12a67 .color-0,#chart-c12009ed-8655-443a-99b8-681999f12a67 .color-0 a:visited{stroke:#F44336;fill:#F44336}#chart-c12009ed-8655-443a-99b8-681999f12a67 .color-1,#chart-c12009ed-8655-443a-99b8-681999f12a67 .color-1 a:visited{stroke:#3F51B5;fill:#3F51B5}#chart-c12009ed-8655-443a-99b8-681999f12a67 .color-2,#chart-c12009ed-8655-443a-99b8-681999f12a67 .color-2 a:visited{stroke:#009688;fill:#009688}#chart-c12009ed-8655-443a-99b8-681999f12a67 .text-overlay .color-0 text{fill:black}#chart-c12009ed-8655-443a-99b8-681999f12a67 .text-overlay .color-1 text{fill:black}#chart-c12009ed-8655-443a-99b8-681999f12a67 .text-overlay .color-2 text{fill:black}\n",
       "#chart-c12009ed-8655-443a-99b8-681999f12a67 text.no_data{text-anchor:middle}#chart-c12009ed-8655-443a-99b8-681999f12a67 .guide.line{fill:none}#chart-c12009ed-8655-443a-99b8-681999f12a67 .centered{text-anchor:middle}#chart-c12009ed-8655-443a-99b8-681999f12a67 .title{text-anchor:middle}#chart-c12009ed-8655-443a-99b8-681999f12a67 .legends .legend text{fill-opacity:1}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.x text{text-anchor:middle}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.x:not(.web) text[transform]{text-anchor:start}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.x:not(.web) text[transform].backwards{text-anchor:end}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.y text{text-anchor:end}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.y text[transform].backwards{text-anchor:start}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.y2 text{text-anchor:start}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.y2 text[transform].backwards{text-anchor:end}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis .guide.line{stroke-dasharray:4,4}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis .major.guide.line{stroke-dasharray:6,6}#chart-c12009ed-8655-443a-99b8-681999f12a67 .horizontal .axis.y .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .horizontal .axis.y2 .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .vertical .axis.x .guide.line{opacity:0}#chart-c12009ed-8655-443a-99b8-681999f12a67 .horizontal .axis.always_show .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .vertical .axis.always_show .guide.line{opacity:1 !important}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.y .guides:hover .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.y2 .guides:hover .guide.line,#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis.x .guides:hover .guide.line{opacity:1}#chart-c12009ed-8655-443a-99b8-681999f12a67 .axis .guides:hover text{opacity:1}#chart-c12009ed-8655-443a-99b8-681999f12a67 .nofill{fill:none}#chart-c12009ed-8655-443a-99b8-681999f12a67 .subtle-fill{fill-opacity:.2}#chart-c12009ed-8655-443a-99b8-681999f12a67 .dot{stroke-width:1px;fill-opacity:1}#chart-c12009ed-8655-443a-99b8-681999f12a67 .dot.active{stroke-width:5px}#chart-c12009ed-8655-443a-99b8-681999f12a67 .dot.negative{fill:transparent}#chart-c12009ed-8655-443a-99b8-681999f12a67 text,#chart-c12009ed-8655-443a-99b8-681999f12a67 tspan{stroke:none !important}#chart-c12009ed-8655-443a-99b8-681999f12a67 .series text.active{opacity:1}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip rect{fill-opacity:.95;stroke-width:.5}#chart-c12009ed-8655-443a-99b8-681999f12a67 .tooltip text{fill-opacity:1}#chart-c12009ed-8655-443a-99b8-681999f12a67 .showable{visibility:hidden}#chart-c12009ed-8655-443a-99b8-681999f12a67 .showable.shown{visibility:visible}#chart-c12009ed-8655-443a-99b8-681999f12a67 .gauge-background{fill:rgba(229,229,229,1);stroke:none}#chart-c12009ed-8655-443a-99b8-681999f12a67 .bg-lines{stroke:rgba(249,249,249,1);stroke-width:2px}</style><script type=\"text/javascript\">window.pygal = window.pygal || {};window.pygal.config = window.pygal.config || {};window.pygal.config['c12009ed-8655-443a-99b8-681999f12a67'] = {\"allow_interruptions\": false, \"box_mode\": \"extremes\", \"classes\": [\"pygal-chart\"], \"css\": [\"file://style.css\", \"file://graph.css\"], \"defs\": [], \"disable_xml_declaration\": false, \"dots_size\": 2.5, \"dynamic_print_values\": false, \"explicit_size\": false, \"fill\": false, \"force_uri_protocol\": \"https\", \"formatter\": null, \"half_pie\": false, \"height\": 400, \"include_x_axis\": false, \"inner_radius\": 0, \"interpolate\": null, \"interpolation_parameters\": {}, \"interpolation_precision\": 250, \"inverse_y_axis\": false, \"js\": [\"//kozea.github.io/pygal.js/2.0.x/pygal-tooltips.min.js\"], \"legend_at_bottom\": false, \"legend_at_bottom_columns\": null, \"legend_box_size\": 12, \"logarithmic\": false, \"margin\": 20, \"margin_bottom\": null, \"margin_left\": null, \"margin_right\": null, \"margin_top\": null, \"max_scale\": 16, \"min_scale\": 4, \"missing_value_fill_truncation\": \"x\", \"no_data_text\": \"No data\", \"no_prefix\": false, \"order_min\": null, \"pretty_print\": false, \"print_labels\": false, \"print_values\": false, \"print_values_position\": \"center\", \"print_zeroes\": true, \"range\": null, \"rounded_bars\": null, \"secondary_range\": null, \"show_dots\": true, \"show_legend\": true, \"show_minor_x_labels\": true, \"show_minor_y_labels\": true, \"show_only_major_dots\": false, \"show_x_guides\": false, \"show_x_labels\": true, \"show_y_guides\": true, \"show_y_labels\": true, \"spacing\": 10, \"stack_from_top\": false, \"strict\": false, \"stroke\": true, \"stroke_style\": null, \"style\": {\"background\": \"rgba(249, 249, 249, 1)\", \"ci_colors\": [], \"colors\": [\"#F44336\", \"#3F51B5\", \"#009688\", \"#FFC107\", \"#FF5722\", \"#9C27B0\", \"#03A9F4\", \"#8BC34A\", \"#FF9800\", \"#E91E63\", \"#2196F3\", \"#4CAF50\", \"#FFEB3B\", \"#673AB7\", \"#00BCD4\", \"#CDDC39\", \"#9E9E9E\", \"#607D8B\"], \"font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"foreground\": \"rgba(0, 0, 0, .87)\", \"foreground_strong\": \"rgba(0, 0, 0, 1)\", \"foreground_subtle\": \"rgba(0, 0, 0, .54)\", \"guide_stroke_dasharray\": \"4,4\", \"inner_radius\": 0.75, \"label_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"label_font_size\": 10, \"legend_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"legend_font_size\": 14, \"major_guide_stroke_dasharray\": \"6,6\", \"major_label_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"major_label_font_size\": 10, \"no_data_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"no_data_font_size\": 64, \"opacity\": \"0.8\", \"opacity_hover\": \"0.5\", \"plot_background\": \"rgba(249, 249, 249, 1)\", \"stroke_opacity\": \".8\", \"stroke_opacity_hover\": \".9\", \"title_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"title_font_size\": 36, \"tooltip_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"tooltip_font_size\": 10, \"transition\": \"150ms\", \"value_background\": \"rgba(229, 229, 229, 1)\", \"value_colors\": [], \"value_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"value_font_size\": 16, \"value_label_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"value_label_font_size\": 10}, \"title\": null, \"tooltip_border_radius\": 1, \"tooltip_fancy_mode\": true, \"truncate_label\": null, \"truncate_legend\": null, \"width\": 800, \"x_label_rotation\": 0, \"x_labels\": null, \"x_labels_major\": null, \"x_labels_major_count\": null, \"x_labels_major_every\": null, \"x_title\": null, \"xrange\": null, \"y_label_rotation\": 0, \"y_labels\": null, \"y_labels_major\": null, \"y_labels_major_count\": null, \"y_labels_major_every\": null, \"y_title\": null, \"zero\": 0, \"legends\": [\"Bad metadata\", \"No metadata\", \"Contains metadata\"]}</script><script type=\"text/javascript\" xlink:href=\"https://kozea.github.io/pygal.js/2.0.x/pygal-tooltips.min.js\" /></defs><title>Pygal</title><g class=\"graph pie-graph vertical\"><rect class=\"background\" height=\"400\" width=\"800\" x=\"0\" y=\"0\" /><g class=\"plot\" transform=\"translate(168, 20)\"><rect class=\"background\" height=\"360\" width=\"612.0\" x=\"0\" y=\"0\" /><g class=\"series serie-0 color-0\"><g class=\"slices\"><g class=\"slice\"><desc class=\"value\">0</desc><desc class=\"x centered\">306.0</desc><desc class=\"y centered\">99.0</desc></g></g></g><g class=\"series serie-1 color-1\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M306.000000 18.000000 A162.000000 162.000000 0 1 1 257.640601 25.386390 L306.000000 180.000000 A0.000000 0.000000 0 1 0 306.000000 180.000000 z\" /><desc class=\"value\">651</desc><desc class=\"x centered\">318.2300609701782</desc><desc class=\"y centered\">260.07137821135416</desc></g></g></g><g class=\"series serie-2 color-2\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M257.640601 25.386390 A162.000000 162.000000 0 0 1 306.000000 18.000000 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">33</desc><desc class=\"x centered\">293.7699390298217</desc><desc class=\"y centered\">99.92862178864584</desc></g></g></g></g><g class=\"titles\" /><g class=\"plot overlay\" transform=\"translate(168, 20)\"><g class=\"series serie-0 color-0\" /><g class=\"series serie-1 color-1\" /><g class=\"series serie-2 color-2\" /></g><g class=\"plot text-overlay\" transform=\"translate(168, 20)\"><g class=\"series serie-0 color-0\" /><g class=\"series serie-1 color-1\" /><g class=\"series serie-2 color-2\" /></g><g class=\"plot tooltip-overlay\" transform=\"translate(168, 20)\"><g class=\"tooltip\" style=\"opacity: 0\" transform=\"translate(0 0)\"><rect class=\"tooltip-box\" height=\"0\" rx=\"1\" ry=\"1\" width=\"0\" /><g class=\"text\" /></g></g><g class=\"legends\" transform=\"translate(10, 30)\"><g class=\"legend reactive activate-serie\" id=\"activate-serie-0\"><rect class=\"color-0 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"1.0\" /><text x=\"17.0\" y=\"11.2\">Bad metadata</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-1\"><rect class=\"color-1 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"22.0\" /><text x=\"17.0\" y=\"32.2\">No metadata</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-2\"><rect class=\"color-2 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"43.0\" /><text x=\"17.0\" y=\"53.2\">Contains metad‚Ä¶</text><title>Contains metadata</title></g></g><g class=\"legends\" transform=\"translate(790, 30)\" /></g></svg>\n",
       "        </figure>\n",
       "      </body>\n",
       "    </html>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 207,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pygal\n",
    "from pygal.style import Style\n",
    "from IPython.display import SVG, display\n",
    "\n",
    "metadata_contents_pie = pygal.Pie(height=400, tooltip_border_radius=1, style=custom_style)\n",
    "\n",
    "metadata_contents_pie.add(\"Bad metadata\", bad_metadata_count)\n",
    "metadata_contents_pie.add(\"No metadata\", no_metadata_count)\n",
    "metadata_contents_pie.add(\"Contains metadata\", total_metadata)\n",
    "\n",
    "display_interactive_pygal(metadata_contents_pie)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>prefix</th>\n",
       "      <th>metadata</th>\n",
       "      <th>metadata_type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ensembl</td>\n",
       "      <td>[{'identifier': 'ENSG00000139618', '@context':...</td>\n",
       "      <td>DataRecord</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>reactome</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>prosite</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "      <td>Organization</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>rgd</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>biogrid</td>\n",
       "      <td>[{'@context': 'https://schema.org/', '@type': ...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>chembl.compound</td>\n",
       "      <td>[{'identifier': 'CHEMBL308052', 'inChIKey': 'B...</td>\n",
       "      <td>MolecularEntity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>dbsnp</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>disprot</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>cath.domain</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>nextprot</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@graph': [...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>narcis</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>CreativeWork</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>cellimage</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>hamap</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "      <td>Organization</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>cst.ab</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>Product</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>atfdb.family</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@graph': ...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>mobidb</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>DataCatalog</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>biosample</td>\n",
       "      <td>[{'@context': ['http://schema.org', {'OBI': 'h...</td>\n",
       "      <td>DataRecord</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>fairsharing</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@id': 'htt...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>neurovault.image</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>Organization</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>rgd.qtl</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>rgd.strain</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>iuphar.ligand</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@id': 'htt...</td>\n",
       "      <td>DataRecord</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>genewiki</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>metanetx.chemical</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>metanetx.reaction</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>metanetx.compartment</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>cellosaurus</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "      <td>Organization</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>cosmic</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>neurovault.collection</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>Organization</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>kaggle</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>Dataset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>encode</td>\n",
       "      <td>[{'documents': ['/documents/0994a79b-44ae-49ef...</td>\n",
       "      <td>Experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>hpscreg</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>lei</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    prefix                                           metadata  \\\n",
       "1                  ensembl  [{'identifier': 'ENSG00000139618', '@context':...   \n",
       "16                reactome  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "29                 prosite  [{'@context': ['https://schema.org/'], '@type'...   \n",
       "43                     rgd  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "54                 biogrid  [{'@context': 'https://schema.org/', '@type': ...   \n",
       "80         chembl.compound  [{'identifier': 'CHEMBL308052', 'inChIKey': 'B...   \n",
       "158                  dbsnp  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "195                disprot  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "206            cath.domain  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "232               nextprot  [{'@context': 'http://schema.org', '@graph': [...   \n",
       "236                 narcis  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "253              cellimage  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "281                  hamap  [{'@context': ['https://schema.org/'], '@type'...   \n",
       "298                 cst.ab  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "301           atfdb.family  [{'@context': 'https://schema.org', '@graph': ...   \n",
       "316                 mobidb  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "336              biosample  [{'@context': ['http://schema.org', {'OBI': 'h...   \n",
       "350            fairsharing  [{'@context': 'http://schema.org', '@id': 'htt...   \n",
       "352       neurovault.image  [{'@context': 'https://schema.org', '@type': '...   \n",
       "435                rgd.qtl  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "436             rgd.strain  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "441          iuphar.ligand  [{'@context': 'http://schema.org', '@id': 'htt...   \n",
       "472               genewiki  [{'@context': 'https://schema.org', '@type': '...   \n",
       "549      metanetx.chemical  [{'@context': 'https://schema.org', '@type': '...   \n",
       "550      metanetx.reaction  [{'@context': 'https://schema.org', '@type': '...   \n",
       "551   metanetx.compartment  [{'@context': 'https://schema.org', '@type': '...   \n",
       "579            cellosaurus  [{'@context': ['https://schema.org/'], '@type'...   \n",
       "582                 cosmic  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "614  neurovault.collection  [{'@context': 'https://schema.org', '@type': '...   \n",
       "638                 kaggle  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "642                 encode  [{'documents': ['/documents/0994a79b-44ae-49ef...   \n",
       "654                hpscreg  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "661                    lei  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "\n",
       "       metadata_type  \n",
       "1         DataRecord  \n",
       "16           WebSite  \n",
       "29      Organization  \n",
       "43           Dataset  \n",
       "54           Dataset  \n",
       "80   MolecularEntity  \n",
       "158          Dataset  \n",
       "195      DataCatalog  \n",
       "206          WebSite  \n",
       "232             None  \n",
       "236     CreativeWork  \n",
       "253          Dataset  \n",
       "281     Organization  \n",
       "298          Product  \n",
       "301             None  \n",
       "316      DataCatalog  \n",
       "336       DataRecord  \n",
       "350          Dataset  \n",
       "352     Organization  \n",
       "435          Dataset  \n",
       "436          Dataset  \n",
       "441       DataRecord  \n",
       "472          Article  \n",
       "549      DataCatalog  \n",
       "550      DataCatalog  \n",
       "551      DataCatalog  \n",
       "579     Organization  \n",
       "582          Dataset  \n",
       "614     Organization  \n",
       "638          Dataset  \n",
       "642       Experiment  \n",
       "654          WebSite  \n",
       "661          WebSite  "
      ]
     },
     "execution_count": 208,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def get_schema_type(metadata):\n",
    "    if len(metadata) == 0:\n",
    "        return None\n",
    "\n",
    "    # flatten lists of lists... :(\n",
    "    if isinstance(metadata[0], list):\n",
    "        metadata[0] = metadata[0][0]\n",
    "    \n",
    "    result = metadata[0].get(\"@type\", None)\n",
    "    if isinstance(result, list):\n",
    "        result = result[0]\n",
    "        \n",
    "    return result\n",
    "\n",
    "df[\"metadata_type\"] = df[\"metadata\"].map(lambda x: get_schema_type(x))\n",
    "\n",
    "df[df[\"metadata\"].map(len) != 0]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 210,
   "metadata": {},
   "outputs": [],
   "source": [
    "from IPython.display import HTML\n",
    "import pygal\n",
    "\n",
    "def display_interactive_pygal(pygal_chart):\n",
    "    base_html = \"\"\"\n",
    "    <!DOCTYPE html>\n",
    "    <html>\n",
    "      <head>\n",
    "      <script type=\"text/javascript\" src=\"http://kozea.github.com/pygal.js/javascripts/svg.jquery.js\"></script>\n",
    "      <script type=\"text/javascript\" src=\"https://kozea.github.io/pygal.js/2.0.x/pygal-tooltips.min.js\"\"></script>\n",
    "      </head>\n",
    "      <body>\n",
    "        <figure>\n",
    "          {rendered_chart}\n",
    "        </figure>\n",
    "      </body>\n",
    "    </html>\n",
    "    \"\"\"\n",
    "\n",
    "    return HTML(base_html.format(rendered_chart=pygal_chart.render(is_unicode=True)))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Dataset            9\n",
       "Organization       5\n",
       "DataCatalog        5\n",
       "WebSite            4\n",
       "DataRecord         3\n",
       "CreativeWork       1\n",
       "Article            1\n",
       "MolecularEntity    1\n",
       "Product            1\n",
       "Experiment         1\n",
       "Name: metadata_type, dtype: int64"
      ]
     },
     "execution_count": 211,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata_type_counts = df.metadata_type.value_counts()\n",
    "metadata_type_counts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 213,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <!DOCTYPE html>\n",
       "    <html>\n",
       "      <head>\n",
       "      <script type=\"text/javascript\" src=\"http://kozea.github.com/pygal.js/javascripts/svg.jquery.js\"></script>\n",
       "      <script type=\"text/javascript\" src=\"https://kozea.github.io/pygal.js/2.0.x/pygal-tooltips.min.js\"\"></script>\n",
       "      </head>\n",
       "      <body>\n",
       "        <figure>\n",
       "          <?xml version='1.0' encoding='utf-8'?>\n",
       "<svg xmlns:xlink=\"http://www.w3.org/1999/xlink\" class=\"pygal-chart\" id=\"chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5\" viewBox=\"0 0 800 400\" xmlns=\"http://www.w3.org/2000/svg\"><!--Generated with pygal 2.4.0 (etree) ¬©Kozea 2012-2016 on 2019-11-20--><!--http://pygal.org--><!--http://github.com/Kozea/pygal--><defs><style type=\"text/css\">#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5{-webkit-user-select:none;-webkit-font-smoothing:antialiased;font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .title{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:36px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .legends .legend text{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:14px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis text{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis text.major{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay text.value{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:16px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay text.label{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:10px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 text.no_data{font-family:Consolas,\"Liberation Mono\",Menlo,Courier,monospace;font-size:64px}\n",
       "#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5{background-color:rgba(249,249,249,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 path,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 rect,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 circle{-webkit-transition:150ms;-moz-transition:150ms;transition:150ms}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .graph &gt; .background{fill:rgba(249,249,249,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .plot &gt; .background{fill:rgba(249,249,249,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .graph{fill:rgba(0,0,0,.87)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 text.no_data{fill:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .title{fill:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .legends .legend text{fill:rgba(0,0,0,.87)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .legends .legend:hover text{fill:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis .line{stroke:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis .guide.line{stroke:rgba(0,0,0,.54)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis .major.line{stroke:rgba(0,0,0,.87)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis text.major{fill:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.y .guides:hover .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .line-graph .axis.x .guides:hover .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .stackedline-graph .axis.x .guides:hover .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .xy-graph .axis.x .guides:hover .guide.line{stroke:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis .guides:hover text{fill:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .reactive{fill-opacity:0.8;stroke-opacity:.8}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .ci{stroke:rgba(0,0,0,.87)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .reactive.active,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .active .reactive{fill-opacity:0.5;stroke-opacity:.9;stroke-width:4}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .ci .reactive.active{stroke-width:1.5}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .series text{fill:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip rect{fill:rgba(249,249,249,1);stroke:rgba(0,0,0,1);-webkit-transition:opacity 150ms;-moz-transition:opacity 150ms;transition:opacity 150ms}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip .label{fill:rgba(0,0,0,.87)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip .label{fill:rgba(0,0,0,.87)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip .legend{font-size:.8em;fill:rgba(0,0,0,.54)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip .x_label{font-size:.6em;fill:rgba(0,0,0,1)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip .xlink{font-size:.5em;text-decoration:underline}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip .value{font-size:1.5em}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .bound{font-size:.5em}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .max-value{font-size:.75em;fill:rgba(0,0,0,.54)}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .map-element{fill:rgba(249,249,249,1);stroke:rgba(0,0,0,.54) !important}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .map-element .reactive{fill-opacity:inherit;stroke-opacity:inherit}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-0,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-0 a:visited{stroke:#F44336;fill:#F44336}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-1,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-1 a:visited{stroke:#3F51B5;fill:#3F51B5}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-2,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-2 a:visited{stroke:#009688;fill:#009688}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-3,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-3 a:visited{stroke:#FFC107;fill:#FFC107}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-4,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-4 a:visited{stroke:#FF5722;fill:#FF5722}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-5,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-5 a:visited{stroke:#9C27B0;fill:#9C27B0}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-6,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-6 a:visited{stroke:#03A9F4;fill:#03A9F4}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-7,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-7 a:visited{stroke:#8BC34A;fill:#8BC34A}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-8,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-8 a:visited{stroke:#FF9800;fill:#FF9800}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-9,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .color-9 a:visited{stroke:#E91E63;fill:#E91E63}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-0 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-1 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-2 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-3 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-4 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-5 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-6 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-7 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-8 text{fill:black}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .text-overlay .color-9 text{fill:black}\n",
       "#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 text.no_data{text-anchor:middle}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .guide.line{fill:none}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .centered{text-anchor:middle}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .title{text-anchor:middle}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .legends .legend text{fill-opacity:1}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.x text{text-anchor:middle}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.x:not(.web) text[transform]{text-anchor:start}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.x:not(.web) text[transform].backwards{text-anchor:end}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.y text{text-anchor:end}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.y text[transform].backwards{text-anchor:start}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.y2 text{text-anchor:start}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.y2 text[transform].backwards{text-anchor:end}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis .guide.line{stroke-dasharray:4,4}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis .major.guide.line{stroke-dasharray:6,6}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .horizontal .axis.y .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .horizontal .axis.y2 .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .vertical .axis.x .guide.line{opacity:0}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .horizontal .axis.always_show .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .vertical .axis.always_show .guide.line{opacity:1 !important}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.y .guides:hover .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.y2 .guides:hover .guide.line,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis.x .guides:hover .guide.line{opacity:1}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .axis .guides:hover text{opacity:1}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .nofill{fill:none}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .subtle-fill{fill-opacity:.2}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .dot{stroke-width:1px;fill-opacity:1}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .dot.active{stroke-width:5px}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .dot.negative{fill:transparent}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 text,#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 tspan{stroke:none !important}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .series text.active{opacity:1}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip rect{fill-opacity:.95;stroke-width:.5}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .tooltip text{fill-opacity:1}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .showable{visibility:hidden}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .showable.shown{visibility:visible}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .gauge-background{fill:rgba(229,229,229,1);stroke:none}#chart-d757d6d4-1de2-4648-a8b7-4e8a93302bc5 .bg-lines{stroke:rgba(249,249,249,1);stroke-width:2px}</style><script type=\"text/javascript\">window.pygal = window.pygal || {};window.pygal.config = window.pygal.config || {};window.pygal.config['d757d6d4-1de2-4648-a8b7-4e8a93302bc5'] = {\"allow_interruptions\": false, \"box_mode\": \"extremes\", \"classes\": [\"pygal-chart\"], \"css\": [\"file://style.css\", \"file://graph.css\"], \"defs\": [], \"disable_xml_declaration\": false, \"dots_size\": 2.5, \"dynamic_print_values\": false, \"explicit_size\": false, \"fill\": false, \"force_uri_protocol\": \"https\", \"formatter\": null, \"half_pie\": false, \"height\": 400, \"include_x_axis\": false, \"inner_radius\": 0, \"interpolate\": null, \"interpolation_parameters\": {}, \"interpolation_precision\": 250, \"inverse_y_axis\": false, \"js\": [\"//kozea.github.io/pygal.js/2.0.x/pygal-tooltips.min.js\"], \"legend_at_bottom\": false, \"legend_at_bottom_columns\": null, \"legend_box_size\": 12, \"logarithmic\": false, \"margin\": 20, \"margin_bottom\": null, \"margin_left\": null, \"margin_right\": null, \"margin_top\": null, \"max_scale\": 16, \"min_scale\": 4, \"missing_value_fill_truncation\": \"x\", \"no_data_text\": \"No data\", \"no_prefix\": false, \"order_min\": null, \"pretty_print\": false, \"print_labels\": false, \"print_values\": false, \"print_values_position\": \"center\", \"print_zeroes\": true, \"range\": null, \"rounded_bars\": null, \"secondary_range\": null, \"show_dots\": true, \"show_legend\": true, \"show_minor_x_labels\": true, \"show_minor_y_labels\": true, \"show_only_major_dots\": false, \"show_x_guides\": false, \"show_x_labels\": true, \"show_y_guides\": true, \"show_y_labels\": true, \"spacing\": 10, \"stack_from_top\": false, \"strict\": false, \"stroke\": true, \"stroke_style\": null, \"style\": {\"background\": \"rgba(249, 249, 249, 1)\", \"ci_colors\": [], \"colors\": [\"#F44336\", \"#3F51B5\", \"#009688\", \"#FFC107\", \"#FF5722\", \"#9C27B0\", \"#03A9F4\", \"#8BC34A\", \"#FF9800\", \"#E91E63\", \"#2196F3\", \"#4CAF50\", \"#FFEB3B\", \"#673AB7\", \"#00BCD4\", \"#CDDC39\", \"#9E9E9E\", \"#607D8B\"], \"font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"foreground\": \"rgba(0, 0, 0, .87)\", \"foreground_strong\": \"rgba(0, 0, 0, 1)\", \"foreground_subtle\": \"rgba(0, 0, 0, .54)\", \"guide_stroke_dasharray\": \"4,4\", \"inner_radius\": 0.75, \"label_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"label_font_size\": 10, \"legend_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"legend_font_size\": 14, \"major_guide_stroke_dasharray\": \"6,6\", \"major_label_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"major_label_font_size\": 10, \"no_data_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"no_data_font_size\": 64, \"opacity\": \"0.8\", \"opacity_hover\": \"0.5\", \"plot_background\": \"rgba(249, 249, 249, 1)\", \"stroke_opacity\": \".8\", \"stroke_opacity_hover\": \".9\", \"title_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"title_font_size\": 36, \"tooltip_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"tooltip_font_size\": 10, \"transition\": \"150ms\", \"value_background\": \"rgba(229, 229, 229, 1)\", \"value_colors\": [], \"value_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"value_font_size\": 16, \"value_label_font_family\": \"Consolas, \\\"Liberation Mono\\\", Menlo, Courier, monospace\", \"value_label_font_size\": 10}, \"title\": null, \"tooltip_border_radius\": 1, \"tooltip_fancy_mode\": true, \"truncate_label\": null, \"truncate_legend\": null, \"width\": 800, \"x_label_rotation\": 0, \"x_labels\": null, \"x_labels_major\": null, \"x_labels_major_count\": null, \"x_labels_major_every\": null, \"x_title\": null, \"xrange\": null, \"y_label_rotation\": 0, \"y_labels\": null, \"y_labels_major\": null, \"y_labels_major_count\": null, \"y_labels_major_every\": null, \"y_title\": null, \"zero\": 0, \"legends\": [\"Dataset\", \"Organization\", \"DataCatalog\", \"WebSite\", \"DataRecord\", \"CreativeWork\", \"Article\", \"MolecularEntity\", \"Product\", \"Experiment\"]}</script><script type=\"text/javascript\" xlink:href=\"https://kozea.github.io/pygal.js/2.0.x/pygal-tooltips.min.js\" /></defs><title>Pygal</title><g class=\"graph pie-graph vertical\"><rect class=\"background\" height=\"400\" width=\"800\" x=\"0\" y=\"0\" /><g class=\"plot\" transform=\"translate(168, 20)\"><rect class=\"background\" height=\"360\" width=\"612.0\" x=\"0\" y=\"0\" /><g class=\"series serie-0 color-0\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M306.000000 18.000000 A162.000000 162.000000 0 0 1 462.828493 220.605710 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">9</desc><desc class=\"x centered\">370.0528346919536</desc><desc class=\"y centered\">130.4194154136393</desc></g></g></g><g class=\"series serie-1 color-1\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M462.828493 220.605710 A162.000000 162.000000 0 0 1 354.496826 334.570560 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">5</desc><desc class=\"x centered\">364.70821576555875</desc><desc class=\"y centered\">235.8063204451306</desc></g></g></g><g class=\"series serie-2 color-2\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M354.496826 334.570560 A162.000000 162.000000 0 0 1 200.477658 302.918816 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">5</desc><desc class=\"x centered\">289.6948198728185</desc><desc class=\"y centered\">259.34192524145203</desc></g></g></g><g class=\"series serie-3 color-3\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M200.477658 302.918816 A162.000000 162.000000 0 0 1 144.207926 188.205165 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">4</desc><desc class=\"x centered\">233.27783229476992</desc><desc class=\"y centered\">215.67192627616836</desc></g></g></g><g class=\"series serie-4 color-4\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M144.207926 188.205165 A162.000000 162.000000 0 0 1 168.519630 94.307830 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">3</desc><desc class=\"x centered\">227.58575337183746</desc><desc class=\"y centered\">159.6971448870436</desc></g></g></g><g class=\"series serie-5 color-5\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M168.519630 94.307830 A162.000000 162.000000 0 0 1 188.583568 68.387359 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">1</desc><desc class=\"x centered\">241.94716530804644</desc><desc class=\"y centered\">130.41941541363929</desc></g></g></g><g class=\"series serie-6 color-6\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M188.583568 68.387359 A162.000000 162.000000 0 0 1 213.454549 47.036323 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">1</desc><desc class=\"x centered\">253.23882889950002</desc><desc class=\"y centered\">118.54059206188393</desc></g></g></g><g class=\"series serie-7 color-7\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M213.454549 47.036323 A162.000000 162.000000 0 0 1 242.114351 31.128835 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">1</desc><desc class=\"x centered\">266.69054103498246</desc><desc class=\"y centered\">109.17792409228883</desc></g></g></g><g class=\"series serie-8 color-8\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M242.114351 31.128835 A162.000000 162.000000 0 0 1 273.389640 21.316150 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">1</desc><desc class=\"x centered\">281.75158703915804</desc><desc class=\"y centered\">102.71472023159603</desc></g></g></g><g class=\"series serie-9 color-9\"><g class=\"slices\"><g class=\"slice\"><path class=\"slice reactive tooltip-trigger\" d=\"M273.389640 21.316150 A162.000000 162.000000 0 0 1 306.000000 18.000000 L306.000000 180.000000 A0.000000 0.000000 0 0 0 306.000000 180.000000 z\" /><desc class=\"value\">1</desc><desc class=\"x centered\">297.805365919018</desc><desc class=\"y centered\">99.41558480525649</desc></g></g></g></g><g class=\"titles\" /><g class=\"plot overlay\" transform=\"translate(168, 20)\"><g class=\"series serie-0 color-0\" /><g class=\"series serie-1 color-1\" /><g class=\"series serie-2 color-2\" /><g class=\"series serie-3 color-3\" /><g class=\"series serie-4 color-4\" /><g class=\"series serie-5 color-5\" /><g class=\"series serie-6 color-6\" /><g class=\"series serie-7 color-7\" /><g class=\"series serie-8 color-8\" /><g class=\"series serie-9 color-9\" /></g><g class=\"plot text-overlay\" transform=\"translate(168, 20)\"><g class=\"series serie-0 color-0\" /><g class=\"series serie-1 color-1\" /><g class=\"series serie-2 color-2\" /><g class=\"series serie-3 color-3\" /><g class=\"series serie-4 color-4\" /><g class=\"series serie-5 color-5\" /><g class=\"series serie-6 color-6\" /><g class=\"series serie-7 color-7\" /><g class=\"series serie-8 color-8\" /><g class=\"series serie-9 color-9\" /></g><g class=\"plot tooltip-overlay\" transform=\"translate(168, 20)\"><g class=\"tooltip\" style=\"opacity: 0\" transform=\"translate(0 0)\"><rect class=\"tooltip-box\" height=\"0\" rx=\"1\" ry=\"1\" width=\"0\" /><g class=\"text\" /></g></g><g class=\"legends\" transform=\"translate(10, 30)\"><g class=\"legend reactive activate-serie\" id=\"activate-serie-0\"><rect class=\"color-0 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"1.0\" /><text x=\"17.0\" y=\"11.2\">Dataset</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-1\"><rect class=\"color-1 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"22.0\" /><text x=\"17.0\" y=\"32.2\">Organization</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-2\"><rect class=\"color-2 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"43.0\" /><text x=\"17.0\" y=\"53.2\">DataCatalog</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-3\"><rect class=\"color-3 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"64.0\" /><text x=\"17.0\" y=\"74.2\">WebSite</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-4\"><rect class=\"color-4 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"85.0\" /><text x=\"17.0\" y=\"95.2\">DataRecord</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-5\"><rect class=\"color-5 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"106.0\" /><text x=\"17.0\" y=\"116.2\">CreativeWork</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-6\"><rect class=\"color-6 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"127.0\" /><text x=\"17.0\" y=\"137.2\">Article</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-7\"><rect class=\"color-7 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"148.0\" /><text x=\"17.0\" y=\"158.2\">MolecularEntity</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-8\"><rect class=\"color-8 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"169.0\" /><text x=\"17.0\" y=\"179.2\">Product</text></g><g class=\"legend reactive activate-serie\" id=\"activate-serie-9\"><rect class=\"color-9 reactive\" height=\"12\" width=\"12\" x=\"0.0\" y=\"190.0\" /><text x=\"17.0\" y=\"200.2\">Experiment</text></g></g><g class=\"legends\" transform=\"translate(790, 30)\" /></g></svg>\n",
       "        </figure>\n",
       "      </body>\n",
       "    </html>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 213,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pygal\n",
    "from pygal.style import Style\n",
    "from IPython.display import SVG, display\n",
    "\n",
    "metadata_type_counts_list = [{\"label\": x, \"value\": y} for x, y in metadata_type_counts.items()]\n",
    "metadata_type_counts_list\n",
    "\n",
    "metadata_contents_pie_exploded = pygal.Pie(height=400, tooltip_border_radius=1, style=custom_style, print_values=False)\n",
    "\n",
    "for metadata_type_count in metadata_type_counts_list:\n",
    "    metadata_contents_pie_exploded.add(metadata_type_count[\"label\"], metadata_type_count[\"value\"])\n",
    "\n",
    "display_interactive_pygal(metadata_contents_pie_exploded)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 214,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>prefix</th>\n",
       "      <th>metadata</th>\n",
       "      <th>metadata_type</th>\n",
       "      <th>properties</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ensembl</td>\n",
       "      <td>[{'identifier': 'ENSG00000139618', '@context':...</td>\n",
       "      <td>DataRecord</td>\n",
       "      <td>[identifier, @context, @type, mainEntity, isPa...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>reactome</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "      <td>[@context, @type, url, potentialAction]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>prosite</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "      <td>Organization</td>\n",
       "      <td>[@context, @type, url, name, logo, memberof, p...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>rgd</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, url, keyw...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>biogrid</td>\n",
       "      <td>[{'@context': 'https://schema.org/', '@type': ...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, url, keyw...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>chembl.compound</td>\n",
       "      <td>[{'identifier': 'CHEMBL308052', 'inChIKey': 'B...</td>\n",
       "      <td>MolecularEntity</td>\n",
       "      <td>[identifier, inChIKey, @context, biochemicalIn...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>dbsnp</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, version, ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>disprot</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "      <td>[@context, @type, @id, name, description, url,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>cath.domain</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "      <td>[@context, @type, url, mainEntityOfPage]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>nextprot</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@graph': [...</td>\n",
       "      <td>None</td>\n",
       "      <td>[@context, @graph]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>narcis</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>CreativeWork</td>\n",
       "      <td>[@context, @type, @id, additionalType, name, h...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>cellimage</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, identifie...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>hamap</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "      <td>Organization</td>\n",
       "      <td>[@context, @type, url, name, logo, memberof, p...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>cst.ab</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>Product</td>\n",
       "      <td>[@context, @type, name, Category, additionalPr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>atfdb.family</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@graph': ...</td>\n",
       "      <td>None</td>\n",
       "      <td>[@context, @graph]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>mobidb</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>DataCatalog</td>\n",
       "      <td>[@context, @type, @id, name, description, url,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>biosample</td>\n",
       "      <td>[{'@context': ['http://schema.org', {'OBI': 'h...</td>\n",
       "      <td>DataRecord</td>\n",
       "      <td>[@context, @type, @id, identifier, dateCreated...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>fairsharing</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@id': 'htt...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @id, @type, alternateName, citation...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>neurovault.image</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>Organization</td>\n",
       "      <td>[@context, @type, name, url, logo, sameAs]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>rgd.qtl</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, url, keyw...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>rgd.strain</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, url, keyw...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>iuphar.ligand</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@id': 'htt...</td>\n",
       "      <td>DataRecord</td>\n",
       "      <td>[@context, @id, @type, dct:conformsTo, include...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>genewiki</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>Article</td>\n",
       "      <td>[@context, @type, name, url, sameAs, mainEntit...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>metanetx.chemical</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "      <td>[@context, @type, description, keywords, name,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>metanetx.reaction</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "      <td>[@context, @type, description, keywords, name,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>metanetx.compartment</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>DataCatalog</td>\n",
       "      <td>[@context, @type, description, keywords, name,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>cellosaurus</td>\n",
       "      <td>[{'@context': ['https://schema.org/'], '@type'...</td>\n",
       "      <td>Organization</td>\n",
       "      <td>[@context, @type, url, name, memberof, parentO...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>cosmic</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'D...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, version, ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>neurovault.collection</td>\n",
       "      <td>[{'@context': 'https://schema.org', '@type': '...</td>\n",
       "      <td>Organization</td>\n",
       "      <td>[@context, @type, name, url, logo, sameAs]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>kaggle</td>\n",
       "      <td>[{'@context': 'http://schema.org/', '@type': '...</td>\n",
       "      <td>Dataset</td>\n",
       "      <td>[@context, @type, name, description, url, same...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>encode</td>\n",
       "      <td>[{'documents': ['/documents/0994a79b-44ae-49ef...</td>\n",
       "      <td>Experiment</td>\n",
       "      <td>[documents, hub, replication_type, revoked_fil...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>hpscreg</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "      <td>[@context, @type, url, potentialAction]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>lei</td>\n",
       "      <td>[{'@context': 'http://schema.org', '@type': 'W...</td>\n",
       "      <td>WebSite</td>\n",
       "      <td>[@context, @type, name, alternateName, url, po...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    prefix                                           metadata  \\\n",
       "1                  ensembl  [{'identifier': 'ENSG00000139618', '@context':...   \n",
       "16                reactome  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "29                 prosite  [{'@context': ['https://schema.org/'], '@type'...   \n",
       "43                     rgd  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "54                 biogrid  [{'@context': 'https://schema.org/', '@type': ...   \n",
       "80         chembl.compound  [{'identifier': 'CHEMBL308052', 'inChIKey': 'B...   \n",
       "158                  dbsnp  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "195                disprot  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "206            cath.domain  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "232               nextprot  [{'@context': 'http://schema.org', '@graph': [...   \n",
       "236                 narcis  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "253              cellimage  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "281                  hamap  [{'@context': ['https://schema.org/'], '@type'...   \n",
       "298                 cst.ab  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "301           atfdb.family  [{'@context': 'https://schema.org', '@graph': ...   \n",
       "316                 mobidb  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "336              biosample  [{'@context': ['http://schema.org', {'OBI': 'h...   \n",
       "350            fairsharing  [{'@context': 'http://schema.org', '@id': 'htt...   \n",
       "352       neurovault.image  [{'@context': 'https://schema.org', '@type': '...   \n",
       "435                rgd.qtl  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "436             rgd.strain  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "441          iuphar.ligand  [{'@context': 'http://schema.org', '@id': 'htt...   \n",
       "472               genewiki  [{'@context': 'https://schema.org', '@type': '...   \n",
       "549      metanetx.chemical  [{'@context': 'https://schema.org', '@type': '...   \n",
       "550      metanetx.reaction  [{'@context': 'https://schema.org', '@type': '...   \n",
       "551   metanetx.compartment  [{'@context': 'https://schema.org', '@type': '...   \n",
       "579            cellosaurus  [{'@context': ['https://schema.org/'], '@type'...   \n",
       "582                 cosmic  [{'@context': 'http://schema.org', '@type': 'D...   \n",
       "614  neurovault.collection  [{'@context': 'https://schema.org', '@type': '...   \n",
       "638                 kaggle  [{'@context': 'http://schema.org/', '@type': '...   \n",
       "642                 encode  [{'documents': ['/documents/0994a79b-44ae-49ef...   \n",
       "654                hpscreg  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "661                    lei  [{'@context': 'http://schema.org', '@type': 'W...   \n",
       "\n",
       "       metadata_type                                         properties  \n",
       "1         DataRecord  [identifier, @context, @type, mainEntity, isPa...  \n",
       "16           WebSite            [@context, @type, url, potentialAction]  \n",
       "29      Organization  [@context, @type, url, name, logo, memberof, p...  \n",
       "43           Dataset  [@context, @type, name, description, url, keyw...  \n",
       "54           Dataset  [@context, @type, name, description, url, keyw...  \n",
       "80   MolecularEntity  [identifier, inChIKey, @context, biochemicalIn...  \n",
       "158          Dataset  [@context, @type, name, description, version, ...  \n",
       "195      DataCatalog  [@context, @type, @id, name, description, url,...  \n",
       "206          WebSite           [@context, @type, url, mainEntityOfPage]  \n",
       "232             None                                 [@context, @graph]  \n",
       "236     CreativeWork  [@context, @type, @id, additionalType, name, h...  \n",
       "253          Dataset  [@context, @type, name, description, identifie...  \n",
       "281     Organization  [@context, @type, url, name, logo, memberof, p...  \n",
       "298          Product  [@context, @type, name, Category, additionalPr...  \n",
       "301             None                                 [@context, @graph]  \n",
       "316      DataCatalog  [@context, @type, @id, name, description, url,...  \n",
       "336       DataRecord  [@context, @type, @id, identifier, dateCreated...  \n",
       "350          Dataset  [@context, @id, @type, alternateName, citation...  \n",
       "352     Organization         [@context, @type, name, url, logo, sameAs]  \n",
       "435          Dataset  [@context, @type, name, description, url, keyw...  \n",
       "436          Dataset  [@context, @type, name, description, url, keyw...  \n",
       "441       DataRecord  [@context, @id, @type, dct:conformsTo, include...  \n",
       "472          Article  [@context, @type, name, url, sameAs, mainEntit...  \n",
       "549      DataCatalog  [@context, @type, description, keywords, name,...  \n",
       "550      DataCatalog  [@context, @type, description, keywords, name,...  \n",
       "551      DataCatalog  [@context, @type, description, keywords, name,...  \n",
       "579     Organization  [@context, @type, url, name, memberof, parentO...  \n",
       "582          Dataset  [@context, @type, name, description, version, ...  \n",
       "614     Organization         [@context, @type, name, url, logo, sameAs]  \n",
       "638          Dataset  [@context, @type, name, description, url, same...  \n",
       "642       Experiment  [documents, hub, replication_type, revoked_fil...  \n",
       "654          WebSite            [@context, @type, url, potentialAction]  \n",
       "661          WebSite  [@context, @type, name, alternateName, url, po...  "
      ]
     },
     "execution_count": 214,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def get_metadata_properties(metadata):\n",
    "    if len(metadata) == 0:\n",
    "        return [None]\n",
    "\n",
    "    # flatten lists of lists... :(\n",
    "    if isinstance(metadata[0], list):\n",
    "        metadata[0] = metadata[0][0]\n",
    "    \n",
    "    result = [x[0] for x in metadata[0].items()]\n",
    "\n",
    "    return result\n",
    "\n",
    "df[\"properties\"] = df[\"metadata\"].map(lambda x: get_metadata_properties(x))\n",
    "df[df[\"metadata\"].map(len) != 0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>prefix</th>\n",
       "      <th>metadata</th>\n",
       "      <th>metadata_type</th>\n",
       "      <th>properties</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chebi</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ensembl</td>\n",
       "      <td>[{'identifier': 'ENSG00000139618', '@context':...</td>\n",
       "      <td>DataRecord</td>\n",
       "      <td>identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ensembl</td>\n",
       "      <td>[{'identifier': 'ENSG00000139618', '@context':...</td>\n",
       "      <td>DataRecord</td>\n",
       "      <td>@context</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ensembl</td>\n",
       "      <td>[{'identifier': 'ENSG00000139618', '@context':...</td>\n",
       "      <td>DataRecord</td>\n",
       "      <td>@type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ensembl</td>\n",
       "      <td>[{'identifier': 'ENSG00000139618', '@context':...</td>\n",
       "      <td>DataRecord</td>\n",
       "      <td>mainEntity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>679</th>\n",
       "      <td>cameo</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>680</th>\n",
       "      <td>nemo</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>681</th>\n",
       "      <td>insdc.gca</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>682</th>\n",
       "      <td>rrid</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>683</th>\n",
       "      <td>spp</td>\n",
       "      <td>[]</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>985 rows √ó 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        prefix                                           metadata  \\\n",
       "0        chebi                                                 []   \n",
       "1      ensembl  [{'identifier': 'ENSG00000139618', '@context':...   \n",
       "1      ensembl  [{'identifier': 'ENSG00000139618', '@context':...   \n",
       "1      ensembl  [{'identifier': 'ENSG00000139618', '@context':...   \n",
       "1      ensembl  [{'identifier': 'ENSG00000139618', '@context':...   \n",
       "..         ...                                                ...   \n",
       "679      cameo                                                 []   \n",
       "680       nemo                                                 []   \n",
       "681  insdc.gca                                                 []   \n",
       "682       rrid                                                 []   \n",
       "683        spp                                                 []   \n",
       "\n",
       "    metadata_type  properties  \n",
       "0            None        None  \n",
       "1      DataRecord  identifier  \n",
       "1      DataRecord    @context  \n",
       "1      DataRecord       @type  \n",
       "1      DataRecord  mainEntity  \n",
       "..            ...         ...  \n",
       "679          None        None  \n",
       "680          None        None  \n",
       "681          None        None  \n",
       "682          None        None  \n",
       "683          None        None  \n",
       "\n",
       "[985 rows x 4 columns]"
      ]
     },
     "execution_count": 216,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "def unnest(df, col, reset_index=False):\n",
    "    col_flat = pd.DataFrame([[i, x] for i, y in df[col].apply(list).iteritems() for x in y], columns=['I', col])\n",
    "    col_flat = col_flat.set_index('I')\n",
    "    df = df.drop(col, 1)\n",
    "    df = df.merge(col_flat, left_index=True, right_index=True)\n",
    "    if reset_index:\n",
    "        df = df.reset_index(drop=True)\n",
    "    return df\n",
    "\n",
    "unnested_df = unnest(df, \"properties\")\n",
    "unnested_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 249,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Properties for schema type Article\n",
      "publisher        1\n",
      "url              1\n",
      "author           1\n",
      "headline         1\n",
      "@context         1\n",
      "datePublished    1\n",
      "image            1\n",
      "sameAs           1\n",
      "@type            1\n",
      "dateModified     1\n",
      "mainEntity       1\n",
      "name             1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type CreativeWork\n",
      "publisher           1\n",
      "url                 1\n",
      "@context            1\n",
      "inLanguage          1\n",
      "name                1\n",
      "mainEntityOfPage    1\n",
      "identifier          1\n",
      "headline            1\n",
      "author              1\n",
      "description         1\n",
      "datePublished       1\n",
      "additionalType      1\n",
      "@type               1\n",
      "@id                 1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type DataCatalog\n",
      "@context              5\n",
      "dataset               5\n",
      "name                  5\n",
      "@type                 5\n",
      "description           5\n",
      "license               5\n",
      "provider              5\n",
      "url                   5\n",
      "keywords              5\n",
      "citation              3\n",
      "fileFormat            3\n",
      "dateCreated           3\n",
      "publication           2\n",
      "encodingFormat        2\n",
      "datePublished         2\n",
      "sourceOrganization    2\n",
      "dateModified          2\n",
      "identifier            2\n",
      "@id                   2\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type DataRecord\n",
      "@context             3\n",
      "@type                3\n",
      "mainEntity           3\n",
      "isPartOf             2\n",
      "identifier           2\n",
      "@id                  2\n",
      "dateCreated          1\n",
      "dct:conformsTo       1\n",
      "dateModified         1\n",
      "includedInDataset    1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type Dataset\n",
      "@context                 9\n",
      "name                     9\n",
      "@type                    9\n",
      "description              9\n",
      "license                  8\n",
      "version                  6\n",
      "url                      6\n",
      "creator                  5\n",
      "distribution             5\n",
      "keywords                 5\n",
      "datePublished            4\n",
      "identifier               4\n",
      "includedInDataCatalog    4\n",
      "sourceOrganization       3\n",
      "citation                 2\n",
      "alternateName            2\n",
      "thumbnailUrl             1\n",
      "about                    1\n",
      "interactionStatistic     1\n",
      "dateModified             1\n",
      "isAccessibleForFree      1\n",
      "@id                      1\n",
      "commentCount             1\n",
      "sameAs                   1\n",
      "discussionUrl            1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type Experiment\n",
      "dbxrefs                 1\n",
      "original_files          1\n",
      "date_released           1\n",
      "lab                     1\n",
      "date_created            1\n",
      "schema_version          1\n",
      "alternate_accessions    1\n",
      "@type                   1\n",
      "description             1\n",
      "assay_term_id           1\n",
      "date_submitted          1\n",
      "biosample_summary       1\n",
      "assay_slims             1\n",
      "uuid                    1\n",
      "biosample_ontology      1\n",
      "possible_controls       1\n",
      "category_slims          1\n",
      "replication_type        1\n",
      "internal_tags           1\n",
      "assay_term_name         1\n",
      "related_series          1\n",
      "award                   1\n",
      "superseded_by           1\n",
      "@id                     1\n",
      "contributing_files      1\n",
      "type_slims              1\n",
      "objective_slims         1\n",
      "references              1\n",
      "revoked_files           1\n",
      "aliases                 1\n",
      "@context                1\n",
      "replicates              1\n",
      "related_files           1\n",
      "visualize               1\n",
      "target                  1\n",
      "hub                     1\n",
      "assay_title             1\n",
      "internal_status         1\n",
      "supersedes              1\n",
      "documents               1\n",
      "audit                   1\n",
      "assembly                1\n",
      "status                  1\n",
      "files                   1\n",
      "submitter_comment       1\n",
      "accession               1\n",
      "submitted_by            1\n",
      "month_released          1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type MolecularEntity\n",
      "biochemicalInteraction         1\n",
      "image                          1\n",
      "url                            1\n",
      "molecularFormula               1\n",
      "@context                       1\n",
      "inChI                          1\n",
      "monoisotopicMolecularWeight    1\n",
      "@type                          1\n",
      "inChIKey                       1\n",
      "canonical_smiles               1\n",
      "identifier                     1\n",
      "molecularWeight                1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type Organization\n",
      "url                   5\n",
      "@context              5\n",
      "name                  5\n",
      "@type                 5\n",
      "logo                  4\n",
      "memberof              3\n",
      "parentOrganization    3\n",
      "sameAs                2\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type Product\n",
      "image                 1\n",
      "Category              1\n",
      "description           1\n",
      "@context              1\n",
      "name                  1\n",
      "@type                 1\n",
      "additionalProperty    1\n",
      "brand                 1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n",
      "Properties for schema type WebSite\n",
      "url                 4\n",
      "@context            4\n",
      "@type               4\n",
      "potentialAction     3\n",
      "name                1\n",
      "mainEntityOfPage    1\n",
      "alternateName       1\n",
      "Name: properties, dtype: int64\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "df_grouped_by_metadata_type = [x for _, x in unnested_df.groupby('metadata_type')]\n",
    "\n",
    "for group_df in df_grouped_by_metadata_type:\n",
    "    print(f\"Properties for schema type {group_df['metadata_type'].values[0]}\")\n",
    "    print(group_df[\"properties\"].value_counts())\n",
    "    print('\\n\\n')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@context                       33\n",
      "@type                          31\n",
      "name                           23\n",
      "url                            23\n",
      "description                    17\n",
      "license                        13\n",
      "identifier                     10\n",
      "keywords                       10\n",
      "datePublished                   8\n",
      "@id                             7\n",
      "version                         6\n",
      "creator                         5\n",
      "dataset                         5\n",
      "sourceOrganization              5\n",
      "dateModified                    5\n",
      "citation                        5\n",
      "provider                        5\n",
      "distribution                    5\n",
      "includedInDataCatalog           4\n",
      "dateCreated                     4\n",
      "mainEntity                      4\n",
      "sameAs                          4\n",
      "logo                            4\n",
      "fileFormat                      3\n",
      "alternateName                   3\n",
      "potentialAction                 3\n",
      "parentOrganization              3\n",
      "image                           3\n",
      "memberof                        3\n",
      "mainEntityOfPage                2\n",
      "publisher                       2\n",
      "@graph                          2\n",
      "encodingFormat                  2\n",
      "author                          2\n",
      "isPartOf                        2\n",
      "publication                     2\n",
      "headline                        2\n",
      "Category                        1\n",
      "category_slims                  1\n",
      "supersedes                      1\n",
      "lab                             1\n",
      "contributing_files              1\n",
      "isAccessibleForFree             1\n",
      "target                          1\n",
      "includedInDataset               1\n",
      "additionalProperty              1\n",
      "canonical_smiles                1\n",
      "accession                       1\n",
      "related_series                  1\n",
      "additionalType                  1\n",
      "date_released                   1\n",
      "assembly                        1\n",
      "audit                           1\n",
      "internal_status                 1\n",
      "uuid                            1\n",
      "dct:conformsTo                  1\n",
      "brand                           1\n",
      "hub                             1\n",
      "schema_version                  1\n",
      "visualize                       1\n",
      "revoked_files                   1\n",
      "thumbnailUrl                    1\n",
      "biosample_ontology              1\n",
      "molecularFormula                1\n",
      "assay_title                     1\n",
      "monoisotopicMolecularWeight     1\n",
      "award                           1\n",
      "references                      1\n",
      "date_created                    1\n",
      "assay_slims                     1\n",
      "inChIKey                        1\n",
      "assay_term_name                 1\n",
      "assay_term_id                   1\n",
      "inChI                           1\n",
      "interactionStatistic            1\n",
      "type_slims                      1\n",
      "possible_controls               1\n",
      "dbxrefs                         1\n",
      "aliases                         1\n",
      "original_files                  1\n",
      "month_released                  1\n",
      "inLanguage                      1\n",
      "discussionUrl                   1\n",
      "superseded_by                   1\n",
      "alternate_accessions            1\n",
      "replicates                      1\n",
      "date_submitted                  1\n",
      "related_files                   1\n",
      "about                           1\n",
      "objective_slims                 1\n",
      "biosample_summary               1\n",
      "submitted_by                    1\n",
      "status                          1\n",
      "commentCount                    1\n",
      "submitter_comment               1\n",
      "internal_tags                   1\n",
      "documents                       1\n",
      "biochemicalInteraction          1\n",
      "files                           1\n",
      "replication_type                1\n",
      "molecularWeight                 1\n",
      "Name: properties, dtype: int64\n"
     ]
    }
   ],
   "source": [
    "pd.set_option('display.max_rows', None)\n",
    "\n",
    "print(unnested_df.properties.value_counts())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Metadata service accuracy\n",
    "\n",
    "Cannot check. Google pagespeed insights do not show if metadata is valid."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 275,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "string indices must be integers",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-275-b8669ccce400>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mnamespace\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mempty_metadata_resource_urls\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m16\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;36m17\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 5\u001b[0;31m     \u001b[0;32mif\u001b[0m \u001b[0mnamespace\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'namespaceEmbeddedInLui'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      6\u001b[0m         \u001b[0mprefix\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnamespace\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'pattern'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m':'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: string indices must be integers"
     ]
    }
   ],
   "source": [
    "resolver_service_url = \"http://resolver.api.identifiers.org\"\n",
    "pagespeed_apikey = \"\" # input api key for google pagespeed\n",
    "\n",
    "for namespace in dataset[16:17]:\n",
    "    if namespace['namespaceEmbeddedInLui']:\n",
    "        prefix = namespace['pattern'][1:].split(':')[0]\n",
    "    else:\n",
    "        prefix = namespace['prefix']\n",
    "\n",
    "    response = requests.get(f\"{resolver_service_url}/{prefix}:{namespace['sampleId']}\").json()\n",
    "    \n",
    "    namespace_provider_urls = []\n",
    "    \n",
    "    if response[\"payload\"] is not None and response[\"payload\"][\"resolvedResources\"] is not None:\n",
    "        for resolvedResource in response[\"payload\"][\"resolvedResources\"]:\n",
    "            namespace_provider_urls.append(resolvedResource[\"compactIdentifierResolvedUrl\"])\n",
    "    \n",
    "    print(namespace_provider_urls)\n",
    "    \n",
    "    for url in namespace_provider_urls:\n",
    "        pagespeed_url = f\"https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url={url}&{pagespeed_apikey}\"\n",
    "        \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 274,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CHEBI:36927',\n",
       " 'uniprot:P0DP23',\n",
       " 'ec-code:1.1.1.1',\n",
       " 'miriam.collection:MIR:00000008',\n",
       " 'taxonomy:9606',\n",
       " 'biomodels.db:BIOMD0000000048',\n",
       " 'intact:EBI-2307691',\n",
       " 'interpro:IPR000100',\n",
       " 'kegg.pathway:hsa00620',\n",
       " 'icd:C34',\n",
       " 'pirsf:PIRSF000100',\n",
       " 'pubmed:16333295',\n",
       " 'mim:603903',\n",
       " 'kegg.compound:C12345',\n",
       " 'kegg.reaction:R00100',\n",
       " 'doi:10.1038/nbt1156',\n",
       " 'pdb:2gc4',\n",
       " 'GO:0006915',\n",
       " 'kegg.drug:D00123',\n",
       " 'sgd:S000003909',\n",
       " 'SBO:0000262',\n",
       " 'kegg.glycan:G00123',\n",
       " 'wb:WBGene00000001',\n",
       " 'pfam:PF01234',\n",
       " 'insdc:X58356',\n",
       " 'fb:FBgn0011293',\n",
       " 'pubchem.compound:100101',\n",
       " 'arxiv:0807.4956v1',\n",
       " 'wormpep:CE28239',\n",
       " 'pubchem.substance:100101',\n",
       " 'sabiork.reaction:75',\n",
       " 'arrayexpress:E-MEXP-1712',\n",
       " 'MGI:2442292',\n",
       " 'refseq:NP_012345',\n",
       " 'tcdb:5.A.1.1.1',\n",
       " 'uniparc:UPI000000000A',\n",
       " 'mint:MINT-10000',\n",
       " 'signaling-gateway:A001094',\n",
       " 'dip:DIP-743N',\n",
       " 'resid:AA0001',\n",
       " 'tair.protein:AASequence:1009107926',\n",
       " 'tair.locus:2200950',\n",
       " 'tair.gene:Gene:2200934',\n",
       " 'peptideatlas:PAp00000009',\n",
       " 'lipidmaps:LMPR0102010012',\n",
       " 'geo:GDS1234',\n",
       " 'hmdb:HMDB00001',\n",
       " 'sgd.pathways:PWY3O-214',\n",
       " 'MOD:00001',\n",
       " 'merops:S01.001',\n",
       " 'panther.family:PTHR12345',\n",
       " 'prints:PR00001',\n",
       " 'ligandexpo:ABC',\n",
       " 'mge:2',\n",
       " 'pride:1',\n",
       " '3dmet:B00162',\n",
       " 'isbn:9781584885658',\n",
       " 'kegg.genes:syn:ssr3451',\n",
       " 'brenda:1.1.1.1',\n",
       " 'FMA:67112',\n",
       " 'matrixdb.association:P00747__P07355',\n",
       " 'ncbigene:100010',\n",
       " 'pubchem.bioassay:1018',\n",
       " 'bao:0002989',\n",
       " 'pathwaycommons:485991',\n",
       " 'hovergen:HBG004341',\n",
       " 'wikipathways:WP100',\n",
       " 'macie:M0001',\n",
       " 'mirbase:MI0000001',\n",
       " 'hgnc:2674',\n",
       " 'zfin:ZDB-GENE-041118-11',\n",
       " 'rhea:12345',\n",
       " 'SO:0000704',\n",
       " 'chembl.target:CHEMBL3467',\n",
       " 'sabiork.kineticrecord:5046',\n",
       " 'atc:A10BA02',\n",
       " 'pharmgkb.pathways:PA146123006',\n",
       " 'pharmgkb.drug:PA448710',\n",
       " 'pharmgkb.disease:PA447218',\n",
       " 'ttd.target:TTDS00056',\n",
       " 'ttd.drug:DAP000773',\n",
       " 'lgic:5HT3Arano',\n",
       " 'neuromorpho:Rosa2',\n",
       " 'chemidplus:57-27-2',\n",
       " 'ctd.chemical:D001151',\n",
       " 'biosystems:001',\n",
       " 'neurondb:265',\n",
       " 'ctd.gene:101',\n",
       " 'ctd.disease:D053716',\n",
       " 'bionumbers:104674',\n",
       " 'drugbank:DB00001',\n",
       " 't3db:T3D0001',\n",
       " 'smpdb:SMP00001',\n",
       " 'genedb:LinJ.20.0070',\n",
       " 'biomodels.teddy:TEDDY_0000066',\n",
       " 'phosphosite.protein:12300',\n",
       " 'CL:0000232',\n",
       " 'BTO:0000146',\n",
       " 'biomodels.kisao:KISAO_0000057',\n",
       " 'pdb-ccd:AB0',\n",
       " 'PATO:0001998',\n",
       " 'glycomedb:G77500AY',\n",
       " 'lipidbank:BBA0001',\n",
       " 'smart:SM00015',\n",
       " 'cdd:cd00400',\n",
       " 'prodom:PD10000',\n",
       " 'kegg.orthology:K00001',\n",
       " 'mmdb:50885',\n",
       " 'imex:IM-19210-3',\n",
       " 'ro:RO_0002533',\n",
       " 'irefweb:617102',\n",
       " 'mpid:172',\n",
       " 'phosphosite.residue:2842',\n",
       " 'neurolex:Birnlex_721',\n",
       " 'asap:ABE-0009634',\n",
       " 'sabiork.ec:2.7.1.1',\n",
       " 'opb:OPB_00573',\n",
       " 'obi:OBI_0000070',\n",
       " 'jws:curien',\n",
       " 'modeldb:45539',\n",
       " 'subtiwiki:BSU29180',\n",
       " 'doqcs.pathway:131',\n",
       " 'doqcs.model:57',\n",
       " 'UO:0000080',\n",
       " 'pid.pathway:pi3kcipathway',\n",
       " 'clinicaltrials:NCT00222573',\n",
       " 'chemspider:56586',\n",
       " 'omia:1000',\n",
       " 'cgd:CAL0003079',\n",
       " 'csa:1a05',\n",
       " 'ncit:C80519',\n",
       " 'antweb:casent0106247',\n",
       " 'biocatalogue.service:614',\n",
       " 'pmc:PMC3084216',\n",
       " 'amoebadb:EDI_244000',\n",
       " 'atcc:11303',\n",
       " 'microsporidia:ECU03_0820i',\n",
       " 'plasmodb:PF11_0344',\n",
       " 'cryptodb:cgd7_230',\n",
       " 'giardiadb:GL50803_102438',\n",
       " 'trichdb:TVAG_386080',\n",
       " 'beetlebase:TC010103',\n",
       " 'bdgp.insertion:KG09531',\n",
       " 'tritrypdb:Tb927.8.620',\n",
       " 'toxoplasma:TGME49_053730',\n",
       " 'bold.taxonomy:27267',\n",
       " 'dbprobe:1000000',\n",
       " 'unists:456789',\n",
       " 'dbest:BP100000',\n",
       " 'ecogene:EG10173',\n",
       " 'greengenes:100000',\n",
       " 'bdgp.est:EY223054.1',\n",
       " 'grin.taxonomy:19333',\n",
       " 'gabi:2679240',\n",
       " 'hinv.locus:HIX0004394',\n",
       " 'hinv.transcript:HIT000195363',\n",
       " 'homd.seq:SEQF1003',\n",
       " 'hinv.protein:HIP000030660',\n",
       " 'homd.taxon:811',\n",
       " 'isfinder:ISA1083-2',\n",
       " 'ird.segment:CY077097',\n",
       " 'jcm:17254',\n",
       " 'img.taxon:648028003',\n",
       " 'mycobank:349124',\n",
       " 'maizegdb.locus:25011',\n",
       " 'img.gene:638309541',\n",
       " 'pseudomonas:PSEEN0001',\n",
       " 'nbrc:00001234',\n",
       " 'gramene.protein:78073',\n",
       " 'gramene.gene:GR:0080039',\n",
       " 'gramene.taxonomy:GR_tax:013681',\n",
       " 'gramene.qtl:CQG5',\n",
       " 'xenbase:922462',\n",
       " 'sgn:0001',\n",
       " 'edam:data_1664',\n",
       " 'bioportal:1046',\n",
       " 'miriam.resource:MIR:00100005',\n",
       " 'arachnoserver:AS000060',\n",
       " 'pmdb:PM0012345',\n",
       " 'biocyc:ECOLI:CYT-D-UBIOX-CPLX',\n",
       " 'paleodb:83088',\n",
       " 'cazy:GT10',\n",
       " 'goa:P12345',\n",
       " 'compulyeast:O08709',\n",
       " 'eggnog:veNOG12876',\n",
       " 'echobase:EB0170',\n",
       " 'ensembl.bacteria:MU9_3181',\n",
       " 'ensembl.protist:PF3D7_1328700',\n",
       " 'ensembl.metazoa:FBtr0084214',\n",
       " 'hcvdb:M58335',\n",
       " 'ensembl.plant:AT1G73965',\n",
       " 'ensembl.fungi:CADAFLAT00006211',\n",
       " 'genatlas:HBB',\n",
       " 'cath.superfamily:1.10.10.200',\n",
       " 'genefarm:4892',\n",
       " 'hogenom:HBG284870',\n",
       " 'gpcrdb:RL3R1_HUMAN',\n",
       " 'hssp:102l',\n",
       " 'genetree:ENSGT00550000074763',\n",
       " 'myco.tuber:Rv1908c',\n",
       " 'myco.marinum:MMAR_2462',\n",
       " 'myco.smeg:MSMEG_3769',\n",
       " 'orthodb:Q9P0K8',\n",
       " 'myco.lepra:ML0224',\n",
       " 'orphanet:85163',\n",
       " 'peroxibase:5282',\n",
       " 'pmap.substratedb:1915',\n",
       " 'phylomedb:Phy000CLXM_RAT',\n",
       " 'pmap.cutdb:25782',\n",
       " 'protclustdb:O80725',\n",
       " 'pmp:Q0VCA6',\n",
       " 'protonet.cluster:4349895',\n",
       " 'protonet.proteincard:16941567',\n",
       " 'swiss-model:P23298',\n",
       " 'rebase:101',\n",
       " 'vectorbase:ISCW007415',\n",
       " 'mirbase.mature:MIMAT0000001',\n",
       " 'cas:50-00-0',\n",
       " 'kegg.metagenome:T30002',\n",
       " 'kegg.genome:eco',\n",
       " 'jcsd:J55.713G',\n",
       " 'pharmgkb.gene:PA131',\n",
       " 'scretf:RSC3',\n",
       " 'insdc.sra:SRX000007',\n",
       " 'PW:0000208',\n",
       " 'napp:351',\n",
       " 'mirnest:MNEST029358',\n",
       " 'noncodev3:377550',\n",
       " 'virsirna:virsi1909',\n",
       " 'elm:CLV_MEL_PAP_1',\n",
       " 'mimodb:1',\n",
       " 'bykdb:A0A009E7X8',\n",
       " 'sitex:1000',\n",
       " 'topfind:Q9UKQ2',\n",
       " 'combine.specifications:sbgn.er.level-1.version-1.2',\n",
       " 'mipmod:HOSAPI0399',\n",
       " 'conoserver:2639',\n",
       " 'huge:KIAA0001',\n",
       " 'cabri:dsmz_mutz-id:ACC 291',\n",
       " 'dictybase.gene:DDB_G0267522',\n",
       " 'string:P53350',\n",
       " 'bindingdb:e999',\n",
       " 'phenolexplorer:75',\n",
       " 'stitch:BQJCRHHNABKAKU',\n",
       " 'atcvet:QJ51RV02',\n",
       " 'snomedct:284196006',\n",
       " 'mesh.2012:17186',\n",
       " 'knapsack:C00000001',\n",
       " 'massbank:PB000166',\n",
       " 'cdpd:63250',\n",
       " 'gmd:68513255-fc44-4041-bc4b-4fd2fae7541d',\n",
       " 'homologene:1000',\n",
       " 'abs:A0014',\n",
       " 'umbbd.compound:c0001',\n",
       " 'dpv:100',\n",
       " 'apd:01001',\n",
       " 'chemdb:3966782',\n",
       " 'aceview.worm:aap-1',\n",
       " 'imgt.ligm:M94112',\n",
       " 'iuphar.receptor:101',\n",
       " 'worfdb:T01B6.1',\n",
       " 'soybase:BARC-013845-01256',\n",
       " 'nextdb:6b1',\n",
       " 'cgsc:74',\n",
       " 'arrayexpress.platform:A-GEOD-50',\n",
       " 'dragondb.dna:3hB06',\n",
       " 'rouge:mKIAA4200',\n",
       " 'merops.family:S1',\n",
       " 'dragondb.locus:DEF',\n",
       " 'dragondb.allele:cho',\n",
       " 'dragondb.protein:AMDEFA',\n",
       " 'issn:0745-4570',\n",
       " 'niaest:J0705A10',\n",
       " 'nasc:N1899',\n",
       " 'pazar:TF0001053',\n",
       " 'mo:ArrayGroup',\n",
       " 'rnamods:101',\n",
       " 'PO:0009089',\n",
       " 'tgd:TTHERM_00648910',\n",
       " 'treebase:TB2:S1000',\n",
       " 'tigrfam:TIGR00010',\n",
       " 'sdbs:4544',\n",
       " 'dbg2introns:Cu.me.I1',\n",
       " 'iuphar.family:78',\n",
       " 'vbase2:humIGHV025',\n",
       " 'genecards:ABL1',\n",
       " 'spike.map:spike00001',\n",
       " 'metlin:1455',\n",
       " 'umbbd.enzyme:e0333',\n",
       " 'mmrrc:70',\n",
       " 'umbbd.reaction:r0001',\n",
       " 'umbbd.pathway:ala',\n",
       " 'dictybase.est:DDB0016567',\n",
       " 'umbbd.rule:bt0001',\n",
       " 'mirex:165a',\n",
       " 'imgt.hla:A*01:01:01:01',\n",
       " 'bdsc:33607',\n",
       " 'opm:1h68',\n",
       " 'allergome:1948',\n",
       " 'pombase:SPCC13B11.01',\n",
       " 'hpa:ENSG00000026508',\n",
       " 'jaxmice:005012',\n",
       " 'ubio.namebank:2555646',\n",
       " 'yetfasco:YOR172W_571.0',\n",
       " 'tarbase:hsa-let-7a-2-3p',\n",
       " 'charprot:CH_001923',\n",
       " 'oma.protein:HUMAN16963',\n",
       " 'genpept:CAA71118.1',\n",
       " 'oma.grp:LCSCCPN',\n",
       " 'unigene:4900',\n",
       " 'ncbiprotein:CAA71118.1',\n",
       " 'bitterdb.rec:1',\n",
       " 'bitterdb.cpd:46',\n",
       " 'bioproject:PRJDB3',\n",
       " 'piroplasma:TA14985',\n",
       " 'unite:UDB000691',\n",
       " 'ncim:C0026339',\n",
       " 'proglyc:AC119',\n",
       " 'polbase:19-T4',\n",
       " 'nuclearbd:prgr_human',\n",
       " 'supfam:SSF57615',\n",
       " 'pina:Q13485',\n",
       " 'bacmap.biog:1050',\n",
       " 'ricegap:LOC_Os02g13300',\n",
       " 'tissuelist:TS-0285',\n",
       " 'hgnc.symbol:DAPK1',\n",
       " 'panther.pathway:P00024',\n",
       " 'fungidb:CNBG_0001',\n",
       " 'drsc:DRSC05221',\n",
       " 'darc:1250',\n",
       " 'oridb.sacch:1',\n",
       " 'oridb.schizo:1',\n",
       " 'scop:47419',\n",
       " 'ena.embl:BN000065',\n",
       " 'pscdb:051',\n",
       " 'panther.node:PTN000000026',\n",
       " 'dommino:2GC4',\n",
       " 'ccds:CCDS13573.1',\n",
       " 'gxa.gene:AT4G01080',\n",
       " 'gxa.expt:E-MTAB-2037',\n",
       " 'hprd:00001',\n",
       " 'lrg:LRG_1',\n",
       " 'metabolights:MTBLS1',\n",
       " 'orcid:0000-0002-5355-2576',\n",
       " 'nbn:urn:nbn:fi:tkk-004781',\n",
       " 'phosphopoint.kinase:AURKA',\n",
       " 'uniprot.isoform:Q5BJF6-3',\n",
       " 'phosphopoint.protein:AURKA',\n",
       " 'inchikey:RYYVLZVUVIJVGH-UHFFFAOYSA-N',\n",
       " 'inchi:InChI=1S/C2H6O/c1-2-3/h3H,2H2,1H3',\n",
       " 'wikipedia.en:SM_UB-81',\n",
       " 'cldb:cl3603',\n",
       " 'kegg.environ:ev:E00032',\n",
       " 'aphidbase.transcript:ACYPI000159',\n",
       " 'efo:0004859',\n",
       " 'hgmd:CALM1',\n",
       " 'affy.probeset:243002_at',\n",
       " 'treefam:TF101014',\n",
       " 'caps:434',\n",
       " 'cubedb:AKR',\n",
       " 'ideal:IID00001',\n",
       " 'pocketome:1433C_TOBAC_1_252',\n",
       " 'stap:1a24',\n",
       " 'gold.meta:Gm00047',\n",
       " 'bugbase.expt:288',\n",
       " 'bugbase.protocol:67',\n",
       " 'VariO:0294',\n",
       " 'tol:98034',\n",
       " 'vipr:BeAn 70563',\n",
       " 'fbol:2224',\n",
       " 'rfam:RF00230',\n",
       " 'epd:TA_H3',\n",
       " 'aftol.taxonomy:959',\n",
       " 'aspgd.locus:ASPL0000349247',\n",
       " 'aspgd.protein:ASPL0000349247',\n",
       " 'autdb:ADA',\n",
       " 'bacmap.map:AP011135',\n",
       " 'ndc:0002-1975-61',\n",
       " 'bgee.gene:FBgn0000015',\n",
       " 'bgee.stage:HsapDO:0000004',\n",
       " 'bgee.family:ENSFM00500000270089',\n",
       " 'bgee.organ:EHDAA:2185',\n",
       " 'biocarta.pathway:h_aktPathway',\n",
       " 'gmd.gcms:53d583d8-40c6-40e1-9296-23f821cd77a5',\n",
       " 'gmd.profile:10b38aaf-b977-4950-85b8-f4775f66658d',\n",
       " 'gmd.ref:8cf84adb-b4db-4807-ac98-0004247c35df',\n",
       " 'gmd.analyte:4f0fa9b6-514f-4ff4-98cc-0009bc08eb80',\n",
       " 'intact.molecule:EBI-366083',\n",
       " 'dailymed:973a9333-fec7-46dd-8eb5-25738f06ee54',\n",
       " 'depod:PTPN1',\n",
       " 'phytozome.locus:Glyma0021s00410',\n",
       " 'cst:Akt_PKB',\n",
       " 'subtilist:BG11523',\n",
       " 'sider.effect:C0017565',\n",
       " 'sider.drug:2244',\n",
       " 'broad:S7000002168151102',\n",
       " 'coriell:GM17027',\n",
       " 'wikigenes:3771877',\n",
       " 'corum:100',\n",
       " 'ecoliwiki:aaeA',\n",
       " 'MA:0002502',\n",
       " 'jstor:3075966',\n",
       " 'genprop:GenProp0699',\n",
       " 'unimod:1200',\n",
       " 'vbrc:35742',\n",
       " 'viralzone:992',\n",
       " 'door:1398574',\n",
       " 'degradome:Ax1',\n",
       " 'datf:AT1G01030.1',\n",
       " 'dbd:0045310',\n",
       " 'molbase:128796-39-4',\n",
       " 'yrcpdr:2673500',\n",
       " 'funcbase.fly:10194',\n",
       " 'funcbase.human:119514',\n",
       " 'yid:SNR17A',\n",
       " 'funcbase.mouse:1351341',\n",
       " 'cbioportal:laml_tcga_pub',\n",
       " 'funcbase.yeast:2701',\n",
       " 'ydpm:YAL001C',\n",
       " 'wb.rnai:WBRNAi00086878',\n",
       " 'pass2:46977',\n",
       " 'iceberg.element:100',\n",
       " 'iceberg.family:1',\n",
       " 'vfdb.gene:VFG2154',\n",
       " 'vfdb.genus:Chlamydia',\n",
       " 'mesh.2013:17165',\n",
       " 'kegg.module:M00002',\n",
       " 'kegg.disease:H00076',\n",
       " 'ligandbox:D00001',\n",
       " 'glycoepitope:EP0311',\n",
       " 'medlineplus:002804',\n",
       " 'jcggdb:JCGG-STR008690',\n",
       " 'noncodev4.rna:NONHSAT000001',\n",
       " 'vfb:00000001',\n",
       " 'noncodev4.gene:NONHSAG00001',\n",
       " 'oryzabase.gene:117',\n",
       " 'oryzabase.mutant:21393',\n",
       " 'otl:AADB12',\n",
       " 'oryzabase.strain:1',\n",
       " 'oryzabase.stage:34',\n",
       " 'paxdb.protein:977869',\n",
       " 'pdb.ligand:TRQ',\n",
       " 'merops.inhibitor:I31.952',\n",
       " 'paxdb.organism:9606',\n",
       " 'glida.gpcr:ACM1_HUMAN',\n",
       " 'adw:Lycalopex_vetulus',\n",
       " 'grsdb:10142',\n",
       " 'glida.ligand:L000001',\n",
       " 'fsnp:rs17852708',\n",
       " 'px:PXD000500',\n",
       " 'hdr:63',\n",
       " 'biomodels.vocabulary:rateRule',\n",
       " 'norine:NOR00681',\n",
       " 'ordb:8497',\n",
       " 'p3db.site:65',\n",
       " 'odor:74',\n",
       " 'cattleqtldb:4685',\n",
       " 'p3db.protein:70',\n",
       " 'topdb:AP00378',\n",
       " 'chickenqtldb:14362',\n",
       " 'mgnify.samp:SRS086444',\n",
       " 'sheepqtldb:19803',\n",
       " 'ega.study:EGAS00000000001',\n",
       " 'pigqtldb:14',\n",
       " 'ega.dataset:EGAD00000000001',\n",
       " 'idot:identifierPattern',\n",
       " 'antibodyregistry:493771',\n",
       " 'mamo:MAMO_0000026',\n",
       " 'pride.project:PXD000440',\n",
       " 'yeastintron:SNR17A',\n",
       " 'hgnc.family:PADI',\n",
       " 'ardb:CAE46076',\n",
       " 'proteomicsdb.protein:53504',\n",
       " 'hpm.protein:1968',\n",
       " 'proteomicsdb.peptide:53504',\n",
       " 'drugbankv4.target:BE0000048',\n",
       " 'foodb.compound:FDB002100',\n",
       " 'hpm.peptide:9606117',\n",
       " 'zinc:ZINC1084',\n",
       " 'unii:3G6A5W338E',\n",
       " 'orphanet.ordo:Orphanet_C023',\n",
       " 'psipar:PAR:0116',\n",
       " 'mgnify.proj:ERP004492',\n",
       " 'clinvar.record:RCV000033555.3',\n",
       " 'euclinicaltrials:2008-005144-16',\n",
       " 'google.patent:US4145692',\n",
       " 'uspto:4145692',\n",
       " 'exac.variant:22-46615880-T-C',\n",
       " 'cpc:A01M1/026',\n",
       " 'gwascentral.study:HGVST1828',\n",
       " 'gwascentral.marker:HGVM15354',\n",
       " 'bigg.metabolite:12dgr161',\n",
       " 'lincs.cell:LCL-2085',\n",
       " 'lincs.protein:200282',\n",
       " 'lincs.smallmolecule:LSM-6306',\n",
       " 'exac.transcript:ENST00000407236',\n",
       " 'exac.gene:ENSG00000169174',\n",
       " 'wikidata:Q2207226',\n",
       " 'unipathway.compound:UPC04349',\n",
       " 'bigg.model:iECABU_c1320',\n",
       " 'seed:Biotin_biosynthesis',\n",
       " 'seed.compound:cpd15380',\n",
       " 'bigg.compartment:c',\n",
       " 'bigg.reaction:13GS',\n",
       " 'umls:C0021390',\n",
       " 'emdb:EMD-1001',\n",
       " 'mesh:C000100',\n",
       " 'mirtarbase:MIRT000002',\n",
       " 'meddra:10015919',\n",
       " 'dashr:hsa-mir-200a',\n",
       " 'splash:splash10-0zq2000000-77302b0326a418630a84',\n",
       " 'dashr.expression:hsa-mir-200a',\n",
       " 'unipathway.reaction:UCR00226',\n",
       " 'sasbdb:SASDAX8',\n",
       " 'HP:0000118',\n",
       " 'hgnc.genefamily:141',\n",
       " 'mdm:4776',\n",
       " 'apid.interactions:P01116',\n",
       " 'probonto:PROB_c0000005',\n",
       " 'storedb:STOREDB:STUDY1040',\n",
       " 'kegg:hsa00190',\n",
       " 'MP:0005452',\n",
       " 'planttfdb:Ath_AT1G01030.1',\n",
       " 'multicellds.collection:MCDS_C_0000000001',\n",
       " 'multicellds.cell_line:MCDS_L_0000000001',\n",
       " 'multicellds.snapshot:MCDS_S_0000000001',\n",
       " 'ecyano.entity:23',\n",
       " 'lincs.data:LDS-1110',\n",
       " 'ecyano.model:26',\n",
       " 'ecyano.rule:56',\n",
       " 'agricola:50018',\n",
       " 'MS:1000001',\n",
       " 'ENVO:09200010',\n",
       " 'grid:grid.225360.0',\n",
       " 'medgen:760050',\n",
       " 'clinvar:12345',\n",
       " 'clinvar.submission:SCV000151292',\n",
       " 'MIR:00100037',\n",
       " 'pgx:icdom:8500_3',\n",
       " 'iao:0000030',\n",
       " 'dbgap:phs000768.v2.p1',\n",
       " 'omit:0005506',\n",
       " 'insdc.cds:AAA35559',\n",
       " 'gdc:ae8c77fe-e6c8-44d5-8265-4a38c637bbef',\n",
       " 'metacyc.reaction:RXN-14904',\n",
       " 'glytoucan:G00054MO',\n",
       " 'metacyc.compound:CPD-10330',\n",
       " 'd1id:00030692-0FE1-4A1B-955E-A2E55D659267',\n",
       " 'mmp.ref:MMP3312982.2',\n",
       " 'cath:1.10.8.10',\n",
       " 'mmp.cat:MMP743597.11',\n",
       " 'mmp.db:MMP02954345.1',\n",
       " 'aop:98',\n",
       " 'biotools:uniprotkb',\n",
       " 'ngl:GLDS-141',\n",
       " 'arraymap:icdom:8500_3',\n",
       " 'begdb:4214',\n",
       " 'erv:THE1B',\n",
       " 'napdi:28',\n",
       " 'crisprdb:551115',\n",
       " 'mzspec:PXD002255::ES_XP_Ubi_97H_HCD_349:scan:9617:LAEIYVNSSFYK/2',\n",
       " 'sugarbind:lectins/172',\n",
       " 'sisu:17:41223048',\n",
       " 'licebase:EMLSAT00000003403',\n",
       " 'gnpis:AY109603',\n",
       " 'rnacentral:URS0000759CF4',\n",
       " 'microscope:5601141',\n",
       " 'swissregulon:AHR',\n",
       " 'validatordb:ByStructure/2h6o',\n",
       " 'biominder:aef4c195-9cf9-46db-a12a-7cfd1ff3eec3',\n",
       " 'vmhmetabolite:h2o',\n",
       " 'NMR:1000003',\n",
       " 'vmhreaction:HEX1',\n",
       " 'ascl:1801.012',\n",
       " 'facebase:FB00000917',\n",
       " 'eu89h:jrc-eurl-ecvam-chemagora',\n",
       " 'gudmap:Q-2958',\n",
       " 'minid:b97957',\n",
       " 'mex:36',\n",
       " 'mw.study:ST000900',\n",
       " 'gpmdb:GPM32310002988',\n",
       " 'mw.project:PR000001',\n",
       " 'massive:MSV000082131',\n",
       " 'fplx:RAS',\n",
       " 'aop.events:3',\n",
       " 'aop.stressor:9',\n",
       " 'aop.relationships:5',\n",
       " 'swh:1:rev:309cf2674ee7a0749978cf8265ab91a60aea0f7d',\n",
       " 'dev.ga4ghdos:23fa7b4b-9d68-429b-aece-658b11124bb3#2017-03-24T18:43:16.886826-05:00',\n",
       " 'comptox:DTXSID2021028',\n",
       " 'complexportal:CPX-2158',\n",
       " 'ligandbook:785',\n",
       " 'rbk:Q-2958',\n",
       " 'idoo:DataCollection',\n",
       " 'oci:1-18',\n",
       " 'FOODON:03307879',\n",
       " 'ricenetdb.gene:LOC_Os01g49190.1',\n",
       " 'ricenetdb.protein:LOC_Os01g49190',\n",
       " 'gtex:BRIP1',\n",
       " 'ricenetdb.compound:OSC1416',\n",
       " 'ricenetdb.reaction:OSR0818',\n",
       " 'ricenetdb.mirna:osa-miR446',\n",
       " 'asin:0471491039',\n",
       " 'oclc:634515043',\n",
       " 'ga4ghdos:dg.4503/01b048d0-e128-4cb0-94e9-b2d2cab7563d',\n",
       " 'oid:2.16.840',\n",
       " 'flowrepository:FR-FCM-ZYGW',\n",
       " 'addgene:50943',\n",
       " 'gdsc:1242',\n",
       " 'bacdive:131392',\n",
       " 'ido:0000504',\n",
       " 'isni:000000012281955X',\n",
       " 'viaf:75121530',\n",
       " 'sabiork.compound:75',\n",
       " 'spdx:CC-BY-1.0',\n",
       " 'peptideatlas.dataset:PASS01237',\n",
       " 'did:sov:WRfXPg8dantKVubE3HX8pw',\n",
       " 'oryzabase.reference:42840',\n",
       " 'biostudies:S-EPMC6266652',\n",
       " 'seed.reaction:rxn00001',\n",
       " 'occ:br/1',\n",
       " 'cadsr:3771992',\n",
       " 'erm:ERM00000044',\n",
       " 'classyfire:C0004828',\n",
       " 'gcst:GCST000035',\n",
       " 'mmp.fun:MMP3888430',\n",
       " 'ecyano.experiment:18',\n",
       " 'dlxb:6VDC956',\n",
       " 'dlxc:M77F7JM',\n",
       " 'cameo:2019-08-03_00000089_1',\n",
       " 'nemo:smp-m3w9hbe',\n",
       " 'insdc.gca:GCA_000155495.1',\n",
       " 'spp:10.1621/vwN2g2HaX3']"
      ]
     },
     "execution_count": 274,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "empty_metadata_resource_urls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 286,
   "metadata": {},
   "outputs": [],
   "source": [
    "namespace_provider_urls = []\n",
    "\n",
    "for ci in empty_metadata_resource_urls:\n",
    "    response = requests.get(f\"{resolver_service_url}/{ci}\").json()\n",
    "\n",
    "    if response[\"payload\"] is not None and response[\"payload\"][\"resolvedResources\"] is not None:\n",
    "        for resolvedResource in response[\"payload\"][\"resolvedResources\"]:\n",
    "            namespace_provider_urls.append(resolvedResource[\"compactIdentifierResolvedUrl\"])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 288,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fchebi%2FsearchId.do%3FchebiId%3DCHEBI%3A36927\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fchebi%2Fterms%3Fobo_id%3DCHEBI%3A36927\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FCHEBI%2FCHEBI%3A36927\n",
      "https%3A%2F%2Fpurl.uniprot.org%2Funiprot%2FP0DP23\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fprotein%2FP0DP23\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fintenz%2Fquery%3Fcmd%3DSearchEC%26ec%3D1.1.1.1\n",
      "https%3A%2F%2Fwww.genome.jp%2Fdbget-bin%2Fwww_bget%3Fec%3A1.1.1.1\n",
      "http%3A%2F%2Fwww.enzyme-database.org%2Fquery.php%3Fec%3D1.1.1.1\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fenzymeportal%2Fsearch%2Fec%2F1.1.1.1\n",
      "https%3A%2F%2Fenzyme.expasy.org%2FEC%2F1.1.1.1\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fmiriam%2Fmain%2FMIR%3A00000008\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2FTaxonomy%2FBrowser%2Fwwwtax.cgi%3Fmode%3DInfo%26id%3D9606\n",
      "https%3A%2F%2Fpurl.uniprot.org%2Ftaxonomy%2F9606\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FTaxon%3A9606\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FNCBITAXON%2F9606\n",
      "http%3A%2F%2Ftaxonomy.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Ftaxonomy%3A9606\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fncbitaxon%2Fterms%3Fshort_form%3DNCBITaxon_9606\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fbiomodels%2FBIOMD0000000048\n",
      "http%3A%2F%2Fbiomodels.caltech.edu%2FBIOMD0000000048\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fbiomodels%2FBIOMD0000000048\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fintact%2Finteraction%2FEBI-2307691\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Finterpro%2Fentry%2FIPR000100\n",
      "http%3A%2F%2Finterpro.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Finterpro%3AIPR000100\n",
      "https%3A%2F%2Fwww.kegg.jp%2Fentry%2Fhsa00620\n",
      "http%3A%2F%2Fapps.who.int%2Fclassifications%2Ficd10%2Fbrowse%2F2010%2Fen%23%2FC34\n",
      "https%3A%2F%2Fpir.georgetown.edu%2Fcgi-bin%2FipcSF%3Fid%3DPIRSF000100\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpubmed%2F16333295\n",
      "http%3A%2F%2Fwww.hubmed.org%2Fdisplay.cgi%3Fuids%3D16333295\n",
      "http%3A%2F%2Feuropepmc.org%2Fabstract%2FMED%2F16333295\n",
      "http%3A%2F%2Flinkedlifedata.com%2Fresource%2Fpubmed%2Fid%2F16333295\n",
      "https%3A%2F%2Fomim.org%2Fentry%2F603903\n",
      "http%3A%2F%2Fmirror.omim.org%2Fentry%2F603903\n",
      "https%3A%2F%2Fwww.kegg.jp%2Fentry%2FC12345\n",
      "https%3A%2F%2Fwww.kegg.jp%2Fentry%2FR00100\n",
      "https%3A%2F%2Fdoi.org%2F10.1038%2Fnbt1156\n",
      "http%3A%2F%2Fproteopedia.org%2Fwiki%2Findex.php%2F2gc4\n",
      "https%3A%2F%2Fwww.rcsb.org%2Fstructure%2F2gc4\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fpdbsum%2F2gc4\n",
      "https%3A%2F%2Fpdbj.org%2Fmine%2Fsummary%2F2gc4\n",
      "http%3A%2F%2Fwww.ebi.ac.uk%2Fpdbe%2Fentry%2Fpdb%2F2gc4\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2FQuickGO%2FGTerm%3Fid%3DGO%3A0006915\n",
      "http%3A%2F%2Famigo.geneontology.org%2Famigo%2Fterm%2FGO%3A0006915\n",
      "http%3A%2F%2Fwww.informatics.jax.org%2Fsearches%2FGO.cgi%3Fid%3DGO%3A0006915\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FGO%2FGO%3A0006915\n",
      "http%3A%2F%2Fwww.pantherdb.org%2Fpanther%2Fcategory.do%3FcategoryAcc%3DGO%3A0006915\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fgo%2Fterms%3Fobo_id%3DGO%3A0006915\n",
      "https%3A%2F%2Fwww.kegg.jp%2Fentry%2FD00123\n",
      "https%3A%2F%2Fwww.yeastgenome.org%2Flocus%2FS000003909\n",
      "http%3A%2F%2Fsgd.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Fsgd%3AS000003909\n",
      "https%3A%2F%2Fbioentity.link%2F%23%2Flexicon%2Fpublic%2FS000003909\n",
      "https%3A%2F%2Fwww.alliancegenome.org%2Fgene%2FSGD%3AS000003909\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fsbo%2Fterms%3Fobo_id%3DSBO%3A0000262\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FSBO%2FSBO%3A0000262\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fsbo%2Fmain%2FSBO%3A0000262\n",
      "https%3A%2F%2Fwww.kegg.jp%2Fentry%2FG00123\n",
      "https%3A%2F%2Fwww.wormbase.org%2Fget%3Fname%3DWBGene00000001\n",
      "https%3A%2F%2Fbioentity.link%2F%23%2Flexicon%2Fpublic%2FWBGene00000001\n",
      "https%3A%2F%2Fwww.alliancegenome.org%2Fgene%2FWB%3AWBGene00000001\n",
      "https%3A%2F%2Fpfam.xfam.org%2Ffamily%2FPF01234\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fnuccore%2FX58356\n",
      "http%3A%2F%2Fgetentry.ddbj.nig.ac.jp%2Fgetentry%3Fdatabase%3Dddbj%26accession_number%3DX58356\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FX58356\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fnuccore%2FX58356\n",
      "http%3A%2F%2Fflybase.org%2Freports%2FFBgn0011293.html\n",
      "https%3A%2F%2Fbioentity.link%2F%23%2Flexicon%2Fpublic%2FFBgn0011293\n",
      "https%3A%2F%2Fwww.alliancegenome.org%2Fgene%2FFB%3AFBgn0011293\n",
      "https%3A%2F%2Fpubchem.ncbi.nlm.nih.gov%2Fsummary%2Fsummary.cgi%3Fcid%3D100101\n",
      "https%3A%2F%2Farxiv.org%2Fabs%2F0807.4956v1\n",
      "https%3A%2F%2Fwww.wormbase.org%2Fdb%2Fseq%2Fprotein%3Fname%3DWP%3ACE28239\n",
      "https%3A%2F%2Fpubchem.ncbi.nlm.nih.gov%2Fsummary%2Fsummary.cgi%3Fsid%3D100101\n",
      "http%3A%2F%2Fsabiork.h-its.org%2FnewSearch%3Fq%3Dsabioreactionid%3A75\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Farrayexpress%2Fexperiments%2FE-MEXP-1712\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Farrayexpress-repository%2FE-MEXP-1712\n",
      "http%3A%2F%2Fwww.informatics.jax.org%2Faccession%2FMGI%3A2442292\n",
      "https%3A%2F%2Fbioentity.link%2F%23%2Flexicon%2Fpublic%2FMGI%3A2442292\n",
      "https%3A%2F%2Fwww.alliancegenome.org%2Fgene%2FMGI%3A2442292\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fprotein%2FNP_012345\n",
      "http%3A%2F%2Fwww.tcdb.org%2Fsearch%2Fresult.php%3Ftc%3D5.A.1.1.1\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fcgi-bin%2Fdbfetch%3Fdb%3Duniparc%26id%3DUPI000000000A\n",
      "http%3A%2F%2Fwww.uniprot.org%2Funiparc%2FUPI000000000A\n",
      "http%3A%2F%2Fmint.bio.uniroma2.it%2Fmint%2Fsearch%2FinFrameInteraction.do%3FinteractionAc%3DMINT-10000\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fintact%2Fquery%2Finteraction_id%3AMINT-10000\n",
      "http%3A%2F%2Fwww.signaling-gateway.org%2Fmolecule%2Fquery%3Fafcsid%3DA001094\n",
      "https%3A%2F%2Fdip.doe-mbi.ucla.edu%2Fdip%2FDIPview.cgi%3FID%3DDIP-743N\n",
      "http%3A%2F%2Fpir0.georgetown.edu%2Fcgi-bin%2Fresid%3Fid%3DAA0001\n",
      "http%3A%2F%2Farabidopsis.org%2Fservlets%2FTairObject%3Faccession%3DAASequence%3A1009107926\n",
      "http%3A%2F%2Fwww.arabidopsis.org%2Fservlets%2FTairObject%3Faccession%3DLocus%3A2200950\n",
      "http%3A%2F%2Farabidopsis.org%2Fservlets%2FTairObject%3Faccession%3DGene%3A2200934\n",
      "https%3A%2F%2Fdb.systemsbiology.net%2Fsbeams%2Fcgi%2FPeptideAtlas%2FSummarize_Peptide%3Fquery%3DQUERY%26searchForThis%3DPAp00000009\n",
      "http%3A%2F%2Fwww.lipidmaps.org%2Fdata%2FLMSDRecord.php%3FLMID%3DLMPR0102010012\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgeo%2Fquery%2Facc.cgi%3Facc%3DGDS1234\n",
      "http%3A%2F%2Fwww.hmdb.ca%2Fmetabolites%2FHMDB00001\n",
      "http%3A%2F%2Fpathway.yeastgenome.org%2FYEAST%2Fnew-image%3Ftype%3DPATHWAY%26object%3DPWY3O-214\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fmod%2Fterms%3Fobo_id%3DMOD%3A00001\n",
      "http%3A%2F%2Fbioportal.bioontology.org%2Fontologies%2F1041%3Fp%3Dterms%26conceptid%3DMOD%3A00001\n",
      "http%3A%2F%2Fmerops.sanger.ac.uk%2Fcgi-bin%2Fpepsum%3Fmid%3DS01.001\n",
      "http%3A%2F%2Fwww.pantherdb.org%2Fpanther%2Ffamily.do%3FclsAccession%3DPTHR12345\n",
      "http%3A%2F%2Fwww.bioinf.manchester.ac.uk%2Fcgi-bin%2Fdbbrowser%2Fsprint%2Fsearchprintss.cgi%3Fprints_accn%3DPR00001%26display_opts%3DPrints%26category%3DNone%26queryform%3Dfalse%26regexpr%3Doff\n",
      "http%3A%2F%2Fligand-depot.rutgers.edu%2Fpyapps%2FldHandler.py%3Fformid%3Dcc-index-search%26target%3DABC%26operation%3Dccid\n",
      "http%3A%2F%2Fligand-expo.rcsb.org%2Fpyapps%2FldHandler.py%3Fformid%3Dcc-index-search%26target%3DABC%26operation%3Dccid\n",
      "http%3A%2F%2Faclame.ulb.ac.be%2Fperl%2FAclame%2FGenomes%2Fmge_view.cgi%3Fview%3Dinfo%26id%3Dmge%3A2\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fpride%2Farchive%2Fassays%2F1\n",
      "http%3A%2F%2Fwww.3dmet.dna.affrc.go.jp%2Fcgi%2Fshow_data.php%3Facc%3DB00162\n",
      "http%3A%2F%2Fisbndb.com%2Fsearch-all.html%3Fkw%3D9781584885658\n",
      "http%3A%2F%2Fwww.worldcat.org%2Fisbn%2F9781584885658\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2Fsyn%3Assr3451\n",
      "https%3A%2F%2Fwww.brenda-enzymes.org%2Fphp%2Fresult_flat.php4%3Fecno%3D1.1.1.1\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Ffma%2Fterms%3Fobo_id%3DFMA%3A67112\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FFMA_subset%2FFMA%3A67112\n",
      "http%3A%2F%2Fmatrixdb.univ-lyon1.fr%2F%2Fcgi-bin%2Fcurrent%2FnewPort%3Ftype%3Dassociation%26value%3DP00747__P07355%26class%3DAssociation\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgene%2F100010\n",
      "http%3A%2F%2Fncbigene.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Fncbigene%3A100010\n",
      "https%3A%2F%2Fpubchem.ncbi.nlm.nih.gov%2Fassay%2Fassay.cgi%3Faid%3D1018\n",
      "http%3A%2F%2Fbioportal.bioontology.org%2Fontologies%2FBAO%2Fbao%3ABAO_0002989\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fbao%2Fterms%3Fobo_id%3DBAO%3A0002989\n",
      "http%3A%2F%2Fwww.pathwaycommons.org%2Fpc%2Frecord2.do%3Fid%3D485991\n",
      "http%3A%2F%2Fpbil.univ-lyon1.fr%2Fcgi-bin%2Fview-tree.pl%3Fquery%3DHBG004341%26db%3DHOVERGEN\n",
      "http%3A%2F%2Fwww.wikipathways.org%2Finstance%2FWP100\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fthornton-srv%2Fdatabases%2Fcgi-bin%2FMACiE%2Fentry%2FgetPage.pl%3Fid%3DM0001\n",
      "http%3A%2F%2Fwww.mirbase.org%2Fcgi-bin%2Fmirna_entry.pl%3Facc%3DMI0000001\n",
      "https%3A%2F%2Fwww.genenames.org%2Fdata%2Fgene-symbol-report%2F%23%21%2Fhgnc_id%2F2674\n",
      "http%3A%2F%2Fzfin.org%2FZDB-GENE-041118-11\n",
      "https%3A%2F%2Fbioentity.link%2F%23%2Flexicon%2Fpublic%2FZDB-GENE-041118-11\n",
      "https%3A%2F%2Ftest.alliancegenome.org%2Fgene%2FZFIN%3AZDB-GENE-041118-11\n",
      "https%3A%2F%2Fwww.rhea-db.org%2Freaction%3Fid%3D12345\n",
      "http%3A%2F%2Fwww.sequenceontology.org%2Fmiso%2Fcurrent_release%2Fterm%2FSO%3A0000704\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FSO%2FSO%3A0000704\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fso%2Fterms%3Fobo_id%3DSO%3A0000704\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fchembl%2Ftarget%2Finspect%2FCHEMBL3467\n",
      "http%3A%2F%2Flinkedchemistry.info%2Fchembl%2Fchemblid%2FCHEMBL3467\n",
      "https%3A%2F%2Frdf.ebi.ac.uk%2Fresource%2Fchembl%2Ftarget%2FCHEMBL3467\n",
      "http%3A%2F%2Fsabiork.h-its.org%2FkineticLawEntry.jsp%3FviewData%3Dtrue%26kinlawid%3D5046\n",
      "http%3A%2F%2Fwww.whocc.no%2Fatc_ddd_index%2F%3Fcode%3DA10BA02\n",
      "http%3A%2F%2Fwww.pharmgkb.org%2Fpathway%2FPA146123006\n",
      "http%3A%2F%2Fwww.pharmgkb.org%2Fdrug%2FPA448710\n",
      "http%3A%2F%2Fwww.pharmgkb.org%2Fdisease%2FPA447218\n",
      "http%3A%2F%2Fbidd.nus.edu.sg%2Fgroup%2FTTD%2FZFTTDDetail.asp%3FID%3DTTDS00056\n",
      "http%3A%2F%2Fbidd.nus.edu.sg%2Fgroup%2FTTD%2FZFTTDDRUG.asp%3FID%3DDAP000773\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fcompneur-srv%2FLGICdb%2FHTML%2F5HT3Arano.php\n",
      "http%3A%2F%2Fneuromorpho.org%2Fneuron_info.jsp%3Fneuron_name%3DRosa2\n",
      "https%3A%2F%2Fchem.nlm.nih.gov%2Fchemidplus%2Frn%2F57-27-2\n",
      "http%3A%2F%2Fctdbase.org%2Fdetail.go%3Ftype%3Dchem%26acc%3DD001151\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbiosystems%2F001\n",
      "http%3A%2F%2Fsenselab.med.yale.edu%2FNeuronDB%2FNeuronProp.aspx%3Fid%3D265\n",
      "http%3A%2F%2Fctdbase.org%2Fdetail.go%3Ftype%3Dgene%26acc%3D101\n",
      "http%3A%2F%2Fctdbase.org%2Fdetail.go%3Ftype%3Ddisease%26db%3DMESH%26acc%3DD053716\n",
      "https%3A%2F%2Fbionumbers.hms.harvard.edu%2Fbionumber.aspx%3Fid%3D104674\n",
      "http%3A%2F%2Fwww.drugbank.ca%2Fdrugs%2FDB00001\n",
      "http%3A%2F%2Fwww.t3db.org%2Ftoxins%2FT3D0001\n",
      "http%3A%2F%2Fsmpdb.ca%2Fview%2FSMP00001\n",
      "https%3A%2F%2Fwww.genedb.org%2Fgene%2FLinJ.20.0070\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FTEDDY%2FTEDDY_0000066\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fteddy%2Fterms%3Fshort_form%3DTEDDY_0000066\n",
      "http%3A%2F%2Fwww.phosphosite.org%2FproteinAction.do%3Fid%3D12300\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fcl%2Fterms%3Fobo_id%3DCL%3A0000232\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FCL%2FCL%3A0000232\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fbto%2Fterms%3Fobo_id%3DBTO%3A0000146\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FBTO%2FBTO%3A0000146\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FKISAO%2Fkisao%3AKISAO_0000057\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fkisao%2Fterms%3Fshort_form%3DKISAO_0000057\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fpdbe-srv%2Fpdbechem%2FchemicalCompound%2Fshow%2FAB0\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fpato%2Fterms%3Fobo_id%3DPATO%3A0001998\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FPATO%2FPATO%3A0001998\n",
      "https%3A%2F%2Fglytoucan.org%2FStructures%2FGlycans%2FG77500AY\n",
      "http%3A%2F%2Flipidbank.jp%2Fcgi-bin%2Fdetail.cgi%3Fid%3DBBA0001\n",
      "http%3A%2F%2Fsmart.embl-heidelberg.de%2Fsmart%2Fdo_annotation.pl%3FDOMAIN%3DSM00015\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2FStructure%2Fcdd%2Fcddsrv.cgi%3Fuid%3Dcd00400\n",
      "http%3A%2F%2Fprodom.prabi.fr%2Fprodom%2Fcurrent%2Fcgi-bin%2Frequest.pl%3Fquestion%3DDBEN%26query%3DPD10000\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2FK00001\n",
      "http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2FStructure%2Fmmdb%2Fmmdbsrv.cgi%3Fuid%3D50885\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fintact%2Fimex%2Fmain.xhtml%3Fquery%3DIM-19210-3\n",
      "https%3A%2F%2Fimexcentral.org%2Ficentral%2Fimex%2Frec%2FIM-19210-3\n",
      "http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FRO_0002533\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fro%2Fterms%3Fshort_form%3DRO_0002533\n",
      "http%3A%2F%2Fwodaklab.org%2FiRefWeb%2Finteraction%2Fshow%2F617102\n",
      "http%3A%2F%2Fwww.jcvi.org%2Fmpidb%2Fexperiment.php%3Finteraction_id%3D172\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fintact%2Fquery%2Finteraction_id%3AMPIDB-INT-172\n",
      "http%3A%2F%2Fwww.phosphosite.org%2FsiteAction.do%3Fid%3D2842\n",
      "http%3A%2F%2Fwww.neurolex.org%2Fwiki%2FBirnlex_721\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FBIRNLEX%2FBirnlex_721\n",
      "http%3A%2F%2Fasap.ahabs.wisc.edu%2Fasap%2Ffeature_info.php%3FLocationID%3DWIS%26FeatureID%3DABE-0009634\n",
      "http%3A%2F%2Fsabiork.h-its.org%2FnewSearch%3Fq%3Decnumber%3A2.7.1.1\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FOPB%3Fconceptid%3Dhttp%253A%252F%252Fbhi.washington.edu%252FOPB%2523OPB_00573\n",
      "http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0000070\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FOBI%2FOBI_0000070\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fobi%2Fterms%3Fobo_id%3DOBI_0000070\n",
      "http%3A%2F%2Fjjj.biochem.sun.ac.za%2Fmodels%2Fcurien%2F\n",
      "http%3A%2F%2Fjjj.bio.vu.nl%2Fmodels%2Fcurien%2F\n",
      "http%3A%2F%2Fjjj.mib.ac.uk%2Fmodels%2Fcurien\n",
      "http%3A%2F%2Fsenselab.med.yale.edu%2FModelDB%2FShowModel.asp%3Fmodel%3D45539\n",
      "http%3A%2F%2Fwww.subtiwiki.uni-goettingen.de%2Fwiki%2Findex.php%2FBSU29180\n",
      "http%3A%2F%2Fdoqcs.ncbs.res.in%2Ftemplate.php%3F%26y%3Dpathwaydetails%26pn%3D131\n",
      "http%3A%2F%2Fdoqcs.ncbs.res.in%2Ftemplate.php%3F%26y%3Daccessiondetails%26an%3D57\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fuo%2Fterms%3Fobo_id%3DUO%3A0000080\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FUO%2FUO%3A0000080\n",
      "http%3A%2F%2Fpid.nci.nih.gov%2Fsearch%2Fpathway_landing.shtml%3Fwhat%3Dgraphic%26jpg%3Don%26pathway_id%3Dpi3kcipathway\n",
      "https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT00222573\n",
      "http%3A%2F%2Fwww.chemspider.com%2FChemical-Structure.56586.html\n",
      "http%3A%2F%2Fomia.angis.org.au%2F1000%2F\n",
      "http%3A%2F%2Fwww.candidagenome.org%2Fcgi-bin%2Flocus.pl%3Fdbid%3DCAL0003079\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fthornton-srv%2Fdatabases%2FCSA%2FSearchResults.php%3FPDBID%3D1a05\n",
      "http%3A%2F%2Fncit.nci.nih.gov%2Fncitbrowser%2FConceptReport.jsp%3Fdictionary%3DNCI%2520Thesaurus%26code%3DC80519\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fncit%2Fterms%3Fshort_form%3DNCIT_C80519\n",
      "http%3A%2F%2Fwww.antweb.org%2Fspecimen.do%3Fname%3Dcasent0106247\n",
      "https%3A%2F%2Fwww.biocatalogue.org%2Fservices%2F614\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3084216%2F%3Ftool%3Dpubmed\n",
      "http%3A%2F%2Feuropepmc.org%2Farticles%2FPMC3084216\n",
      "https%3A%2F%2Famoebadb.org%2Famoeba%2Fapp%2Frecord%2Fgene%2FEDI_244000\n",
      "http%3A%2F%2Fwww.lgcstandards-atcc.org%2FProducts%2FAll%2F11303.aspx\n",
      "http%3A%2F%2Fmicrosporidiadb.org%2Fmicro%2FshowRecord.do%3Fname%3DGeneRecordClasses.GeneRecordClass%26source_id%3DECU03_0820i\n",
      "http%3A%2F%2Fplasmodb.org%2Fplasmo%2FshowRecord.do%3Fname%3DGeneRecordClasses.GeneRecordClass%26source_id%3DPF11_0344\n",
      "https%3A%2F%2Fcryptodb.org%2Fcryptodb%2Fapp%2Frecord%2Fgene%2Fcgd7_230\n",
      "https%3A%2F%2Fgiardiadb.org%2Fgiardiadb%2Fapp%2Frecord%2Fgene%2FGL50803_102438\n",
      "http%3A%2F%2Ftrichdb.org%2Ftrichdb%2FshowRecord.do%3Fname%3DGeneRecordClasses.GeneRecordClass%26source_id%3DTVAG_386080\n",
      "http%3A%2F%2Fbeetlebase.org%2Fcgi-bin%2Fgbrowse%2FBeetleBase3.gff3%2F%3Fname%3DTC010103\n",
      "http%3A%2F%2Fflypush.imgen.bcm.tmc.edu%2Fpscreen%2Fdetails.php%3Fline%3DKG09531\n",
      "http%3A%2F%2Ftritrypdb.org%2Ftritrypdb%2FshowRecord.do%3Fname%3DGeneRecordClasses.GeneRecordClass%26source_id%3DTb927.8.620\n",
      "http%3A%2F%2Ftoxodb.org%2Ftoxo%2FshowRecord.do%3Fname%3DGeneRecordClasses.GeneRecordClass%26source_id%3DTGME49_053730\n",
      "http%3A%2F%2Fwww.boldsystems.org%2Findex.php%2FTaxbrowser_Taxonpage%3Ftaxid%3D27267\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fprobe%2F%3Fterm%3D1000000\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgenome%2Fsts%2Fsts.cgi%3Fuid%3D456789\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fnucest%2FBP100000\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FBP100000\n",
      "http%3A%2F%2Fgetentry.ddbj.nig.ac.jp%2Fgetentry%2Fna%2FBP100000\n",
      "http%3A%2F%2Fwww.ecogene.org%2Fgene%2FEG10173\n",
      "http%3A%2F%2Fgreengenes.lbl.gov%2Fcgi-bin%2Fshow_one_record_v2.pl%3FprokMSA_id%3D100000\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fnucest%2FEY223054.1\n",
      "http%3A%2F%2Fwww.ars-grin.gov%2Fcgi-bin%2Fnpgs%2Fhtml%2Ftaxon.pl%3F19333\n",
      "http%3A%2F%2Fwww.gabipd.org%2Fdatabase%2Fcgi-bin%2FGreenCards.pl.cgi%3FBioObjectId%3D2679240%26Mode%3DShowBioObject\n",
      "http%3A%2F%2Fh-invitational.jp%2Fhinv%2Fspsoup%2Flocus_view%3Fhix_id%3DHIX0004394\n",
      "http%3A%2F%2Fh-invitational.jp%2Fhinv%2Fspsoup%2Ftranscript_view%3Fhit_id%3DHIT000195363\n",
      "http%3A%2F%2Fwww.homd.org%2Fmodules.php%3Fop%3Dmodload%26name%3DGenomeList%26file%3Dindex%26link%3Ddetailinfo%26seqid%3DSEQF1003\n",
      "http%3A%2F%2Fh-invitational.jp%2Fhinv%2Fprotein%2Fprotein_view.cgi%3Fhip_id%3DHIP000030660\n",
      "http%3A%2F%2Fwww.homd.org%2Fmodules.php%3Fop%3Dmodload%26name%3DHOMD%26file%3Dindex%26oraltaxonid%3D811%26view%3Ddynamic\n",
      "https%3A%2F%2Fwww-is.biotoul.fr%2Fscripts%2FficheIS.php%3Fname%3DISA1083-2\n",
      "http%3A%2F%2Fwww.fludb.org%2Fbrc%2FfluSegmentDetails.do%3FncbiGenomicAccession%3DCY077097\n",
      "http%3A%2F%2Fwww.jcm.riken.go.jp%2Fcgi-bin%2Fjcm%2Fjcm_number%3FJCM%3D17254\n",
      "http%3A%2F%2Fimg.jgi.doe.gov%2Fcgi-bin%2Fw%2Fmain.cgi%3Fsection%3DTaxonDetail%26taxon_oid%3D648028003\n",
      "http%3A%2F%2Fwww.mycobank.org%2FBiolomics.aspx%3FTable%3DMycobank%26MycoBankNr_%3D349124\n",
      "http%3A%2F%2Fwww.maizegdb.org%2Fcgi-bin%2Fdisplaylocusrecord.cgi%3Fid%3D25011\n",
      "http%3A%2F%2Fimg.jgi.doe.gov%2Fcgi-bin%2Fw%2Fmain.cgi%3Fsection%3DGeneDetail%26gene_oid%3D638309541\n",
      "http%3A%2F%2Fwww.pseudomonas.com%2Ffeature%2Fshow%2F%3Flocus_tag%3DPSEEN0001\n",
      "http%3A%2F%2Fwww.nbrc.nite.go.jp%2FNBRC2%2FNBRCCatalogueDetailServlet%3FID%3DNBRC%26CAT%3D00001234\n",
      "http%3A%2F%2Fwww.gramene.org%2Fdb%2Fprotein%2Fprotein_search%3Fprotein_id%3D78073\n",
      "http%3A%2F%2Fwww.gramene.org%2Fdb%2Fgenes%2Fsearch_gene%3Facc%3DGR%3A0080039\n",
      "http%3A%2F%2Fwww.gramene.org%2Fdb%2Fontology%2Fsearch%3Fid%3DGR_tax%3A013681\n",
      "http%3A%2F%2Fwww.gramene.org%2Fdb%2Fqtl%2Fqtl_display%3Fqtl_accession_id%3DCQG5\n",
      "http%3A%2F%2Fwww.xenbase.org%2Fgene%2Fshowgene.do%3Fmethod%3DdisplayGeneSummary%26geneId%3D922462\n",
      "http%3A%2F%2Fsolgenomics.net%2Fphenome%2Flocus_display.pl%3Flocus_id%3D0001\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FEDAM%2Fdata_1664\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fedam%2Fterms%3Fshort_form%3Ddata_1664\n",
      "http%3A%2F%2Fbioportal.bioontology.org%2Fontologies%2F1046\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fmiriam%2Fmain%2Fresources%2FMIR%3A00100005\n",
      "http%3A%2F%2Fwww.arachnoserver.org%2Ftoxincard.html%3Fid%3DAS000060\n",
      "https%3A%2F%2Fbioinformatics.cineca.it%2FPMDB%2Fuser%2F%2Fsearch.php%3Fidsearch%3DPM0012345\n",
      "http%3A%2F%2Fbiocyc.org%2Fgetid%3Fid%3DECOLI%3ACYT-D-UBIOX-CPLX\n",
      "http%3A%2F%2Ffossilworks.org%2F%3Fa%3DtaxonInfo%26taxon_no%3D83088\n",
      "http%3A%2F%2Fwww.cazy.org%2FGT10.html\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2FQuickGO%2FGProtein%3Fac%3DP12345\n",
      "http%3A%2F%2Fcompluyeast2dpage.dacya.ucm.es%2Fcgi-bin%2F2d%2F2d.cgi%3Fac%3DO08709\n",
      "http%3A%2F%2Feggnog.embl.de%2Fversion_3.0%2Fcgi%2Fsearch.py%3Fsearch_term_0%3DveNOG12876\n",
      "http%3A%2F%2Fwww.york.ac.uk%2Fres%2Fthomas%2FGene.cfm%3FrecordID%3DEB0170\n",
      "https%3A%2F%2Fbacteria.ensembl.org%2Fid%2FMU9_3181\n",
      "https%3A%2F%2Fprotists.ensembl.org%2Fid%2FPF3D7_1328700\n",
      "https%3A%2F%2Fmetazoa.ensembl.org%2Fid%2FFBtr0084214\n",
      "https%3A%2F%2Feuhcvdb.ibcp.fr%2FeuHCVdb%2Fdo%2FdisplayHCVEntry%3FprimaryAC%3DM58335\n",
      "https%3A%2F%2Fplants.ensembl.org%2Fid%2FAT1G73965\n",
      "https%3A%2F%2Ffungi.ensembl.org%2Fid%2FCADAFLAT00006211\n",
      "http%3A%2F%2Fgenatlas.medecine.univ-paris5.fr%2Ffiche.php%3Fsymbol%3DHBB\n",
      "http%3A%2F%2Fwww.cathdb.info%2Fcathnode%2F1.10.10.200\n",
      "https%3A%2F%2Furgi.versailles.inra.fr%2FGenefarm%2FGene%2Fdisplay_gene.htpl%3FGENE_ID%3D4892\n",
      "http%3A%2F%2Fpbil.univ-lyon1.fr%2Fcgi-bin%2Fview-tree.pl%3Fdb%3DHOGENOM5%26query%3DHBG284870\n",
      "http%3A%2F%2Fwww.gpcrdb.org%2Fprotein%2FRL3R1_HUMAN%2F\n",
      "ftp%3A%2F%2Fftp.embl-heidelberg.de%2Fpub%2Fdatabases%2Fprotein_extras%2Fhssp%2F102l.hssp.bz2\n",
      "ftp%3A%2F%2Fftp.cmbi.ru.nl%2Fpub%2Fmolbio%2Fdata%2Fhssp%2F102l.hssp.bz2\n",
      "http%3A%2F%2Fwww.ensembl.org%2FMulti%2FGeneTree%2FImage%3Fdb%3Dcore%3Bgt%3DENSGT00550000074763\n",
      "http%3A%2F%2Ftuberculist.epfl.ch%2Fquicksearch.php%3Fgene%2Bname%3DRv1908c\n",
      "http%3A%2F%2Fmycobrowser.epfl.ch%2Fmarinosearch.php%3Fgene%2Bname%3DMMAR_2462\n",
      "http%3A%2F%2Fmycobrowser.epfl.ch%2Fsmegmasearch.php%3Fgene%2Bname%3DMSMEG_3769\n",
      "http%3A%2F%2Fcegg.unige.ch%2Forthodb%2Fresults%3Fsearchtext%3DQ9P0K8\n",
      "http%3A%2F%2Fmycobrowser.epfl.ch%2Fleprosysearch.php%3Fgene%2Bname%3DML0224\n",
      "http%3A%2F%2Fwww.orpha.net%2Fconsor%2Fcgi-bin%2FOC_Exp.php%3FLng%3DEN%26Expert%3D85163\n",
      "http%3A%2F%2Forphanet.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Forphanet%3A85163\n",
      "http%3A%2F%2Fperoxibase.toulouse.inra.fr%2Fbrowse%2Fprocess%2Fview_perox.php%3Fid%3D5282\n",
      "http%3A%2F%2Fsubstrate.burnham.org%2Fprotein%2Fannotation%2F1915%2Fhtml\n",
      "http%3A%2F%2Fphylomedb.org%2F%3Fseqid%3DPhy000CLXM_RAT\n",
      "http%3A%2F%2Fcutdb.burnham.org%2Frelation%2Fshow%2F25782\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fsites%2Fentrez%3FDb%3Dproteinclusters%26Cmd%3DDetailsSearch%26Term%3DO80725\n",
      "http%3A%2F%2Fwww.proteinmodelportal.org%2Fquery%2Funiprot%2FQ0VCA6\n",
      "http%3A%2F%2Fwww.protonet.cs.huji.ac.il%2Frequested%2Fcluster_card.php%3Fcluster%3D4349895\n",
      "http%3A%2F%2Fwww.protonet.cs.huji.ac.il%2Frequested%2Fprotein_card.php%3Fprotein_id%3D16941567\n",
      "https%3A%2F%2Fswissmodel.expasy.org%2Frepository%2Funiprot%2FP23298\n",
      "http%3A%2F%2Frebase.neb.com%2Frebase%2Fenz%2F101.html\n",
      "https%3A%2F%2Fwww.vectorbase.org%2Fsearch%2Fsite%2FISCW007415%3F%26site%3D%22Genome%22\n",
      "http%3A%2F%2Fwww.mirbase.org%2Fcgi-bin%2Fmature.pl%3Fmature_acc%3DMIMAT0000001\n",
      "http%3A%2F%2Fcommonchemistry.org%2FChemicalDetail.aspx%3Fref%3D50-00-0\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2FT30002\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2Feco\n",
      "http%3A%2F%2Fjglobal.jst.go.jp%2Fen%2Fredirect%3FNikkaji_No%3DJ55.713G\n",
      "http%3A%2F%2Fwww.pharmgkb.org%2Fgene%2FPA131\n",
      "http%3A%2F%2Fstormo.wustl.edu%2FScerTF%2Fdetails%2FRSC3%2F\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fsra%2FSRX000007\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FSRX000007\n",
      "http%3A%2F%2Ftrace.ddbj.nig.ac.jp%2FDRASearch%2Fexperiment%3Facc%3DSRX000007\n",
      "http%3A%2F%2Frgd.mcw.edu%2Frgdweb%2Fontology%2Fannot.html%3Facc_id%3DPW%3A0000208\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fpw%2Fterms%3Fobo_id%3DPW%3A0000208\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FPW%2FPW%3A0000208\n",
      "http%3A%2F%2Frna.igmors.u-psud.fr%2FNAPP%2FNiveau2.php%3Fspecie%3D351\n",
      "http%3A%2F%2Frhesus.amu.edu.pl%2Fmirnest%2Fcopy%2Fdetails.php%3Fid%3DMNEST029358\n",
      "http%3A%2F%2Fwww.noncode.org%2FNONCODERv3%2Fncrna.php%3Fncid%3D377550\n",
      "http%3A%2F%2Fcrdd.osdd.net%2Fservers%2Fvirsirnadb%2Frecord.php%3Fdetails%3Dvirsi1909\n",
      "http%3A%2F%2Felm.eu.org%2Felms%2FelmPages%2FCLV_MEL_PAP_1.html\n",
      "http%3A%2F%2Fimmunet.cn%2Fbdb%2Findex.php%2Fmimoset%2F1\n",
      "https%3A%2F%2Fbykdb.ibcp.fr%2Fdata%2Fhtml%2Fannotated%2FA0A009E7X8.html\n",
      "http%3A%2F%2Fwww-bionet.sscc.ru%2Fsitex%2Findex.php%3Fsiteid%3D1000\n",
      "http%3A%2F%2Fclipserve.clip.ubc.ca%2Ftopfind%2Fproteins%2FQ9UKQ2\n",
      "https%3A%2F%2Fco.mbine.org%2Fspecifications%2Fsbgn.er.level-1.version-1.2\n",
      "http%3A%2F%2Fbioinfo.iitk.ac.in%2FMIPModDB%2Fresult.php%3Fcode%3DHOSAPI0399\n",
      "http%3A%2F%2Fwww.conoserver.org%2F%3Fpage%3Dcard%26table%3Dprotein%26id%3D2639\n",
      "http%3A%2F%2Fwww.kazusa.or.jp%2Fhuge%2Fgfpage%2FKIAA0001%2F\n",
      "http%3A%2F%2Fwww.cabri.org%2FCABRI%2Fsrs-bin%2Fwgetz%3F-e%2B-page%2BEntryPage%2B%5Bdsmz_mutz-id%3AACC%20291%5D\n",
      "http%3A%2F%2Fwww.be.cabri.org%2FCABRI%2Fsrs-bin%2Fwgetz%3F-e%2B-page%2BEntryPage%2B%5Bdsmz_mutz-id%3AACC%20291%5D\n",
      "http%3A%2F%2Fdictybase.org%2Fgene%2FDDB_G0267522\n",
      "http%3A%2F%2Fstring.embl.de%2Finteractions%2FP53350\n",
      "http%3A%2F%2Fstring-db.org%2Finteractions%2FP53350\n",
      "http%3A%2F%2Fwww.bindingdb.org%2Fcompact%2Fe999\n",
      "http%3A%2F%2Fphenol-explorer.eu%2Ffoods%2F75\n",
      "http%3A%2F%2Fstitch.embl.de%2Finteractions%2FBQJCRHHNABKAKU\n",
      "http%3A%2F%2Fwww.whocc.no%2Fatcvet%2Fatcvet_index%2F%3Fcode%3DQJ51RV02\n",
      "http%3A%2F%2Fwww.snomedbrowser.com%2FCodes%2FDetails%2F284196006\n",
      "http%3A%2F%2Fwww.nlm.nih.gov%2Fcgi%2Fmesh%2F2012%2FMB_cgi%3Fmode%3D%26index%3D17186%26view%3Dexpanded\n",
      "http%3A%2F%2Fkanaya.naist.jp%2Fknapsack_jsp%2Finformation.jsp%3Fword%3DC00000001\n",
      "http%3A%2F%2Fwww.massbank.jp%2Fjsp%2FFwdRecord.jsp%3Fid%3DPB000166\n",
      "http%3A%2F%2Fwww.massbank.eu%2FMassBank%2Fjsp%2FFwdRecord.jsp%3Fid%3DPB000166\n",
      "http%3A%2F%2Fwebprod3.hc-sc.gc.ca%2Fdpd-bdpp%2Finfo.do%3Flang%3Deng%26code%3D63250\n",
      "http%3A%2F%2Fgmd.mpimp-golm.mpg.de%2FMetabolites%2F68513255-fc44-4041-bc4b-4fd2fae7541d.aspx\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fhomologene%2F1000\n",
      "http%3A%2F%2Fhomologene.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Fhomologene%3A1000\n",
      "http%3A%2F%2Fgenome.crg.es%2Fdatasets%2Fabs2005%2Fentries%2FA0014.html\n",
      "http%3A%2F%2Fumbbd.ethz.ch%2Fservlets%2Fpageservlet%3Fptype%3Dc%26compID%3Dc0001\n",
      "http%3A%2F%2Fwww.dpvweb.net%2Fdpv%2Fshowdpv.php%3Fdpvno%3D100\n",
      "http%3A%2F%2Faps.unmc.edu%2FAP%2Fdatabase%2Fquery_output.php%3FID%3D01001\n",
      "http%3A%2F%2Fcdb.ics.uci.edu%2Fcgibin%2FChemicalDetailWeb.py%3Fchemical_id%3D3966782\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2FIEB%2FResearch%2FAcembly%2Fav.cgi%3Fdb%3Dworm%26c%3DGene%26l%3Daap-1\n",
      "http%3A%2F%2Fgenius.embnet.dkfz-heidelberg.de%2Fmenu%2Fcgi-bin%2Fsrs7.1.3.1%2Fwgetz%3F-id%2B4ktAl1fNMTA%2B-e%2B%5BIMGT%3A%27M94112%27%5D\n",
      "http%3A%2F%2Fwww.imgt.org%2Fligmdb%2Fview%3Fid%3DM94112\n",
      "http%3A%2F%2Fwww.guidetopharmacology.org%2FGRAC%2FObjectDisplayForward%3FobjectId%3D101\n",
      "http%3A%2F%2Fworfdb.dfci.harvard.edu%2Findex.php%3Fsearch_type%3Dname%26page%3Dshowresultrc%26race_query%3DT01B6.1\n",
      "http%3A%2F%2Fwww.soybase.org%2Fsbt%2Fsearch%2Fsearch_results.php%3Fcategory%3DSNP%26search_term%3DBARC-013845-01256\n",
      "http%3A%2F%2Fnematode.lab.nig.ac.jp%2Fdb2%2FShowCloneInfo.php%3Fclone%3D6b1\n",
      "http%3A%2F%2Fcgsc.biology.yale.edu%2FSite.php%3FID%3D74\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Farrayexpress%2Farrays%2FA-GEOD-50\n",
      "http%3A%2F%2Fantirrhinum.net%2Fcgi-bin%2Face%2Fgeneric%2Ftree%2FDragonDB%3Fname%3D3hB06%3Bclass%3DDNA\n",
      "http%3A%2F%2Fwww.kazusa.or.jp%2Frouge%2Fgfpage%2FmKIAA4200%2F\n",
      "http%3A%2F%2Fmerops.sanger.ac.uk%2Fcgi-bin%2Ffamsum%3Ffamily%3DS1\n",
      "http%3A%2F%2Fantirrhinum.net%2Fcgi-bin%2Face%2Fgeneric%2Ftree%2FDragonDB%3Fname%3DDEF%26amp%3Bclass%3DLocus\n",
      "http%3A%2F%2Fantirrhinum.net%2Fcgi-bin%2Face%2Fgeneric%2Ftree%2FDragonDB%3Fname%3Dcho%26amp%3Bclass%3DAllele\n",
      "http%3A%2F%2Fantirrhinum.net%2Fcgi-bin%2Face%2Fgeneric%2Ftree%2FDragonDB%3Fname%3DAMDEFA%3Bclass%3DPeptide\n",
      "https%3A%2F%2Fcatalog.loc.gov%2Fvwebv%2Fsearch%3FsearchCode%3DSTNO%26searchArg%3D0745-4570%26searchType%3D1%26recCount%3D25\n",
      "https%3A%2F%2Fportal.issn.org%2Fresource%2FISSN%2F0745-4570\n",
      "http%3A%2F%2Flgsun.grc.nia.nih.gov%2Fcgi-bin%2Fpro3%3Fsname1%3DJ0705A10\n",
      "http%3A%2F%2Farabidopsis.info%2FStockInfo%3FNASC_id%3DN1899\n",
      "http%3A%2F%2Fwww.pazar.info%2Fcgi-bin%2Ftf_search.cgi%3FgeneID%3DTF0001053\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMO%2FArrayGroup\n",
      "https%3A%2F%2Fmged.sourceforge.net%2Fontologies%2FMGEDontology.php%23ArrayGroup\n",
      "http%3A%2F%2Fmods.rna.albany.edu%2Fmods%2Fmodifications%2Fview%2F101\n",
      "http%3A%2F%2Fwww.plantontology.org%2Famigo%2Fgo.cgi%3Fview%3Ddetails%26query%3DPO%3A0009089\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FPO%2FPO%3A0009089\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fpo%2Fterms%3Fobo_id%3DPO%3A0009089\n",
      "http%3A%2F%2Fciliate.org%2Findex.php%2Ffeature%2Fdetails%2FTTHERM_00648910\n",
      "http%3A%2F%2Fpurl.org%2Fphylo%2Ftreebase%2Fphylows%2Fstudy%2FTB2%3AS1000%3Fformat%3Dhtml\n",
      "http%3A%2F%2Fwww.jcvi.org%2Fcgi-bin%2Ftigrfams%2FHmmReportPage.cgi%3Facc%3DTIGR00010\n",
      "http%3A%2F%2Friodb01.ibase.aist.go.jp%2Fsdbs%2Fcgi-bin%2Fcre_frame_disp.cgi%3Fsdbsno%3D4544\n",
      "http%3A%2F%2Fwebapps2.ucalgary.ca%2F~groupii%2Fcgi-bin%2Fintron.cgi%3Fname%3DCu.me.I1\n",
      "http%3A%2F%2Fwww.guidetopharmacology.org%2FGRAC%2FFamilyDisplayForward%3FfamilyId%3D78\n",
      "http%3A%2F%2Fwww.vbase2.org%2Fvgene.php%3Fid%3DhumIGHV025\n",
      "https%3A%2F%2Fwww.genecards.org%2Fcgi-bin%2Fcarddisp.pl%3Fgene%3DABL1\n",
      "http%3A%2F%2Fwww.cs.tau.ac.il%2F~spike%2Fmaps%2Fspike00001.html\n",
      "http%3A%2F%2Fmetlin.scripps.edu%2Fmetabo_info.php%3Fmolid%3D1455\n",
      "http%3A%2F%2Fumbbd.ethz.ch%2Fservlets%2Fpageservlet%3Fptype%3Dep%26enzymeID%3De0333\n",
      "http%3A%2F%2Fwww.mmrrc.org%2Fcatalog%2FgetSDS.php%3Fmmrrc_id%3D70\n",
      "http%3A%2F%2Fumbbd.ethz.ch%2Fservlets%2Fpageservlet%3Fptype%3Dr%26reacID%3Dr0001\n",
      "http%3A%2F%2Fumbbd.ethz.ch%2Fservlets%2Fpageservlet%3Fptype%3Dp%26pathway_abbr%3Dala\n",
      "http%3A%2F%2Fdictybase.org%2Fdb%2Fcgi-bin%2Ffeature_page.pl%3Fprimary_id%3DDDB0016567\n",
      "http%3A%2F%2Fwww.umbbd.ethz.ch%2Fservlets%2Frule.jsp%3Frule%3Dbt0001\n",
      "http%3A%2F%2Fcomgen.pl%2Fmirex1%2F%3Fpage%3Dresults%2Frecord%26name%3D165a%26exref%3Dpp2a%26limit%3Dyes\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fcgi-bin%2Fimgt%2Fhla%2Fget_allele.cgi%3FA%2A01%3A01%3A01%3A01\n",
      "https%3A%2F%2Fbdsc.indiana.edu%2Fstocks%2F33607\n",
      "http%3A%2F%2Fopm.phar.umich.edu%2Fprotein.php%3Fpdbid%3D1h68\n",
      "http%3A%2F%2Fwww.allergome.org%2Fscript%2Fdettaglio.php%3Fid_molecule%3D1948\n",
      "http%3A%2F%2Fwww.pombase.org%2Fspombe%2Fresult%2FSPCC13B11.01\n",
      "http%3A%2F%2Fwww.proteinatlas.org%2FENSG00000026508\n",
      "http%3A%2F%2Fjaxmice.jax.org%2Fstrain%2F005012.html\n",
      "http%3A%2F%2Fwww.ubio.org%2Fbrowser%2Fdetails.php%3FnamebankID%3D2555646\n",
      "http%3A%2F%2Fyetfasco.ccbr.utoronto.ca%2FshowPFM.php%3Fmot%3DYOR172W_571.0\n",
      "http%3A%2F%2Fdiana.imis.athena-innovation.gr%2FDianaTools%2Findex.php%3Fr%3Dtarbase%2Findex%26mirnas%3Dhsa-let-7a-2-3p\n",
      "http%3A%2F%2Fwww.jcvi.org%2Fcharprotdb%2Findex.cgi%2Fview%2FCH_001923\n",
      "https%3A%2F%2Fomabrowser.org%2Fcgi-bin%2Fgateway.pl%3Ff%3DDisplayEntry%26p1%3DHUMAN16963\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fprotein%2FCAA71118.1%3Freport%3Dgenpept\n",
      "https%3A%2F%2Fomabrowser.org%2Fcgi-bin%2Fgateway.pl%3Ff%3DDisplayGroup%26p1%3DLCSCCPN\n",
      "http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2FUniGene%2Fclust.cgi%3FUGID%3D4900\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fprotein%2FCAA71118.1\n",
      "http%3A%2F%2Fbitterdb.agri.huji.ac.il%2FReceptor.php%3Fid%3D1\n",
      "http%3A%2F%2Fbitterdb.agri.huji.ac.il%2Fbitterdb%2Fcompound.php%3Fid%3D46\n",
      "http%3A%2F%2Ftrace.ddbj.nig.ac.jp%2FBPSearch%2Fbioproject%3Facc%3DPRJDB3\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbioproject%3Fterm%3DPRJDB3\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FPRJDB3\n",
      "http%3A%2F%2Fpiroplasmadb.org%2Fpiro%2FshowRecord.do%3Fname%3DGeneRecordClasses.GeneRecordClass%26source_id%3DTA14985\n",
      "http%3A%2F%2Funite.ut.ee%2Fbl_forw.php%3Fnimi%3DUDB000691\n",
      "http%3A%2F%2Fncim.nci.nih.gov%2Fncimbrowser%2FConceptReport.jsp%3Fdictionary%3DNCI%2520MetaThesaurus%26code%3DC0026339\n",
      "http%3A%2F%2Fwww.proglycprot.org%2Fdetail.aspx%3FProId%3DAC119\n",
      "https%3A%2F%2Fpolbase.neb.com%2Fpolymerases%2F19-T4%23sequences\n",
      "http%3A%2F%2Fwww.receptors.org%2Fnucleardb%2Fproteins%2Fprgr_human\n",
      "http%3A%2F%2Fsupfam.org%2FSUPERFAMILY%2Fcgi-bin%2Fscop.cgi%3Fipid%3DSSF57615\n",
      "http%3A%2F%2Fcbg.garvan.unsw.edu.au%2Fpina%2Finteractome.oneP.do%3Fac%3DQ13485%26showExtend%3Dnull\n",
      "http%3A%2F%2Fbacmap.wishartlab.com%2Forganisms%2F1050\n",
      "http%3A%2F%2Frice.plantbiology.msu.edu%2Fcgi-bin%2FORF_infopage.cgi%3F%26orf%3DLOC_Os02g13300\n",
      "https%3A%2F%2Fwww.uniprot.org%2Ftissues%2FTS-0285\n",
      "https%3A%2F%2Fwww.genenames.org%2Fcgi-bin%2Fgene_symbol_report%3Fmatch%3DDAPK1\n",
      "http%3A%2F%2Fwww.pantherdb.org%2Fpathway%2FpathwayDiagram.jsp%3FcatAccession%3DP00024\n",
      "https%3A%2F%2Ffungidb.org%2Ffungidb%2Fapp%2Frecord%2Fgene%2FCNBG_0001\n",
      "http%3A%2F%2Fwww.flyrnai.org%2Fcgi-bin%2FRNAi_gene_lookup_public.pl%3Fgname%3DDRSC05221\n",
      "http%3A%2F%2Fdarcsite.genzentrum.lmu.de%2Fdarc%2Fview.php%3Fid%3D1250\n",
      "http%3A%2F%2Fcerevisiae.oridb.org%2Fdetails.php%3Fid%3D1\n",
      "http%3A%2F%2Fpombe.oridb.org%2Fdetails.php%3Fid%3D1\n",
      "http%3A%2F%2Fscop.mrc-lmb.cam.ac.uk%2Fscop%2Fsearch.cgi%3Fsunid%3D47419\n",
      "http%3A%2F%2Fscop.berkeley.edu%2Fsunid%3D47419\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FBN000065\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fnuccore%2FBN000065\n",
      "http%3A%2F%2Fidp1.force.cs.is.nagoya-u.ac.jp%2Fpscdb%2F051.html\n",
      "http%3A%2F%2Fwww.pantree.org%2Fnode%2FannotationNode.jsp%3Fid%3DPTN000000026\n",
      "http%3A%2F%2Forion.rnet.missouri.edu%2F~nz953%2FDOMMINO%2Findex.php%2Fresult%2Fshow_network%2F2GC4\n",
      "http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2FCCDS%2FCcdsBrowse.cgi%3FREQUEST%3DCCDS%26DATA%3DCCDS13573.1\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fgxa%2Fgenes%2FAT4G01080\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fgxa%2Fexperiments%2FE-MTAB-2037\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fatlas-experiments%2FE-MTAB-2037\n",
      "http%3A%2F%2Fwww.hprd.org%2Fprotein%2F00001\n",
      "ftp%3A%2F%2Fftp.ebi.ac.uk%2Fpub%2Fdatabases%2Flrgex%2FLRG_1.xml\n",
      "http%3A%2F%2Fwww.ensembl.org%2FHomo_sapiens%2FLRG%2FSummary%3Flrg%3DLRG_1\n",
      "http%3A%2F%2Fasia.ensembl.org%2FHomo_sapiens%2FLRG%2FSummary%3Flrg%3DLRG_1\n",
      "http%3A%2F%2Fuswest.ensembl.org%2FHomo_sapiens%2FLRG%2FSummary%3Flrg%3DLRG_1\n",
      "http%3A%2F%2Fuseast.ensembl.org%2FHomo_sapiens%2FLRG%2FSummary%3Flrg%3DLRG_1\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fmetabolights%2FMTBLS1\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fmetabolights_dataset%2FMTBLS1\n",
      "https%3A%2F%2Forcid.org%2F0000-0002-5355-2576\n",
      "http%3A%2F%2Fnbn-resolving.org%2Fresolver%3Fidentifier%3Durn%3Anbn%3Afi%3Atkk-004781%26verb%3Dredirect\n",
      "http%3A%2F%2Fkinase.bioinformatics.tw%2Fshowall.jsp%3Ftype%3DKinase%26info%3DGene%26name%3DAURKA%26drawing%3D0%26sorting%3D0%26kinome%3D1\n",
      "http%3A%2F%2Fwww.uniprot.org%2Funiprot%2FQ5BJF6-3\n",
      "http%3A%2F%2Fpurl.uniprot.org%2Funiprot%2FQ5BJF6-3\n",
      "http%3A%2F%2Fwww.uniprot.org%2Funiparc%2F%3Fquery%3DQ5BJF6-3\n",
      "http%3A%2F%2Fkinase.bioinformatics.tw%2Fshowall.jsp%3Ftype%3DPhosphoProtein%26info%3DGene%26name%3DAURKA%26drawing%3D0%26sorting%3D0%26kinome%3D0\n",
      "http%3A%2F%2Fwww.chemspider.com%2Finchikey%3DRYYVLZVUVIJVGH-UHFFFAOYSA-N\n",
      "http%3A%2F%2Fcactus.nci.nih.gov%2Fchemical%2Fstructure%2FRYYVLZVUVIJVGH-UHFFFAOYSA-N%2Fnames\n",
      "http%3A%2F%2Fwww.chemspider.com%2FInChI%3D1S%2FC2H6O%2Fc1-2-3%2Fh3H%2C2H2%2C1H3\n",
      "http%3A%2F%2Fwebbook.nist.gov%2Fcgi%2Fcbook.cgi%3FInChI%3D1S%2FC2H6O%2Fc1-2-3%2Fh3H%2C2H2%2C1H3\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fchebi%2FadvancedSearchFT.do%3FsearchString%3DInChI%3D1S%2FC2H6O%2Fc1-2-3%2Fh3H%2C2H2%2C1H3\n",
      "http%3A%2F%2Frdf.openmolecules.net%2F%3FInChI%3D1S%2FC2H6O%2Fc1-2-3%2Fh3H%2C2H2%2C1H3\n",
      "http%3A%2F%2Fen.wikipedia.org%2Fwiki%2FSM_UB-81\n",
      "http%3A%2F%2Fdbpedia.org%2Fpage%2FSM_UB-81\n",
      "http%3A%2F%2Fbioinformatics.hsanmartino.it%2Fhypercldb%2Fcl3603.html\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2Fev%3AE00032\n",
      "http%3A%2F%2Fbipaa.genouest.org%2Fapps%2Fgrs-2.3%2Fgrs%3FreportID%3Daphidbase_transcript_report%26objectID%3DACYPI000159\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fefo%2Fterms%3Fobo_id%3DEFO%3A0004859\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FEFO%2Fefo%3AEFO_0004859\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0004859\n",
      "http%3A%2F%2Fwww.hgmd.cf.ac.uk%2Fac%2Fgene.php%3Fgene%3DCALM1\n",
      "https%3A%2F%2Fwww.affymetrix.com%2FLinkServlet%3Fprobeset%3D243002_at\n",
      "http%3A%2F%2Fcu.affymetrix.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Faffymetrix%3A243002_at\n",
      "http%3A%2F%2Fwww.treefam.org%2Ffamily%2FTF101014\n",
      "http%3A%2F%2Fwww.bioinsilico.org%2Fcgi-bin%2FCAPSDB%2FgetCAPScluster%3Fnidcl%3D434\n",
      "http%3A%2F%2Fepsf.bmad.bii.a-star.edu.sg%2Fcube%2Fdb%2Fdata%2FAKR%2F\n",
      "http%3A%2F%2Fidp1.force.cs.is.nagoya-u.ac.jp%2FIDEAL%2FidealItem.php%3Fid%3DIID00001\n",
      "http%3A%2F%2Fwww.pocketome.org%2Ffiles%2F1433C_TOBAC_1_252.html\n",
      "http%3A%2F%2Fpsb.kobic.re.kr%2FSTAP%2Frefinement1%2Fresult.php%3Fsearch%3D1a24\n",
      "http%3A%2F%2Fgenomesonline.org%2Fcgi-bin%2FGOLD%2Fbin%2FGOLDCards.cgi%3Fgoldstamp%3DGm00047\n",
      "http%3A%2F%2Fbugs.sgul.ac.uk%2Fbugsbase%2Ftabs%2Fexperiment.php%3Fexpt_id%3D288%26action%3Dview\n",
      "http%3A%2F%2Fbugs.sgul.ac.uk%2Fbugsbase%2Ftabs%2Fprotocol.php%3Fprotocol_id%3D67%26amp%3Baction%3Dview\n",
      "http%3A%2F%2Fwww.variationontology.org%2Fcgi-bin%2Famivario%2Fterm-details.cgi%3Fterm%3DVariO%3A0294\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FVARIO%2FVariO%3A0294\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fvario%2Fterms%3Fobo_id%3DVariO%3A0294\n",
      "http%3A%2F%2Ftolweb.org%2F98034\n",
      "http%3A%2F%2Fwww.viprbrc.org%2Fbrc%2FviprStrainDetails.do%3FstrainName%3DBeAn%2070563%26decorator%3Darena\n",
      "http%3A%2F%2Fwww.fungalbarcoding.org%2FBioloMICS.aspx%3FTable%3DFungal%20barcodes%26Rec%3D2224%26Fields%3DAll%26ExactMatch%3DT\n",
      "https%3A%2F%2Frfam.xfam.org%2Ffamily%2FRF00230\n",
      "http%3A%2F%2Fepd.vital-it.ch%2Fcgi-bin%2Fquery_result.pl%3Fout_format%3DNICE%26Entry_0%3DTA_H3\n",
      "http%3A%2F%2Fwasabi.lutzonilab.net%2Fpub%2FdisplayTaxonInfo%3Faftol_id%3D959\n",
      "http%3A%2F%2Fwww.aspergillusgenome.org%2Fcgi-bin%2Flocus.pl%3Fdbid%3DASPL0000349247\n",
      "http%3A%2F%2Fwww.aspergillusgenome.org%2Fcgi-bin%2Fprotein%2FproteinPage.pl%3Fdbid%3DASPL0000349247\n",
      "http%3A%2F%2Fautism.mindspec.org%2FGeneDetail%2FADA\n",
      "http%3A%2F%2Fbacmap.wishartlab.com%2Fmaps%2FAP011135%2Findex.html\n",
      "http%3A%2F%2Fwww.hipaaspace.com%2FMedical_Billing%2FCoding%2FNational.Drug.Codes%2F0002-1975-61\n",
      "http%3A%2F%2Fndc.bio2rdf.org%2Fdescribe%2F%3Furl%3Dhttp%3A%2F%2Fbio2rdf.org%2Fndc%3A0002-1975-61\n",
      "https%3A%2F%2Fbgee.org%2F%3Fpage%3Dgene%26gene_id%3DFBgn0000015\n",
      "http%3A%2F%2Fbgee.unil.ch%2Fbgee%2Fbgee%3Fpage%3Danatomy%26action%3Dorgans%26stage_id%3DHsapDO%3A0000004%26stage_children%3Don\n",
      "http%3A%2F%2Fbgee.unil.ch%2Fbgee%2Fbgee%3Fpage%3Dgene_family%26action%3Dfamily_details%26gene_family_id%3DENSFM00500000270089\n",
      "http%3A%2F%2Fbgee.unil.ch%2Fbgee%2Fbgee%3Fpage%3Danatomy%26action%3Dorgan_details%26organ_id%3DEHDAA%3A2185%26organ_children%3Don\n",
      "https%3A%2F%2Fcgap.nci.nih.gov%2FPathways%2FBioCarta%2Fh_aktPathway\n",
      "http%3A%2F%2Fgmd.mpimp-golm.mpg.de%2FSpectrums%2F53d583d8-40c6-40e1-9296-23f821cd77a5\n",
      "http%3A%2F%2Fgmd.mpimp-golm.mpg.de%2Fprofile%2Fdefault.aspx%3FXemlId%3D10b38aaf-b977-4950-85b8-f4775f66658d\n",
      "http%3A%2F%2Fgmd.mpimp-golm.mpg.de%2FReferenceSubstances%2F8cf84adb-b4db-4807-ac98-0004247c35df\n",
      "http%3A%2F%2Fgmd.mpimp-golm.mpg.de%2FAnalytes%2F4f0fa9b6-514f-4ff4-98cc-0009bc08eb80\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fintact%2Fmolecule%2FEBI-366083\n",
      "https%3A%2F%2Fdailymed.nlm.nih.gov%2Fdailymed%2FdrugInfo.cfm%3Fsetid%3D973a9333-fec7-46dd-8eb5-25738f06ee54\n",
      "http%3A%2F%2Fwww.depod.bioss.uni-freiburg.de%2Fshowp.php%3Fgene%3DPTPN1\n",
      "http%3A%2F%2Fwww.phytozome.net%2FgenePage.php%3Fcrown%26method%3D0%26search%3D1%26detail%3D1%26searchText%3Dlocusname%3AGlyma0021s00410\n",
      "http%3A%2F%2Fwww.cellsignal.com%2Freference%2Fpathway%2FAkt_PKB.html\n",
      "http%3A%2F%2Fgenolist.pasteur.fr%2FSubtiList%2Fgenome.cgi%3Fexternal_query%2BBG11523\n",
      "http%3A%2F%2Fsideeffects.embl.de%2Fse%2FC0017565%2F\n",
      "http%3A%2F%2Fsideeffects.embl.de%2Fdrugs%2F2244%2F\n",
      "https%3A%2F%2Fwww.broadinstitute.org%2Fannotation%2Fgenome%2Fmagnaporthe_grisea%2FGeneDetails.html%3Fsp%3DS7000002168151102\n",
      "http%3A%2F%2Fccr.coriell.org%2FSections%2FSearch%2FSample_Detail.aspx%3FRef%3DGM17027\n",
      "http%3A%2F%2Fwww.wikigenes.org%2Fe%2Fgene%2Fe%2F3771877.html\n",
      "https%3A%2F%2Fmips.helmholtz-muenchen.de%2Fcorum%2F%3Fid%3D100\n",
      "http%3A%2F%2Fecoliwiki.net%2Fcolipedia%2Findex.php%2FaaeA%3AGene\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMA%2FMA%3A0002502\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fma%2Fterms%3Fobo_id%3DMA%3A0002502\n",
      "http%3A%2F%2Fwww.informatics.jax.org%2Fsearches%2FAMA.cgi%3Fid%3DMA%3A0002502\n",
      "http%3A%2F%2Fwww.jstor.org%2Fstable%2F3075966\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Finterpro%2Fgenomeproperties%2F%23GenProp0699\n",
      "http%3A%2F%2Fwww.unimod.org%2Fmodifications_view.php%3Feditid1%3D1200\n",
      "http%3A%2F%2Fvbrc.org%2Fgene_detail.asp%3Fgene_id%3D35742\n",
      "http%3A%2F%2Fviralzone.expasy.org%2Fall_by_protein%2F992.html\n",
      "http%3A%2F%2Fcsbl.bmb.uga.edu%2FDOOR%2Foperon.php%3Fid%3D1398574\n",
      "http%3A%2F%2Fdegradome.uniovi.es%2Fcgi-bin%2Fprotease%2FAx1\n",
      "http%3A%2F%2Fplanttfdb.cbi.pku.edu.cn%2Ftf.php%3Fsp%3DAth%26did%3DAT1G01030.1\n",
      "http%3A%2F%2Fwww.transcriptionfactor.org%2Findex.cgi%3FSearch%2FDomain%2Bdomain%3A0045310%2Bcat%3ADBD\n",
      "http%3A%2F%2Fwww.molbase.com%2Fen%2Findex.php%3Fapp%3Dsearch%26search_keyword%3D128796-39-4\n",
      "http%3A%2F%2Fyeastrc.org%2Fpdr%2FviewProtein.do%3Fid%3D2673500\n",
      "http%3A%2F%2Ffunc.mshri.on.ca%2Ffly%2Fgenes%2Flist_functional_scores%2F10194\n",
      "http%3A%2F%2Ffunc.mshri.on.ca%2Fhuman%2Fgenes%2Flist_functional_scores%2F119514\n",
      "http%3A%2F%2Fmetarray.ucsc.edu%2Fcgi-bin%2Fintron%2FyirIntrondb%3ForfName%3DSNR17A\n",
      "http%3A%2F%2Ffunc.mshri.on.ca%2Fmouse%2Fgenes%2Flist_functional_scores%2F1351341\n",
      "http%3A%2F%2Fwww.cbioportal.org%2Fstudy%3Fid%3Dlaml_tcga_pub%23summary\n",
      "http%3A%2F%2Ffunc.mshri.on.ca%2Fyeast%2Fgenes%2Flist_functional_scores%2F2701\n",
      "http%3A%2F%2Fwww-deletion.stanford.edu%2Fcgi-bin%2FYDPM%2FYDPM_search.cgi%3Fthelist%3DYAL001C\n",
      "https%3A%2F%2Fwww.wormbase.org%2Fspecies%2Fc_elegans%2Frnai%2FWBRNAi00086878\n",
      "http%3A%2F%2Fcaps.ncbs.res.in%2Fcgi-bin%2Fpass2%2F%2Fshow_sf.py%3Fsf_id%3D46977\n",
      "http%3A%2F%2Fdb-mml.sjtu.edu.cn%2FICEberg%2Ffeature_page.php%3Fice_id%3D100\n",
      "http%3A%2F%2Fdb-mml.sjtu.edu.cn%2FICEberg%2Fbrowse_result.php%3Ftype%3Dfam%26fam_id%3D1\n",
      "http%3A%2F%2Fwww.mgc.ac.cn%2Fcgi-bin%2FVFs%2Fgene.cgi%3FGeneID%3DVFG2154\n",
      "http%3A%2F%2Fwww.mgc.ac.cn%2Fcgi-bin%2FVFs%2Fgenus.cgi%3FGenus%3DChlamydia\n",
      "http%3A%2F%2Fwww.nlm.nih.gov%2Fcgi%2Fmesh%2F2013%2FMB_cgi%3Fmode%3D%26index%3D17165%26view%3Dexpanded\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2FM00002\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2FH00076\n",
      "http%3A%2F%2Fligandbox.protein.osaka-u.ac.jp%2Fligandbox%2Fcgi-bin%2Fliginf.cgi%3Fid%3DD00001\n",
      "https%3A%2F%2Fwww.glycoepitope.jp%2Fepitopes%2FEP0311\n",
      "http%3A%2F%2Fwww.nlm.nih.gov%2Fmedlineplus%2Fency%2Farticle%2F002804.htm\n",
      "http%3A%2F%2Fjcggdb.jp%2Fidb%2Fjcggdb%2FJCGG-STR008690\n",
      "http%3A%2F%2Fwww.bioinfo.org%2FNONCODEv4%2Fshow_rna.php%3Fid%3DNONHSAT000001\n",
      "http%3A%2F%2Fvirtualflybrain.org%2Freports%2F00000001\n",
      "http%3A%2F%2Fwww.bioinfo.org%2FNONCODEv4%2Fshow_gene.php%3Fid%3DNONHSAG00001\n",
      "http%3A%2F%2Fwww.shigen.nig.ac.jp%2Frice%2ForyzabaseV4%2Fgene%2Fdetail%2F117\n",
      "http%3A%2F%2Fwww.shigen.nig.ac.jp%2Frice%2ForyzabaseV4%2Fstrain%2FinducedMutationLine%2Fdetail%2F21393\n",
      "http%3A%2F%2Foryzatagline.cirad.fr%2Fcgi-bin%2Fgeneral_mutant.pl%3Fline%3DAADB12\n",
      "http%3A%2F%2Fwww.shigen.nig.ac.jp%2Frice%2ForyzabaseV4%2Fstrain%2FwildCore%2Fdetail%2F1\n",
      "http%3A%2F%2Fwww.shigen.nig.ac.jp%2Frice%2ForyzabaseV4%2Fdevstageineachorgan%2Fdetail%2F34\n",
      "http%3A%2F%2Fpax-db.org%2F%23%21protein%2F977869\n",
      "http%3A%2F%2Fwww.rcsb.org%2Fpdb%2Fligand%2Fligandsummary.do%3FhetId%3DTRQ\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fpdbe-srv%2Fpdbechem%2FchemicalCompound%2Fshow%2FTRQ\n",
      "http%3A%2F%2Fmerops.sanger.ac.uk%2Fcgi-bin%2Fpepsum%3Fid%3DI31.952\n",
      "http%3A%2F%2Fpax-db.org%2F%23%21species%2F9606\n",
      "http%3A%2F%2Fpharminfo.pharm.kyoto-u.ac.jp%2Fservices%2Fglida%2Fgpcr_information.php%3Fid%3DACM1_HUMAN\n",
      "https%3A%2F%2Fanimaldiversity.org%2Faccounts%2FLycalopex_vetulus%2F\n",
      "http%3A%2F%2Fbioinformatics.ramapo.edu%2FGRSDB2%2Fgeneview.php%3FgeneID%3D10142\n",
      "http%3A%2F%2Fpharminfo.pharm.kyoto-u.ac.jp%2Fservices%2Fglida%2Fligand_information.php%3Fid%3DL000001\n",
      "http%3A%2F%2Fcompbio.cs.queensu.ca%2Fcgi-bin%2Fcompbio%2Fsearch%2Fmain.cgi%3Fsearch_mode%3Did%26amp%3Bid_type%3Dsnp_id%26amp%3Bid_val%3Drs17852708\n",
      "http%3A%2F%2Fproteomecentral.proteomexchange.org%2Fcgi%2FGetDataset%3FID%3DPXD000500\n",
      "http%3A%2F%2Fresearch.nhgri.nih.gov%2Fapps%2Fhomeodomain%2Fweb%2Findex.cgi%3Fmode%3Dview%26amp%3Bview%3Dproteins%26amp%3Bid%3D63\n",
      "http%3A%2F%2Fbiomodels.net%2Frdf%2Fvocabulary.rdf%23rateRule\n",
      "http%3A%2F%2Fbioinfo.lifl.fr%2Fnorine%2Fresult.jsp%3FID%3DNOR00681\n",
      "http%3A%2F%2Fsenselab.med.yale.edu%2FORDB%2FData%2F8497\n",
      "http%3A%2F%2Fwww.p3db.org%2Fphosphosite.php%3Fid%3D65%26ref%3D0\n",
      "http%3A%2F%2Fsenselab.med.yale.edu%2FOdorDB%2FData%2F74%2F%3Fdb%3D5\n",
      "http%3A%2F%2Fwww.animalgenome.org%2Fcgi-bin%2FQTLdb%2FBT%2Fqdetails%3FQTL_ID%3D4685\n",
      "http%3A%2F%2Fwww.p3db.org%2Fprotein.php%3Fid%3D70%26amp%3Bref%3D0\n",
      "http%3A%2F%2Ftopdb.enzim.hu%2F%3Fm%3Dshow%26id%3DAP00378\n",
      "http%3A%2F%2Fwww.animalgenome.org%2Fcgi-bin%2FQTLdb%2FGG%2Fqdetails%3FQTL_ID%3D14362\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fmetagenomics%2Fsamples%2FSRS086444\n",
      "http%3A%2F%2Fwww.animalgenome.org%2Fcgi-bin%2FQTLdb%2FOA%2Fqdetails%3FQTL_ID%3D19803\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fega%2Fstudies%2FEGAS00000000001\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fega%2FEGAS00000000001\n",
      "http%3A%2F%2Fwww.animalgenome.org%2Fcgi-bin%2FQTLdb%2FSS%2Fqdetails%3FQTL_ID%3D14\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fega%2Fdatasets%2FEGAD00000000001\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fega%2FEGAD00000000001\n",
      "https%3A%2F%2Fbiomodels.net%2Fvocab%2Fidot.rdf%23identifierPattern\n",
      "http%3A%2F%2Fantibodyregistry.org%2FAB_493771\n",
      "http%3A%2F%2Fbioportal.bioontology.org%2Fontologies%2FMAMO%2F%3Fp%3Dclasses%26conceptid%3Dhttp%3A%2F%2Fidentifiers.org%2Fmamo%2FMAMO_0000026\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fmamo%2Fterms%3Fshort_form%3DMAMO_0000026\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fpride%2Farchive%2Fprojects%2FPXD000440\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fpride%2FPXD000440\n",
      "http%3A%2F%2Fintron.ucsc.edu%2Fcgi-bin%2Fyeast4.3%2Fintronreports.pl%3Foutputformat%3Dfull%26amp%3BorfName%3DSNR17A\n",
      "https%3A%2F%2Fwww.genenames.org%2Fgenefamilies%2FPADI\n",
      "http%3A%2F%2Fardb.cbcb.umd.edu%2Fcgi%2Fsearch.cgi%3Fdb%3DL%26amp%3Bfield%3Dni%26amp%3Bterm%3DCAE46076\n",
      "https%3A%2F%2Fwww.proteomicsdb.org%2F%23human%2FproteinDetails%2F53504%2Fsummary\n",
      "http%3A%2F%2Fwww.humanproteomemap.org%2Fprotein.php%3Fhpm_id%3D1968\n",
      "https%3A%2F%2Fwww.proteomicsdb.org%2F%23human%2FproteinDetails%2F53504%2Fpeptides%2F\n",
      "http%3A%2F%2Fwww.drugbank.ca%2Fbiodb%2Fbio_entities%2FBE0000048\n",
      "http%3A%2F%2Ffoodb.ca%2Fcompounds%2FFDB002100\n",
      "http%3A%2F%2Fwww.humanproteomemap.org%2Fspectrum.php%3Fpep_id%3D9606117\n",
      "http%3A%2F%2Fzinc15.docking.org%2Fsubstances%2FZINC1084\n",
      "http%3A%2F%2Ffdasis.nlm.nih.gov%2Fsrs%2Fsrsdirect.jsp%3Fregno%3D3G6A5W338E\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fordo%2Fterms%3Fshort_form%3DOrphanet_C023\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fontology-lookup%2F%3FtermId%3DPAR%3A0116\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fmetagenomics%2Fprojects%2FERP004492\n",
      "http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fclinvar%2FRCV000033555.3%2F\n",
      "https%3A%2F%2Fwww.clinicaltrialsregister.eu%2Fctr-search%2Fsearch%3Fquery%3D2008-005144-16\n",
      "https%3A%2F%2Fwww.google.com%2Fpatents%2FUS4145692\n",
      "http%3A%2F%2Fpatft.uspto.gov%2Fnetacgi%2Fnph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D%2Fnetahtml%2FPTO%2Fsearch-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN%2F4145692\n",
      "http%3A%2F%2Fexac.broadinstitute.org%2Fvariant%2F22-46615880-T-C\n",
      "https%3A%2F%2Fworldwide.espacenet.com%2Fclassification%3Flocale%3Den_EP%23%21%2FCPC%3DA01M1%2F026\n",
      "https%3A%2F%2Fwww.gwascentral.org%2Fstudy%2FHGVST1828\n",
      "https%3A%2F%2Fwww.gwascentral.org%2Fmarker%2FHGVM15354\n",
      "http%3A%2F%2Fbigg.ucsd.edu%2Fmodels%2Funiversal%2Fmetabolites%2F12dgr161\n",
      "http%3A%2F%2Flincsportal.ccs.miami.edu%2Fcells%2F%23%2Fview%2FLCL-2085\n",
      "http%3A%2F%2Flincs.hms.harvard.edu%2Fdb%2Fproteins%2F200282%2F\n",
      "http%3A%2F%2Flincsportal.ccs.miami.edu%2FSmallMolecules%2F%23%2Fview%2FLSM-6306\n",
      "http%3A%2F%2Fexac.broadinstitute.org%2Ftranscript%2FENST00000407236\n",
      "http%3A%2F%2Fexac.broadinstitute.org%2Fgene%2FENSG00000169174\n",
      "https%3A%2F%2Fwww.wikidata.org%2Fwiki%2FQ2207226\n",
      "http%3A%2F%2Fwww.grenoble.prabi.fr%2Fobiwarehouse%2Funipathway%2Fupc%3Fupid%3DUPC04349\n",
      "http%3A%2F%2Fbigg.ucsd.edu%2Fmodels%2FiECABU_c1320\n",
      "https%3A%2F%2Fregistry.identifiers.org%2Fdeprecation%2Fresources%2FMIR%3A00100729%2FBiotin_biosynthesis\n",
      "http%3A%2F%2Fmodelseed.org%2Fbiochem%2Fcompounds%2Fcpd15380\n",
      "http%3A%2F%2Fbigg.ucsd.edu%2Fcompartments%2Fc\n",
      "http%3A%2F%2Fbigg.ucsd.edu%2Fmodels%2Funiversal%2Freactions%2F13GS\n",
      "http%3A%2F%2Flinkedlifedata.com%2Fresource%2Fumls%2Fid%2FC0021390\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fpdbe%2Fentry%2Femdb%2FEMD-1001\n",
      "http%3A%2F%2Fid.nlm.nih.gov%2Fmesh%2FC000100\n",
      "http%3A%2F%2Fmirtarbase.mbc.nctu.edu.tw%2Fphp%2Fdetail.php%3Fmirtid%3DMIRT000002\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMEDDRA%2F10015919\n",
      "http%3A%2F%2Flisanwanglab.org%2FDASHR%2Fentry%2Fhsa-mir-200a\n",
      "http%3A%2F%2Fmona.fiehnlab.ucdavis.edu%2F%23%2Fspectra%2Fsplash%2Fsplash10-0zq2000000-77302b0326a418630a84\n",
      "https%3A%2F%2Fdashr1.lisanwanglab.org%2Fentry%2Fhsa-mir-200a%23hsa-mir-200a%23exprPerTissueTable\n",
      "http%3A%2F%2Fwww.grenoble.prabi.fr%2Fobiwarehouse%2Funipathway%2Fucr%3Fupid%3DUCR00226\n",
      "http%3A%2F%2Fwww.sasbdb.org%2Fdata%2FSASDAX8\n",
      "http%3A%2F%2Fcompbio.charite.de%2Fhpoweb%2Fshowterm%3Fid%3DHP%3A0000118\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fhp%2Fterms%3Fobo_id%3DHP%3A0000118\n",
      "https%3A%2F%2Fwww.genenames.org%2Fcgi-bin%2Fgenefamilies%2Fset%2F141\n",
      "https%3A%2F%2Fmedical-data-models.org%2Fforms%2F4776\n",
      "http%3A%2F%2Fcicblade.dep.usal.es%3A8080%2FAPID%2FInteractions.action%3Fprotein%3DP01116\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fprobonto%2Fterms%3Fshort_form%3DPROB_c0000005\n",
      "https%3A%2F%2Fwww.storedb.org%2F%3FSTOREDB%3ASTUDY1040\n",
      "http%3A%2F%2Fwww.kegg.jp%2Fentry%2Fhsa00190\n",
      "http%3A%2F%2Fwww.informatics.jax.org%2Fsearches%2FPhat.cgi%3Fid%3DMP%3A0005452\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fmp%2Fterms%3Fobo_id%3DMP%3A0005452\n",
      "https%3A%2F%2Fbioportal.bioontology.org%2Fontologies%2FMP%2FMP%3A0005452\n",
      "http%3A%2F%2Fplanttfdb.cbi.pku.edu.cn%2Ftf.php%3Fuid%3DAth_AT1G01030.1\n",
      "http%3A%2F%2Fmulticellds.org%2FMultiCellDB%2FMCDS_C_0000000001\n",
      "http%3A%2F%2Fmulticellds.org%2FMultiCellDB%2FMCDS_L_0000000001\n",
      "http%3A%2F%2Fmulticellds.org%2FMultiCellDB%2FMCDS_S_0000000001\n",
      "http%3A%2F%2Fe-cyanobacterium.org%2Fbcs%2Fentity%2F23%2F\n",
      "http%3A%2F%2Flincsportal.ccs.miami.edu%2Fdatasets%2F%23%2Fview%2FLDS-1110\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Flincs%2FLDS-1110\n",
      "http%3A%2F%2Fe-cyanobacterium.org%2Fmodels%2Fmodel%2F26%2F\n",
      "http%3A%2F%2Fe-cyanobacterium.org%2Fbcs%2Frule%2F56%2F\n",
      "http%3A%2F%2Fddr.nal.usda.gov%2Fdspace%2Fhandle%2F10113%2F50018\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fms%2Fterms%3Fobo_id%3DMS%3A1000001\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMS%2FMS%3A1000001\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fenvo%2Fterms%3Fobo_id%3DENVO%3A09200010\n",
      "http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FENVO%2FENVO%3A09200010\n",
      "https%3A%2F%2Fwww.grid.ac%2Finstitutes%2Fgrid.225360.0\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fmedgen%2F760050\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fclinvar%2Fvariation%2F12345\n",
      "http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fclinvar%3Fterm%3DSCV000151292\n",
      "https%3A%2F%2Fregistry.identifiers.org%2Fregistry%3Fquery%3D%22MIR%3A00100037%22\n",
      "http%3A%2F%2Fprogenetix.org%2Fpgx%3Aicdom%3A8500_3\n",
      "http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000030\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fprojects%2Fgap%2Fcgi-bin%2Fstudy.cgi%3Fstudy_id%3Dphs000768.v2.p1\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fomit%2Fterms%3Fshort_form%3DOMIT_0005506\n",
      "http%3A%2F%2Fgetentry.ddbj.nig.ac.jp%2Fgetentry%2Fdad%2FAAA35559\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FAAA35559\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fprotein%2FAAA35559\n",
      "https%3A%2F%2Fportal.gdc.cancer.gov%2Fcases%2Fae8c77fe-e6c8-44d5-8265-4a38c637bbef\n",
      "https%3A%2F%2Fmetacyc.org%2FMETA%2FNEW-IMAGE%3Ftype%3DREACTION%26object%3DRXN-14904\n",
      "https%3A%2F%2Fglytoucan.org%2FStructures%2FGlycans%2FG00054MO\n",
      "https%3A%2F%2Fmetacyc.org%2Fcompound%3Forgid%3DMETA%26id%3DCPD-10330\n",
      "https%3A%2F%2Fcn.dataone.org%2Fcn%2Fv2%2Fresolve%2F%7B00030692-0FE1-4A1B-955E-A2E55D659267%7D\n",
      "https%3A%2F%2Fmmp.sfb.uit.no%2Fdatabases%2Fmarref%2F%23%2Frecords%2FMMP3312982.2\n",
      "http%3A%2F%2Fwww.cathdb.info%2Fcathnode%2F1.10.8.10\n",
      "https%3A%2F%2Fmmp.sfb.uit.no%2Fdatabases%2Fmarcat%2F%23%2Frecords%2FMMP743597.11\n",
      "https%3A%2F%2Fmmp.sfb.uit.no%2Fdatabases%2Fmardb%2F%23%2Frecords%2FMMP02954345.1\n",
      "https%3A%2F%2Faopwiki.org%2Faops%2F98\n",
      "https%3A%2F%2Fbio.tools%2Funiprotkb\n",
      "https%3A%2F%2Fgenelab-data.ndc.nasa.gov%2Fgenelab%2Faccession%2FGLDS-141\n",
      "https%3A%2F%2Fwww.arraymap.org%2Fpgx%3Aicdom%3A8500_3\n",
      "http%3A%2F%2Fwww.begdb.com%2Findex.php%3Faction%3DoneMolecule%26state%3Dshow%26id%3D4214\n",
      "https%3A%2F%2Fherv.img.cas.cz%2Fs%2FTHE1B\n",
      "https%3A%2F%2Frepo.napdi.org%2Fstudy%2F28\n",
      "http%3A%2F%2Fcrispr.i2bc.paris-saclay.fr%2Fcgi-bin%2Fcrispr%2FSpecieProperties_db.cgi%3FTaxon_id%5B%5D%3D551115\n",
      "https%3A%2F%2Fdb.systemsbiology.net%2Fsbeams%2Fcgi%2FPeptideAtlas%2FShowObservedSpectrum%3Fusi%3Dmzspec%3APXD002255%3A%3AES_XP_Ubi_97H_HCD_349%3Ascan%3A9617%3ALAEIYVNSSFYK%2F2\n",
      "http%3A%2F%2Fsugarbind.expasy.org%2Flectins%2F172\n",
      "http%3A%2F%2Fsearch.sisuproject.fi%2F%23%2Fvariant%2F17%3A41223048\n",
      "https%3A%2F%2Flicebase.org%2F%3Fq%3DEMLSAT00000003403\n",
      "https%3A%2F%2Furgi.versailles.inra.fr%2Fgnpis%2F%23result%2Fterm%3DAY109603\n",
      "https%3A%2F%2Frnacentral.org%2Frna%2FURS0000759CF4\n",
      "http%3A%2F%2Fwww.genoscope.cns.fr%2Fagc%2Fmicroscope%2Fmage%2Finfo.php%3Fid%3D5601141\n",
      "http%3A%2F%2Fswissregulon.unibas.ch%2Fquery%2FAHR\n",
      "https%3A%2F%2Fwebchem.ncbr.muni.cz%2FPlatform%2FValidatorDb%2FByStructure%2F2h6o\n",
      "https%3A%2F%2Fdatalab.rwth-aachen.de%2FMINDER%2Fresource%2Faef4c195-9cf9-46db-a12a-7cfd1ff3eec3\n",
      "http%3A%2F%2Fvmh.uni.lu%2F%23metabolite%2Fh2o\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fnmrcv%2Fterms%3Fshort_form%3DNMR%3A1000003\n",
      "https%3A%2F%2Fvmh.uni.lu%2F%23reaction%2FHEX1\n",
      "http%3A%2F%2Fascl.net%2F1801.012\n",
      "https%3A%2F%2Fwww.facebase.org%2Fdata%2Frecord%2F%231%2Fisa%3Adataset%2Faccession%3DFB00000917\n",
      "http%3A%2F%2Fdata.europa.eu%2F89h%2Fjrc-eurl-ecvam-chemagora\n",
      "https%3A%2F%2Fgudmap.org%2Fid%2FQ-2958\n",
      "http%3A%2F%2Fn2t.net%2Fark%3A%2F57799%2Fb97957\n",
      "https%3A%2F%2Fwww.metabolome-express.org%2Fdatasetview.php%3Fdatasetid%3D36\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fmetabolome_express%2FMEX36\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fmetabolomics_workbench%2FST000900\n",
      "http%3A%2F%2Fwww.metabolomicsworkbench.org%2Fdata%2FDRCCMetadata.php%3FMode%3DStudy%26StudyID%3DST000900\n",
      "http%3A%2F%2Fgpmdb.thegpm.org%2F~%2Fdblist_gpmnum%2Fgpmnum%3DGPM32310002988\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fgpmdb%2FGPM32310002988\n",
      "http%3A%2F%2Fwww.metabolomicsworkbench.org%2Fdata%2FDRCCMetadata.php%3FMode%3DProject%26ProjectID%3DPR000001\n",
      "https%3A%2F%2Fmassive.ucsd.edu%2FProteoSAFe%2FQueryMSV%3Fid%3DMSV000082131\n",
      "https%3A%2F%2Fwww.omicsdi.org%2Fdataset%2Fmassive%2FMSV000082131\n",
      "http%3A%2F%2Fbioportal.bioontology.org%2Fontologies%2FFPLX%2F%3Fp%3Dclasses%26conceptid%3Dhttp%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FFPLX%2523_RAS\n",
      "https%3A%2F%2Faopwiki.org%2Fevents%2F3\n",
      "https%3A%2F%2Faopwiki.org%2Fstressors%2F9\n",
      "https%3A%2F%2Faopwiki.org%2Frelationships%2F5\n",
      "https%3A%2F%2Farchive.softwareheritage.org%2Fbrowse%2Fswh%3A1%3Arev%3A309cf2674ee7a0749978cf8265ab91a60aea0f7d\n",
      "https%3A%2F%2Fdos-gdc.ucsc-cgp-dev.org%2Fga4gh%2Fdos%2Fv1%2Fdataobjects%2F23fa7b4b-9d68-429b-aece-658b11124bb3\n",
      "https%3A%2F%2Fcomptox.epa.gov%2Fdashboard%2FDTXSID2021028\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fcomplexportal%2Fcomplex%2FCPX-2158\n",
      "https%3A%2F%2Fligandbook.org%2Fpackage%2F785\n",
      "https%3A%2F%2Fwww.rebuildingakidney.org%2Fid%2FQ-2958\n",
      "http%3A%2F%2Fregistry.api.hq.identifiers.org%2FsemanticApi%2FgetRegistryOntology%23DataCollection\n",
      "https%3A%2F%2Fw3id.org%2Foc%2Foci%2F1-18\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Ffoodon%2Fterms%3Fobo_id%3DFOODON%3A03307879\n",
      "http%3A%2F%2Fbis.zju.edu.cn%2Fricenetdb%2Fgenedetails.php%3FID%3DLOC_Os01g49190.1\n",
      "http%3A%2F%2Fbis.zju.edu.cn%2Fricenetdb%2Fproteindetails.php%3FID%3DLOC_Os01g49190\n",
      "https%3A%2F%2Fwww.gtexportal.org%2Fhome%2Fgene%2FBRIP1\n",
      "http%3A%2F%2Fbis.zju.edu.cn%2Fricenetdb%2Fcompounddetails.php%3FID%3DOSC1416\n",
      "http%3A%2F%2Fbis.zju.edu.cn%2Fricenetdb%2Freactiondetails.php%3FID%3DOSR0818\n",
      "http%3A%2F%2Fbis.zju.edu.cn%2Fricenetdb%2FmiRNAdetails.php%3FID%3Dosa-miR446\n",
      "https%3A%2F%2Famzn.com%2F0471491039\n",
      "https%3A%2F%2Fwww.worldcat.org%2Foclc%2F634515043\n",
      "https%3A%2F%2Fdataguids.org%2Fga4gh%2Fdos%2Fv1%2Fdataobjects%2Fdg.4503%2F01b048d0-e128-4cb0-94e9-b2d2cab7563d\n",
      "http%3A%2F%2Fwww.oid-info.com%2Fcgi-bin%2Fdisplay%3Foid%3D2.16.840%26submit%3DDisplay%26action%3Ddisplay\n",
      "https%3A%2F%2Fflowrepository.org%2Fid%2FFR-FCM-ZYGW\n",
      "http%3A%2F%2Faddgene.org%2F50943\n",
      "https%3A%2F%2Fwww.cancerrxgene.org%2Ftranslation%2FDrug%2F1242\n",
      "https%3A%2F%2Fbacdive.dsmz.de%2Fstrain%2F131392\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fols%2Fontologies%2Fido%2Fterms%3Fobo_id%3DIDO%3A0000504\n",
      "http%3A%2F%2Fwww.isni.org%2Fisni%2F000000012281955X\n",
      "http%3A%2F%2Fviaf.org%2Fviaf%2F75121530\n",
      "http%3A%2F%2Fsabiork.h-its.org%2FnewSearch%3Fq%3D75\n",
      "https%3A%2F%2Fspdx.org%2Flicenses%2FCC-BY-1.0\n",
      "http%3A%2F%2Fwww.peptideatlas.org%2FPASS%2FPASS01237\n",
      "https%3A%2F%2Funiresolver.io%2F%23did%3Asov%3AWRfXPg8dantKVubE3HX8pw\n",
      "https%3A%2F%2Fshigen.nig.ac.jp%2Frice%2Foryzabase%2Freference%2Fdetail%2F42840\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fbiostudies%2Fstudies%2FS-EPMC6266652\n",
      "http%3A%2F%2Fmodelseed.org%2Fbiochem%2Freactions%2Frxn00001\n",
      "https%3A%2F%2Fw3id.org%2Foc%2Fcorpus%2Fbr%2F1\n",
      "https%3A%2F%2Fcadsrapi.nci.nih.gov%2Fcadsrapi4%2FGetXML%3Fquery%3DDataElement%5B%40publicId%3D3771992%5D\n",
      "https%3A%2F%2Fnanocommons.github.io%2Fidentifiers%2Fregistry%23ERM00000044\n",
      "http%3A%2F%2Fclassyfire.wishartlab.com%2Ftax_nodes%2FC0004828\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fgwas%2Fstudies%2FGCST000035\n",
      "https%3A%2F%2Fmmp.sfb.uit.no%2Fdatabases%2Fmarfun%2F%23%2Frecords%2FMMP3888430\n",
      "https%3A%2F%2Fwww.e-cyanobacterium.org%2Fexperiments-repository%2F%3Faction%3Ddetail%26id%3D18\n",
      "https%3A%2F%2Fdoulix.com%2Fbiomodules%2F6VDC956%2F\n",
      "https%3A%2F%2Fdoulix.com%2Fconstructs%2FM77F7JM%2F\n",
      "https%3A%2F%2Fwww.cameo3d.org%2Fsp%2Ftargets%2Ftarget%2F2019-08-03_00000089_1\n",
      "https%3A%2F%2Fassets.nemoarchive.org%2Fsmp-m3w9hbe\n",
      "https%3A%2F%2Fwww.ebi.ac.uk%2Fena%2Fdata%2Fview%2FGCA_000155495.1\n",
      "https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fassembly%2FGCA_000155495.1\n",
      "https%3A%2F%2Fwww.signalingpathways.org%2Fdatasets%2Fdataset.jsf%3Fdoi%3D10.1621%2FvwN2g2HaX3\n"
     ]
    }
   ],
   "source": [
    "import urllib.parse\n",
    "\n",
    "print(\"\\n\".join([urllib.parse.quote(x, safe=\"\") for x in namespace_provider_urls]))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
